<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Michael C. Saul (michael.saul [at] jax.org)" />


<title>DO Drug-Naïve Expression Summarize</title>

<script src="site_libs/header-attrs-2.17/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Diversity Outbred Reference Traits Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">DO Drug-Naïve Expression Summarize</h1>
<h4 class="author">Michael C. Saul (michael.saul [at] jax.org)</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-03-29
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rta/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20200501code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20200501)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20200501code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20200501)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongbd41812">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong> bd41812
</a>
</p>
</div>
<div id="strongRepositoryversionstrongbd41812"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version bd41812.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    code/.Rapp.history
    Ignored:    data/.Rhistory
    Ignored:    data/RDS/
    Ignored:    run/

Untracked files:
    Untracked:  LICENSE
    Untracked:  Rplot.pdf
    Untracked:  SENSI_chr7_snps.R
    Untracked:  Untitled.R
    Untracked:  analysis/17_mapping_interpretation_old.Rmd
    Untracked:  analysis/temp.txt
    Untracked:  code/DO_IVSA_SALINE_pleiotropy.R
    Untracked:  code/DO_IVSA_SENS_pleiotropy.R
    Untracked:  code/DO_SENS_SALINE_pleiotropy.R
    Untracked:  code/DO_eQTL_perms.R
    Untracked:  code/DO_eQTL_score_calc.R
    Untracked:  code/SENSI_chr7_snps.R
    Untracked:  code/STAR_RSEM_416.sh
    Untracked:  code/eQTL_plotting_DO_striatum_416.R
    Untracked:  code/ggplot_pub_themes.R
    Untracked:  code/pleitropy_analysis.sbatch
    Untracked:  code/pleitropy_saline_analysis.sbatch
    Untracked:  code/pleitropy_sensi_saline_analysis.sbatch
    Untracked:  code/rta_mapping_figure_leftover.R
    Untracked:  data/20_elastic_mv_results.csv
    Untracked:  data/20_elasticnet_predictions.RDS
    Untracked:  data/DO_416_DGEList.RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 3).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 4).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_DGEList_unfiltered.RDS
    Untracked:  data/DO_416_ERCC_DGEList.RDS
    Untracked:  data/DO_416_ERCC_cpm.RDS
    Untracked:  data/DO_416_normrt.RDS
    Untracked:  data/DO_416_pheno.RDS
    Untracked:  data/DO_416_pheno_allgenes.RDS
    Untracked:  data/DO_416_voom (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom.RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom_allgenes.RDS
    Untracked:  data/DO_416_voom_cpresiduals.RDS
    Untracked:  data/DO_AQ_active_CoV_toptable_annotated.RDS
    Untracked:  data/DO_AQ_lever_ratio_toptable_annotated.RDS
    Untracked:  data/DO_AQ_mean_inf_toptable_annotated.RDS
    Untracked:  data/DO_AQ_rate_toptable_annotated.RDS
    Untracked:  data/DO_IVSA_SALINE_pleio_lods.RDS
    Untracked:  data/DO_IVSA_SENS_pleio_lods.RDS
    Untracked:  data/DO_IVSA_SENS_pleio_pvl.RDS
    Untracked:  data/DO_IVSA_acquisition_cancor_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_glm.RDS
    Untracked:  data/DO_IVSA_all_cancor_df.RDS
    Untracked:  data/DO_IVSA_all_cancor_df_2024-03-27.RDS
    Untracked:  data/DO_IVSA_data.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_ALL.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_AQ.RDS
    Untracked:  data/DO_IVSA_novelty_data.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_old.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_a1_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_a2_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor_2024-03-25.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor_2024-03-27.RDS
    Untracked:  data/DO_IVSA_prediction_a4_male_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a5_female_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_raw_data.RDS
    Untracked:  data/DO_IVSA_transformed_data.RDS
    Untracked:  data/DO_LIMS_data.RDS
    Untracked:  data/DO_RTG_a1_new.qtl.RData
    Untracked:  data/DO_RTG_a1_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a1_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a2_new.qtl.RData
    Untracked:  data/DO_RTG_a2_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a2_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a3_new.qtl.RData
    Untracked:  data/DO_RTG_a3_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_01.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_02.RData
    Untracked:  data/DO_RTG_a4_new.qtl.RData
    Untracked:  data/DO_RTG_a5_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_project_df_for_mapping.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_2020-11-06.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_a2.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_sexspecific.RDS
    Untracked:  data/DO_SENS_SALINE_pleio_lods.RDS
    Untracked:  data/DO_cocaine_measures_data.RDS
    Untracked:  data/DO_demographic_data_df.RDS
    Untracked:  data/DO_eigengene_significance.RDS
    Untracked:  data/DO_eigengenes.RDS
    Untracked:  data/DO_elasticnet_GEX_pvalue_histograms.RData
    Untracked:  data/DO_highcor_cpresiduals_datatable.RDS
    Untracked:  data/DO_holeboard_data_df.RDS
    Untracked:  data/DO_holeboard_summarized.RDS
    Untracked:  data/DO_lightdark_data_df.RDS
    Untracked:  data/DO_mQTL_peaks.RDS
    Untracked:  data/DO_mQTL_scan1s.RDS
    Untracked:  data/DO_module_membership.RDS
    Untracked:  data/DO_novelplace_data_df.RDS
    Untracked:  data/DO_novelty_COC_cancor.RDS
    Untracked:  data/DO_novelty_SAL_cancor.RDS
    Untracked:  data/DO_novelty_data.RDS
    Untracked:  data/DO_novelty_response_data.RDS
    Untracked:  data/DO_novelty_response_data_for_RTG.RDS
    Untracked:  data/DO_novelty_response_data_with_transform.csv
    Untracked:  data/DO_novelty_saline_cocaine_cancor.RDS
    Untracked:  data/DO_openfield_data_df.RDS
    Untracked:  data/DO_paraclique_data.el
    Untracked:  data/DO_paraclique_eigengenes.RDS
    Untracked:  data/DO_paraclique_significance_elasticnet.RDS
    Untracked:  data/DO_paracliques.RDS
    Untracked:  data/DO_paracliques_significance.RDS
    Untracked:  data/DO_ploidy_calls.RDS
    Untracked:  data/DO_pmap.RDS
    Untracked:  data/DO_pvalue_histograms.RData
    Untracked:  data/DO_rtg_a1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a3_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m3_toptable_annotated.RDS
    Untracked:  data/DO_saline.qtl.RData
    Untracked:  data/DO_saline_measures_data.RDS
    Untracked:  data/DO_sensi_cocaine.qtl.RData
    Untracked:  data/DO_str_2016_eQTL.RData
    Untracked:  data/DO_str_2016_gm4qtl2.zip
    Untracked:  data/DO_str_2016_gm4qtl2_intensities.fst
    Untracked:  data/DO_voom_cpresiduals_NArm.RDS
    Untracked:  data/DO_wgcna.RDS
    Untracked:  data/IVSA_RTG_a3_blup.pdf
    Untracked:  data/IVSA_h2_2019-11-18.tsv
    Untracked:  data/IVSA_metadata_sheet.RDS
    Untracked:  data/ME16_edges.tsv
    Untracked:  data/Mus_musculus.GRCm38.94.parsed.RDS
    Untracked:  data/Paraclique_19_MF_AMIGO.txt
    Untracked:  data/RTG_QTL_peaks.RDS
    Untracked:  data/RTG_figure.RDS
    Untracked:  data/SENSI_chr7_snps.RData
    Untracked:  data/SENS_RTG/
    Untracked:  data/Tyr2_genotypes.RDS
    Untracked:  data/Tyr_genotypes.RDS
    Untracked:  data/blup_chr7_DO_RTG_a3.RDS
    Untracked:  data/blup_spearman_cors.RData
    Untracked:  data/cc_variants/
    Untracked:  data/chr18_QTL_results.RDS
    Untracked:  data/chr4_QTL_results.RDS
    Untracked:  data/chr7_a3_QTL_results.RDS
    Untracked:  data/chr7_blups_with_rtg.RDS
    Untracked:  data/chr7_cocaine_BLUP.RDS
    Untracked:  data/chr7_gene_blup.RDS
    Untracked:  data/chr7_qtl_cors_with_spearman_blups.RDS
    Untracked:  data/chr7_sensi_cocaine_BLUP.RDS
    Untracked:  data/cis/
    Untracked:  data/eQTL_annotated_peaks.RDS
    Untracked:  data/eQTL_plots.RData
    Untracked:  data/figures/
    Untracked:  data/founders_key/
    Untracked:  data/gigamuga/
    Untracked:  data/gwas/
    Untracked:  data/info/
    Untracked:  data/m2G.a2.RTG.chr4snp.RData
    Untracked:  data/mapping/
    Untracked:  data/mapping_from_hao/
    Untracked:  data/mpd_measures_saline.txt
    Untracked:  data/mpd_measures_saline_cocaine.txt
    Untracked:  data/perms/
    Untracked:  data/pleio_df.RDS
    Untracked:  data/qtl2/
    Untracked:  data/raw/
    Untracked:  data/resources/
    Untracked:  data/tads_loops/
    Untracked:  data/transcripts/
    Untracked:  data/variants/
    Untracked:  logs/
    Untracked:  new_RTG_a3.jpeg
    Untracked:  output/DO_module_membership.tsv
    Untracked:  output/DO_paraclique_df.tsv
    Untracked:  output/RDS/
    Untracked:  output/RTG_out/
    Untracked:  output/docs_backup_2020-07-30.tar.gz
    Untracked:  output/elastic_net_out/
    Untracked:  output/mapping_figure.pdf
    Untracked:  output/mapping_plot.pdf
    Untracked:  output/mediation_all_rta_figure.pdf
    Untracked:  output/mediation_all_rta_figure_with_eed.pdf
    Untracked:  output/pleio_plot2.pdf
    Untracked:  rstudio_rta.sif
    Untracked:  singularity/
    Untracked:  tmp/
    Untracked:  var/

Unstaged changes:
    Deleted:    .Rprofile
    Modified:   .gitignore
    Modified:   README.md
    Modified:   analysis/17_mapping_interpretation.Rmd
    Modified:   analysis/_site.yml

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/05_DO_expression_prep_STAR_RSEM.Rmd</code>) and HTML
(<code>docs/05_DO_expression_prep_STAR_RSEM.html</code>) files. If
you’ve configured a remote Git repository (see
<code>?wflow_git_remote</code>), click on the hyperlinks in the table
below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
bd41812
</td>
<td>
Michael C. Saul
</td>
<td>
2024-03-29
</td>
<td>
Committing in the new repository for the first time. Changing biomaRt
repository to January 2019.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
362d53f
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
<td>
Initial commit for Sumner
</td>
</tr>
<tr>
<td>
html
</td>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
8a53a3d
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
<td>
Prepping DO differential expression with filterByExpr used as filtering
criterion
</td>
</tr>
<tr>
<td>
html
</td>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
<td>
Rolling back to working version.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
be1304f
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
06f74b7
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-29
</td>
<td>
Updated to fix bug with subject IDs that affected X aneuploidy
</td>
</tr>
<tr>
<td>
html
</td>
<td>
cfd5b48
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
c4e9a5f
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
<td>
Minor tweak of md5 checksums of output files as a table
</td>
</tr>
<tr>
<td>
html
</td>
<td>
661f8d8
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
f5d51a3
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
<td>
Fixing md5 checksums of output files as a table
</td>
</tr>
<tr>
<td>
html
</td>
<td>
6dc66a4
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
0d0753e
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
<td>
Adding in md5 checksums of output files as a table
</td>
</tr>
<tr>
<td>
html
</td>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
a50fe70
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
<td>
Initial commit of expression prep data.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="background" class="section level1">
<h1>Background</h1>
<div id="motivation" class="section level2">
<h2>Motivation</h2>
<div id="purpose-of-analysis" class="section level3">
<h3>Purpose of analysis</h3>
<p>This analysis is performed to preprocess and summarize striatum
expression data prior to expression correlation analysis.</p>
</div>
</div>
</div>
<div id="analysis" class="section level1">
<h1>Analysis</h1>
<div id="setup" class="section level2">
<h2>Setup</h2>
<div id="current-date" class="section level3">
<h3>Current Date</h3>
<p>Getting current date</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a>current_date <span class="ot">=</span> <span class="fu">format</span>(<span class="fu">Sys.time</span>(),<span class="st">&quot;%Y-%m-%d&quot;</span>)</span></code></pre></div>
</div>
<div id="r-libraries" class="section level3">
<h3>R libraries</h3>
<p>Calling R libraries necessary for this analysis.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;limma&quot;</span>)</span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="co"># library(&quot;GenomicRanges&quot;)</span></span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;tibble&quot;</span>)</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;tximport&quot;</span>)</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;edgeR&quot;</span>)</span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggplot2&quot;</span>)</span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;gplots&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    lowess</code></pre>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;RColorBrewer&quot;</span>)</span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;biomaRt&quot;</span>)</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggbeeswarm&quot;</span>)</span></code></pre></div>
</div>
<div id="norm_rank_transform-function" class="section level3">
<h3><code>norm_rank_transform()</code> function</h3>
<p>Getting a normal rank transformation function that can handle
<code>NA</code> values from the following formula:</p>
<p><span
class="math inline">\(Y^t_i={\phi}^{-1}({{r_i-c}\over{N-2c+1}})\)</span></p>
<p>where <span class="math inline">\(r_i\)</span> is the rank, <span
class="math inline">\(N\)</span> is the number of ranked samples, <span
class="math inline">\(\phi^{-1}\)</span> is the inverse normal funcion
(equivalent to <code>qnorm()</code> in R). The original recommended
value of <span class="math inline">\(c=3/8\)</span> was per <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2921808/">Beasley and
Erickson, 2010</a>, but we use <span class="math inline">\(c=0\)</span>.
Our work appears to meet their sample size recommendations.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>norm_rank_transform <span class="ot">=</span> <span class="cf">function</span>(x, <span class="at">c =</span> <span class="dv">0</span>) {</span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stopifnot</span>(<span class="fu">is.numeric</span>(x) <span class="sc">&amp;</span> <span class="fu">is.vector</span>(x))</span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>  x_noNA <span class="ot">=</span> <span class="fu">which</span>(<span class="sc">!</span><span class="fu">is.na</span>(x))</span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>  N <span class="ot">=</span> <span class="fu">length</span>(x_noNA)</span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a>  x[x_noNA] <span class="ot">=</span> <span class="fu">qnorm</span>((<span class="fu">rank</span>(x[x_noNA], <span class="at">ties.method =</span> <span class="st">&quot;average&quot;</span>) <span class="sc">-</span> c) <span class="sc">/</span> (N <span class="sc">-</span> (<span class="dv">2</span> <span class="sc">*</span> c) <span class="sc">+</span> <span class="dv">1</span>))</span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(x)</span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
</div>
</div>
<div id="importing-data" class="section level2">
<h2>Importing data</h2>
<div id="importing-demographic-data" class="section level3">
<h3>Importing demographic data</h3>
<p>Using the demographic data to generate a data frame that can hold
phenotype data.</p>
<p>The md5 checksum of the input file
(<code>416_Matched_DO_Generation_and_Sex_Data.csv</code>) is:
<code>1f3c8e14eba6754b3e93c5e31246c3ad</code>.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>DO_demodata <span class="ot">=</span> <span class="fu">read.csv</span>(<span class="st">&quot;./data/raw/416_Matched_DO_Generation_and_Sex_Data.csv&quot;</span>, <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(DO_demodata) <span class="ot">=</span> <span class="fu">as.character</span>(DO_demodata<span class="sc">$</span>ExpSess.Subject)</span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>DO_pheno <span class="ot">=</span> DO_demodata[,<span class="fu">c</span>(<span class="st">&quot;Sex&quot;</span>,<span class="st">&quot;Generation&quot;</span>)]</span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a><span class="fu">rm</span>(<span class="at">list =</span> <span class="fu">c</span>(<span class="st">&quot;DO_demodata&quot;</span>))</span></code></pre></div>
</div>
<div id="importing-expression-data" class="section level3">
<h3>Importing expression data</h3>
<p><em>Note: the transcript EM results have been gzipped for use in this
workflowr project.</em></p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Preparing to read in aligned and summarized data</span></span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>DO_RSEM_dir <span class="ot">=</span> <span class="st">&quot;./data/transcripts/&quot;</span></span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a>DO_RSEM_files <span class="ot">=</span> <span class="fu">list.files</span>(<span class="at">path =</span> DO_RSEM_dir)</span>
<span id="cb8-4"><a href="#cb8-4" aria-hidden="true" tabindex="-1"></a>DO_RSEM_files <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">base =</span> DO_RSEM_files,</span>
<span id="cb8-5"><a href="#cb8-5" aria-hidden="true" tabindex="-1"></a>                           <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb8-6"><a href="#cb8-6" aria-hidden="true" tabindex="-1"></a>DO_RSEM_files<span class="sc">$</span>behavior_id <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^16NGS-001-Chesler_(</span><span class="sc">\\</span><span class="st">d{4})_[sS]t_(GT16-</span><span class="sc">\\</span><span class="st">d{5})_</span><span class="sc">\\</span><span class="st">w{6}_S</span><span class="sc">\\</span><span class="st">d{1,3}_RSEM_quant.isoforms.results.gz$&quot;</span>, <span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>, DO_RSEM_files<span class="sc">$</span>base)</span>
<span id="cb8-7"><a href="#cb8-7" aria-hidden="true" tabindex="-1"></a>DO_RSEM_files<span class="sc">$</span>sample_id <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^16NGS-001-Chesler_(</span><span class="sc">\\</span><span class="st">d{4})_[sS]t_(GT16-</span><span class="sc">\\</span><span class="st">d{5})_</span><span class="sc">\\</span><span class="st">w{6}_S</span><span class="sc">\\</span><span class="st">d{1,3}_RSEM_quant.isoforms.results.gz$&quot;</span>, <span class="st">&quot;</span><span class="sc">\\</span><span class="st">2&quot;</span>, DO_RSEM_files<span class="sc">$</span>base)</span>
<span id="cb8-8"><a href="#cb8-8" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(DO_RSEM_files) <span class="ot">=</span> DO_RSEM_files<span class="sc">$</span>behavior_id</span>
<span id="cb8-9"><a href="#cb8-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-10"><a href="#cb8-10" aria-hidden="true" tabindex="-1"></a>file_i <span class="ot">=</span> <span class="fu">read.table</span>(<span class="fu">paste</span>(DO_RSEM_dir,DO_RSEM_files[<span class="dv">1</span>,<span class="st">&quot;base&quot;</span>],<span class="at">sep=</span><span class="st">&quot;&quot;</span>), <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">&quot;</span><span class="sc">\t</span><span class="st">&quot;</span>, <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb8-11"><a href="#cb8-11" aria-hidden="true" tabindex="-1"></a>DO_tx2gene <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">TXNAME =</span> file_i<span class="sc">$</span>transcript_id,</span>
<span id="cb8-12"><a href="#cb8-12" aria-hidden="true" tabindex="-1"></a>                        <span class="at">GENEID =</span> file_i<span class="sc">$</span>gene_id,</span>
<span id="cb8-13"><a href="#cb8-13" aria-hidden="true" tabindex="-1"></a>                        <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb8-14"><a href="#cb8-14" aria-hidden="true" tabindex="-1"></a>DO_tx2gene <span class="ot">=</span> <span class="fu">as_tibble</span>(DO_tx2gene)</span>
<span id="cb8-15"><a href="#cb8-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-16"><a href="#cb8-16" aria-hidden="true" tabindex="-1"></a>txi_i <span class="ot">=</span> <span class="fu">tximport</span>(<span class="at">files =</span> <span class="fu">paste</span>(DO_RSEM_dir,</span>
<span id="cb8-17"><a href="#cb8-17" aria-hidden="true" tabindex="-1"></a>                               DO_RSEM_files[,<span class="st">&quot;base&quot;</span>],</span>
<span id="cb8-18"><a href="#cb8-18" aria-hidden="true" tabindex="-1"></a>                               <span class="at">sep=</span><span class="st">&quot;&quot;</span>),</span>
<span id="cb8-19"><a href="#cb8-19" aria-hidden="true" tabindex="-1"></a>                 <span class="at">type =</span> <span class="st">&quot;rsem&quot;</span>,</span>
<span id="cb8-20"><a href="#cb8-20" aria-hidden="true" tabindex="-1"></a>                 <span class="at">tx2gene =</span> DO_tx2gene,</span>
<span id="cb8-21"><a href="#cb8-21" aria-hidden="true" tabindex="-1"></a>                 <span class="at">countsFromAbundance =</span> <span class="st">&quot;lengthScaledTPM&quot;</span>,</span>
<span id="cb8-22"><a href="#cb8-22" aria-hidden="true" tabindex="-1"></a>                 <span class="at">abundanceCol =</span> <span class="st">&quot;TPM&quot;</span>,</span>
<span id="cb8-23"><a href="#cb8-23" aria-hidden="true" tabindex="-1"></a>                 <span class="at">ignoreTxVersion =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
<pre><code>reading in files with read_tsv</code></pre>
<pre><code>1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 242 243 244 245 246 247 248 249 250 251 252 253 254 255 256 257 258 259 260 261 262 263 264 265 266 267 268 269 270 271 272 273 274 275 276 277 278 279 280 281 282 283 284 285 286 287 288 289 290 291 292 293 294 295 296 297 298 299 300 301 302 303 304 305 306 307 308 309 310 311 312 313 314 315 316 317 318 319 320 321 322 323 324 325 326 327 328 329 330 331 332 333 334 335 336 337 338 339 340 341 342 343 344 345 346 347 348 349 350 351 352 353 354 355 356 357 358 359 360 361 362 363 364 365 366 367 368 369 370 371 372 373 374 375 376 377 378 379 380 381 382 383 384 385 386 387 388 389 390 391 392 393 394 395 396 397 398 399 400 401 402 403 404 405 406 407 408 409 410 411 412 413 414 415 416 
transcripts missing from tx2gene: 1082
summarizing abundance
summarizing counts
summarizing length</code></pre>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a>gene_mat_i <span class="ot">=</span> txi_i<span class="sc">$</span>counts</span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(gene_mat_i) <span class="ot">=</span> <span class="fu">row.names</span>(DO_RSEM_files)</span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>DO_gene_counts <span class="ot">=</span> <span class="fu">as.data.frame</span>(gene_mat_i)</span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a><span class="fu">rm</span>(<span class="at">list =</span> <span class="fu">c</span>(<span class="st">&quot;txi_i&quot;</span>, <span class="st">&quot;file_i&quot;</span>, <span class="st">&quot;gene_mat_i&quot;</span>))</span>
<span id="cb11-5"><a href="#cb11-5" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb11-6"><a href="#cb11-6" aria-hidden="true" tabindex="-1"></a>DO_gene_counts <span class="ot">=</span> <span class="fu">as.matrix</span>(DO_gene_counts[<span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_gene_counts)) <span class="sc">==</span> <span class="dv">0</span>),])</span>
<span id="cb11-7"><a href="#cb11-7" aria-hidden="true" tabindex="-1"></a>DO_ERCC_counts <span class="ot">=</span> DO_gene_counts[<span class="fu">grep</span>(<span class="st">&quot;ERCC&quot;</span>, <span class="fu">row.names</span>(DO_gene_counts)),]</span>
<span id="cb11-8"><a href="#cb11-8" aria-hidden="true" tabindex="-1"></a>DO_gene_counts <span class="ot">=</span> DO_gene_counts[<span class="fu">grep</span>(<span class="st">&quot;^ENSMUSG&quot;</span>, <span class="fu">row.names</span>(DO_gene_counts)),]</span>
<span id="cb11-9"><a href="#cb11-9" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(DO_gene_counts) <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^(ENSMUSG</span><span class="sc">\\</span><span class="st">d{11})_.*$&quot;</span>, <span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>, <span class="fu">row.names</span>(DO_gene_counts))</span></code></pre></div>
<p>Preparing edgeR DGEList from the data at the gene level.</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>DO_exprs_SubjectIDs <span class="ot">=</span> <span class="fu">colnames</span>(DO_gene_counts)</span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a>DO_dge <span class="ot">=</span> <span class="fu">DGEList</span>(<span class="fu">as.matrix</span>(DO_gene_counts))</span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>DO_dge<span class="sc">$</span>samples <span class="ot">=</span> <span class="fu">cbind</span>(DO_dge<span class="sc">$</span>samples, DO_pheno[DO_exprs_SubjectIDs,])</span></code></pre></div>
</div>
<div id="qc-of-expression-data" class="section level3">
<h3>QC of expression data</h3>
<p>Starting with the data from the <code>Sex</code> factor.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a>DGE_sex <span class="ot">=</span> DO_pheno[DO_exprs_SubjectIDs,<span class="st">&quot;Sex&quot;</span>]</span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plotMDS</span>(DO_dge, <span class="at">col =</span> <span class="fu">ifelse</span>(DGE_sex <span class="sc">==</span> <span class="st">&quot;M&quot;</span>,<span class="st">&quot;#66CC66&quot;</span>,<span class="st">&quot;#3399CC&quot;</span>))</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-6-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The MDS indicates that there are some weird points in the middle of
males and females. The male samples 7998, 8497, 7993, 8080, 7732, 8496,
7977, 8077, 7731, 7981, 8081, 8605, and 7623 appear somewhere betwewen
males (green) and females (blue).</p>
<p>However, these points show a substantial amount of noise that is not
distinguishing the sexes. Based upon previous investigations of striatum
data, we found that a number of samples have likely choroid plexus
contamination. We can use samples with <em>Prlr</em> expression of
greater than 10 CPM as a filter for this.</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>high_choroid_plexus <span class="ot">=</span> <span class="fu">colnames</span>(DO_dge)[<span class="fu">which</span>(<span class="fu">cpm</span>(DO_dge)[<span class="st">&quot;ENSMUSG00000005268&quot;</span>,] <span class="sc">&gt;</span> <span class="dv">10</span>)]</span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>choroid_plexus_color <span class="ot">=</span> <span class="fu">ifelse</span>(<span class="fu">colnames</span>(DO_dge) <span class="sc">%in%</span> high_choroid_plexus, <span class="st">&quot;#CC3333&quot;</span>, <span class="st">&quot;#666666&quot;</span>)</span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a><span class="fu">plotMDS</span>(DO_dge, <span class="at">col =</span> choroid_plexus_color)</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-7-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The samples in this location have choroid plexus contamination
Building a nuissance covariate using the log-mean of an unambiguous set
of markers for the choroid plexus contamination. Using the Sathyanesan
et al., 2012 paper from <em>Translational Psychiatry</em><a href="#fn1"
class="footnote-ref" id="fnref1"><sup>1</sup></a>, which reports that
<em>Kl</em> and <em>Ttr</em> are good and unambiguous markers. Their
Ensembl gene IDs are <code>ENSMUSG00000058488</code> and
<code>ENSMUSG00000061808</code> respectively.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a>choroid_plexus_genes <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;ENSMUSG00000058488&quot;</span>,<span class="st">&quot;ENSMUSG00000061808&quot;</span>)</span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a>logmean <span class="ot">=</span> <span class="cf">function</span>(x) {</span>
<span id="cb15-3"><a href="#cb15-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(<span class="fu">exp</span>(<span class="fu">mean</span>(<span class="fu">log</span>(x))))</span>
<span id="cb15-4"><a href="#cb15-4" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb15-5"><a href="#cb15-5" aria-hidden="true" tabindex="-1"></a>choroid_plexus_covariate <span class="ot">=</span> <span class="fu">apply</span>(<span class="fu">cpm</span>(DO_dge[choroid_plexus_genes,]), <span class="at">MARGIN =</span> <span class="dv">2</span>, <span class="at">FUN =</span> logmean)</span>
<span id="cb15-6"><a href="#cb15-6" aria-hidden="true" tabindex="-1"></a>DO_pheno<span class="sc">$</span>choroid_plexus_covariate <span class="ot">=</span> choroid_plexus_covariate[<span class="fu">row.names</span>(DO_pheno)]</span></code></pre></div>
<p>For points assorting between males and females, the candidate
hypothesis is that these are partial X duplication DO mice documented in
Chesler et al., 2016<a href="#fn2" class="footnote-ref"
id="fnref2"><sup>2</sup></a>. This can be tested by comparing how these
mice cluster relative to other males on both chromosome X and chromosome
1.</p>
<p>First, identifying these subjects and getting all genes in Ensembl on
chromosomes 1, X, and Y.</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting male subject IDs</span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>male_subjects <span class="ot">=</span> <span class="fu">row.names</span>(DO_pheno[<span class="fu">which</span>(DO_pheno<span class="sc">$</span>Sex <span class="sc">==</span> <span class="st">&quot;M&quot;</span>),])</span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a>male_subjects_exprs <span class="ot">=</span> male_subjects[<span class="fu">which</span>(male_subjects <span class="sc">%in%</span> <span class="fu">colnames</span>(DO_dge))]</span>
<span id="cb16-4"><a href="#cb16-4" aria-hidden="true" tabindex="-1"></a>males_intermediate <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">c</span>(<span class="dv">7998</span>,<span class="dv">8497</span>,<span class="dv">7993</span>,<span class="dv">8080</span>,<span class="dv">7732</span>,<span class="dv">8496</span>,<span class="dv">7977</span>,<span class="dv">8077</span>,<span class="dv">7731</span>,<span class="dv">7981</span>,<span class="dv">8081</span>,<span class="dv">8605</span>,<span class="dv">7623</span>))</span>
<span id="cb16-5"><a href="#cb16-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-6"><a href="#cb16-6" aria-hidden="true" tabindex="-1"></a><span class="co"># Setting up biomaRt</span></span>
<span id="cb16-7"><a href="#cb16-7" aria-hidden="true" tabindex="-1"></a>chr_maRt <span class="ot">=</span> <span class="fu">useMart</span>(<span class="at">biomart =</span> <span class="st">&quot;ENSEMBL_MART_ENSEMBL&quot;</span>,</span>
<span id="cb16-8"><a href="#cb16-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">host =</span> <span class="st">&quot;https://jan2019.archive.ensembl.org&quot;</span>,</span>
<span id="cb16-9"><a href="#cb16-9" aria-hidden="true" tabindex="-1"></a>                   <span class="at">dataset =</span> <span class="st">&quot;mmusculus_gene_ensembl&quot;</span>)</span>
<span id="cb16-10"><a href="#cb16-10" aria-hidden="true" tabindex="-1"></a>chr_maRt_filter <span class="ot">=</span> <span class="st">&quot;chromosome_name&quot;</span></span>
<span id="cb16-11"><a href="#cb16-11" aria-hidden="true" tabindex="-1"></a>chr_maRt_attributes <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;ensembl_gene_id&quot;</span>)</span>
<span id="cb16-12"><a href="#cb16-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-13"><a href="#cb16-13" aria-hidden="true" tabindex="-1"></a><span class="co"># Setting heatmap colors</span></span>
<span id="cb16-14"><a href="#cb16-14" aria-hidden="true" tabindex="-1"></a>heatmapColors <span class="ot">=</span> <span class="fu">colorRampPalette</span>(<span class="fu">c</span>(<span class="st">&quot;#660066&quot;</span>,<span class="st">&quot;#FFFFFF&quot;</span>))(<span class="at">n=</span><span class="dv">299</span>)</span>
<span id="cb16-15"><a href="#cb16-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-16"><a href="#cb16-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting genes on various combinations of chromosome</span></span>
<span id="cb16-17"><a href="#cb16-17" aria-hidden="true" tabindex="-1"></a>chr1_genes <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">getBM</span>(chr_maRt_attributes, chr_maRt_filter, <span class="fu">c</span>(<span class="st">&quot;1&quot;</span>), chr_maRt)<span class="sc">$</span>ensembl_gene_id)</span>
<span id="cb16-18"><a href="#cb16-18" aria-hidden="true" tabindex="-1"></a>chrX_genes <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">getBM</span>(chr_maRt_attributes, chr_maRt_filter, <span class="fu">c</span>(<span class="st">&quot;X&quot;</span>), chr_maRt)<span class="sc">$</span>ensembl_gene_id)</span>
<span id="cb16-19"><a href="#cb16-19" aria-hidden="true" tabindex="-1"></a>chrY_genes <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">getBM</span>(chr_maRt_attributes, chr_maRt_filter, <span class="fu">c</span>(<span class="st">&quot;Y&quot;</span>), chr_maRt)<span class="sc">$</span>ensembl_gene_id)</span>
<span id="cb16-20"><a href="#cb16-20" aria-hidden="true" tabindex="-1"></a>autosomal_genes <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">getBM</span>(chr_maRt_attributes, chr_maRt_filter, <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">19</span>), chr_maRt)<span class="sc">$</span>ensembl_gene_id)</span></code></pre></div>
<p>Now clustering samples together based upon their chromosome 1
expression correlations.</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtering chr1 genes and looking at heatmap</span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>chr1_genes <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">getBM</span>(chr_maRt_attributes, chr_maRt_filter, <span class="fu">c</span>(<span class="st">&quot;1&quot;</span>), chr_maRt)<span class="sc">$</span>ensembl_gene_id)</span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>chr1_genes_dge <span class="ot">=</span> chr1_genes[<span class="fu">which</span>(chr1_genes <span class="sc">%in%</span> <span class="fu">row.names</span>(DO_dge))]</span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a>males_chr1 <span class="ot">=</span> DO_dge[chr1_genes_dge,male_subjects_exprs]</span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>males_chr1 <span class="ot">=</span> <span class="fu">cpm</span>(males_chr1)</span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>males_chr1 <span class="ot">=</span> males_chr1[<span class="fu">which</span>(<span class="fu">rowSums</span>(males_chr1 <span class="sc">&gt;=</span> <span class="dv">1</span>) <span class="sc">&gt;=</span> <span class="dv">13</span>),]</span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a>males_col1 <span class="ot">=</span> <span class="fu">ifelse</span>(<span class="fu">colnames</span>(males_chr1) <span class="sc">%in%</span> males_intermediate,</span>
<span id="cb17-8"><a href="#cb17-8" aria-hidden="true" tabindex="-1"></a>                    <span class="st">&quot;#66CC66&quot;</span>,<span class="st">&quot;#AAAAAA&quot;</span>)</span>
<span id="cb17-9"><a href="#cb17-9" aria-hidden="true" tabindex="-1"></a><span class="fu">heatmap.2</span>(<span class="fu">cor</span>(males_chr1),</span>
<span id="cb17-10"><a href="#cb17-10" aria-hidden="true" tabindex="-1"></a>          <span class="at">trace =</span> <span class="st">&quot;none&quot;</span>,</span>
<span id="cb17-11"><a href="#cb17-11" aria-hidden="true" tabindex="-1"></a>          <span class="at">col =</span> heatmapColors,</span>
<span id="cb17-12"><a href="#cb17-12" aria-hidden="true" tabindex="-1"></a>          <span class="at">dendrogram =</span> <span class="st">&quot;both&quot;</span>,</span>
<span id="cb17-13"><a href="#cb17-13" aria-hidden="true" tabindex="-1"></a>          <span class="at">ColSideColors =</span> males_col1,</span>
<span id="cb17-14"><a href="#cb17-14" aria-hidden="true" tabindex="-1"></a>          <span class="at">RowSideColors =</span> males_col1,</span>
<span id="cb17-15"><a href="#cb17-15" aria-hidden="true" tabindex="-1"></a>          <span class="at">main =</span> <span class="st">&quot;chr1 male gene expression correlation&quot;</span>)</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-10-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-10-1">
Past versions of unnamed-chunk-10-1.png
</button>
</p>
<div id="fig-unnamed-chunk-10-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Nothing looks too strange on chromosome 1 with respect to these
samples. However, there does appear to be an outlier point or two. We
will come back to that later.</p>
<p>Clustering samples together based upon their chromosome X
correlations.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtering chrX genes and looking at heatmap</span></span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a>chrX_genes_dge <span class="ot">=</span> chrX_genes[<span class="fu">which</span>(chrX_genes <span class="sc">%in%</span> <span class="fu">row.names</span>(DO_dge))]</span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a>males_chrX <span class="ot">=</span> DO_dge[chrX_genes_dge,male_subjects_exprs]</span>
<span id="cb18-4"><a href="#cb18-4" aria-hidden="true" tabindex="-1"></a>males_chrX <span class="ot">=</span> <span class="fu">cpm</span>(males_chrX)</span>
<span id="cb18-5"><a href="#cb18-5" aria-hidden="true" tabindex="-1"></a>males_chrX <span class="ot">=</span> males_chrX[<span class="fu">which</span>(<span class="fu">rowSums</span>(males_chrX <span class="sc">&gt;=</span> <span class="dv">1</span>) <span class="sc">&gt;=</span> <span class="dv">13</span>),]</span>
<span id="cb18-6"><a href="#cb18-6" aria-hidden="true" tabindex="-1"></a>males_colX <span class="ot">=</span> <span class="fu">ifelse</span>(<span class="fu">colnames</span>(males_chrX) <span class="sc">%in%</span> males_intermediate,</span>
<span id="cb18-7"><a href="#cb18-7" aria-hidden="true" tabindex="-1"></a>                    <span class="st">&quot;#66CC66&quot;</span>,<span class="st">&quot;#AAAAAA&quot;</span>)</span>
<span id="cb18-8"><a href="#cb18-8" aria-hidden="true" tabindex="-1"></a><span class="fu">heatmap.2</span>(<span class="fu">cor</span>(males_chrX),</span>
<span id="cb18-9"><a href="#cb18-9" aria-hidden="true" tabindex="-1"></a>          <span class="at">trace =</span> <span class="st">&quot;none&quot;</span>,</span>
<span id="cb18-10"><a href="#cb18-10" aria-hidden="true" tabindex="-1"></a>          <span class="at">col =</span> heatmapColors,</span>
<span id="cb18-11"><a href="#cb18-11" aria-hidden="true" tabindex="-1"></a>          <span class="at">dendrogram =</span> <span class="st">&quot;both&quot;</span>,</span>
<span id="cb18-12"><a href="#cb18-12" aria-hidden="true" tabindex="-1"></a>          <span class="at">ColSideColors =</span> males_colX,</span>
<span id="cb18-13"><a href="#cb18-13" aria-hidden="true" tabindex="-1"></a>          <span class="at">RowSideColors =</span> males_colX,</span>
<span id="cb18-14"><a href="#cb18-14" aria-hidden="true" tabindex="-1"></a>          <span class="at">main =</span> <span class="st">&quot;chrX male gene expression correlation&quot;</span>)</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-11-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>This is pretty clear. The unusual samples are most likely partial X
duplication. They cluster on chromosome X expression, but not on
chromosome 1 expression.</p>
<p>Chesler et al., 2016<a href="#fn3" class="footnote-ref"
id="fnref3"><sup>3</sup></a> also documents the presence of XO females.
Looking at the MDS in just females to see if there are any odd candidate
samples.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>female_subjects <span class="ot">=</span> <span class="fu">row.names</span>(DO_pheno[<span class="fu">which</span>(DO_pheno<span class="sc">$</span>Sex <span class="sc">==</span> <span class="st">&quot;F&quot;</span>),])</span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>female_subjects_exprs <span class="ot">=</span> female_subjects[<span class="fu">which</span>(female_subjects <span class="sc">%in%</span> <span class="fu">colnames</span>(DO_dge))]</span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a>DO_female_dge <span class="ot">=</span> DO_dge[,female_subjects_exprs]</span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a><span class="fu">plotMDS</span>(DO_female_dge, <span class="at">col =</span> <span class="fu">rep</span>(<span class="st">&quot;#3399CC&quot;</span>, <span class="at">times =</span> <span class="fu">ncol</span>(DO_female_dge)))</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-12-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-12-1">
Past versions of unnamed-chunk-12-1.png
</button>
</p>
<div id="fig-unnamed-chunk-12-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>There are many points off the main cloud. Plotting female chromosomes
1 and X data like the male data to be certain.</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Female chr1</span></span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>chr1_genes_dge <span class="ot">=</span> chr1_genes[<span class="fu">which</span>(chr1_genes <span class="sc">%in%</span> <span class="fu">row.names</span>(DO_female_dge))]</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a>females_chr1 <span class="ot">=</span> DO_dge[chr1_genes_dge,female_subjects_exprs]</span>
<span id="cb20-4"><a href="#cb20-4" aria-hidden="true" tabindex="-1"></a>females_chr1 <span class="ot">=</span> <span class="fu">cpm</span>(females_chr1)</span>
<span id="cb20-5"><a href="#cb20-5" aria-hidden="true" tabindex="-1"></a>females_chr1 <span class="ot">=</span> females_chr1[<span class="fu">which</span>(<span class="fu">rowSums</span>(females_chr1 <span class="sc">&gt;=</span> <span class="dv">1</span>) <span class="sc">&gt;=</span> <span class="dv">13</span>),]</span>
<span id="cb20-6"><a href="#cb20-6" aria-hidden="true" tabindex="-1"></a><span class="fu">heatmap.2</span>(<span class="fu">cor</span>(females_chr1),</span>
<span id="cb20-7"><a href="#cb20-7" aria-hidden="true" tabindex="-1"></a>          <span class="at">trace =</span> <span class="st">&quot;none&quot;</span>,</span>
<span id="cb20-8"><a href="#cb20-8" aria-hidden="true" tabindex="-1"></a>          <span class="at">col =</span> heatmapColors,</span>
<span id="cb20-9"><a href="#cb20-9" aria-hidden="true" tabindex="-1"></a>          <span class="at">dendrogram =</span> <span class="st">&quot;both&quot;</span>,</span>
<span id="cb20-10"><a href="#cb20-10" aria-hidden="true" tabindex="-1"></a>          <span class="at">main =</span> <span class="st">&quot;chr1 female gene expression correlation&quot;</span>)</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-13-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-13-1">
Past versions of unnamed-chunk-13-1.png
</button>
</p>
<div id="fig-unnamed-chunk-13-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Again, nothing unusual, though there is a single strange sample in
the chromosome 1 data again that has low correlation with the rest.</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Female chrX</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>chrX_genes_dge <span class="ot">=</span> chrX_genes[<span class="fu">which</span>(chrX_genes <span class="sc">%in%</span> <span class="fu">row.names</span>(DO_female_dge))]</span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>females_chrX <span class="ot">=</span> DO_dge[chrX_genes_dge,female_subjects_exprs]</span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>females_chrX <span class="ot">=</span> <span class="fu">cpm</span>(females_chrX)</span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>females_chrX <span class="ot">=</span> females_chrX[<span class="fu">which</span>(<span class="fu">rowSums</span>(females_chrX <span class="sc">&gt;=</span> <span class="dv">1</span>) <span class="sc">&gt;=</span> <span class="dv">13</span>),]</span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a><span class="fu">heatmap.2</span>(<span class="fu">cor</span>(females_chrX),</span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>          <span class="at">trace =</span> <span class="st">&quot;none&quot;</span>,</span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a>          <span class="at">col =</span> heatmapColors,</span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a>          <span class="at">dendrogram =</span> <span class="st">&quot;both&quot;</span>,</span>
<span id="cb21-10"><a href="#cb21-10" aria-hidden="true" tabindex="-1"></a>          <span class="at">main =</span> <span class="st">&quot;chrX female gene expression correlation&quot;</span>)</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-14-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-14-1">
Past versions of unnamed-chunk-14-1.png
</button>
</p>
<div id="fig-unnamed-chunk-14-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>This looks reminiscent of the male data. Cutting the tree into 2 to
identify outlier points.</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>female_treeX <span class="ot">=</span> <span class="fu">hclust</span>(<span class="fu">dist</span>(<span class="fu">cor</span>(females_chrX)))</span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>female_cutX <span class="ot">=</span> <span class="fu">cutree</span>(female_treeX, <span class="at">k =</span> <span class="dv">2</span>)</span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a>female_colX <span class="ot">=</span> <span class="fu">ifelse</span>(female_cutX <span class="sc">==</span> <span class="dv">1</span>, <span class="st">&quot;#3399CC&quot;</span>,<span class="st">&quot;#AAAAAA&quot;</span>)</span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a><span class="fu">heatmap.2</span>(<span class="fu">cor</span>(females_chrX),</span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a>          <span class="at">trace =</span> <span class="st">&quot;none&quot;</span>,</span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a>          <span class="at">col =</span> heatmapColors,</span>
<span id="cb22-7"><a href="#cb22-7" aria-hidden="true" tabindex="-1"></a>          <span class="at">dendrogram =</span> <span class="st">&quot;both&quot;</span>,</span>
<span id="cb22-8"><a href="#cb22-8" aria-hidden="true" tabindex="-1"></a>          <span class="at">RowSideColors =</span> female_colX,</span>
<span id="cb22-9"><a href="#cb22-9" aria-hidden="true" tabindex="-1"></a>          <span class="at">ColSideColors =</span> female_colX,</span>
<span id="cb22-10"><a href="#cb22-10" aria-hidden="true" tabindex="-1"></a>          <span class="at">main =</span> <span class="st">&quot;chrX female gene expression correlation&quot;</span>)</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-15-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-15-1">
Past versions of unnamed-chunk-15-1.png
</button>
</p>
<div id="fig-unnamed-chunk-15-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>These samples do fall out together as a block. Trying the Y
chromosome.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Female chrY</span></span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>chrY_genes_dge <span class="ot">=</span> chrY_genes[<span class="fu">which</span>(chrY_genes <span class="sc">%in%</span> <span class="fu">row.names</span>(DO_female_dge))]</span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a>females_chrY <span class="ot">=</span> DO_dge[chrY_genes_dge,female_subjects_exprs]</span>
<span id="cb23-4"><a href="#cb23-4" aria-hidden="true" tabindex="-1"></a>females_chrY <span class="ot">=</span> <span class="fu">cpm</span>(females_chrY)</span>
<span id="cb23-5"><a href="#cb23-5" aria-hidden="true" tabindex="-1"></a>females_chrY <span class="ot">=</span> females_chrY[<span class="fu">which</span>(<span class="fu">rowSums</span>(females_chrY <span class="sc">&gt;=</span> <span class="dv">1</span>) <span class="sc">&gt;=</span> <span class="dv">1</span>),]</span>
<span id="cb23-6"><a href="#cb23-6" aria-hidden="true" tabindex="-1"></a><span class="fu">heatmap.2</span>(<span class="fu">cor</span>(females_chrY),</span>
<span id="cb23-7"><a href="#cb23-7" aria-hidden="true" tabindex="-1"></a>          <span class="at">trace =</span> <span class="st">&quot;none&quot;</span>,</span>
<span id="cb23-8"><a href="#cb23-8" aria-hidden="true" tabindex="-1"></a>          <span class="at">col =</span> heatmapColors,</span>
<span id="cb23-9"><a href="#cb23-9" aria-hidden="true" tabindex="-1"></a>          <span class="at">dendrogram =</span> <span class="st">&quot;both&quot;</span>,</span>
<span id="cb23-10"><a href="#cb23-10" aria-hidden="true" tabindex="-1"></a>          <span class="at">RowSideColors =</span> female_colX,</span>
<span id="cb23-11"><a href="#cb23-11" aria-hidden="true" tabindex="-1"></a>          <span class="at">ColSideColors =</span> female_colX,</span>
<span id="cb23-12"><a href="#cb23-12" aria-hidden="true" tabindex="-1"></a>          <span class="at">main =</span> <span class="st">&quot;chrY female gene expression correlation&quot;</span>)</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-16-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-16-1">
Past versions of unnamed-chunk-16-1.png
</button>
</p>
<div id="fig-unnamed-chunk-16-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>These females may be XY, but it’s hard to say off of 11 genes.
Redoing MDS with new information.</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plotMDS</span>(DO_female_dge, <span class="at">col =</span> female_colX)</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-17-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-17-1">
Past versions of unnamed-chunk-17-1.png
</button>
</p>
<div id="fig-unnamed-chunk-17-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The presumed female XO samples do not seem to account for a large
proportion of variance, but these samples should still be identified and
dealt with through either censoring or an additive covariate. Also,
these genomes are effectively a third and fourth sex. Creating new
levels of the sex variable and creating a new phenotypic covariate that
can be used to block out the X aneuoploidy independent of sex.</p>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a>males_partial_X_dup <span class="ot">=</span> males_intermediate</span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a>females_XO <span class="ot">=</span> <span class="fu">names</span>(female_cutX[<span class="fu">which</span>(female_cutX <span class="sc">==</span> <span class="dv">1</span>)])</span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a>DO_pheno<span class="sc">$</span>XAneuploidy <span class="ot">=</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(DO_pheno))</span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a>DO_pheno[<span class="fu">which</span>(<span class="fu">row.names</span>(DO_pheno) <span class="sc">%in%</span> <span class="fu">colnames</span>(DO_dge)),<span class="st">&quot;XAneuploidy&quot;</span>] <span class="ot">=</span> <span class="st">&quot;NotAneuoploid&quot;</span></span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a>DO_pheno[<span class="fu">which</span>(<span class="fu">row.names</span>(DO_pheno) <span class="sc">%in%</span> males_partial_X_dup),<span class="st">&quot;XAneuploidy&quot;</span>] <span class="ot">=</span> <span class="st">&quot;XPartialDup&quot;</span></span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a>DO_pheno[<span class="fu">which</span>(<span class="fu">row.names</span>(DO_pheno) <span class="sc">%in%</span> females_XO),<span class="st">&quot;XAneuploidy&quot;</span>] <span class="ot">=</span> <span class="st">&quot;XOFemale&quot;</span></span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a>DO_pheno[<span class="fu">which</span>(<span class="fu">row.names</span>(DO_pheno) <span class="sc">%in%</span> males_partial_X_dup),<span class="st">&quot;Sex&quot;</span>] <span class="ot">=</span> <span class="st">&quot;M_XPartialDup&quot;</span></span>
<span id="cb25-8"><a href="#cb25-8" aria-hidden="true" tabindex="-1"></a>DO_pheno[<span class="fu">which</span>(<span class="fu">row.names</span>(DO_pheno) <span class="sc">%in%</span> females_XO),<span class="st">&quot;Sex&quot;</span>] <span class="ot">=</span> <span class="st">&quot;F_XO&quot;</span></span></code></pre></div>
<p>To reassess the other samples, looking at correlations between
samples for all highly expressed (&gt; 1 CPM average expression) genes
on autosomes.</p>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting genes on autosomes</span></span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a>autosomal_genes_dge <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">row.names</span>(DO_dge))</span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a>autosomal_genes_dge <span class="ot">=</span> autosomal_genes_dge[<span class="fu">which</span>(autosomal_genes_dge <span class="sc">%in%</span> autosomal_genes)]</span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a>DO_dge_autosomal <span class="ot">=</span> DO_dge[autosomal_genes_dge,]</span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a>DO_dge_autosomal <span class="ot">=</span> <span class="fu">cpm</span>(DO_dge_autosomal[<span class="fu">which</span>(<span class="fu">rowMeans</span>(<span class="fu">cpm</span>(DO_dge_autosomal)) <span class="sc">&gt;=</span> <span class="dv">1</span>),])</span>
<span id="cb26-6"><a href="#cb26-6" aria-hidden="true" tabindex="-1"></a>DO_dge_autosomal_Sex <span class="ot">=</span> DO_pheno[<span class="fu">colnames</span>(DO_dge_autosomal),<span class="st">&quot;Sex&quot;</span>]</span>
<span id="cb26-7"><a href="#cb26-7" aria-hidden="true" tabindex="-1"></a>DO_dge_autosomal_XAneuploidy <span class="ot">=</span> DO_pheno[<span class="fu">colnames</span>(DO_dge_autosomal),<span class="st">&quot;XAneuploidy&quot;</span>]</span>
<span id="cb26-8"><a href="#cb26-8" aria-hidden="true" tabindex="-1"></a>DO_dge_autosomal_SubjectID <span class="ot">=</span> <span class="fu">row.names</span>(DO_pheno[<span class="fu">colnames</span>(DO_dge_autosomal),])</span>
<span id="cb26-9"><a href="#cb26-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-10"><a href="#cb26-10" aria-hidden="true" tabindex="-1"></a><span class="fu">heatmap.2</span>(<span class="fu">cor</span>(DO_dge_autosomal),</span>
<span id="cb26-11"><a href="#cb26-11" aria-hidden="true" tabindex="-1"></a>          <span class="at">trace =</span> <span class="st">&quot;none&quot;</span>,</span>
<span id="cb26-12"><a href="#cb26-12" aria-hidden="true" tabindex="-1"></a>          <span class="at">col =</span> heatmapColors,</span>
<span id="cb26-13"><a href="#cb26-13" aria-hidden="true" tabindex="-1"></a>          <span class="at">dendrogram =</span> <span class="st">&quot;both&quot;</span>,</span>
<span id="cb26-14"><a href="#cb26-14" aria-hidden="true" tabindex="-1"></a>          <span class="at">main =</span> <span class="st">&quot;autosomal expression correlation&quot;</span>)</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-19-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-19-1">
Past versions of unnamed-chunk-19-1.png
</button>
</p>
<div id="fig-unnamed-chunk-19-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>In the GBRS data, there was weird blockiness that was related to the
rRNA 7S data. Seeing what the effects of black listing rRNA 7S is in
this dataset.</p>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a>rRNA_7S_genes <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;ENSMUSG00000065037&quot;</span>,</span>
<span id="cb27-2"><a href="#cb27-2" aria-hidden="true" tabindex="-1"></a>                  <span class="st">&quot;ENSMUSG00000099021&quot;</span>,</span>
<span id="cb27-3"><a href="#cb27-3" aria-hidden="true" tabindex="-1"></a>                  <span class="st">&quot;ENSMUSG00000099250&quot;</span>)</span>
<span id="cb27-4"><a href="#cb27-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb27-5"><a href="#cb27-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting genes on autosomes</span></span>
<span id="cb27-6"><a href="#cb27-6" aria-hidden="true" tabindex="-1"></a>autosomal_genes_dge <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">row.names</span>(DO_dge))</span>
<span id="cb27-7"><a href="#cb27-7" aria-hidden="true" tabindex="-1"></a>autosomal_genes_dge <span class="ot">=</span> autosomal_genes_dge[<span class="fu">which</span>(autosomal_genes_dge <span class="sc">%in%</span> autosomal_genes)]</span>
<span id="cb27-8"><a href="#cb27-8" aria-hidden="true" tabindex="-1"></a>DO_dge_autosomal <span class="ot">=</span> DO_dge[autosomal_genes_dge,]</span>
<span id="cb27-9"><a href="#cb27-9" aria-hidden="true" tabindex="-1"></a>DO_dge_autosomal <span class="ot">=</span> <span class="fu">cpm</span>(DO_dge_autosomal[<span class="fu">which</span>(<span class="fu">rowMeans</span>(<span class="fu">cpm</span>(DO_dge_autosomal)) <span class="sc">&gt;=</span> <span class="dv">1</span>),])</span>
<span id="cb27-10"><a href="#cb27-10" aria-hidden="true" tabindex="-1"></a>DO_dge_autosomal <span class="ot">=</span> DO_dge_autosomal[<span class="fu">which</span>(<span class="sc">!</span>(<span class="fu">row.names</span>(DO_dge_autosomal) <span class="sc">%in%</span> rRNA_7S_genes)),]</span>
<span id="cb27-11"><a href="#cb27-11" aria-hidden="true" tabindex="-1"></a>DO_dge_autosomal_Sex <span class="ot">=</span> DO_pheno[<span class="fu">colnames</span>(DO_dge_autosomal),<span class="st">&quot;Sex&quot;</span>]</span>
<span id="cb27-12"><a href="#cb27-12" aria-hidden="true" tabindex="-1"></a>DO_dge_autosomal_XAneuploidy <span class="ot">=</span> DO_pheno[<span class="fu">colnames</span>(DO_dge_autosomal),<span class="st">&quot;XAneuploidy&quot;</span>]</span>
<span id="cb27-13"><a href="#cb27-13" aria-hidden="true" tabindex="-1"></a>DO_dge_autosomal_SubjectID <span class="ot">=</span> <span class="fu">row.names</span>(DO_pheno[<span class="fu">colnames</span>(DO_dge_autosomal),])</span>
<span id="cb27-14"><a href="#cb27-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb27-15"><a href="#cb27-15" aria-hidden="true" tabindex="-1"></a><span class="fu">heatmap.2</span>(<span class="fu">cor</span>(DO_dge_autosomal),</span>
<span id="cb27-16"><a href="#cb27-16" aria-hidden="true" tabindex="-1"></a>          <span class="at">trace =</span> <span class="st">&quot;none&quot;</span>,</span>
<span id="cb27-17"><a href="#cb27-17" aria-hidden="true" tabindex="-1"></a>          <span class="at">col =</span> heatmapColors,</span>
<span id="cb27-18"><a href="#cb27-18" aria-hidden="true" tabindex="-1"></a>          <span class="at">dendrogram =</span> <span class="st">&quot;both&quot;</span>,</span>
<span id="cb27-19"><a href="#cb27-19" aria-hidden="true" tabindex="-1"></a>          <span class="at">main =</span> <span class="st">&quot;autosomal expression correlation&quot;</span>)</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-20-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-20-1">
Past versions of unnamed-chunk-20-1.png
</button>
</p>
<div id="fig-unnamed-chunk-20-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
dee6333
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The samples 7613, 7624, 7641 are poorly correlated with the rest of
the samples and can be considered outliers. They may be filtered out of
the expression set for downstream work.</p>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a><span class="fu">heatmap.2</span>(<span class="fu">cor</span>(DO_dge_autosomal[,(<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(<span class="fu">colnames</span>(DO_dge_autosomal) <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">&quot;7624&quot;</span>,<span class="st">&quot;7641&quot;</span>,<span class="st">&quot;7613&quot;</span>)))]),</span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>          <span class="at">trace =</span> <span class="st">&quot;none&quot;</span>,</span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a>          <span class="at">col =</span> heatmapColors,</span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a>          <span class="at">dendrogram =</span> <span class="st">&quot;both&quot;</span>,</span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>          <span class="at">main =</span> <span class="st">&quot;autosomal expression correlation with exclusions&quot;</span>)</span></code></pre></div>
<p><img src="figure/05_DO_expression_prep_STAR_RSEM.Rmd/unnamed-chunk-21-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-21-1">
Past versions of unnamed-chunk-21-1.png
</button>
</p>
<div id="fig-unnamed-chunk-21-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
a292466
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
</tr>
<tr>
<td>
7687eb6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-04-19
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
661f8d8
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>This looks as clean as the dataset will get. Using these criteria for
downstream analysis.</p>
<p><em>X aneuploidy may be interesting later, but for now, they are
filtered out as adding variance that is not specifically of interest to
the question of what genes’ expression is correlated with the
behaviors.</em> Using this and a number of other factors to generate a
variable of columns that should not be used in downstream expression
analyses.</p>
<p>Because this is a correlational analysis, using a filtering criterion
of at least 1 CPM in at least 50% of the samples.</p>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a>DO_col_filter <span class="ot">=</span> <span class="fu">unique</span>(<span class="fu">c</span>(<span class="st">&quot;7624&quot;</span>,<span class="st">&quot;7641&quot;</span>,<span class="st">&quot;7613&quot;</span>,<span class="fu">row.names</span>(DO_pheno[<span class="fu">which</span>(DO_pheno<span class="sc">$</span>XAneuploidy <span class="sc">!=</span> <span class="st">&quot;NotAneuoploid&quot;</span>),])))</span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a>DO_dge_filter <span class="ot">=</span> <span class="fu">filterByExpr</span>(DO_dge) <span class="co"># rowSums(cpm(DO_dge) &gt; 1) &gt; (ncol(DO_dge) / 2)</span></span></code></pre></div>
<pre><code>Warning in filterByExpr.DGEList(DO_dge): All samples appear to belong to the
same group.</code></pre>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a>DO_dge_filter[rRNA_7S_genes] <span class="ot">=</span> <span class="cn">FALSE</span></span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a>DO_dge_filter <span class="ot">=</span> DO_dge_filter[<span class="fu">row.names</span>(DO_dge)]</span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a>DO_dge_notkept_rows <span class="ot">=</span> DO_dge[<span class="fu">which</span>(<span class="sc">!</span>(DO_dge_filter)),]</span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a>DO_dge_notkept_cols <span class="ot">=</span> DO_dge[,<span class="fu">which</span>(<span class="fu">colnames</span>(DO_dge) <span class="sc">%in%</span> DO_col_filter)]</span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a>DO_pheno<span class="sc">$</span>notkept_cols <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;kept&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(DO_pheno))</span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a>DO_pheno[DO_col_filter,<span class="st">&quot;notkept_cols&quot;</span>] <span class="ot">=</span> <span class="st">&quot;not_kept&quot;</span></span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a>DO_dge_unfiltered <span class="ot">=</span> DO_dge</span>
<span id="cb31-8"><a href="#cb31-8" aria-hidden="true" tabindex="-1"></a>DO_dge <span class="ot">=</span> DO_dge[<span class="fu">which</span>(DO_dge_filter),]</span></code></pre></div>
<p>Performing TMM normalization on the DGEList.</p>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a>DO_dge <span class="ot">=</span> <span class="fu">calcNormFactors</span>(DO_dge, <span class="at">method =</span> <span class="st">&quot;TMM&quot;</span>)</span></code></pre></div>
<p>Doing a log2 transform and a normal rank-transform of the data.</p>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>DO_voom <span class="ot">=</span> <span class="fu">voom</span>(DO_dge)</span></code></pre></div>
<div class="sourceCode" id="cb34"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb34-1"><a href="#cb34-1" aria-hidden="true" tabindex="-1"></a>DO_normrt <span class="ot">=</span> <span class="fu">matrix</span>(<span class="at">nrow =</span> <span class="fu">nrow</span>(DO_voom), <span class="at">ncol =</span> <span class="fu">ncol</span>(DO_voom))</span>
<span id="cb34-2"><a href="#cb34-2" aria-hidden="true" tabindex="-1"></a><span class="fu">rownames</span>(DO_normrt) <span class="ot">=</span> <span class="fu">rownames</span>(DO_voom)</span>
<span id="cb34-3"><a href="#cb34-3" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_normrt) <span class="ot">=</span> <span class="fu">colnames</span>(DO_voom)</span>
<span id="cb34-4"><a href="#cb34-4" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(DO_normrt)) {</span>
<span id="cb34-5"><a href="#cb34-5" aria-hidden="true" tabindex="-1"></a>  gene_i <span class="ot">=</span> <span class="fu">rownames</span>(DO_normrt)[i]</span>
<span id="cb34-6"><a href="#cb34-6" aria-hidden="true" tabindex="-1"></a>  DO_normrt[i,] <span class="ot">=</span> <span class="fu">norm_rank_transform</span>(DO_voom<span class="sc">$</span>E[gene_i,<span class="fu">colnames</span>(DO_normrt)])</span>
<span id="cb34-7"><a href="#cb34-7" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb34-8"><a href="#cb34-8" aria-hidden="true" tabindex="-1"></a>DO_normrt <span class="ot">=</span> DO_normrt[,<span class="fu">colnames</span>(DO_voom)]</span></code></pre></div>
<p>Collapsing and reordering the phenotype data to the same size as the
<code>voom</code> object</p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a>DO_pheno <span class="ot">=</span> DO_pheno[<span class="fu">colnames</span>(DO_voom),]</span></code></pre></div>
<p>Preparing ERCC DGEList</p>
<div class="sourceCode" id="cb36"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1" aria-hidden="true" tabindex="-1"></a>DO_ERCC_DGEList <span class="ot">=</span> <span class="fu">DGEList</span>(<span class="fu">as.matrix</span>(DO_ERCC_counts))</span>
<span id="cb36-2"><a href="#cb36-2" aria-hidden="true" tabindex="-1"></a>DO_ERCC_DGEList<span class="sc">$</span>samples[<span class="fu">colnames</span>(DO_dge),<span class="st">&quot;norm.factors&quot;</span>] <span class="ot">=</span> DO_dge<span class="sc">$</span>samples<span class="sc">$</span>norm.factors</span>
<span id="cb36-3"><a href="#cb36-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-4"><a href="#cb36-4" aria-hidden="true" tabindex="-1"></a>DO_ERCC_cpm <span class="ot">=</span> <span class="fu">cpm</span>(DO_ERCC_DGEList, <span class="at">normalized.lib.sizes =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</div>
<div id="saving-data-for-downstream-analysis." class="section level3">
<h3>Saving data for downstream analysis.</h3>
<p>Saving <code>voom</code> and <code>DGEList</code> objects as well as
the phenotype data frame.</p>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_dge,<span class="fu">paste</span>(<span class="st">&quot;./data/DO_&quot;</span>,<span class="fu">ncol</span>(DO_dge),<span class="st">&quot;_DGEList.RDS&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>),<span class="at">compress=</span><span class="st">&quot;xz&quot;</span>)</span>
<span id="cb37-2"><a href="#cb37-2" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_dge_unfiltered,<span class="fu">paste</span>(<span class="st">&quot;./data/DO_&quot;</span>,<span class="fu">ncol</span>(DO_dge_unfiltered),<span class="st">&quot;_DGEList_unfiltered.RDS&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>),<span class="at">compress=</span><span class="st">&quot;xz&quot;</span>)</span>
<span id="cb37-3"><a href="#cb37-3" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_voom,<span class="fu">paste</span>(<span class="st">&quot;./data/DO_&quot;</span>,<span class="fu">ncol</span>(DO_voom),<span class="st">&quot;_voom.RDS&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>),<span class="at">compress=</span><span class="st">&quot;xz&quot;</span>)</span>
<span id="cb37-4"><a href="#cb37-4" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_normrt,<span class="fu">paste</span>(<span class="st">&quot;./data/DO_&quot;</span>,<span class="fu">ncol</span>(DO_normrt),<span class="st">&quot;_normrt.RDS&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>),<span class="at">compress=</span><span class="st">&quot;xz&quot;</span>)</span>
<span id="cb37-5"><a href="#cb37-5" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_pheno,<span class="fu">paste</span>(<span class="st">&quot;./data/DO_&quot;</span>,<span class="fu">nrow</span>(DO_pheno),<span class="st">&quot;_pheno.RDS&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>),<span class="at">compress=</span><span class="st">&quot;xz&quot;</span>)</span>
<span id="cb37-6"><a href="#cb37-6" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_ERCC_cpm,<span class="fu">paste</span>(<span class="st">&quot;./data/DO_&quot;</span>,<span class="fu">nrow</span>(DO_pheno),<span class="st">&quot;_ERCC_cpm.RDS&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>),<span class="at">compress=</span><span class="st">&quot;xz&quot;</span>)</span>
<span id="cb37-7"><a href="#cb37-7" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_ERCC_DGEList,<span class="fu">paste</span>(<span class="st">&quot;./data/DO_&quot;</span>,<span class="fu">nrow</span>(DO_pheno),<span class="st">&quot;_ERCC_DGEList.RDS&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>),<span class="at">compress=</span><span class="st">&quot;xz&quot;</span>)</span></code></pre></div>
<p>The md5 checksums for the output files are:</p>
<table>
<colgroup>
<col width="46%" />
<col width="53%" />
</colgroup>
<thead>
<tr class="header">
<th>file</th>
<th>md5 checksum</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><code>./data/DO_416_DGEList.RDS</code></td>
<td><code>dac563acbd2231713221a8a962385a50</code></td>
</tr>
<tr class="even">
<td><code>./data/DO_416_voom.RDS</code></td>
<td><code>2440e7216cbdf9c68a0343babfc7c264</code></td>
</tr>
<tr class="odd">
<td><code>./data/DO_416_normrt.RDS</code></td>
<td><code>7dd9ef204e017571e25961667d10f015</code></td>
</tr>
<tr class="even">
<td><code>./data/DO_416_pheno.RDS</code></td>
<td><code>0f0cd88dcb1483202194f5ea78b832d5</code></td>
</tr>
<tr class="odd">
<td><code>./data/DO_416_ERCC_DGEList.RDS</code></td>
<td><code>47b4cb26c3234d3dbbd7d95ea5e61cb6</code></td>
</tr>
</tbody>
</table>
</div>
<div id="document-control" class="section level3">
<h3>Document Control</h3>
<p>This document was prepared using RMarkdown in RStudio. This document
was imported into workflowr on 2020-05-04.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb38"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.6 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/liblapack.so.3

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ggbeeswarm_0.7.2   biomaRt_2.54.1     RColorBrewer_1.1-3 gplots_3.1.3      
[5] ggplot2_3.4.2      edgeR_3.40.2       tximport_1.26.1    tibble_3.2.1      
[9] limma_3.54.2      

loaded via a namespace (and not attached):
 [1] bitops_1.0-7           fs_1.5.2               bit64_4.0.5           
 [4] filelock_1.0.2         progress_1.2.2         httr_1.4.4            
 [7] rprojroot_2.0.3        GenomeInfoDb_1.34.9    tools_4.2.1           
[10] bslib_0.4.0            utf8_1.2.2             R6_2.5.1              
[13] vipor_0.4.5            KernSmooth_2.23-20     DBI_1.1.3             
[16] BiocGenerics_0.44.0    colorspace_2.0-3       withr_2.5.0           
[19] tidyselect_1.2.0       prettyunits_1.1.1      bit_4.0.4             
[22] curl_4.3.3             compiler_4.2.1         git2r_0.32.0          
[25] cli_3.4.1              Biobase_2.58.0         xml2_1.3.3            
[28] sass_0.4.2             caTools_1.18.2         scales_1.2.1          
[31] readr_2.1.3            rappdirs_0.3.3         stringr_1.5.0         
[34] digest_0.6.30          rmarkdown_2.17         XVector_0.38.0        
[37] pkgconfig_2.0.3        htmltools_0.5.3        highr_0.9             
[40] dbplyr_2.2.1           fastmap_1.1.0          rlang_1.1.1           
[43] rstudioapi_0.14        RSQLite_2.2.18         jquerylib_0.1.4       
[46] generics_0.1.3         jsonlite_1.8.3         vroom_1.6.0           
[49] gtools_3.9.4           dplyr_1.1.2            RCurl_1.98-1.9        
[52] magrittr_2.0.3         GenomeInfoDbData_1.2.9 Rcpp_1.0.9            
[55] munsell_0.5.0          S4Vectors_0.36.2       fansi_1.0.3           
[58] lifecycle_1.0.3        stringi_1.7.8          whisker_0.4           
[61] yaml_2.3.6             zlibbioc_1.44.0        BiocFileCache_2.6.1   
[64] grid_4.2.1             blob_1.2.3             parallel_4.2.1        
[67] promises_1.2.0.1       crayon_1.5.2           lattice_0.20-45       
[70] Biostrings_2.66.0      hms_1.1.2              KEGGREST_1.38.0       
[73] locfit_1.5-9.6         knitr_1.42             pillar_1.9.0          
[76] stats4_4.2.1           XML_3.99-0.11          glue_1.6.2            
[79] evaluate_0.17          tzdb_0.3.0             png_0.1-7             
[82] vctrs_0.6.2            httpuv_1.6.6           purrr_1.0.1           
[85] gtable_0.3.1           assertthat_0.2.1       cachem_1.0.6          
[88] xfun_0.39              later_1.3.0            beeswarm_0.4.0        
[91] AnnotationDbi_1.60.2   memoise_2.0.1          IRanges_2.32.0        
[94] workflowr_1.7.0        ellipsis_0.3.2        </code></pre>
</div>
</div>
</div>
</div>
<div class="footnotes footnotes-end-of-document">
<hr />
<ol>
<li id="fn1"><p>Sathyanesan M, Girgenti MJ, Banasr M, Stone K, Bruce C,
Guilchicek E, Wilczak-Havill K, Nairn A, Williams K, Sass S, Duman JG,
Newton SS. 2012. <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3410626/">A molecular
characterization of the choroid plexus and stress-induced gene
regulation.</a> <em>Transl Psychiatry</em> 2:e139. <a
href="https://doi.org/10.1038/tp.2012.64">doi:10.1038/tp.2012.64</a><a
href="#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>Chesler EJ, Gatti DM, Morgan AP, Strobel M, Trepanier L,
Oberbeck D, McWeeney S, Hitzemann R, Ferris M, McMullan R, Clayshultle
A, Bell TA, Pardo-Manuel de Villena F, Churchill GA. 2016. <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5144960/">Diversity
Outbred mice at 21: maintaining allelic variation in the face of
selection.</a> <em>G3: Genes, Genomes, Genetics</em> 6:3893–3902. <a
href="https://doi.org/10.1534/g3.116.035527">doi:10.1534/g3.116.035527</a><a
href="#fnref2" class="footnote-back">↩︎</a></p></li>
<li id="fn3"><p>Chesler EJ, Gatti DM, Morgan AP, Strobel M, Trepanier L,
Oberbeck D, McWeeney S, Hitzemann R, Ferris M, McMullan R, Clayshultle
A, Bell TA, Pardo-Manuel de Villena F, Churchill GA. 2016. <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5144960/">Diversity
Outbred mice at 21: maintaining allelic variation in the face of
selection.</a> <em>G3: Genes, Genomes, Genetics</em> 6:3893–3902. <a
href="https://doi.org/10.1534/g3.116.035527">doi:10.1534/g3.116.035527</a><a
href="#fnref3" class="footnote-back">↩︎</a></p></li>
</ol>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
