<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Michael C. Saul (michael.saul [at] jax.org)" />


<title>DO Aneuploidy: Followup to Expression</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>


<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Diversity Outbred Reference Traits Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">DO Aneuploidy: Followup to Expression</h1>
<h4 class="author">Michael C. Saul (michael.saul [at] jax.org)</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-07
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rta/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20200501code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20200501)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20200501code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20200501)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong868cd0f"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 868cd0f </a>
</p>
</div>
<div id="strongRepositoryversionstrong868cd0f" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 868cd0f. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/.Rapp.history
    Ignored:    data/.Rhistory
    Ignored:    data/RDS/
    Ignored:    run/

Untracked files:
    Untracked:  LICENSE
    Untracked:  Untitled.R
    Untracked:  analysis/17_mapping_interpretation_old.Rmd
    Untracked:  analysis/20_predict_acquisition_from_novelty.Rmd
    Untracked:  analysis/21_DO_elasticnet_correlation.Rmd
    Untracked:  analysis/temp.txt
    Untracked:  code/DO_eQTL_perms.R
    Untracked:  code/DO_eQTL_score_calc.R
    Untracked:  code/STAR_RSEM_416.sh
    Untracked:  code/eQTL_plotting_DO_striatum_416.R
    Untracked:  code/ggplot_pub_themes.R
    Untracked:  data/20_elastic_mv_results.csv
    Untracked:  data/20_elasticnet_predictions.RDS
    Untracked:  data/DO_416_DGEList.RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 3).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 4).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_DGEList_unfiltered.RDS
    Untracked:  data/DO_416_ERCC_DGEList.RDS
    Untracked:  data/DO_416_ERCC_cpm.RDS
    Untracked:  data/DO_416_normrt.RDS
    Untracked:  data/DO_416_pheno.RDS
    Untracked:  data/DO_416_pheno_allgenes.RDS
    Untracked:  data/DO_416_voom (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom.RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom_allgenes.RDS
    Untracked:  data/DO_416_voom_cpresiduals.RDS
    Untracked:  data/DO_AQ_active_CoV_toptable_annotated.RDS
    Untracked:  data/DO_AQ_lever_ratio_toptable_annotated.RDS
    Untracked:  data/DO_AQ_mean_inf_toptable_annotated.RDS
    Untracked:  data/DO_AQ_rate_toptable_annotated.RDS
    Untracked:  data/DO_IVSA_acquisition_cancor_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_glm.RDS
    Untracked:  data/DO_IVSA_all_cancor_df.RDS
    Untracked:  data/DO_IVSA_data.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_ALL.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_AQ.RDS
    Untracked:  data/DO_IVSA_novelty_data.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_old.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_a1_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_a2_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a4_male_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a5_female_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_raw_data.RDS
    Untracked:  data/DO_IVSA_transformed_data.RDS
    Untracked:  data/DO_LIMS_data.RDS
    Untracked:  data/DO_RTG_a1_new.qtl.RData
    Untracked:  data/DO_RTG_a1_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a1_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a2_new.qtl.RData
    Untracked:  data/DO_RTG_a2_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a2_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a3_new.qtl.RData
    Untracked:  data/DO_RTG_a3_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_01.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_02.RData
    Untracked:  data/DO_RTG_a4_new.qtl.RData
    Untracked:  data/DO_RTG_a5_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_project_df_for_mapping.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_2020-11-06.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_a2.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_sexspecific.RDS
    Untracked:  data/DO_demographic_data_df.RDS
    Untracked:  data/DO_eigengene_significance.RDS
    Untracked:  data/DO_eigengenes.RDS
    Untracked:  data/DO_highcor_cpresiduals_datatable.RDS
    Untracked:  data/DO_holeboard_data_df.RDS
    Untracked:  data/DO_holeboard_summarized.RDS
    Untracked:  data/DO_lightdark_data_df.RDS
    Untracked:  data/DO_mQTL_peaks.RDS
    Untracked:  data/DO_mQTL_scan1s.RDS
    Untracked:  data/DO_module_membership.RDS
    Untracked:  data/DO_novelplace_data_df.RDS
    Untracked:  data/DO_novelty_data.RDS
    Untracked:  data/DO_novelty_response_data.RDS
    Untracked:  data/DO_novelty_response_data_for_RTG.RDS
    Untracked:  data/DO_novelty_response_data_with_transform.csv
    Untracked:  data/DO_openfield_data_df.RDS
    Untracked:  data/DO_paraclique_data.el
    Untracked:  data/DO_paraclique_eigengenes.RDS
    Untracked:  data/DO_paracliques.RDS
    Untracked:  data/DO_paracliques_significance.RDS
    Untracked:  data/DO_ploidy_calls.RDS
    Untracked:  data/DO_pmap.RDS
    Untracked:  data/DO_pvalue_histograms.RData
    Untracked:  data/DO_rtg_a1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a3_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m3_toptable_annotated.RDS
    Untracked:  data/DO_str_2016_eQTL.RData
    Untracked:  data/DO_str_2016_gm4qtl2.zip
    Untracked:  data/DO_str_2016_gm4qtl2_intensities.fst
    Untracked:  data/DO_voom_cpresiduals_NArm.RDS
    Untracked:  data/DO_wgcna.RDS
    Untracked:  data/IVSA_RTG_a3_blup.pdf
    Untracked:  data/IVSA_h2_2019-11-18.tsv
    Untracked:  data/IVSA_metadata_sheet.RDS
    Untracked:  data/ME16_edges.tsv
    Untracked:  data/Mus_musculus.GRCm38.94.parsed.RDS
    Untracked:  data/Paraclique_19_MF_AMIGO.txt
    Untracked:  data/RTG_QTL_peaks.RDS
    Untracked:  data/RTG_figure.RDS
    Untracked:  data/SENS_RTG/
    Untracked:  data/Tyr2_genotypes.RDS
    Untracked:  data/Tyr_genotypes.RDS
    Untracked:  data/blup_chr7_DO_RTG_a3.RDS
    Untracked:  data/cc_variants/
    Untracked:  data/chr18_QTL_results.RDS
    Untracked:  data/chr4_QTL_results.RDS
    Untracked:  data/cis/
    Untracked:  data/eQTL_annotated_peaks.RDS
    Untracked:  data/eQTL_plots.RData
    Untracked:  data/figures/
    Untracked:  data/founders_key/
    Untracked:  data/gigamuga/
    Untracked:  data/gwas/
    Untracked:  data/info/
    Untracked:  data/m2G.a2.RTG.chr4snp.RData
    Untracked:  data/mapping_from_hao/
    Untracked:  data/perms/
    Untracked:  data/qtl2/
    Untracked:  data/raw/
    Untracked:  data/resources/
    Untracked:  data/tads_loops/
    Untracked:  data/transcripts/
    Untracked:  data/variants/
    Untracked:  logs/
    Untracked:  new_RTG_a3.jpeg
    Untracked:  output/RTG_out/
    Untracked:  output/docs_backup_2020-07-30.tar.gz
    Untracked:  output/elastic_net_out/
    Untracked:  singularity/
    Untracked:  tmp/
    Untracked:  var/

Unstaged changes:
    Deleted:    .Rprofile
    Modified:   .gitignore
    Modified:   README.md
    Modified:   analysis/17_mapping_interpretation.Rmd
    Modified:   analysis/_site.yml

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/06_DO_ploidy_followup_to_expression.Rmd</code>) and HTML (<code>docs/06_DO_ploidy_followup_to_expression.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
868cd0f
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-07
</td>
<td>
Initial commit for Sumner with alteration for ggplot2
</td>
</tr>
<tr>
<td>
html
</td>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
<td>
Rolling back to working version.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
3799d63
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
839900c
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
ae65e65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
<td>
Commit ploidy analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="background" class="section level1">
<h1>Background</h1>
<div id="motivation" class="section level2">
<h2>Motivation</h2>
<div id="purpose-of-analysis" class="section level3">
<h3>Purpose of analysis</h3>
<p>This analysis is performed to follow up X and Y chromosome aneuploidy suggestion from DO striatum expression analysis.</p>
</div>
</div>
</div>
<div id="analysis" class="section level1">
<h1>Analysis</h1>
<div id="setup" class="section level2">
<h2>Setup</h2>
<div id="r-libraries" class="section level3">
<h3>R libraries</h3>
<p>Calling R libraries necessary for this analysis.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Using the argyle library from the following GitHub repository: andrewparkermorgan/argyle</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;argyle&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;argyle&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:stats&#39;:

    dist, heatmap</code></pre>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;tools&quot;</span>)</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;cowplot&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;cowplot&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:argyle&#39;:

    theme_nothing</code></pre>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggplot2&quot;</span>)</span></code></pre></div>
</div>
</div>
</div>
<div id="data-import" class="section level1">
<h1>Data Import</h1>
<div id="pre-data-import" class="section level2">
<h2>Pre-data import</h2>
<div id="getting-norm_rank_transform-function-that-can-handle-na-values." class="section level3">
<h3>Getting <code>norm_rank_transform</code> function that can handle <code>NA</code> values.</h3>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>norm_rank_transform <span class="ot">=</span> <span class="cf">function</span>(x, <span class="at">c =</span> (<span class="dv">0</span>)) {</span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stopifnot</span>(<span class="fu">is.numeric</span>(x) <span class="sc">&amp;</span> <span class="fu">is.vector</span>(x))</span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a>  x_noNA <span class="ot">=</span> <span class="fu">which</span>(<span class="sc">!</span><span class="fu">is.na</span>(x))</span>
<span id="cb8-4"><a href="#cb8-4" aria-hidden="true" tabindex="-1"></a>  N <span class="ot">=</span> <span class="fu">length</span>(x_noNA)</span>
<span id="cb8-5"><a href="#cb8-5" aria-hidden="true" tabindex="-1"></a>  x[x_noNA] <span class="ot">=</span> <span class="fu">qnorm</span>((<span class="fu">rank</span>(x[x_noNA], <span class="at">ties.method =</span> <span class="st">&quot;random&quot;</span>) <span class="sc">-</span> c) <span class="sc">/</span> (N <span class="sc">-</span> (<span class="dv">2</span> <span class="sc">*</span> c) <span class="sc">+</span> <span class="dv">1</span>))</span>
<span id="cb8-6"><a href="#cb8-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(x)</span>
<span id="cb8-7"><a href="#cb8-7" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
</div>
</div>
<div id="importing-data" class="section level2">
<h2>Importing data</h2>
<div id="importing-snp-data-for-the-gigamuga-chipset." class="section level3">
<h3>Importing SNP data for the GigaMUGA chipset.</h3>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="co"># The GigaMUGA consensus file was found at the following location on the JAX filesystem:</span></span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a><span class="co"># /projects/churchill-lab/resource_data/muga_reference/gigamuga_consensus_alleles.csv</span></span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a><span class="co"># The file on the server has the following MD5 checksum:</span></span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a><span class="co"># cfb5e353f959cab0451619a3eaa8a796</span></span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="fu">ifelse</span>(<span class="fu">md5sum</span>(<span class="st">&quot;./data/resources/gigamuga_consensus_alleles.csv&quot;</span>) <span class="sc">==</span> <span class="st">&quot;cfb5e353f959cab0451619a3eaa8a796&quot;</span>,</span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a>           <span class="st">&quot;The md5 checksum of the v38 GigaMUGA consensus allele file MATCHES the Churchill Lab&#39;s file.&quot;</span>,</span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a>           <span class="st">&quot;The md5 checksum of the v38 GigaMUGA consensus allele file DOES NOT MATCH the Churchill Lab&#39;s file.&quot;</span>))</span></code></pre></div>
<pre><code>The md5 checksum of the v38 GigaMUGA consensus allele file MATCHES the Churchill Lab&#39;s file.</code></pre>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a>gigamuga_consensus_alleles <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">&quot;./data/resources/gigamuga_consensus_alleles.csv&quot;</span>,</span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a>                                        <span class="at">sep =</span> <span class="st">&quot;,&quot;</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="fu">url</span>(<span class="st">&quot;ftp://ftp.jax.org/MUGA/GM_snps.Rdata&quot;</span>))</span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="fu">url</span>(<span class="st">&quot;http://csbio.unc.edu/MUGA/snps.gigamuga.Rdata&quot;</span>))</span></code></pre></div>
<p>Refining the GigaMUGA consensus allele file</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(gigamuga_consensus_alleles) <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;Marker&quot;</span>,<span class="st">&quot;ChrB38&quot;</span>,<span class="st">&quot;PosB38&quot;</span>,</span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a>                                         <span class="st">&quot;allele_129&quot;</span>,<span class="st">&quot;allele_AJ&quot;</span>,<span class="st">&quot;allele_B6&quot;</span>,<span class="st">&quot;allele_CAST&quot;</span>,</span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>                                         <span class="st">&quot;allele_NOD&quot;</span>,<span class="st">&quot;allele_NZO&quot;</span>,<span class="st">&quot;allele_PWK&quot;</span>,<span class="st">&quot;allele_WSB&quot;</span>)</span>
<span id="cb12-4"><a href="#cb12-4" aria-hidden="true" tabindex="-1"></a>gigamuga_allele_cols <span class="ot">=</span> <span class="fu">grep</span>(<span class="st">&quot;^allele&quot;</span>, <span class="fu">colnames</span>(gigamuga_consensus_alleles))</span>
<span id="cb12-5"><a href="#cb12-5" aria-hidden="true" tabindex="-1"></a>gigamuga_consensus_alleles<span class="sc">$</span>ChrB38 <span class="ot">=</span> <span class="fu">paste</span>(<span class="st">&quot;chr&quot;</span>,gigamuga_consensus_alleles<span class="sc">$</span>ChrB38,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)</span>
<span id="cb12-6"><a href="#cb12-6" aria-hidden="true" tabindex="-1"></a>gigamuga_informative_alleles <span class="ot">=</span> gigamuga_consensus_alleles[<span class="fu">which</span>(gigamuga_consensus_alleles<span class="sc">$</span>ChrB38 <span class="sc">!=</span> <span class="st">&quot;chrNA&quot;</span>),]</span>
<span id="cb12-7"><a href="#cb12-7" aria-hidden="true" tabindex="-1"></a>gigamuga_informative_alleles <span class="ot">=</span> gigamuga_informative_alleles[<span class="fu">which</span>(<span class="fu">rowSums</span>(gigamuga_informative_alleles[,gigamuga_allele_cols] <span class="sc">==</span> <span class="st">&quot;N&quot;</span>) <span class="sc">!=</span> <span class="dv">8</span>),]</span>
<span id="cb12-8"><a href="#cb12-8" aria-hidden="true" tabindex="-1"></a>alleles_129 <span class="ot">=</span> gigamuga_informative_alleles[,<span class="st">&quot;allele_129&quot;</span>]</span>
<span id="cb12-9"><a href="#cb12-9" aria-hidden="true" tabindex="-1"></a>gigamuga_informative_alleles <span class="ot">=</span> gigamuga_informative_alleles[<span class="fu">which</span>(<span class="fu">rowSums</span>(gigamuga_informative_alleles[,gigamuga_allele_cols] <span class="sc">==</span> alleles_129) <span class="sc">!=</span> <span class="dv">8</span>),]</span>
<span id="cb12-10"><a href="#cb12-10" aria-hidden="true" tabindex="-1"></a><span class="fu">rownames</span>(gigamuga_informative_alleles) <span class="ot">=</span> <span class="fu">as.character</span>(gigamuga_informative_alleles[,<span class="st">&quot;Marker&quot;</span>])</span></code></pre></div>
</div>
<div id="importing-the-gigamuga-data-using-argyle" class="section level3">
<h3>Importing the GigaMUGA data using <code>argyle</code></h3>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Using input file from Vivek downloaded off of cluster.</span></span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a>gigamuga_input <span class="ot">=</span> <span class="fu">read.table</span>(<span class="at">file =</span> <span class="st">&quot;./data/gigamuga/gigamuga_input_20181120.txt&quot;</span>, </span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a>                            <span class="at">sep =</span> <span class="st">&quot;</span><span class="sc">\t</span><span class="st">&quot;</span>, <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>, <span class="at">header =</span> <span class="cn">TRUE</span>)</span>
<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-5"><a href="#cb13-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Changing directory IDs to local IDs.</span></span>
<span id="cb13-6"><a href="#cb13-6" aria-hidden="true" tabindex="-1"></a>gigamuga_input<span class="sc">$</span>dir <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^/fastscratch/vmp/csna/&quot;</span>, </span>
<span id="cb13-7"><a href="#cb13-7" aria-hidden="true" tabindex="-1"></a>                          <span class="st">&quot;./data/gigamuga/&quot;</span>,</span>
<span id="cb13-8"><a href="#cb13-8" aria-hidden="true" tabindex="-1"></a>                          gigamuga_input<span class="sc">$</span>dir)</span>
<span id="cb13-9"><a href="#cb13-9" aria-hidden="true" tabindex="-1"></a>gigamuga_input<span class="sc">$</span>id <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^./data/gigamuga/&quot;</span>,</span>
<span id="cb13-10"><a href="#cb13-10" aria-hidden="true" tabindex="-1"></a>                         <span class="st">&quot;&quot;</span>,</span>
<span id="cb13-11"><a href="#cb13-11" aria-hidden="true" tabindex="-1"></a>                         gigamuga_input<span class="sc">$</span>dir)</span>
<span id="cb13-12"><a href="#cb13-12" aria-hidden="true" tabindex="-1"></a>gigamuga_input<span class="sc">$</span>id <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;/$&quot;</span>,</span>
<span id="cb13-13"><a href="#cb13-13" aria-hidden="true" tabindex="-1"></a>                         <span class="st">&quot;&quot;</span>,</span>
<span id="cb13-14"><a href="#cb13-14" aria-hidden="true" tabindex="-1"></a>                         gigamuga_input<span class="sc">$</span>id)</span>
<span id="cb13-15"><a href="#cb13-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-16"><a href="#cb13-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Looping through genotype files to get intensities</span></span>
<span id="cb13-17"><a href="#cb13-17" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(gigamuga_input)) {</span>
<span id="cb13-18"><a href="#cb13-18" aria-hidden="true" tabindex="-1"></a>  geno_i <span class="ot">=</span> <span class="fu">read.beadstudio</span>(<span class="at">prefix =</span> gigamuga_input[i,<span class="st">&quot;prefix&quot;</span>], </span>
<span id="cb13-19"><a href="#cb13-19" aria-hidden="true" tabindex="-1"></a>                           <span class="at">snps =</span> snps, </span>
<span id="cb13-20"><a href="#cb13-20" aria-hidden="true" tabindex="-1"></a>                           <span class="at">in.path =</span> gigamuga_input[i,<span class="st">&quot;dir&quot;</span>])</span>
<span id="cb13-21"><a href="#cb13-21" aria-hidden="true" tabindex="-1"></a>  <span class="co"># This line saves each file.</span></span>
<span id="cb13-22"><a href="#cb13-22" aria-hidden="true" tabindex="-1"></a>  <span class="co"># saveRDS(geno_i, file = paste(&quot;./data/genotypes/geno_&quot;,gigamuga_input[i,&quot;id&quot;],&quot;.RDS&quot;,sep=&quot;&quot;))</span></span>
<span id="cb13-23"><a href="#cb13-23" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-24"><a href="#cb13-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">assign</span>(<span class="fu">paste</span>(<span class="st">&quot;samples_in_&quot;</span>,gigamuga_input[i,<span class="st">&quot;id&quot;</span>],<span class="at">sep=</span><span class="st">&quot;&quot;</span>), <span class="fu">colnames</span>(geno_i))</span>
<span id="cb13-25"><a href="#cb13-25" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-26"><a href="#cb13-26" aria-hidden="true" tabindex="-1"></a>  sex_predict_xy_i <span class="ot">=</span> <span class="fu">predict.sex</span>(geno_i, <span class="at">method =</span> <span class="st">&quot;xy&quot;</span>, <span class="at">platform =</span> <span class="st">&quot;giga&quot;</span>)</span>
<span id="cb13-27"><a href="#cb13-27" aria-hidden="true" tabindex="-1"></a>  sex_predict_ycalls_i <span class="ot">=</span> <span class="fu">predict.sex</span>(geno_i, <span class="at">method =</span> <span class="st">&quot;ycalls&quot;</span>, <span class="at">platform =</span> <span class="st">&quot;giga&quot;</span>)</span>
<span id="cb13-28"><a href="#cb13-28" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-29"><a href="#cb13-29" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Subsetting to only genomic loci</span></span>
<span id="cb13-30"><a href="#cb13-30" aria-hidden="true" tabindex="-1"></a>  geno_chr_i <span class="ot">=</span> geno_i[<span class="fu">which</span>(<span class="fu">as.character</span>(<span class="fu">attr</span>(geno_i,<span class="st">&quot;map&quot;</span>)[,<span class="st">&quot;chr&quot;</span>]) <span class="sc">%in%</span> <span class="fu">paste</span>(<span class="st">&quot;chr&quot;</span>,<span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">19</span>,<span class="st">&quot;X&quot;</span>,<span class="st">&quot;Y&quot;</span>,<span class="st">&quot;M&quot;</span>),<span class="at">sep=</span><span class="st">&quot;&quot;</span>)),]</span>
<span id="cb13-31"><a href="#cb13-31" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-32"><a href="#cb13-32" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Getting raw genotypes as a matrix</span></span>
<span id="cb13-33"><a href="#cb13-33" aria-hidden="true" tabindex="-1"></a>  geno_mat_i <span class="ot">=</span> <span class="fu">matrix</span>(<span class="fu">as.vector</span>(geno_chr_i), <span class="at">ncol =</span> <span class="fu">dim</span>(geno_chr_i)[<span class="dv">2</span>], <span class="at">nrow =</span> <span class="fu">dim</span>(geno_chr_i)[<span class="dv">1</span>], <span class="at">byrow =</span> <span class="cn">FALSE</span>,</span>
<span id="cb13-34"><a href="#cb13-34" aria-hidden="true" tabindex="-1"></a>                      <span class="at">dimnames =</span> <span class="fu">list</span>(<span class="fu">row.names</span>(geno_chr_i), <span class="fu">colnames</span>(geno_chr_i)))</span>
<span id="cb13-35"><a href="#cb13-35" aria-hidden="true" tabindex="-1"></a>  geno_0_i <span class="ot">=</span> <span class="fu">which</span>(geno_mat_i <span class="sc">==</span> <span class="st">&quot;A&quot;</span> <span class="sc">|</span> geno_mat_i <span class="sc">==</span> <span class="st">&quot;T&quot;</span>, <span class="at">arr.ind =</span> <span class="cn">TRUE</span>)</span>
<span id="cb13-36"><a href="#cb13-36" aria-hidden="true" tabindex="-1"></a>  geno_1_i <span class="ot">=</span> <span class="fu">which</span>(geno_mat_i <span class="sc">==</span> <span class="st">&quot;H&quot;</span>, <span class="at">arr.ind =</span> <span class="cn">TRUE</span>)</span>
<span id="cb13-37"><a href="#cb13-37" aria-hidden="true" tabindex="-1"></a>  geno_2_i <span class="ot">=</span> <span class="fu">which</span>(geno_mat_i <span class="sc">==</span> <span class="st">&quot;C&quot;</span> <span class="sc">|</span> geno_mat_i <span class="sc">==</span> <span class="st">&quot;G&quot;</span>, <span class="at">arr.ind =</span> <span class="cn">TRUE</span>)</span>
<span id="cb13-38"><a href="#cb13-38" aria-hidden="true" tabindex="-1"></a>  geno_NA_i <span class="ot">=</span> <span class="fu">which</span>(geno_mat_i <span class="sc">==</span> <span class="st">&quot;N&quot;</span> <span class="sc">|</span> <span class="fu">is.na</span>(geno_mat_i), <span class="at">arr.ind =</span> <span class="cn">TRUE</span>)</span>
<span id="cb13-39"><a href="#cb13-39" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-40"><a href="#cb13-40" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Getting raw intensity</span></span>
<span id="cb13-41"><a href="#cb13-41" aria-hidden="true" tabindex="-1"></a>  intensity_x_i <span class="ot">=</span> <span class="fu">as.matrix</span>(<span class="fu">intensity</span>(geno_chr_i)<span class="sc">$</span>x)</span>
<span id="cb13-42"><a href="#cb13-42" aria-hidden="true" tabindex="-1"></a>  intensity_y_i <span class="ot">=</span> <span class="fu">as.matrix</span>(<span class="fu">intensity</span>(geno_chr_i)<span class="sc">$</span>y)</span>
<span id="cb13-43"><a href="#cb13-43" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-44"><a href="#cb13-44" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Calculating R for each marker, intensity of x plus intensity of y</span></span>
<span id="cb13-45"><a href="#cb13-45" aria-hidden="true" tabindex="-1"></a>  R_i <span class="ot">=</span> intensity_x_i <span class="sc">+</span> intensity_y_i</span>
<span id="cb13-46"><a href="#cb13-46" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-47"><a href="#cb13-47" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Calculating theta for each marker, ((2 / pi) * atan2(y, x))</span></span>
<span id="cb13-48"><a href="#cb13-48" aria-hidden="true" tabindex="-1"></a>  theta_i <span class="ot">=</span> (<span class="dv">2</span> <span class="sc">/</span> pi) <span class="sc">*</span> <span class="fu">atan2</span>(intensity_y_i, intensity_x_i)</span>
<span id="cb13-49"><a href="#cb13-49" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-50"><a href="#cb13-50" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Calculating clusters data frame for tQN calculations</span></span>
<span id="cb13-51"><a href="#cb13-51" aria-hidden="true" tabindex="-1"></a>  blank_df_i <span class="ot">=</span> <span class="fu">matrix</span>(<span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(geno_mat_i) <span class="sc">*</span> <span class="fu">ncol</span>(geno_mat_i)), </span>
<span id="cb13-52"><a href="#cb13-52" aria-hidden="true" tabindex="-1"></a>                      <span class="at">ncol =</span> <span class="fu">ncol</span>(geno_mat_i), <span class="at">nrow =</span> <span class="fu">nrow</span>(geno_mat_i), <span class="at">byrow =</span> <span class="cn">FALSE</span>,</span>
<span id="cb13-53"><a href="#cb13-53" aria-hidden="true" tabindex="-1"></a>                      <span class="at">dimnames =</span> <span class="fu">list</span>(<span class="fu">row.names</span>(geno_mat_i), <span class="fu">colnames</span>(geno_mat_i)))</span>
<span id="cb13-54"><a href="#cb13-54" aria-hidden="true" tabindex="-1"></a>  clusters_df_i <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">row.names =</span> <span class="fu">row.names</span>(blank_df_i),</span>
<span id="cb13-55"><a href="#cb13-55" aria-hidden="true" tabindex="-1"></a>                             <span class="at">A.R =</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(blank_df_i)),</span>
<span id="cb13-56"><a href="#cb13-56" aria-hidden="true" tabindex="-1"></a>                             <span class="at">A.T =</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(blank_df_i)),</span>
<span id="cb13-57"><a href="#cb13-57" aria-hidden="true" tabindex="-1"></a>                             <span class="at">B.R =</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(blank_df_i)),</span>
<span id="cb13-58"><a href="#cb13-58" aria-hidden="true" tabindex="-1"></a>                             <span class="at">B.T =</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(blank_df_i)),</span>
<span id="cb13-59"><a href="#cb13-59" aria-hidden="true" tabindex="-1"></a>                             <span class="at">H.R =</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(blank_df_i)),</span>
<span id="cb13-60"><a href="#cb13-60" aria-hidden="true" tabindex="-1"></a>                             <span class="at">H.T =</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(blank_df_i)))</span>
<span id="cb13-61"><a href="#cb13-61" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-62"><a href="#cb13-62" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Getting mean R for A</span></span>
<span id="cb13-63"><a href="#cb13-63" aria-hidden="true" tabindex="-1"></a>  A.R_i <span class="ot">=</span> blank_df_i</span>
<span id="cb13-64"><a href="#cb13-64" aria-hidden="true" tabindex="-1"></a>  A.R_i[geno_0_i] <span class="ot">=</span> R_i[geno_0_i]</span>
<span id="cb13-65"><a href="#cb13-65" aria-hidden="true" tabindex="-1"></a>  clusters_df_i<span class="sc">$</span>A.R <span class="ot">=</span> <span class="fu">rowMeans</span>(A.R_i, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb13-66"><a href="#cb13-66" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-67"><a href="#cb13-67" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Getting mean theta for A</span></span>
<span id="cb13-68"><a href="#cb13-68" aria-hidden="true" tabindex="-1"></a>  A.T_i <span class="ot">=</span> blank_df_i</span>
<span id="cb13-69"><a href="#cb13-69" aria-hidden="true" tabindex="-1"></a>  A.T_i[geno_0_i] <span class="ot">=</span> theta_i[geno_0_i]</span>
<span id="cb13-70"><a href="#cb13-70" aria-hidden="true" tabindex="-1"></a>  clusters_df_i<span class="sc">$</span>A.T <span class="ot">=</span> <span class="fu">rowMeans</span>(A.T_i, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb13-71"><a href="#cb13-71" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-72"><a href="#cb13-72" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Getting mean R for B</span></span>
<span id="cb13-73"><a href="#cb13-73" aria-hidden="true" tabindex="-1"></a>  B.R_i <span class="ot">=</span> blank_df_i</span>
<span id="cb13-74"><a href="#cb13-74" aria-hidden="true" tabindex="-1"></a>  B.R_i[geno_2_i] <span class="ot">=</span> R_i[geno_2_i]</span>
<span id="cb13-75"><a href="#cb13-75" aria-hidden="true" tabindex="-1"></a>  clusters_df_i<span class="sc">$</span>B.R <span class="ot">=</span> <span class="fu">rowMeans</span>(B.R_i, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb13-76"><a href="#cb13-76" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-77"><a href="#cb13-77" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Getting mean theta for B</span></span>
<span id="cb13-78"><a href="#cb13-78" aria-hidden="true" tabindex="-1"></a>  B.T_i <span class="ot">=</span> blank_df_i</span>
<span id="cb13-79"><a href="#cb13-79" aria-hidden="true" tabindex="-1"></a>  B.T_i[geno_2_i] <span class="ot">=</span> theta_i[geno_2_i]</span>
<span id="cb13-80"><a href="#cb13-80" aria-hidden="true" tabindex="-1"></a>  clusters_df_i<span class="sc">$</span>B.T <span class="ot">=</span> <span class="fu">rowMeans</span>(B.T_i, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb13-81"><a href="#cb13-81" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-82"><a href="#cb13-82" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Getting mean R for A</span></span>
<span id="cb13-83"><a href="#cb13-83" aria-hidden="true" tabindex="-1"></a>  H.R_i <span class="ot">=</span> blank_df_i</span>
<span id="cb13-84"><a href="#cb13-84" aria-hidden="true" tabindex="-1"></a>  H.R_i[geno_1_i] <span class="ot">=</span> R_i[geno_1_i]</span>
<span id="cb13-85"><a href="#cb13-85" aria-hidden="true" tabindex="-1"></a>  clusters_df_i<span class="sc">$</span>H.R <span class="ot">=</span> <span class="fu">rowMeans</span>(H.R_i, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb13-86"><a href="#cb13-86" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-87"><a href="#cb13-87" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Getting mean theta for H</span></span>
<span id="cb13-88"><a href="#cb13-88" aria-hidden="true" tabindex="-1"></a>  H.T_i <span class="ot">=</span> blank_df_i</span>
<span id="cb13-89"><a href="#cb13-89" aria-hidden="true" tabindex="-1"></a>  H.T_i[geno_1_i] <span class="ot">=</span> theta_i[geno_1_i]</span>
<span id="cb13-90"><a href="#cb13-90" aria-hidden="true" tabindex="-1"></a>  clusters_df_i<span class="sc">$</span>H.T <span class="ot">=</span> <span class="fu">rowMeans</span>(H.T_i, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb13-91"><a href="#cb13-91" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-92"><a href="#cb13-92" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Calculating tQN</span></span>
<span id="cb13-93"><a href="#cb13-93" aria-hidden="true" tabindex="-1"></a>  geno_chr_i <span class="ot">=</span> <span class="fu">tQN</span>(geno_chr_i, <span class="at">clusters =</span> clusters_df_i, <span class="at">xynorm =</span> <span class="cn">FALSE</span>, <span class="at">adjust.lrr =</span> <span class="cn">FALSE</span>)</span>
<span id="cb13-94"><a href="#cb13-94" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-95"><a href="#cb13-95" aria-hidden="true" tabindex="-1"></a>  geno_lrr_i <span class="ot">=</span> <span class="fu">attr</span>(geno_chr_i, <span class="st">&quot;lrr&quot;</span>)</span>
<span id="cb13-96"><a href="#cb13-96" aria-hidden="true" tabindex="-1"></a>  geno_baf_i <span class="ot">=</span> <span class="fu">attr</span>(geno_chr_i, <span class="st">&quot;baf&quot;</span>)</span>
<span id="cb13-97"><a href="#cb13-97" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb13-98"><a href="#cb13-98" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (i <span class="sc">==</span> <span class="dv">1</span>) {</span>
<span id="cb13-99"><a href="#cb13-99" aria-hidden="true" tabindex="-1"></a>    geno_matrix <span class="ot">=</span> geno_mat_i</span>
<span id="cb13-100"><a href="#cb13-100" aria-hidden="true" tabindex="-1"></a>    intensity_x_matrix <span class="ot">=</span> intensity_x_i</span>
<span id="cb13-101"><a href="#cb13-101" aria-hidden="true" tabindex="-1"></a>    intensity_y_matrix <span class="ot">=</span> intensity_y_i</span>
<span id="cb13-102"><a href="#cb13-102" aria-hidden="true" tabindex="-1"></a>    R_matrix <span class="ot">=</span> R_i</span>
<span id="cb13-103"><a href="#cb13-103" aria-hidden="true" tabindex="-1"></a>    theta_matrix <span class="ot">=</span> theta_i</span>
<span id="cb13-104"><a href="#cb13-104" aria-hidden="true" tabindex="-1"></a>    lrr_matrix <span class="ot">=</span> geno_lrr_i</span>
<span id="cb13-105"><a href="#cb13-105" aria-hidden="true" tabindex="-1"></a>    baf_matrix <span class="ot">=</span> geno_baf_i</span>
<span id="cb13-106"><a href="#cb13-106" aria-hidden="true" tabindex="-1"></a>    sex_predict_xy <span class="ot">=</span> sex_predict_xy_i</span>
<span id="cb13-107"><a href="#cb13-107" aria-hidden="true" tabindex="-1"></a>    sex_predict_ycalls <span class="ot">=</span> sex_predict_ycalls_i</span>
<span id="cb13-108"><a href="#cb13-108" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb13-109"><a href="#cb13-109" aria-hidden="true" tabindex="-1"></a>    geno_matrix <span class="ot">=</span> <span class="fu">cbind</span>(geno_matrix, geno_mat_i)</span>
<span id="cb13-110"><a href="#cb13-110" aria-hidden="true" tabindex="-1"></a>    intensity_x_matrix <span class="ot">=</span> <span class="fu">cbind</span>(intensity_x_matrix, intensity_x_i)</span>
<span id="cb13-111"><a href="#cb13-111" aria-hidden="true" tabindex="-1"></a>    intensity_y_matrix <span class="ot">=</span> <span class="fu">cbind</span>(intensity_y_matrix, intensity_y_i)</span>
<span id="cb13-112"><a href="#cb13-112" aria-hidden="true" tabindex="-1"></a>    R_matrix <span class="ot">=</span> <span class="fu">cbind</span>(R_matrix, R_i)</span>
<span id="cb13-113"><a href="#cb13-113" aria-hidden="true" tabindex="-1"></a>    theta_matrix <span class="ot">=</span> <span class="fu">cbind</span>(theta_matrix, theta_i)</span>
<span id="cb13-114"><a href="#cb13-114" aria-hidden="true" tabindex="-1"></a>    lrr_matrix <span class="ot">=</span> <span class="fu">cbind</span>(lrr_matrix, geno_lrr_i)</span>
<span id="cb13-115"><a href="#cb13-115" aria-hidden="true" tabindex="-1"></a>    baf_matrix <span class="ot">=</span> <span class="fu">cbind</span>(baf_matrix, geno_baf_i)</span>
<span id="cb13-116"><a href="#cb13-116" aria-hidden="true" tabindex="-1"></a>    sex_predict_xy <span class="ot">=</span> <span class="fu">rbind</span>(sex_predict_xy, sex_predict_xy_i)</span>
<span id="cb13-117"><a href="#cb13-117" aria-hidden="true" tabindex="-1"></a>    sex_predict_ycalls <span class="ot">=</span> <span class="fu">rbind</span>(sex_predict_ycalls, sex_predict_ycalls_i)</span>
<span id="cb13-118"><a href="#cb13-118" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb13-119"><a href="#cb13-119" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rm</span>(<span class="at">list =</span> <span class="fu">ls</span>()[<span class="fu">grep</span>(<span class="st">&quot;_i$&quot;</span>,<span class="fu">ls</span>())])</span>
<span id="cb13-120"><a href="#cb13-120" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<pre><code>Reading sample manifest from &lt; ./data/gigamuga/Jackson_Lab_Bubier_MURGIGIV01_20171001//Sample_Map.zip &gt; ...</code></pre>
<pre><code>Reading genotypes and intensities for 143259 markers x 11 samples from &lt; ./data/gigamuga/Jackson_Lab_Bubier_MURGIGIV01_20171001//Jackson_Lab_Bubier_MURGIGV01_20171001_FinalReport.zip &gt; ...</code></pre>
<pre><code>Taking input= as a system command (&#39;unzip -ap &#39;./data/gigamuga/Jackson_Lab_Bubier_MURGIGIV01_20171001//Jackson_Lab_Bubier_MURGIGV01_20171001_FinalReport.zip&#39;&#39;) and a variable has been used in the expression passed to `input=`. Please use fread(cmd=...). There is a security concern if you are creating an app, and the app could have a malicious user, and the app is not running in a secure environment; e.g. the app is running as root. Please read item 5 in the NEWS file for v1.11.6 for more information and for the option to suppress this message.</code></pre>
<pre><code>Constructing genotype matrix...</code></pre>
<pre><code>Constructing intensity matrices...</code></pre>
<pre><code>     143259 sites x 11 samples</code></pre>
<pre><code>Warning in genotypes(.copy.matrix.noattr(calls), map = attr(calls, &quot;map&quot;), : REF and ALT alleles shouldn&#39;t match, but they do at these markers:
 ChloR001,ChloR002,TK001</code></pre>
<pre><code>Done.</code></pre>
<pre><code>Predicting sex using count of good calls on chrY...</code></pre>
<pre><code>Recoding to 0/1/2 using reference alleles.</code></pre>
<pre><code>Performing initial quantile normalization...</code></pre>
<pre><code>Performing tQN normalization without additional within-sample normalization.</code></pre>
<pre><code>Done.</code></pre>
<pre><code>Reading sample manifest from &lt; ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20160908//Sample_Map.zip &gt; ...</code></pre>
<pre><code>Reading genotypes and intensities for 143259 markers x 286 samples from &lt; ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20160908//Jackson_Lab_Bubier_MURGIGV01_20160908_FinalReport.zip &gt; ...</code></pre>
<pre><code>Taking input= as a system command (&#39;unzip -ap &#39;./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20160908//Jackson_Lab_Bubier_MURGIGV01_20160908_FinalReport.zip&#39;&#39;) and a variable has been used in the expression passed to `input=`. Please use fread(cmd=...). There is a security concern if you are creating an app, and the app could have a malicious user, and the app is not running in a secure environment; e.g. the app is running as root. Please read item 5 in the NEWS file for v1.11.6 for more information and for the option to suppress this message.</code></pre>
<pre><code>Constructing genotype matrix...</code></pre>
<pre><code>Constructing intensity matrices...</code></pre>
<pre><code>     143259 sites x 286 samples</code></pre>
<pre><code>Warning in genotypes(.copy.matrix.noattr(calls), map = attr(calls, &quot;map&quot;), : REF and ALT alleles shouldn&#39;t match, but they do at these markers:
 ChloR001,ChloR002,TK001</code></pre>
<pre><code>Done.</code></pre>
<pre><code>Predicting sex using count of good calls on chrY...</code></pre>
<pre><code>Recoding to 0/1/2 using reference alleles.</code></pre>
<pre><code>Performing initial quantile normalization...</code></pre>
<pre><code>Performing tQN normalization without additional within-sample normalization.</code></pre>
<pre><code>Done.</code></pre>
<pre><code>Reading sample manifest from &lt; ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20161227//Sample_Map.zip &gt; ...</code></pre>
<pre><code>Reading genotypes and intensities for 143259 markers x 192 samples from &lt; ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20161227//Jackson_Lab_Bubier_MURGIGV01_20161227_FinalReport.zip &gt; ...</code></pre>
<pre><code>Taking input= as a system command (&#39;unzip -ap &#39;./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20161227//Jackson_Lab_Bubier_MURGIGV01_20161227_FinalReport.zip&#39;&#39;) and a variable has been used in the expression passed to `input=`. Please use fread(cmd=...). There is a security concern if you are creating an app, and the app could have a malicious user, and the app is not running in a secure environment; e.g. the app is running as root. Please read item 5 in the NEWS file for v1.11.6 for more information and for the option to suppress this message.</code></pre>
<pre><code>Constructing genotype matrix...</code></pre>
<pre><code>Constructing intensity matrices...</code></pre>
<pre><code>     143259 sites x 192 samples</code></pre>
<pre><code>Warning in genotypes(.copy.matrix.noattr(calls), map = attr(calls, &quot;map&quot;), : REF and ALT alleles shouldn&#39;t match, but they do at these markers:
 ChloR001,ChloR002,TK001</code></pre>
<pre><code>Done.</code></pre>
<pre><code>Predicting sex using count of good calls on chrY...</code></pre>
<pre><code>Recoding to 0/1/2 using reference alleles.</code></pre>
<pre><code>Performing initial quantile normalization...</code></pre>
<pre><code>Performing tQN normalization without additional within-sample normalization.</code></pre>
<pre><code>Done.</code></pre>
<pre><code>Reading sample manifest from &lt; ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20170904//Sample_Map.zip &gt; ...</code></pre>
<pre><code>Reading genotypes and intensities for 143259 markers x 60 samples from &lt; ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20170904//Jackson_Lab_Bubier_MURGIGV01_20170904_FinalReport.zip &gt; ...</code></pre>
<pre><code>Taking input= as a system command (&#39;unzip -ap &#39;./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20170904//Jackson_Lab_Bubier_MURGIGV01_20170904_FinalReport.zip&#39;&#39;) and a variable has been used in the expression passed to `input=`. Please use fread(cmd=...). There is a security concern if you are creating an app, and the app could have a malicious user, and the app is not running in a secure environment; e.g. the app is running as root. Please read item 5 in the NEWS file for v1.11.6 for more information and for the option to suppress this message.</code></pre>
<pre><code>Constructing genotype matrix...</code></pre>
<pre><code>Constructing intensity matrices...</code></pre>
<pre><code>     143259 sites x 60 samples</code></pre>
<pre><code>Warning in genotypes(.copy.matrix.noattr(calls), map = attr(calls, &quot;map&quot;), : REF and ALT alleles shouldn&#39;t match, but they do at these markers:
 ChloR001,ChloR002,TK001</code></pre>
<pre><code>Done.</code></pre>
<pre><code>Predicting sex using count of good calls on chrY...</code></pre>
<pre><code>Recoding to 0/1/2 using reference alleles.</code></pre>
<pre><code>Performing initial quantile normalization...</code></pre>
<pre><code>Performing tQN normalization without additional within-sample normalization.</code></pre>
<pre><code>Done.</code></pre>
<pre><code>Reading sample manifest from &lt; ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20180518//Sample_Map.zip &gt; ...</code></pre>
<pre><code>Reading genotypes and intensities for 143259 markers x 312 samples from &lt; ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20180518//Jackson_Lab_Bubier_MURGIGV01_20180518_FinalReport.zip &gt; ...</code></pre>
<pre><code>Taking input= as a system command (&#39;unzip -ap &#39;./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20180518//Jackson_Lab_Bubier_MURGIGV01_20180518_FinalReport.zip&#39;&#39;) and a variable has been used in the expression passed to `input=`. Please use fread(cmd=...). There is a security concern if you are creating an app, and the app could have a malicious user, and the app is not running in a secure environment; e.g. the app is running as root. Please read item 5 in the NEWS file for v1.11.6 for more information and for the option to suppress this message.</code></pre>
<pre><code>Constructing genotype matrix...</code></pre>
<pre><code>Constructing intensity matrices...</code></pre>
<pre><code>     143259 sites x 312 samples</code></pre>
<pre><code>Warning in genotypes(.copy.matrix.noattr(calls), map = attr(calls, &quot;map&quot;), : REF and ALT alleles shouldn&#39;t match, but they do at these markers:
 ChloR001,ChloR002,TK001</code></pre>
<pre><code>Done.</code></pre>
<pre><code>Predicting sex using count of good calls on chrY...</code></pre>
<pre><code>Recoding to 0/1/2 using reference alleles.</code></pre>
<pre><code>Performing initial quantile normalization...</code></pre>
<pre><code>Performing tQN normalization without additional within-sample normalization.</code></pre>
<pre><code>Done.</code></pre>
</div>
<div id="evaluating-x-and-y-chromosome-content" class="section level3">
<h3>Evaluating X and Y chromosome content</h3>
<div class="sourceCode" id="cb79"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb79-1"><a href="#cb79-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Y chromosome content: normalized intensity (R rank-normal transformed) will be low for</span></span>
<span id="cb79-2"><a href="#cb79-2" aria-hidden="true" tabindex="-1"></a><span class="co"># samples not containing a Y chromosome.</span></span>
<span id="cb79-3"><a href="#cb79-3" aria-hidden="true" tabindex="-1"></a>R_norm_matrix <span class="ot">=</span> <span class="fu">apply</span>(R_matrix, <span class="at">MARGIN =</span> <span class="dv">2</span>, <span class="at">FUN =</span> norm_rank_transform)</span>
<span id="cb79-4"><a href="#cb79-4" aria-hidden="true" tabindex="-1"></a>R_norm_chrY <span class="ot">=</span> R_norm_matrix[<span class="fu">which</span>(<span class="fu">row.names</span>(R_norm_matrix) <span class="sc">%in%</span> snps[<span class="fu">which</span>(snps<span class="sc">$</span>chr <span class="sc">==</span> <span class="st">&quot;chrY&quot;</span>),<span class="st">&quot;marker&quot;</span>]),]</span>
<span id="cb79-5"><a href="#cb79-5" aria-hidden="true" tabindex="-1"></a>R_norm_meanY <span class="ot">=</span> <span class="fu">apply</span>(R_norm_chrY, <span class="at">MARGIN =</span> <span class="dv">2</span>, <span class="at">FUN =</span> mean)</span>
<span id="cb79-6"><a href="#cb79-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb79-7"><a href="#cb79-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting X chromosome ploidy using number of heterozygous loci on X</span></span>
<span id="cb79-8"><a href="#cb79-8" aria-hidden="true" tabindex="-1"></a>geno_chrX <span class="ot">=</span> geno_matrix[<span class="fu">which</span>(<span class="fu">row.names</span>(geno_matrix) <span class="sc">%in%</span> snps[<span class="fu">which</span>(snps<span class="sc">$</span>chr <span class="sc">==</span> <span class="st">&quot;chrX&quot;</span>),<span class="st">&quot;marker&quot;</span>]),]</span>
<span id="cb79-9"><a href="#cb79-9" aria-hidden="true" tabindex="-1"></a>het_chrX <span class="ot">=</span> <span class="fu">which</span>(geno_chrX <span class="sc">==</span> <span class="st">&quot;H&quot;</span>, <span class="at">arr.ind =</span> <span class="cn">TRUE</span>)</span>
<span id="cb79-10"><a href="#cb79-10" aria-hidden="true" tabindex="-1"></a>ishet_chrX <span class="ot">=</span> <span class="fu">matrix</span>(<span class="fu">rep</span>(<span class="cn">FALSE</span>, <span class="at">times =</span> <span class="fu">nrow</span>(geno_chrX) <span class="sc">*</span> <span class="fu">ncol</span>(geno_chrX)),</span>
<span id="cb79-11"><a href="#cb79-11" aria-hidden="true" tabindex="-1"></a>                    <span class="at">nrow =</span> <span class="fu">nrow</span>(geno_chrX), <span class="at">ncol =</span> <span class="fu">ncol</span>(geno_chrX), </span>
<span id="cb79-12"><a href="#cb79-12" aria-hidden="true" tabindex="-1"></a>                    <span class="at">dimnames =</span> <span class="fu">list</span>(<span class="fu">row.names</span>(geno_chrX), <span class="fu">colnames</span>(geno_chrX)))</span>
<span id="cb79-13"><a href="#cb79-13" aria-hidden="true" tabindex="-1"></a>ishet_chrX[het_chrX] <span class="ot">=</span> <span class="cn">TRUE</span></span>
<span id="cb79-14"><a href="#cb79-14" aria-hidden="true" tabindex="-1"></a>nhet_chrX <span class="ot">=</span> <span class="fu">apply</span>(ishet_chrX, <span class="at">MARGIN =</span> <span class="dv">2</span>, <span class="at">FUN =</span> sum)</span>
<span id="cb79-15"><a href="#cb79-15" aria-hidden="true" tabindex="-1"></a>ploidy <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">row.names =</span> <span class="fu">row.names</span>(sex_predict_ycalls),</span>
<span id="cb79-16"><a href="#cb79-16" aria-hidden="true" tabindex="-1"></a>                    <span class="at">mean.R.chrY =</span> R_norm_meanY[<span class="fu">row.names</span>(sex_predict_ycalls)],</span>
<span id="cb79-17"><a href="#cb79-17" aria-hidden="true" tabindex="-1"></a>                    <span class="at">has.chrY =</span> <span class="fu">ifelse</span>(sex_predict_ycalls<span class="sc">$</span>predicted <span class="sc">==</span> <span class="dv">1</span>, <span class="cn">TRUE</span>, <span class="cn">FALSE</span>),</span>
<span id="cb79-18"><a href="#cb79-18" aria-hidden="true" tabindex="-1"></a>                    <span class="at">nhet.chrX =</span> nhet_chrX[<span class="fu">row.names</span>(sex_predict_ycalls)])</span>
<span id="cb79-19"><a href="#cb79-19" aria-hidden="true" tabindex="-1"></a>ploidy<span class="sc">$</span>chrX.ploidy <span class="ot">=</span> <span class="fu">ifelse</span>(ploidy<span class="sc">$</span>nhet.chrX <span class="sc">&lt;</span> <span class="dv">150</span>, <span class="st">&quot;haploid&quot;</span>, <span class="st">&quot;diploid&quot;</span>)</span>
<span id="cb79-20"><a href="#cb79-20" aria-hidden="true" tabindex="-1"></a>ploidy[<span class="fu">which</span>(ploidy<span class="sc">$</span>nhet.chrX <span class="sc">&gt;</span> <span class="dv">150</span> <span class="sc">&amp;</span> ploidy<span class="sc">$</span>has.chrY),<span class="st">&quot;chrX.ploidy&quot;</span>] <span class="ot">=</span> <span class="st">&quot;possible_partial_xdup&quot;</span></span>
<span id="cb79-21"><a href="#cb79-21" aria-hidden="true" tabindex="-1"></a>ploidy<span class="sc">$</span>sex <span class="ot">=</span> <span class="fu">ifelse</span>(ploidy<span class="sc">$</span>has.chrY, <span class="st">&quot;male&quot;</span>, <span class="st">&quot;female&quot;</span>)</span>
<span id="cb79-22"><a href="#cb79-22" aria-hidden="true" tabindex="-1"></a>ploidy<span class="sc">$</span>class <span class="ot">=</span> ploidy<span class="sc">$</span>sex</span>
<span id="cb79-23"><a href="#cb79-23" aria-hidden="true" tabindex="-1"></a>ploidy[<span class="fu">which</span>(ploidy<span class="sc">$</span>class <span class="sc">==</span> <span class="st">&quot;male&quot;</span> <span class="sc">&amp;</span> ploidy<span class="sc">$</span>chrX.ploidy <span class="sc">==</span> <span class="st">&quot;possible_partial_xdup&quot;</span>),<span class="st">&quot;class&quot;</span>] <span class="ot">=</span> <span class="st">&quot;male_partial_X_dup&quot;</span></span>
<span id="cb79-24"><a href="#cb79-24" aria-hidden="true" tabindex="-1"></a>ploidy[<span class="fu">which</span>(ploidy<span class="sc">$</span>class <span class="sc">==</span> <span class="st">&quot;female&quot;</span> <span class="sc">&amp;</span> ploidy<span class="sc">$</span>chrX.ploidy <span class="sc">==</span> <span class="st">&quot;haploid&quot;</span>),<span class="st">&quot;class&quot;</span>] <span class="ot">=</span> <span class="st">&quot;female_X0&quot;</span></span>
<span id="cb79-25"><a href="#cb79-25" aria-hidden="true" tabindex="-1"></a>ploidy_plot <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> ploidy, <span class="fu">aes</span>(<span class="at">x =</span> nhet.chrX, <span class="at">y =</span> mean.R.chrY, <span class="at">color =</span> class)) <span class="sc">+</span> </span>
<span id="cb79-26"><a href="#cb79-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span> </span>
<span id="cb79-27"><a href="#cb79-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_log10</span>() <span class="sc">+</span></span>
<span id="cb79-28"><a href="#cb79-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;chrX Het. Calls (log10 scale)&quot;</span>) <span class="sc">+</span></span>
<span id="cb79-29"><a href="#cb79-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Mean Rank-Normal R: chrY&quot;</span>) <span class="sc">+</span></span>
<span id="cb79-30"><a href="#cb79-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_cowplot</span>() <span class="sc">+</span></span>
<span id="cb79-31"><a href="#cb79-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>) <span class="sc">+</span> </span>
<span id="cb79-32"><a href="#cb79-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#3399CC&quot;</span>, <span class="st">&quot;#66CC66&quot;</span>, <span class="st">&quot;#CC33CC&quot;</span>, <span class="st">&quot;#FFCC33&quot;</span>))</span>
<span id="cb79-33"><a href="#cb79-33" aria-hidden="true" tabindex="-1"></a>ploidy_plot</span></code></pre></div>
<pre><code>Warning: Removed 21 rows containing missing values (geom_point).</code></pre>
<p><img src="figure/06_DO_ploidy_followup_to_expression.Rmd/unnamed-chunk-6-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
839900c
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Saving ploidy calls as a file.</p>
<div class="sourceCode" id="cb81"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb81-1"><a href="#cb81-1" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(ploidy, <span class="at">file =</span> <span class="st">&quot;./data/DO_ploidy_calls.RDS&quot;</span>)</span></code></pre></div>
<p>The md5 checksums for the output files are:</p>
<table>
<thead>
<tr class="header">
<th>file</th>
<th>md5 checksum</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><code>DO_ploidy_calls.RDS</code></td>
<td><code>f8ccfdbf5b0dde3077e39c80745599af</code></td>
</tr>
</tbody>
</table>
</div>
<div id="document-control" class="section level3">
<h3>Document Control</h3>
<p>This document was prepared using RMarkdown in RStudio and was integrated into workflowr on 2020-05-04.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb82"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb82-1"><a href="#cb82-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.3 LTS

Matrix products: default
BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] tools     stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
[1] ggplot2_3.3.5 cowplot_1.1.1 argyle_0.2.2 

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.7            highr_0.9             pillar_1.6.4         
 [4] compiler_4.1.2        later_1.3.0           jquerylib_0.1.4      
 [7] git2r_0.29.0          workflowr_1.6.2       digest_0.6.29        
[10] preprocessCore_1.56.0 evaluate_0.14         lifecycle_1.0.1      
[13] tibble_3.1.6          gtable_0.3.0          pkgconfig_2.0.3      
[16] rlang_0.4.12          DBI_1.1.1             yaml_2.2.1           
[19] mvtnorm_1.1-3         xfun_0.28             fastmap_1.1.0        
[22] withr_2.4.3           dplyr_1.0.7           stringr_1.4.0        
[25] knitr_1.36            generics_0.1.1        fs_1.5.1             
[28] vctrs_0.3.8           tidyselect_1.1.1      rprojroot_2.0.2      
[31] grid_4.1.2            data.table_1.14.2     glue_1.5.1           
[34] R6_2.5.1              fansi_0.5.0           rmarkdown_2.11       
[37] farver_2.1.0          purrr_0.3.4           magrittr_2.0.1       
[40] whisker_0.4           scales_1.1.1          promises_1.2.0.1     
[43] ellipsis_0.3.2        htmltools_0.5.2       assertthat_0.2.1     
[46] colorspace_2.0-2      httpuv_1.6.3          labeling_0.4.2       
[49] utf8_1.2.2            stringi_1.7.6         munsell_0.5.0        
[52] crayon_1.4.2         </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
