<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Michael C. Saul (michael.saul [at] jax.org)" />


<title>DO Expression: eQTL Prep</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>


<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Diversity Outbred Reference Traits Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">DO Expression: eQTL Prep</h1>
<h4 class="author">Michael C. Saul (michael.saul [at] jax.org)</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-07
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rta/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20200501code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20200501)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20200501code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20200501)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong06d35f8"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 06d35f8 </a>
</p>
</div>
<div id="strongRepositoryversionstrong06d35f8" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 06d35f8. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/.Rapp.history
    Ignored:    data/.Rhistory
    Ignored:    data/RDS/
    Ignored:    run/

Untracked files:
    Untracked:  LICENSE
    Untracked:  Untitled.R
    Untracked:  analysis/17_mapping_interpretation_old.Rmd
    Untracked:  analysis/20_predict_acquisition_from_novelty.Rmd
    Untracked:  analysis/21_DO_elasticnet_correlation.Rmd
    Untracked:  analysis/temp.txt
    Untracked:  code/DO_eQTL_perms.R
    Untracked:  code/DO_eQTL_score_calc.R
    Untracked:  code/STAR_RSEM_416.sh
    Untracked:  code/eQTL_plotting_DO_striatum_416.R
    Untracked:  code/ggplot_pub_themes.R
    Untracked:  data/20_elastic_mv_results.csv
    Untracked:  data/20_elasticnet_predictions.RDS
    Untracked:  data/DO_416_DGEList.RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 3).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 4).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_DGEList_unfiltered.RDS
    Untracked:  data/DO_416_ERCC_DGEList.RDS
    Untracked:  data/DO_416_ERCC_cpm.RDS
    Untracked:  data/DO_416_normrt.RDS
    Untracked:  data/DO_416_pheno.RDS
    Untracked:  data/DO_416_pheno_allgenes.RDS
    Untracked:  data/DO_416_voom (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom.RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom_allgenes.RDS
    Untracked:  data/DO_416_voom_cpresiduals.RDS
    Untracked:  data/DO_AQ_active_CoV_toptable_annotated.RDS
    Untracked:  data/DO_AQ_lever_ratio_toptable_annotated.RDS
    Untracked:  data/DO_AQ_mean_inf_toptable_annotated.RDS
    Untracked:  data/DO_AQ_rate_toptable_annotated.RDS
    Untracked:  data/DO_IVSA_acquisition_cancor_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_glm.RDS
    Untracked:  data/DO_IVSA_all_cancor_df.RDS
    Untracked:  data/DO_IVSA_data.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_ALL.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_AQ.RDS
    Untracked:  data/DO_IVSA_novelty_data.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_old.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_a1_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_a2_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a4_male_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a5_female_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_raw_data.RDS
    Untracked:  data/DO_IVSA_transformed_data.RDS
    Untracked:  data/DO_LIMS_data.RDS
    Untracked:  data/DO_RTG_a1_new.qtl.RData
    Untracked:  data/DO_RTG_a1_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a1_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a2_new.qtl.RData
    Untracked:  data/DO_RTG_a2_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a2_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a3_new.qtl.RData
    Untracked:  data/DO_RTG_a3_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_01.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_02.RData
    Untracked:  data/DO_RTG_a4_new.qtl.RData
    Untracked:  data/DO_RTG_a5_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_project_df_for_mapping.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_2020-11-06.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_a2.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_sexspecific.RDS
    Untracked:  data/DO_demographic_data_df.RDS
    Untracked:  data/DO_eigengene_significance.RDS
    Untracked:  data/DO_eigengenes.RDS
    Untracked:  data/DO_highcor_cpresiduals_datatable.RDS
    Untracked:  data/DO_holeboard_data_df.RDS
    Untracked:  data/DO_holeboard_summarized.RDS
    Untracked:  data/DO_lightdark_data_df.RDS
    Untracked:  data/DO_mQTL_peaks.RDS
    Untracked:  data/DO_mQTL_scan1s.RDS
    Untracked:  data/DO_module_membership.RDS
    Untracked:  data/DO_novelplace_data_df.RDS
    Untracked:  data/DO_novelty_data.RDS
    Untracked:  data/DO_novelty_response_data.RDS
    Untracked:  data/DO_novelty_response_data_for_RTG.RDS
    Untracked:  data/DO_novelty_response_data_with_transform.csv
    Untracked:  data/DO_openfield_data_df.RDS
    Untracked:  data/DO_paraclique_data.el
    Untracked:  data/DO_paraclique_eigengenes.RDS
    Untracked:  data/DO_paracliques.RDS
    Untracked:  data/DO_paracliques_significance.RDS
    Untracked:  data/DO_ploidy_calls.RDS
    Untracked:  data/DO_pmap.RDS
    Untracked:  data/DO_pvalue_histograms.RData
    Untracked:  data/DO_rtg_a1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a3_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m3_toptable_annotated.RDS
    Untracked:  data/DO_str_2016_eQTL.RData
    Untracked:  data/DO_str_2016_gm4qtl2.zip
    Untracked:  data/DO_str_2016_gm4qtl2_intensities.fst
    Untracked:  data/DO_voom_cpresiduals_NArm.RDS
    Untracked:  data/DO_wgcna.RDS
    Untracked:  data/IVSA_RTG_a3_blup.pdf
    Untracked:  data/IVSA_h2_2019-11-18.tsv
    Untracked:  data/IVSA_metadata_sheet.RDS
    Untracked:  data/ME16_edges.tsv
    Untracked:  data/Mus_musculus.GRCm38.94.parsed.RDS
    Untracked:  data/Paraclique_19_MF_AMIGO.txt
    Untracked:  data/RTG_QTL_peaks.RDS
    Untracked:  data/RTG_figure.RDS
    Untracked:  data/SENS_RTG/
    Untracked:  data/Tyr2_genotypes.RDS
    Untracked:  data/Tyr_genotypes.RDS
    Untracked:  data/blup_chr7_DO_RTG_a3.RDS
    Untracked:  data/cc_variants/
    Untracked:  data/chr18_QTL_results.RDS
    Untracked:  data/chr4_QTL_results.RDS
    Untracked:  data/cis/
    Untracked:  data/eQTL_annotated_peaks.RDS
    Untracked:  data/eQTL_plots.RData
    Untracked:  data/figures/
    Untracked:  data/founders_key/
    Untracked:  data/gigamuga/
    Untracked:  data/gwas/
    Untracked:  data/info/
    Untracked:  data/m2G.a2.RTG.chr4snp.RData
    Untracked:  data/mapping_from_hao/
    Untracked:  data/perms/
    Untracked:  data/qtl2/
    Untracked:  data/raw/
    Untracked:  data/resources/
    Untracked:  data/tads_loops/
    Untracked:  data/transcripts/
    Untracked:  data/variants/
    Untracked:  logs/
    Untracked:  new_RTG_a3.jpeg
    Untracked:  output/RTG_out/
    Untracked:  output/docs_backup_2020-07-30.tar.gz
    Untracked:  output/elastic_net_out/
    Untracked:  singularity/
    Untracked:  tmp/
    Untracked:  var/

Unstaged changes:
    Deleted:    .Rprofile
    Modified:   .gitignore
    Modified:   README.md
    Modified:   analysis/17_mapping_interpretation.Rmd
    Modified:   analysis/_site.yml

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/07_DO_eQTL_genotype_prep.Rmd</code>) and HTML (<code>docs/07_DO_eQTL_genotype_prep.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
<td>
Rolling back to working version.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
a743c27
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
5f89658
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
c68f180
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-29
</td>
<td>
Updated output file to exclude genotype probabilities
</td>
</tr>
<tr>
<td>
html
</td>
<td>
b495515
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
1b34c81
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
<td>
Add eQTL_maRt to output Rdata object
</td>
</tr>
<tr>
<td>
html
</td>
<td>
84aab57
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
fabb7a2
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
<td>
Initial commit of eQTL genotype prep
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
dbeeeff
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-05
</td>
<td>
Initial commit of eQTL prep
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="background" class="section level1">
<h1>Background</h1>
<div id="motivation" class="section level2">
<h2>Motivation</h2>
<div id="purpose-of-analysis" class="section level3">
<h3>Purpose of analysis</h3>
<p>This analysis is performed to prepare the data for downstream use in eQTL analysis.</p>
</div>
</div>
</div>
<div id="analysis" class="section level1">
<h1>Analysis</h1>
<div id="setup" class="section level2">
<h2>Setup</h2>
<div id="r-libraries" class="section level3">
<h3>R libraries</h3>
<p>Calling R libraries necessary for this analysis.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;limma&quot;</span>)</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;edgeR&quot;</span>)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;cowplot&quot;</span>)</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggbeeswarm&quot;</span>)</span></code></pre></div>
<pre><code>Loading required package: ggplot2</code></pre>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;gplots&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    lowess</code></pre>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;RColorBrewer&quot;</span>)</span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;data.table&quot;</span>)</span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;biomaRt&quot;</span>)</span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;MASS&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;MASS&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:biomaRt&#39;:

    select</code></pre>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;qtl2&quot;</span>)</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;qtl2convert&quot;</span>)</span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;tools&quot;</span>)</span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;fst&quot;</span>)</span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;lubridate&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;lubridate&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:data.table&#39;:

    hour, isoweek, mday, minute, month, quarter, second, wday, week,
    yday, year</code></pre>
<pre><code>The following object is masked from &#39;package:cowplot&#39;:

    stamp</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    date, intersect, setdiff, union</code></pre>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;broman&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;broman&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:gplots&#39;:

    venn</code></pre>
</div>
<div id="getting-norm_rank_transform-function" class="section level3">
<h3>Getting <code>norm_rank_transform</code> function</h3>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>norm_rank_transform <span class="ot">=</span> <span class="cf">function</span>(x, <span class="at">c =</span> <span class="dv">0</span>) {</span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stopifnot</span>(<span class="fu">is.numeric</span>(x) <span class="sc">&amp;</span> <span class="fu">is.vector</span>(x))</span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>  x_noNA <span class="ot">=</span> <span class="fu">which</span>(<span class="sc">!</span><span class="fu">is.na</span>(x))</span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a>  N <span class="ot">=</span> <span class="fu">length</span>(x_noNA)</span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>  x[x_noNA] <span class="ot">=</span> <span class="fu">qnorm</span>((<span class="fu">rank</span>(x[x_noNA], <span class="at">ties.method =</span> <span class="st">&quot;average&quot;</span>) <span class="sc">-</span> c) <span class="sc">/</span> (N <span class="sc">-</span> (<span class="dv">2</span> <span class="sc">*</span> c) <span class="sc">+</span> <span class="dv">1</span>))</span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(x)</span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
</div>
<div id="importing-data" class="section level3">
<h3>Importing data</h3>
<div id="importing-expression-data" class="section level4">
<h4>Importing expression data</h4>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a>DO_voom <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_416_voom.RDS&quot;</span>)</span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a>DO_pheno <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_416_pheno.RDS&quot;</span>)</span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a>DO_demodata <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_demographic_data_df.RDS&quot;</span>)</span>
<span id="cb18-4"><a href="#cb18-4" aria-hidden="true" tabindex="-1"></a>DO_MUGA_ploidy <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_ploidy_calls.RDS&quot;</span>)</span>
<span id="cb18-5"><a href="#cb18-5" aria-hidden="true" tabindex="-1"></a>DO_pheno<span class="sc">$</span>MUGA_sex <span class="ot">=</span> DO_MUGA_ploidy[<span class="fu">row.names</span>(DO_pheno),<span class="st">&quot;sex&quot;</span>]</span>
<span id="cb18-6"><a href="#cb18-6" aria-hidden="true" tabindex="-1"></a>DO_pheno<span class="sc">$</span>MUGA_ploidy <span class="ot">=</span> DO_MUGA_ploidy[<span class="fu">row.names</span>(DO_pheno),<span class="st">&quot;class&quot;</span>]</span>
<span id="cb18-7"><a href="#cb18-7" aria-hidden="true" tabindex="-1"></a>DO_pheno<span class="sc">$</span>DOB <span class="ot">=</span> DO_demodata[<span class="fu">row.names</span>(DO_pheno),<span class="st">&quot;DOB&quot;</span>]</span>
<span id="cb18-8"><a href="#cb18-8" aria-hidden="true" tabindex="-1"></a><span class="fu">rm</span>(<span class="at">list =</span> <span class="fu">c</span>(<span class="st">&quot;DO_demodata&quot;</span>))</span></code></pre></div>
</div>
</div>
<div id="getting-choroid-plexus-residuals" class="section level3">
<h3>Getting choroid plexus residuals</h3>
<p>Calculating expression results with choroid plexus contamination regressed out.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>DO_voom_cpresid <span class="ot">=</span> <span class="fu">matrix</span>(<span class="at">nrow =</span> <span class="fu">nrow</span>(DO_voom<span class="sc">$</span>E), <span class="at">ncol =</span> <span class="fu">ncol</span>(DO_voom<span class="sc">$</span>E))</span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(DO_voom_cpresid) <span class="ot">=</span> <span class="fu">row.names</span>(DO_voom<span class="sc">$</span>E)</span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_voom_cpresid) <span class="ot">=</span> <span class="fu">colnames</span>(DO_voom<span class="sc">$</span>E)</span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="fu">row.names</span>(DO_voom_cpresid)) {</span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a>  DO_voom_cpresid[i,] <span class="ot">=</span> <span class="fu">resid</span>(<span class="fu">lm</span>(DO_voom<span class="sc">$</span>E[i,] <span class="sc">~</span> DO_pheno<span class="sc">$</span>choroid_plexus_covariate))</span>
<span id="cb19-7"><a href="#cb19-7" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb19-8"><a href="#cb19-8" aria-hidden="true" tabindex="-1"></a>DO_voom <span class="ot">=</span> DO_voom_cpresid</span>
<span id="cb19-9"><a href="#cb19-9" aria-hidden="true" tabindex="-1"></a><span class="fu">rm</span>(<span class="at">list =</span> <span class="fu">c</span>(<span class="st">&quot;DO_voom_cpresid&quot;</span>))</span></code></pre></div>
</div>
<div id="getting-expression-annotation-information" class="section level3">
<h3>Getting expression annotation information</h3>
<p>Downloading expresison annotation information from Ensembl using <code>biomaRt</code>.</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>maRt <span class="ot">=</span> <span class="fu">useMart</span>(<span class="at">biomart =</span> <span class="st">&quot;ENSEMBL_MART_ENSEMBL&quot;</span>,</span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>               <span class="at">host =</span> <span class="st">&quot;oct2018.archive.ensembl.org&quot;</span>,</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a>               <span class="at">dataset =</span> <span class="st">&quot;mmusculus_gene_ensembl&quot;</span>)</span></code></pre></div>
<pre><code>Warning: Ensembl will soon enforce the use of https.
Ensure the &#39;host&#39; argument includes &quot;https://&quot;</code></pre>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>maRt_filter <span class="ot">=</span> <span class="st">&quot;ensembl_gene_id&quot;</span></span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>maRt_attributes <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;mgi_symbol&quot;</span>,<span class="st">&quot;mgi_description&quot;</span>,<span class="st">&quot;mgi_id&quot;</span>,<span class="st">&quot;chromosome_name&quot;</span>,<span class="st">&quot;start_position&quot;</span>,</span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a>                    <span class="st">&quot;end_position&quot;</span>,<span class="st">&quot;strand&quot;</span>,<span class="st">&quot;band&quot;</span>,<span class="st">&quot;ensembl_gene_id&quot;</span>)</span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a>eQTL_maRt <span class="ot">=</span> <span class="fu">getBM</span>(maRt_attributes, maRt_filter, <span class="fu">row.names</span>(DO_voom), maRt)</span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(eQTL_maRt) <span class="ot">=</span> eQTL_maRt<span class="sc">$</span>ensembl_gene_id</span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a>eQTL_maRt<span class="sc">$</span>marker_start <span class="ot">=</span> <span class="fu">ifelse</span>(<span class="fu">sign</span>(eQTL_maRt<span class="sc">$</span>strand) <span class="sc">==</span> <span class="dv">1</span>,</span>
<span id="cb22-7"><a href="#cb22-7" aria-hidden="true" tabindex="-1"></a>                                eQTL_maRt<span class="sc">$</span>start_position,</span>
<span id="cb22-8"><a href="#cb22-8" aria-hidden="true" tabindex="-1"></a>                                eQTL_maRt<span class="sc">$</span>end_position)</span></code></pre></div>
<div id="importing-snp-data-for-the-gigamuga-chipset." class="section level4">
<h4>Importing SNP data for the GigaMUGA chipset.</h4>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="co"># The GigaMUGA consensus file was found at the following location on the JAX filesystem:</span></span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a><span class="co"># /projects/churchill-lab/resource_data/muga_reference/gigamuga_consensus_alleles.csv</span></span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a><span class="co"># The file on the server has the following MD5 checksum:</span></span>
<span id="cb23-4"><a href="#cb23-4" aria-hidden="true" tabindex="-1"></a><span class="co"># cfb5e353f959cab0451619a3eaa8a796</span></span>
<span id="cb23-5"><a href="#cb23-5" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="fu">ifelse</span>(<span class="fu">md5sum</span>(<span class="st">&quot;./data/resources/gigamuga_consensus_alleles.csv&quot;</span>) <span class="sc">==</span> <span class="st">&quot;cfb5e353f959cab0451619a3eaa8a796&quot;</span>,</span>
<span id="cb23-6"><a href="#cb23-6" aria-hidden="true" tabindex="-1"></a>           <span class="st">&quot;The md5 checksum of the v38 GigaMUGA consensus allele file MATCHES the Churchill Lab&#39;s file.&quot;</span>,</span>
<span id="cb23-7"><a href="#cb23-7" aria-hidden="true" tabindex="-1"></a>           <span class="st">&quot;The md5 checksum of the v38 GigaMUGA consensus allele file DOES NOT MATCH the Churchill Lab&#39;s file.&quot;</span>))</span></code></pre></div>
<pre><code>The md5 checksum of the v38 GigaMUGA consensus allele file MATCHES the Churchill Lab&#39;s file.</code></pre>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a>gigamuga_consensus_alleles <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">&quot;./data/resources/gigamuga_consensus_alleles.csv&quot;</span>,</span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a>                                        <span class="at">sep =</span> <span class="st">&quot;,&quot;</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="fu">url</span>(<span class="st">&quot;ftp://ftp.jax.org/MUGA/GM_snps.Rdata&quot;</span>))</span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="fu">url</span>(<span class="st">&quot;http://csbio.unc.edu/MUGA/snps.gigamuga.Rdata&quot;</span>))</span></code></pre></div>
<p>Refining the GigaMUGA consensus allele file</p>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(gigamuga_consensus_alleles) <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;Marker&quot;</span>,<span class="st">&quot;ChrB38&quot;</span>,<span class="st">&quot;PosB38&quot;</span>,</span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a>                                         <span class="st">&quot;allele_129&quot;</span>,<span class="st">&quot;allele_AJ&quot;</span>,<span class="st">&quot;allele_B6&quot;</span>,<span class="st">&quot;allele_CAST&quot;</span>,</span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a>                                         <span class="st">&quot;allele_NOD&quot;</span>,<span class="st">&quot;allele_NZO&quot;</span>,<span class="st">&quot;allele_PWK&quot;</span>,<span class="st">&quot;allele_WSB&quot;</span>)</span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a>gigamuga_allele_cols <span class="ot">=</span> <span class="fu">grep</span>(<span class="st">&quot;^allele&quot;</span>, <span class="fu">colnames</span>(gigamuga_consensus_alleles))</span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a>gigamuga_consensus_alleles<span class="sc">$</span>ChrB38 <span class="ot">=</span> <span class="fu">paste</span>(<span class="st">&quot;chr&quot;</span>,gigamuga_consensus_alleles<span class="sc">$</span>ChrB38,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)</span>
<span id="cb26-6"><a href="#cb26-6" aria-hidden="true" tabindex="-1"></a>gigamuga_informative_alleles <span class="ot">=</span> gigamuga_consensus_alleles[<span class="fu">which</span>(gigamuga_consensus_alleles<span class="sc">$</span>ChrB38 <span class="sc">!=</span> <span class="st">&quot;chrNA&quot;</span>),]</span>
<span id="cb26-7"><a href="#cb26-7" aria-hidden="true" tabindex="-1"></a>gigamuga_informative_alleles <span class="ot">=</span> gigamuga_informative_alleles[<span class="fu">which</span>(<span class="fu">rowSums</span>(gigamuga_informative_alleles[,gigamuga_allele_cols] <span class="sc">==</span> <span class="st">&quot;N&quot;</span>) <span class="sc">!=</span> <span class="dv">8</span>),]</span>
<span id="cb26-8"><a href="#cb26-8" aria-hidden="true" tabindex="-1"></a>alleles_129 <span class="ot">=</span> gigamuga_informative_alleles[,<span class="st">&quot;allele_129&quot;</span>]</span>
<span id="cb26-9"><a href="#cb26-9" aria-hidden="true" tabindex="-1"></a>gigamuga_informative_alleles <span class="ot">=</span> gigamuga_informative_alleles[<span class="fu">which</span>(<span class="fu">rowSums</span>(gigamuga_informative_alleles[,gigamuga_allele_cols] <span class="sc">==</span> alleles_129) <span class="sc">!=</span> <span class="dv">8</span>),]</span>
<span id="cb26-10"><a href="#cb26-10" aria-hidden="true" tabindex="-1"></a><span class="fu">rownames</span>(gigamuga_informative_alleles) <span class="ot">=</span> <span class="fu">as.character</span>(gigamuga_informative_alleles[,<span class="st">&quot;Marker&quot;</span>])</span></code></pre></div>
<p>Getting list of all data up to this point.</p>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a>loaded_data <span class="ot">=</span> <span class="fu">ls</span>()</span></code></pre></div>
</div>
<div id="converting-genotype-and-phenotype-data-to-the-correct-format-for-rqtl2" class="section level4">
<h4>Converting genotype and phenotype data to the correct format for R/qtl2</h4>
<p>Converting GigaMUGA SNP data from GeneSeek format to R/qtl2 using code from the <code>geneseek2qtl.R</code> script by the Broman lab.</p>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a><span class="co"># convert GeneSeek FinalReport files to format for R/qtl2</span></span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a><span class="co">#</span></span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a><span class="co"># - creates one genotype CSV file for each chromosome</span></span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a><span class="co">#</span></span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a><span class="co"># - also creates 2 files containing the two channels of SNP intensities for markers on the X and Y chr</span></span>
<span id="cb28-6"><a href="#cb28-6" aria-hidden="true" tabindex="-1"></a><span class="co">#   (these are useful for verifying the sex of the mice)</span></span>
<span id="cb28-7"><a href="#cb28-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-8"><a href="#cb28-8" aria-hidden="true" tabindex="-1"></a><span class="co"># file containing allele codes for GigaMUGA data</span></span>
<span id="cb28-9"><a href="#cb28-9" aria-hidden="true" tabindex="-1"></a><span class="co">#   - from GM_processed_files.zip, https://doi.org/10.6084/m9.figshare.5404759</span></span>
<span id="cb28-10"><a href="#cb28-10" aria-hidden="true" tabindex="-1"></a>codefile <span class="ot">&lt;-</span> <span class="st">&quot;cat ./data/resources/GM/GM_allelecodes.csv&quot;</span></span>
<span id="cb28-11"><a href="#cb28-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-12"><a href="#cb28-12" aria-hidden="true" tabindex="-1"></a><span class="co"># input files with GigaMUGA genotypes</span></span>
<span id="cb28-13"><a href="#cb28-13" aria-hidden="true" tabindex="-1"></a><span class="co">#  - can be a single file or a vector of multiple files</span></span>
<span id="cb28-14"><a href="#cb28-14" aria-hidden="true" tabindex="-1"></a><span class="co">#  - if samples appear in multiple files, the genotypes in later files</span></span>
<span id="cb28-15"><a href="#cb28-15" aria-hidden="true" tabindex="-1"></a><span class="co">#    will be used in place of genotypes in earlier files</span></span>
<span id="cb28-16"><a href="#cb28-16" aria-hidden="true" tabindex="-1"></a><span class="co">#  - files can be gzipped (&quot;.gz&quot; extension)</span></span>
<span id="cb28-17"><a href="#cb28-17" aria-hidden="true" tabindex="-1"></a>ifiles <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">&quot;unzip -cq ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20160908/Jackson_Lab_Bubier_MURGIGV01_20160908_FinalReport.zip&quot;</span>,</span>
<span id="cb28-18"><a href="#cb28-18" aria-hidden="true" tabindex="-1"></a>            <span class="st">&quot;unzip -cq ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20161227/Jackson_Lab_Bubier_MURGIGV01_20161227_FinalReport.zip&quot;</span>,</span>
<span id="cb28-19"><a href="#cb28-19" aria-hidden="true" tabindex="-1"></a>            <span class="st">&quot;unzip -cq ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20170904/Jackson_Lab_Bubier_MURGIGV01_20170904_FinalReport.zip&quot;</span>,</span>
<span id="cb28-20"><a href="#cb28-20" aria-hidden="true" tabindex="-1"></a>            <span class="st">&quot;unzip -cq ./data/gigamuga/Jackson_Lab_Bubier_MURGIGIV01_20171001/Jackson_Lab_Bubier_MURGIGV01_20171001_FinalReport.zip&quot;</span>,</span>
<span id="cb28-21"><a href="#cb28-21" aria-hidden="true" tabindex="-1"></a>            <span class="st">&quot;unzip -cq ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20180518/Jackson_Lab_Bubier_MURGIGV01_20180518_FinalReport.zip&quot;</span>)</span>
<span id="cb28-22"><a href="#cb28-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-23"><a href="#cb28-23" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting the dates as we will assume later dates with the same ID are reruns of the same data from bad microarrays to deduplicate</span></span>
<span id="cb28-24"><a href="#cb28-24" aria-hidden="true" tabindex="-1"></a>ifiles_dates <span class="ot">=</span> <span class="fu">ymd</span>(<span class="fu">gsub</span>(<span class="st">&quot;^.*MURGIGV01_(</span><span class="sc">\\</span><span class="st">d{4})(</span><span class="sc">\\</span><span class="st">d{2})(</span><span class="sc">\\</span><span class="st">d{2})_FinalReport</span><span class="sc">\\</span><span class="st">.zip$&quot;</span>,<span class="st">&quot;</span><span class="sc">\\</span><span class="st">1-</span><span class="sc">\\</span><span class="st">2-</span><span class="sc">\\</span><span class="st">3&quot;</span>,ifiles))</span>
<span id="cb28-25"><a href="#cb28-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-26"><a href="#cb28-26" aria-hidden="true" tabindex="-1"></a><span class="co"># file &quot;stem&quot; for output files</span></span>
<span id="cb28-27"><a href="#cb28-27" aria-hidden="true" tabindex="-1"></a><span class="co"># output files will be like &quot;gm4qtl2_geno19.csv&quot;</span></span>
<span id="cb28-28"><a href="#cb28-28" aria-hidden="true" tabindex="-1"></a>ostem <span class="ot">&lt;-</span> <span class="st">&quot;./data/qtl2/DO_str_2016_gm4qtl2_&quot;</span></span>
<span id="cb28-29"><a href="#cb28-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-30"><a href="#cb28-30" aria-hidden="true" tabindex="-1"></a><span class="do">##############################</span></span>
<span id="cb28-31"><a href="#cb28-31" aria-hidden="true" tabindex="-1"></a><span class="co"># define a couple of functions</span></span>
<span id="cb28-32"><a href="#cb28-32" aria-hidden="true" tabindex="-1"></a><span class="do">##############################</span></span>
<span id="cb28-33"><a href="#cb28-33" aria-hidden="true" tabindex="-1"></a><span class="co"># version of data.table::fread() where data.table=FALSE is the default</span></span>
<span id="cb28-34"><a href="#cb28-34" aria-hidden="true" tabindex="-1"></a>myfread <span class="ot">&lt;-</span> <span class="cf">function</span>(filename, <span class="at">data.table=</span><span class="cn">FALSE</span>, ...) data.table<span class="sc">::</span><span class="fu">fread</span>(filename, <span class="at">data.table=</span>data.table,...)</span>
<span id="cb28-35"><a href="#cb28-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-36"><a href="#cb28-36" aria-hidden="true" tabindex="-1"></a><span class="co"># cbind, replacing matching columns with second set and adding unique ones</span></span>
<span id="cb28-37"><a href="#cb28-37" aria-hidden="true" tabindex="-1"></a>cbind_smother <span class="ot">&lt;-</span></span>
<span id="cb28-38"><a href="#cb28-38" aria-hidden="true" tabindex="-1"></a>  <span class="cf">function</span>(mat1, mat2)</span>
<span id="cb28-39"><a href="#cb28-39" aria-hidden="true" tabindex="-1"></a>  {</span>
<span id="cb28-40"><a href="#cb28-40" aria-hidden="true" tabindex="-1"></a>    cn1 <span class="ot">&lt;-</span> <span class="fu">colnames</span>(mat1)</span>
<span id="cb28-41"><a href="#cb28-41" aria-hidden="true" tabindex="-1"></a>    cn2 <span class="ot">&lt;-</span> <span class="fu">colnames</span>(mat2)</span>
<span id="cb28-42"><a href="#cb28-42" aria-hidden="true" tabindex="-1"></a>    m <span class="ot">&lt;-</span> (cn2 <span class="sc">%in%</span> cn1)</span>
<span id="cb28-43"><a href="#cb28-43" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span>(<span class="fu">any</span>(m)) {</span>
<span id="cb28-44"><a href="#cb28-44" aria-hidden="true" tabindex="-1"></a>      mat1[,cn2[m]] <span class="ot">&lt;-</span> mat2[,cn2[m],drop<span class="ot">=</span><span class="cn">FALSE</span>]</span>
<span id="cb28-45"><a href="#cb28-45" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span>(<span class="fu">any</span>(<span class="sc">!</span>m)) {</span>
<span id="cb28-46"><a href="#cb28-46" aria-hidden="true" tabindex="-1"></a>        mat1 <span class="ot">&lt;-</span> <span class="fu">cbind</span>(mat1, mat2[,cn2[<span class="sc">!</span>m],<span class="at">drop=</span><span class="cn">FALSE</span>])</span>
<span id="cb28-47"><a href="#cb28-47" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb28-48"><a href="#cb28-48" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb28-49"><a href="#cb28-49" aria-hidden="true" tabindex="-1"></a>    <span class="cf">else</span> {</span>
<span id="cb28-50"><a href="#cb28-50" aria-hidden="true" tabindex="-1"></a>      mat1 <span class="ot">&lt;-</span> <span class="fu">cbind</span>(mat1, mat2)</span>
<span id="cb28-51"><a href="#cb28-51" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb28-52"><a href="#cb28-52" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb28-53"><a href="#cb28-53" aria-hidden="true" tabindex="-1"></a>    mat1</span>
<span id="cb28-54"><a href="#cb28-54" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb28-55"><a href="#cb28-55" aria-hidden="true" tabindex="-1"></a><span class="do">##############################</span></span>
<span id="cb28-56"><a href="#cb28-56" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-57"><a href="#cb28-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-58"><a href="#cb28-58" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-59"><a href="#cb28-59" aria-hidden="true" tabindex="-1"></a><span class="co"># read genotype codes</span></span>
<span id="cb28-60"><a href="#cb28-60" aria-hidden="true" tabindex="-1"></a>codes <span class="ot">&lt;-</span> data.table<span class="sc">::</span><span class="fu">fread</span>(<span class="at">cmd =</span> codefile, <span class="at">skip =</span> <span class="st">&quot;marker&quot;</span>, <span class="at">data.table =</span> <span class="cn">FALSE</span>)</span>
<span id="cb28-61"><a href="#cb28-61" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-62"><a href="#cb28-62" aria-hidden="true" tabindex="-1"></a>full_geno <span class="ot">&lt;-</span> <span class="cn">NULL</span></span>
<span id="cb28-63"><a href="#cb28-63" aria-hidden="true" tabindex="-1"></a>cXint <span class="ot">&lt;-</span> cYint <span class="ot">&lt;-</span> <span class="cn">NULL</span></span>
<span id="cb28-64"><a href="#cb28-64" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-65"><a href="#cb28-65" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span>(i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(ifiles)) {</span>
<span id="cb28-66"><a href="#cb28-66" aria-hidden="true" tabindex="-1"></a>  ifile <span class="ot">=</span> ifiles[i]</span>
<span id="cb28-67"><a href="#cb28-67" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cat</span>(<span class="st">&quot; -File:&quot;</span>, ifile, <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>)</span>
<span id="cb28-68"><a href="#cb28-68" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb28-69"><a href="#cb28-69" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb28-70"><a href="#cb28-70" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cat</span>(<span class="st">&quot; -Reading data</span><span class="sc">\n</span><span class="st">&quot;</span>)</span>
<span id="cb28-71"><a href="#cb28-71" aria-hidden="true" tabindex="-1"></a>  g <span class="ot">&lt;-</span> data.table<span class="sc">::</span><span class="fu">fread</span>(<span class="at">cmd =</span> ifile, <span class="at">skip =</span> <span class="st">&quot;SNP Name&quot;</span>, <span class="at">data.table =</span> <span class="cn">FALSE</span>)</span>
<span id="cb28-72"><a href="#cb28-72" aria-hidden="true" tabindex="-1"></a>  <span class="co"># subset to the markers in the codes object</span></span>
<span id="cb28-73"><a href="#cb28-73" aria-hidden="true" tabindex="-1"></a>  g <span class="ot">&lt;-</span> g[g[,<span class="st">&quot;SNP Name&quot;</span>] <span class="sc">%in%</span> codes[,<span class="st">&quot;marker&quot;</span>],]</span>
<span id="cb28-74"><a href="#cb28-74" aria-hidden="true" tabindex="-1"></a>  g<span class="sc">$</span><span class="st">`</span><span class="at">Sample ID</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">paste0</span>(g<span class="sc">$</span><span class="st">`</span><span class="at">Sample ID</span><span class="st">`</span>,<span class="st">&quot;_file&quot;</span>,<span class="fu">as.character</span>(ifiles_dates)[i])</span>
<span id="cb28-75"><a href="#cb28-75" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb28-76"><a href="#cb28-76" aria-hidden="true" tabindex="-1"></a>  <span class="co"># </span><span class="al">NOTE</span><span class="co">: may need to revise the IDs in the 2nd column</span></span>
<span id="cb28-77"><a href="#cb28-77" aria-hidden="true" tabindex="-1"></a>  samples <span class="ot">&lt;-</span> <span class="fu">unique</span>(g[,<span class="st">&quot;Sample ID&quot;</span>])</span>
<span id="cb28-78"><a href="#cb28-78" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb28-79"><a href="#cb28-79" aria-hidden="true" tabindex="-1"></a>  <span class="co"># matrix to contain the genotypes</span></span>
<span id="cb28-80"><a href="#cb28-80" aria-hidden="true" tabindex="-1"></a>  geno <span class="ot">&lt;-</span> <span class="fu">matrix</span>(<span class="at">nrow=</span><span class="fu">nrow</span>(codes), <span class="at">ncol=</span><span class="fu">length</span>(samples))</span>
<span id="cb28-81"><a href="#cb28-81" aria-hidden="true" tabindex="-1"></a>  <span class="fu">dimnames</span>(geno) <span class="ot">&lt;-</span> <span class="fu">list</span>(codes[,<span class="st">&quot;marker&quot;</span>], samples)</span>
<span id="cb28-82"><a href="#cb28-82" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb28-83"><a href="#cb28-83" aria-hidden="true" tabindex="-1"></a>  <span class="co"># fill in matrix</span></span>
<span id="cb28-84"><a href="#cb28-84" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cat</span>(<span class="st">&quot; -Reorganizing data</span><span class="sc">\n</span><span class="st">&quot;</span>)</span>
<span id="cb28-85"><a href="#cb28-85" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span>(i <span class="cf">in</span> <span class="fu">seq</span>(<span class="at">along=</span>samples)) {</span>
<span id="cb28-86"><a href="#cb28-86" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span>(i<span class="sc">==</span><span class="fu">round</span>(i,<span class="sc">-</span><span class="dv">1</span>)) <span class="fu">cat</span>(<span class="st">&quot; --Sample&quot;</span>, i, <span class="st">&quot;of&quot;</span>, <span class="fu">length</span>(samples), <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>)</span>
<span id="cb28-87"><a href="#cb28-87" aria-hidden="true" tabindex="-1"></a>    wh <span class="ot">&lt;-</span> (g[,<span class="st">&quot;Sample ID&quot;</span>]<span class="sc">==</span>samples[i])</span>
<span id="cb28-88"><a href="#cb28-88" aria-hidden="true" tabindex="-1"></a>    geno[g[wh,<span class="st">&quot;SNP Name&quot;</span>],i] <span class="ot">&lt;-</span> <span class="fu">paste0</span>(g[wh,<span class="st">&quot;Allele1 - Forward&quot;</span>], g[wh,<span class="st">&quot;Allele2 - Forward&quot;</span>])</span>
<span id="cb28-89"><a href="#cb28-89" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb28-90"><a href="#cb28-90" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb28-91"><a href="#cb28-91" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cat</span>(<span class="st">&quot; -Encode genotypes</span><span class="sc">\n</span><span class="st">&quot;</span>)</span>
<span id="cb28-92"><a href="#cb28-92" aria-hidden="true" tabindex="-1"></a>  geno <span class="ot">&lt;-</span> qtl2convert<span class="sc">::</span><span class="fu">encode_geno</span>(geno, <span class="fu">as.matrix</span>(codes[,<span class="fu">c</span>(<span class="st">&quot;A&quot;</span>,<span class="st">&quot;B&quot;</span>)]))</span>
<span id="cb28-93"><a href="#cb28-93" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb28-94"><a href="#cb28-94" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span>(<span class="fu">is.null</span>(full_geno)) {</span>
<span id="cb28-95"><a href="#cb28-95" aria-hidden="true" tabindex="-1"></a>    full_geno <span class="ot">&lt;-</span> geno</span>
<span id="cb28-96"><a href="#cb28-96" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb28-97"><a href="#cb28-97" aria-hidden="true" tabindex="-1"></a>    <span class="co"># if any columns in both, use those from second set</span></span>
<span id="cb28-98"><a href="#cb28-98" aria-hidden="true" tabindex="-1"></a>    full_geno <span class="ot">&lt;-</span> <span class="fu">cbind_smother</span>(full_geno, geno)</span>
<span id="cb28-99"><a href="#cb28-99" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb28-100"><a href="#cb28-100" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb28-101"><a href="#cb28-101" aria-hidden="true" tabindex="-1"></a>  <span class="co"># grab X and Y intensities</span></span>
<span id="cb28-102"><a href="#cb28-102" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cat</span>(<span class="st">&quot; -Grab X and Y intensities</span><span class="sc">\n</span><span class="st">&quot;</span>)</span>
<span id="cb28-103"><a href="#cb28-103" aria-hidden="true" tabindex="-1"></a>  gX <span class="ot">&lt;-</span> g[g[,<span class="st">&quot;SNP Name&quot;</span>] <span class="sc">%in%</span> codes[codes<span class="sc">$</span>chr<span class="sc">==</span><span class="st">&quot;X&quot;</span>,<span class="st">&quot;marker&quot;</span>],]</span>
<span id="cb28-104"><a href="#cb28-104" aria-hidden="true" tabindex="-1"></a>  gY <span class="ot">&lt;-</span> g[g[,<span class="st">&quot;SNP Name&quot;</span>] <span class="sc">%in%</span> codes[codes<span class="sc">$</span>chr<span class="sc">==</span><span class="st">&quot;Y&quot;</span>,<span class="st">&quot;marker&quot;</span>],]</span>
<span id="cb28-105"><a href="#cb28-105" aria-hidden="true" tabindex="-1"></a>  cX <span class="ot">&lt;-</span> <span class="fu">matrix</span>(<span class="at">nrow=</span><span class="fu">sum</span>(codes<span class="sc">$</span>chr<span class="sc">==</span><span class="st">&quot;X&quot;</span>),</span>
<span id="cb28-106"><a href="#cb28-106" aria-hidden="true" tabindex="-1"></a>               <span class="at">ncol=</span><span class="fu">length</span>(samples))</span>
<span id="cb28-107"><a href="#cb28-107" aria-hidden="true" tabindex="-1"></a>  <span class="fu">dimnames</span>(cX) <span class="ot">&lt;-</span> <span class="fu">list</span>(codes[codes<span class="sc">$</span>chr<span class="sc">==</span><span class="st">&quot;X&quot;</span>,<span class="st">&quot;marker&quot;</span>], samples)</span>
<span id="cb28-108"><a href="#cb28-108" aria-hidden="true" tabindex="-1"></a>  cY <span class="ot">&lt;-</span> <span class="fu">matrix</span>(<span class="at">nrow=</span><span class="fu">sum</span>(codes<span class="sc">$</span>chr<span class="sc">==</span><span class="st">&quot;Y&quot;</span>),</span>
<span id="cb28-109"><a href="#cb28-109" aria-hidden="true" tabindex="-1"></a>               <span class="at">ncol=</span><span class="fu">length</span>(samples))</span>
<span id="cb28-110"><a href="#cb28-110" aria-hidden="true" tabindex="-1"></a>  <span class="fu">dimnames</span>(cY) <span class="ot">&lt;-</span> <span class="fu">list</span>(codes[codes<span class="sc">$</span>chr<span class="sc">==</span><span class="st">&quot;Y&quot;</span>,<span class="st">&quot;marker&quot;</span>], samples)</span>
<span id="cb28-111"><a href="#cb28-111" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span>(i <span class="cf">in</span> <span class="fu">seq</span>(<span class="at">along=</span>samples)) {</span>
<span id="cb28-112"><a href="#cb28-112" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span>(i<span class="sc">==</span><span class="fu">round</span>(i,<span class="sc">-</span><span class="dv">1</span>))  {<span class="fu">cat</span>(<span class="st">&quot; --Sample&quot;</span>, i, <span class="st">&quot;of&quot;</span>, <span class="fu">length</span>(samples), <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>)</span>
<span id="cb28-113"><a href="#cb28-113" aria-hidden="true" tabindex="-1"></a>    wh <span class="ot">&lt;-</span> (gX[,<span class="st">&quot;Sample ID&quot;</span>]<span class="sc">==</span>samples[i])</span>
<span id="cb28-114"><a href="#cb28-114" aria-hidden="true" tabindex="-1"></a>    cX[gX[wh,<span class="st">&quot;SNP Name&quot;</span>],i] <span class="ot">&lt;-</span> (gX<span class="sc">$</span>X[wh] <span class="sc">+</span> gX<span class="sc">$</span>Y[wh])<span class="sc">/</span><span class="dv">2</span></span>
<span id="cb28-115"><a href="#cb28-115" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb28-116"><a href="#cb28-116" aria-hidden="true" tabindex="-1"></a>    wh <span class="ot">&lt;-</span> (gY[,<span class="st">&quot;Sample ID&quot;</span>]<span class="sc">==</span>samples[i])</span>
<span id="cb28-117"><a href="#cb28-117" aria-hidden="true" tabindex="-1"></a>    cY[gY[wh,<span class="st">&quot;SNP Name&quot;</span>],i] <span class="ot">&lt;-</span> (gY<span class="sc">$</span>X[wh] <span class="sc">+</span> gY<span class="sc">$</span>Y[wh])<span class="sc">/</span><span class="dv">2</span></span>
<span id="cb28-118"><a href="#cb28-118" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb28-119"><a href="#cb28-119" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span>(<span class="fu">is.null</span>(cXint)) {</span>
<span id="cb28-120"><a href="#cb28-120" aria-hidden="true" tabindex="-1"></a>    cXint <span class="ot">&lt;-</span> cX</span>
<span id="cb28-121"><a href="#cb28-121" aria-hidden="true" tabindex="-1"></a>    cYint <span class="ot">&lt;-</span> cY</span>
<span id="cb28-122"><a href="#cb28-122" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb28-123"><a href="#cb28-123" aria-hidden="true" tabindex="-1"></a>    <span class="co"># if any columns in both, use those from second set</span></span>
<span id="cb28-124"><a href="#cb28-124" aria-hidden="true" tabindex="-1"></a>    cXint <span class="ot">&lt;-</span> <span class="fu">cbind_smother</span>(cXint, cX)</span>
<span id="cb28-125"><a href="#cb28-125" aria-hidden="true" tabindex="-1"></a>    cYint <span class="ot">&lt;-</span> <span class="fu">cbind_smother</span>(cYint, cY)</span>
<span id="cb28-126"><a href="#cb28-126" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb28-127"><a href="#cb28-127" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb28-128"><a href="#cb28-128" aria-hidden="true" tabindex="-1"></a>  } </span>
<span id="cb28-129"><a href="#cb28-129" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<pre><code> -File: unzip -cq ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20160908/Jackson_Lab_Bubier_MURGIGV01_20160908_FinalReport.zip 
 -Reading data
 -Reorganizing data
 --Sample 10 of 286 
 --Sample 20 of 286 
 --Sample 30 of 286 
 --Sample 40 of 286 
 --Sample 50 of 286 
 --Sample 60 of 286 
 --Sample 70 of 286 
 --Sample 80 of 286 
 --Sample 90 of 286 
 --Sample 100 of 286 
 --Sample 110 of 286 
 --Sample 120 of 286 
 --Sample 130 of 286 
 --Sample 140 of 286 
 --Sample 150 of 286 
 --Sample 160 of 286 
 --Sample 170 of 286 
 --Sample 180 of 286 
 --Sample 190 of 286 
 --Sample 200 of 286 
 --Sample 210 of 286 
 --Sample 220 of 286 
 --Sample 230 of 286 
 --Sample 240 of 286 
 --Sample 250 of 286 
 --Sample 260 of 286 
 --Sample 270 of 286 
 --Sample 280 of 286 
 -Encode genotypes
 -Grab X and Y intensities
 --Sample 10 of 286 
 --Sample 20 of 286 
 --Sample 30 of 286 
 --Sample 40 of 286 
 --Sample 50 of 286 
 --Sample 60 of 286 
 --Sample 70 of 286 
 --Sample 80 of 286 
 --Sample 90 of 286 
 --Sample 100 of 286 
 --Sample 110 of 286 
 --Sample 120 of 286 
 --Sample 130 of 286 
 --Sample 140 of 286 
 --Sample 150 of 286 
 --Sample 160 of 286 
 --Sample 170 of 286 
 --Sample 180 of 286 
 --Sample 190 of 286 
 --Sample 200 of 286 
 --Sample 210 of 286 
 --Sample 220 of 286 
 --Sample 230 of 286 
 --Sample 240 of 286 
 --Sample 250 of 286 
 --Sample 260 of 286 
 --Sample 270 of 286 
 --Sample 280 of 286 
 -File: unzip -cq ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20161227/Jackson_Lab_Bubier_MURGIGV01_20161227_FinalReport.zip 
 -Reading data
 -Reorganizing data
 --Sample 10 of 192 
 --Sample 20 of 192 
 --Sample 30 of 192 
 --Sample 40 of 192 
 --Sample 50 of 192 
 --Sample 60 of 192 
 --Sample 70 of 192 
 --Sample 80 of 192 
 --Sample 90 of 192 
 --Sample 100 of 192 
 --Sample 110 of 192 
 --Sample 120 of 192 
 --Sample 130 of 192 
 --Sample 140 of 192 
 --Sample 150 of 192 
 --Sample 160 of 192 
 --Sample 170 of 192 
 --Sample 180 of 192 
 --Sample 190 of 192 
 -Encode genotypes
 -Grab X and Y intensities
 --Sample 10 of 192 
 --Sample 20 of 192 
 --Sample 30 of 192 
 --Sample 40 of 192 
 --Sample 50 of 192 
 --Sample 60 of 192 
 --Sample 70 of 192 
 --Sample 80 of 192 
 --Sample 90 of 192 
 --Sample 100 of 192 
 --Sample 110 of 192 
 --Sample 120 of 192 
 --Sample 130 of 192 
 --Sample 140 of 192 
 --Sample 150 of 192 
 --Sample 160 of 192 
 --Sample 170 of 192 
 --Sample 180 of 192 
 --Sample 190 of 192 
 -File: unzip -cq ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20170904/Jackson_Lab_Bubier_MURGIGV01_20170904_FinalReport.zip 
 -Reading data
 -Reorganizing data
 --Sample 10 of 60 
 --Sample 20 of 60 
 --Sample 30 of 60 
 --Sample 40 of 60 
 --Sample 50 of 60 
 --Sample 60 of 60 
 -Encode genotypes
 -Grab X and Y intensities
 --Sample 10 of 60 
 --Sample 20 of 60 
 --Sample 30 of 60 
 --Sample 40 of 60 
 --Sample 50 of 60 
 --Sample 60 of 60 
 -File: unzip -cq ./data/gigamuga/Jackson_Lab_Bubier_MURGIGIV01_20171001/Jackson_Lab_Bubier_MURGIGV01_20171001_FinalReport.zip 
 -Reading data
 -Reorganizing data
 --Sample 10 of 11 
 -Encode genotypes
 -Grab X and Y intensities
 --Sample 10 of 11 
 -File: unzip -cq ./data/gigamuga/Jackson_Lab_Bubier_MURGIGV01_20180518/Jackson_Lab_Bubier_MURGIGV01_20180518_FinalReport.zip 
 -Reading data
 -Reorganizing data
 --Sample 10 of 312 
 --Sample 20 of 312 
 --Sample 30 of 312 
 --Sample 40 of 312 
 --Sample 50 of 312 
 --Sample 60 of 312 
 --Sample 70 of 312 
 --Sample 80 of 312 
 --Sample 90 of 312 
 --Sample 100 of 312 
 --Sample 110 of 312 
 --Sample 120 of 312 
 --Sample 130 of 312 
 --Sample 140 of 312 
 --Sample 150 of 312 
 --Sample 160 of 312 
 --Sample 170 of 312 
 --Sample 180 of 312 
 --Sample 190 of 312 
 --Sample 200 of 312 
 --Sample 210 of 312 
 --Sample 220 of 312 
 --Sample 230 of 312 
 --Sample 240 of 312 
 --Sample 250 of 312 
 --Sample 260 of 312 
 --Sample 270 of 312 
 --Sample 280 of 312 
 --Sample 290 of 312 
 --Sample 300 of 312 
 --Sample 310 of 312 
 -Encode genotypes
 -Grab X and Y intensities
 --Sample 10 of 312 
 --Sample 20 of 312 
 --Sample 30 of 312 
 --Sample 40 of 312 
 --Sample 50 of 312 
 --Sample 60 of 312 
 --Sample 70 of 312 
 --Sample 80 of 312 
 --Sample 90 of 312 
 --Sample 100 of 312 
 --Sample 110 of 312 
 --Sample 120 of 312 
 --Sample 130 of 312 
 --Sample 140 of 312 
 --Sample 150 of 312 
 --Sample 160 of 312 
 --Sample 170 of 312 
 --Sample 180 of 312 
 --Sample 190 of 312 
 --Sample 200 of 312 
 --Sample 210 of 312 
 --Sample 220 of 312 
 --Sample 230 of 312 
 --Sample 240 of 312 
 --Sample 250 of 312 
 --Sample 260 of 312 
 --Sample 270 of 312 
 --Sample 280 of 312 
 --Sample 290 of 312 
 --Sample 300 of 312 
 --Sample 310 of 312 </code></pre>
<div class="sourceCode" id="cb30"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1" aria-hidden="true" tabindex="-1"></a><span class="co"># simple version of data.table::fread()</span></span>
<span id="cb30-2"><a href="#cb30-2" aria-hidden="true" tabindex="-1"></a>myfread2 <span class="ot">&lt;-</span> <span class="cf">function</span>(filename) data.table<span class="sc">::</span><span class="fu">fread</span>(<span class="at">cmd =</span> filename, <span class="at">data.table=</span><span class="cn">FALSE</span>, <span class="at">skip=</span><span class="dv">9</span>)</span>
<span id="cb30-3"><a href="#cb30-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-4"><a href="#cb30-4" aria-hidden="true" tabindex="-1"></a><span class="co"># data at https://doi.org/10.6084/m9.figshare.7359542.v1</span></span>
<span id="cb30-5"><a href="#cb30-5" aria-hidden="true" tabindex="-1"></a><span class="co">#     also see https://github.com/rqtl/qtl2data/tree/master/DO_Svenson291</span></span>
<span id="cb30-6"><a href="#cb30-6" aria-hidden="true" tabindex="-1"></a>zip_files <span class="ot">&lt;-</span> ifiles</span>
<span id="cb30-7"><a href="#cb30-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-8"><a href="#cb30-8" aria-hidden="true" tabindex="-1"></a><span class="co"># unzip, and read the data</span></span>
<span id="cb30-9"><a href="#cb30-9" aria-hidden="true" tabindex="-1"></a>dat <span class="ot">&lt;-</span> <span class="fu">vector</span>(<span class="st">&quot;list&quot;</span>, <span class="fu">length</span>(zip_files))</span>
<span id="cb30-10"><a href="#cb30-10" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span>(i <span class="cf">in</span> <span class="fu">seq_along</span>(zip_files)) {</span>
<span id="cb30-11"><a href="#cb30-11" aria-hidden="true" tabindex="-1"></a>  zipfile <span class="ot">&lt;-</span> zip_files[i]</span>
<span id="cb30-12"><a href="#cb30-12" aria-hidden="true" tabindex="-1"></a>  dat[[i]] <span class="ot">&lt;-</span> <span class="fu">myfread2</span>(zipfile)</span>
<span id="cb30-13"><a href="#cb30-13" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb30-14"><a href="#cb30-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-15"><a href="#cb30-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Duplicate items exist in multiple files. Resolving these duplicates.</span></span>
<span id="cb30-16"><a href="#cb30-16" aria-hidden="true" tabindex="-1"></a>datlen <span class="ot">=</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(dat)</span>
<span id="cb30-17"><a href="#cb30-17" aria-hidden="true" tabindex="-1"></a>lenobjdat <span class="ot">=</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">length</span>(datlen))</span>
<span id="cb30-18"><a href="#cb30-18" aria-hidden="true" tabindex="-1"></a>filedate <span class="ot">=</span> <span class="fu">c</span>()</span>
<span id="cb30-19"><a href="#cb30-19" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> datlen) {</span>
<span id="cb30-20"><a href="#cb30-20" aria-hidden="true" tabindex="-1"></a>  lenobjdat[i] <span class="ot">=</span> <span class="fu">nrow</span>(dat[[i]])</span>
<span id="cb30-21"><a href="#cb30-21" aria-hidden="true" tabindex="-1"></a>  filedate <span class="ot">=</span> <span class="fu">c</span>(filedate, <span class="fu">rep</span>(<span class="fu">as.character</span>(ifiles_dates[i]), <span class="at">times =</span> lenobjdat[i] ))</span>
<span id="cb30-22"><a href="#cb30-22" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb30-23"><a href="#cb30-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-24"><a href="#cb30-24" aria-hidden="true" tabindex="-1"></a><span class="co"># rbind the results together, saving selected columns</span></span>
<span id="cb30-25"><a href="#cb30-25" aria-hidden="true" tabindex="-1"></a>dat <span class="ot">&lt;-</span> <span class="fu">do.call</span>(<span class="st">&quot;rbind&quot;</span>, dat)[,<span class="fu">c</span>(<span class="st">&quot;SNP Name&quot;</span>, <span class="st">&quot;Sample ID&quot;</span>, <span class="st">&quot;X&quot;</span>, <span class="st">&quot;Y&quot;</span>)]</span>
<span id="cb30-26"><a href="#cb30-26" aria-hidden="true" tabindex="-1"></a>dat<span class="sc">$</span><span class="st">`</span><span class="at">Sample ID</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">paste0</span>(dat<span class="sc">$</span><span class="st">`</span><span class="at">Sample ID</span><span class="st">`</span>, <span class="st">&quot;_file&quot;</span>, filedate)</span>
<span id="cb30-27"><a href="#cb30-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-28"><a href="#cb30-28" aria-hidden="true" tabindex="-1"></a><span class="co"># create matrices that are snps x samples</span></span>
<span id="cb30-29"><a href="#cb30-29" aria-hidden="true" tabindex="-1"></a>snps <span class="ot">&lt;-</span> <span class="fu">unique</span>(dat[,<span class="st">&quot;SNP Name&quot;</span>])</span>
<span id="cb30-30"><a href="#cb30-30" aria-hidden="true" tabindex="-1"></a>samples <span class="ot">&lt;-</span> <span class="fu">unique</span>(dat[,<span class="st">&quot;Sample ID&quot;</span>])</span>
<span id="cb30-31"><a href="#cb30-31" aria-hidden="true" tabindex="-1"></a>X <span class="ot">&lt;-</span> Y <span class="ot">&lt;-</span> <span class="fu">matrix</span>(<span class="at">ncol=</span><span class="fu">length</span>(samples), <span class="at">nrow=</span><span class="fu">length</span>(snps))</span>
<span id="cb30-32"><a href="#cb30-32" aria-hidden="true" tabindex="-1"></a><span class="fu">dimnames</span>(X) <span class="ot">&lt;-</span> <span class="fu">dimnames</span>(Y) <span class="ot">&lt;-</span> <span class="fu">list</span>(snps, samples)</span>
<span id="cb30-33"><a href="#cb30-33" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span>(i <span class="cf">in</span> <span class="fu">seq</span>(<span class="at">along=</span>samples)) {</span>
<span id="cb30-34"><a href="#cb30-34" aria-hidden="true" tabindex="-1"></a>  <span class="fu">message</span>(i, <span class="st">&quot; of &quot;</span>, <span class="fu">length</span>(samples))</span>
<span id="cb30-35"><a href="#cb30-35" aria-hidden="true" tabindex="-1"></a>  tmp <span class="ot">&lt;-</span> dat[dat[,<span class="st">&quot;Sample ID&quot;</span>]<span class="sc">==</span>samples[i],]</span>
<span id="cb30-36"><a href="#cb30-36" aria-hidden="true" tabindex="-1"></a>  X[,samples[i]] <span class="ot">&lt;-</span> tmp[,<span class="st">&quot;X&quot;</span>]</span>
<span id="cb30-37"><a href="#cb30-37" aria-hidden="true" tabindex="-1"></a>  Y[,samples[i]] <span class="ot">&lt;-</span> tmp[,<span class="st">&quot;Y&quot;</span>]</span>
<span id="cb30-38"><a href="#cb30-38" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<pre><code>1 of 861</code></pre>
<pre><code>2 of 861</code></pre>
<pre><code>3 of 861</code></pre>
<pre><code>4 of 861</code></pre>
<pre><code>5 of 861</code></pre>
<pre><code>6 of 861</code></pre>
<pre><code>7 of 861</code></pre>
<pre><code>8 of 861</code></pre>
<pre><code>9 of 861</code></pre>
<pre><code>10 of 861</code></pre>
<pre><code>11 of 861</code></pre>
<pre><code>12 of 861</code></pre>
<pre><code>13 of 861</code></pre>
<pre><code>14 of 861</code></pre>
<pre><code>15 of 861</code></pre>
<pre><code>16 of 861</code></pre>
<pre><code>17 of 861</code></pre>
<pre><code>18 of 861</code></pre>
<pre><code>19 of 861</code></pre>
<pre><code>20 of 861</code></pre>
<pre><code>21 of 861</code></pre>
<pre><code>22 of 861</code></pre>
<pre><code>23 of 861</code></pre>
<pre><code>24 of 861</code></pre>
<pre><code>25 of 861</code></pre>
<pre><code>26 of 861</code></pre>
<pre><code>27 of 861</code></pre>
<pre><code>28 of 861</code></pre>
<pre><code>29 of 861</code></pre>
<pre><code>30 of 861</code></pre>
<pre><code>31 of 861</code></pre>
<pre><code>32 of 861</code></pre>
<pre><code>33 of 861</code></pre>
<pre><code>34 of 861</code></pre>
<pre><code>35 of 861</code></pre>
<pre><code>36 of 861</code></pre>
<pre><code>37 of 861</code></pre>
<pre><code>38 of 861</code></pre>
<pre><code>39 of 861</code></pre>
<pre><code>40 of 861</code></pre>
<pre><code>41 of 861</code></pre>
<pre><code>42 of 861</code></pre>
<pre><code>43 of 861</code></pre>
<pre><code>44 of 861</code></pre>
<pre><code>45 of 861</code></pre>
<pre><code>46 of 861</code></pre>
<pre><code>47 of 861</code></pre>
<pre><code>48 of 861</code></pre>
<pre><code>49 of 861</code></pre>
<pre><code>50 of 861</code></pre>
<pre><code>51 of 861</code></pre>
<pre><code>52 of 861</code></pre>
<pre><code>53 of 861</code></pre>
<pre><code>54 of 861</code></pre>
<pre><code>55 of 861</code></pre>
<pre><code>56 of 861</code></pre>
<pre><code>57 of 861</code></pre>
<pre><code>58 of 861</code></pre>
<pre><code>59 of 861</code></pre>
<pre><code>60 of 861</code></pre>
<pre><code>61 of 861</code></pre>
<pre><code>62 of 861</code></pre>
<pre><code>63 of 861</code></pre>
<pre><code>64 of 861</code></pre>
<pre><code>65 of 861</code></pre>
<pre><code>66 of 861</code></pre>
<pre><code>67 of 861</code></pre>
<pre><code>68 of 861</code></pre>
<pre><code>69 of 861</code></pre>
<pre><code>70 of 861</code></pre>
<pre><code>71 of 861</code></pre>
<pre><code>72 of 861</code></pre>
<pre><code>73 of 861</code></pre>
<pre><code>74 of 861</code></pre>
<pre><code>75 of 861</code></pre>
<pre><code>76 of 861</code></pre>
<pre><code>77 of 861</code></pre>
<pre><code>78 of 861</code></pre>
<pre><code>79 of 861</code></pre>
<pre><code>80 of 861</code></pre>
<pre><code>81 of 861</code></pre>
<pre><code>82 of 861</code></pre>
<pre><code>83 of 861</code></pre>
<pre><code>84 of 861</code></pre>
<pre><code>85 of 861</code></pre>
<pre><code>86 of 861</code></pre>
<pre><code>87 of 861</code></pre>
<pre><code>88 of 861</code></pre>
<pre><code>89 of 861</code></pre>
<pre><code>90 of 861</code></pre>
<pre><code>91 of 861</code></pre>
<pre><code>92 of 861</code></pre>
<pre><code>93 of 861</code></pre>
<pre><code>94 of 861</code></pre>
<pre><code>95 of 861</code></pre>
<pre><code>96 of 861</code></pre>
<pre><code>97 of 861</code></pre>
<pre><code>98 of 861</code></pre>
<pre><code>99 of 861</code></pre>
<pre><code>100 of 861</code></pre>
<pre><code>101 of 861</code></pre>
<pre><code>102 of 861</code></pre>
<pre><code>103 of 861</code></pre>
<pre><code>104 of 861</code></pre>
<pre><code>105 of 861</code></pre>
<pre><code>106 of 861</code></pre>
<pre><code>107 of 861</code></pre>
<pre><code>108 of 861</code></pre>
<pre><code>109 of 861</code></pre>
<pre><code>110 of 861</code></pre>
<pre><code>111 of 861</code></pre>
<pre><code>112 of 861</code></pre>
<pre><code>113 of 861</code></pre>
<pre><code>114 of 861</code></pre>
<pre><code>115 of 861</code></pre>
<pre><code>116 of 861</code></pre>
<pre><code>117 of 861</code></pre>
<pre><code>118 of 861</code></pre>
<pre><code>119 of 861</code></pre>
<pre><code>120 of 861</code></pre>
<pre><code>121 of 861</code></pre>
<pre><code>122 of 861</code></pre>
<pre><code>123 of 861</code></pre>
<pre><code>124 of 861</code></pre>
<pre><code>125 of 861</code></pre>
<pre><code>126 of 861</code></pre>
<pre><code>127 of 861</code></pre>
<pre><code>128 of 861</code></pre>
<pre><code>129 of 861</code></pre>
<pre><code>130 of 861</code></pre>
<pre><code>131 of 861</code></pre>
<pre><code>132 of 861</code></pre>
<pre><code>133 of 861</code></pre>
<pre><code>134 of 861</code></pre>
<pre><code>135 of 861</code></pre>
<pre><code>136 of 861</code></pre>
<pre><code>137 of 861</code></pre>
<pre><code>138 of 861</code></pre>
<pre><code>139 of 861</code></pre>
<pre><code>140 of 861</code></pre>
<pre><code>141 of 861</code></pre>
<pre><code>142 of 861</code></pre>
<pre><code>143 of 861</code></pre>
<pre><code>144 of 861</code></pre>
<pre><code>145 of 861</code></pre>
<pre><code>146 of 861</code></pre>
<pre><code>147 of 861</code></pre>
<pre><code>148 of 861</code></pre>
<pre><code>149 of 861</code></pre>
<pre><code>150 of 861</code></pre>
<pre><code>151 of 861</code></pre>
<pre><code>152 of 861</code></pre>
<pre><code>153 of 861</code></pre>
<pre><code>154 of 861</code></pre>
<pre><code>155 of 861</code></pre>
<pre><code>156 of 861</code></pre>
<pre><code>157 of 861</code></pre>
<pre><code>158 of 861</code></pre>
<pre><code>159 of 861</code></pre>
<pre><code>160 of 861</code></pre>
<pre><code>161 of 861</code></pre>
<pre><code>162 of 861</code></pre>
<pre><code>163 of 861</code></pre>
<pre><code>164 of 861</code></pre>
<pre><code>165 of 861</code></pre>
<pre><code>166 of 861</code></pre>
<pre><code>167 of 861</code></pre>
<pre><code>168 of 861</code></pre>
<pre><code>169 of 861</code></pre>
<pre><code>170 of 861</code></pre>
<pre><code>171 of 861</code></pre>
<pre><code>172 of 861</code></pre>
<pre><code>173 of 861</code></pre>
<pre><code>174 of 861</code></pre>
<pre><code>175 of 861</code></pre>
<pre><code>176 of 861</code></pre>
<pre><code>177 of 861</code></pre>
<pre><code>178 of 861</code></pre>
<pre><code>179 of 861</code></pre>
<pre><code>180 of 861</code></pre>
<pre><code>181 of 861</code></pre>
<pre><code>182 of 861</code></pre>
<pre><code>183 of 861</code></pre>
<pre><code>184 of 861</code></pre>
<pre><code>185 of 861</code></pre>
<pre><code>186 of 861</code></pre>
<pre><code>187 of 861</code></pre>
<pre><code>188 of 861</code></pre>
<pre><code>189 of 861</code></pre>
<pre><code>190 of 861</code></pre>
<pre><code>191 of 861</code></pre>
<pre><code>192 of 861</code></pre>
<pre><code>193 of 861</code></pre>
<pre><code>194 of 861</code></pre>
<pre><code>195 of 861</code></pre>
<pre><code>196 of 861</code></pre>
<pre><code>197 of 861</code></pre>
<pre><code>198 of 861</code></pre>
<pre><code>199 of 861</code></pre>
<pre><code>200 of 861</code></pre>
<pre><code>201 of 861</code></pre>
<pre><code>202 of 861</code></pre>
<pre><code>203 of 861</code></pre>
<pre><code>204 of 861</code></pre>
<pre><code>205 of 861</code></pre>
<pre><code>206 of 861</code></pre>
<pre><code>207 of 861</code></pre>
<pre><code>208 of 861</code></pre>
<pre><code>209 of 861</code></pre>
<pre><code>210 of 861</code></pre>
<pre><code>211 of 861</code></pre>
<pre><code>212 of 861</code></pre>
<pre><code>213 of 861</code></pre>
<pre><code>214 of 861</code></pre>
<pre><code>215 of 861</code></pre>
<pre><code>216 of 861</code></pre>
<pre><code>217 of 861</code></pre>
<pre><code>218 of 861</code></pre>
<pre><code>219 of 861</code></pre>
<pre><code>220 of 861</code></pre>
<pre><code>221 of 861</code></pre>
<pre><code>222 of 861</code></pre>
<pre><code>223 of 861</code></pre>
<pre><code>224 of 861</code></pre>
<pre><code>225 of 861</code></pre>
<pre><code>226 of 861</code></pre>
<pre><code>227 of 861</code></pre>
<pre><code>228 of 861</code></pre>
<pre><code>229 of 861</code></pre>
<pre><code>230 of 861</code></pre>
<pre><code>231 of 861</code></pre>
<pre><code>232 of 861</code></pre>
<pre><code>233 of 861</code></pre>
<pre><code>234 of 861</code></pre>
<pre><code>235 of 861</code></pre>
<pre><code>236 of 861</code></pre>
<pre><code>237 of 861</code></pre>
<pre><code>238 of 861</code></pre>
<pre><code>239 of 861</code></pre>
<pre><code>240 of 861</code></pre>
<pre><code>241 of 861</code></pre>
<pre><code>242 of 861</code></pre>
<pre><code>243 of 861</code></pre>
<pre><code>244 of 861</code></pre>
<pre><code>245 of 861</code></pre>
<pre><code>246 of 861</code></pre>
<pre><code>247 of 861</code></pre>
<pre><code>248 of 861</code></pre>
<pre><code>249 of 861</code></pre>
<pre><code>250 of 861</code></pre>
<pre><code>251 of 861</code></pre>
<pre><code>252 of 861</code></pre>
<pre><code>253 of 861</code></pre>
<pre><code>254 of 861</code></pre>
<pre><code>255 of 861</code></pre>
<pre><code>256 of 861</code></pre>
<pre><code>257 of 861</code></pre>
<pre><code>258 of 861</code></pre>
<pre><code>259 of 861</code></pre>
<pre><code>260 of 861</code></pre>
<pre><code>261 of 861</code></pre>
<pre><code>262 of 861</code></pre>
<pre><code>263 of 861</code></pre>
<pre><code>264 of 861</code></pre>
<pre><code>265 of 861</code></pre>
<pre><code>266 of 861</code></pre>
<pre><code>267 of 861</code></pre>
<pre><code>268 of 861</code></pre>
<pre><code>269 of 861</code></pre>
<pre><code>270 of 861</code></pre>
<pre><code>271 of 861</code></pre>
<pre><code>272 of 861</code></pre>
<pre><code>273 of 861</code></pre>
<pre><code>274 of 861</code></pre>
<pre><code>275 of 861</code></pre>
<pre><code>276 of 861</code></pre>
<pre><code>277 of 861</code></pre>
<pre><code>278 of 861</code></pre>
<pre><code>279 of 861</code></pre>
<pre><code>280 of 861</code></pre>
<pre><code>281 of 861</code></pre>
<pre><code>282 of 861</code></pre>
<pre><code>283 of 861</code></pre>
<pre><code>284 of 861</code></pre>
<pre><code>285 of 861</code></pre>
<pre><code>286 of 861</code></pre>
<pre><code>287 of 861</code></pre>
<pre><code>288 of 861</code></pre>
<pre><code>289 of 861</code></pre>
<pre><code>290 of 861</code></pre>
<pre><code>291 of 861</code></pre>
<pre><code>292 of 861</code></pre>
<pre><code>293 of 861</code></pre>
<pre><code>294 of 861</code></pre>
<pre><code>295 of 861</code></pre>
<pre><code>296 of 861</code></pre>
<pre><code>297 of 861</code></pre>
<pre><code>298 of 861</code></pre>
<pre><code>299 of 861</code></pre>
<pre><code>300 of 861</code></pre>
<pre><code>301 of 861</code></pre>
<pre><code>302 of 861</code></pre>
<pre><code>303 of 861</code></pre>
<pre><code>304 of 861</code></pre>
<pre><code>305 of 861</code></pre>
<pre><code>306 of 861</code></pre>
<pre><code>307 of 861</code></pre>
<pre><code>308 of 861</code></pre>
<pre><code>309 of 861</code></pre>
<pre><code>310 of 861</code></pre>
<pre><code>311 of 861</code></pre>
<pre><code>312 of 861</code></pre>
<pre><code>313 of 861</code></pre>
<pre><code>314 of 861</code></pre>
<pre><code>315 of 861</code></pre>
<pre><code>316 of 861</code></pre>
<pre><code>317 of 861</code></pre>
<pre><code>318 of 861</code></pre>
<pre><code>319 of 861</code></pre>
<pre><code>320 of 861</code></pre>
<pre><code>321 of 861</code></pre>
<pre><code>322 of 861</code></pre>
<pre><code>323 of 861</code></pre>
<pre><code>324 of 861</code></pre>
<pre><code>325 of 861</code></pre>
<pre><code>326 of 861</code></pre>
<pre><code>327 of 861</code></pre>
<pre><code>328 of 861</code></pre>
<pre><code>329 of 861</code></pre>
<pre><code>330 of 861</code></pre>
<pre><code>331 of 861</code></pre>
<pre><code>332 of 861</code></pre>
<pre><code>333 of 861</code></pre>
<pre><code>334 of 861</code></pre>
<pre><code>335 of 861</code></pre>
<pre><code>336 of 861</code></pre>
<pre><code>337 of 861</code></pre>
<pre><code>338 of 861</code></pre>
<pre><code>339 of 861</code></pre>
<pre><code>340 of 861</code></pre>
<pre><code>341 of 861</code></pre>
<pre><code>342 of 861</code></pre>
<pre><code>343 of 861</code></pre>
<pre><code>344 of 861</code></pre>
<pre><code>345 of 861</code></pre>
<pre><code>346 of 861</code></pre>
<pre><code>347 of 861</code></pre>
<pre><code>348 of 861</code></pre>
<pre><code>349 of 861</code></pre>
<pre><code>350 of 861</code></pre>
<pre><code>351 of 861</code></pre>
<pre><code>352 of 861</code></pre>
<pre><code>353 of 861</code></pre>
<pre><code>354 of 861</code></pre>
<pre><code>355 of 861</code></pre>
<pre><code>356 of 861</code></pre>
<pre><code>357 of 861</code></pre>
<pre><code>358 of 861</code></pre>
<pre><code>359 of 861</code></pre>
<pre><code>360 of 861</code></pre>
<pre><code>361 of 861</code></pre>
<pre><code>362 of 861</code></pre>
<pre><code>363 of 861</code></pre>
<pre><code>364 of 861</code></pre>
<pre><code>365 of 861</code></pre>
<pre><code>366 of 861</code></pre>
<pre><code>367 of 861</code></pre>
<pre><code>368 of 861</code></pre>
<pre><code>369 of 861</code></pre>
<pre><code>370 of 861</code></pre>
<pre><code>371 of 861</code></pre>
<pre><code>372 of 861</code></pre>
<pre><code>373 of 861</code></pre>
<pre><code>374 of 861</code></pre>
<pre><code>375 of 861</code></pre>
<pre><code>376 of 861</code></pre>
<pre><code>377 of 861</code></pre>
<pre><code>378 of 861</code></pre>
<pre><code>379 of 861</code></pre>
<pre><code>380 of 861</code></pre>
<pre><code>381 of 861</code></pre>
<pre><code>382 of 861</code></pre>
<pre><code>383 of 861</code></pre>
<pre><code>384 of 861</code></pre>
<pre><code>385 of 861</code></pre>
<pre><code>386 of 861</code></pre>
<pre><code>387 of 861</code></pre>
<pre><code>388 of 861</code></pre>
<pre><code>389 of 861</code></pre>
<pre><code>390 of 861</code></pre>
<pre><code>391 of 861</code></pre>
<pre><code>392 of 861</code></pre>
<pre><code>393 of 861</code></pre>
<pre><code>394 of 861</code></pre>
<pre><code>395 of 861</code></pre>
<pre><code>396 of 861</code></pre>
<pre><code>397 of 861</code></pre>
<pre><code>398 of 861</code></pre>
<pre><code>399 of 861</code></pre>
<pre><code>400 of 861</code></pre>
<pre><code>401 of 861</code></pre>
<pre><code>402 of 861</code></pre>
<pre><code>403 of 861</code></pre>
<pre><code>404 of 861</code></pre>
<pre><code>405 of 861</code></pre>
<pre><code>406 of 861</code></pre>
<pre><code>407 of 861</code></pre>
<pre><code>408 of 861</code></pre>
<pre><code>409 of 861</code></pre>
<pre><code>410 of 861</code></pre>
<pre><code>411 of 861</code></pre>
<pre><code>412 of 861</code></pre>
<pre><code>413 of 861</code></pre>
<pre><code>414 of 861</code></pre>
<pre><code>415 of 861</code></pre>
<pre><code>416 of 861</code></pre>
<pre><code>417 of 861</code></pre>
<pre><code>418 of 861</code></pre>
<pre><code>419 of 861</code></pre>
<pre><code>420 of 861</code></pre>
<pre><code>421 of 861</code></pre>
<pre><code>422 of 861</code></pre>
<pre><code>423 of 861</code></pre>
<pre><code>424 of 861</code></pre>
<pre><code>425 of 861</code></pre>
<pre><code>426 of 861</code></pre>
<pre><code>427 of 861</code></pre>
<pre><code>428 of 861</code></pre>
<pre><code>429 of 861</code></pre>
<pre><code>430 of 861</code></pre>
<pre><code>431 of 861</code></pre>
<pre><code>432 of 861</code></pre>
<pre><code>433 of 861</code></pre>
<pre><code>434 of 861</code></pre>
<pre><code>435 of 861</code></pre>
<pre><code>436 of 861</code></pre>
<pre><code>437 of 861</code></pre>
<pre><code>438 of 861</code></pre>
<pre><code>439 of 861</code></pre>
<pre><code>440 of 861</code></pre>
<pre><code>441 of 861</code></pre>
<pre><code>442 of 861</code></pre>
<pre><code>443 of 861</code></pre>
<pre><code>444 of 861</code></pre>
<pre><code>445 of 861</code></pre>
<pre><code>446 of 861</code></pre>
<pre><code>447 of 861</code></pre>
<pre><code>448 of 861</code></pre>
<pre><code>449 of 861</code></pre>
<pre><code>450 of 861</code></pre>
<pre><code>451 of 861</code></pre>
<pre><code>452 of 861</code></pre>
<pre><code>453 of 861</code></pre>
<pre><code>454 of 861</code></pre>
<pre><code>455 of 861</code></pre>
<pre><code>456 of 861</code></pre>
<pre><code>457 of 861</code></pre>
<pre><code>458 of 861</code></pre>
<pre><code>459 of 861</code></pre>
<pre><code>460 of 861</code></pre>
<pre><code>461 of 861</code></pre>
<pre><code>462 of 861</code></pre>
<pre><code>463 of 861</code></pre>
<pre><code>464 of 861</code></pre>
<pre><code>465 of 861</code></pre>
<pre><code>466 of 861</code></pre>
<pre><code>467 of 861</code></pre>
<pre><code>468 of 861</code></pre>
<pre><code>469 of 861</code></pre>
<pre><code>470 of 861</code></pre>
<pre><code>471 of 861</code></pre>
<pre><code>472 of 861</code></pre>
<pre><code>473 of 861</code></pre>
<pre><code>474 of 861</code></pre>
<pre><code>475 of 861</code></pre>
<pre><code>476 of 861</code></pre>
<pre><code>477 of 861</code></pre>
<pre><code>478 of 861</code></pre>
<pre><code>479 of 861</code></pre>
<pre><code>480 of 861</code></pre>
<pre><code>481 of 861</code></pre>
<pre><code>482 of 861</code></pre>
<pre><code>483 of 861</code></pre>
<pre><code>484 of 861</code></pre>
<pre><code>485 of 861</code></pre>
<pre><code>486 of 861</code></pre>
<pre><code>487 of 861</code></pre>
<pre><code>488 of 861</code></pre>
<pre><code>489 of 861</code></pre>
<pre><code>490 of 861</code></pre>
<pre><code>491 of 861</code></pre>
<pre><code>492 of 861</code></pre>
<pre><code>493 of 861</code></pre>
<pre><code>494 of 861</code></pre>
<pre><code>495 of 861</code></pre>
<pre><code>496 of 861</code></pre>
<pre><code>497 of 861</code></pre>
<pre><code>498 of 861</code></pre>
<pre><code>499 of 861</code></pre>
<pre><code>500 of 861</code></pre>
<pre><code>501 of 861</code></pre>
<pre><code>502 of 861</code></pre>
<pre><code>503 of 861</code></pre>
<pre><code>504 of 861</code></pre>
<pre><code>505 of 861</code></pre>
<pre><code>506 of 861</code></pre>
<pre><code>507 of 861</code></pre>
<pre><code>508 of 861</code></pre>
<pre><code>509 of 861</code></pre>
<pre><code>510 of 861</code></pre>
<pre><code>511 of 861</code></pre>
<pre><code>512 of 861</code></pre>
<pre><code>513 of 861</code></pre>
<pre><code>514 of 861</code></pre>
<pre><code>515 of 861</code></pre>
<pre><code>516 of 861</code></pre>
<pre><code>517 of 861</code></pre>
<pre><code>518 of 861</code></pre>
<pre><code>519 of 861</code></pre>
<pre><code>520 of 861</code></pre>
<pre><code>521 of 861</code></pre>
<pre><code>522 of 861</code></pre>
<pre><code>523 of 861</code></pre>
<pre><code>524 of 861</code></pre>
<pre><code>525 of 861</code></pre>
<pre><code>526 of 861</code></pre>
<pre><code>527 of 861</code></pre>
<pre><code>528 of 861</code></pre>
<pre><code>529 of 861</code></pre>
<pre><code>530 of 861</code></pre>
<pre><code>531 of 861</code></pre>
<pre><code>532 of 861</code></pre>
<pre><code>533 of 861</code></pre>
<pre><code>534 of 861</code></pre>
<pre><code>535 of 861</code></pre>
<pre><code>536 of 861</code></pre>
<pre><code>537 of 861</code></pre>
<pre><code>538 of 861</code></pre>
<pre><code>539 of 861</code></pre>
<pre><code>540 of 861</code></pre>
<pre><code>541 of 861</code></pre>
<pre><code>542 of 861</code></pre>
<pre><code>543 of 861</code></pre>
<pre><code>544 of 861</code></pre>
<pre><code>545 of 861</code></pre>
<pre><code>546 of 861</code></pre>
<pre><code>547 of 861</code></pre>
<pre><code>548 of 861</code></pre>
<pre><code>549 of 861</code></pre>
<pre><code>550 of 861</code></pre>
<pre><code>551 of 861</code></pre>
<pre><code>552 of 861</code></pre>
<pre><code>553 of 861</code></pre>
<pre><code>554 of 861</code></pre>
<pre><code>555 of 861</code></pre>
<pre><code>556 of 861</code></pre>
<pre><code>557 of 861</code></pre>
<pre><code>558 of 861</code></pre>
<pre><code>559 of 861</code></pre>
<pre><code>560 of 861</code></pre>
<pre><code>561 of 861</code></pre>
<pre><code>562 of 861</code></pre>
<pre><code>563 of 861</code></pre>
<pre><code>564 of 861</code></pre>
<pre><code>565 of 861</code></pre>
<pre><code>566 of 861</code></pre>
<pre><code>567 of 861</code></pre>
<pre><code>568 of 861</code></pre>
<pre><code>569 of 861</code></pre>
<pre><code>570 of 861</code></pre>
<pre><code>571 of 861</code></pre>
<pre><code>572 of 861</code></pre>
<pre><code>573 of 861</code></pre>
<pre><code>574 of 861</code></pre>
<pre><code>575 of 861</code></pre>
<pre><code>576 of 861</code></pre>
<pre><code>577 of 861</code></pre>
<pre><code>578 of 861</code></pre>
<pre><code>579 of 861</code></pre>
<pre><code>580 of 861</code></pre>
<pre><code>581 of 861</code></pre>
<pre><code>582 of 861</code></pre>
<pre><code>583 of 861</code></pre>
<pre><code>584 of 861</code></pre>
<pre><code>585 of 861</code></pre>
<pre><code>586 of 861</code></pre>
<pre><code>587 of 861</code></pre>
<pre><code>588 of 861</code></pre>
<pre><code>589 of 861</code></pre>
<pre><code>590 of 861</code></pre>
<pre><code>591 of 861</code></pre>
<pre><code>592 of 861</code></pre>
<pre><code>593 of 861</code></pre>
<pre><code>594 of 861</code></pre>
<pre><code>595 of 861</code></pre>
<pre><code>596 of 861</code></pre>
<pre><code>597 of 861</code></pre>
<pre><code>598 of 861</code></pre>
<pre><code>599 of 861</code></pre>
<pre><code>600 of 861</code></pre>
<pre><code>601 of 861</code></pre>
<pre><code>602 of 861</code></pre>
<pre><code>603 of 861</code></pre>
<pre><code>604 of 861</code></pre>
<pre><code>605 of 861</code></pre>
<pre><code>606 of 861</code></pre>
<pre><code>607 of 861</code></pre>
<pre><code>608 of 861</code></pre>
<pre><code>609 of 861</code></pre>
<pre><code>610 of 861</code></pre>
<pre><code>611 of 861</code></pre>
<pre><code>612 of 861</code></pre>
<pre><code>613 of 861</code></pre>
<pre><code>614 of 861</code></pre>
<pre><code>615 of 861</code></pre>
<pre><code>616 of 861</code></pre>
<pre><code>617 of 861</code></pre>
<pre><code>618 of 861</code></pre>
<pre><code>619 of 861</code></pre>
<pre><code>620 of 861</code></pre>
<pre><code>621 of 861</code></pre>
<pre><code>622 of 861</code></pre>
<pre><code>623 of 861</code></pre>
<pre><code>624 of 861</code></pre>
<pre><code>625 of 861</code></pre>
<pre><code>626 of 861</code></pre>
<pre><code>627 of 861</code></pre>
<pre><code>628 of 861</code></pre>
<pre><code>629 of 861</code></pre>
<pre><code>630 of 861</code></pre>
<pre><code>631 of 861</code></pre>
<pre><code>632 of 861</code></pre>
<pre><code>633 of 861</code></pre>
<pre><code>634 of 861</code></pre>
<pre><code>635 of 861</code></pre>
<pre><code>636 of 861</code></pre>
<pre><code>637 of 861</code></pre>
<pre><code>638 of 861</code></pre>
<pre><code>639 of 861</code></pre>
<pre><code>640 of 861</code></pre>
<pre><code>641 of 861</code></pre>
<pre><code>642 of 861</code></pre>
<pre><code>643 of 861</code></pre>
<pre><code>644 of 861</code></pre>
<pre><code>645 of 861</code></pre>
<pre><code>646 of 861</code></pre>
<pre><code>647 of 861</code></pre>
<pre><code>648 of 861</code></pre>
<pre><code>649 of 861</code></pre>
<pre><code>650 of 861</code></pre>
<pre><code>651 of 861</code></pre>
<pre><code>652 of 861</code></pre>
<pre><code>653 of 861</code></pre>
<pre><code>654 of 861</code></pre>
<pre><code>655 of 861</code></pre>
<pre><code>656 of 861</code></pre>
<pre><code>657 of 861</code></pre>
<pre><code>658 of 861</code></pre>
<pre><code>659 of 861</code></pre>
<pre><code>660 of 861</code></pre>
<pre><code>661 of 861</code></pre>
<pre><code>662 of 861</code></pre>
<pre><code>663 of 861</code></pre>
<pre><code>664 of 861</code></pre>
<pre><code>665 of 861</code></pre>
<pre><code>666 of 861</code></pre>
<pre><code>667 of 861</code></pre>
<pre><code>668 of 861</code></pre>
<pre><code>669 of 861</code></pre>
<pre><code>670 of 861</code></pre>
<pre><code>671 of 861</code></pre>
<pre><code>672 of 861</code></pre>
<pre><code>673 of 861</code></pre>
<pre><code>674 of 861</code></pre>
<pre><code>675 of 861</code></pre>
<pre><code>676 of 861</code></pre>
<pre><code>677 of 861</code></pre>
<pre><code>678 of 861</code></pre>
<pre><code>679 of 861</code></pre>
<pre><code>680 of 861</code></pre>
<pre><code>681 of 861</code></pre>
<pre><code>682 of 861</code></pre>
<pre><code>683 of 861</code></pre>
<pre><code>684 of 861</code></pre>
<pre><code>685 of 861</code></pre>
<pre><code>686 of 861</code></pre>
<pre><code>687 of 861</code></pre>
<pre><code>688 of 861</code></pre>
<pre><code>689 of 861</code></pre>
<pre><code>690 of 861</code></pre>
<pre><code>691 of 861</code></pre>
<pre><code>692 of 861</code></pre>
<pre><code>693 of 861</code></pre>
<pre><code>694 of 861</code></pre>
<pre><code>695 of 861</code></pre>
<pre><code>696 of 861</code></pre>
<pre><code>697 of 861</code></pre>
<pre><code>698 of 861</code></pre>
<pre><code>699 of 861</code></pre>
<pre><code>700 of 861</code></pre>
<pre><code>701 of 861</code></pre>
<pre><code>702 of 861</code></pre>
<pre><code>703 of 861</code></pre>
<pre><code>704 of 861</code></pre>
<pre><code>705 of 861</code></pre>
<pre><code>706 of 861</code></pre>
<pre><code>707 of 861</code></pre>
<pre><code>708 of 861</code></pre>
<pre><code>709 of 861</code></pre>
<pre><code>710 of 861</code></pre>
<pre><code>711 of 861</code></pre>
<pre><code>712 of 861</code></pre>
<pre><code>713 of 861</code></pre>
<pre><code>714 of 861</code></pre>
<pre><code>715 of 861</code></pre>
<pre><code>716 of 861</code></pre>
<pre><code>717 of 861</code></pre>
<pre><code>718 of 861</code></pre>
<pre><code>719 of 861</code></pre>
<pre><code>720 of 861</code></pre>
<pre><code>721 of 861</code></pre>
<pre><code>722 of 861</code></pre>
<pre><code>723 of 861</code></pre>
<pre><code>724 of 861</code></pre>
<pre><code>725 of 861</code></pre>
<pre><code>726 of 861</code></pre>
<pre><code>727 of 861</code></pre>
<pre><code>728 of 861</code></pre>
<pre><code>729 of 861</code></pre>
<pre><code>730 of 861</code></pre>
<pre><code>731 of 861</code></pre>
<pre><code>732 of 861</code></pre>
<pre><code>733 of 861</code></pre>
<pre><code>734 of 861</code></pre>
<pre><code>735 of 861</code></pre>
<pre><code>736 of 861</code></pre>
<pre><code>737 of 861</code></pre>
<pre><code>738 of 861</code></pre>
<pre><code>739 of 861</code></pre>
<pre><code>740 of 861</code></pre>
<pre><code>741 of 861</code></pre>
<pre><code>742 of 861</code></pre>
<pre><code>743 of 861</code></pre>
<pre><code>744 of 861</code></pre>
<pre><code>745 of 861</code></pre>
<pre><code>746 of 861</code></pre>
<pre><code>747 of 861</code></pre>
<pre><code>748 of 861</code></pre>
<pre><code>749 of 861</code></pre>
<pre><code>750 of 861</code></pre>
<pre><code>751 of 861</code></pre>
<pre><code>752 of 861</code></pre>
<pre><code>753 of 861</code></pre>
<pre><code>754 of 861</code></pre>
<pre><code>755 of 861</code></pre>
<pre><code>756 of 861</code></pre>
<pre><code>757 of 861</code></pre>
<pre><code>758 of 861</code></pre>
<pre><code>759 of 861</code></pre>
<pre><code>760 of 861</code></pre>
<pre><code>761 of 861</code></pre>
<pre><code>762 of 861</code></pre>
<pre><code>763 of 861</code></pre>
<pre><code>764 of 861</code></pre>
<pre><code>765 of 861</code></pre>
<pre><code>766 of 861</code></pre>
<pre><code>767 of 861</code></pre>
<pre><code>768 of 861</code></pre>
<pre><code>769 of 861</code></pre>
<pre><code>770 of 861</code></pre>
<pre><code>771 of 861</code></pre>
<pre><code>772 of 861</code></pre>
<pre><code>773 of 861</code></pre>
<pre><code>774 of 861</code></pre>
<pre><code>775 of 861</code></pre>
<pre><code>776 of 861</code></pre>
<pre><code>777 of 861</code></pre>
<pre><code>778 of 861</code></pre>
<pre><code>779 of 861</code></pre>
<pre><code>780 of 861</code></pre>
<pre><code>781 of 861</code></pre>
<pre><code>782 of 861</code></pre>
<pre><code>783 of 861</code></pre>
<pre><code>784 of 861</code></pre>
<pre><code>785 of 861</code></pre>
<pre><code>786 of 861</code></pre>
<pre><code>787 of 861</code></pre>
<pre><code>788 of 861</code></pre>
<pre><code>789 of 861</code></pre>
<pre><code>790 of 861</code></pre>
<pre><code>791 of 861</code></pre>
<pre><code>792 of 861</code></pre>
<pre><code>793 of 861</code></pre>
<pre><code>794 of 861</code></pre>
<pre><code>795 of 861</code></pre>
<pre><code>796 of 861</code></pre>
<pre><code>797 of 861</code></pre>
<pre><code>798 of 861</code></pre>
<pre><code>799 of 861</code></pre>
<pre><code>800 of 861</code></pre>
<pre><code>801 of 861</code></pre>
<pre><code>802 of 861</code></pre>
<pre><code>803 of 861</code></pre>
<pre><code>804 of 861</code></pre>
<pre><code>805 of 861</code></pre>
<pre><code>806 of 861</code></pre>
<pre><code>807 of 861</code></pre>
<pre><code>808 of 861</code></pre>
<pre><code>809 of 861</code></pre>
<pre><code>810 of 861</code></pre>
<pre><code>811 of 861</code></pre>
<pre><code>812 of 861</code></pre>
<pre><code>813 of 861</code></pre>
<pre><code>814 of 861</code></pre>
<pre><code>815 of 861</code></pre>
<pre><code>816 of 861</code></pre>
<pre><code>817 of 861</code></pre>
<pre><code>818 of 861</code></pre>
<pre><code>819 of 861</code></pre>
<pre><code>820 of 861</code></pre>
<pre><code>821 of 861</code></pre>
<pre><code>822 of 861</code></pre>
<pre><code>823 of 861</code></pre>
<pre><code>824 of 861</code></pre>
<pre><code>825 of 861</code></pre>
<pre><code>826 of 861</code></pre>
<pre><code>827 of 861</code></pre>
<pre><code>828 of 861</code></pre>
<pre><code>829 of 861</code></pre>
<pre><code>830 of 861</code></pre>
<pre><code>831 of 861</code></pre>
<pre><code>832 of 861</code></pre>
<pre><code>833 of 861</code></pre>
<pre><code>834 of 861</code></pre>
<pre><code>835 of 861</code></pre>
<pre><code>836 of 861</code></pre>
<pre><code>837 of 861</code></pre>
<pre><code>838 of 861</code></pre>
<pre><code>839 of 861</code></pre>
<pre><code>840 of 861</code></pre>
<pre><code>841 of 861</code></pre>
<pre><code>842 of 861</code></pre>
<pre><code>843 of 861</code></pre>
<pre><code>844 of 861</code></pre>
<pre><code>845 of 861</code></pre>
<pre><code>846 of 861</code></pre>
<pre><code>847 of 861</code></pre>
<pre><code>848 of 861</code></pre>
<pre><code>849 of 861</code></pre>
<pre><code>850 of 861</code></pre>
<pre><code>851 of 861</code></pre>
<pre><code>852 of 861</code></pre>
<pre><code>853 of 861</code></pre>
<pre><code>854 of 861</code></pre>
<pre><code>855 of 861</code></pre>
<pre><code>856 of 861</code></pre>
<pre><code>857 of 861</code></pre>
<pre><code>858 of 861</code></pre>
<pre><code>859 of 861</code></pre>
<pre><code>860 of 861</code></pre>
<pre><code>861 of 861</code></pre>
<div class="sourceCode" id="cb892"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb892-1"><a href="#cb892-1" aria-hidden="true" tabindex="-1"></a><span class="co"># bring together in one matrix</span></span>
<span id="cb892-2"><a href="#cb892-2" aria-hidden="true" tabindex="-1"></a>result <span class="ot">&lt;-</span> <span class="fu">cbind</span>(<span class="at">snp=</span><span class="fu">rep</span>(snps, <span class="dv">2</span>),</span>
<span id="cb892-3"><a href="#cb892-3" aria-hidden="true" tabindex="-1"></a>                <span class="at">channel=</span><span class="fu">rep</span>(<span class="fu">c</span>(<span class="st">&quot;X&quot;</span>, <span class="st">&quot;Y&quot;</span>), <span class="at">each=</span><span class="fu">length</span>(snps)),</span>
<span id="cb892-4"><a href="#cb892-4" aria-hidden="true" tabindex="-1"></a>                <span class="fu">as.data.frame</span>(<span class="fu">rbind</span>(X, Y)))</span>
<span id="cb892-5"><a href="#cb892-5" aria-hidden="true" tabindex="-1"></a><span class="fu">rownames</span>(result) <span class="ot">&lt;-</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(result)</span>
<span id="cb892-6"><a href="#cb892-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb892-7"><a href="#cb892-7" aria-hidden="true" tabindex="-1"></a><span class="co"># bring SNP rows together</span></span>
<span id="cb892-8"><a href="#cb892-8" aria-hidden="true" tabindex="-1"></a>result <span class="ot">&lt;-</span> result[<span class="fu">as.numeric</span>(<span class="fu">t</span>(<span class="fu">cbind</span>(<span class="fu">seq_along</span>(snps), <span class="fu">seq_along</span>(snps)<span class="sc">+</span><span class="fu">length</span>(snps)))),]</span>
<span id="cb892-9"><a href="#cb892-9" aria-hidden="true" tabindex="-1"></a><span class="fu">rownames</span>(result) <span class="ot">&lt;-</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(result)</span></code></pre></div>
<p>Making phenotype and covariate tables and writing these data out to csv files.</p>
<div class="sourceCode" id="cb893"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb893-1"><a href="#cb893-1" aria-hidden="true" tabindex="-1"></a>pheno_full <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">row.names =</span> <span class="fu">colnames</span>(full_geno),</span>
<span id="cb893-2"><a href="#cb893-2" aria-hidden="true" tabindex="-1"></a>                        <span class="at">id =</span> <span class="fu">colnames</span>(full_geno),</span>
<span id="cb893-3"><a href="#cb893-3" aria-hidden="true" tabindex="-1"></a>                        <span class="at">sample =</span> <span class="fu">gsub</span>(<span class="st">&quot;^(</span><span class="sc">\\</span><span class="st">d{4,5})_.*$&quot;</span>,<span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>,<span class="fu">colnames</span>(full_geno)),</span>
<span id="cb893-4"><a href="#cb893-4" aria-hidden="true" tabindex="-1"></a>                        <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb893-5"><a href="#cb893-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb893-6"><a href="#cb893-6" aria-hidden="true" tabindex="-1"></a>has_genotype_exprs                <span class="ot">=</span> <span class="fu">row.names</span>(DO_pheno)[<span class="fu">which</span>((<span class="fu">row.names</span>(DO_pheno) <span class="sc">%in%</span> pheno_full<span class="sc">$</span>sample))]</span>
<span id="cb893-7"><a href="#cb893-7" aria-hidden="true" tabindex="-1"></a>pheno_full <span class="ot">=</span> pheno_full[<span class="fu">which</span>(pheno_full<span class="sc">$</span>sample <span class="sc">%in%</span> has_genotype_exprs),]</span>
<span id="cb893-8"><a href="#cb893-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb893-9"><a href="#cb893-9" aria-hidden="true" tabindex="-1"></a>DO_pheno_has_exprs_geno           <span class="ot">=</span> <span class="fu">cbind</span>(pheno_full, DO_pheno[pheno_full<span class="sc">$</span>sample,])</span>
<span id="cb893-10"><a href="#cb893-10" aria-hidden="true" tabindex="-1"></a>DO_pheno_has_exprs_geno           <span class="ot">=</span> DO_pheno_has_exprs_geno[<span class="fu">which</span>(DO_pheno_has_exprs_geno<span class="sc">$</span>notkept_cols <span class="sc">==</span> <span class="st">&quot;kept&quot;</span>),]</span>
<span id="cb893-11"><a href="#cb893-11" aria-hidden="true" tabindex="-1"></a>DO_pheno_has_exprs_geno           <span class="ot">=</span> DO_pheno_has_exprs_geno[<span class="fu">which</span>(DO_pheno_has_exprs_geno<span class="sc">$</span>MUGA_ploidy <span class="sc">==</span> <span class="st">&quot;female&quot;</span> <span class="sc">|</span> </span>
<span id="cb893-12"><a href="#cb893-12" aria-hidden="true" tabindex="-1"></a>                                                                    DO_pheno_has_exprs_geno<span class="sc">$</span>MUGA_ploidy <span class="sc">==</span> <span class="st">&quot;male&quot;</span> ),]</span>
<span id="cb893-13"><a href="#cb893-13" aria-hidden="true" tabindex="-1"></a>DO_pheno_has_exprs_geno           <span class="ot">=</span> DO_pheno_has_exprs_geno[,<span class="fu">c</span>(<span class="st">&quot;id&quot;</span>,<span class="st">&quot;sample&quot;</span>,<span class="st">&quot;MUGA_sex&quot;</span>,<span class="st">&quot;Generation&quot;</span>,<span class="st">&quot;choroid_plexus_covariate&quot;</span>,<span class="st">&quot;DOB&quot;</span>)]</span>
<span id="cb893-14"><a href="#cb893-14" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_pheno_has_exprs_geno) <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;id&quot;</span>,<span class="st">&quot;subject&quot;</span>,<span class="st">&quot;sex&quot;</span>,<span class="st">&quot;ngen&quot;</span>,<span class="st">&quot;choroid_plexus_covariate&quot;</span>,<span class="st">&quot;dob&quot;</span>)</span>
<span id="cb893-15"><a href="#cb893-15" aria-hidden="true" tabindex="-1"></a>DO_pheno_has_exprs_geno<span class="sc">$</span>ngen <span class="ot">=</span> <span class="fu">as.numeric</span>(<span class="fu">gsub</span>(<span class="st">&quot;G&quot;</span>,<span class="st">&quot;&quot;</span>,DO_pheno_has_exprs_geno<span class="sc">$</span>ngen))</span>
<span id="cb893-16"><a href="#cb893-16" aria-hidden="true" tabindex="-1"></a>DO_pheno_has_exprs_geno[<span class="fu">which</span>(DO_pheno_has_exprs_geno<span class="sc">$</span>sex <span class="sc">==</span> <span class="st">&quot;female&quot;</span>),<span class="st">&quot;sex&quot;</span>] <span class="ot">=</span> <span class="st">&quot;F&quot;</span></span>
<span id="cb893-17"><a href="#cb893-17" aria-hidden="true" tabindex="-1"></a>DO_pheno_has_exprs_geno[<span class="fu">which</span>(DO_pheno_has_exprs_geno<span class="sc">$</span>sex <span class="sc">==</span> <span class="st">&quot;male&quot;</span>),<span class="st">&quot;sex&quot;</span>] <span class="ot">=</span> <span class="st">&quot;M&quot;</span></span>
<span id="cb893-18"><a href="#cb893-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb893-19"><a href="#cb893-19" aria-hidden="true" tabindex="-1"></a>qtl2convert<span class="sc">::</span><span class="fu">write2csv</span>(DO_pheno_has_exprs_geno, </span>
<span id="cb893-20"><a href="#cb893-20" aria-hidden="true" tabindex="-1"></a>                       <span class="fu">paste0</span>(ostem, <span class="st">&quot;covar.csv&quot;</span>),</span>
<span id="cb893-21"><a href="#cb893-21" aria-hidden="true" tabindex="-1"></a>                       <span class="fu">paste</span>(ostem, <span class="st">&quot;Phenotypes&quot;</span>),</span>
<span id="cb893-22"><a href="#cb893-22" aria-hidden="true" tabindex="-1"></a>                       <span class="at">overwrite =</span> <span class="cn">TRUE</span>)</span>
<span id="cb893-23"><a href="#cb893-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb893-24"><a href="#cb893-24" aria-hidden="true" tabindex="-1"></a>DO_voom_has_exprs_geno            <span class="ot">=</span> <span class="fu">t</span>(DO_voom)</span>
<span id="cb893-25"><a href="#cb893-25" aria-hidden="true" tabindex="-1"></a>DO_voom_has_exprs_geno            <span class="ot">=</span> DO_voom_has_exprs_geno[DO_pheno_has_exprs_geno<span class="sc">$</span>subject,]</span>
<span id="cb893-26"><a href="#cb893-26" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(DO_voom_has_exprs_geno) <span class="ot">=</span> <span class="fu">row.names</span>(DO_pheno_has_exprs_geno)</span>
<span id="cb893-27"><a href="#cb893-27" aria-hidden="true" tabindex="-1"></a>DO_voom_has_exprs_geno            <span class="ot">=</span> <span class="fu">as.data.frame</span>(DO_voom_has_exprs_geno)</span>
<span id="cb893-28"><a href="#cb893-28" aria-hidden="true" tabindex="-1"></a>DO_voom_cols                      <span class="ot">=</span> <span class="fu">colnames</span>(DO_voom_has_exprs_geno)</span>
<span id="cb893-29"><a href="#cb893-29" aria-hidden="true" tabindex="-1"></a>DO_voom_has_exprs_geno<span class="sc">$</span>id         <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">row.names</span>(DO_voom_has_exprs_geno))</span>
<span id="cb893-30"><a href="#cb893-30" aria-hidden="true" tabindex="-1"></a>DO_voom_has_exprs_geno            <span class="ot">=</span> DO_voom_has_exprs_geno[,<span class="fu">c</span>(<span class="st">&quot;id&quot;</span>,DO_voom_cols)]</span>
<span id="cb893-31"><a href="#cb893-31" aria-hidden="true" tabindex="-1"></a>qtl2convert<span class="sc">::</span><span class="fu">write2csv</span>(DO_voom_has_exprs_geno, </span>
<span id="cb893-32"><a href="#cb893-32" aria-hidden="true" tabindex="-1"></a>                       <span class="fu">paste0</span>(ostem, <span class="st">&quot;pheno.csv&quot;</span>),</span>
<span id="cb893-33"><a href="#cb893-33" aria-hidden="true" tabindex="-1"></a>                       <span class="fu">paste</span>(ostem, <span class="st">&quot;Phenotypes&quot;</span>),</span>
<span id="cb893-34"><a href="#cb893-34" aria-hidden="true" tabindex="-1"></a>                       <span class="at">overwrite =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
<p><strong>Note that the phenotype data now include duplicated information for genotype data that are duplicated.</strong></p>
<p>Writing out intensity data for these same files.</p>
<div class="sourceCode" id="cb894"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb894-1"><a href="#cb894-1" aria-hidden="true" tabindex="-1"></a>result <span class="ot">=</span> result[,<span class="fu">which</span>(<span class="fu">colnames</span>(result) <span class="sc">%in%</span> <span class="fu">row.names</span>(DO_pheno_has_exprs_geno))]</span>
<span id="cb894-2"><a href="#cb894-2" aria-hidden="true" tabindex="-1"></a><span class="co"># write to fst file, maximally compressed</span></span>
<span id="cb894-3"><a href="#cb894-3" aria-hidden="true" tabindex="-1"></a><span class="fu">write.fst</span>(result, <span class="st">&quot;./data/DO_str_2016_gm4qtl2_intensities.fst&quot;</span>, <span class="at">compress=</span><span class="dv">100</span>)</span></code></pre></div>
<p>Getting genotype data for just these chromosomes.</p>
<div class="sourceCode" id="cb895"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb895-1"><a href="#cb895-1" aria-hidden="true" tabindex="-1"></a>cXint     <span class="ot">=</span> cXint[,<span class="fu">row.names</span>(DO_pheno_has_exprs_geno)]</span>
<span id="cb895-2"><a href="#cb895-2" aria-hidden="true" tabindex="-1"></a>cYint     <span class="ot">=</span> cYint[,<span class="fu">row.names</span>(DO_pheno_has_exprs_geno)]</span>
<span id="cb895-3"><a href="#cb895-3" aria-hidden="true" tabindex="-1"></a>full_geno <span class="ot">=</span> full_geno[,<span class="fu">row.names</span>(DO_pheno_has_exprs_geno)]</span></code></pre></div>
<p>Writing out genotype data.</p>
<div class="sourceCode" id="cb896"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb896-1"><a href="#cb896-1" aria-hidden="true" tabindex="-1"></a><span class="co"># write X and Y intensities</span></span>
<span id="cb896-2"><a href="#cb896-2" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot; -Writing X and Y intensities</span><span class="sc">\n</span><span class="st">&quot;</span>)</span></code></pre></div>
<pre><code> -Writing X and Y intensities</code></pre>
<div class="sourceCode" id="cb898"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb898-1"><a href="#cb898-1" aria-hidden="true" tabindex="-1"></a>qtl2convert<span class="sc">::</span><span class="fu">write2csv</span>(<span class="fu">cbind</span>(<span class="at">marker=</span><span class="fu">rownames</span>(cXint), cXint),</span>
<span id="cb898-2"><a href="#cb898-2" aria-hidden="true" tabindex="-1"></a>                       <span class="fu">paste0</span>(ostem, <span class="st">&quot;chrXint.csv&quot;</span>),</span>
<span id="cb898-3"><a href="#cb898-3" aria-hidden="true" tabindex="-1"></a>                       <span class="fu">paste</span>(ostem, <span class="st">&quot;X chr intensities&quot;</span>),</span>
<span id="cb898-4"><a href="#cb898-4" aria-hidden="true" tabindex="-1"></a>                       <span class="at">overwrite=</span><span class="cn">TRUE</span>)</span>
<span id="cb898-5"><a href="#cb898-5" aria-hidden="true" tabindex="-1"></a>qtl2convert<span class="sc">::</span><span class="fu">write2csv</span>(<span class="fu">cbind</span>(<span class="at">marker=</span><span class="fu">rownames</span>(cYint), cYint),</span>
<span id="cb898-6"><a href="#cb898-6" aria-hidden="true" tabindex="-1"></a>                       <span class="fu">paste0</span>(ostem, <span class="st">&quot;chrYint.csv&quot;</span>),</span>
<span id="cb898-7"><a href="#cb898-7" aria-hidden="true" tabindex="-1"></a>                       <span class="fu">paste</span>(ostem, <span class="st">&quot;Y chr intensities&quot;</span>),</span>
<span id="cb898-8"><a href="#cb898-8" aria-hidden="true" tabindex="-1"></a>                       <span class="at">overwrite=</span><span class="cn">TRUE</span>)</span>
<span id="cb898-9"><a href="#cb898-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb898-10"><a href="#cb898-10" aria-hidden="true" tabindex="-1"></a><span class="co"># write data to chromosome-specific files</span></span>
<span id="cb898-11"><a href="#cb898-11" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot; -Writing genotypes</span><span class="sc">\n</span><span class="st">&quot;</span>)</span></code></pre></div>
<pre><code> -Writing genotypes</code></pre>
<div class="sourceCode" id="cb900"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb900-1"><a href="#cb900-1" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span>(chr <span class="cf">in</span> <span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">19</span>,<span class="st">&quot;X&quot;</span>,<span class="st">&quot;Y&quot;</span>,<span class="st">&quot;M&quot;</span>)) {</span>
<span id="cb900-2"><a href="#cb900-2" aria-hidden="true" tabindex="-1"></a>  mar <span class="ot">&lt;-</span> codes[codes<span class="sc">$</span>chr<span class="sc">==</span>chr,<span class="st">&quot;marker&quot;</span>]</span>
<span id="cb900-3"><a href="#cb900-3" aria-hidden="true" tabindex="-1"></a>  g <span class="ot">&lt;-</span> full_geno[mar,]</span>
<span id="cb900-4"><a href="#cb900-4" aria-hidden="true" tabindex="-1"></a>  qtl2convert<span class="sc">::</span><span class="fu">write2csv</span>(<span class="fu">cbind</span>(<span class="at">marker=</span><span class="fu">rownames</span>(g), g),</span>
<span id="cb900-5"><a href="#cb900-5" aria-hidden="true" tabindex="-1"></a>                         <span class="fu">paste0</span>(ostem, <span class="st">&quot;geno&quot;</span>, chr, <span class="st">&quot;.csv&quot;</span>),</span>
<span id="cb900-6"><a href="#cb900-6" aria-hidden="true" tabindex="-1"></a>                         <span class="fu">paste0</span>(ostem, <span class="st">&quot;_genotypes_for_chr &quot;</span>, chr),</span>
<span id="cb900-7"><a href="#cb900-7" aria-hidden="true" tabindex="-1"></a>                         <span class="at">overwrite=</span><span class="cn">TRUE</span>)</span>
<span id="cb900-8"><a href="#cb900-8" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<p>Making control file (code taken from Karl Broman’s code on the <code>qtl2</code> documentation).</p>
<div class="sourceCode" id="cb901"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb901-1"><a href="#cb901-1" aria-hidden="true" tabindex="-1"></a>chr <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">19</span>, <span class="st">&quot;X&quot;</span>)</span>
<span id="cb901-2"><a href="#cb901-2" aria-hidden="true" tabindex="-1"></a><span class="fu">write_control_file</span>(<span class="st">&quot;./data/qtl2/DO_str_2016_gm4qtl2.json&quot;</span>,</span>
<span id="cb901-3"><a href="#cb901-3" aria-hidden="true" tabindex="-1"></a>                   <span class="at">crosstype=</span><span class="st">&quot;do&quot;</span>,</span>
<span id="cb901-4"><a href="#cb901-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">description=</span><span class="st">&quot;DO Striatum 416 eQTL Data&quot;</span>,</span>
<span id="cb901-5"><a href="#cb901-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">founder_geno_file=</span><span class="fu">paste0</span>(<span class="st">&quot;GM/GM_foundergeno&quot;</span>, chr, <span class="st">&quot;.csv&quot;</span>),</span>
<span id="cb901-6"><a href="#cb901-6" aria-hidden="true" tabindex="-1"></a>                   <span class="at">founder_geno_transposed=</span><span class="cn">TRUE</span>,</span>
<span id="cb901-7"><a href="#cb901-7" aria-hidden="true" tabindex="-1"></a>                   <span class="at">gmap_file=</span><span class="fu">paste0</span>(<span class="st">&quot;GM/GM_gmap&quot;</span>, chr, <span class="st">&quot;.csv&quot;</span>),</span>
<span id="cb901-8"><a href="#cb901-8" aria-hidden="true" tabindex="-1"></a>                   <span class="at">pmap_file=</span><span class="fu">paste0</span>(<span class="st">&quot;GM/GM_pmap&quot;</span>, chr, <span class="st">&quot;.csv&quot;</span>),</span>
<span id="cb901-9"><a href="#cb901-9" aria-hidden="true" tabindex="-1"></a>                   <span class="at">geno_file=</span><span class="fu">paste0</span>(<span class="st">&quot;DO_str_2016_gm4qtl2_geno&quot;</span>, chr, <span class="st">&quot;.csv&quot;</span>),</span>
<span id="cb901-10"><a href="#cb901-10" aria-hidden="true" tabindex="-1"></a>                   <span class="at">geno_transposed=</span><span class="cn">TRUE</span>,</span>
<span id="cb901-11"><a href="#cb901-11" aria-hidden="true" tabindex="-1"></a>                   <span class="at">geno_codes=</span><span class="fu">list</span>(<span class="at">A=</span><span class="dv">1</span>, <span class="at">H=</span><span class="dv">2</span>, <span class="at">B=</span><span class="dv">3</span>),</span>
<span id="cb901-12"><a href="#cb901-12" aria-hidden="true" tabindex="-1"></a>                   <span class="at">xchr=</span><span class="st">&quot;X&quot;</span>,</span>
<span id="cb901-13"><a href="#cb901-13" aria-hidden="true" tabindex="-1"></a>                   <span class="at">pheno_file=</span><span class="st">&quot;DO_str_2016_gm4qtl2_pheno.csv&quot;</span>,</span>
<span id="cb901-14"><a href="#cb901-14" aria-hidden="true" tabindex="-1"></a>                   <span class="at">covar_file=</span><span class="st">&quot;DO_str_2016_gm4qtl2_covar.csv&quot;</span>,</span>
<span id="cb901-15"><a href="#cb901-15" aria-hidden="true" tabindex="-1"></a>                   <span class="at">sex_covar=</span><span class="st">&quot;sex&quot;</span>,</span>
<span id="cb901-16"><a href="#cb901-16" aria-hidden="true" tabindex="-1"></a>                   <span class="at">sex_codes=</span><span class="fu">list</span>(<span class="at">F=</span><span class="st">&quot;Female&quot;</span>, <span class="at">M=</span><span class="st">&quot;Male&quot;</span>),</span>
<span id="cb901-17"><a href="#cb901-17" aria-hidden="true" tabindex="-1"></a>                   <span class="at">crossinfo_covar=</span><span class="st">&quot;ngen&quot;</span>,</span>
<span id="cb901-18"><a href="#cb901-18" aria-hidden="true" tabindex="-1"></a>                   <span class="at">overwrite=</span><span class="cn">TRUE</span>)</span></code></pre></div>
<p>Zipping the data files</p>
<div class="sourceCode" id="cb902"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb902-1"><a href="#cb902-1" aria-hidden="true" tabindex="-1"></a><span class="fu">zip_datafiles</span>(<span class="st">&quot;./data/qtl2/DO_str_2016_gm4qtl2.json&quot;</span>,</span>
<span id="cb902-2"><a href="#cb902-2" aria-hidden="true" tabindex="-1"></a>              <span class="at">zip_file =</span> <span class="st">&quot;./data/DO_str_2016_gm4qtl2.zip&quot;</span>,</span>
<span id="cb902-3"><a href="#cb902-3" aria-hidden="true" tabindex="-1"></a>              <span class="at">overwrite =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
<p>The md5 checksums for the output files are:</p>
<table>
<thead>
<tr class="header">
<th>file</th>
<th>md5 checksum</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><code>DO_str_2016_gm4qtl2.zip</code></td>
<td><code>8d3e883ca5e01a1e44a2ad20e89d12cf</code></td>
</tr>
</tbody>
</table>
<p>Removing objects generated during the conversion to <code>qtl2</code> format.</p>
<div class="sourceCode" id="cb903"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb903-1"><a href="#cb903-1" aria-hidden="true" tabindex="-1"></a><span class="fu">rm</span>(<span class="at">list =</span> <span class="fu">ls</span>()[<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(<span class="fu">ls</span>() <span class="sc">%in%</span> loaded_data)])</span>
<span id="cb903-2"><a href="#cb903-2" aria-hidden="true" tabindex="-1"></a><span class="fu">rm</span>(<span class="at">list =</span> <span class="fu">c</span>(<span class="st">&quot;DO_pheno&quot;</span>,<span class="st">&quot;DO_voom&quot;</span>,<span class="st">&quot;DO_MUGA_ploidy&quot;</span>))</span></code></pre></div>
</div>
<div id="importing-data-for-rqtl2" class="section level4">
<h4>Importing data for R/qtl2</h4>
<p>Reading in the cross data.</p>
<div class="sourceCode" id="cb904"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb904-1"><a href="#cb904-1" aria-hidden="true" tabindex="-1"></a>DO_str_2016_cross <span class="ot">=</span> <span class="fu">read_cross2</span>(<span class="st">&quot;./data/DO_str_2016_gm4qtl2.zip&quot;</span>)</span></code></pre></div>
</div>
</div>
</div>
<div id="do-eqtl-analysis" class="section level2">
<h2>DO eQTL Analysis</h2>
<div id="cleaning-up-the-cross-data-percent-missing-sex-aneuploidies-and-duplicate-samples" class="section level3">
<h3>Cleaning up the cross data: percent missing, sex aneuploidies, and duplicate samples</h3>
<p>Using the procedures from the <a href="https://kbroman.org/qtl2/assets/vignettes/do_diagnostics.html">R/qtl2 vignette on DO diagnostics</a>.</p>
<div class="sourceCode" id="cb905"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb905-1"><a href="#cb905-1" aria-hidden="true" tabindex="-1"></a>DO_str_2016_cross <span class="ot">=</span> <span class="fu">drop_nullmarkers</span>(DO_str_2016_cross)</span></code></pre></div>
<pre><code>Dropping 140 markers with no data</code></pre>
<p>We have already done a systematic examination for X chromosome aneuploidies and sex mismatches and resolved or excluded those data from this analysis.</p>
<p>Looking for duplicated samples in the dataset.</p>
<div class="sourceCode" id="cb907"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb907-1"><a href="#cb907-1" aria-hidden="true" tabindex="-1"></a>cg <span class="ot">=</span> <span class="fu">compare_geno</span>(DO_str_2016_cross, <span class="at">cores =</span> <span class="dv">0</span>)</span>
<span id="cb907-2"><a href="#cb907-2" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(cg)</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["ind1"],"name":[1],"type":["chr"],"align":["left"]},{"label":["ind2"],"name":[2],"type":["chr"],"align":["left"]},{"label":["prop_match"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["n_mismatch"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["n_typed"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["n_match"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["index1"],"name":[7],"type":["int"],"align":["right"]},{"label":["index2"],"name":[8],"type":["int"],"align":["right"]}],"data":[{"1":"8673_file2017-09-04","2":"8669_file2017-09-04","3":"0.9995872","4":"46","5":"111435","6":"111389","7":"358","8":"365","_rn_":"8673_file2017-09-04"},{"1":"8616_file2017-09-04","2":"8597_file2017-09-04","3":"0.9992341","4":"85","5":"110987","6":"110902","7":"359","8":"366","_rn_":"8616_file2017-09-04"},{"1":"8618_file2017-09-04","2":"8614_file2017-09-04","3":"0.9992247","4":"86","5":"110927","6":"110841","7":"355","8":"363","_rn_":"8618_file2017-09-04"},{"1":"8664_file2016-12-27","2":"8664_file2017-09-04","3":"0.9990058","4":"111","5":"111653","6":"111542","7":"320","8":"341","_rn_":"8664_file2016-12-27"},{"1":"8585_file2016-12-27","2":"8585_file2017-09-04","3":"0.9984564","4":"172","5":"111428","6":"111256","7":"304","8":"340","_rn_":"8585_file2016-12-27"},{"1":"8674_file2017-09-04","2":"8670_file2017-09-04","3":"0.9977470","4":"246","5":"109186","6":"108940","7":"356","8":"364","_rn_":"8674_file2017-09-04"},{"1":"8167_file2016-09-08","2":"8167_file2016-12-27","3":"0.9965352","4":"383","5":"110540","6":"110157","7":"210","8":"237","_rn_":"8167_file2016-09-08"},{"1":"8168_file2016-09-08","2":"8168_file2016-12-27","3":"0.9939314","4":"666","5":"109746","6":"109080","7":"209","8":"230","_rn_":"8168_file2016-09-08"},{"1":"8183_file2016-09-08","2":"8183_file2016-12-27","3":"0.9937976","4":"677","5":"109152","6":"108475","7":"211","8":"231","_rn_":"8183_file2016-09-08"},{"1":"8185_file2016-09-08","2":"8185_file2016-12-27","3":"0.9884772","4":"1253","5":"108741","6":"107488","7":"208","8":"233","_rn_":"8185_file2016-09-08"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>There are many samples with identical or near-identical genotypes. Some of these are reruns of the same chip that might’ve failed QC (note: this is a known issue from above). Plotting percent missing as an initial step in resolving this problem.</p>
<div class="sourceCode" id="cb908"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb908-1"><a href="#cb908-1" aria-hidden="true" tabindex="-1"></a>percent_missing <span class="ot">=</span> <span class="fu">n_missing</span>(DO_str_2016_cross, <span class="st">&quot;ind&quot;</span>, <span class="st">&quot;prop&quot;</span>) <span class="sc">*</span> <span class="dv">100</span></span>
<span id="cb908-2"><a href="#cb908-2" aria-hidden="true" tabindex="-1"></a>percent_missing <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">id =</span> <span class="fu">names</span>(percent_missing),</span>
<span id="cb908-3"><a href="#cb908-3" aria-hidden="true" tabindex="-1"></a>                             <span class="at">order =</span> <span class="fu">as.numeric</span>(<span class="fu">order</span>(<span class="fu">names</span>(percent_missing))),</span>
<span id="cb908-4"><a href="#cb908-4" aria-hidden="true" tabindex="-1"></a>                             <span class="at">percent =</span> percent_missing,</span>
<span id="cb908-5"><a href="#cb908-5" aria-hidden="true" tabindex="-1"></a>                             <span class="at">gtrthan3 =</span> <span class="fu">ifelse</span>(percent_missing <span class="sc">&gt;</span> <span class="dv">3</span>, <span class="st">&quot;YES&quot;</span>,<span class="st">&quot;no&quot;</span>),</span>
<span id="cb908-6"><a href="#cb908-6" aria-hidden="true" tabindex="-1"></a>                             <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb908-7"><a href="#cb908-7" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(<span class="at">data =</span> percent_missing, <span class="fu">aes</span>(<span class="at">x =</span> order, <span class="at">y =</span> percent, <span class="at">color =</span> gtrthan3)) <span class="sc">+</span></span>
<span id="cb908-8"><a href="#cb908-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb908-9"><a href="#cb908-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_log10</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="fl">0.001</span>,<span class="dv">100</span>)) <span class="sc">+</span></span>
<span id="cb908-10"><a href="#cb908-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#555555&quot;</span>,<span class="st">&quot;#CC3333&quot;</span>))</span></code></pre></div>
<p><img src="figure/07_DO_eQTL_genotype_prep.Rmd/unnamed-chunk-11-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
a743c27
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-30
</td>
</tr>
<tr>
<td>
84aab57
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>8 points have greater than 3% missing markers. 8 (all) of those are unique. Excluding the bad files from the <code>cg</code> repeated files set.</p>
<div class="sourceCode" id="cb909"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb909-1"><a href="#cb909-1" aria-hidden="true" tabindex="-1"></a>duplicated_cg <span class="ot">=</span> <span class="fu">unique</span>(<span class="fu">c</span>(<span class="fu">summary</span>(cg)<span class="sc">$</span>ind1, <span class="fu">summary</span>(cg)<span class="sc">$</span>ind2))</span>
<span id="cb909-2"><a href="#cb909-2" aria-hidden="true" tabindex="-1"></a>exclude_cg <span class="ot">=</span> duplicated_cg[<span class="fu">which</span>(duplicated_cg <span class="sc">%in%</span> percent_missing[<span class="fu">which</span>(percent_missing<span class="sc">$</span>gtrthan3 <span class="sc">==</span> <span class="st">&quot;YES&quot;</span>),<span class="st">&quot;id&quot;</span>])]</span>
<span id="cb909-3"><a href="#cb909-3" aria-hidden="true" tabindex="-1"></a>DO_str_2016_cross <span class="ot">=</span> DO_str_2016_cross[<span class="fu">paste0</span>(<span class="st">&quot;-&quot;</span>,exclude_cg),]</span>
<span id="cb909-4"><a href="#cb909-4" aria-hidden="true" tabindex="-1"></a>cg <span class="ot">=</span> <span class="fu">compare_geno</span>(DO_str_2016_cross, <span class="at">cores =</span> <span class="dv">0</span>)</span>
<span id="cb909-5"><a href="#cb909-5" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(cg)</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["ind1"],"name":[1],"type":["chr"],"align":["left"]},{"label":["ind2"],"name":[2],"type":["chr"],"align":["left"]},{"label":["prop_match"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["n_mismatch"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["n_typed"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["n_match"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["index1"],"name":[7],"type":["int"],"align":["right"]},{"label":["index2"],"name":[8],"type":["int"],"align":["right"]}],"data":[{"1":"8673_file2017-09-04","2":"8669_file2017-09-04","3":"0.9995872","4":"46","5":"111435","6":"111389","7":"357","8":"364","_rn_":"8673_file2017-09-04"},{"1":"8616_file2017-09-04","2":"8597_file2017-09-04","3":"0.9992341","4":"85","5":"110987","6":"110902","7":"358","8":"365","_rn_":"8616_file2017-09-04"},{"1":"8618_file2017-09-04","2":"8614_file2017-09-04","3":"0.9992247","4":"86","5":"110927","6":"110841","7":"354","8":"362","_rn_":"8618_file2017-09-04"},{"1":"8664_file2016-12-27","2":"8664_file2017-09-04","3":"0.9990058","4":"111","5":"111653","6":"111542","7":"319","8":"340","_rn_":"8664_file2016-12-27"},{"1":"8585_file2016-12-27","2":"8585_file2017-09-04","3":"0.9984564","4":"172","5":"111428","6":"111256","7":"303","8":"339","_rn_":"8585_file2016-12-27"},{"1":"8674_file2017-09-04","2":"8670_file2017-09-04","3":"0.9977470","4":"246","5":"109186","6":"108940","7":"355","8":"363","_rn_":"8674_file2017-09-04"},{"1":"8167_file2016-09-08","2":"8167_file2016-12-27","3":"0.9965352","4":"383","5":"110540","6":"110157","7":"209","8":"236","_rn_":"8167_file2016-09-08"},{"1":"8168_file2016-09-08","2":"8168_file2016-12-27","3":"0.9939314","4":"666","5":"109746","6":"109080","7":"208","8":"229","_rn_":"8168_file2016-09-08"},{"1":"8183_file2016-09-08","2":"8183_file2016-12-27","3":"0.9937976","4":"677","5":"109152","6":"108475","7":"210","8":"230","_rn_":"8183_file2016-09-08"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>For the remaining genotypes, taking the one at a later date for the ones not excluded by the initial QC deduplication. Assuming that later date means there was a reason for regenotyping the animal.</p>
<div class="sourceCode" id="cb910"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb910-1"><a href="#cb910-1" aria-hidden="true" tabindex="-1"></a>duplicated_cg <span class="ot">=</span> <span class="fu">unique</span>(<span class="fu">c</span>(<span class="fu">summary</span>(cg)<span class="sc">$</span>ind1, <span class="fu">summary</span>(cg)<span class="sc">$</span>ind2))</span>
<span id="cb910-2"><a href="#cb910-2" aria-hidden="true" tabindex="-1"></a>duplicated_cg <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">id =</span> duplicated_cg,</span>
<span id="cb910-3"><a href="#cb910-3" aria-hidden="true" tabindex="-1"></a>                           <span class="at">subject =</span> <span class="fu">gsub</span>(<span class="st">&quot;^(</span><span class="sc">\\</span><span class="st">d{4,5}).*$&quot;</span>,<span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>,duplicated_cg),</span>
<span id="cb910-4"><a href="#cb910-4" aria-hidden="true" tabindex="-1"></a>                           <span class="at">date =</span> <span class="fu">ymd</span>(<span class="fu">gsub</span>(<span class="st">&quot;^.*file(</span><span class="sc">\\</span><span class="st">d{4}-</span><span class="sc">\\</span><span class="st">d{2}-</span><span class="sc">\\</span><span class="st">d{2})$&quot;</span>,<span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>,duplicated_cg)),</span>
<span id="cb910-5"><a href="#cb910-5" aria-hidden="true" tabindex="-1"></a>                           <span class="at">dup_subject =</span> <span class="fu">rep</span>(<span class="cn">FALSE</span>, <span class="at">times =</span> <span class="fu">length</span>(duplicated_cg)),</span>
<span id="cb910-6"><a href="#cb910-6" aria-hidden="true" tabindex="-1"></a>                           <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb910-7"><a href="#cb910-7" aria-hidden="true" tabindex="-1"></a>dup_subjects <span class="ot">=</span> <span class="fu">unique</span>(duplicated_cg[<span class="fu">which</span>(<span class="fu">duplicated</span>(duplicated_cg<span class="sc">$</span>subject)),<span class="st">&quot;subject&quot;</span>])</span>
<span id="cb910-8"><a href="#cb910-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb910-9"><a href="#cb910-9" aria-hidden="true" tabindex="-1"></a>exclude_cg <span class="ot">=</span> <span class="fu">c</span>()</span>
<span id="cb910-10"><a href="#cb910-10" aria-hidden="true" tabindex="-1"></a>unresolved_cg <span class="ot">=</span> <span class="fu">c</span>()</span>
<span id="cb910-11"><a href="#cb910-11" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> dup_subjects) {</span>
<span id="cb910-12"><a href="#cb910-12" aria-hidden="true" tabindex="-1"></a>  dup_subjects_rows_i <span class="ot">=</span> <span class="fu">which</span>(duplicated_cg<span class="sc">$</span>subject <span class="sc">==</span> i)</span>
<span id="cb910-13"><a href="#cb910-13" aria-hidden="true" tabindex="-1"></a>  dup_dates_i <span class="ot">=</span> duplicated_cg[dup_subjects_rows_i,<span class="st">&quot;date&quot;</span>]</span>
<span id="cb910-14"><a href="#cb910-14" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="fu">length</span>(<span class="fu">which</span>(<span class="fu">duplicated</span>(dup_dates_i))) <span class="sc">!=</span> <span class="dv">0</span>) {</span>
<span id="cb910-15"><a href="#cb910-15" aria-hidden="true" tabindex="-1"></a>    unresolved_cg <span class="ot">=</span> <span class="fu">c</span>(unresolved_cg, duplicated_cg[dup_subjects_rows_i,<span class="st">&quot;id&quot;</span>])</span>
<span id="cb910-16"><a href="#cb910-16" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb910-17"><a href="#cb910-17" aria-hidden="true" tabindex="-1"></a>    exclude_cg <span class="ot">=</span> <span class="fu">c</span>(exclude_cg, duplicated_cg[dup_subjects_rows_i[<span class="fu">which</span>(dup_dates_i <span class="sc">!=</span> <span class="fu">max</span>(dup_dates_i))],<span class="st">&quot;id&quot;</span>])</span>
<span id="cb910-18"><a href="#cb910-18" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb910-19"><a href="#cb910-19" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb910-20"><a href="#cb910-20" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;IDs &quot;</span>, <span class="fu">paste</span>(unresolved_cg, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>, <span class="at">collapse =</span> <span class="st">&quot;, &quot;</span>), <span class="st">&quot; cannot be resolved by date.</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>IDs  cannot be resolved by date.</code></pre>
<p>Excluding what appears to be the second file from the file that cannot be resolved. Rerunning duplication analysis.</p>
<div class="sourceCode" id="cb912"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb912-1"><a href="#cb912-1" aria-hidden="true" tabindex="-1"></a><span class="co"># exclude_cg = c(exclude_cg, &quot;8166_200840140018_R01C01_file2016-12-27&quot;)</span></span>
<span id="cb912-2"><a href="#cb912-2" aria-hidden="true" tabindex="-1"></a>DO_str_2016_cross <span class="ot">=</span> DO_str_2016_cross[<span class="fu">paste0</span>(<span class="st">&quot;-&quot;</span>,exclude_cg),]</span>
<span id="cb912-3"><a href="#cb912-3" aria-hidden="true" tabindex="-1"></a>cg <span class="ot">=</span> <span class="fu">compare_geno</span>(DO_str_2016_cross, <span class="at">cores =</span> <span class="dv">0</span>)</span>
<span id="cb912-4"><a href="#cb912-4" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(cg)</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["ind1"],"name":[1],"type":["chr"],"align":["left"]},{"label":["ind2"],"name":[2],"type":["chr"],"align":["left"]},{"label":["prop_match"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["n_mismatch"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["n_typed"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["n_match"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["index1"],"name":[7],"type":["int"],"align":["right"]},{"label":["index2"],"name":[8],"type":["int"],"align":["right"]}],"data":[{"1":"8673_file2017-09-04","2":"8669_file2017-09-04","3":"0.9995872","4":"46","5":"111435","6":"111389","7":"352","8":"359","_rn_":"8673_file2017-09-04"},{"1":"8616_file2017-09-04","2":"8597_file2017-09-04","3":"0.9992341","4":"85","5":"110987","6":"110902","7":"353","8":"360","_rn_":"8616_file2017-09-04"},{"1":"8618_file2017-09-04","2":"8614_file2017-09-04","3":"0.9992247","4":"86","5":"110927","6":"110841","7":"349","8":"357","_rn_":"8618_file2017-09-04"},{"1":"8674_file2017-09-04","2":"8670_file2017-09-04","3":"0.9977470","4":"246","5":"109186","6":"108940","7":"350","8":"358","_rn_":"8674_file2017-09-04"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>The rest of these files have different subject IDs. After spot-checking GBRS data on a couple of these, it appears to be an error on the genotyping. Excluding these data as it is impossible to reconcile them with the behavioral data associated with these animals.</p>
<div class="sourceCode" id="cb913"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb913-1"><a href="#cb913-1" aria-hidden="true" tabindex="-1"></a>exclude_cg <span class="ot">=</span> <span class="fu">unique</span>(<span class="fu">c</span>(<span class="fu">summary</span>(cg)<span class="sc">$</span>ind1, <span class="fu">summary</span>(cg)<span class="sc">$</span>ind2))</span>
<span id="cb913-2"><a href="#cb913-2" aria-hidden="true" tabindex="-1"></a>DO_str_2016_cross <span class="ot">=</span> DO_str_2016_cross[<span class="fu">paste0</span>(<span class="st">&quot;-&quot;</span>,exclude_cg),]</span></code></pre></div>
<p>Finally, looking at duplications with high percent missing numbers that may not have been caught in the deduplication analysis above.</p>
<div class="sourceCode" id="cb914"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb914-1"><a href="#cb914-1" aria-hidden="true" tabindex="-1"></a>covar_duplicate_subjects <span class="ot">=</span> DO_str_2016_cross<span class="sc">$</span>covar[<span class="fu">which</span>(<span class="fu">duplicated</span>(DO_str_2016_cross<span class="sc">$</span>covar<span class="sc">$</span>subject)),<span class="st">&quot;subject&quot;</span>]</span>
<span id="cb914-2"><a href="#cb914-2" aria-hidden="true" tabindex="-1"></a>covar_duplicate_ids <span class="ot">=</span> <span class="fu">row.names</span>(DO_str_2016_cross<span class="sc">$</span>covar[<span class="fu">which</span>(DO_str_2016_cross<span class="sc">$</span>covar<span class="sc">$</span>subject <span class="sc">%in%</span> covar_duplicate_subjects),])</span>
<span id="cb914-3"><a href="#cb914-3" aria-hidden="true" tabindex="-1"></a>percent_missing[covar_duplicate_ids,]</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["id"],"name":[1],"type":["chr"],"align":["left"]},{"label":["order"],"name":[2],"type":["dbl"],"align":["right"]},{"label":["percent"],"name":[3],"type":["dbl"],"align":["right"]},{"label":["gtrthan3"],"name":[4],"type":["chr"],"align":["left"]}],"data":[{"1":"7747_file2016-09-08","2":"87","3":"18.8478564","4":"YES","_rn_":"7747_file2016-09-08"},{"1":"8042_200735410007_R09C01_file2016-09-08","2":"186","3":"1.6215881","4":"no","_rn_":"8042_200735410007_R09C01_file2016-09-08"},{"1":"8042_200735410008_R11C02_file2016-09-08","2":"202","3":"2.3605283","4":"no","_rn_":"8042_200735410008_R11C02_file2016-09-08"},{"1":"8066_file2016-09-08","2":"207","3":"2.9229391","4":"no","_rn_":"8066_file2016-09-08"},{"1":"8066_file2016-12-27","2":"226","3":"1.1505469","4":"no","_rn_":"8066_file2016-12-27"},{"1":"7747_file2016-12-27","2":"245","3":"0.7282953","4":"no","_rn_":"7747_file2016-12-27"},{"1":"8663_200840140080_R08C01_file2016-12-27","2":"348","3":"1.1301440","4":"no","_rn_":"8663_200840140080_R08C01_file2016-12-27"},{"1":"8663_200840140005_R10C02_file2016-12-27","2":"363","3":"1.0086136","4":"no","_rn_":"8663_200840140005_R10C02_file2016-12-27"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Excluding those files with very different percent missing numbers and both files when the numbers aren’t very different.</p>
<div class="sourceCode" id="cb915"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb915-1"><a href="#cb915-1" aria-hidden="true" tabindex="-1"></a>exclude_id <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;7649_file2016-09-08&quot;</span>,<span class="st">&quot;7747_file2016-09-08&quot;</span>,<span class="st">&quot;8042_200735410008_R11C02_file2016-09-08&quot;</span>,</span>
<span id="cb915-2"><a href="#cb915-2" aria-hidden="true" tabindex="-1"></a>               <span class="st">&quot;8066_file2016-09-08&quot;</span>,<span class="st">&quot;8663_200840140080_R08C01_file2016-12-27&quot;</span>,<span class="st">&quot;8663_200840140005_R10C02_file2016-12-27&quot;</span>)</span>
<span id="cb915-3"><a href="#cb915-3" aria-hidden="true" tabindex="-1"></a>DO_str_2016_cross <span class="ot">=</span> DO_str_2016_cross[<span class="fu">paste0</span>(<span class="st">&quot;-&quot;</span>,exclude_id),]</span></code></pre></div>
<pre><code>Warning in subset_ind(ind, ind_g, allow_logical): Some individuals not in cross:
7649_file2016-09-08</code></pre>
<div class="sourceCode" id="cb917"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb917-1"><a href="#cb917-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="fu">ifelse</span>(<span class="fu">length</span>(<span class="fu">unique</span>(DO_str_2016_cross<span class="sc">$</span>covar<span class="sc">$</span>subject)) <span class="sc">==</span> <span class="fu">length</span>(DO_str_2016_cross<span class="sc">$</span>covar<span class="sc">$</span>subject),</span>
<span id="cb917-2"><a href="#cb917-2" aria-hidden="true" tabindex="-1"></a>           <span class="st">&quot;Deduplication successful.</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="st">&quot;Deduplication NOT successful.</span><span class="sc">\n</span><span class="st">&quot;</span>), <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Deduplication successful.</code></pre>
<p>There are a total of 362 data points that will be included.</p>
<div id="markers-with-missing-data" class="section level4">
<h4>Markers with missing data</h4>
<p>Finding markers with lots of missing data.</p>
<div class="sourceCode" id="cb919"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb919-1"><a href="#cb919-1" aria-hidden="true" tabindex="-1"></a>pmis_mar <span class="ot">&lt;-</span> <span class="fu">n_missing</span>(DO_str_2016_cross, <span class="st">&quot;marker&quot;</span>, <span class="st">&quot;proportion&quot;</span>) <span class="sc">*</span> <span class="dv">100</span></span>
<span id="cb919-2"><a href="#cb919-2" aria-hidden="true" tabindex="-1"></a><span class="fu">par</span>(<span class="at">mar=</span><span class="fu">c</span>(<span class="fl">5.1</span>,<span class="fl">0.6</span>,<span class="fl">0.6</span>, <span class="fl">0.6</span>))</span>
<span id="cb919-3"><a href="#cb919-3" aria-hidden="true" tabindex="-1"></a><span class="fu">hist</span>(pmis_mar, <span class="at">breaks=</span><span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">100</span>, <span class="at">length=</span><span class="dv">201</span>),</span>
<span id="cb919-4"><a href="#cb919-4" aria-hidden="true" tabindex="-1"></a>     <span class="at">main=</span><span class="st">&quot;&quot;</span>, <span class="at">yaxt=</span><span class="st">&quot;n&quot;</span>, <span class="at">ylab=</span><span class="st">&quot;&quot;</span>, <span class="at">xlab=</span><span class="st">&quot;Percent missing genotypes&quot;</span>)</span>
<span id="cb919-5"><a href="#cb919-5" aria-hidden="true" tabindex="-1"></a><span class="fu">rug</span>(pmis_mar)</span></code></pre></div>
<p><img src="figure/07_DO_eQTL_genotype_prep.Rmd/unnamed-chunk-18-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-18-1">
Past versions of unnamed-chunk-18-1.png
</button>
</p>
<div id="fig-unnamed-chunk-18-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
a743c27
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-30
</td>
</tr>
<tr>
<td>
84aab57
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>As in Broman’s vignette, excluding markers with &gt; 15% missing genotypes.</p>
<div class="sourceCode" id="cb920"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb920-1"><a href="#cb920-1" aria-hidden="true" tabindex="-1"></a>exclude_missing <span class="ot">=</span> <span class="fu">names</span>(pmis_mar[<span class="fu">which</span>(pmis_mar <span class="sc">&gt;</span> <span class="dv">15</span>)])</span>
<span id="cb920-2"><a href="#cb920-2" aria-hidden="true" tabindex="-1"></a>DO_str_2016_cross <span class="ot">=</span> <span class="fu">drop_markers</span>(DO_str_2016_cross, exclude_missing)</span></code></pre></div>
</div>
</div>
<div id="getting-number-of-cores-to-use" class="section level3">
<h3>Getting number of cores to use</h3>
<div class="sourceCode" id="cb921"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb921-1"><a href="#cb921-1" aria-hidden="true" tabindex="-1"></a>n_cores <span class="ot">=</span> <span class="dv">2</span></span></code></pre></div>
</div>
<div id="calculating-genotype-and-allele-probabilities" class="section level3">
<h3>Calculating genotype and allele probabilities</h3>
<div class="sourceCode" id="cb922"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb922-1"><a href="#cb922-1" aria-hidden="true" tabindex="-1"></a>DO_str_2016_gprobs <span class="ot">=</span> <span class="fu">calc_genoprob</span>(DO_str_2016_cross, <span class="at">error_prob=</span><span class="fl">0.002</span>, <span class="at">map_function=</span><span class="st">&quot;c-f&quot;</span>, <span class="at">cores=</span>n_cores)</span>
<span id="cb922-2"><a href="#cb922-2" aria-hidden="true" tabindex="-1"></a>DO_str_2016_aprobs <span class="ot">=</span> <span class="fu">genoprob_to_alleleprob</span>(DO_str_2016_gprobs, <span class="at">cores=</span>n_cores)</span></code></pre></div>
</div>
<div id="calculating-kinship-matrix" class="section level3">
<h3>Calculating kinship matrix</h3>
<div class="sourceCode" id="cb923"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb923-1"><a href="#cb923-1" aria-hidden="true" tabindex="-1"></a>DO_str_2016_kinship <span class="ot">=</span> <span class="fu">calc_kinship</span>(DO_str_2016_aprobs, <span class="st">&quot;loco&quot;</span>, <span class="at">cores=</span>n_cores)</span></code></pre></div>
<p>Displaying kinship matrix for chromosome 1.</p>
<div class="sourceCode" id="cb924"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb924-1"><a href="#cb924-1" aria-hidden="true" tabindex="-1"></a><span class="fu">image</span>(<span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(DO_str_2016_kinship[[<span class="dv">1</span>]]), <span class="dv">1</span><span class="sc">:</span><span class="fu">ncol</span>(DO_str_2016_kinship[[<span class="dv">1</span>]]), DO_str_2016_kinship[[<span class="dv">1</span>]][,<span class="fu">ncol</span>(DO_str_2016_kinship[[<span class="dv">1</span>]])<span class="sc">:</span><span class="dv">1</span>], <span class="at">xlab =</span> <span class="st">&quot;Samples&quot;</span>, </span>
<span id="cb924-2"><a href="#cb924-2" aria-hidden="true" tabindex="-1"></a>      <span class="at">ylab =</span> <span class="st">&quot;Samples&quot;</span>, <span class="at">yaxt =</span> <span class="st">&quot;n&quot;</span>, <span class="at">main =</span> <span class="st">&quot;Kinship between samples (chr1)&quot;</span>, </span>
<span id="cb924-3"><a href="#cb924-3" aria-hidden="true" tabindex="-1"></a>      <span class="at">breaks =</span> <span class="dv">0</span><span class="sc">:</span><span class="dv">100</span><span class="sc">/</span><span class="dv">100</span>, <span class="at">col =</span> <span class="fu">colorRampPalette</span>(<span class="fu">brewer.pal</span>(<span class="at">n =</span> <span class="dv">9</span>, <span class="at">name =</span> <span class="st">&quot;YlGn&quot;</span>))(<span class="fu">length</span>(<span class="dv">0</span><span class="sc">:</span><span class="dv">100</span><span class="sc">/</span><span class="dv">100</span>) <span class="sc">-</span> <span class="dv">1</span>))</span></code></pre></div>
<p><img src="figure/07_DO_eQTL_genotype_prep.Rmd/unnamed-chunk-23-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-23-1">
Past versions of unnamed-chunk-23-1.png
</button>
</p>
<div id="fig-unnamed-chunk-23-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
a743c27
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-30
</td>
</tr>
<tr>
<td>
84aab57
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="qcing-the-data-for-crossovers." class="section level3">
<h3>QCing the data for crossovers.</h3>
<div class="sourceCode" id="cb925"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb925-1"><a href="#cb925-1" aria-hidden="true" tabindex="-1"></a>m <span class="ot">&lt;-</span> <span class="fu">maxmarg</span>(DO_str_2016_gprobs, <span class="at">minprob=</span><span class="fl">0.5</span>, <span class="at">cores=</span><span class="dv">0</span>)</span>
<span id="cb925-2"><a href="#cb925-2" aria-hidden="true" tabindex="-1"></a>nxo <span class="ot">&lt;-</span> <span class="fu">count_xo</span>(m, <span class="at">cores=</span><span class="dv">0</span>)</span>
<span id="cb925-3"><a href="#cb925-3" aria-hidden="true" tabindex="-1"></a>totxo <span class="ot">&lt;-</span> <span class="fu">rowSums</span>(nxo)</span></code></pre></div>
<p>Plotting crossovers by mouse ID and generation</p>
<div class="sourceCode" id="cb926"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb926-1"><a href="#cb926-1" aria-hidden="true" tabindex="-1"></a>DO_str_2016_XOs <span class="ot">=</span> DO_str_2016_cross<span class="sc">$</span>covar</span>
<span id="cb926-2"><a href="#cb926-2" aria-hidden="true" tabindex="-1"></a>DO_str_2016_XOs<span class="sc">$</span>totxo <span class="ot">=</span> totxo[<span class="fu">row.names</span>(DO_str_2016_XOs)]</span>
<span id="cb926-3"><a href="#cb926-3" aria-hidden="true" tabindex="-1"></a>DO_str_2016_XOs<span class="sc">$</span>order_subject <span class="ot">=</span> <span class="fu">rank</span>(<span class="fu">as.numeric</span>(DO_str_2016_XOs<span class="sc">$</span>subject))</span>
<span id="cb926-4"><a href="#cb926-4" aria-hidden="true" tabindex="-1"></a>DO_str_2016_XOs<span class="sc">$</span>ngen <span class="ot">=</span> <span class="fu">factor</span>(<span class="fu">paste0</span>(<span class="st">&quot;generation &quot;</span>, DO_str_2016_XOs<span class="sc">$</span>ngen),</span>
<span id="cb926-5"><a href="#cb926-5" aria-hidden="true" tabindex="-1"></a>                              <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">&quot;generation 21&quot;</span>, <span class="st">&quot;generation 22&quot;</span>, <span class="st">&quot;generation 23&quot;</span>),</span>
<span id="cb926-6"><a href="#cb926-6" aria-hidden="true" tabindex="-1"></a>                              <span class="at">ordered =</span> <span class="cn">TRUE</span>)</span>
<span id="cb926-7"><a href="#cb926-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb926-8"><a href="#cb926-8" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(<span class="at">data =</span> DO_str_2016_XOs, <span class="fu">aes</span>(<span class="at">x =</span> order_subject, <span class="at">y =</span> totxo, <span class="at">color =</span> ngen)) <span class="sc">+</span></span>
<span id="cb926-9"><a href="#cb926-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb926-10"><a href="#cb926-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb926-11"><a href="#cb926-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">panel.grid =</span> <span class="fu">element_blank</span>()) <span class="sc">+</span></span>
<span id="cb926-12"><a href="#cb926-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Subject Order (by JMCRS ID)&quot;</span>) <span class="sc">+</span></span>
<span id="cb926-13"><a href="#cb926-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Number of Crossovers&quot;</span>) <span class="sc">+</span></span>
<span id="cb926-14"><a href="#cb926-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#3399CC&quot;</span>,<span class="st">&quot;#FFCC33&quot;</span>,<span class="st">&quot;#CC3333&quot;</span>))</span></code></pre></div>
<p><img src="figure/07_DO_eQTL_genotype_prep.Rmd/unnamed-chunk-25-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-25-1">
Past versions of unnamed-chunk-25-1.png
</button>
</p>
<div id="fig-unnamed-chunk-25-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
a743c27
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-30
</td>
</tr>
<tr>
<td>
5f89658
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-29
</td>
</tr>
<tr>
<td>
b495515
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
</tr>
<tr>
<td>
84aab57
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>One of these points has an absurdly low number for crossovers, but the rest appear to obey the effect of more crossovers per generation. Dropping the very low number of crossovers sample as this appears to be problematic.</p>
<div class="sourceCode" id="cb927"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb927-1"><a href="#cb927-1" aria-hidden="true" tabindex="-1"></a>DO_str_2016_cross <span class="ot">=</span> DO_str_2016_cross[<span class="fu">paste0</span>(<span class="st">&quot;-&quot;</span>,<span class="fu">row.names</span>(DO_str_2016_XOs[<span class="fu">which</span>(DO_str_2016_XOs<span class="sc">$</span>totxo <span class="sc">&lt;</span> <span class="dv">200</span>),])),]</span></code></pre></div>
</div>
<div id="qcing-samples-for-minor-allele-frequency-outliers" class="section level3">
<h3>QCing samples for minor allele frequency outliers</h3>
<p>Examining the minor allele frequencies.</p>
<div class="sourceCode" id="cb928"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb928-1"><a href="#cb928-1" aria-hidden="true" tabindex="-1"></a>g <span class="ot">&lt;-</span> <span class="fu">do.call</span>(<span class="st">&quot;cbind&quot;</span>, DO_str_2016_cross<span class="sc">$</span>geno[<span class="dv">1</span><span class="sc">:</span><span class="dv">19</span>])</span>
<span id="cb928-2"><a href="#cb928-2" aria-hidden="true" tabindex="-1"></a>fg <span class="ot">&lt;-</span> <span class="fu">do.call</span>(<span class="st">&quot;cbind&quot;</span>, DO_str_2016_cross<span class="sc">$</span>founder_geno[<span class="dv">1</span><span class="sc">:</span><span class="dv">19</span>])</span>
<span id="cb928-3"><a href="#cb928-3" aria-hidden="true" tabindex="-1"></a>g <span class="ot">&lt;-</span> g[,<span class="fu">colSums</span>(fg<span class="sc">==</span><span class="dv">0</span>)<span class="sc">==</span><span class="dv">0</span>]</span>
<span id="cb928-4"><a href="#cb928-4" aria-hidden="true" tabindex="-1"></a>fg <span class="ot">&lt;-</span> fg[,<span class="fu">colSums</span>(fg<span class="sc">==</span><span class="dv">0</span>)<span class="sc">==</span><span class="dv">0</span>]</span>
<span id="cb928-5"><a href="#cb928-5" aria-hidden="true" tabindex="-1"></a>fgn <span class="ot">&lt;-</span> <span class="fu">colSums</span>(fg<span class="sc">==</span><span class="dv">3</span>)</span>
<span id="cb928-6"><a href="#cb928-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb928-7"><a href="#cb928-7" aria-hidden="true" tabindex="-1"></a>gf_ind <span class="ot">&lt;-</span> <span class="fu">vector</span>(<span class="st">&quot;list&quot;</span>, <span class="dv">4</span>)</span>
<span id="cb928-8"><a href="#cb928-8" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span>(i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>) {</span>
<span id="cb928-9"><a href="#cb928-9" aria-hidden="true" tabindex="-1"></a>  gf_ind[[i]] <span class="ot">&lt;-</span> <span class="fu">t</span>(<span class="fu">apply</span>(g[,fgn<span class="sc">==</span>i], <span class="dv">1</span>, <span class="cf">function</span>(a) <span class="fu">table</span>(<span class="fu">factor</span>(a, <span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>))<span class="sc">/</span><span class="fu">sum</span>(a <span class="sc">!=</span> <span class="dv">0</span>)))</span>
<span id="cb928-10"><a href="#cb928-10" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb928-11"><a href="#cb928-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb928-12"><a href="#cb928-12" aria-hidden="true" tabindex="-1"></a><span class="fu">par</span>(<span class="at">mfrow=</span><span class="fu">c</span>(<span class="dv">2</span>,<span class="dv">2</span>), <span class="at">mar=</span><span class="fu">c</span>(<span class="fl">0.6</span>, <span class="fl">0.6</span>, <span class="fl">2.6</span>, <span class="fl">0.6</span>))</span>
<span id="cb928-13"><a href="#cb928-13" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span>(i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>) {</span>
<span id="cb928-14"><a href="#cb928-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">triplot</span>(<span class="fu">c</span>(<span class="st">&quot;AA&quot;</span>, <span class="st">&quot;AB&quot;</span>, <span class="st">&quot;BB&quot;</span>), <span class="at">main=</span><span class="fu">paste0</span>(<span class="st">&quot;MAF = &quot;</span>, i, <span class="st">&quot;/8&quot;</span>))</span>
<span id="cb928-15"><a href="#cb928-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tripoints</span>(gf_ind[[i]], <span class="at">pch=</span><span class="dv">21</span>, <span class="at">bg=</span><span class="st">&quot;lightblue&quot;</span>)</span>
<span id="cb928-16"><a href="#cb928-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tripoints</span>(<span class="fu">c</span>((<span class="dv">1</span><span class="sc">-</span>i<span class="sc">/</span><span class="dv">8</span>)<span class="sc">^</span><span class="dv">2</span>, <span class="dv">2</span><span class="sc">*</span>i<span class="sc">/</span><span class="dv">8</span><span class="sc">*</span>(<span class="dv">1</span><span class="sc">-</span>i<span class="sc">/</span><span class="dv">8</span>), (i<span class="sc">/</span><span class="dv">8</span>)<span class="sc">^</span><span class="dv">2</span>), <span class="at">pch=</span><span class="dv">21</span>, <span class="at">bg=</span><span class="st">&quot;violetred&quot;</span>)</span>
<span id="cb928-17"><a href="#cb928-17" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb928-18"><a href="#cb928-18" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span>(i<span class="sc">&gt;=</span><span class="dv">3</span>) { <span class="co"># label mouse with lowest het</span></span>
<span id="cb928-19"><a href="#cb928-19" aria-hidden="true" tabindex="-1"></a>    wh <span class="ot">&lt;-</span> <span class="fu">which</span>(gf_ind[[i]][,<span class="dv">2</span>] <span class="sc">==</span> <span class="fu">min</span>(gf_ind[[i]][,<span class="dv">2</span>]))</span>
<span id="cb928-20"><a href="#cb928-20" aria-hidden="true" tabindex="-1"></a>    <span class="fu">tritext</span>(gf_ind[[i]][wh,,<span class="at">drop=</span><span class="cn">FALSE</span>] <span class="sc">+</span> <span class="fu">c</span>(<span class="fl">0.02</span>, <span class="sc">-</span><span class="fl">0.02</span>, <span class="dv">0</span>),</span>
<span id="cb928-21"><a href="#cb928-21" aria-hidden="true" tabindex="-1"></a>            <span class="fu">names</span>(wh), <span class="at">adj=</span><span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">1</span>))</span>
<span id="cb928-22"><a href="#cb928-22" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb928-23"><a href="#cb928-23" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb928-24"><a href="#cb928-24" aria-hidden="true" tabindex="-1"></a>  <span class="co"># label other mice</span></span>
<span id="cb928-25"><a href="#cb928-25" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span>(i<span class="sc">==</span><span class="dv">1</span>) {</span>
<span id="cb928-26"><a href="#cb928-26" aria-hidden="true" tabindex="-1"></a>    lab <span class="ot">&lt;-</span> <span class="fu">rownames</span>(gf_ind[[i]])[gf_ind[[i]][,<span class="dv">2</span>]<span class="sc">&gt;</span><span class="fl">0.3</span>]</span>
<span id="cb928-27"><a href="#cb928-27" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb928-28"><a href="#cb928-28" aria-hidden="true" tabindex="-1"></a>  <span class="cf">else</span> <span class="cf">if</span>(i<span class="sc">==</span><span class="dv">2</span>) {</span>
<span id="cb928-29"><a href="#cb928-29" aria-hidden="true" tabindex="-1"></a>    lab <span class="ot">&lt;-</span> <span class="fu">rownames</span>(gf_ind[[i]])[gf_ind[[i]][,<span class="dv">2</span>]<span class="sc">&gt;</span><span class="fl">0.48</span>]</span>
<span id="cb928-30"><a href="#cb928-30" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb928-31"><a href="#cb928-31" aria-hidden="true" tabindex="-1"></a>  <span class="cf">else</span> <span class="cf">if</span>(i<span class="sc">==</span><span class="dv">3</span>) {</span>
<span id="cb928-32"><a href="#cb928-32" aria-hidden="true" tabindex="-1"></a>    lab <span class="ot">&lt;-</span> <span class="fu">rownames</span>(gf_ind[[i]])[gf_ind[[i]][,<span class="dv">2</span>]<span class="sc">&gt;</span><span class="fl">0.51</span>]</span>
<span id="cb928-33"><a href="#cb928-33" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb928-34"><a href="#cb928-34" aria-hidden="true" tabindex="-1"></a>  <span class="cf">else</span> <span class="cf">if</span>(i<span class="sc">==</span><span class="dv">4</span>) {</span>
<span id="cb928-35"><a href="#cb928-35" aria-hidden="true" tabindex="-1"></a>    lab <span class="ot">&lt;-</span> <span class="fu">rownames</span>(gf_ind[[i]])[gf_ind[[i]][,<span class="dv">2</span>]<span class="sc">&gt;</span><span class="fl">0.6</span>]</span>
<span id="cb928-36"><a href="#cb928-36" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb928-37"><a href="#cb928-37" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb928-38"><a href="#cb928-38" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span>(ind <span class="cf">in</span> lab) {</span>
<span id="cb928-39"><a href="#cb928-39" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span>(<span class="fu">grepl</span>(<span class="st">&quot;^F&quot;</span>, ind) <span class="sc">&amp;&amp;</span> i <span class="sc">!=</span> <span class="dv">3</span>) {</span>
<span id="cb928-40"><a href="#cb928-40" aria-hidden="true" tabindex="-1"></a>      <span class="fu">tritext</span>(gf_ind[[i]][ind,,<span class="at">drop=</span><span class="cn">FALSE</span>] <span class="sc">+</span> <span class="fu">c</span>(<span class="sc">-</span><span class="fl">0.01</span>, <span class="dv">0</span>, <span class="sc">+</span><span class="fl">0.01</span>), ind, <span class="at">adj=</span><span class="fu">c</span>(<span class="dv">1</span>,<span class="fl">0.5</span>))</span>
<span id="cb928-41"><a href="#cb928-41" aria-hidden="true" tabindex="-1"></a>    } <span class="cf">else</span> {</span>
<span id="cb928-42"><a href="#cb928-42" aria-hidden="true" tabindex="-1"></a>      <span class="fu">tritext</span>(gf_ind[[i]][ind,,<span class="at">drop=</span><span class="cn">FALSE</span>] <span class="sc">+</span> <span class="fu">c</span>(<span class="fl">0.01</span>, <span class="dv">0</span>, <span class="sc">-</span><span class="fl">0.01</span>), ind, <span class="at">adj=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="fl">0.5</span>))</span>
<span id="cb928-43"><a href="#cb928-43" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb928-44"><a href="#cb928-44" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb928-45"><a href="#cb928-45" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<p><img src="figure/07_DO_eQTL_genotype_prep.Rmd/unnamed-chunk-27-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-27-1">
Past versions of unnamed-chunk-27-1.png
</button>
</p>
<div id="fig-unnamed-chunk-27-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
a743c27
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-30
</td>
</tr>
<tr>
<td>
84aab57
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>For no sample do the MAFs appear to have systematic problems. There are a couple of samples that appear to be outliers in a couple of the MAF conditions, but these look appropriate for DO mice. Not dropping any samples based upon this criterion.</p>
</div>
<div id="examining-genotyping-errors" class="section level3">
<h3>Examining genotyping errors</h3>
<p>First, using error LOD scores.</p>
<div class="sourceCode" id="cb929"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb929-1"><a href="#cb929-1" aria-hidden="true" tabindex="-1"></a>e <span class="ot">&lt;-</span> <span class="fu">calc_errorlod</span>(DO_str_2016_cross, DO_str_2016_gprobs, <span class="at">cores=</span><span class="dv">0</span>)</span></code></pre></div>
<pre><code>Warning in selectChildren(ac[!fin], -1): error &#39;No child processes&#39; in select

Warning in selectChildren(ac[!fin], -1): error &#39;No child processes&#39; in select

Warning in selectChildren(ac[!fin], -1): error &#39;No child processes&#39; in select</code></pre>
<div class="sourceCode" id="cb931"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb931-1"><a href="#cb931-1" aria-hidden="true" tabindex="-1"></a>e <span class="ot">&lt;-</span> <span class="fu">do.call</span>(<span class="st">&quot;cbind&quot;</span>, e)</span>
<span id="cb931-2"><a href="#cb931-2" aria-hidden="true" tabindex="-1"></a>errors_ind <span class="ot">&lt;-</span> <span class="fu">rowSums</span>(e<span class="sc">&gt;</span><span class="dv">2</span>)<span class="sc">/</span><span class="fu">n_typed</span>(DO_str_2016_cross)<span class="sc">*</span><span class="dv">100</span></span>
<span id="cb931-3"><a href="#cb931-3" aria-hidden="true" tabindex="-1"></a>lab <span class="ot">&lt;-</span> <span class="fu">paste0</span>(<span class="fu">gsub</span>(<span class="st">&quot;^(</span><span class="sc">\\</span><span class="st">d{4,5})_.*$&quot;</span>,<span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>,<span class="fu">names</span>(errors_ind)), <span class="st">&quot; (&quot;</span>, <span class="fu">myround</span>(percent_missing[<span class="fu">names</span>(errors_ind),<span class="st">&quot;percent&quot;</span>],<span class="dv">1</span>), <span class="st">&quot;% missing)&quot;</span>)</span>
<span id="cb931-4"><a href="#cb931-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb931-5"><a href="#cb931-5" aria-hidden="true" tabindex="-1"></a>error_df <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">index =</span> <span class="fu">seq_along</span>(errors_ind),</span>
<span id="cb931-6"><a href="#cb931-6" aria-hidden="true" tabindex="-1"></a>                      <span class="at">label =</span> lab,</span>
<span id="cb931-7"><a href="#cb931-7" aria-hidden="true" tabindex="-1"></a>                      <span class="at">error =</span> errors_ind,</span>
<span id="cb931-8"><a href="#cb931-8" aria-hidden="true" tabindex="-1"></a>                      <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb931-9"><a href="#cb931-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb931-10"><a href="#cb931-10" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(<span class="at">data =</span> error_df, <span class="fu">aes</span>(<span class="at">x =</span> index, <span class="at">y =</span> error, <span class="at">label =</span> label)) <span class="sc">+</span></span>
<span id="cb931-11"><a href="#cb931-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb931-12"><a href="#cb931-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_label</span>()</span></code></pre></div>
<p><img src="figure/07_DO_eQTL_genotype_prep.Rmd/unnamed-chunk-28-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-28-1">
Past versions of unnamed-chunk-28-1.png
</button>
</p>
<div id="fig-unnamed-chunk-28-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
a743c27
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-30
</td>
</tr>
<tr>
<td>
84aab57
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>There are 1 samples with error LODs &gt; 2 in at least 1% of samples. This does not look very bad. We needn’t necessarily remove samples based upon this criterion.</p>
<div class="sourceCode" id="cb932"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb932-1"><a href="#cb932-1" aria-hidden="true" tabindex="-1"></a>snpg <span class="ot">&lt;-</span> <span class="fu">predict_snpgeno</span>(DO_str_2016_cross, m, <span class="at">cores=</span><span class="dv">0</span>)</span>
<span id="cb932-2"><a href="#cb932-2" aria-hidden="true" tabindex="-1"></a>snpg <span class="ot">&lt;-</span> <span class="fu">do.call</span>(<span class="st">&quot;cbind&quot;</span>, snpg)</span>
<span id="cb932-3"><a href="#cb932-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb932-4"><a href="#cb932-4" aria-hidden="true" tabindex="-1"></a>gobs <span class="ot">&lt;-</span> <span class="fu">do.call</span>(<span class="st">&quot;cbind&quot;</span>, DO_str_2016_cross<span class="sc">$</span>geno)</span>
<span id="cb932-5"><a href="#cb932-5" aria-hidden="true" tabindex="-1"></a>gobs[gobs<span class="sc">==</span><span class="dv">0</span>] <span class="ot">&lt;-</span> <span class="cn">NA</span></span>
<span id="cb932-6"><a href="#cb932-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb932-7"><a href="#cb932-7" aria-hidden="true" tabindex="-1"></a><span class="fu">par</span>(<span class="at">pty=</span><span class="st">&quot;s&quot;</span>)</span>
<span id="cb932-8"><a href="#cb932-8" aria-hidden="true" tabindex="-1"></a>err_direct <span class="ot">&lt;-</span> <span class="fu">rowMeans</span>(snpg <span class="sc">!=</span> gobs, <span class="at">na.rm=</span><span class="cn">TRUE</span>)<span class="sc">*</span><span class="dv">100</span></span>
<span id="cb932-9"><a href="#cb932-9" aria-hidden="true" tabindex="-1"></a>errors_ind_0 <span class="ot">&lt;-</span> <span class="fu">rowSums</span>(e <span class="sc">&gt;</span> <span class="dv">0</span>)<span class="sc">/</span><span class="fu">n_typed</span>(DO_str_2016_cross)<span class="sc">*</span><span class="dv">100</span></span>
<span id="cb932-10"><a href="#cb932-10" aria-hidden="true" tabindex="-1"></a><span class="fu">par</span>(<span class="at">mar=</span><span class="fu">c</span>(<span class="fl">4.1</span>,<span class="fl">4.1</span>,<span class="fl">0.6</span>, <span class="fl">0.6</span>), <span class="at">mfrow =</span> <span class="fu">c</span>(<span class="dv">2</span>,<span class="dv">1</span>))</span>
<span id="cb932-11"><a href="#cb932-11" aria-hidden="true" tabindex="-1"></a><span class="fu">grayplot</span>(errors_ind_0, err_direct,</span>
<span id="cb932-12"><a href="#cb932-12" aria-hidden="true" tabindex="-1"></a>         <span class="at">xlab=</span><span class="st">&quot;Percent errors (error LOD &gt; 0)&quot;</span>,</span>
<span id="cb932-13"><a href="#cb932-13" aria-hidden="true" tabindex="-1"></a>         <span class="at">ylab=</span><span class="st">&quot;Percent errors (obs vs predicted)&quot;</span>,</span>
<span id="cb932-14"><a href="#cb932-14" aria-hidden="true" tabindex="-1"></a>         <span class="at">xlim=</span><span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">21.2</span>), <span class="at">ylim=</span><span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">21.2</span>))</span>
<span id="cb932-15"><a href="#cb932-15" aria-hidden="true" tabindex="-1"></a><span class="fu">abline</span>(<span class="dv">0</span>,<span class="dv">1</span>,<span class="at">lty=</span><span class="dv">2</span>, <span class="at">col=</span><span class="st">&quot;gray60&quot;</span>)</span>
<span id="cb932-16"><a href="#cb932-16" aria-hidden="true" tabindex="-1"></a><span class="fu">grayplot</span>(errors_ind_0, err_direct,</span>
<span id="cb932-17"><a href="#cb932-17" aria-hidden="true" tabindex="-1"></a>         <span class="at">xlab=</span><span class="st">&quot;Percent errors (error LOD &gt; 0)&quot;</span>,</span>
<span id="cb932-18"><a href="#cb932-18" aria-hidden="true" tabindex="-1"></a>         <span class="at">ylab=</span><span class="st">&quot;Percent errors (obs vs predicted)&quot;</span>,</span>
<span id="cb932-19"><a href="#cb932-19" aria-hidden="true" tabindex="-1"></a>         <span class="at">xlim=</span><span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">3</span>), <span class="at">ylim=</span><span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">3</span>))</span>
<span id="cb932-20"><a href="#cb932-20" aria-hidden="true" tabindex="-1"></a><span class="fu">abline</span>(<span class="dv">0</span>,<span class="dv">1</span>,<span class="at">lty=</span><span class="dv">2</span>, <span class="at">col=</span><span class="st">&quot;gray60&quot;</span>)</span></code></pre></div>
<p><img src="figure/07_DO_eQTL_genotype_prep.Rmd/unnamed-chunk-29-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-29-1">
Past versions of unnamed-chunk-29-1.png
</button>
</p>
<div id="fig-unnamed-chunk-29-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
a743c27
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-30
</td>
</tr>
<tr>
<td>
84aab57
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The highest of proportion of errors doesn’t appear to go much over 1.5%. These samples can likely be maintained, but the markers with high amounts of errors might need to be censored.</p>
</div>
<div id="marker-qc-maf" class="section level3">
<h3>Marker QC: MAF</h3>
<p>QCing markers by comparing their realized genotype frequencies to those expected under the theoretical MAFs.</p>
<div class="sourceCode" id="cb933"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb933-1"><a href="#cb933-1" aria-hidden="true" tabindex="-1"></a>gf_mar <span class="ot">&lt;-</span> <span class="fu">t</span>(<span class="fu">apply</span>(g, <span class="dv">2</span>, <span class="cf">function</span>(a) <span class="fu">table</span>(<span class="fu">factor</span>(a, <span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>))<span class="sc">/</span><span class="fu">sum</span>(a <span class="sc">!=</span> <span class="dv">0</span>)))</span>
<span id="cb933-2"><a href="#cb933-2" aria-hidden="true" tabindex="-1"></a>gn_mar <span class="ot">&lt;-</span> <span class="fu">t</span>(<span class="fu">apply</span>(g, <span class="dv">2</span>, <span class="cf">function</span>(a) <span class="fu">table</span>(<span class="fu">factor</span>(a, <span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>))))</span>
<span id="cb933-3"><a href="#cb933-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb933-4"><a href="#cb933-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb933-5"><a href="#cb933-5" aria-hidden="true" tabindex="-1"></a><span class="fu">par</span>(<span class="at">mfrow=</span><span class="fu">c</span>(<span class="dv">2</span>,<span class="dv">2</span>), <span class="at">mar=</span><span class="fu">c</span>(<span class="fl">0.6</span>, <span class="fl">0.6</span>, <span class="fl">2.6</span>, <span class="fl">0.6</span>))</span>
<span id="cb933-6"><a href="#cb933-6" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span>(i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>) {</span>
<span id="cb933-7"><a href="#cb933-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">triplot</span>(<span class="fu">c</span>(<span class="st">&quot;AA&quot;</span>, <span class="st">&quot;AB&quot;</span>, <span class="st">&quot;BB&quot;</span>), <span class="at">main=</span><span class="fu">paste0</span>(<span class="st">&quot;MAF = &quot;</span>, i, <span class="st">&quot;/8&quot;</span>))</span>
<span id="cb933-8"><a href="#cb933-8" aria-hidden="true" tabindex="-1"></a>  z <span class="ot">&lt;-</span> gf_mar[fgn<span class="sc">==</span>i,]</span>
<span id="cb933-9"><a href="#cb933-9" aria-hidden="true" tabindex="-1"></a>  z <span class="ot">&lt;-</span> z[<span class="fu">rowSums</span>(<span class="fu">is.na</span>(z)) <span class="sc">&lt;</span> <span class="dv">3</span>,]</span>
<span id="cb933-10"><a href="#cb933-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tripoints</span>(z, <span class="at">pch=</span><span class="dv">21</span>, <span class="at">bg=</span><span class="st">&quot;gray80&quot;</span>, <span class="at">cex=</span><span class="fl">0.6</span>)</span>
<span id="cb933-11"><a href="#cb933-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tripoints</span>(<span class="fu">c</span>((<span class="dv">1</span><span class="sc">-</span>i<span class="sc">/</span><span class="dv">8</span>)<span class="sc">^</span><span class="dv">2</span>, <span class="dv">2</span><span class="sc">*</span>i<span class="sc">/</span><span class="dv">8</span><span class="sc">*</span>(<span class="dv">1</span><span class="sc">-</span>i<span class="sc">/</span><span class="dv">8</span>), (i<span class="sc">/</span><span class="dv">8</span>)<span class="sc">^</span><span class="dv">2</span>), <span class="at">pch=</span><span class="dv">21</span>, <span class="at">bg=</span><span class="st">&quot;violetred&quot;</span>)</span>
<span id="cb933-12"><a href="#cb933-12" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<p><img src="figure/07_DO_eQTL_genotype_prep.Rmd/unnamed-chunk-30-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-30-1">
Past versions of unnamed-chunk-30-1.png
</button>
</p>
<div id="fig-unnamed-chunk-30-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
a743c27
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-30
</td>
</tr>
<tr>
<td>
84aab57
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>On the <span class="math inline">\(MAF = 1/8\)</span> plot, no evidence of the WSB R2D2, which was flushed by this generation of DO. There are a number of genotypes that appear unusual. However, these plots appear better behaved than the ones in Broman’s vignette.</p>
<p>Using the genotype LOD scores to identify markers needing exclusion.</p>
<div class="sourceCode" id="cb934"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb934-1"><a href="#cb934-1" aria-hidden="true" tabindex="-1"></a>errors_mar <span class="ot">&lt;-</span> <span class="fu">colSums</span>(e<span class="sc">&gt;</span><span class="dv">2</span>)<span class="sc">/</span><span class="fu">n_typed</span>(DO_str_2016_cross, <span class="st">&quot;marker&quot;</span>)<span class="sc">*</span><span class="dv">100</span></span>
<span id="cb934-2"><a href="#cb934-2" aria-hidden="true" tabindex="-1"></a>errors_mar_df <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">marker =</span> <span class="fu">names</span>(errors_mar),</span>
<span id="cb934-3"><a href="#cb934-3" aria-hidden="true" tabindex="-1"></a>                           <span class="at">gtr_than_5 =</span> <span class="fu">ifelse</span>(errors_mar <span class="sc">&gt;</span> <span class="dv">5</span>, <span class="st">&quot;YES&quot;</span>, <span class="st">&quot;no&quot;</span>),</span>
<span id="cb934-4"><a href="#cb934-4" aria-hidden="true" tabindex="-1"></a>                           <span class="at">errors =</span> errors_mar,</span>
<span id="cb934-5"><a href="#cb934-5" aria-hidden="true" tabindex="-1"></a>                           <span class="at">missing =</span> pmis_mar[<span class="fu">names</span>(errors_mar)],</span>
<span id="cb934-6"><a href="#cb934-6" aria-hidden="true" tabindex="-1"></a>                           <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb934-7"><a href="#cb934-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb934-8"><a href="#cb934-8" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(<span class="at">data =</span> errors_mar_df, <span class="fu">aes</span>(<span class="at">x =</span> missing, <span class="at">y =</span> errors, <span class="at">color =</span> gtr_than_5)) <span class="sc">+</span></span>
<span id="cb934-9"><a href="#cb934-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb934-10"><a href="#cb934-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Proportion missing&quot;</span>) <span class="sc">+</span></span>
<span id="cb934-11"><a href="#cb934-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Proportion genotyping errors&quot;</span>) <span class="sc">+</span></span>
<span id="cb934-12"><a href="#cb934-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb934-13"><a href="#cb934-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">panel.grid =</span> <span class="fu">element_blank</span>())</span></code></pre></div>
<p><img src="figure/07_DO_eQTL_genotype_prep.Rmd/unnamed-chunk-31-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-31-1">
Past versions of unnamed-chunk-31-1.png
</button>
</p>
<div id="fig-unnamed-chunk-31-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
a743c27
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-30
</td>
</tr>
<tr>
<td>
84aab57
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>There are many markers with more than 5% errant genotypes. For these 728 markers (0.6540528) percent of the total markers), will exclude them.</p>
<div class="sourceCode" id="cb935"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb935-1"><a href="#cb935-1" aria-hidden="true" tabindex="-1"></a>DO_str_2016_cross <span class="ot">=</span> <span class="fu">drop_markers</span>(DO_str_2016_cross, errors_mar_df[<span class="fu">which</span>(errors_mar_df<span class="sc">$</span>gtr_than_5 <span class="sc">==</span> <span class="st">&quot;YES&quot;</span>),<span class="st">&quot;marker&quot;</span>])</span></code></pre></div>
</div>
<div id="recalculating-genotype-and-allele-probabilities-and-kinship" class="section level3">
<h3>Recalculating genotype and allele probabilities and kinship</h3>
<p>Now that markers have been excluded, recalculating genotype and allele probabilities and kinship.</p>
<div class="sourceCode" id="cb936"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb936-1"><a href="#cb936-1" aria-hidden="true" tabindex="-1"></a>DO_str_2016_gprobs <span class="ot">=</span> <span class="fu">calc_genoprob</span>(DO_str_2016_cross, <span class="at">error_prob=</span><span class="fl">0.002</span>, <span class="at">map_function=</span><span class="st">&quot;c-f&quot;</span>, <span class="at">cores=</span>n_cores)</span>
<span id="cb936-2"><a href="#cb936-2" aria-hidden="true" tabindex="-1"></a>DO_str_2016_aprobs <span class="ot">=</span> <span class="fu">genoprob_to_alleleprob</span>(DO_str_2016_gprobs, <span class="at">cores=</span>n_cores)</span>
<span id="cb936-3"><a href="#cb936-3" aria-hidden="true" tabindex="-1"></a>DO_str_2016_kinship <span class="ot">=</span> <span class="fu">calc_kinship</span>(DO_str_2016_aprobs, <span class="st">&quot;loco&quot;</span>, <span class="at">cores=</span>n_cores)</span></code></pre></div>
<p>Displaying kinship matrix for chromosome 1.</p>
<div class="sourceCode" id="cb937"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb937-1"><a href="#cb937-1" aria-hidden="true" tabindex="-1"></a><span class="fu">image</span>(<span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(DO_str_2016_kinship[[<span class="dv">1</span>]]), <span class="dv">1</span><span class="sc">:</span><span class="fu">ncol</span>(DO_str_2016_kinship[[<span class="dv">1</span>]]), DO_str_2016_kinship[[<span class="dv">1</span>]][,<span class="fu">ncol</span>(DO_str_2016_kinship[[<span class="dv">1</span>]])<span class="sc">:</span><span class="dv">1</span>], <span class="at">xlab =</span> <span class="st">&quot;Samples&quot;</span>, </span>
<span id="cb937-2"><a href="#cb937-2" aria-hidden="true" tabindex="-1"></a>      <span class="at">ylab =</span> <span class="st">&quot;Samples&quot;</span>, <span class="at">yaxt =</span> <span class="st">&quot;n&quot;</span>, <span class="at">main =</span> <span class="st">&quot;Kinship between samples (chr1)&quot;</span>, </span>
<span id="cb937-3"><a href="#cb937-3" aria-hidden="true" tabindex="-1"></a>      <span class="at">breaks =</span> <span class="dv">0</span><span class="sc">:</span><span class="dv">100</span><span class="sc">/</span><span class="dv">100</span>, <span class="at">col =</span> <span class="fu">colorRampPalette</span>(<span class="fu">brewer.pal</span>(<span class="at">n =</span> <span class="dv">9</span>, <span class="at">name =</span> <span class="st">&quot;YlGn&quot;</span>))(<span class="fu">length</span>(<span class="dv">0</span><span class="sc">:</span><span class="dv">100</span><span class="sc">/</span><span class="dv">100</span>) <span class="sc">-</span> <span class="dv">1</span>))</span></code></pre></div>
<p><img src="figure/07_DO_eQTL_genotype_prep.Rmd/unnamed-chunk-34-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-34-1">
Past versions of unnamed-chunk-34-1.png
</button>
</p>
<div id="fig-unnamed-chunk-34-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
a743c27
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-30
</td>
</tr>
<tr>
<td>
84aab57
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="saving-all-data-as-rdata-file" class="section level3">
<h3>Saving all data as RData file</h3>
<div class="sourceCode" id="cb938"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb938-1"><a href="#cb938-1" aria-hidden="true" tabindex="-1"></a><span class="fu">save</span>(<span class="at">list =</span> <span class="fu">c</span>(<span class="st">&quot;DO_str_2016_cross&quot;</span>,</span>
<span id="cb938-2"><a href="#cb938-2" aria-hidden="true" tabindex="-1"></a>              <span class="st">&quot;DO_str_2016_aprobs&quot;</span>,</span>
<span id="cb938-3"><a href="#cb938-3" aria-hidden="true" tabindex="-1"></a>              <span class="st">&quot;DO_str_2016_kinship&quot;</span>,</span>
<span id="cb938-4"><a href="#cb938-4" aria-hidden="true" tabindex="-1"></a>              <span class="st">&quot;eQTL_maRt&quot;</span>), <span class="at">file =</span> <span class="st">&quot;./data/DO_str_2016_eQTL.RData&quot;</span>,</span>
<span id="cb938-5"><a href="#cb938-5" aria-hidden="true" tabindex="-1"></a>     <span class="at">compress =</span> <span class="st">&quot;gzip&quot;</span>)</span></code></pre></div>
<p>The md5 checksums for the output files are:</p>
<table>
<thead>
<tr class="header">
<th>file</th>
<th>md5 checksum</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><code>DO_str_2016_eQTL.RData</code></td>
<td><code>cc1eb40b78683efe86145c7cb880c910</code></td>
</tr>
</tbody>
</table>
</div>
<div id="document-control" class="section level3">
<h3>Document Control</h3>
<p>This document was prepared using RMarkdown in RStudio. It was converted to workflowr on 2020-05-04.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb939"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb939-1"><a href="#cb939-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.3 LTS

Matrix products: default
BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] tools     stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] broman_0.76        lubridate_1.8.0    fst_0.9.4          qtl2convert_0.26  
 [5] qtl2_0.28          MASS_7.3-54        biomaRt_2.50.1     data.table_1.14.2 
 [9] RColorBrewer_1.1-2 gplots_3.1.1       ggbeeswarm_0.6.0   ggplot2_3.3.5     
[13] cowplot_1.1.1      edgeR_3.36.0       limma_3.50.0      

loaded via a namespace (and not attached):
 [1] bitops_1.0-7           fs_1.5.1               bit64_4.0.5           
 [4] filelock_1.0.2         progress_1.2.2         httr_1.4.2            
 [7] rprojroot_2.0.2        GenomeInfoDb_1.30.0    utf8_1.2.2            
[10] R6_2.5.1               KernSmooth_2.23-20     vipor_0.4.5           
[13] DBI_1.1.1              BiocGenerics_0.40.0    colorspace_2.0-2      
[16] withr_2.4.3            tidyselect_1.1.1       prettyunits_1.1.1     
[19] bit_4.0.4              curl_4.3.2             compiler_4.1.2        
[22] git2r_0.29.0           Biobase_2.54.0         xml2_1.3.3            
[25] labeling_0.4.2         caTools_1.18.2         scales_1.1.1          
[28] rappdirs_0.3.3         stringr_1.4.0          digest_0.6.29         
[31] rmarkdown_2.11         XVector_0.34.0         pkgconfig_2.0.3       
[34] htmltools_0.5.2        highr_0.9              dbplyr_2.1.1          
[37] fastmap_1.1.0          rlang_0.4.12           RSQLite_2.2.9         
[40] farver_2.1.0           jquerylib_0.1.4        generics_0.1.1        
[43] jsonlite_1.7.2         gtools_3.9.2           dplyr_1.0.7           
[46] RCurl_1.98-1.5         magrittr_2.0.1         qtl_1.50              
[49] GenomeInfoDbData_1.2.7 Rcpp_1.0.7             munsell_0.5.0         
[52] S4Vectors_0.32.3       fansi_0.5.0            lifecycle_1.0.1       
[55] stringi_1.7.6          whisker_0.4            yaml_2.2.1            
[58] zlibbioc_1.40.0        BiocFileCache_2.2.0    grid_4.1.2            
[61] blob_1.2.2             parallel_4.1.2         promises_1.2.0.1      
[64] crayon_1.4.2           lattice_0.20-45        Biostrings_2.62.0     
[67] hms_1.1.1              KEGGREST_1.34.0        locfit_1.5-9.4        
[70] knitr_1.36             pillar_1.6.4           stats4_4.1.2          
[73] XML_3.99-0.8           glue_1.5.1             evaluate_0.14         
[76] png_0.1-7              vctrs_0.3.8            httpuv_1.6.3          
[79] gtable_0.3.0           purrr_0.3.4            assertthat_0.2.1      
[82] cachem_1.0.6           xfun_0.28              later_1.3.0           
[85] tibble_3.1.6           AnnotationDbi_1.56.2   beeswarm_0.4.0        
[88] memoise_2.0.1          IRanges_2.28.0         workflowr_1.6.2       
[91] ellipsis_0.3.2        </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
