<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Michael C. Saul (michael.saul [at] jax.org)" />


<title>DO IVSA Summarization</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>


<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Diversity Outbred Reference Traits Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">DO IVSA Summarization</h1>
<h4 class="author">Michael C. Saul (michael.saul [at] jax.org)</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-08
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rta/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20200501code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20200501)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20200501code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20200501)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong535b06b"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 535b06b </a>
</p>
</div>
<div id="strongRepositoryversionstrong535b06b" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 535b06b. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/.Rapp.history
    Ignored:    data/.Rhistory
    Ignored:    data/RDS/
    Ignored:    run/

Untracked files:
    Untracked:  LICENSE
    Untracked:  Untitled.R
    Untracked:  analysis/17_mapping_interpretation_old.Rmd
    Untracked:  analysis/20_predict_acquisition_from_novelty.Rmd
    Untracked:  analysis/21_DO_elasticnet_correlation.Rmd
    Untracked:  analysis/temp.txt
    Untracked:  code/DO_eQTL_perms.R
    Untracked:  code/DO_eQTL_score_calc.R
    Untracked:  code/STAR_RSEM_416.sh
    Untracked:  code/eQTL_plotting_DO_striatum_416.R
    Untracked:  code/ggplot_pub_themes.R
    Untracked:  data/20_elastic_mv_results.csv
    Untracked:  data/20_elasticnet_predictions.RDS
    Untracked:  data/DO_416_DGEList.RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 3).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 4).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_DGEList_unfiltered.RDS
    Untracked:  data/DO_416_ERCC_DGEList.RDS
    Untracked:  data/DO_416_ERCC_cpm.RDS
    Untracked:  data/DO_416_normrt.RDS
    Untracked:  data/DO_416_pheno.RDS
    Untracked:  data/DO_416_pheno_allgenes.RDS
    Untracked:  data/DO_416_voom (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom.RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom_allgenes.RDS
    Untracked:  data/DO_416_voom_cpresiduals.RDS
    Untracked:  data/DO_AQ_active_CoV_toptable_annotated.RDS
    Untracked:  data/DO_AQ_lever_ratio_toptable_annotated.RDS
    Untracked:  data/DO_AQ_mean_inf_toptable_annotated.RDS
    Untracked:  data/DO_AQ_rate_toptable_annotated.RDS
    Untracked:  data/DO_IVSA_acquisition_cancor_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_glm.RDS
    Untracked:  data/DO_IVSA_all_cancor_df.RDS
    Untracked:  data/DO_IVSA_data.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_ALL.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_AQ.RDS
    Untracked:  data/DO_IVSA_novelty_data.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_old.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_a1_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_a2_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a4_male_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a5_female_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_raw_data.RDS
    Untracked:  data/DO_IVSA_transformed_data.RDS
    Untracked:  data/DO_LIMS_data.RDS
    Untracked:  data/DO_RTG_a1_new.qtl.RData
    Untracked:  data/DO_RTG_a1_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a1_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a2_new.qtl.RData
    Untracked:  data/DO_RTG_a2_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a2_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a3_new.qtl.RData
    Untracked:  data/DO_RTG_a3_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_01.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_02.RData
    Untracked:  data/DO_RTG_a4_new.qtl.RData
    Untracked:  data/DO_RTG_a5_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_project_df_for_mapping.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_2020-11-06.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_a2.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_sexspecific.RDS
    Untracked:  data/DO_demographic_data_df.RDS
    Untracked:  data/DO_eigengene_significance.RDS
    Untracked:  data/DO_eigengenes.RDS
    Untracked:  data/DO_highcor_cpresiduals_datatable.RDS
    Untracked:  data/DO_holeboard_data_df.RDS
    Untracked:  data/DO_holeboard_summarized.RDS
    Untracked:  data/DO_lightdark_data_df.RDS
    Untracked:  data/DO_mQTL_peaks.RDS
    Untracked:  data/DO_mQTL_scan1s.RDS
    Untracked:  data/DO_module_membership.RDS
    Untracked:  data/DO_novelplace_data_df.RDS
    Untracked:  data/DO_novelty_data.RDS
    Untracked:  data/DO_novelty_response_data.RDS
    Untracked:  data/DO_novelty_response_data_for_RTG.RDS
    Untracked:  data/DO_novelty_response_data_with_transform.csv
    Untracked:  data/DO_openfield_data_df.RDS
    Untracked:  data/DO_paraclique_data.el
    Untracked:  data/DO_paraclique_eigengenes.RDS
    Untracked:  data/DO_paracliques.RDS
    Untracked:  data/DO_paracliques_significance.RDS
    Untracked:  data/DO_ploidy_calls.RDS
    Untracked:  data/DO_pmap.RDS
    Untracked:  data/DO_pvalue_histograms.RData
    Untracked:  data/DO_rtg_a1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a3_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m3_toptable_annotated.RDS
    Untracked:  data/DO_str_2016_eQTL.RData
    Untracked:  data/DO_str_2016_gm4qtl2.zip
    Untracked:  data/DO_str_2016_gm4qtl2_intensities.fst
    Untracked:  data/DO_voom_cpresiduals_NArm.RDS
    Untracked:  data/DO_wgcna.RDS
    Untracked:  data/IVSA_RTG_a3_blup.pdf
    Untracked:  data/IVSA_h2_2019-11-18.tsv
    Untracked:  data/IVSA_metadata_sheet.RDS
    Untracked:  data/ME16_edges.tsv
    Untracked:  data/Mus_musculus.GRCm38.94.parsed.RDS
    Untracked:  data/Paraclique_19_MF_AMIGO.txt
    Untracked:  data/RTG_QTL_peaks.RDS
    Untracked:  data/RTG_figure.RDS
    Untracked:  data/SENS_RTG/
    Untracked:  data/Tyr2_genotypes.RDS
    Untracked:  data/Tyr_genotypes.RDS
    Untracked:  data/blup_chr7_DO_RTG_a3.RDS
    Untracked:  data/cc_variants/
    Untracked:  data/chr18_QTL_results.RDS
    Untracked:  data/chr4_QTL_results.RDS
    Untracked:  data/cis/
    Untracked:  data/eQTL_annotated_peaks.RDS
    Untracked:  data/eQTL_plots.RData
    Untracked:  data/figures/
    Untracked:  data/founders_key/
    Untracked:  data/gigamuga/
    Untracked:  data/gwas/
    Untracked:  data/info/
    Untracked:  data/m2G.a2.RTG.chr4snp.RData
    Untracked:  data/mapping_from_hao/
    Untracked:  data/perms/
    Untracked:  data/qtl2/
    Untracked:  data/raw/
    Untracked:  data/resources/
    Untracked:  data/tads_loops/
    Untracked:  data/transcripts/
    Untracked:  data/variants/
    Untracked:  logs/
    Untracked:  new_RTG_a3.jpeg
    Untracked:  output/RTG_out/
    Untracked:  output/docs_backup_2020-07-30.tar.gz
    Untracked:  output/elastic_net_out/
    Untracked:  singularity/
    Untracked:  tmp/
    Untracked:  var/

Unstaged changes:
    Deleted:    .Rprofile
    Modified:   .gitignore
    Modified:   README.md
    Modified:   analysis/17_mapping_interpretation.Rmd
    Modified:   analysis/_site.yml

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/10_DO_IVSA_summarize.Rmd</code>) and HTML (<code>docs/10_DO_IVSA_summarize.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
eba8a35
</td>
<td>
Michael C. Saul
</td>
<td>
2021-01-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
e8800f8
</td>
<td>
Michael C. Saul
</td>
<td>
2021-01-04
</td>
<td>
Performing batch-transformation of these data in this script instead of canonical correlation script
</td>
</tr>
<tr>
<td>
html
</td>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
<td>
Rolling back to working version.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
c49a753
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
4ff98f5
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Adding in data dictionary
</td>
</tr>
<tr>
<td>
html
</td>
<td>
916d622
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
91a3b9e
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Minor tweaks.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
ada5f4e
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
0965aff
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
workflowr::wflow_publish("./analysis/10_DO_IVSA_summarize.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
c970c21
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
6623879
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Initial commit of IVSA preprocessing
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="background" class="section level1">
<h1>Background</h1>
<div id="motivation" class="section level2">
<h2>Motivation</h2>
<div id="purpose-of-analysis" class="section level3">
<h3>Purpose of analysis</h3>
<p>This script is used to summarize the intravenous self-administration data upstream of using them for reference trait analysis.</p>
</div>
<div id="data-dictionary-of-ivsa-variables" class="section level3">
<h3>Data Dictionary of IVSA Variables</h3>
<p>These variables will been alterred from the original output file. The original file has inconsistent column header IDs that are difficult to query programatically and reference using regular expressions. The correspondence to the current variable names and the original names is in the IVSA metadata sheet created in this notebook. Abbreviations are as follows:</p>
<ul>
<li>Prefices
<ul>
<li>Prefix <code>AQ_</code> is acquisition</li>
<li>Prefix <code>DR_</code> is dose-response</li>
<li>Prefix <code>EX_</code> is extinction</li>
<li>Prefix <code>RI_</code> is reinstatement</li>
<li>Prefix <code>WT_</code> is weight</li>
</ul></li>
<li>Phenotypes
<ul>
<li><code>SessionsToAcquisition</code> is the number of sessions it took for the animal to acquire IVSA</li>
<li><code>ALP</code> is active lever presses</li>
<li><code>ILP</code> is inactive lever presses</li>
<li><code>Inf</code> is infusions</li>
<li><code>IEI</code> is inter-event interval</li>
<li><code>pAP</code> is percentage of active presses</li>
<li><code>TCI</code> is total cocaine intake</li>
<li><code>TfACDiE</code> is total for all cocaine doses in experiment</li>
<li><code>Tf7Es</code> is total for all seven extinction sessions</li>
<li><code>RIn</code> is reinstatement</li>
<li><code>StS</code> is sessions to stabilization</li>
<li><code>AUC</code> is area under the curve</li>
</ul></li>
<li>General terms
<ul>
<li><code>mgkg</code> is milligrams per kilogram, mg/kg, and is displayed in NpNNN with the p standing for the decimal point (e.g., 0p032mgkg is 0.032 mg/kg)</li>
<li><code>X2</code> is second exposure, which happens on the 1.0 mg/kg dose to stabilize the mouse.</li>
<li><code>dTO</code> is during time out</li>
<li><code>Mean</code> is mean</li>
<li><code>Total</code> is total</li>
<li><code>Variance</code> is variance</li>
<li><code>CoV</code> is coefficient of variation, <span class="math inline">\({\sigma}\over{\mu}\)</span></li>
<li><code>TfASoTD</code> is total for all sessions on this dose</li>
<li><code>Mean2s</code> is mean of two sessions</li>
<li><code>Sal</code> is saline</li>
</ul></li>
</ul>
</div>
</div>
</div>
<div id="analysis" class="section level1">
<h1>Analysis</h1>
<div id="setup" class="section level2">
<h2>Setup</h2>
<div id="r-libraries" class="section level3">
<h3>R libraries</h3>
<p>Calling R libraries necessary for this analysis.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;MASS&quot;</span>)</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;readxl&quot;</span>)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;tidyr&quot;</span>)</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;lubridate&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;lubridate&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    date, intersect, setdiff, union</code></pre>
</div>
<div id="functions" class="section level3">
<h3>Functions</h3>
<p>Getting a normal rank transformation function that can handle <code>NA</code> values from the following formula:</p>
<p><span class="math inline">\(Y^t_i={\phi}^{-1}({{r_i-c}\over{N-2c+1}})\)</span></p>
<p>where <span class="math inline">\(r_i\)</span> is the rank, <span class="math inline">\(N\)</span> is the number of ranked samples, <span class="math inline">\(\phi^{-1}\)</span> is the inverse normal function (equivalent to <code>qnorm()</code> in R). A recommended value of <span class="math inline">\(c=3/8\)</span> in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2921808/">Beasley and Erickson, 2010</a>, but we use the <a href="https://scholar.google.com/scholar_lookup?journal=Proc+Koninklijke+Nederlandse+Akademie+van+Wetenschappen&amp;title=Order+tests+for+the+two-sample+problem+and+their+power&amp;author=BL+van+der+Waerden&amp;volume=55&amp;publication_year=1952&amp;pages=453-458&amp;">van der Waerden (1952)</a> value of <span class="math inline">\(c=0\)</span> per Vivek Philip. Additionally, after discussing this with Vivek, I’ve changed the rank tie method to <code>"average"</code> whereas I had <code>"random"</code> previously.</p>
<p>Our work appears to meet Beasley and Erickson’s sample size recommendations.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>norm_rank_transform <span class="ot">=</span> <span class="cf">function</span>(x, <span class="at">c =</span> <span class="dv">0</span>) {</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stopifnot</span>(<span class="fu">is.numeric</span>(x) <span class="sc">&amp;</span> <span class="fu">is.vector</span>(x))</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>  x_noNA <span class="ot">=</span> <span class="fu">which</span>(<span class="sc">!</span><span class="fu">is.na</span>(x))</span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>  N <span class="ot">=</span> <span class="fu">length</span>(x_noNA)</span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>  x[x_noNA] <span class="ot">=</span> <span class="fu">qnorm</span>((<span class="fu">rank</span>(x[x_noNA], <span class="at">ties.method =</span> <span class="st">&quot;average&quot;</span>) <span class="sc">-</span> c) <span class="sc">/</span> (N <span class="sc">-</span> (<span class="dv">2</span> <span class="sc">*</span> c) <span class="sc">+</span> <span class="dv">1</span>))</span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(x)</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
</div>
<div id="importing-data" class="section level3">
<h3>Importing data</h3>
<div id="getting-data-locations" class="section level4">
<h4>Getting data locations</h4>
<p><em>Note: have added underscores to file names.</em></p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>IVSA_file_path <span class="ot">=</span> <span class="st">&quot;./data/raw/IVSA_export_to_3-22-20_preprocessed_multivariate.xlsx&quot;</span></span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>LIMS_file_path <span class="ot">=</span> <span class="st">&quot;./data/raw/LIMS_export_2020-04-08.csv&quot;</span></span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>PED_JMCRS_LIMS_file_path <span class="ot">=</span> <span class="st">&quot;./data/raw/Matching_PED_and_JCMS_numbers.xlsx&quot;</span></span></code></pre></div>
</div>
<div id="importing-lims-data" class="section level4">
<h4>Importing LIMS Data</h4>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Extracting LIMS information</span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>LIMS_data <span class="ot">=</span> <span class="fu">read.table</span>(<span class="at">file =</span> LIMS_file_path, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">&quot;,&quot;</span>, <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>, <span class="at">quote =</span> <span class="st">&quot;</span><span class="sc">\&quot;</span><span class="st">&quot;</span>)</span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Deduplicting LIMS data</span></span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a><span class="co"># There are duplicate 24h circadian rhythm data</span></span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a><span class="co"># The IDs to keep are those from the study ID &quot;24H Circadian&quot;</span></span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Dropping the ones from the duplicates with the study ID &quot;Circadian R&quot;</span></span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a>duplicate_IDs <span class="ot">=</span> <span class="fu">unique</span>(LIMS_data[<span class="fu">which</span>(<span class="fu">duplicated</span>(LIMS_data<span class="sc">$</span>Organism.ID)),<span class="st">&quot;Organism.ID&quot;</span>])</span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a>LIMS_data_deduplicate <span class="ot">=</span> <span class="fu">which</span>(LIMS_data<span class="sc">$</span>Organism.ID <span class="sc">%in%</span> duplicate_IDs <span class="sc">&amp;</span> LIMS_data<span class="sc">$</span>Study <span class="sc">==</span> <span class="st">&quot;Circadian R&quot;</span>)</span>
<span id="cb6-10"><a href="#cb6-10" aria-hidden="true" tabindex="-1"></a>LIMS_data <span class="ot">=</span> LIMS_data[(<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> LIMS_data_deduplicate),]</span>
<span id="cb6-11"><a href="#cb6-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-12"><a href="#cb6-12" aria-hidden="true" tabindex="-1"></a><span class="co"># Making row IDs as the organism IDs.</span></span>
<span id="cb6-13"><a href="#cb6-13" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(LIMS_data) <span class="ot">=</span> <span class="fu">as.character</span>(LIMS_data<span class="sc">$</span>Organism.ID)</span></code></pre></div>
<p>Reformating dates to individual columns for each year, month, and day of each relevant date.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Coercing dates to lubridate format</span></span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>Birth.Date <span class="ot">=</span> <span class="fu">mdy</span>(LIMS_data<span class="sc">$</span>Birth.Date, <span class="at">tz =</span> <span class="st">&quot;America/New_York&quot;</span>)</span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>Arrival.Date <span class="ot">=</span> <span class="fu">mdy</span>(LIMS_data<span class="sc">$</span>Arrival.Date, <span class="at">tz =</span> <span class="st">&quot;America/New_York&quot;</span>)</span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>Exit.Date <span class="ot">=</span> <span class="fu">mdy</span>(LIMS_data<span class="sc">$</span>Exit.Date, <span class="at">tz =</span> <span class="st">&quot;America/New_York&quot;</span>)</span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a><span class="co"># Parsing dates to separate columns for year, month, and day</span></span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>birth_date_year <span class="ot">=</span> <span class="fu">year</span>(LIMS_data<span class="sc">$</span>Birth.Date)</span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>birth_date_month <span class="ot">=</span> <span class="fu">month</span>(LIMS_data<span class="sc">$</span>Birth.Date)</span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>birth_date_day <span class="ot">=</span> <span class="fu">day</span>(LIMS_data<span class="sc">$</span>Birth.Date)</span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>arrival_date_year <span class="ot">=</span> <span class="fu">year</span>(LIMS_data<span class="sc">$</span>Arrival.Date)</span>
<span id="cb7-11"><a href="#cb7-11" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>arrival_date_month <span class="ot">=</span> <span class="fu">month</span>(LIMS_data<span class="sc">$</span>Arrival.Date)</span>
<span id="cb7-12"><a href="#cb7-12" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>arrival_date_day <span class="ot">=</span> <span class="fu">day</span>(LIMS_data<span class="sc">$</span>Arrival.Date)</span>
<span id="cb7-13"><a href="#cb7-13" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>exit_date_year <span class="ot">=</span> <span class="fu">year</span>(LIMS_data<span class="sc">$</span>Exit.Date)</span>
<span id="cb7-14"><a href="#cb7-14" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>exit_date_month <span class="ot">=</span> <span class="fu">month</span>(LIMS_data<span class="sc">$</span>Exit.Date)</span>
<span id="cb7-15"><a href="#cb7-15" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>exit_date_day <span class="ot">=</span> <span class="fu">day</span>(LIMS_data<span class="sc">$</span>Exit.Date)</span>
<span id="cb7-16"><a href="#cb7-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-17"><a href="#cb7-17" aria-hidden="true" tabindex="-1"></a><span class="co"># Removing LIMS data birth, arrival, and exit date columns</span></span>
<span id="cb7-18"><a href="#cb7-18" aria-hidden="true" tabindex="-1"></a>LIMS_data <span class="ot">=</span> LIMS_data[,(<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(<span class="fu">colnames</span>(LIMS_data) <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">&quot;Birth.Date&quot;</span>,<span class="st">&quot;Arrival.Date&quot;</span>,<span class="st">&quot;Exit.Date&quot;</span>)))]</span></code></pre></div>
<p>Removing the periods from the column names and replacing them with underscores.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(LIMS_data) <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;</span><span class="sc">\\</span><span class="st">.&quot;</span>,<span class="st">&quot;_&quot;</span>,<span class="fu">colnames</span>(LIMS_data))</span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(LIMS_data) <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;_s_&quot;</span>,<span class="st">&quot;&quot;</span>,<span class="fu">colnames</span>(LIMS_data))</span></code></pre></div>
<p>Per communication with Jason Bubier, DO animals with mouse IDs of &lt; 10000 are pre-CSNA mice (<code>precenter</code>) while &gt; 10000 are CSNA mice (<code>CSNA</code>). There were large intralaboratory changes in personnel, equipment, and procedures to scale up the IVSA pipeline when the CSNA started, making this specific factor a primary batch variable that appears to account for both differences in the means and variances of all measures. Consequently, this cannot just be residualized out.</p>
<p>Per discussions with Elissa Chesler, the procedure we arrived at for normalizing these data between arms of the project is to rank-normal transform the residualized data for <code>precenter</code> and <code>CSNA</code> separately.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>DO_Center_Status <span class="ot">=</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(LIMS_data))</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a>LIMS_data[<span class="fu">which</span>(<span class="fu">as.numeric</span>(LIMS_data<span class="sc">$</span>Organism_ID) <span class="sc">&lt;</span> <span class="dv">10000</span> <span class="sc">&amp;</span> LIMS_data<span class="sc">$</span>Line_Name <span class="sc">==</span> <span class="st">&quot;J:DO&quot;</span>),<span class="st">&quot;DO_Center_Status&quot;</span>] <span class="ot">=</span> <span class="st">&quot;precenter&quot;</span></span></code></pre></div>
<pre><code>Warning in which(as.numeric(LIMS_data$Organism_ID) &lt; 10000 &amp; LIMS_data$Line_Name
== : NAs introduced by coercion</code></pre>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a>LIMS_data[<span class="fu">which</span>(<span class="fu">as.numeric</span>(LIMS_data<span class="sc">$</span>Organism_ID) <span class="sc">&gt;</span> <span class="dv">10000</span> <span class="sc">&amp;</span> LIMS_data<span class="sc">$</span>Line_Name <span class="sc">==</span> <span class="st">&quot;J:DO&quot;</span>),<span class="st">&quot;DO_Center_Status&quot;</span>] <span class="ot">=</span> <span class="st">&quot;CSNA&quot;</span></span></code></pre></div>
<pre><code>Warning in which(as.numeric(LIMS_data$Organism_ID) &gt; 10000 &amp; LIMS_data$Line_Name
== : NAs introduced by coercion</code></pre>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>Sex <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^Female$&quot;</span>,<span class="st">&quot;F&quot;</span>,LIMS_data<span class="sc">$</span>Sex)</span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a>LIMS_data<span class="sc">$</span>Sex <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^Male$&quot;</span>,<span class="st">&quot;M&quot;</span>,LIMS_data<span class="sc">$</span>Sex)</span></code></pre></div>
</div>
<div id="importing-intravenous-self-administration-data" class="section level4">
<h4>Importing intravenous self-administration data</h4>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>IVSA_raw <span class="ot">=</span> <span class="fu">as.data.frame</span>(<span class="fu">read_excel</span>(IVSA_file_path), </span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>                         <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a>IVSA_Descriptive_vars <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;Subject&quot;</span>,<span class="st">&quot;Experiment&quot;</span>,<span class="st">&quot;Active_Lever&quot;</span>,</span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a>                          <span class="st">&quot;Control_Unit&quot;</span>,<span class="st">&quot;Group&quot;</span>,<span class="st">&quot;Box&quot;</span>)</span>
<span id="cb14-5"><a href="#cb14-5" aria-hidden="true" tabindex="-1"></a>IVSA_Weight_vars <span class="ot">=</span> <span class="fu">colnames</span>(IVSA_raw)[<span class="fu">grep</span>(<span class="st">&quot;^Weight&quot;</span>, <span class="fu">colnames</span>(IVSA_raw))]</span>
<span id="cb14-6"><a href="#cb14-6" aria-hidden="true" tabindex="-1"></a>IVSA_Acquisition_vars <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;SessionsToAcquisition&quot;</span>,</span>
<span id="cb14-7"><a href="#cb14-7" aria-hidden="true" tabindex="-1"></a>                          <span class="fu">colnames</span>(IVSA_raw)[<span class="fu">grep</span>(<span class="st">&quot;^Acquisition at &quot;</span>, <span class="fu">colnames</span>(IVSA_raw))],</span>
<span id="cb14-8"><a href="#cb14-8" aria-hidden="true" tabindex="-1"></a>                          <span class="st">&quot;ActiveLeverPressesDuringTimeOut_Block01 Acquisition at 1.0 mg/kg-Session 01&quot;</span>,</span>
<span id="cb14-9"><a href="#cb14-9" aria-hidden="true" tabindex="-1"></a>                          <span class="st">&quot;ActiveLeverPressesDuringTimeOut_Block01 Acquisition at 1.0 mg/kg-Session 02&quot;</span>)</span>
<span id="cb14-10"><a href="#cb14-10" aria-hidden="true" tabindex="-1"></a>IVSA_DoseResponse_vars <span class="ot">=</span> <span class="fu">colnames</span>(IVSA_raw)[<span class="fu">grep</span>(<span class="st">&quot;^Dose [Rr]esponse FR1&quot;</span>, <span class="fu">colnames</span>(IVSA_raw))]</span>
<span id="cb14-11"><a href="#cb14-11" aria-hidden="true" tabindex="-1"></a>IVSA_Extinction_vars <span class="ot">=</span> <span class="fu">colnames</span>(IVSA_raw)[<span class="fu">grep</span>(<span class="st">&quot;^Extinction&quot;</span>, <span class="fu">colnames</span>(IVSA_raw))]</span>
<span id="cb14-12"><a href="#cb14-12" aria-hidden="true" tabindex="-1"></a>IVSA_Reinstatement_vars <span class="ot">=</span> <span class="fu">colnames</span>(IVSA_raw)[<span class="fu">grep</span>(<span class="st">&quot;^Reinstatement&quot;</span>, <span class="fu">colnames</span>(IVSA_raw))]</span>
<span id="cb14-13"><a href="#cb14-13" aria-hidden="true" tabindex="-1"></a>IVSA_AllCols <span class="ot">=</span> <span class="fu">c</span>(IVSA_Descriptive_vars,</span>
<span id="cb14-14"><a href="#cb14-14" aria-hidden="true" tabindex="-1"></a>                 IVSA_Weight_vars,</span>
<span id="cb14-15"><a href="#cb14-15" aria-hidden="true" tabindex="-1"></a>                 IVSA_Acquisition_vars,</span>
<span id="cb14-16"><a href="#cb14-16" aria-hidden="true" tabindex="-1"></a>                 IVSA_DoseResponse_vars,</span>
<span id="cb14-17"><a href="#cb14-17" aria-hidden="true" tabindex="-1"></a>                 IVSA_Extinction_vars,</span>
<span id="cb14-18"><a href="#cb14-18" aria-hidden="true" tabindex="-1"></a>                 IVSA_Reinstatement_vars)</span>
<span id="cb14-19"><a href="#cb14-19" aria-hidden="true" tabindex="-1"></a><span class="fu">ifelse</span>(<span class="fu">length</span>(<span class="fu">colnames</span>(IVSA_raw)[<span class="fu">which</span>(<span class="sc">!</span>(<span class="fu">colnames</span>(IVSA_raw) <span class="sc">%in%</span> IVSA_AllCols))]) <span class="sc">==</span> <span class="dv">0</span>,</span>
<span id="cb14-20"><a href="#cb14-20" aria-hidden="true" tabindex="-1"></a>       <span class="st">&quot;All columns categorized&quot;</span>, </span>
<span id="cb14-21"><a href="#cb14-21" aria-hidden="true" tabindex="-1"></a>       <span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">colnames</span>(IVSA_raw)[<span class="fu">which</span>(<span class="sc">!</span>(<span class="fu">colnames</span>(IVSA_raw) <span class="sc">%in%</span> IVSA_AllCols))]),</span>
<span id="cb14-22"><a href="#cb14-22" aria-hidden="true" tabindex="-1"></a>             <span class="st">&quot; columns not categorized.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>))</span></code></pre></div>
<pre><code>[1] &quot;All columns categorized&quot;</code></pre>
<p>Cleaning up data and coercing response variables to numeric.</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a>IVSA_raw[<span class="fu">which</span>(IVSA_raw <span class="sc">==</span> <span class="st">&quot;x&quot;</span>, <span class="at">arr.ind =</span> <span class="cn">TRUE</span>)] <span class="ot">=</span> <span class="cn">NA</span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>IVSA_Data_vars <span class="ot">=</span> <span class="fu">c</span>(IVSA_Weight_vars,</span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a>                   IVSA_Acquisition_vars,</span>
<span id="cb16-4"><a href="#cb16-4" aria-hidden="true" tabindex="-1"></a>                   IVSA_DoseResponse_vars,</span>
<span id="cb16-5"><a href="#cb16-5" aria-hidden="true" tabindex="-1"></a>                   IVSA_Extinction_vars,</span>
<span id="cb16-6"><a href="#cb16-6" aria-hidden="true" tabindex="-1"></a>                   IVSA_Reinstatement_vars)</span>
<span id="cb16-7"><a href="#cb16-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-8"><a href="#cb16-8" aria-hidden="true" tabindex="-1"></a>IVSA_raw[,IVSA_Data_vars] <span class="ot">=</span> <span class="fu">sapply</span>(IVSA_raw[,IVSA_Data_vars], as.numeric)</span>
<span id="cb16-9"><a href="#cb16-9" aria-hidden="true" tabindex="-1"></a>data_col_numeric <span class="ot">=</span> <span class="fu">c</span>()</span>
<span id="cb16-10"><a href="#cb16-10" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (var_i <span class="cf">in</span> IVSA_Data_vars) {</span>
<span id="cb16-11"><a href="#cb16-11" aria-hidden="true" tabindex="-1"></a>  data_col_numeric[var_i] <span class="ot">=</span> <span class="fu">is.numeric</span>(IVSA_raw[,var_i])</span>
<span id="cb16-12"><a href="#cb16-12" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb16-13"><a href="#cb16-13" aria-hidden="true" tabindex="-1"></a><span class="fu">ifelse</span>(<span class="fu">length</span>(<span class="fu">which</span>(<span class="sc">!</span>data_col_numeric)) <span class="sc">==</span> <span class="dv">0</span>,</span>
<span id="cb16-14"><a href="#cb16-14" aria-hidden="true" tabindex="-1"></a>       <span class="st">&quot;All IVSA data variables successfully coerced to numeric.&quot;</span>,</span>
<span id="cb16-15"><a href="#cb16-15" aria-hidden="true" tabindex="-1"></a>       <span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(<span class="sc">!</span>data_col_numeric)), <span class="st">&quot; IVSA data variables not successfully coerced to numeric.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>))</span></code></pre></div>
<pre><code>[1] &quot;All IVSA data variables successfully coerced to numeric.&quot;</code></pre>
<p>Adding in LIMS columns to the raw IVSA dataset and into the descriptive variables.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(IVSA_raw) <span class="ot">=</span> <span class="fu">as.character</span>(IVSA_raw<span class="sc">$</span>Subject)</span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>Strain <span class="ot">=</span> LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;Line_Name&quot;</span>]</span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>Stock_ID <span class="ot">=</span> LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;Stock_ID&quot;</span>]</span>
<span id="cb18-4"><a href="#cb18-4" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>Sex <span class="ot">=</span> LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;Sex&quot;</span>]</span>
<span id="cb18-5"><a href="#cb18-5" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>Breeding <span class="ot">=</span> LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;Breeding&quot;</span>]</span>
<span id="cb18-6"><a href="#cb18-6" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>Protocol <span class="ot">=</span> LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;Protocol&quot;</span>]</span>
<span id="cb18-7"><a href="#cb18-7" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>Study <span class="ot">=</span> LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;Study&quot;</span>]</span>
<span id="cb18-8"><a href="#cb18-8" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>birth_date_year <span class="ot">=</span> <span class="fu">as.integer</span>(LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;birth_date_year&quot;</span>])</span>
<span id="cb18-9"><a href="#cb18-9" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>birth_date_month <span class="ot">=</span> <span class="fu">as.integer</span>(LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;birth_date_month&quot;</span>])</span>
<span id="cb18-10"><a href="#cb18-10" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>birth_date_day <span class="ot">=</span> <span class="fu">as.integer</span>(LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;birth_date_day&quot;</span>])</span>
<span id="cb18-11"><a href="#cb18-11" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>arrival_date_year <span class="ot">=</span> <span class="fu">as.integer</span>(LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;arrival_date_year&quot;</span>])</span>
<span id="cb18-12"><a href="#cb18-12" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>arrival_date_month <span class="ot">=</span> <span class="fu">as.integer</span>(LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;arrival_date_month&quot;</span>])</span>
<span id="cb18-13"><a href="#cb18-13" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>arrival_date_day <span class="ot">=</span> <span class="fu">as.integer</span>(LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;arrival_date_day&quot;</span>])</span>
<span id="cb18-14"><a href="#cb18-14" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>exit_date_year <span class="ot">=</span> <span class="fu">as.integer</span>(LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;exit_date_year&quot;</span>])</span>
<span id="cb18-15"><a href="#cb18-15" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>exit_date_month <span class="ot">=</span> <span class="fu">as.integer</span>(LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;exit_date_month&quot;</span>])</span>
<span id="cb18-16"><a href="#cb18-16" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>exit_date_day <span class="ot">=</span> <span class="fu">as.integer</span>(LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;exit_date_day&quot;</span>])</span>
<span id="cb18-17"><a href="#cb18-17" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>Exit_Reason <span class="ot">=</span> LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;Exit_Reason&quot;</span>]</span>
<span id="cb18-18"><a href="#cb18-18" aria-hidden="true" tabindex="-1"></a>IVSA_raw<span class="sc">$</span>DO_Center_Status <span class="ot">=</span> LIMS_data[<span class="fu">row.names</span>(IVSA_raw),<span class="st">&quot;DO_Center_Status&quot;</span>]</span>
<span id="cb18-19"><a href="#cb18-19" aria-hidden="true" tabindex="-1"></a>IVSA_from_LIMS_vars <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;Strain&quot;</span>,<span class="st">&quot;Stock_ID&quot;</span>,<span class="st">&quot;Sex&quot;</span>,<span class="st">&quot;Breeding&quot;</span>,</span>
<span id="cb18-20"><a href="#cb18-20" aria-hidden="true" tabindex="-1"></a>                          <span class="st">&quot;Protocol&quot;</span>,<span class="st">&quot;Study&quot;</span>,</span>
<span id="cb18-21"><a href="#cb18-21" aria-hidden="true" tabindex="-1"></a>                          <span class="st">&quot;birth_date_year&quot;</span>,<span class="st">&quot;birth_date_month&quot;</span>,<span class="st">&quot;birth_date_day&quot;</span>,</span>
<span id="cb18-22"><a href="#cb18-22" aria-hidden="true" tabindex="-1"></a>                          <span class="st">&quot;arrival_date_year&quot;</span>,<span class="st">&quot;arrival_date_month&quot;</span>,<span class="st">&quot;arrival_date_day&quot;</span>,</span>
<span id="cb18-23"><a href="#cb18-23" aria-hidden="true" tabindex="-1"></a>                          <span class="st">&quot;exit_date_year&quot;</span>,<span class="st">&quot;exit_date_month&quot;</span>,<span class="st">&quot;exit_date_day&quot;</span>,</span>
<span id="cb18-24"><a href="#cb18-24" aria-hidden="true" tabindex="-1"></a>                          <span class="st">&quot;Exit_Reason&quot;</span>,<span class="st">&quot;DO_Center_Status&quot;</span>)</span>
<span id="cb18-25"><a href="#cb18-25" aria-hidden="true" tabindex="-1"></a>IVSA_Descriptive_vars <span class="ot">=</span> <span class="fu">c</span>(IVSA_Descriptive_vars,</span>
<span id="cb18-26"><a href="#cb18-26" aria-hidden="true" tabindex="-1"></a>                          IVSA_from_LIMS_vars)</span>
<span id="cb18-27"><a href="#cb18-27" aria-hidden="true" tabindex="-1"></a>IVSA_raw <span class="ot">=</span> IVSA_raw[,<span class="fu">c</span>(IVSA_Descriptive_vars,IVSA_Data_vars)]</span></code></pre></div>
<p>Renaming IVSA variables and creating a metadata sheet to track the names.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">variable =</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">ncol</span>(IVSA_raw)),</span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">long_varname =</span> <span class="fu">colnames</span>(IVSA_raw),</span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">vartype =</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">ncol</span>(IVSA_raw)),</span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a><span class="co"># Labeling all variables</span></span>
<span id="cb19-7"><a href="#cb19-7" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">which</span>(IVSA_metadata_sheet<span class="sc">$</span>long_varname <span class="sc">%in%</span> IVSA_Descriptive_vars),<span class="st">&quot;vartype&quot;</span>]   <span class="ot">=</span> <span class="st">&quot;Descriptive&quot;</span></span>
<span id="cb19-8"><a href="#cb19-8" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">which</span>(IVSA_metadata_sheet<span class="sc">$</span>long_varname <span class="sc">%in%</span> IVSA_Weight_vars),<span class="st">&quot;vartype&quot;</span>]        <span class="ot">=</span> <span class="st">&quot;Weight&quot;</span></span>
<span id="cb19-9"><a href="#cb19-9" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">which</span>(IVSA_metadata_sheet<span class="sc">$</span>long_varname <span class="sc">%in%</span> IVSA_Acquisition_vars),<span class="st">&quot;vartype&quot;</span>]   <span class="ot">=</span> <span class="st">&quot;Acquisition&quot;</span></span>
<span id="cb19-10"><a href="#cb19-10" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">which</span>(IVSA_metadata_sheet<span class="sc">$</span>long_varname <span class="sc">%in%</span> IVSA_DoseResponse_vars),<span class="st">&quot;vartype&quot;</span>]  <span class="ot">=</span> <span class="st">&quot;DoseResponse&quot;</span></span>
<span id="cb19-11"><a href="#cb19-11" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">which</span>(IVSA_metadata_sheet<span class="sc">$</span>long_varname <span class="sc">%in%</span> IVSA_Extinction_vars),<span class="st">&quot;vartype&quot;</span>]    <span class="ot">=</span> <span class="st">&quot;Extinction&quot;</span></span>
<span id="cb19-12"><a href="#cb19-12" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">which</span>(IVSA_metadata_sheet<span class="sc">$</span>long_varname <span class="sc">%in%</span> IVSA_Reinstatement_vars),<span class="st">&quot;vartype&quot;</span>] <span class="ot">=</span> <span class="st">&quot;Reinstatement&quot;</span></span>
<span id="cb19-13"><a href="#cb19-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-14"><a href="#cb19-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Using regular expressions to replace variable names with things that make sense</span></span>
<span id="cb19-15"><a href="#cb19-15" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;Session (</span><span class="sc">\\</span><span class="st">d{2})&quot;</span>, <span class="st">&quot;Session</span><span class="sc">\\</span><span class="st">1&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>long_varname)</span>
<span id="cb19-16"><a href="#cb19-16" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot; &quot;</span>, <span class="st">&quot;_&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-17"><a href="#cb19-17" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;</span><span class="sc">\\</span><span class="st">.&quot;</span>, <span class="st">&quot;p&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-18"><a href="#cb19-18" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;/&quot;</span>, <span class="st">&quot;per&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-19"><a href="#cb19-19" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;-&quot;</span>, <span class="st">&quot;_&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-20"><a href="#cb19-20" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;–&quot;</span>, <span class="st">&quot;_&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-21"><a href="#cb19-21" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;–&quot;</span>, <span class="st">&quot;_&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-22"><a href="#cb19-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-23"><a href="#cb19-23" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting dose</span></span>
<span id="cb19-24"><a href="#cb19-24" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>dose_mgperkg <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^.*([01])p(</span><span class="sc">\\</span><span class="st">d+)_mgperkg.*$&quot;</span>, <span class="st">&quot;</span><span class="sc">\\</span><span class="st">1.</span><span class="sc">\\</span><span class="st">2&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-25"><a href="#cb19-25" aria-hidden="true" tabindex="-1"></a>DoseResponse_rows <span class="ot">=</span> <span class="fu">which</span>(IVSA_metadata_sheet<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;DoseResponse&quot;</span>)</span>
<span id="cb19-26"><a href="#cb19-26" aria-hidden="true" tabindex="-1"></a>DoseResponse_rows_saline <span class="ot">=</span> DoseResponse_rows[<span class="fu">grep</span>(<span class="st">&quot;[Ss]aline&quot;</span>,IVSA_metadata_sheet[DoseResponse_rows,<span class="st">&quot;variable&quot;</span>])]</span>
<span id="cb19-27"><a href="#cb19-27" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[DoseResponse_rows_saline,<span class="st">&quot;dose_mgperkg&quot;</span>] <span class="ot">=</span> <span class="st">&quot;0.0&quot;</span></span>
<span id="cb19-28"><a href="#cb19-28" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">grep</span>(<span class="st">&quot;</span><span class="sc">\\</span><span class="st">.&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>dose_mgperkg, <span class="at">invert =</span> <span class="cn">TRUE</span>),<span class="st">&quot;dose_mgperkg&quot;</span>] <span class="ot">=</span> <span class="cn">NA</span></span>
<span id="cb19-29"><a href="#cb19-29" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>dose_mgperkg <span class="ot">=</span> <span class="fu">as.numeric</span>(IVSA_metadata_sheet<span class="sc">$</span>dose_mgperkg)</span>
<span id="cb19-30"><a href="#cb19-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-31"><a href="#cb19-31" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting session</span></span>
<span id="cb19-32"><a href="#cb19-32" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>session <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^.*Session(</span><span class="sc">\\</span><span class="st">d{2})$&quot;</span>, <span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-33"><a href="#cb19-33" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">grep</span>(<span class="st">&quot;^</span><span class="sc">\\</span><span class="st">d{2}$&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>session, <span class="at">invert =</span> <span class="cn">TRUE</span>),<span class="st">&quot;session&quot;</span>] <span class="ot">=</span> <span class="cn">NA</span></span>
<span id="cb19-34"><a href="#cb19-34" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>session <span class="ot">=</span> <span class="fu">as.numeric</span>(IVSA_metadata_sheet<span class="sc">$</span>session)</span>
<span id="cb19-35"><a href="#cb19-35" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;Session(</span><span class="sc">\\</span><span class="st">d{2})$&quot;</span>, <span class="st">&quot;s</span><span class="sc">\\</span><span class="st">1&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-36"><a href="#cb19-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-37"><a href="#cb19-37" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting block</span></span>
<span id="cb19-38"><a href="#cb19-38" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>block <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^.*Block(</span><span class="sc">\\</span><span class="st">d{2})_.*$&quot;</span>, <span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-39"><a href="#cb19-39" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">grep</span>(<span class="st">&quot;^</span><span class="sc">\\</span><span class="st">d{2}$&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>block, <span class="at">invert =</span> <span class="cn">TRUE</span>),<span class="st">&quot;block&quot;</span>] <span class="ot">=</span> <span class="cn">NA</span></span>
<span id="cb19-40"><a href="#cb19-40" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>block <span class="ot">=</span> <span class="fu">as.numeric</span>(IVSA_metadata_sheet<span class="sc">$</span>block)</span>
<span id="cb19-41"><a href="#cb19-41" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;Block&quot;</span>, <span class="st">&quot;b&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-42"><a href="#cb19-42" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-43"><a href="#cb19-43" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting columns with totals in them</span></span>
<span id="cb19-44"><a href="#cb19-44" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>is_total <span class="ot">=</span> <span class="fu">rep</span>(<span class="cn">FALSE</span>, <span class="at">times =</span> <span class="fu">nrow</span>(IVSA_metadata_sheet))</span>
<span id="cb19-45"><a href="#cb19-45" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">grep</span>(<span class="st">&quot;Total&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable, <span class="at">ignore.case =</span> <span class="cn">TRUE</span>),<span class="st">&quot;is_total&quot;</span>] <span class="ot">=</span> <span class="cn">TRUE</span></span>
<span id="cb19-46"><a href="#cb19-46" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-47"><a href="#cb19-47" aria-hidden="true" tabindex="-1"></a><span class="co"># Reducing variable prefices</span></span>
<span id="cb19-48"><a href="#cb19-48" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">which</span>(IVSA_metadata_sheet<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;Descriptive&quot;</span>),<span class="st">&quot;variable&quot;</span>] <span class="ot">=</span> <span class="fu">paste</span>(<span class="st">&quot;DE_&quot;</span>, IVSA_metadata_sheet[<span class="fu">which</span>(IVSA_metadata_sheet<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;Descriptive&quot;</span>),<span class="st">&quot;variable&quot;</span>], <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span>
<span id="cb19-49"><a href="#cb19-49" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="fu">which</span>(IVSA_metadata_sheet<span class="sc">$</span>variable <span class="sc">==</span> <span class="st">&quot;SessionsToAcquisition&quot;</span>), <span class="st">&quot;variable&quot;</span>] <span class="ot">=</span> <span class="st">&quot;AQ_SessionsToAcquisition&quot;</span></span>
<span id="cb19-50"><a href="#cb19-50" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^Acquisition_at_1p0_mgperkg_&quot;</span>, <span class="st">&quot;AQ_&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-51"><a href="#cb19-51" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^Dose_[rR]esponse_FR1_&quot;</span>, <span class="st">&quot;DR_&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-52"><a href="#cb19-52" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^Extinction_&quot;</span>, <span class="st">&quot;EX_&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-53"><a href="#cb19-53" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^Reinstatement_&quot;</span>, <span class="st">&quot;RI_&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-54"><a href="#cb19-54" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^Weight_&quot;</span>, <span class="st">&quot;WT_&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-55"><a href="#cb19-55" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;_mgperkg&quot;</span>, <span class="st">&quot;mgkg&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-56"><a href="#cb19-56" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;DuringTime[Oo]ut&quot;</span>, <span class="st">&quot;dTO&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-57"><a href="#cb19-57" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;CoefOfVariation&quot;</span>, <span class="st">&quot;CoV&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-58"><a href="#cb19-58" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;Mean&quot;</span>, <span class="st">&quot;Mean&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-59"><a href="#cb19-59" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;Variance&quot;</span>, <span class="st">&quot;Var&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-60"><a href="#cb19-60" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;InactiveLeverPresses&quot;</span>, <span class="st">&quot;ILP&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-61"><a href="#cb19-61" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;ActiveLeverPresses&quot;</span>, <span class="st">&quot;ALP&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-62"><a href="#cb19-62" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;Active_LeverPresses&quot;</span>, <span class="st">&quot;ALP&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-63"><a href="#cb19-63" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;Infusions&quot;</span>, <span class="st">&quot;Inf&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-64"><a href="#cb19-64" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;InterEventInterval&quot;</span>, <span class="st">&quot;IEI&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-65"><a href="#cb19-65" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;PercentageActivePresses&quot;</span>, <span class="st">&quot;pAP&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-66"><a href="#cb19-66" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;_2nd_exposure&quot;</span>, <span class="st">&quot;X2&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-67"><a href="#cb19-67" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;TotalForAllSessionsOnThisDose&quot;</span>, <span class="st">&quot;TfASoTD&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-68"><a href="#cb19-68" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;Total_Cocaine_Intake&quot;</span>, <span class="st">&quot;TCI&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-69"><a href="#cb19-69" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;TotalForAllCocaineDosesInExperiment&quot;</span>, <span class="st">&quot;TfACDiE&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-70"><a href="#cb19-70" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;TotalForAllCocaineDosesInExperiment&quot;</span>, <span class="st">&quot;TfACDiE&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-71"><a href="#cb19-71" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;mean_of_2_sessions&quot;</span>, <span class="st">&quot;Mean2s&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-72"><a href="#cb19-72" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;TotalForAll7ExtinctionSessions&quot;</span>, <span class="st">&quot;Tf7Es&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-73"><a href="#cb19-73" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;Reinstatement&quot;</span>, <span class="st">&quot;RIn&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-74"><a href="#cb19-74" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;Sessions_To_Stabilization&quot;</span>, <span class="st">&quot;StS&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-75"><a href="#cb19-75" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;[Ss]aline&quot;</span>, <span class="st">&quot;Sal&quot;</span>, IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-76"><a href="#cb19-76" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>variable <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^(ALPdTO_b</span><span class="sc">\\</span><span class="st">d{2})_Acquisition_at_1p0mgkg(_s</span><span class="sc">\\</span><span class="st">d{2})$&quot;</span>,<span class="st">&quot;AQ_</span><span class="sc">\\</span><span class="st">1</span><span class="sc">\\</span><span class="st">2&quot;</span>,IVSA_metadata_sheet<span class="sc">$</span>variable)</span>
<span id="cb19-77"><a href="#cb19-77" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-78"><a href="#cb19-78" aria-hidden="true" tabindex="-1"></a><span class="co"># Introducing the metadata names into the IVSA sheet.</span></span>
<span id="cb19-79"><a href="#cb19-79" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(IVSA_metadata_sheet) <span class="ot">=</span> IVSA_metadata_sheet<span class="sc">$</span>variable</span>
<span id="cb19-80"><a href="#cb19-80" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>varclass <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">sapply</span>(IVSA_raw, class))</span>
<span id="cb19-81"><a href="#cb19-81" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-82"><a href="#cb19-82" aria-hidden="true" tabindex="-1"></a><span class="co"># Adding in notes</span></span>
<span id="cb19-83"><a href="#cb19-83" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet<span class="sc">$</span>notes <span class="ot">=</span> <span class="fu">ifelse</span>(IVSA_metadata_sheet<span class="sc">$</span>long_varname <span class="sc">%in%</span> IVSA_from_LIMS_vars,</span>
<span id="cb19-84"><a href="#cb19-84" aria-hidden="true" tabindex="-1"></a>                                   <span class="st">&quot;Data populated from LIMS export on 2020-04-08&quot;</span>,<span class="st">&quot;&quot;</span>)</span>
<span id="cb19-85"><a href="#cb19-85" aria-hidden="true" tabindex="-1"></a>IVSA_metadata_sheet[<span class="st">&quot;DE_Subject&quot;</span>,<span class="st">&quot;notes&quot;</span>] <span class="ot">=</span> <span class="st">&quot;Data corrected to PED IDs&quot;</span></span>
<span id="cb19-86"><a href="#cb19-86" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-87"><a href="#cb19-87" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(IVSA_raw) <span class="ot">=</span> IVSA_metadata_sheet[,<span class="st">&quot;variable&quot;</span>]</span></code></pre></div>
<p>Performing rank-normal transformation on <code>J:DO</code> mice. Starting by selecting important variables. Note that the CSNA changed dose-response data collection from many doses to just 4 doses in an SOP revision after an extermal advisory board meeting. We are only using dose-response curve values that appear in both SOPs (1.0, 0.32, 0.1, and 0.032 mg/kg doses).</p>
<p>We select variables that do not contain many <code>NA</code> values relative to the first day of cued reinstatement. This minimizes data loss for the purposes of canonical correlation analysis.</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Selecting DO only.</span></span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>DO_IVSA_raw <span class="ot">=</span> IVSA_raw[<span class="fu">which</span>(IVSA_raw<span class="sc">$</span>DE_Strain <span class="sc">==</span> <span class="st">&quot;J:DO&quot;</span>),]</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-4"><a href="#cb20-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting descriptive columns</span></span>
<span id="cb20-5"><a href="#cb20-5" aria-hidden="true" tabindex="-1"></a>IVSA_DE_vars <span class="ot">=</span> IVSA_metadata_sheet[<span class="fu">which</span>(IVSA_metadata_sheet<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;Descriptive&quot;</span>),<span class="st">&quot;variable&quot;</span>]</span>
<span id="cb20-6"><a href="#cb20-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-7"><a href="#cb20-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Removing data columns broken down by sessions and blocks</span></span>
<span id="cb20-8"><a href="#cb20-8" aria-hidden="true" tabindex="-1"></a>IVSA_AQ_vars_noblocks_nosessions <span class="ot">=</span> IVSA_metadata_sheet[<span class="fu">which</span>(<span class="fu">is.na</span>(IVSA_metadata_sheet<span class="sc">$</span>session) <span class="sc">&amp;</span></span>
<span id="cb20-9"><a href="#cb20-9" aria-hidden="true" tabindex="-1"></a>                                                         <span class="fu">is.na</span>(IVSA_metadata_sheet<span class="sc">$</span>block) <span class="sc">&amp;</span></span>
<span id="cb20-10"><a href="#cb20-10" aria-hidden="true" tabindex="-1"></a>                                                         IVSA_metadata_sheet<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;Acquisition&quot;</span>),</span>
<span id="cb20-11"><a href="#cb20-11" aria-hidden="true" tabindex="-1"></a>                                                 <span class="st">&quot;variable&quot;</span>]</span>
<span id="cb20-12"><a href="#cb20-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-13"><a href="#cb20-13" aria-hidden="true" tabindex="-1"></a><span class="co"># Removing data columns not at 0.032, 0.1, 0.32, and 1 mg/kg</span></span>
<span id="cb20-14"><a href="#cb20-14" aria-hidden="true" tabindex="-1"></a>IVSA_DR_currentdoses <span class="ot">=</span> <span class="fu">c</span>(<span class="fl">0.032</span>, <span class="fl">0.1</span>, <span class="fl">0.32</span>, <span class="fl">1.0</span>)</span>
<span id="cb20-15"><a href="#cb20-15" aria-hidden="true" tabindex="-1"></a>IVSA_DR_vars_currentprotocol <span class="ot">=</span> IVSA_metadata_sheet[<span class="fu">which</span>(<span class="fu">is.na</span>(IVSA_metadata_sheet<span class="sc">$</span>session) <span class="sc">&amp;</span></span>
<span id="cb20-16"><a href="#cb20-16" aria-hidden="true" tabindex="-1"></a>                                                     <span class="fu">is.na</span>(IVSA_metadata_sheet<span class="sc">$</span>block) <span class="sc">&amp;</span></span>
<span id="cb20-17"><a href="#cb20-17" aria-hidden="true" tabindex="-1"></a>                                                     IVSA_metadata_sheet<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;DoseResponse&quot;</span> <span class="sc">&amp;</span></span>
<span id="cb20-18"><a href="#cb20-18" aria-hidden="true" tabindex="-1"></a>                                                     IVSA_metadata_sheet<span class="sc">$</span>dose_mgperkg <span class="sc">%in%</span> IVSA_DR_currentdoses),</span>
<span id="cb20-19"><a href="#cb20-19" aria-hidden="true" tabindex="-1"></a>                                             <span class="st">&quot;variable&quot;</span>]</span>
<span id="cb20-20"><a href="#cb20-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-21"><a href="#cb20-21" aria-hidden="true" tabindex="-1"></a>IVSA_EX_noblocks <span class="ot">=</span> IVSA_metadata_sheet[<span class="fu">which</span>(<span class="fu">is.na</span>(IVSA_metadata_sheet<span class="sc">$</span>block) <span class="sc">&amp;</span></span>
<span id="cb20-22"><a href="#cb20-22" aria-hidden="true" tabindex="-1"></a>                                         IVSA_metadata_sheet<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;Extinction&quot;</span>),</span>
<span id="cb20-23"><a href="#cb20-23" aria-hidden="true" tabindex="-1"></a>                                 <span class="st">&quot;variable&quot;</span>]</span>
<span id="cb20-24"><a href="#cb20-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-25"><a href="#cb20-25" aria-hidden="true" tabindex="-1"></a>IVSA_RI_noblocks <span class="ot">=</span> IVSA_metadata_sheet[<span class="fu">which</span>(<span class="fu">is.na</span>(IVSA_metadata_sheet<span class="sc">$</span>block) <span class="sc">&amp;</span></span>
<span id="cb20-26"><a href="#cb20-26" aria-hidden="true" tabindex="-1"></a>                                         IVSA_metadata_sheet<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;Reinstatement&quot;</span>),</span>
<span id="cb20-27"><a href="#cb20-27" aria-hidden="true" tabindex="-1"></a>                                 <span class="st">&quot;variable&quot;</span>]</span>
<span id="cb20-28"><a href="#cb20-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-29"><a href="#cb20-29" aria-hidden="true" tabindex="-1"></a>IVSA_data_vars <span class="ot">=</span> <span class="fu">c</span>(IVSA_AQ_vars_noblocks_nosessions,</span>
<span id="cb20-30"><a href="#cb20-30" aria-hidden="true" tabindex="-1"></a>                   IVSA_DR_vars_currentprotocol,</span>
<span id="cb20-31"><a href="#cb20-31" aria-hidden="true" tabindex="-1"></a>                   IVSA_EX_noblocks,</span>
<span id="cb20-32"><a href="#cb20-32" aria-hidden="true" tabindex="-1"></a>                   IVSA_RI_noblocks)</span>
<span id="cb20-33"><a href="#cb20-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-34"><a href="#cb20-34" aria-hidden="true" tabindex="-1"></a><span class="co"># Using the RI_Inf_Total_s01 variable to filter out columns with too many NAs in them</span></span>
<span id="cb20-35"><a href="#cb20-35" aria-hidden="true" tabindex="-1"></a>noNA_col <span class="ot">=</span> <span class="st">&quot;RI_Inf_Total_s01&quot;</span></span>
<span id="cb20-36"><a href="#cb20-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-37"><a href="#cb20-37" aria-hidden="true" tabindex="-1"></a>IVSA_NAs <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">row.names =</span> IVSA_data_vars,</span>
<span id="cb20-38"><a href="#cb20-38" aria-hidden="true" tabindex="-1"></a>                      <span class="at">IVSA_var =</span> IVSA_data_vars,</span>
<span id="cb20-39"><a href="#cb20-39" aria-hidden="true" tabindex="-1"></a>                      <span class="at">NAs =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">1</span>, <span class="at">times =</span> <span class="fu">length</span>(IVSA_data_vars)),</span>
<span id="cb20-40"><a href="#cb20-40" aria-hidden="true" tabindex="-1"></a>                      <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb20-41"><a href="#cb20-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-42"><a href="#cb20-42" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> IVSA_data_vars) {</span>
<span id="cb20-43"><a href="#cb20-43" aria-hidden="true" tabindex="-1"></a>  IVSA_df_i <span class="ot">=</span> DO_IVSA_raw[,<span class="fu">c</span>(i,noNA_col)]</span>
<span id="cb20-44"><a href="#cb20-44" aria-hidden="true" tabindex="-1"></a>  na_i <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">is.na</span>(IVSA_df_i[,i]) <span class="sc">&amp;</span> (<span class="sc">!</span><span class="fu">is.na</span>(IVSA_df_i[,noNA_col])))</span>
<span id="cb20-45"><a href="#cb20-45" aria-hidden="true" tabindex="-1"></a>  IVSA_NAs[i,<span class="st">&quot;NAs&quot;</span>] <span class="ot">=</span> <span class="fu">length</span>(na_i)</span>
<span id="cb20-46"><a href="#cb20-46" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb20-47"><a href="#cb20-47" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-48"><a href="#cb20-48" aria-hidden="true" tabindex="-1"></a>IVSA_use <span class="ot">=</span> IVSA_NAs[<span class="fu">which</span>(IVSA_NAs<span class="sc">$</span>NAs <span class="sc">&lt;</span> <span class="dv">50</span>),<span class="st">&quot;IVSA_var&quot;</span>]</span>
<span id="cb20-49"><a href="#cb20-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-50"><a href="#cb20-50" aria-hidden="true" tabindex="-1"></a>IVSA_transformed <span class="ot">=</span> DO_IVSA_raw[,<span class="fu">c</span>(IVSA_DE_vars, IVSA_use)]</span>
<span id="cb20-51"><a href="#cb20-51" aria-hidden="true" tabindex="-1"></a>precenter_rows <span class="ot">=</span> <span class="fu">row.names</span>(IVSA_transformed)[<span class="fu">which</span>(IVSA_transformed<span class="sc">$</span>DE_DO_Center_Status <span class="sc">==</span> <span class="st">&quot;precenter&quot;</span>)]</span>
<span id="cb20-52"><a href="#cb20-52" aria-hidden="true" tabindex="-1"></a>CSNA_rows <span class="ot">=</span>  <span class="fu">row.names</span>(IVSA_transformed)[<span class="fu">which</span>(IVSA_transformed<span class="sc">$</span>DE_DO_Center_Status <span class="sc">==</span> <span class="st">&quot;CSNA&quot;</span>)]</span></code></pre></div>
<p>We append values corresponding to the maximum time allowed to acquire for <code>AQ_SessionsToAcquisition</code> for mice that did not acquire. This depended upon the center status.</p>
<ul>
<li>For <code>precenter</code> mice, this value was 28.</li>
<li>FOr <code>CSNA</code> mice, this value was 18.</li>
</ul>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>IVSA_transformed[<span class="fu">which</span>(IVSA_transformed<span class="sc">$</span>DE_Subject <span class="sc">%in%</span> precenter_rows <span class="sc">&amp;</span> IVSA_transformed<span class="sc">$</span>DE_Exit_Reason <span class="sc">==</span> <span class="st">&quot;IVSA Acquisition Only&quot;</span>),<span class="st">&quot;AQ_SessionsToAcquisition&quot;</span>] <span class="ot">=</span> <span class="dv">28</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>IVSA_transformed[<span class="fu">which</span>(IVSA_transformed<span class="sc">$</span>DE_Subject <span class="sc">%in%</span> CSNA_rows <span class="sc">&amp;</span> IVSA_transformed<span class="sc">$</span>DE_Exit_Reason <span class="sc">==</span> <span class="st">&quot;IVSA Acquisition Only&quot;</span>),<span class="st">&quot;AQ_SessionsToAcquisition&quot;</span>] <span class="ot">=</span> <span class="dv">18</span></span></code></pre></div>
<p>Next, we rank-normal transform the pre-center and CSNA data separately as with the novelty response data.</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> IVSA_use) {</span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>  IVSA_transformed[,<span class="fu">paste</span>(i,<span class="st">&quot;_batch_ranknorm&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)] <span class="ot">=</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(IVSA_transformed))</span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a>  IVSA_transformed[precenter_rows,<span class="fu">paste</span>(i,<span class="st">&quot;_batch_ranknorm&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)] <span class="ot">=</span> <span class="fu">norm_rank_transform</span>(IVSA_transformed[precenter_rows,i])</span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a>  IVSA_transformed[CSNA_rows,<span class="fu">paste</span>(i,<span class="st">&quot;_batch_ranknorm&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)] <span class="ot">=</span> <span class="fu">norm_rank_transform</span>(IVSA_transformed[CSNA_rows,i])</span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
</div>
</div>
<div id="output" class="section level3">
<h3>Output</h3>
<p>Saving the DO IVSA, LIMS data, and metadata.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_IVSA_raw, <span class="at">file =</span> <span class="st">&quot;./data/DO_IVSA_raw_data.RDS&quot;</span>)</span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(IVSA_metadata_sheet, <span class="at">file =</span> <span class="st">&quot;./data/IVSA_metadata_sheet.RDS&quot;</span>)</span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(LIMS_data[<span class="fu">which</span>(LIMS_data<span class="sc">$</span>Line_Name <span class="sc">==</span> <span class="st">&quot;J:DO&quot;</span>),], <span class="at">file =</span> <span class="st">&quot;./data/DO_LIMS_data.RDS&quot;</span>)</span>
<span id="cb23-4"><a href="#cb23-4" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(IVSA_transformed, <span class="at">file =</span> <span class="st">&quot;./data/DO_IVSA_transformed_data.RDS&quot;</span>)</span></code></pre></div>
<p>The output data are saved in <code>./data/</code>.</p>
</div>
<div id="document-control" class="section level3">
<h3>Document Control</h3>
<p>This document was prepared using RMarkdown in RStudio.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.3 LTS

Matrix products: default
BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] lubridate_1.8.0 tidyr_1.1.4     readxl_1.3.1    MASS_7.3-54    

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.7       pillar_1.6.4     compiler_4.1.2   cellranger_1.1.0
 [5] later_1.3.0      jquerylib_0.1.4  git2r_0.29.0     workflowr_1.6.2 
 [9] tools_4.1.2      digest_0.6.29    evaluate_0.14    lifecycle_1.0.1 
[13] tibble_3.1.6     pkgconfig_2.0.3  rlang_0.4.12     DBI_1.1.1       
[17] yaml_2.2.1       xfun_0.28        fastmap_1.1.0    stringr_1.4.0   
[21] dplyr_1.0.7      knitr_1.36       generics_0.1.1   fs_1.5.1        
[25] vctrs_0.3.8      tidyselect_1.1.1 rprojroot_2.0.2  glue_1.5.1      
[29] R6_2.5.1         fansi_0.5.0      rmarkdown_2.11   purrr_0.3.4     
[33] magrittr_2.0.1   whisker_0.4      promises_1.2.0.1 ellipsis_0.3.2  
[37] htmltools_0.5.2  assertthat_0.2.1 httpuv_1.6.3     utf8_1.2.2      
[41] stringi_1.7.6    crayon_1.4.2    </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
