<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Michael C. Saul (michael.saul [at] jax.org)" />


<title>DO IVSA and Novelty Response Canonical Correlation</title>

<script src="site_libs/header-attrs-2.17/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Diversity Outbred Reference Traits Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">DO IVSA and Novelty Response Canonical
Correlation</h1>
<h4 class="author">Michael C. Saul (michael.saul [at] jax.org)</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-03-29
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rta/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20200501code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20200501)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20200501code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20200501)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstronge35887d">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong> e35887d
</a>
</p>
</div>
<div id="strongRepositoryversionstronge35887d"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version e35887d.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/.Rapp.history
    Ignored:    data/.Rhistory
    Ignored:    data/RDS/
    Ignored:    run/

Untracked files:
    Untracked:  LICENSE
    Untracked:  Rplot.pdf
    Untracked:  SENSI_chr7_snps.R
    Untracked:  Untitled.R
    Untracked:  analysis/17_mapping_interpretation_old.Rmd
    Untracked:  analysis/temp.txt
    Untracked:  code/DO_IVSA_SALINE_pleiotropy.R
    Untracked:  code/DO_IVSA_SENS_pleiotropy.R
    Untracked:  code/DO_SENS_SALINE_pleiotropy.R
    Untracked:  code/DO_eQTL_perms.R
    Untracked:  code/DO_eQTL_score_calc.R
    Untracked:  code/SENSI_chr7_snps.R
    Untracked:  code/STAR_RSEM_416.sh
    Untracked:  code/eQTL_plotting_DO_striatum_416.R
    Untracked:  code/ggplot_pub_themes.R
    Untracked:  code/pleitropy_analysis.sbatch
    Untracked:  code/pleitropy_saline_analysis.sbatch
    Untracked:  code/pleitropy_sensi_saline_analysis.sbatch
    Untracked:  code/rta_mapping_figure_leftover.R
    Untracked:  data/20_elastic_mv_results.csv
    Untracked:  data/20_elasticnet_predictions.RDS
    Untracked:  data/DO_416_DGEList.RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 3).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 4).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_DGEList_unfiltered.RDS
    Untracked:  data/DO_416_ERCC_DGEList.RDS
    Untracked:  data/DO_416_ERCC_cpm.RDS
    Untracked:  data/DO_416_normrt.RDS
    Untracked:  data/DO_416_pheno.RDS
    Untracked:  data/DO_416_pheno_allgenes.RDS
    Untracked:  data/DO_416_voom (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom.RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom_allgenes.RDS
    Untracked:  data/DO_416_voom_cpresiduals.RDS
    Untracked:  data/DO_AQ_active_CoV_toptable_annotated.RDS
    Untracked:  data/DO_AQ_lever_ratio_toptable_annotated.RDS
    Untracked:  data/DO_AQ_mean_inf_toptable_annotated.RDS
    Untracked:  data/DO_AQ_rate_toptable_annotated.RDS
    Untracked:  data/DO_IVSA_SALINE_pleio_lods.RDS
    Untracked:  data/DO_IVSA_SENS_pleio_lods.RDS
    Untracked:  data/DO_IVSA_SENS_pleio_pvl.RDS
    Untracked:  data/DO_IVSA_acquisition_cancor_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_glm.RDS
    Untracked:  data/DO_IVSA_all_cancor_df.RDS
    Untracked:  data/DO_IVSA_all_cancor_df_2024-03-27.RDS
    Untracked:  data/DO_IVSA_all_cancor_df_2024-03-29.RDS
    Untracked:  data/DO_IVSA_data.RDS
    Untracked:  data/DO_IVSA_data_2024-03-29.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_ALL.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_ALL_2024-03-29.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_AQ.RDS
    Untracked:  data/DO_IVSA_novelty_data.RDS
    Untracked:  data/DO_IVSA_novelty_data_2024-03-29.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_old.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_a1_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_a2_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor_
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor_2024-03-25.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor_2024-03-27.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor_2024-03-29.RDS
    Untracked:  data/DO_IVSA_prediction_a4_male_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a5_female_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_raw_data.RDS
    Untracked:  data/DO_IVSA_transformed_data.RDS
    Untracked:  data/DO_LIMS_data.RDS
    Untracked:  data/DO_RTG_a1_new.qtl.RData
    Untracked:  data/DO_RTG_a1_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a1_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a2_new.qtl.RData
    Untracked:  data/DO_RTG_a2_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a2_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a3_new.qtl.RData
    Untracked:  data/DO_RTG_a3_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_01.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_02.RData
    Untracked:  data/DO_RTG_a4_new.qtl.RData
    Untracked:  data/DO_RTG_a5_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_project_df_for_mapping.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_2020-11-06.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_a2.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_sexspecific.RDS
    Untracked:  data/DO_SENS_SALINE_pleio_lods.RDS
    Untracked:  data/DO_cocaine_measures_data.RDS
    Untracked:  data/DO_demographic_data_df.RDS
    Untracked:  data/DO_eigengene_significance.RDS
    Untracked:  data/DO_eigengenes.RDS
    Untracked:  data/DO_elasticnet_GEX_pvalue_histograms.RData
    Untracked:  data/DO_highcor_cpresiduals_datatable.RDS
    Untracked:  data/DO_holeboard_data_df.RDS
    Untracked:  data/DO_holeboard_summarized.RDS
    Untracked:  data/DO_lightdark_data_df.RDS
    Untracked:  data/DO_mQTL_peaks.RDS
    Untracked:  data/DO_mQTL_scan1s.RDS
    Untracked:  data/DO_module_membership.RDS
    Untracked:  data/DO_novelplace_data_df.RDS
    Untracked:  data/DO_novelty_COC_cancor.RDS
    Untracked:  data/DO_novelty_SAL_cancor.RDS
    Untracked:  data/DO_novelty_data.RDS
    Untracked:  data/DO_novelty_data_2024-03-29.RDS
    Untracked:  data/DO_novelty_response_data.RDS
    Untracked:  data/DO_novelty_response_data_for_RTG.RDS
    Untracked:  data/DO_novelty_response_data_with_transform.csv
    Untracked:  data/DO_novelty_saline_cocaine_cancor.RDS
    Untracked:  data/DO_openfield_data_df.RDS
    Untracked:  data/DO_paraclique_data.el
    Untracked:  data/DO_paraclique_eigengenes.RDS
    Untracked:  data/DO_paraclique_significance_elasticnet.RDS
    Untracked:  data/DO_paracliques.RDS
    Untracked:  data/DO_paracliques_significance.RDS
    Untracked:  data/DO_ploidy_calls.RDS
    Untracked:  data/DO_pmap.RDS
    Untracked:  data/DO_pvalue_histograms.RData
    Untracked:  data/DO_rtg_a1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a3_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m3_toptable_annotated.RDS
    Untracked:  data/DO_saline.qtl.RData
    Untracked:  data/DO_saline_measures_data.RDS
    Untracked:  data/DO_sensi_cocaine.qtl.RData
    Untracked:  data/DO_str_2016_eQTL.RData
    Untracked:  data/DO_str_2016_gm4qtl2.zip
    Untracked:  data/DO_str_2016_gm4qtl2_intensities.fst
    Untracked:  data/DO_voom_cpresiduals_NArm.RDS
    Untracked:  data/DO_wgcna.RDS
    Untracked:  data/IVSA_RTG_a3_blup.pdf
    Untracked:  data/IVSA_h2_2019-11-18.tsv
    Untracked:  data/IVSA_metadata_sheet.RDS
    Untracked:  data/ME16_edges.tsv
    Untracked:  data/Mus_musculus.GRCm38.94.parsed.RDS
    Untracked:  data/Paraclique_19_MF_AMIGO.txt
    Untracked:  data/RTG_QTL_peaks.RDS
    Untracked:  data/RTG_figure.RDS
    Untracked:  data/SENSI_chr7_snps.RData
    Untracked:  data/SENS_RTG/
    Untracked:  data/Tyr2_genotypes.RDS
    Untracked:  data/Tyr_genotypes.RDS
    Untracked:  data/blup_chr7_DO_RTG_a3.RDS
    Untracked:  data/blup_spearman_cors.RData
    Untracked:  data/cc_variants/
    Untracked:  data/chr18_QTL_results.RDS
    Untracked:  data/chr4_QTL_results.RDS
    Untracked:  data/chr7_a3_QTL_results.RDS
    Untracked:  data/chr7_blups_with_rtg.RDS
    Untracked:  data/chr7_cocaine_BLUP.RDS
    Untracked:  data/chr7_gene_blup.RDS
    Untracked:  data/chr7_qtl_cors_with_spearman_blups.RDS
    Untracked:  data/chr7_sensi_cocaine_BLUP.RDS
    Untracked:  data/cis/
    Untracked:  data/eQTL_annotated_peaks.RDS
    Untracked:  data/eQTL_plots.RData
    Untracked:  data/figures/
    Untracked:  data/founders_key/
    Untracked:  data/gigamuga/
    Untracked:  data/gwas/
    Untracked:  data/info/
    Untracked:  data/m2G.a2.RTG.chr4snp.RData
    Untracked:  data/mapping/
    Untracked:  data/mapping_from_hao/
    Untracked:  data/mpd_measures_saline.txt
    Untracked:  data/mpd_measures_saline_cocaine.txt
    Untracked:  data/perms/
    Untracked:  data/pleio_df.RDS
    Untracked:  data/qtl2/
    Untracked:  data/raw/
    Untracked:  data/resources/
    Untracked:  data/tads_loops/
    Untracked:  data/transcripts/
    Untracked:  data/variants/
    Untracked:  logs/
    Untracked:  new_RTG_a3.jpeg
    Untracked:  output/DO_module_membership.tsv
    Untracked:  output/DO_paraclique_df.tsv
    Untracked:  output/RDS/
    Untracked:  output/RTG_out/
    Untracked:  output/docs_backup_2020-07-30.tar.gz
    Untracked:  output/elastic_net_out/
    Untracked:  output/mapping_figure.pdf
    Untracked:  output/mapping_plot.pdf
    Untracked:  output/mediation_all_rta_figure.pdf
    Untracked:  output/mediation_all_rta_figure_with_eed.pdf
    Untracked:  output/pleio_plot2.pdf
    Untracked:  rstudio_rta.sif
    Untracked:  singularity/
    Untracked:  tmp/
    Untracked:  var/

Unstaged changes:
    Deleted:    .Rprofile
    Modified:   .gitignore
    Modified:   README.md
    Modified:   analysis/17_mapping_interpretation.Rmd
    Modified:   analysis/_site.yml

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/12_DO_IVSA_novelty_cancor.Rmd</code>) and HTML
(<code>docs/12_DO_IVSA_novelty_cancor.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
e35887d
</td>
<td>
Michael C. Saul
</td>
<td>
2024-03-29
</td>
<td>
Pulling out commented lines that no longer had relevance.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
e8251da
</td>
<td>
Michael C. Saul
</td>
<td>
2024-03-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
92ef26b
</td>
<td>
Michael C. Saul
</td>
<td>
2024-03-29
</td>
<td>
Pulling irrelevant analysis out of this document (note: trakced in git)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
2cb3df3
</td>
<td>
Michael C. Saul
</td>
<td>
2024-03-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
02e05e1
</td>
<td>
Michael C. Saul
</td>
<td>
2024-03-29
</td>
<td>
Committing changes since altering to a correlation-based initial
variable selection.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
b194018
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
5600b79
</td>
<td>
Michael C. Saul
</td>
<td>
2021-02-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
4bf260b
</td>
<td>
Michael C. Saul
</td>
<td>
2021-02-09
</td>
<td>
Integrating adjustment for sex
</td>
</tr>
<tr>
<td>
html
</td>
<td>
b5b4d59
</td>
<td>
Michael C. Saul
</td>
<td>
2021-01-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
d57c5e0
</td>
<td>
Michael C. Saul
</td>
<td>
2021-01-14
</td>
<td>
Going back to analysis 2 with acquisition included
</td>
</tr>
<tr>
<td>
html
</td>
<td>
21a5d69
</td>
<td>
Michael C. Saul
</td>
<td>
2021-01-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
3f55b08
</td>
<td>
Michael C. Saul
</td>
<td>
2021-01-06
</td>
<td>
Redoing with raw sessions to acquisition
</td>
</tr>
<tr>
<td>
html
</td>
<td>
e4e9312
</td>
<td>
Michael C. Saul
</td>
<td>
2021-01-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
d4b5610
</td>
<td>
Michael C. Saul
</td>
<td>
2021-01-05
</td>
<td>
Redoing the canonical correlation without acquiring as a binary variable
in analysis 2
</td>
</tr>
<tr>
<td>
html
</td>
<td>
1c4a09a
</td>
<td>
Michael C. Saul
</td>
<td>
2021-01-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
2ae87b6
</td>
<td>
Michael C. Saul
</td>
<td>
2021-01-04
</td>
<td>
Updating canonical correlation to work on batch-corrected datasets
</td>
</tr>
<tr>
<td>
html
</td>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
<td>
Rolling back to working version.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
72db5a1
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
1c079bf
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-22
</td>
<td>
Fixing save file names
</td>
</tr>
<tr>
<td>
html
</td>
<td>
e028695
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
5890af6
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-21
</td>
<td>
Committing saves of data frames for logistic regression and canonical
correlations
</td>
</tr>
<tr>
<td>
html
</td>
<td>
08b2bcf
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
dcbe6c0
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-03
</td>
<td>
Second tweaking sizes for publication figure
</td>
</tr>
<tr>
<td>
html
</td>
<td>
6c09258
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
40a48fd
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-03
</td>
<td>
Tweaking sizes for publication figure
</td>
</tr>
<tr>
<td>
html
</td>
<td>
f05c929
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
a592de0
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-03
</td>
<td>
Saving publication-quality figure
</td>
</tr>
<tr>
<td>
html
</td>
<td>
c7c5610
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
0a03183
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-03
</td>
<td>
Saved canonical correlations, logistic regression, and underlying data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
401d3f9
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
0e17fff
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Tweaking canonical correlation to begin Model 3 with Open Field Slope
</td>
</tr>
<tr>
<td>
html
</td>
<td>
c2c9117
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
cc746cb
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Tweaked canonical correlation for Model 3 slightly
</td>
</tr>
<tr>
<td>
html
</td>
<td>
ec4048f
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
116de33
</td>
<td>
Michael C. Saul
</td>
<td>
2020-06-02
</td>
<td>
workflowr::wflow_publish("./analysis/12_DO_IVSA_novelty_cancor.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="background" class="section level1">
<h1>Background</h1>
<div id="motivation" class="section level2">
<h2>Motivation</h2>
<div id="purpose-of-analysis" class="section level3">
<h3>Purpose of analysis</h3>
<p>This script is used to summarize the novelty data upstream of using
them for reference trait analysis.</p>
<p><em>NOTE: on 2024-03-29, Michael Saul pulled a couple of analyses out
of this document that were no longer relevant to the reference trait
analysis paper. These can be restored with <code>git</code> as this
repository is tracked with <code>git</code>.</em></p>
</div>
</div>
</div>
<div id="analysis" class="section level1">
<h1>Analysis</h1>
<div id="setup" class="section level2">
<h2>Setup</h2>
<div id="r-libraries" class="section level3">
<h3>R libraries</h3>
<p>Calling R libraries necessary for this analysis.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;MASS&quot;</span>)</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;cowplot&quot;</span>)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;lubridate&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;lubridate&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:cowplot&#39;:

    stamp</code></pre>
<pre><code>The following objects are masked from &#39;package:base&#39;:

    date, intersect, setdiff, union</code></pre>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;car&quot;</span>)</span></code></pre></div>
<pre><code>Loading required package: carData</code></pre>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;corrplot&quot;</span>)</span></code></pre></div>
<pre><code>corrplot 0.92 loaded</code></pre>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;tidyr&quot;</span>)</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;gplots&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    lowess</code></pre>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggpubr&quot;</span>)</span></code></pre></div>
<pre><code>Loading required package: ggplot2</code></pre>
<pre><code>
Attaching package: &#39;ggpubr&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:cowplot&#39;:

    get_legend</code></pre>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggbeeswarm&quot;</span>)</span></code></pre></div>
<p>Getting a normal rank transformation function that can handle
<code>NA</code> values from the following formula:</p>
<p><span
class="math inline">\(Y^t_i={\phi}^{-1}({{r_i-c}\over{N-2c+1}})\)</span></p>
<p>where <span class="math inline">\(r_i\)</span> is the rank, <span
class="math inline">\(N\)</span> is the number of ranked samples, <span
class="math inline">\(\phi^{-1}\)</span> is the inverse normal function
(equivalent to <code>qnorm()</code> in R). A recommended value of <span
class="math inline">\(c=3/8\)</span> in <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2921808/">Beasley and
Erickson, 2010</a>, but we use the <a
href="https://scholar.google.com/scholar_lookup?journal=Proc+Koninklijke+Nederlandse+Akademie+van+Wetenschappen&amp;title=Order+tests+for+the+two-sample+problem+and+their+power&amp;author=BL+van+der+Waerden&amp;volume=55&amp;publication_year=1952&amp;pages=453-458&amp;">van
der Waerden (1952)</a> value of <span class="math inline">\(c=0\)</span>
per Vivek Philip. Additionally, after discussing this with Vivek, I’ve
changed the rank tie method to <code>"average"</code> whereas I had
<code>"random"</code> previously.</p>
<p>Our work appears to meet Beasley and Erickson’s sample size
recommendations.</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>norm_rank_transform <span class="ot">=</span> <span class="cf">function</span>(x, <span class="at">c =</span> <span class="dv">0</span>) {</span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stopifnot</span>(<span class="fu">is.numeric</span>(x) <span class="sc">&amp;</span> <span class="fu">is.vector</span>(x))</span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>  x_noNA <span class="ot">=</span> <span class="fu">which</span>(<span class="sc">!</span><span class="fu">is.na</span>(x))</span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a>  N <span class="ot">=</span> <span class="fu">length</span>(x_noNA)</span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>  x[x_noNA] <span class="ot">=</span> <span class="fu">qnorm</span>((<span class="fu">rank</span>(x[x_noNA], <span class="at">ties.method =</span> <span class="st">&quot;average&quot;</span>) <span class="sc">-</span> c) <span class="sc">/</span> (N <span class="sc">-</span> (<span class="dv">2</span> <span class="sc">*</span> c) <span class="sc">+</span> <span class="dv">1</span>))</span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(x)</span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<div id="getting-publication-themes" class="section level4">
<h4>Getting publication themes</h4>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="fu">source</span>(<span class="st">&quot;./code/ggplot_pub_themes.R&quot;</span>)</span></code></pre></div>
<pre><code>Warning: The `size` argument of `element_line()` is deprecated as of ggplot2 3.4.0.
ℹ Please use the `linewidth` argument instead.
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
generated.</code></pre>
</div>
</div>
<div id="importing-data" class="section level3">
<h3>Importing data</h3>
<div id="importing-ivsa-and-novelty-response-data"
class="section level4">
<h4>Importing IVSA and novelty response data</h4>
<p>Bringing in datasets from RDS files as well as founders IVSA
heritability.</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>DO_novelty <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_novelty_response_data.RDS&quot;</span>)</span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>DO_IVSA <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_IVSA_transformed_data.RDS&quot;</span>)</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a>IVSA_metadata <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/IVSA_metadata_sheet.RDS&quot;</span>)</span>
<span id="cb20-4"><a href="#cb20-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-5"><a href="#cb20-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting IVSA measure heritability calculations derived from the CC/DO founders</span></span>
<span id="cb20-6"><a href="#cb20-6" aria-hidden="true" tabindex="-1"></a>IVSA_h2 <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">&quot;./data/IVSA_h2_2019-11-18.tsv&quot;</span>,</span>
<span id="cb20-7"><a href="#cb20-7" aria-hidden="true" tabindex="-1"></a>                     <span class="at">sep =</span> <span class="st">&quot;</span><span class="sc">\t</span><span class="st">&quot;</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span></code></pre></div>
<p>Getting IVSA metadata. Selecting only those doses that are in the SOP
v2 (1.0 mg/kg, 0.32 mg/kg, 0.1 mg/kg, and 0.032 mg/kg).</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>IVSA_DE_vars <span class="ot">=</span> IVSA_metadata[<span class="fu">which</span>(IVSA_metadata<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;Descriptive&quot;</span>),<span class="st">&quot;variable&quot;</span>]</span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Removing data columns broken down by sessions and blocks</span></span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>IVSA_AQ_vars_noblocks_nosessions <span class="ot">=</span> IVSA_metadata[<span class="fu">which</span>(<span class="fu">is.na</span>(IVSA_metadata<span class="sc">$</span>session) <span class="sc">&amp;</span></span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>                                                         <span class="fu">is.na</span>(IVSA_metadata<span class="sc">$</span>block) <span class="sc">&amp;</span></span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a>                                                         IVSA_metadata<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;Acquisition&quot;</span>),</span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>                                                 <span class="st">&quot;variable&quot;</span>]</span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Removing data columns not at 0.032, 0.1, 0.32, and 1 mg/kg</span></span>
<span id="cb21-10"><a href="#cb21-10" aria-hidden="true" tabindex="-1"></a>IVSA_DR_currentdoses <span class="ot">=</span> <span class="fu">c</span>(<span class="fl">0.032</span>, <span class="fl">0.1</span>, <span class="fl">0.32</span>, <span class="fl">1.0</span>)</span>
<span id="cb21-11"><a href="#cb21-11" aria-hidden="true" tabindex="-1"></a>IVSA_DR_vars_currentprotocol <span class="ot">=</span> IVSA_metadata[<span class="fu">which</span>(<span class="fu">is.na</span>(IVSA_metadata<span class="sc">$</span>session) <span class="sc">&amp;</span></span>
<span id="cb21-12"><a href="#cb21-12" aria-hidden="true" tabindex="-1"></a>                                                     <span class="fu">is.na</span>(IVSA_metadata<span class="sc">$</span>block) <span class="sc">&amp;</span></span>
<span id="cb21-13"><a href="#cb21-13" aria-hidden="true" tabindex="-1"></a>                                                     IVSA_metadata<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;DoseResponse&quot;</span> <span class="sc">&amp;</span></span>
<span id="cb21-14"><a href="#cb21-14" aria-hidden="true" tabindex="-1"></a>                                                     IVSA_metadata<span class="sc">$</span>dose_mgperkg <span class="sc">%in%</span> IVSA_DR_currentdoses),</span>
<span id="cb21-15"><a href="#cb21-15" aria-hidden="true" tabindex="-1"></a>                                             <span class="st">&quot;variable&quot;</span>]</span>
<span id="cb21-16"><a href="#cb21-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-17"><a href="#cb21-17" aria-hidden="true" tabindex="-1"></a>IVSA_EX_noblocks <span class="ot">=</span> IVSA_metadata[<span class="fu">which</span>(<span class="fu">is.na</span>(IVSA_metadata<span class="sc">$</span>block) <span class="sc">&amp;</span></span>
<span id="cb21-18"><a href="#cb21-18" aria-hidden="true" tabindex="-1"></a>                                         IVSA_metadata<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;Extinction&quot;</span>),</span>
<span id="cb21-19"><a href="#cb21-19" aria-hidden="true" tabindex="-1"></a>                                 <span class="st">&quot;variable&quot;</span>]</span>
<span id="cb21-20"><a href="#cb21-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-21"><a href="#cb21-21" aria-hidden="true" tabindex="-1"></a>IVSA_RI_noblocks <span class="ot">=</span> IVSA_metadata[<span class="fu">which</span>(<span class="fu">is.na</span>(IVSA_metadata<span class="sc">$</span>block) <span class="sc">&amp;</span></span>
<span id="cb21-22"><a href="#cb21-22" aria-hidden="true" tabindex="-1"></a>                                         IVSA_metadata<span class="sc">$</span>vartype <span class="sc">==</span> <span class="st">&quot;Reinstatement&quot;</span>),</span>
<span id="cb21-23"><a href="#cb21-23" aria-hidden="true" tabindex="-1"></a>                                 <span class="st">&quot;variable&quot;</span>]</span>
<span id="cb21-24"><a href="#cb21-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-25"><a href="#cb21-25" aria-hidden="true" tabindex="-1"></a>IVSA_data_vars <span class="ot">=</span> <span class="fu">c</span>(IVSA_AQ_vars_noblocks_nosessions,</span>
<span id="cb21-26"><a href="#cb21-26" aria-hidden="true" tabindex="-1"></a>                   IVSA_DR_vars_currentprotocol,</span>
<span id="cb21-27"><a href="#cb21-27" aria-hidden="true" tabindex="-1"></a>                   IVSA_EX_noblocks,</span>
<span id="cb21-28"><a href="#cb21-28" aria-hidden="true" tabindex="-1"></a>                   IVSA_RI_noblocks)</span></code></pre></div>
</div>
</div>
<div id="analysis-1" class="section level3">
<h3>Analysis</h3>
<div id="novelty-and-ivsa-data" class="section level4">
<h4>Novelty and IVSA Data</h4>
<p>Making data frame to carry novelty and IVSA data</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>novelty_vars <span class="ot">=</span> <span class="fu">colnames</span>(DO_novelty)</span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>novelty_vars <span class="ot">=</span> novelty_vars[<span class="fu">grep</span>(<span class="st">&quot;_batch_ranknorm$&quot;</span>,novelty_vars)]</span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a>IVSA_vars <span class="ot">=</span> <span class="fu">colnames</span>(DO_IVSA)</span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a>IVSA_vars <span class="ot">=</span> IVSA_vars[<span class="fu">grep</span>(<span class="st">&quot;_batch_ranknorm$&quot;</span>, IVSA_vars)]</span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a>IVSA_vars <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;AQ_SessionsToAcquisition&quot;</span>,IVSA_vars)</span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-7"><a href="#cb22-7" aria-hidden="true" tabindex="-1"></a>DO_novelty_IVSA_IDs <span class="ot">=</span> <span class="fu">intersect</span>(<span class="fu">row.names</span>(DO_IVSA), <span class="fu">row.names</span>(DO_novelty))</span>
<span id="cb22-8"><a href="#cb22-8" aria-hidden="true" tabindex="-1"></a>DO_novelty_IVSA <span class="ot">=</span> <span class="fu">cbind</span>(DO_IVSA[DO_novelty_IVSA_IDs,<span class="fu">c</span>(IVSA_DE_vars, IVSA_vars)],</span>
<span id="cb22-9"><a href="#cb22-9" aria-hidden="true" tabindex="-1"></a>                        DO_novelty[DO_novelty_IVSA_IDs,novelty_vars])</span>
<span id="cb22-10"><a href="#cb22-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-11"><a href="#cb22-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtering for only animals that acquired IVSA or were acquisition-only.</span></span>
<span id="cb22-12"><a href="#cb22-12" aria-hidden="true" tabindex="-1"></a>DO_novelty_IVSA_raw <span class="ot">=</span> DO_novelty_IVSA</span>
<span id="cb22-13"><a href="#cb22-13" aria-hidden="true" tabindex="-1"></a>DO_novelty_IVSA <span class="ot">=</span> DO_novelty_IVSA[<span class="fu">which</span>(DO_novelty_IVSA<span class="sc">$</span>DE_Exit_Reason <span class="sc">==</span> <span class="st">&quot;Finished Pipeline&quot;</span> <span class="sc">|</span></span>
<span id="cb22-14"><a href="#cb22-14" aria-hidden="true" tabindex="-1"></a>                                        DO_novelty_IVSA<span class="sc">$</span>DE_Exit_Reason <span class="sc">==</span> <span class="st">&quot;IVSA Acquisition Only&quot;</span>),]</span></code></pre></div>
<p>Looking at ratios of acquisition.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a>DO_novelty_IVSA<span class="sc">$</span>acquisition <span class="ot">=</span> <span class="fu">ifelse</span>(DO_novelty_IVSA<span class="sc">$</span>DE_Exit_Reason <span class="sc">==</span> <span class="st">&quot;Finished Pipeline&quot;</span>,<span class="dv">1</span>,<span class="dv">0</span>)</span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>DO_acquisition_ratios <span class="ot">=</span> <span class="fu">as.matrix</span>(<span class="fu">table</span>(DO_novelty_IVSA<span class="sc">$</span>DE_Sex, DO_novelty_IVSA<span class="sc">$</span>acquisition))</span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Overall acquisition: &quot;</span>, <span class="fu">round</span>(<span class="dv">100</span> <span class="sc">*</span> <span class="fu">sum</span>(DO_acquisition_ratios[,<span class="dv">2</span>]) <span class="sc">/</span> <span class="fu">sum</span>(DO_acquisition_ratios),<span class="dv">1</span>),<span class="st">&quot;%.</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Overall acquisition: 59.1%.</code></pre>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Female acquisition: &quot;</span>, <span class="fu">round</span>(<span class="dv">100</span> <span class="sc">*</span> <span class="fu">sum</span>(DO_acquisition_ratios[<span class="dv">1</span>,<span class="dv">2</span>]) <span class="sc">/</span> <span class="fu">sum</span>(DO_acquisition_ratios[<span class="dv">1</span>,]),<span class="dv">1</span>),<span class="st">&quot;%.</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Female acquisition: 61.2%.</code></pre>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Male acquisition: &quot;</span>, <span class="fu">round</span>(<span class="dv">100</span> <span class="sc">*</span> <span class="fu">sum</span>(DO_acquisition_ratios[<span class="dv">2</span>,<span class="dv">2</span>]) <span class="sc">/</span> <span class="fu">sum</span>(DO_acquisition_ratios[<span class="dv">2</span>,]),<span class="dv">1</span>),<span class="st">&quot;%.</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Male acquisition: 56.9%.</code></pre>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a>DO_acquisition_ratios</span></code></pre></div>
<pre><code>   
      0   1
  F 186 294
  M 208 275</code></pre>
</div>
<div id="model-3-canonical-correlation-on-all-ivsa-data."
class="section level4">
<h4>Model 3: Canonical correlation on all IVSA data.</h4>
<p>Running the same canonical correlation, but on acquisition plus all
other variables for animals with complete data. Requiring at least 300
samples to fit. Starting by finding the highest correlation in animals
that finished the pipeline.</p>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Excluding hole board repeat entries as it does not allow traverse to data collected before CSNA</span></span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a>novelty_vars <span class="ot">=</span> novelty_vars[<span class="fu">which</span>(novelty_vars <span class="sc">!=</span> <span class="st">&quot;HB_Repeat_Entries_batch_ranknorm&quot;</span>)]</span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Making a data frame for these data.</span></span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a>DO_novelty_IVSA <span class="ot">=</span> DO_novelty_IVSA_raw</span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a>DO_novelty_IVSA <span class="ot">=</span> DO_novelty_IVSA[<span class="fu">which</span>(DO_novelty_IVSA<span class="sc">$</span>DE_Exit_Reason <span class="sc">==</span> <span class="st">&quot;Finished Pipeline&quot;</span>),]</span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-8"><a href="#cb31-8" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting highest correlation variables to start out</span></span>
<span id="cb31-9"><a href="#cb31-9" aria-hidden="true" tabindex="-1"></a>highest_cor <span class="ot">=</span> <span class="fu">cor</span>(DO_novelty_IVSA[,IVSA_vars], </span>
<span id="cb31-10"><a href="#cb31-10" aria-hidden="true" tabindex="-1"></a>                  DO_novelty_IVSA[,novelty_vars],</span>
<span id="cb31-11"><a href="#cb31-11" aria-hidden="true" tabindex="-1"></a>                  <span class="at">use =</span> <span class="st">&quot;pairwise.complete.obs&quot;</span>)</span></code></pre></div>
<pre><code>Warning in cor(DO_novelty_IVSA[, IVSA_vars], DO_novelty_IVSA[, novelty_vars], :
the standard deviation is zero</code></pre>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>highest_cor_r <span class="ot">=</span> <span class="fu">max</span>(<span class="fu">abs</span>(highest_cor), <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>highest_cor_ind <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">abs</span>(highest_cor) <span class="sc">==</span> highest_cor_r, <span class="at">arr.ind =</span> <span class="cn">TRUE</span>)</span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a>ivsa_highcor_id <span class="ot">=</span> <span class="fu">row.names</span>(highest_cor)[highest_cor_ind[<span class="dv">1</span>,<span class="st">&quot;row&quot;</span>]] <span class="co"># &quot;RI_Inf_Total_s01_batch_ranknorm&quot;</span></span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a>novelty_highcor_id <span class="ot">=</span> <span class="fu">colnames</span>(highest_cor)[highest_cor_ind[<span class="dv">1</span>,<span class="st">&quot;col&quot;</span>]]</span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a><span class="fu">paste0</span>(<span class="st">&quot;The highest correlation (&quot;</span>, <span class="fu">round</span>(highest_cor_r, <span class="dv">2</span>),<span class="st">&quot;) is between &quot;</span>, ivsa_highcor_id, <span class="st">&quot; and &quot;</span>,</span>
<span id="cb33-7"><a href="#cb33-7" aria-hidden="true" tabindex="-1"></a>       novelty_highcor_id, <span class="st">&quot;.&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;The highest correlation (0.22) is between RI_IEI_Mean_Inf_Total_s02_batch_ranknorm and OFA_total_distance_traveled_in_perimeter_batch_ranknorm.&quot;</code></pre>
<p>Doing the canonical correlation analysis. Starting by testing
correlations.</p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a>DO_novelty_IVSA <span class="ot">=</span> DO_novelty_IVSA_raw</span>
<span id="cb35-2"><a href="#cb35-2" aria-hidden="true" tabindex="-1"></a>DO_novelty_IVSA <span class="ot">=</span> DO_novelty_IVSA[<span class="fu">which</span>(DO_novelty_IVSA<span class="sc">$</span>DE_Exit_Reason <span class="sc">==</span> <span class="st">&quot;Finished Pipeline&quot;</span>),]</span>
<span id="cb35-3"><a href="#cb35-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-4"><a href="#cb35-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Starting with these as best novelty and IVSA variables</span></span>
<span id="cb35-5"><a href="#cb35-5" aria-hidden="true" tabindex="-1"></a>best_novelty_vars <span class="ot">=</span> novelty_highcor_id</span>
<span id="cb35-6"><a href="#cb35-6" aria-hidden="true" tabindex="-1"></a>best_IVSA_vars <span class="ot">=</span> ivsa_highcor_id</span>
<span id="cb35-7"><a href="#cb35-7" aria-hidden="true" tabindex="-1"></a>noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_novelty_IVSA[,<span class="fu">c</span>(best_novelty_vars, best_IVSA_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb35-8"><a href="#cb35-8" aria-hidden="true" tabindex="-1"></a>starting_cancor <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_novelty_IVSA[noNA_rows,best_novelty_vars], DO_novelty_IVSA[noNA_rows,best_IVSA_vars],</span>
<span id="cb35-9"><a href="#cb35-9" aria-hidden="true" tabindex="-1"></a>                         <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb35-10"><a href="#cb35-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-11"><a href="#cb35-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Selecting heritable IVSA variables</span></span>
<span id="cb35-12"><a href="#cb35-12" aria-hidden="true" tabindex="-1"></a>IVSA_vars_unselected <span class="ot">=</span> IVSA_vars[(<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(IVSA_vars <span class="sc">==</span> best_IVSA_vars))]</span>
<span id="cb35-13"><a href="#cb35-13" aria-hidden="true" tabindex="-1"></a>novelty_vars_unselected <span class="ot">=</span> novelty_vars[(<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(novelty_vars <span class="sc">==</span> best_novelty_vars))]</span>
<span id="cb35-14"><a href="#cb35-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-15"><a href="#cb35-15" aria-hidden="true" tabindex="-1"></a>not_optimized <span class="ot">=</span> <span class="cn">TRUE</span></span>
<span id="cb35-16"><a href="#cb35-16" aria-hidden="true" tabindex="-1"></a>min_obs <span class="ot">=</span> <span class="dv">300</span></span>
<span id="cb35-17"><a href="#cb35-17" aria-hidden="true" tabindex="-1"></a>max_var <span class="ot">=</span> <span class="dv">20</span></span>
<span id="cb35-18"><a href="#cb35-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-19"><a href="#cb35-19" aria-hidden="true" tabindex="-1"></a><span class="cf">while</span> (not_optimized) {</span>
<span id="cb35-20"><a href="#cb35-20" aria-hidden="true" tabindex="-1"></a>  best_cancor <span class="ot">=</span> starting_cancor<span class="sc">$</span>cor[<span class="dv">1</span>]</span>
<span id="cb35-21"><a href="#cb35-21" aria-hidden="true" tabindex="-1"></a>  best_var <span class="ot">=</span> <span class="st">&quot;&quot;</span></span>
<span id="cb35-22"><a href="#cb35-22" aria-hidden="true" tabindex="-1"></a>  best_paradigm <span class="ot">=</span> <span class="st">&quot;&quot;</span></span>
<span id="cb35-23"><a href="#cb35-23" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb35-24"><a href="#cb35-24" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (i <span class="cf">in</span> novelty_vars_unselected) {</span>
<span id="cb35-25"><a href="#cb35-25" aria-hidden="true" tabindex="-1"></a>    noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_novelty_IVSA[,<span class="fu">c</span>(best_novelty_vars, i, best_IVSA_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb35-26"><a href="#cb35-26" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="fu">length</span>(noNA_rows) <span class="sc">&gt;=</span> min_obs) {</span>
<span id="cb35-27"><a href="#cb35-27" aria-hidden="true" tabindex="-1"></a>      cancor_i <span class="ot">=</span> stats<span class="sc">::</span><span class="fu">cancor</span>(DO_novelty_IVSA[noNA_rows, <span class="fu">c</span>(best_novelty_vars, i)], DO_novelty_IVSA[noNA_rows, best_IVSA_vars],</span>
<span id="cb35-28"><a href="#cb35-28" aria-hidden="true" tabindex="-1"></a>                        <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb35-29"><a href="#cb35-29" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span> (cancor_i<span class="sc">$</span>cor[<span class="dv">1</span>] <span class="sc">&gt;</span> best_cancor) {</span>
<span id="cb35-30"><a href="#cb35-30" aria-hidden="true" tabindex="-1"></a>        best_cancor <span class="ot">=</span> cancor_i<span class="sc">$</span>cor[<span class="dv">1</span>]</span>
<span id="cb35-31"><a href="#cb35-31" aria-hidden="true" tabindex="-1"></a>        best_var <span class="ot">=</span> i</span>
<span id="cb35-32"><a href="#cb35-32" aria-hidden="true" tabindex="-1"></a>        best_paradigm <span class="ot">=</span> <span class="st">&quot;novelty&quot;</span></span>
<span id="cb35-33"><a href="#cb35-33" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb35-34"><a href="#cb35-34" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb35-35"><a href="#cb35-35" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb35-36"><a href="#cb35-36" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb35-37"><a href="#cb35-37" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (j <span class="cf">in</span> IVSA_vars_unselected) {</span>
<span id="cb35-38"><a href="#cb35-38" aria-hidden="true" tabindex="-1"></a>    noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_novelty_IVSA[,<span class="fu">c</span>(best_novelty_vars, j, best_IVSA_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb35-39"><a href="#cb35-39" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="fu">length</span>(noNA_rows) <span class="sc">&gt;=</span> min_obs) {</span>
<span id="cb35-40"><a href="#cb35-40" aria-hidden="true" tabindex="-1"></a>      cancor_j <span class="ot">=</span> stats<span class="sc">::</span><span class="fu">cancor</span>(DO_novelty_IVSA[noNA_rows, best_novelty_vars], DO_novelty_IVSA[noNA_rows,<span class="fu">c</span>(best_IVSA_vars, j)],</span>
<span id="cb35-41"><a href="#cb35-41" aria-hidden="true" tabindex="-1"></a>                        <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb35-42"><a href="#cb35-42" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span> (cancor_j<span class="sc">$</span>cor[<span class="dv">1</span>] <span class="sc">&gt;</span> best_cancor) {</span>
<span id="cb35-43"><a href="#cb35-43" aria-hidden="true" tabindex="-1"></a>        best_cancor <span class="ot">=</span> cancor_j<span class="sc">$</span>cor[<span class="dv">1</span>]</span>
<span id="cb35-44"><a href="#cb35-44" aria-hidden="true" tabindex="-1"></a>        best_var <span class="ot">=</span> j</span>
<span id="cb35-45"><a href="#cb35-45" aria-hidden="true" tabindex="-1"></a>        best_paradigm <span class="ot">=</span> <span class="st">&quot;IVSA&quot;</span></span>
<span id="cb35-46"><a href="#cb35-46" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb35-47"><a href="#cb35-47" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb35-48"><a href="#cb35-48" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb35-49"><a href="#cb35-49" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb35-50"><a href="#cb35-50" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;novelty&quot;</span>) {</span>
<span id="cb35-51"><a href="#cb35-51" aria-hidden="true" tabindex="-1"></a>    novelty_vars_test <span class="ot">=</span> <span class="fu">c</span>(best_novelty_vars, best_var)</span>
<span id="cb35-52"><a href="#cb35-52" aria-hidden="true" tabindex="-1"></a>    IVSA_vars_test <span class="ot">=</span> best_IVSA_vars</span>
<span id="cb35-53"><a href="#cb35-53" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;IVSA&quot;</span>) {</span>
<span id="cb35-54"><a href="#cb35-54" aria-hidden="true" tabindex="-1"></a>    novelty_vars_test <span class="ot">=</span> best_novelty_vars</span>
<span id="cb35-55"><a href="#cb35-55" aria-hidden="true" tabindex="-1"></a>    IVSA_vars_test <span class="ot">=</span> <span class="fu">c</span>(best_IVSA_vars, best_var)</span>
<span id="cb35-56"><a href="#cb35-56" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb35-57"><a href="#cb35-57" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb35-58"><a href="#cb35-58" aria-hidden="true" tabindex="-1"></a>  noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_novelty_IVSA[,<span class="fu">c</span>(novelty_vars_test, IVSA_vars_test)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb35-59"><a href="#cb35-59" aria-hidden="true" tabindex="-1"></a>  k_gtr <span class="ot">=</span> <span class="fu">c</span>()</span>
<span id="cb35-60"><a href="#cb35-60" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb35-61"><a href="#cb35-61" aria-hidden="true" tabindex="-1"></a>  B <span class="ot">=</span> <span class="fl">1e4</span></span>
<span id="cb35-62"><a href="#cb35-62" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (k <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span>B) {</span>
<span id="cb35-63"><a href="#cb35-63" aria-hidden="true" tabindex="-1"></a>    cancor_k <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_novelty_IVSA[<span class="fu">sample</span>(noNA_rows), novelty_vars_test], </span>
<span id="cb35-64"><a href="#cb35-64" aria-hidden="true" tabindex="-1"></a>                      DO_novelty_IVSA[noNA_rows, IVSA_vars_test],</span>
<span id="cb35-65"><a href="#cb35-65" aria-hidden="true" tabindex="-1"></a>                      <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb35-66"><a href="#cb35-66" aria-hidden="true" tabindex="-1"></a>    k_gtr <span class="ot">=</span> <span class="fu">c</span>(k_gtr, cancor_k<span class="sc">$</span>cor[<span class="dv">1</span>])</span>
<span id="cb35-67"><a href="#cb35-67" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb35-68"><a href="#cb35-68" aria-hidden="true" tabindex="-1"></a>  p_value_improvement <span class="ot">=</span> (<span class="fu">length</span>(<span class="fu">which</span>(k_gtr <span class="sc">&gt;</span> best_cancor)) <span class="sc">+</span> <span class="dv">1</span>) <span class="sc">/</span> (B <span class="sc">+</span> <span class="dv">1</span>)</span>
<span id="cb35-69"><a href="#cb35-69" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb35-70"><a href="#cb35-70" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> ((<span class="fu">length</span>(best_novelty_vars) <span class="sc">+</span> <span class="fu">length</span>(best_IVSA_vars)) <span class="sc">&gt;</span> max_var) {</span>
<span id="cb35-71"><a href="#cb35-71" aria-hidden="true" tabindex="-1"></a>    not_optimized <span class="ot">=</span> <span class="cn">FALSE</span></span>
<span id="cb35-72"><a href="#cb35-72" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> <span class="cf">if</span> (p_value_improvement <span class="sc">&lt;</span> <span class="fl">0.001</span>) {</span>
<span id="cb35-73"><a href="#cb35-73" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;novelty&quot;</span>) {</span>
<span id="cb35-74"><a href="#cb35-74" aria-hidden="true" tabindex="-1"></a>      best_novelty_vars <span class="ot">=</span> <span class="fu">c</span>(best_novelty_vars, best_var)</span>
<span id="cb35-75"><a href="#cb35-75" aria-hidden="true" tabindex="-1"></a>      novelty_vars_unselected <span class="ot">=</span> novelty_vars_unselected[<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(novelty_vars_unselected <span class="sc">==</span> best_var)]</span>
<span id="cb35-76"><a href="#cb35-76" aria-hidden="true" tabindex="-1"></a>    } <span class="cf">else</span> <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;IVSA&quot;</span>) {</span>
<span id="cb35-77"><a href="#cb35-77" aria-hidden="true" tabindex="-1"></a>      best_IVSA_vars <span class="ot">=</span> <span class="fu">c</span>(best_IVSA_vars, best_var)</span>
<span id="cb35-78"><a href="#cb35-78" aria-hidden="true" tabindex="-1"></a>      IVSA_vars_unselected <span class="ot">=</span> IVSA_vars_unselected[<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(IVSA_vars_unselected <span class="sc">==</span> best_var)]</span>
<span id="cb35-79"><a href="#cb35-79" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb35-80"><a href="#cb35-80" aria-hidden="true" tabindex="-1"></a>    starting_cancor <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_novelty_IVSA[noNA_rows,best_novelty_vars], DO_novelty_IVSA[noNA_rows,best_IVSA_vars],</span>
<span id="cb35-81"><a href="#cb35-81" aria-hidden="true" tabindex="-1"></a>                             <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb35-82"><a href="#cb35-82" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb35-83"><a href="#cb35-83" aria-hidden="true" tabindex="-1"></a>    not_optimized <span class="ot">=</span> <span class="cn">FALSE</span></span>
<span id="cb35-84"><a href="#cb35-84" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb35-85"><a href="#cb35-85" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb35-86"><a href="#cb35-86" aria-hidden="true" tabindex="-1"></a>noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_novelty_IVSA[,<span class="fu">c</span>(best_novelty_vars, best_IVSA_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb35-87"><a href="#cb35-87" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-88"><a href="#cb35-88" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Best novelty variables:</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="fu">paste</span>(best_novelty_vars, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>, <span class="at">collapse =</span> <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>),<span class="st">&quot;</span><span class="sc">\n\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Best novelty variables:
OFA_total_distance_traveled_in_perimeter_batch_ranknorm
LD_pct_ambulatory_time_in_light_batch_ranknorm
OFA_total_distance_traveled_in_corner_batch_ranknorm
LD_pct_ambulatory_Counts_in_light_batch_ranknorm
LD_total_transitions_batch_ranknorm</code></pre>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Best IVSA variables:</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="fu">paste</span>(best_IVSA_vars, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>, <span class="at">collapse =</span> <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>), <span class="st">&quot;</span><span class="sc">\n\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Best IVSA variables:
RI_IEI_Mean_Inf_Total_s02_batch_ranknorm
RI_IEI_Var_ALP_Total_s02_batch_ranknorm
AQ_SessionsToAcquisition
DR_IEI_CoV_ALPdTO_Total_1p0mgkgX2_batch_ranknorm
DR_ILP_Total_1p0mgkg_batch_ranknorm
RI_Inf_Total_s02_batch_ranknorm
DR_IEI_Var_ALPdTO_Total_0p32mgkg_batch_ranknorm
DR_Inf_Total_1p0mgkg_batch_ranknorm
RI_IEI_CoV_ILP_Total_s02_batch_ranknorm
RI_IEI_CoV_ILP_Total_s01_batch_ranknorm
DR_Inf_Total_1p0mgkgX2_batch_ranknorm
RI_ALPdTO_Total_s02_batch_ranknorm
DR_ILPdTO_TfASoTD_1p0mgkg_batch_ranknorm
DR_ILP_TfASoTD_0p32mgkg_batch_ranknorm
DR_IEI_CoV_ILPdTO_Total_0p1mgkg_batch_ranknorm
DR_IEI_Mean_Inf_Total_1p0mgkg_batch_ranknorm</code></pre>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;First canonical correlate magnitude: &quot;</span>, <span class="fu">round</span>(starting_cancor<span class="sc">$</span>cor[<span class="dv">1</span>], <span class="dv">4</span>), <span class="st">&quot;</span><span class="sc">\n\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>First canonical correlate magnitude: 0.4402</code></pre>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb41-1"><a href="#cb41-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Canonical correlation trained on &quot;</span>, <span class="fu">length</span>(noNA_rows), <span class="st">&quot; complete observations.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Canonical correlation trained on 473 complete observations.</code></pre>
<p>Adjusting this canonical correlation for sex.</p>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1" aria-hidden="true" tabindex="-1"></a>DO_novelty_IVSA<span class="sc">$</span>Sex <span class="ot">=</span> <span class="fu">ifelse</span>(DO_novelty_IVSA<span class="sc">$</span>DE_Sex <span class="sc">==</span> <span class="st">&quot;M&quot;</span>, <span class="dv">1</span>, <span class="dv">0</span>)</span>
<span id="cb43-2"><a href="#cb43-2" aria-hidden="true" tabindex="-1"></a>novelty_cancor_vars_ALL <span class="ot">=</span> <span class="fu">c</span>(best_novelty_vars,<span class="st">&quot;Sex&quot;</span>)</span>
<span id="cb43-3"><a href="#cb43-3" aria-hidden="true" tabindex="-1"></a>IVSA_cancor_vars_ALL <span class="ot">=</span> <span class="fu">c</span>(best_IVSA_vars,<span class="st">&quot;Sex&quot;</span>)</span>
<span id="cb43-4"><a href="#cb43-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb43-5"><a href="#cb43-5" aria-hidden="true" tabindex="-1"></a>IVSA_cancor_rows_ALL <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_novelty_IVSA[,<span class="fu">c</span>(novelty_cancor_vars_ALL,IVSA_cancor_vars_ALL)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb43-6"><a href="#cb43-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb43-7"><a href="#cb43-7" aria-hidden="true" tabindex="-1"></a>IVSA_novelty_cancor_ALL <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_novelty_IVSA[IVSA_cancor_rows_ALL,IVSA_cancor_vars_ALL],</span>
<span id="cb43-8"><a href="#cb43-8" aria-hidden="true" tabindex="-1"></a>                                 DO_novelty_IVSA[IVSA_cancor_rows_ALL,novelty_cancor_vars_ALL],</span>
<span id="cb43-9"><a href="#cb43-9" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb43-10"><a href="#cb43-10" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="st">&quot;The first canonical correlate of these IVSA variables to novelty variables has a magnitude of &quot;</span>, </span>
<span id="cb43-11"><a href="#cb43-11" aria-hidden="true" tabindex="-1"></a>      <span class="fu">round</span>(IVSA_novelty_cancor_ALL<span class="sc">$</span>cor[<span class="dv">2</span>], <span class="at">digits =</span> <span class="dv">3</span>), </span>
<span id="cb43-12"><a href="#cb43-12" aria-hidden="true" tabindex="-1"></a>      <span class="st">&quot;.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;The first canonical correlate of these IVSA variables to novelty variables has a magnitude of 0.435.&quot;</code></pre>
<div class="sourceCode" id="cb45"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="st">&quot;This canonical correlation is based upon &quot;</span>,</span>
<span id="cb45-2"><a href="#cb45-2" aria-hidden="true" tabindex="-1"></a>      <span class="fu">length</span>(IVSA_cancor_rows_ALL), <span class="st">&quot; complete observations.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;This canonical correlation is based upon 473 complete observations.&quot;</code></pre>
<p>This is an adequate fit for the canonical correlation. Plotting the
variables.</p>
<div class="sourceCode" id="cb47"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb47-1"><a href="#cb47-1" aria-hidden="true" tabindex="-1"></a>IVSA_cancor <span class="ot">=</span> <span class="fu">as.matrix</span>(DO_novelty_IVSA[IVSA_cancor_rows_ALL,IVSA_cancor_vars_ALL]) <span class="sc">%*%</span> IVSA_novelty_cancor_ALL<span class="sc">$</span>xcoef[,<span class="dv">2</span>]</span>
<span id="cb47-2"><a href="#cb47-2" aria-hidden="true" tabindex="-1"></a>novelty_cancor <span class="ot">=</span> <span class="fu">as.matrix</span>(DO_novelty_IVSA[IVSA_cancor_rows_ALL,novelty_cancor_vars_ALL]) <span class="sc">%*%</span> IVSA_novelty_cancor_ALL<span class="sc">$</span>ycoef[,<span class="dv">2</span>]</span>
<span id="cb47-3"><a href="#cb47-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-4"><a href="#cb47-4" aria-hidden="true" tabindex="-1"></a>IVSA_novelty_cancor_ALL_df <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">IVSA_cancor =</span> IVSA_cancor, <span class="at">novelty_cancor =</span> novelty_cancor)</span>
<span id="cb47-5"><a href="#cb47-5" aria-hidden="true" tabindex="-1"></a>IVSA_novelty_cancor_ALL_df<span class="sc">$</span>Sex <span class="ot">=</span> DO_novelty_IVSA[<span class="fu">row.names</span>(IVSA_novelty_cancor_ALL_df),<span class="st">&quot;DE_Sex&quot;</span>]</span>
<span id="cb47-6"><a href="#cb47-6" aria-hidden="true" tabindex="-1"></a>IVSA_novelty_cancor_ALL_df<span class="sc">$</span>Subject <span class="ot">=</span> <span class="fu">as.numeric</span>(<span class="fu">as.character</span>(<span class="fu">row.names</span>(IVSA_novelty_cancor_ALL_df)))</span>
<span id="cb47-7"><a href="#cb47-7" aria-hidden="true" tabindex="-1"></a>IVSA_novelty_cancor_ALL_df<span class="sc">$</span>Acquired <span class="ot">=</span> DO_novelty_IVSA[<span class="fu">row.names</span>(IVSA_novelty_cancor_ALL_df),<span class="st">&quot;DE_Exit_Reason&quot;</span>]</span>
<span id="cb47-8"><a href="#cb47-8" aria-hidden="true" tabindex="-1"></a>IVSA_novelty_cancor_lm <span class="ot">=</span> <span class="fu">lm</span>(IVSA_cancor <span class="sc">~</span> novelty_cancor, <span class="at">data =</span> IVSA_novelty_cancor_ALL_df)</span>
<span id="cb47-9"><a href="#cb47-9" aria-hidden="true" tabindex="-1"></a>novelty_IVSA_ALL_cancor_plot <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> IVSA_novelty_cancor_ALL_df, <span class="fu">aes</span>(<span class="at">x =</span> novelty_cancor, <span class="at">y =</span> IVSA_cancor, <span class="at">color =</span> Sex)) <span class="sc">+</span></span>
<span id="cb47-10"><a href="#cb47-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">size =</span> (pt_to_mm <span class="sc">*</span> <span class="fl">1.5</span>)) <span class="sc">+</span></span>
<span id="cb47-11"><a href="#cb47-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_abline</span>(<span class="at">slope =</span> IVSA_novelty_cancor_lm<span class="sc">$</span>coefficients[<span class="st">&quot;novelty_cancor&quot;</span>], </span>
<span id="cb47-12"><a href="#cb47-12" aria-hidden="true" tabindex="-1"></a>              <span class="at">intercept =</span> IVSA_novelty_cancor_lm<span class="sc">$</span>coefficients[<span class="st">&quot;(Intercept)&quot;</span>],</span>
<span id="cb47-13"><a href="#cb47-13" aria-hidden="true" tabindex="-1"></a>              <span class="at">size =</span> (<span class="dv">1</span> <span class="sc">*</span> pt_to_mm)) <span class="sc">+</span></span>
<span id="cb47-14"><a href="#cb47-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#3399CC&quot;</span>,<span class="st">&quot;#66CC66&quot;</span>)) <span class="sc">+</span></span>
<span id="cb47-15"><a href="#cb47-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Novelty Canonical Correlate&quot;</span>) <span class="sc">+</span></span>
<span id="cb47-16"><a href="#cb47-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;IVSA Canonical Correlate&quot;</span>)</span></code></pre></div>
<pre><code>Warning: Using `size` aesthetic for lines was deprecated in ggplot2 3.4.0.
ℹ Please use `linewidth` instead.
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
generated.</code></pre>
<div class="sourceCode" id="cb49"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb49-1"><a href="#cb49-1" aria-hidden="true" tabindex="-1"></a>novelty_IVSA_ALL_cancor_plot <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">panel.grid =</span> <span class="fu">element_line</span>(<span class="at">color =</span> <span class="st">&quot;#FFFFFF&quot;</span>))</span></code></pre></div>
<p><img src="figure/12_DO_IVSA_novelty_cancor.Rmd/unnamed-chunk-11-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
e8251da
</td>
<td>
Michael C. Saul
</td>
<td>
2024-03-29
</td>
</tr>
<tr>
<td>
2cb3df3
</td>
<td>
Michael C. Saul
</td>
<td>
2024-03-29
</td>
</tr>
<tr>
<td>
b194018
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-08
</td>
</tr>
<tr>
<td>
1c4a09a
</td>
<td>
Michael C. Saul
</td>
<td>
2021-01-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="making-predictions" class="section level4">
<h4>Making Predictions</h4>
<p>Using canonical correlations and logistic regression to make
predictions.</p>
<div class="sourceCode" id="cb50"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb50-1"><a href="#cb50-1" aria-hidden="true" tabindex="-1"></a>DO_novelty<span class="sc">$</span>DE_Sex <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^([MF]).*$&quot;</span>,<span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>,DO_novelty<span class="sc">$</span>Sex)</span>
<span id="cb50-2"><a href="#cb50-2" aria-hidden="true" tabindex="-1"></a>DO_novelty<span class="sc">$</span>Sex <span class="ot">=</span> <span class="fu">ifelse</span>(DO_novelty<span class="sc">$</span>DE_Sex <span class="sc">==</span> <span class="st">&quot;M&quot;</span>, <span class="dv">1</span>, <span class="dv">0</span>)</span>
<span id="cb50-3"><a href="#cb50-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-4"><a href="#cb50-4" aria-hidden="true" tabindex="-1"></a>DO_predict_rows_ALL <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_novelty[,novelty_cancor_vars_ALL])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb50-5"><a href="#cb50-5" aria-hidden="true" tabindex="-1"></a>IVSA_prediction_ALL <span class="ot">=</span> <span class="fu">as.matrix</span>(DO_novelty[DO_predict_rows_ALL,novelty_cancor_vars_ALL]) <span class="sc">%*%</span> IVSA_novelty_cancor_ALL<span class="sc">$</span>ycoef[,<span class="dv">2</span>]</span>
<span id="cb50-6"><a href="#cb50-6" aria-hidden="true" tabindex="-1"></a>IVSA_prediction_ALL <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">row.names =</span> <span class="fu">row.names</span>(IVSA_prediction_ALL),</span>
<span id="cb50-7"><a href="#cb50-7" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">Mouse_ID =</span> <span class="fu">row.names</span>(IVSA_prediction_ALL),</span>
<span id="cb50-8"><a href="#cb50-8" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">IVSA_novelty_cancor =</span> IVSA_prediction_ALL,</span>
<span id="cb50-9"><a href="#cb50-9" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">Sex =</span> DO_novelty[<span class="fu">row.names</span>(IVSA_prediction_ALL),<span class="st">&quot;Sex&quot;</span>],</span>
<span id="cb50-10"><a href="#cb50-10" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">Generation =</span> DO_novelty[<span class="fu">row.names</span>(IVSA_prediction_ALL),<span class="st">&quot;Generation&quot;</span>],</span>
<span id="cb50-11"><a href="#cb50-11" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span></code></pre></div>
</div>
</div>
<div id="output" class="section level3">
<h3>Output</h3>
<p>Saving RTG predictions as output.</p>
<div class="sourceCode" id="cb51"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb51-1"><a href="#cb51-1" aria-hidden="true" tabindex="-1"></a>today <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">today</span>())</span>
<span id="cb51-2"><a href="#cb51-2" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(IVSA_prediction_ALL, <span class="at">file =</span> <span class="fu">paste0</span>(<span class="st">&quot;./data/DO_IVSA_prediction_a3_ALL_cancor_&quot;</span>,today,<span class="st">&quot;.RDS&quot;</span>))</span></code></pre></div>
<p>Saving canonical correlations and prediction-training data as
output.</p>
<div class="sourceCode" id="cb52"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb52-1"><a href="#cb52-1" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(IVSA_novelty_cancor_ALL, <span class="fu">paste0</span>(<span class="st">&quot;./data/DO_IVSA_novelty_cancor_ALL_&quot;</span>,today,<span class="st">&quot;.RDS&quot;</span>))</span>
<span id="cb52-2"><a href="#cb52-2" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_novelty_IVSA_raw, <span class="fu">paste0</span>(<span class="st">&quot;./data/DO_IVSA_novelty_data_&quot;</span>,today,<span class="st">&quot;.RDS&quot;</span>))</span>
<span id="cb52-3"><a href="#cb52-3" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_novelty, <span class="fu">paste0</span>(<span class="st">&quot;./data/DO_novelty_data_&quot;</span>,today,<span class="st">&quot;.RDS&quot;</span>))</span>
<span id="cb52-4"><a href="#cb52-4" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_IVSA, <span class="fu">paste0</span>(<span class="st">&quot;./data/DO_IVSA_data_&quot;</span>,today,<span class="st">&quot;.RDS&quot;</span>))</span></code></pre></div>
<p>Saving data frames of each analysis as output.</p>
<div class="sourceCode" id="cb53"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb53-1"><a href="#cb53-1" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(IVSA_novelty_cancor_ALL_df, <span class="fu">paste0</span>(<span class="st">&quot;./data/DO_IVSA_all_cancor_df_&quot;</span>,today,<span class="st">&quot;.RDS&quot;</span>))</span></code></pre></div>
<p>The output data are saved in <code>./data/</code>.</p>
</div>
<div id="document-control" class="section level3">
<h3>Document Control</h3>
<p>This document was prepared using RMarkdown in RStudio.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb54"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb54-1"><a href="#cb54-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.6 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/liblapack.so.3

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] ggbeeswarm_0.7.2 ggpubr_0.6.0     ggplot2_3.4.2    gplots_3.1.3    
 [5] tidyr_1.3.0      corrplot_0.92    car_3.1-2        carData_3.0-5   
 [9] lubridate_1.8.0  cowplot_1.1.1    MASS_7.3-57     

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.9         gtools_3.9.4       rprojroot_2.0.3    digest_0.6.30     
 [5] utf8_1.2.2         R6_2.5.1           backports_1.4.1    evaluate_0.17     
 [9] highr_0.9          pillar_1.9.0       rlang_1.1.1        rstudioapi_0.14   
[13] whisker_0.4        jquerylib_0.1.4    rmarkdown_2.17     labeling_0.4.2    
[17] stringr_1.5.0      munsell_0.5.0      broom_1.0.1        vipor_0.4.5       
[21] compiler_4.2.1     httpuv_1.6.6       xfun_0.39          pkgconfig_2.0.3   
[25] htmltools_0.5.3    tidyselect_1.2.0   tibble_3.2.1       workflowr_1.7.0   
[29] fansi_1.0.3        crayon_1.5.2       dplyr_1.1.2        withr_2.5.0       
[33] later_1.3.0        bitops_1.0-7       grid_4.2.1         jsonlite_1.8.3    
[37] gtable_0.3.1       lifecycle_1.0.3    git2r_0.32.0       magrittr_2.0.3    
[41] scales_1.2.1       KernSmooth_2.23-20 cli_3.4.1          stringi_1.7.8     
[45] cachem_1.0.6       farver_2.1.1       ggsignif_0.6.4     fs_1.5.2          
[49] promises_1.2.0.1   bslib_0.4.0        generics_0.1.3     vctrs_0.6.2       
[53] tools_4.2.1        beeswarm_0.4.0     glue_1.6.2         purrr_1.0.1       
[57] abind_1.4-5        fastmap_1.1.0      yaml_2.3.6         colorspace_2.0-3  
[61] caTools_1.18.2     rstatix_0.7.2      knitr_1.42         sass_0.4.2        </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
