<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Michael C. Saul (michael.saul [at] jax.org)" />


<title>DO Mapping Interpretation for RTG Analysis 2</title>

<script src="site_libs/header-attrs-2.17/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Diversity Outbred Reference Traits Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">DO Mapping Interpretation for RTG Analysis
2</h1>
<h4 class="author">Michael C. Saul (michael.saul [at] jax.org)</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-11-28
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rta/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20200501code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20200501)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20200501code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20200501)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongb37c551">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong> b37c551
</a>
</p>
</div>
<div id="strongRepositoryversionstrongb37c551"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version b37c551.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    code/.Rapp.history
    Ignored:    data/.Rhistory
    Ignored:    data/RDS/
    Ignored:    run/

Untracked files:
    Untracked:  LICENSE
    Untracked:  Rplot.pdf
    Untracked:  SENSI_chr7_snps.R
    Untracked:  Untitled.R
    Untracked:  analysis/17_mapping_interpretation_old.Rmd
    Untracked:  analysis/temp.txt
    Untracked:  code/DO_IVSA_SALINE_pleiotropy.R
    Untracked:  code/DO_IVSA_SENS_pleiotropy.R
    Untracked:  code/DO_SENS_SALINE_pleiotropy.R
    Untracked:  code/DO_eQTL_perms.R
    Untracked:  code/DO_eQTL_score_calc.R
    Untracked:  code/SENSI_chr7_snps.R
    Untracked:  code/STAR_RSEM_416.sh
    Untracked:  code/eQTL_plotting_DO_striatum_416.R
    Untracked:  code/ggplot_pub_themes.R
    Untracked:  code/pleitropy_analysis.sbatch
    Untracked:  code/pleitropy_saline_analysis.sbatch
    Untracked:  code/pleitropy_sensi_saline_analysis.sbatch
    Untracked:  code/rta_mapping_figure_leftover.R
    Untracked:  data/20_elastic_mv_results.csv
    Untracked:  data/20_elasticnet_predictions.RDS
    Untracked:  data/DO_416_DGEList.RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 3).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 4).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_DGEList_unfiltered.RDS
    Untracked:  data/DO_416_ERCC_DGEList.RDS
    Untracked:  data/DO_416_ERCC_cpm.RDS
    Untracked:  data/DO_416_normrt.RDS
    Untracked:  data/DO_416_pheno.RDS
    Untracked:  data/DO_416_pheno_allgenes.RDS
    Untracked:  data/DO_416_voom (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom.RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom_allgenes.RDS
    Untracked:  data/DO_416_voom_cpresiduals.RDS
    Untracked:  data/DO_AQ_active_CoV_toptable_annotated.RDS
    Untracked:  data/DO_AQ_lever_ratio_toptable_annotated.RDS
    Untracked:  data/DO_AQ_mean_inf_toptable_annotated.RDS
    Untracked:  data/DO_AQ_rate_toptable_annotated.RDS
    Untracked:  data/DO_IVSA_SENS_pleio_lods.RDS
    Untracked:  data/DO_IVSA_SENS_pleio_pvl.RDS
    Untracked:  data/DO_IVSA_acquisition_cancor_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_glm.RDS
    Untracked:  data/DO_IVSA_all_cancor_df.RDS
    Untracked:  data/DO_IVSA_data.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_ALL.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_AQ.RDS
    Untracked:  data/DO_IVSA_novelty_data.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_old.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_a1_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_a2_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a4_male_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a5_female_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_raw_data.RDS
    Untracked:  data/DO_IVSA_transformed_data.RDS
    Untracked:  data/DO_LIMS_data.RDS
    Untracked:  data/DO_RTG_a1_new.qtl.RData
    Untracked:  data/DO_RTG_a1_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a1_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a2_new.qtl.RData
    Untracked:  data/DO_RTG_a2_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a2_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a3_new.qtl.RData
    Untracked:  data/DO_RTG_a3_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_01.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_02.RData
    Untracked:  data/DO_RTG_a4_new.qtl.RData
    Untracked:  data/DO_RTG_a5_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_project_df_for_mapping.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_2020-11-06.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_a2.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_sexspecific.RDS
    Untracked:  data/DO_cocaine_measures_data.RDS
    Untracked:  data/DO_demographic_data_df.RDS
    Untracked:  data/DO_eigengene_significance.RDS
    Untracked:  data/DO_eigengenes.RDS
    Untracked:  data/DO_elasticnet_GEX_pvalue_histograms.RData
    Untracked:  data/DO_highcor_cpresiduals_datatable.RDS
    Untracked:  data/DO_holeboard_data_df.RDS
    Untracked:  data/DO_holeboard_summarized.RDS
    Untracked:  data/DO_lightdark_data_df.RDS
    Untracked:  data/DO_mQTL_peaks.RDS
    Untracked:  data/DO_mQTL_scan1s.RDS
    Untracked:  data/DO_module_membership.RDS
    Untracked:  data/DO_novelplace_data_df.RDS
    Untracked:  data/DO_novelty_COC_cancor.RDS
    Untracked:  data/DO_novelty_SAL_cancor.RDS
    Untracked:  data/DO_novelty_data.RDS
    Untracked:  data/DO_novelty_response_data.RDS
    Untracked:  data/DO_novelty_response_data_for_RTG.RDS
    Untracked:  data/DO_novelty_response_data_with_transform.csv
    Untracked:  data/DO_novelty_saline_cocaine_cancor.RDS
    Untracked:  data/DO_openfield_data_df.RDS
    Untracked:  data/DO_paraclique_data.el
    Untracked:  data/DO_paraclique_eigengenes.RDS
    Untracked:  data/DO_paraclique_significance_elasticnet.RDS
    Untracked:  data/DO_paracliques.RDS
    Untracked:  data/DO_paracliques_significance.RDS
    Untracked:  data/DO_ploidy_calls.RDS
    Untracked:  data/DO_pmap.RDS
    Untracked:  data/DO_pvalue_histograms.RData
    Untracked:  data/DO_rtg_a1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a3_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m3_toptable_annotated.RDS
    Untracked:  data/DO_saline.qtl.RData
    Untracked:  data/DO_saline_measures_data.RDS
    Untracked:  data/DO_sensi_cocaine.qtl.RData
    Untracked:  data/DO_str_2016_eQTL.RData
    Untracked:  data/DO_str_2016_gm4qtl2.zip
    Untracked:  data/DO_str_2016_gm4qtl2_intensities.fst
    Untracked:  data/DO_voom_cpresiduals_NArm.RDS
    Untracked:  data/DO_wgcna.RDS
    Untracked:  data/IVSA_RTG_a3_blup.pdf
    Untracked:  data/IVSA_h2_2019-11-18.tsv
    Untracked:  data/IVSA_metadata_sheet.RDS
    Untracked:  data/ME16_edges.tsv
    Untracked:  data/Mus_musculus.GRCm38.94.parsed.RDS
    Untracked:  data/Paraclique_19_MF_AMIGO.txt
    Untracked:  data/RTG_QTL_peaks.RDS
    Untracked:  data/RTG_figure.RDS
    Untracked:  data/SENSI_chr7_snps.RData
    Untracked:  data/SENS_RTG/
    Untracked:  data/Tyr2_genotypes.RDS
    Untracked:  data/Tyr_genotypes.RDS
    Untracked:  data/blup_chr7_DO_RTG_a3.RDS
    Untracked:  data/cc_variants/
    Untracked:  data/chr18_QTL_results.RDS
    Untracked:  data/chr4_QTL_results.RDS
    Untracked:  data/chr7_a3_QTL_results.RDS
    Untracked:  data/cis/
    Untracked:  data/eQTL_annotated_peaks.RDS
    Untracked:  data/eQTL_plots.RData
    Untracked:  data/figures/
    Untracked:  data/founders_key/
    Untracked:  data/gigamuga/
    Untracked:  data/gwas/
    Untracked:  data/info/
    Untracked:  data/m2G.a2.RTG.chr4snp.RData
    Untracked:  data/mapping/
    Untracked:  data/mapping_from_hao/
    Untracked:  data/mpd_measures_saline.txt
    Untracked:  data/mpd_measures_saline_cocaine.txt
    Untracked:  data/perms/
    Untracked:  data/qtl2/
    Untracked:  data/raw/
    Untracked:  data/resources/
    Untracked:  data/tads_loops/
    Untracked:  data/transcripts/
    Untracked:  data/variants/
    Untracked:  logs/
    Untracked:  new_RTG_a3.jpeg
    Untracked:  output/DO_module_membership.tsv
    Untracked:  output/DO_paraclique_df.tsv
    Untracked:  output/RDS/
    Untracked:  output/RTG_out/
    Untracked:  output/docs_backup_2020-07-30.tar.gz
    Untracked:  output/elastic_net_out/
    Untracked:  output/mapping_figure.pdf
    Untracked:  output/mapping_plot.pdf
    Untracked:  output/mediation_all_rta_figure.pdf
    Untracked:  output/mediation_all_rta_figure_with_eed.pdf
    Untracked:  rstudio_rta.sif
    Untracked:  singularity/
    Untracked:  tmp/
    Untracked:  var/

Unstaged changes:
    Deleted:    .Rprofile
    Modified:   .gitignore
    Modified:   README.md
    Modified:   analysis/42_RTG_mapping.Rmd
    Modified:   analysis/_site.yml

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/17_mapping_interpretation.Rmd</code>) and HTML
(<code>docs/17_mapping_interpretation.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
b37c551
</td>
<td>
Michael C. Saul
</td>
<td>
2023-11-28
</td>
<td>
Committing most recent changes (adding new plots)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
aa52960
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
a9a19d5
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-11
</td>
<td>
Adding in new sensitization to cocaine data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
2485d7d
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
78e83fe
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-14
</td>
<td>
Committing for Sumner and integrating new analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
973cb1c
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-09
</td>
<td>
Recommitting for Sumner plus adding new analyses
</td>
</tr>
<tr>
<td>
html
</td>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
<td>
Rolling back to working version.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
1eb2885
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
34bb611
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-21
</td>
<td>
Committing table of QTL results
</td>
</tr>
<tr>
<td>
html
</td>
<td>
c86ee90
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
0702711
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-20
</td>
<td>
Committed modifications to mapping analyses figures and fix to chr18
mediation analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
9b43495
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
168cb84
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-19
</td>
<td>
Committing chr18 peak analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
da1922f
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
61a60a0
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-17
</td>
<td>
Committing mapping figure
</td>
</tr>
<tr>
<td>
html
</td>
<td>
f9c5856
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
322facd
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-16
</td>
<td>
Committing work on variants that might contribute to differences in TAD
boundary
</td>
</tr>
<tr>
<td>
html
</td>
<td>
bf1ce2e
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
b14409e
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-15
</td>
<td>
Commiting new plots with shared synteny
</td>
</tr>
<tr>
<td>
html
</td>
<td>
a6193f0
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
4944529
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-14
</td>
<td>
workflowr::wflow_publish("./analysis/17_mapping_interpretation.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
6fb042a
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
b6a22e2
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Re-committed information about discussion to fix plot order error
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
883b0eb
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Re-committed information about discussion to fix plot order error
</td>
</tr>
<tr>
<td>
html
</td>
<td>
3cafd1b
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
9519172
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Committed information about discussion
</td>
</tr>
<tr>
<td>
html
</td>
<td>
2a2643d
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
497c8be
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
workflowr::wflow_publish("./analysis/17_mapping_interpretation.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
601e3ff
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
0b25f4e
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Commit changes to images (corrected to use the knitr::include_graphics()
</td>
</tr>
<tr>
<td>
html
</td>
<td>
ae7f4f9
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
945f4e8
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Commit changes to images (as jpg, corrected file path)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
74a70db
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
4296fe8
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Committing images
</td>
</tr>
<tr>
<td>
html
</td>
<td>
c52718d
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
8986fb3
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-13
</td>
<td>
Initial commit of QTL interpretation.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="background" class="section level1">
<h1>Background</h1>
<div id="motivation" class="section level2">
<h2>Motivation</h2>
<div id="purpose-of-analysis" class="section level3">
<h3>Purpose of analysis</h3>
<p>This script is used to interpret the RTG mapping work results.</p>
<p>The goal of this analysis is to resolve these QTL mapping data to one
(or a few) genes.</p>
</div>
</div>
</div>
<div id="analysis" class="section level1">
<h1>Analysis</h1>
<div id="r-libraries" class="section level3">
<h3>R libraries</h3>
<p>Calling R libraries necessary for this analysis.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;tidyverse&quot;</span>)</span></code></pre></div>
<pre><code>── Attaching packages ─────────────────────────────────────── tidyverse 1.3.2 ──
✔ ggplot2 3.4.2     ✔ purrr   1.0.1
✔ tibble  3.2.1     ✔ dplyr   1.1.2
✔ tidyr   1.3.0     ✔ stringr 1.5.0
✔ readr   2.1.3     ✔ forcats 0.5.2
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()</code></pre>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggplot2&quot;</span>)</span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggrepel&quot;</span>)</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;biomaRt&quot;</span>)</span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;qtl2&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;qtl2&#39;

The following object is masked from &#39;package:readr&#39;:

    read_csv</code></pre>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;grid&quot;</span>)</span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;cowplot&quot;</span>)</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggplotify&quot;</span>)</span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;data.table&quot;</span>)</span></code></pre></div>
<pre><code>
Attaching package: &#39;data.table&#39;

The following objects are masked from &#39;package:dplyr&#39;:

    between, first, last

The following object is masked from &#39;package:purrr&#39;:

    transpose</code></pre>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;edgeR&quot;</span>)</span></code></pre></div>
<pre><code>Loading required package: limma</code></pre>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;limma&quot;</span>)</span></code></pre></div>
</div>
<div id="getting-publication-themes" class="section level3">
<h3>Getting publication themes</h3>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="fu">source</span>(<span class="st">&quot;./code/ggplot_pub_themes.R&quot;</span>)</span></code></pre></div>
<pre><code>Warning: The `size` argument of `element_line()` is deprecated as of ggplot2 3.4.0.
ℹ Please use the `linewidth` argument instead.
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
generated.</code></pre>
</div>
<div id="norm_rank_transform-function" class="section level3">
<h3><code>norm_rank_transform()</code> function</h3>
<p>Getting a normal rank transformation function that can handle
<code>NA</code> values.</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>norm_rank_transform <span class="ot">=</span> <span class="cf">function</span>(x, <span class="at">c =</span> (<span class="dv">0</span>)) {</span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stopifnot</span>(<span class="fu">is.numeric</span>(x) <span class="sc">&amp;</span> <span class="fu">is.vector</span>(x))</span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>  x_noNA <span class="ot">=</span> <span class="fu">which</span>(<span class="sc">!</span><span class="fu">is.na</span>(x))</span>
<span id="cb12-4"><a href="#cb12-4" aria-hidden="true" tabindex="-1"></a>  N <span class="ot">=</span> <span class="fu">length</span>(x_noNA)</span>
<span id="cb12-5"><a href="#cb12-5" aria-hidden="true" tabindex="-1"></a>  x[x_noNA] <span class="ot">=</span> <span class="fu">qnorm</span>((<span class="fu">rank</span>(x[x_noNA], <span class="at">ties.method =</span> <span class="st">&quot;average&quot;</span>) <span class="sc">-</span> c) <span class="sc">/</span> (N <span class="sc">-</span> (<span class="dv">2</span> <span class="sc">*</span> c) <span class="sc">+</span> <span class="dv">1</span>))</span>
<span id="cb12-6"><a href="#cb12-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(x)</span>
<span id="cb12-7"><a href="#cb12-7" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
</div>
<div id="importing-data" class="section level3">
<h3>Importing data</h3>
<p>Importing data.</p>
<p>First, importing eQTL peaks.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Importing significant eQTL results</span></span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a>DO_eQTL_results <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/eQTL_annotated_peaks.RDS&quot;</span>)</span></code></pre></div>
<p>Next, importing SNP data from the interval around the chr7 QTL from
Hao He.</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./data/DO_RTG_a3_new_mapping_1000x_01.RData&quot;</span>)</span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>RTG_a3_chr7_SNPs <span class="ot">=</span> <span class="fu">as.data.frame</span>(m2.sigqtl.snps<span class="sc">$</span>RTG_a3[[<span class="dv">2</span>]]<span class="sc">$</span>lod)</span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a>RTG_a3_chr7_info <span class="ot">=</span> <span class="fu">as.data.frame</span>(m2.sigqtl.snps<span class="sc">$</span>RTG_a3[[<span class="dv">2</span>]]<span class="sc">$</span>snpinfo)</span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(RTG_a3_chr7_info) <span class="ot">=</span> RTG_a3_chr7_info<span class="sc">$</span>snp_id</span>
<span id="cb14-5"><a href="#cb14-5" aria-hidden="true" tabindex="-1"></a>RTG_a3_chr7_SNPs <span class="ot">=</span> <span class="fu">cbind</span>(RTG_a3_chr7_info[<span class="fu">row.names</span>(RTG_a3_chr7_SNPs),], RTG_a3_chr7_SNPs)</span>
<span id="cb14-6"><a href="#cb14-6" aria-hidden="true" tabindex="-1"></a><span class="fu">rm</span>(<span class="at">list =</span> <span class="fu">c</span>(<span class="st">&quot;RTG_a3_chr7_info&quot;</span>,<span class="st">&quot;m2.sigqtl.genes&quot;</span>,<span class="st">&quot;m2.sigqtl.snps&quot;</span>))</span></code></pre></div>
<p>Loading sensi SNPs for SNP correlations.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./data/SENSI_chr7_snps.RData&quot;</span>)</span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a>chr7_cocaine_snps_lod <span class="ot">=</span> <span class="fu">as.data.frame</span>(chr7_cocaine_snps<span class="sc">$</span>lod)</span>
<span id="cb15-3"><a href="#cb15-3" aria-hidden="true" tabindex="-1"></a>chr7_saline_snps_lod  <span class="ot">=</span> <span class="fu">as.data.frame</span>(chr7_saline_snps<span class="sc">$</span>lod)</span></code></pre></div>
<p>Next, importing <a
href="ftp://ftp.jax.org/dgatti/CC_SNP_DB/cc_variants.sqlite">CC variants
file</a> (downloaded 2020-07-09). Creating a query function.</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a>query_func <span class="ot">=</span> <span class="fu">create_variant_query_func</span>(<span class="st">&quot;./data/cc_variants/cc_variants.sqlite&quot;</span>)</span></code></pre></div>
<p>Next, loading in the eQTL data</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./data/DO_str_2016_eQTL.RData&quot;</span>)</span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a><span class="co"># DO_pheno = DO_str_2016_cross$pheno</span></span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>DO_covar <span class="ot">=</span> DO_str_2016_cross<span class="sc">$</span>covar</span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>DO_unfiltered_DGE <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_416_DGEList_unfiltered.RDS&quot;</span>)</span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>DO_unfiltered_DGE <span class="ot">=</span> <span class="fu">calcNormFactors</span>(DO_unfiltered_DGE)</span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a>DO_pheno <span class="ot">=</span> <span class="fu">t</span>(<span class="fu">voom</span>(DO_unfiltered_DGE)<span class="sc">$</span>E)</span>
<span id="cb17-8"><a href="#cb17-8" aria-hidden="true" tabindex="-1"></a><span class="fu">rm</span>(<span class="at">list =</span> <span class="fu">c</span>(<span class="st">&quot;DO_unfiltered_DGE&quot;</span>))</span>
<span id="cb17-9"><a href="#cb17-9" aria-hidden="true" tabindex="-1"></a>DO_pheno <span class="ot">=</span> DO_pheno[<span class="fu">gsub</span>(<span class="st">&quot;^(</span><span class="sc">\\</span><span class="st">d{4,5})_.*$&quot;</span>,<span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>,<span class="fu">row.names</span>(DO_covar)),]</span>
<span id="cb17-10"><a href="#cb17-10" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(DO_pheno) <span class="ot">=</span> <span class="fu">row.names</span>(DO_covar)</span>
<span id="cb17-11"><a href="#cb17-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-12"><a href="#cb17-12" aria-hidden="true" tabindex="-1"></a>addcovar_mat <span class="ot">=</span> <span class="fu">model.matrix</span>(<span class="sc">~</span> ngen <span class="sc">+</span> sex, <span class="at">data =</span> DO_covar)[,<span class="sc">-</span><span class="dv">1</span>]</span>
<span id="cb17-13"><a href="#cb17-13" aria-hidden="true" tabindex="-1"></a>addcovar_i <span class="ot">=</span> addcovar_mat</span>
<span id="cb17-14"><a href="#cb17-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-15"><a href="#cb17-15" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;qtl2&quot;</span>)</span>
<span id="cb17-16"><a href="#cb17-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-17"><a href="#cb17-17" aria-hidden="true" tabindex="-1"></a>plot_eQTL_str_2016 <span class="ot">=</span> <span class="cf">function</span>(gene_i, <span class="at">n_cores =</span> <span class="dv">9</span>, <span class="at">label =</span> gene_i, <span class="at">color =</span> <span class="st">&quot;darkblue&quot;</span>) {</span>
<span id="cb17-18"><a href="#cb17-18" aria-hidden="true" tabindex="-1"></a>  pheno_col_i <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">colnames</span>(DO_pheno) <span class="sc">==</span> gene_i)</span>
<span id="cb17-19"><a href="#cb17-19" aria-hidden="true" tabindex="-1"></a>  qtl_cis_i <span class="ot">=</span> <span class="fu">scan1</span>(<span class="at">genoprobs =</span> DO_str_2016_aprobs,</span>
<span id="cb17-20"><a href="#cb17-20" aria-hidden="true" tabindex="-1"></a>                    <span class="at">pheno =</span> DO_pheno[,pheno_col_i, <span class="at">drop =</span> <span class="cn">FALSE</span>],</span>
<span id="cb17-21"><a href="#cb17-21" aria-hidden="true" tabindex="-1"></a>                    <span class="at">kinship =</span> DO_str_2016_kinship,</span>
<span id="cb17-22"><a href="#cb17-22" aria-hidden="true" tabindex="-1"></a>                    <span class="at">addcovar =</span> addcovar_i,</span>
<span id="cb17-23"><a href="#cb17-23" aria-hidden="true" tabindex="-1"></a>                    <span class="at">cores =</span> n_cores)</span>
<span id="cb17-24"><a href="#cb17-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(<span class="fu">plot_scan1</span>(qtl_cis_i, <span class="at">map =</span> DO_str_2016_cross<span class="sc">$</span>pmap, <span class="at">col =</span> color, <span class="at">main =</span> label))</span>
<span id="cb17-25"><a href="#cb17-25" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb17-26"><a href="#cb17-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-27"><a href="#cb17-27" aria-hidden="true" tabindex="-1"></a>plot_coefCC2 <span class="ot">=</span> <span class="cf">function</span>(x,</span>
<span id="cb17-28"><a href="#cb17-28" aria-hidden="true" tabindex="-1"></a>                        map, </span>
<span id="cb17-29"><a href="#cb17-29" aria-hidden="true" tabindex="-1"></a>                        <span class="at">columns =</span> <span class="fu">c</span>(<span class="st">&quot;A&quot;</span>,<span class="st">&quot;B&quot;</span>,<span class="st">&quot;C&quot;</span>,<span class="st">&quot;D&quot;</span>,<span class="st">&quot;E&quot;</span>,<span class="st">&quot;F&quot;</span>,<span class="st">&quot;G&quot;</span>,<span class="st">&quot;H&quot;</span>), </span>
<span id="cb17-30"><a href="#cb17-30" aria-hidden="true" tabindex="-1"></a>                        <span class="at">col =</span>  <span class="fu">c</span>(<span class="st">&quot;#F0E442&quot;</span>, <span class="st">&quot;#555555&quot;</span>, <span class="st">&quot;#E69F00&quot;</span>, <span class="st">&quot;#0072B2&quot;</span>, </span>
<span id="cb17-31"><a href="#cb17-31" aria-hidden="true" tabindex="-1"></a>                                 <span class="st">&quot;#56B4E9&quot;</span>, <span class="st">&quot;#009E73&quot;</span>, <span class="st">&quot;#D55E00&quot;</span>, <span class="st">&quot;#CC79A7&quot;</span>),</span>
<span id="cb17-32"><a href="#cb17-32" aria-hidden="true" tabindex="-1"></a>                        <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">&quot;AJ&quot;</span>, <span class="st">&quot;B6&quot;</span>, <span class="st">&quot;129&quot;</span>, <span class="st">&quot;NOD&quot;</span>, </span>
<span id="cb17-33"><a href="#cb17-33" aria-hidden="true" tabindex="-1"></a>                                   <span class="st">&quot;NZO&quot;</span>, <span class="st">&quot;CAST&quot;</span>, <span class="st">&quot;PWK&quot;</span>, <span class="st">&quot;WSB&quot;</span>),</span>
<span id="cb17-34"><a href="#cb17-34" aria-hidden="true" tabindex="-1"></a>                        <span class="at">legend =</span> <span class="cn">NULL</span>,</span>
<span id="cb17-35"><a href="#cb17-35" aria-hidden="true" tabindex="-1"></a>                        ...) {</span>
<span id="cb17-36"><a href="#cb17-36" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="fu">length</span>(columns) <span class="sc">!=</span> <span class="fu">length</span>(labels)) {</span>
<span id="cb17-37"><a href="#cb17-37" aria-hidden="true" tabindex="-1"></a>    <span class="fu">stop</span>(<span class="st">&quot;Column length different than label length&quot;</span>)</span>
<span id="cb17-38"><a href="#cb17-38" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> <span class="cf">if</span> (<span class="fu">length</span>(<span class="fu">which</span>(columns <span class="sc">%in%</span> <span class="fu">colnames</span>(x))) <span class="sc">!=</span> <span class="fu">length</span>(columns)) {</span>
<span id="cb17-39"><a href="#cb17-39" aria-hidden="true" tabindex="-1"></a>    <span class="fu">stop</span>(<span class="st">&quot;Columns not in coefficient matrix&quot;</span>) </span>
<span id="cb17-40"><a href="#cb17-40" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> <span class="cf">if</span> (<span class="fu">length</span>(col) <span class="sc">!=</span> <span class="fu">length</span>(columns)) {</span>
<span id="cb17-41"><a href="#cb17-41" aria-hidden="true" tabindex="-1"></a>    <span class="fu">stop</span>(<span class="st">&quot;Different number of colors to column length&quot;</span>)</span>
<span id="cb17-42"><a href="#cb17-42" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb17-43"><a href="#cb17-43" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stopifnot</span>(<span class="fu">require</span>(<span class="st">&quot;qtl2&quot;</span>))</span>
<span id="cb17-44"><a href="#cb17-44" aria-hidden="true" tabindex="-1"></a>  x <span class="ot">=</span> x[,columns]</span>
<span id="cb17-45"><a href="#cb17-45" aria-hidden="true" tabindex="-1"></a>  <span class="fu">colnames</span>(x) <span class="ot">=</span> labels</span>
<span id="cb17-46"><a href="#cb17-46" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(<span class="fu">plot_coef</span>(<span class="at">x =</span> x, <span class="at">map =</span> map, <span class="at">col =</span> col, <span class="at">legend =</span> legend, ...))</span>
<span id="cb17-47"><a href="#cb17-47" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb17-48"><a href="#cb17-48" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-49"><a href="#cb17-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-50"><a href="#cb17-50" aria-hidden="true" tabindex="-1"></a>plot_BLUP_str_2016 <span class="ot">=</span> <span class="cf">function</span>(gene_i, chr_i, <span class="at">label =</span> gene_i, <span class="at">n_cores =</span> <span class="dv">9</span>, <span class="at">legendpos =</span> <span class="st">&quot;bottomleft&quot;</span>) {</span>
<span id="cb17-51"><a href="#cb17-51" aria-hidden="true" tabindex="-1"></a>  pheno_col_i <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">colnames</span>(DO_pheno) <span class="sc">==</span> gene_i)</span>
<span id="cb17-52"><a href="#cb17-52" aria-hidden="true" tabindex="-1"></a>  blup_i <span class="ot">=</span> <span class="fu">scan1blup</span>(DO_str_2016_aprobs[,chr_i], </span>
<span id="cb17-53"><a href="#cb17-53" aria-hidden="true" tabindex="-1"></a>                     <span class="at">pheno =</span> DO_pheno[,pheno_col_i, <span class="at">drop =</span> <span class="cn">FALSE</span>], </span>
<span id="cb17-54"><a href="#cb17-54" aria-hidden="true" tabindex="-1"></a>                     <span class="at">kinship =</span> DO_str_2016_kinship[[chr_i]], </span>
<span id="cb17-55"><a href="#cb17-55" aria-hidden="true" tabindex="-1"></a>                     <span class="at">addcovar =</span> addcovar_i,</span>
<span id="cb17-56"><a href="#cb17-56" aria-hidden="true" tabindex="-1"></a>                     <span class="at">cores =</span> n_cores)</span>
<span id="cb17-57"><a href="#cb17-57" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(<span class="fu">plot_coefCC2</span>(blup_i, <span class="at">map =</span> DO_str_2016_cross<span class="sc">$</span>pmap, <span class="at">legend =</span> legendpos, <span class="at">main =</span> label, <span class="at">bgcolor =</span> <span class="st">&quot;white&quot;</span>))</span>
<span id="cb17-58"><a href="#cb17-58" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<p>Next, loading the <code>RTG_a3</code> data for mediation
analysis.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a>RTG_phenotypes <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_RTG_project_df_for_mapping.RDS&quot;</span>)</span></code></pre></div>
<p>Also loading the <code>SENSI</code> data for mediation analysis</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>SENSI_cocaine_sensi <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_novelty_saline_cocaine_cancor.RDS&quot;</span>)</span></code></pre></div>
<p>Next, loading CC/DO founders key.</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>cc_do_founders_key <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/founders_key/cc_do_founders_key.RDS&quot;</span>)</span></code></pre></div>
<p>Next, loading in <code>biomaRt</code> and getting a key for
expression data.</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="co"># 10</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting biomaRt data</span></span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>mm10_maRt <span class="ot">=</span> <span class="fu">useMart</span>(<span class="at">biomart =</span> <span class="st">&quot;ENSEMBL_MART_ENSEMBL&quot;</span>,</span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>                    <span class="at">host =</span> <span class="st">&quot;https://oct2018.archive.ensembl.org&quot;</span>,</span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>                    <span class="at">dataset =</span> <span class="st">&quot;mmusculus_gene_ensembl&quot;</span>)</span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a>maRt_filter <span class="ot">=</span> <span class="st">&quot;ensembl_gene_id&quot;</span></span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>maRt_attributes <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;ensembl_gene_id&quot;</span>,<span class="st">&quot;mgi_symbol&quot;</span>,<span class="st">&quot;mgi_description&quot;</span>,<span class="st">&quot;chromosome_name&quot;</span>,<span class="st">&quot;start_position&quot;</span>,<span class="st">&quot;end_position&quot;</span>,<span class="st">&quot;strand&quot;</span>)</span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a>Mm_maRt <span class="ot">=</span> <span class="fu">getBM</span>(maRt_attributes, maRt_filter, <span class="fu">colnames</span>(DO_pheno), mm10_maRt)</span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a><span class="co"># row.names(Mm_maRt) = Mm_maRt$ensembl_gene_id</span></span>
<span id="cb21-10"><a href="#cb21-10" aria-hidden="true" tabindex="-1"></a>Mm_maRt<span class="sc">$</span>start_position <span class="ot">=</span> Mm_maRt<span class="sc">$</span>start_position <span class="sc">/</span> <span class="fl">1e6</span></span>
<span id="cb21-11"><a href="#cb21-11" aria-hidden="true" tabindex="-1"></a>Mm_maRt<span class="sc">$</span>end_position <span class="ot">=</span> Mm_maRt<span class="sc">$</span>end_position <span class="sc">/</span> <span class="fl">1e6</span></span></code></pre></div>
<p>Finally, loading the <code>RTG_a2</code>, <code>RTG_a3</code>, and
<code>SENS</code> QTL results.</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Loading RTG a1</span></span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./data/DO_RTG_a1_new.qtl.RData&quot;</span>)</span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a>RTG_a1_qtl <span class="ot">=</span> m2.qtl.out<span class="sc">$</span>RTG_a1</span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./data/DO_RTG_a1_new_1000x.qtlpermu.RData&quot;</span>)</span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a>RTG_a1_perms <span class="ot">=</span> m2.permu<span class="sc">$</span>RTG_a1</span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-7"><a href="#cb22-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Loading RTG a2</span></span>
<span id="cb22-8"><a href="#cb22-8" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./data/DO_RTG_a2_new.qtl.RData&quot;</span>)</span>
<span id="cb22-9"><a href="#cb22-9" aria-hidden="true" tabindex="-1"></a>RTG_a2_qtl <span class="ot">=</span> m2.qtl.out<span class="sc">$</span>RTG_a2</span>
<span id="cb22-10"><a href="#cb22-10" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./data/DO_RTG_a2_new_1000x.qtlpermu.RData&quot;</span>)</span>
<span id="cb22-11"><a href="#cb22-11" aria-hidden="true" tabindex="-1"></a>RTG_a2_perms <span class="ot">=</span> m2.permu<span class="sc">$</span>RTG_a2</span>
<span id="cb22-12"><a href="#cb22-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-13"><a href="#cb22-13" aria-hidden="true" tabindex="-1"></a><span class="co"># Loading RTG a3</span></span>
<span id="cb22-14"><a href="#cb22-14" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./data/DO_RTG_a3_new.qtl.RData&quot;</span>)</span>
<span id="cb22-15"><a href="#cb22-15" aria-hidden="true" tabindex="-1"></a>RTG_a3_qtl <span class="ot">=</span> m2.qtl.out<span class="sc">$</span>RTG_a3</span>
<span id="cb22-16"><a href="#cb22-16" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./data/DO_RTG_a3_new_1000x.qtlpermu.RData&quot;</span>)</span>
<span id="cb22-17"><a href="#cb22-17" aria-hidden="true" tabindex="-1"></a>RTG_a3_perms <span class="ot">=</span> m2.permu<span class="sc">$</span>RTG_a3</span>
<span id="cb22-18"><a href="#cb22-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-19"><a href="#cb22-19" aria-hidden="true" tabindex="-1"></a><span class="co"># Loading SENS</span></span>
<span id="cb22-20"><a href="#cb22-20" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./data/DO_sensi_cocaine.qtl.RData&quot;</span>)</span>
<span id="cb22-21"><a href="#cb22-21" aria-hidden="true" tabindex="-1"></a>SENS_COC_qtl <span class="ot">=</span> m2.qtl.out<span class="sc">$</span>cocaine_novelty_cancor</span>
<span id="cb22-22"><a href="#cb22-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-23"><a href="#cb22-23" aria-hidden="true" tabindex="-1"></a><span class="co"># Loading SENSI Saline</span></span>
<span id="cb22-24"><a href="#cb22-24" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./data/DO_saline.qtl.RData&quot;</span>)</span>
<span id="cb22-25"><a href="#cb22-25" aria-hidden="true" tabindex="-1"></a>SENS_SAL_qtl <span class="ot">=</span> m2.qtl.out<span class="sc">$</span>saline_novelty_cancor</span>
<span id="cb22-26"><a href="#cb22-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-27"><a href="#cb22-27" aria-hidden="true" tabindex="-1"></a><span class="co"># Loading physical map</span></span>
<span id="cb22-28"><a href="#cb22-28" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">&quot;./output/RTG_out/gm_qc_pgmap.rdata&quot;</span>)</span></code></pre></div>
</div>
<div id="analysis-1" class="section level3">
<h3>Analysis</h3>
<p>Identifying all genes within a 10 Mb window around the peaks.</p>
<div id="rtg-a1" class="section level4">
<h4>RTG a1</h4>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting thresholds</span></span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>RTG_a1_permu_thresholds <span class="ot">=</span> <span class="fu">quantile</span>(RTG_a1_perms, <span class="at">probs =</span> <span class="dv">1</span> <span class="sc">-</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">0.01</span>, <span class="fl">0.05</span>, <span class="fl">0.10</span>, <span class="fl">0.63</span>, <span class="dv">1</span>))</span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a>RTG_a1_suggestive <span class="ot">=</span> RTG_a1_permu_thresholds[<span class="st">&quot;37%&quot;</span>]</span>
<span id="cb23-4"><a href="#cb23-4" aria-hidden="true" tabindex="-1"></a>RTG_a1_ten_percent <span class="ot">=</span> RTG_a1_permu_thresholds[<span class="st">&quot;90%&quot;</span>]</span>
<span id="cb23-5"><a href="#cb23-5" aria-hidden="true" tabindex="-1"></a>RTG_a1_five_percent <span class="ot">=</span> RTG_a1_permu_thresholds[<span class="st">&quot;95%&quot;</span>]</span>
<span id="cb23-6"><a href="#cb23-6" aria-hidden="true" tabindex="-1"></a>RTG_a1_one_percent <span class="ot">=</span> RTG_a1_permu_thresholds[<span class="st">&quot;99%&quot;</span>]</span>
<span id="cb23-7"><a href="#cb23-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-8"><a href="#cb23-8" aria-hidden="true" tabindex="-1"></a>RTG_a1_peaks_pos <span class="ot">=</span> <span class="fu">find_peaks</span>(RTG_a1_qtl,</span>
<span id="cb23-9"><a href="#cb23-9" aria-hidden="true" tabindex="-1"></a>                              <span class="at">map =</span> pmap,</span>
<span id="cb23-10"><a href="#cb23-10" aria-hidden="true" tabindex="-1"></a>                              <span class="at">threshold =</span> RTG_a1_permu_thresholds[<span class="st">&quot;37%&quot;</span>],</span>
<span id="cb23-11"><a href="#cb23-11" aria-hidden="true" tabindex="-1"></a>                              <span class="at">prob =</span> <span class="fl">0.95</span>)</span>
<span id="cb23-12"><a href="#cb23-12" aria-hidden="true" tabindex="-1"></a>RTG_a1_peaks_pos</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["lodindex"],"name":[1],"type":["int"],"align":["right"]},{"label":["lodcolumn"],"name":[2],"type":["chr"],"align":["left"]},{"label":["chr"],"name":[3],"type":["fct"],"align":["left"]},{"label":["pos"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["lod"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["ci_lo"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["ci_hi"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"1","2":"pheno1","3":"3","4":"64.79428","5":"6.266369","6":"27.90608","7":"87.26804"},{"1":"1","2":"pheno1","3":"4","4":"66.94180","5":"6.672830","6":"65.31461","7":"104.19205"},{"1":"1","2":"pheno1","3":"7","4":"90.55231","5":"5.964469","6":"19.44355","7":"139.35725"},{"1":"1","2":"pheno1","3":"8","4":"102.90059","5":"6.628390","6":"95.51528","7":"104.04931"},{"1":"1","2":"pheno1","3":"10","4":"126.19694","5":"6.449488","6":"73.68332","7":"128.58355"},{"1":"1","2":"pheno1","3":"17","4":"30.79443","5":"6.767366","6":"24.31971","7":"31.19120"},{"1":"1","2":"pheno1","3":"X","4":"126.06304","5":"6.871369","6":"126.06304","7":"126.66864"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="rtg-a2" class="section level4">
<h4>RTG a2</h4>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a>RTG_a2_permu_thresholds <span class="ot">=</span> <span class="fu">quantile</span>(RTG_a2_perms, <span class="at">probs =</span> <span class="dv">1</span> <span class="sc">-</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">0.01</span>, <span class="fl">0.05</span>, <span class="fl">0.10</span>, <span class="fl">0.63</span>, <span class="dv">1</span>))</span>
<span id="cb24-2"><a href="#cb24-2" aria-hidden="true" tabindex="-1"></a>RTG_a2_suggestive <span class="ot">=</span> RTG_a2_permu_thresholds[<span class="st">&quot;37%&quot;</span>]</span>
<span id="cb24-3"><a href="#cb24-3" aria-hidden="true" tabindex="-1"></a>RTG_a2_ten_percent <span class="ot">=</span> RTG_a2_permu_thresholds[<span class="st">&quot;90%&quot;</span>]</span>
<span id="cb24-4"><a href="#cb24-4" aria-hidden="true" tabindex="-1"></a>RTG_a2_five_percent <span class="ot">=</span> RTG_a2_permu_thresholds[<span class="st">&quot;95%&quot;</span>]</span>
<span id="cb24-5"><a href="#cb24-5" aria-hidden="true" tabindex="-1"></a>RTG_a2_one_percent <span class="ot">=</span> RTG_a2_permu_thresholds[<span class="st">&quot;99%&quot;</span>]</span>
<span id="cb24-6"><a href="#cb24-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb24-7"><a href="#cb24-7" aria-hidden="true" tabindex="-1"></a>RTG_a2_peaks_pos <span class="ot">=</span> <span class="fu">find_peaks</span>(RTG_a2_qtl,</span>
<span id="cb24-8"><a href="#cb24-8" aria-hidden="true" tabindex="-1"></a>                              <span class="at">map =</span> pmap,</span>
<span id="cb24-9"><a href="#cb24-9" aria-hidden="true" tabindex="-1"></a>                              <span class="at">threshold =</span> RTG_a2_permu_thresholds[<span class="st">&quot;37%&quot;</span>],</span>
<span id="cb24-10"><a href="#cb24-10" aria-hidden="true" tabindex="-1"></a>                              <span class="at">prob =</span> <span class="fl">0.95</span>)</span>
<span id="cb24-11"><a href="#cb24-11" aria-hidden="true" tabindex="-1"></a>RTG_a2_peaks_pos</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["lodindex"],"name":[1],"type":["int"],"align":["right"]},{"label":["lodcolumn"],"name":[2],"type":["chr"],"align":["left"]},{"label":["chr"],"name":[3],"type":["fct"],"align":["left"]},{"label":["pos"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["lod"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["ci_lo"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["ci_hi"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"1","2":"pheno1","3":"2","4":"152.69142","5":"6.298269","6":"5.700593","7":"178.75988"},{"1":"1","2":"pheno1","3":"3","4":"87.08223","5":"6.147783","6":"27.906078","7":"147.10265"},{"1":"1","2":"pheno1","3":"4","4":"68.40926","5":"6.886803","6":"65.622424","7":"104.85803"},{"1":"1","2":"pheno1","3":"10","4":"79.06056","5":"6.192272","6":"28.485286","7":"127.64426"},{"1":"1","2":"pheno1","3":"17","4":"26.58020","5":"6.646707","6":"23.927610","7":"30.92497"},{"1":"1","2":"pheno1","3":"18","4":"79.07333","5":"6.502239","6":"35.133799","7":"88.55710"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="rtg-a3" class="section level4">
<h4>RTG a3</h4>
<p>Starting with chr7.</p>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a>RTG_a3_permu_thresholds <span class="ot">=</span> <span class="fu">quantile</span>(RTG_a3_perms, <span class="at">probs =</span> <span class="dv">1</span> <span class="sc">-</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">0.01</span>, <span class="fl">0.05</span>, <span class="fl">0.10</span>, <span class="fl">0.63</span>, <span class="dv">1</span>))</span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a>RTG_a3_suggestive <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;37%&quot;</span>]</span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a>RTG_a3_ten_percent <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;90%&quot;</span>]</span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a>RTG_a3_five_percent <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;95%&quot;</span>]</span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a>RTG_a3_one_percent <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;99%&quot;</span>]</span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a>RTG_a3_peaks_pos <span class="ot">=</span> <span class="fu">find_peaks</span>(RTG_a3_qtl,</span>
<span id="cb25-8"><a href="#cb25-8" aria-hidden="true" tabindex="-1"></a>                              <span class="at">map =</span> pmap,</span>
<span id="cb25-9"><a href="#cb25-9" aria-hidden="true" tabindex="-1"></a>                              <span class="at">threshold =</span> RTG_a3_permu_thresholds[<span class="st">&quot;37%&quot;</span>],</span>
<span id="cb25-10"><a href="#cb25-10" aria-hidden="true" tabindex="-1"></a>                              <span class="at">prob =</span> <span class="fl">0.95</span>)</span>
<span id="cb25-11"><a href="#cb25-11" aria-hidden="true" tabindex="-1"></a>RTG_a3_peaks_pos</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["lodindex"],"name":[1],"type":["int"],"align":["right"]},{"label":["lodcolumn"],"name":[2],"type":["chr"],"align":["left"]},{"label":["chr"],"name":[3],"type":["fct"],"align":["left"]},{"label":["pos"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["lod"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["ci_lo"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["ci_hi"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"1","2":"pheno1","3":"2","4":"156.94727","5":"7.516119","6":"21.537364","7":"159.97494"},{"1":"1","2":"pheno1","3":"3","4":"41.71768","5":"6.103629","6":"8.849682","7":"141.22427"},{"1":"1","2":"pheno1","3":"4","4":"97.75826","5":"6.789763","6":"66.847420","7":"137.20284"},{"1":"1","2":"pheno1","3":"6","4":"117.44650","5":"6.100421","6":"4.997746","7":"142.47792"},{"1":"1","2":"pheno1","3":"7","4":"87.65045","5":"13.247377","6":"87.127600","7":"89.52247"},{"1":"1","2":"pheno1","3":"10","4":"126.99230","5":"9.573921","6":"126.910230","7":"128.35783"},{"1":"1","2":"pheno1","3":"14","4":"109.43210","5":"6.314892","6":"31.239525","7":"113.91478"},{"1":"1","2":"pheno1","3":"18","4":"81.67797","5":"7.063818","6":"77.645166","7":"82.53084"},{"1":"1","2":"pheno1","3":"19","4":"28.10394","5":"6.289914","6":"27.612116","7":"32.73827"},{"1":"1","2":"pheno1","3":"X","4":"145.41292","5":"7.697957","6":"143.607480","7":"153.18451"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="sens-cocaine-peaks" class="section level4">
<h4>SENS Cocaine Peaks</h4>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a>SENS_COC_peaks_pos <span class="ot">=</span> <span class="fu">find_peaks</span>(SENS_COC_qtl,</span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a>                              <span class="at">map =</span> pmap,</span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a>                              <span class="at">threshold =</span> RTG_a3_permu_thresholds[<span class="st">&quot;37%&quot;</span>],</span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a>                              <span class="at">prob =</span> <span class="fl">0.95</span>)</span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a>SENS_COC_peaks_pos</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["lodindex"],"name":[1],"type":["int"],"align":["right"]},{"label":["lodcolumn"],"name":[2],"type":["chr"],"align":["left"]},{"label":["chr"],"name":[3],"type":["fct"],"align":["left"]},{"label":["pos"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["lod"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["ci_lo"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["ci_hi"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"1","2":"pheno1","3":"3","4":"68.721822","5":"7.457026","6":"66.826817","7":"139.243295"},{"1":"1","2":"pheno1","3":"6","4":"5.211712","5":"9.122582","6":"4.997746","7":"5.753635"},{"1":"1","2":"pheno1","3":"7","4":"89.522474","5":"9.410413","6":"87.244659","7":"91.284113"},{"1":"1","2":"pheno1","3":"10","4":"105.630599","5":"6.078273","6":"17.234900","7":"128.382937"},{"1":"1","2":"pheno1","3":"16","4":"30.819494","5":"6.287702","6":"24.498310","7":"78.155446"},{"1":"1","2":"pheno1","3":"18","4":"81.704609","5":"7.331789","6":"77.645166","7":"83.757465"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="sens-saline-peaks" class="section level4">
<h4>SENS Saline Peaks</h4>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a>SENS_SAL_peaks_pos <span class="ot">=</span> <span class="fu">find_peaks</span>(SENS_SAL_qtl,</span>
<span id="cb27-2"><a href="#cb27-2" aria-hidden="true" tabindex="-1"></a>                              <span class="at">map =</span> pmap,</span>
<span id="cb27-3"><a href="#cb27-3" aria-hidden="true" tabindex="-1"></a>                              <span class="at">threshold =</span> RTG_a3_permu_thresholds[<span class="st">&quot;37%&quot;</span>],</span>
<span id="cb27-4"><a href="#cb27-4" aria-hidden="true" tabindex="-1"></a>                              <span class="at">prob =</span> <span class="fl">0.95</span>)</span>
<span id="cb27-5"><a href="#cb27-5" aria-hidden="true" tabindex="-1"></a>SENS_SAL_peaks_pos</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["lodindex"],"name":[1],"type":["int"],"align":["right"]},{"label":["lodcolumn"],"name":[2],"type":["chr"],"align":["left"]},{"label":["chr"],"name":[3],"type":["fct"],"align":["left"]},{"label":["pos"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["lod"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["ci_lo"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["ci_hi"],"name":[7],"type":["dbl"],"align":["right"]}],"data":[{"1":"1","2":"pheno1","3":"2","4":"85.24720","5":"6.979980","6":"82.318968","7":"90.57100"},{"1":"1","2":"pheno1","3":"3","4":"69.26026","5":"7.741544","6":"66.826817","7":"76.84885"},{"1":"1","2":"pheno1","3":"4","4":"64.08704","5":"6.676130","6":"62.856157","7":"155.08023"},{"1":"1","2":"pheno1","3":"6","4":"12.09362","5":"7.454458","6":"4.997746","7":"16.06701"},{"1":"1","2":"pheno1","3":"7","4":"87.46413","5":"11.671472","6":"84.039573","7":"90.65869"},{"1":"1","2":"pheno1","3":"8","4":"103.42172","5":"6.710438","6":"33.468734","7":"128.37172"},{"1":"1","2":"pheno1","3":"9","4":"94.34335","5":"6.424863","6":"89.591139","7":"94.66214"},{"1":"1","2":"pheno1","3":"10","4":"128.35050","5":"7.013327","6":"41.866826","7":"130.41683"},{"1":"1","2":"pheno1","3":"16","4":"26.74108","5":"6.137962","6":"8.472590","7":"79.10995"},{"1":"1","2":"pheno1","3":"17","4":"86.50368","5":"6.755416","6":"86.180453","7":"88.43695"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
<div id="chromosome-7-peak" class="section level4">
<h4>Chromosome 7 peak</h4>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a>RTG_a3_chr7_peak <span class="ot">=</span> RTG_a3_peaks_pos[<span class="fu">which</span>(RTG_a3_peaks_pos<span class="sc">$</span>chr <span class="sc">==</span> <span class="st">&quot;7&quot;</span>),]</span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>RTG_a3_chr7_peak_pos <span class="ot">=</span> RTG_a3_chr7_peak[,<span class="st">&quot;pos&quot;</span>]</span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a>SENSI_cocaine_chr7_peak <span class="ot">=</span> SENS_COC_peaks_pos[<span class="fu">which</span>(SENS_COC_peaks_pos<span class="sc">$</span>chr <span class="sc">==</span> <span class="st">&quot;7&quot;</span>),]</span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>SENSI_cocaine_chr7_peak_pos <span class="ot">=</span> SENSI_cocaine_chr7_peak[,<span class="st">&quot;pos&quot;</span>]</span>
<span id="cb28-6"><a href="#cb28-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-7"><a href="#cb28-7" aria-hidden="true" tabindex="-1"></a>SENSI_saline_chr7_peak <span class="ot">=</span> SENS_SAL_peaks_pos[<span class="fu">which</span>(SENS_SAL_peaks_pos<span class="sc">$</span>chr <span class="sc">==</span> <span class="st">&quot;7&quot;</span>),]</span>
<span id="cb28-8"><a href="#cb28-8" aria-hidden="true" tabindex="-1"></a>SENSI_saline_chr7_peak_pos <span class="ot">=</span> SENSI_saline_chr7_peak[,<span class="st">&quot;pos&quot;</span>]</span>
<span id="cb28-9"><a href="#cb28-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-10"><a href="#cb28-10" aria-hidden="true" tabindex="-1"></a>max_chr7_peak_pos <span class="ot">=</span> <span class="fu">max</span>(<span class="fu">c</span>(RTG_a3_chr7_peak_pos, SENSI_cocaine_chr7_peak_pos, SENSI_saline_chr7_peak_pos))</span>
<span id="cb28-11"><a href="#cb28-11" aria-hidden="true" tabindex="-1"></a>min_chr7_peak_pos <span class="ot">=</span> <span class="fu">min</span>(<span class="fu">c</span>(RTG_a3_chr7_peak_pos, SENSI_cocaine_chr7_peak_pos, SENSI_saline_chr7_peak_pos))</span>
<span id="cb28-12"><a href="#cb28-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-13"><a href="#cb28-13" aria-hidden="true" tabindex="-1"></a>chr7_genes <span class="ot">=</span> Mm_maRt[<span class="fu">which</span>(Mm_maRt<span class="sc">$</span>chromosome_name <span class="sc">==</span> <span class="st">&quot;7&quot;</span>),]</span>
<span id="cb28-14"><a href="#cb28-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-15"><a href="#cb28-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting genes within 10 Mb of the peak</span></span>
<span id="cb28-16"><a href="#cb28-16" aria-hidden="true" tabindex="-1"></a>chr7_genes <span class="ot">=</span> chr7_genes[<span class="fu">which</span>(chr7_genes<span class="sc">$</span>start_position <span class="sc">&gt;=</span> (min_chr7_peak_pos <span class="sc">-</span> <span class="dv">10</span>) <span class="sc">|</span> chr7_genes<span class="sc">$</span>end_position <span class="sc">&gt;=</span> (min_chr7_peak_pos <span class="sc">-</span> <span class="dv">10</span>)),]</span>
<span id="cb28-17"><a href="#cb28-17" aria-hidden="true" tabindex="-1"></a>chr7_genes <span class="ot">=</span> chr7_genes[<span class="fu">which</span>(chr7_genes<span class="sc">$</span>start_position <span class="sc">&lt;=</span> (max_chr7_peak_pos <span class="sc">+</span> <span class="dv">10</span>) <span class="sc">|</span> chr7_genes<span class="sc">$</span>end_position <span class="sc">&lt;=</span> (max_chr7_peak_pos <span class="sc">+</span> <span class="dv">10</span>)),]</span></code></pre></div>
<p>Finding SNP associations for all of these genes and calculating
correlations with the <code>RTG_a3</code> SNP associations. Starting
with chr4.</p>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting SNP range for the chr7 resuls</span></span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a>chr <span class="ot">=</span> <span class="st">&quot;7&quot;</span></span>
<span id="cb29-3"><a href="#cb29-3" aria-hidden="true" tabindex="-1"></a>start <span class="ot">=</span> <span class="fu">min</span>(RTG_a3_chr7_SNPs<span class="sc">$</span>pos)</span>
<span id="cb29-4"><a href="#cb29-4" aria-hidden="true" tabindex="-1"></a>end <span class="ot">=</span> <span class="fu">max</span>(RTG_a3_chr7_SNPs<span class="sc">$</span>pos)</span>
<span id="cb29-5"><a href="#cb29-5" aria-hidden="true" tabindex="-1"></a>chr7_pheno <span class="ot">=</span> DO_pheno[,<span class="fu">which</span>(<span class="fu">colnames</span>(DO_pheno) <span class="sc">%in%</span> chr7_genes<span class="sc">$</span>ensembl_gene_id)]</span>
<span id="cb29-6"><a href="#cb29-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-7"><a href="#cb29-7" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">ncol</span>(chr7_pheno)) {</span>
<span id="cb29-8"><a href="#cb29-8" aria-hidden="true" tabindex="-1"></a>  col_i <span class="ot">=</span> <span class="fu">colnames</span>(chr7_pheno)[i]</span>
<span id="cb29-9"><a href="#cb29-9" aria-hidden="true" tabindex="-1"></a>  ranknorm_resid_i <span class="ot">=</span> <span class="fu">norm_rank_transform</span>(<span class="fu">resid</span>(<span class="fu">lm</span>(DO_pheno[,col_i] <span class="sc">~</span> <span class="fu">as.numeric</span>(DO_covar<span class="sc">$</span>choroid_plexus_covariate))))</span>
<span id="cb29-10"><a href="#cb29-10" aria-hidden="true" tabindex="-1"></a>  chr7_pheno[,i] <span class="ot">=</span> ranknorm_resid_i[<span class="fu">row.names</span>(chr7_pheno)]</span>
<span id="cb29-11"><a href="#cb29-11" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb29-12"><a href="#cb29-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-13"><a href="#cb29-13" aria-hidden="true" tabindex="-1"></a><span class="co"># Looping through to make eQTL results</span></span>
<span id="cb29-14"><a href="#cb29-14" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(chr7_genes)) {</span>
<span id="cb29-15"><a href="#cb29-15" aria-hidden="true" tabindex="-1"></a>  sig_gene_i <span class="ot">=</span> chr7_genes[i,<span class="st">&quot;ensembl_gene_id&quot;</span>]</span>
<span id="cb29-16"><a href="#cb29-16" aria-hidden="true" tabindex="-1"></a>  sig_label_i <span class="ot">=</span> chr7_genes[i,<span class="st">&quot;mgi_symbol&quot;</span>]</span>
<span id="cb29-17"><a href="#cb29-17" aria-hidden="true" tabindex="-1"></a>  pheno_col_i <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">colnames</span>(chr7_pheno) <span class="sc">==</span> sig_gene_i)</span>
<span id="cb29-18"><a href="#cb29-18" aria-hidden="true" tabindex="-1"></a>  assoc <span class="ot">=</span> <span class="fu">scan1snps</span>(<span class="at">genoprobs =</span> DO_str_2016_aprobs[,chr], </span>
<span id="cb29-19"><a href="#cb29-19" aria-hidden="true" tabindex="-1"></a>                    <span class="at">map =</span> DO_str_2016_cross<span class="sc">$</span>pmap, </span>
<span id="cb29-20"><a href="#cb29-20" aria-hidden="true" tabindex="-1"></a>                    <span class="at">pheno =</span> chr7_pheno[,pheno_col_i, <span class="at">drop =</span> <span class="cn">FALSE</span>], </span>
<span id="cb29-21"><a href="#cb29-21" aria-hidden="true" tabindex="-1"></a>                    <span class="at">kinship =</span> DO_str_2016_kinship, </span>
<span id="cb29-22"><a href="#cb29-22" aria-hidden="true" tabindex="-1"></a>                    <span class="at">addcovar =</span> addcovar_i, </span>
<span id="cb29-23"><a href="#cb29-23" aria-hidden="true" tabindex="-1"></a>                    <span class="at">query_func =</span> query_func, </span>
<span id="cb29-24"><a href="#cb29-24" aria-hidden="true" tabindex="-1"></a>                    <span class="at">chr =</span> chr, </span>
<span id="cb29-25"><a href="#cb29-25" aria-hidden="true" tabindex="-1"></a>                    <span class="at">start =</span> start, </span>
<span id="cb29-26"><a href="#cb29-26" aria-hidden="true" tabindex="-1"></a>                    <span class="at">end =</span> end, </span>
<span id="cb29-27"><a href="#cb29-27" aria-hidden="true" tabindex="-1"></a>                    <span class="at">keep_all_snps =</span> <span class="cn">TRUE</span>)</span>
<span id="cb29-28"><a href="#cb29-28" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="fu">exists</span>(<span class="st">&quot;qtl_df&quot;</span>)) {</span>
<span id="cb29-29"><a href="#cb29-29" aria-hidden="true" tabindex="-1"></a>    qtl_df <span class="ot">=</span> <span class="fu">cbind</span>(qtl_df, <span class="fu">as.data.frame</span>(assoc<span class="sc">$</span>lod)[<span class="fu">row.names</span>(qtl_df),])</span>
<span id="cb29-30"><a href="#cb29-30" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb29-31"><a href="#cb29-31" aria-hidden="true" tabindex="-1"></a>    qtl_df <span class="ot">=</span> <span class="fu">as.data.frame</span>(assoc<span class="sc">$</span>lod)</span>
<span id="cb29-32"><a href="#cb29-32" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb29-33"><a href="#cb29-33" aria-hidden="true" tabindex="-1"></a>  <span class="fu">colnames</span>(qtl_df)[i] <span class="ot">=</span> sig_label_i</span>
<span id="cb29-34"><a href="#cb29-34" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb29-35"><a href="#cb29-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-36"><a href="#cb29-36" aria-hidden="true" tabindex="-1"></a><span class="co">#  colnames(qtl_df) = chr7_genes$mgi_symbol</span></span>
<span id="cb29-37"><a href="#cb29-37" aria-hidden="true" tabindex="-1"></a>qtl_df<span class="sc">$</span>RTG_a3 <span class="ot">=</span> RTG_a3_chr7_SNPs[<span class="fu">row.names</span>(qtl_df),<span class="st">&quot;pheno1&quot;</span>]</span>
<span id="cb29-38"><a href="#cb29-38" aria-hidden="true" tabindex="-1"></a>qtl_df<span class="sc">$</span>SENSI_cocaine <span class="ot">=</span> chr7_cocaine_snps_lod[<span class="fu">row.names</span>(qtl_df),<span class="st">&quot;pheno1&quot;</span>]</span>
<span id="cb29-39"><a href="#cb29-39" aria-hidden="true" tabindex="-1"></a>qtl_df<span class="sc">$</span>SENSI_saline <span class="ot">=</span> chr7_saline_snps_lod[<span class="fu">row.names</span>(qtl_df),<span class="st">&quot;pheno1&quot;</span>]</span></code></pre></div>
<p>Calculating correlations between the <code>RTG_a3</code>,
<code>SENSI_cocaine</code>, and <code>SENSI_saline</code> SNP mapping
for chr7.</p>
<div class="sourceCode" id="cb30"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1" aria-hidden="true" tabindex="-1"></a>RTG_a3_cor <span class="ot">=</span> <span class="fu">cor</span>(qtl_df<span class="sc">$</span>RTG_a3,qtl_df[,chr7_genes<span class="sc">$</span>mgi_symbol],</span>
<span id="cb30-2"><a href="#cb30-2" aria-hidden="true" tabindex="-1"></a>                 <span class="at">use =</span> <span class="st">&quot;pairwise.complete.obs&quot;</span>)</span>
<span id="cb30-3"><a href="#cb30-3" aria-hidden="true" tabindex="-1"></a>SENSI_cocaine_cor <span class="ot">=</span> <span class="fu">cor</span>(qtl_df<span class="sc">$</span>SENSI_cocaine,qtl_df[,chr7_genes<span class="sc">$</span>mgi_symbol],</span>
<span id="cb30-4"><a href="#cb30-4" aria-hidden="true" tabindex="-1"></a>                        <span class="at">use =</span> <span class="st">&quot;pairwise.complete.obs&quot;</span>)</span>
<span id="cb30-5"><a href="#cb30-5" aria-hidden="true" tabindex="-1"></a>SENSI_saline_cor <span class="ot">=</span> <span class="fu">cor</span>(qtl_df<span class="sc">$</span>SENSI_saline,qtl_df[,chr7_genes<span class="sc">$</span>mgi_symbol],</span>
<span id="cb30-6"><a href="#cb30-6" aria-hidden="true" tabindex="-1"></a>                 <span class="at">use =</span> <span class="st">&quot;pairwise.complete.obs&quot;</span>)</span>
<span id="cb30-7"><a href="#cb30-7" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>cor_RTG_a3 <span class="ot">=</span> RTG_a3_cor[,<span class="fu">as.character</span>(chr7_genes<span class="sc">$</span>mgi_symbol)]</span>
<span id="cb30-8"><a href="#cb30-8" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>cor_SENSI_cocaine <span class="ot">=</span> SENSI_cocaine_cor[,<span class="fu">as.character</span>(chr7_genes<span class="sc">$</span>mgi_symbol)]</span>
<span id="cb30-9"><a href="#cb30-9" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>cor_SENSI_saline <span class="ot">=</span> SENSI_saline_cor[,<span class="fu">as.character</span>(chr7_genes<span class="sc">$</span>mgi_symbol)]</span></code></pre></div>
<p>Performing mediation analysis with <code>RTG_a3</code> phenotype on
chr7 genes. Beginning with a plot of the <code>RTG_a3</code> result on
the data with expression information.</p>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting phenotype matrices for IVSA, SENSI Cocaine, and SENSI Saline</span></span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a>RTG_a3_phenotype_mat <span class="ot">=</span> <span class="fu">matrix</span>(RTG_phenotypes[<span class="fu">gsub</span>(<span class="st">&quot;^(</span><span class="sc">\\</span><span class="st">d{4,5})_.*$&quot;</span>,<span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>,<span class="fu">row.names</span>(chr7_pheno)),<span class="st">&quot;RTG_a3&quot;</span>], <span class="at">ncol =</span> <span class="dv">1</span>)</span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(RTG_a3_phenotype_mat) <span class="ot">=</span> <span class="fu">row.names</span>(chr7_pheno)</span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(RTG_a3_phenotype_mat) <span class="ot">=</span> <span class="st">&quot;RTG_a3&quot;</span></span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a>SENSI_cocaine_phenotype_mat <span class="ot">=</span> <span class="fu">matrix</span>(SENSI_cocaine_sensi[<span class="fu">gsub</span>(<span class="st">&quot;^(</span><span class="sc">\\</span><span class="st">d{4,5})_.*$&quot;</span>,<span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>,<span class="fu">row.names</span>(chr7_pheno)),<span class="st">&quot;cocaine_novelty_cancor&quot;</span>], <span class="at">ncol =</span> <span class="dv">1</span>)</span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(SENSI_cocaine_phenotype_mat) <span class="ot">=</span> <span class="fu">row.names</span>(chr7_pheno)</span>
<span id="cb31-8"><a href="#cb31-8" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(SENSI_cocaine_phenotype_mat) <span class="ot">=</span> <span class="st">&quot;SENSI_cocaine&quot;</span></span>
<span id="cb31-9"><a href="#cb31-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-10"><a href="#cb31-10" aria-hidden="true" tabindex="-1"></a>SENSI_saline_phenotype_mat <span class="ot">=</span> <span class="fu">matrix</span>(SENSI_cocaine_sensi[<span class="fu">gsub</span>(<span class="st">&quot;^(</span><span class="sc">\\</span><span class="st">d{4,5})_.*$&quot;</span>,<span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>,<span class="fu">row.names</span>(chr7_pheno)),<span class="st">&quot;saline_novelty_cancor&quot;</span>], <span class="at">ncol =</span> <span class="dv">1</span>)</span>
<span id="cb31-11"><a href="#cb31-11" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(SENSI_saline_phenotype_mat) <span class="ot">=</span> <span class="fu">row.names</span>(chr7_pheno)</span>
<span id="cb31-12"><a href="#cb31-12" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(SENSI_saline_phenotype_mat) <span class="ot">=</span> <span class="st">&quot;SENSI_saline&quot;</span></span>
<span id="cb31-13"><a href="#cb31-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-14"><a href="#cb31-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Mapping RTG_a3 on chromosome 7</span></span>
<span id="cb31-15"><a href="#cb31-15" aria-hidden="true" tabindex="-1"></a>qtl_RTG_a3_exprs <span class="ot">=</span> <span class="fu">scan1</span>(<span class="at">genoprobs =</span> DO_str_2016_aprobs[,<span class="st">&quot;7&quot;</span>],</span>
<span id="cb31-16"><a href="#cb31-16" aria-hidden="true" tabindex="-1"></a>                         <span class="at">pheno =</span> RTG_a3_phenotype_mat[,<span class="st">&quot;RTG_a3&quot;</span>],</span>
<span id="cb31-17"><a href="#cb31-17" aria-hidden="true" tabindex="-1"></a>                         <span class="at">kinship =</span> DO_str_2016_kinship<span class="sc">$</span><span class="st">`</span><span class="at">7</span><span class="st">`</span>,</span>
<span id="cb31-18"><a href="#cb31-18" aria-hidden="true" tabindex="-1"></a>                         <span class="at">addcovar =</span> addcovar_i,</span>
<span id="cb31-19"><a href="#cb31-19" aria-hidden="true" tabindex="-1"></a>                         <span class="at">cores =</span> <span class="dv">9</span>)</span>
<span id="cb31-20"><a href="#cb31-20" aria-hidden="true" tabindex="-1"></a>chr7_markers <span class="ot">=</span> DO_str_2016_cross<span class="sc">$</span>pmap<span class="sc">$</span><span class="st">`</span><span class="at">7</span><span class="st">`</span></span>
<span id="cb31-21"><a href="#cb31-21" aria-hidden="true" tabindex="-1"></a>chr7_markers <span class="ot">=</span> chr7_markers[<span class="fu">which</span>(chr7_markers <span class="sc">&gt;=</span> (RTG_a3_chr7_peak_pos <span class="sc">-</span> <span class="dv">5</span>) <span class="sc">&amp;</span></span>
<span id="cb31-22"><a href="#cb31-22" aria-hidden="true" tabindex="-1"></a>                                  chr7_markers <span class="sc">&lt;=</span> (RTG_a3_chr7_peak_pos <span class="sc">+</span> <span class="dv">5</span>))]</span>
<span id="cb31-23"><a href="#cb31-23" aria-hidden="true" tabindex="-1"></a>chr7_pheno1 <span class="ot">=</span> <span class="fu">as.data.frame</span>(qtl_RTG_a3_exprs)[<span class="fu">names</span>(chr7_markers),<span class="st">&quot;pheno1&quot;</span>]</span>
<span id="cb31-24"><a href="#cb31-24" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(chr7_pheno1) <span class="ot">=</span> <span class="fu">names</span>(chr7_markers)</span>
<span id="cb31-25"><a href="#cb31-25" aria-hidden="true" tabindex="-1"></a>max_marker_RTG_a3 <span class="ot">=</span> <span class="fu">names</span>(chr7_pheno1[<span class="fu">which</span>(chr7_pheno1 <span class="sc">==</span> <span class="fu">max</span>(chr7_pheno1))])</span>
<span id="cb31-26"><a href="#cb31-26" aria-hidden="true" tabindex="-1"></a>max_marker_lod_RTG_a3 <span class="ot">=</span> chr7_pheno1[max_marker_RTG_a3]</span>
<span id="cb31-27"><a href="#cb31-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-28"><a href="#cb31-28" aria-hidden="true" tabindex="-1"></a><span class="co"># Mapping SENSI_cocaine on chromosome 7</span></span>
<span id="cb31-29"><a href="#cb31-29" aria-hidden="true" tabindex="-1"></a>qtl_SENSI_cocaine_exprs <span class="ot">=</span> <span class="fu">scan1</span>(<span class="at">genoprobs =</span> DO_str_2016_aprobs[,<span class="st">&quot;7&quot;</span>],</span>
<span id="cb31-30"><a href="#cb31-30" aria-hidden="true" tabindex="-1"></a>                                <span class="at">pheno =</span> SENSI_cocaine_phenotype_mat[,<span class="st">&quot;SENSI_cocaine&quot;</span>],</span>
<span id="cb31-31"><a href="#cb31-31" aria-hidden="true" tabindex="-1"></a>                                <span class="at">kinship =</span> DO_str_2016_kinship<span class="sc">$</span><span class="st">`</span><span class="at">7</span><span class="st">`</span>,</span>
<span id="cb31-32"><a href="#cb31-32" aria-hidden="true" tabindex="-1"></a>                                <span class="at">addcovar =</span> addcovar_i,</span>
<span id="cb31-33"><a href="#cb31-33" aria-hidden="true" tabindex="-1"></a>                                <span class="at">cores =</span> <span class="dv">9</span>)</span>
<span id="cb31-34"><a href="#cb31-34" aria-hidden="true" tabindex="-1"></a>chr7_markers <span class="ot">=</span> DO_str_2016_cross<span class="sc">$</span>pmap<span class="sc">$</span><span class="st">`</span><span class="at">7</span><span class="st">`</span></span>
<span id="cb31-35"><a href="#cb31-35" aria-hidden="true" tabindex="-1"></a>chr7_markers <span class="ot">=</span> chr7_markers[<span class="fu">which</span>(chr7_markers <span class="sc">&gt;=</span> (SENSI_cocaine_chr7_peak_pos <span class="sc">-</span> <span class="dv">5</span>) <span class="sc">&amp;</span></span>
<span id="cb31-36"><a href="#cb31-36" aria-hidden="true" tabindex="-1"></a>                                  chr7_markers <span class="sc">&lt;=</span> (SENSI_cocaine_chr7_peak_pos <span class="sc">+</span> <span class="dv">5</span>))]</span>
<span id="cb31-37"><a href="#cb31-37" aria-hidden="true" tabindex="-1"></a>chr7_pheno1 <span class="ot">=</span> <span class="fu">as.data.frame</span>(qtl_SENSI_cocaine_exprs)[<span class="fu">names</span>(chr7_markers),<span class="dv">1</span>]</span>
<span id="cb31-38"><a href="#cb31-38" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(chr7_pheno1) <span class="ot">=</span> <span class="fu">names</span>(chr7_markers)</span>
<span id="cb31-39"><a href="#cb31-39" aria-hidden="true" tabindex="-1"></a>max_marker_SENSI_cocaine <span class="ot">=</span> <span class="fu">names</span>(chr7_pheno1[<span class="fu">which</span>(chr7_pheno1 <span class="sc">==</span> <span class="fu">max</span>(chr7_pheno1))])</span>
<span id="cb31-40"><a href="#cb31-40" aria-hidden="true" tabindex="-1"></a>max_marker_lod_SENSI_cocaine <span class="ot">=</span> chr7_pheno1[max_marker_SENSI_cocaine]</span>
<span id="cb31-41"><a href="#cb31-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-42"><a href="#cb31-42" aria-hidden="true" tabindex="-1"></a><span class="co"># Mapping SENSI_saline on chromosome 7</span></span>
<span id="cb31-43"><a href="#cb31-43" aria-hidden="true" tabindex="-1"></a>qtl_SENSI_saline_exprs <span class="ot">=</span> <span class="fu">scan1</span>(<span class="at">genoprobs =</span> DO_str_2016_aprobs[,<span class="st">&quot;7&quot;</span>],</span>
<span id="cb31-44"><a href="#cb31-44" aria-hidden="true" tabindex="-1"></a>                         <span class="at">pheno =</span> SENSI_saline_phenotype_mat[,<span class="st">&quot;SENSI_saline&quot;</span>],</span>
<span id="cb31-45"><a href="#cb31-45" aria-hidden="true" tabindex="-1"></a>                         <span class="at">kinship =</span> DO_str_2016_kinship<span class="sc">$</span><span class="st">`</span><span class="at">7</span><span class="st">`</span>,</span>
<span id="cb31-46"><a href="#cb31-46" aria-hidden="true" tabindex="-1"></a>                         <span class="at">addcovar =</span> addcovar_i,</span>
<span id="cb31-47"><a href="#cb31-47" aria-hidden="true" tabindex="-1"></a>                         <span class="at">cores =</span> <span class="dv">9</span>)</span>
<span id="cb31-48"><a href="#cb31-48" aria-hidden="true" tabindex="-1"></a>chr7_markers <span class="ot">=</span> DO_str_2016_cross<span class="sc">$</span>pmap<span class="sc">$</span><span class="st">`</span><span class="at">7</span><span class="st">`</span></span>
<span id="cb31-49"><a href="#cb31-49" aria-hidden="true" tabindex="-1"></a>chr7_markers <span class="ot">=</span> chr7_markers[<span class="fu">which</span>(chr7_markers <span class="sc">&gt;=</span> (SENSI_saline_chr7_peak_pos <span class="sc">-</span> <span class="dv">5</span>) <span class="sc">&amp;</span></span>
<span id="cb31-50"><a href="#cb31-50" aria-hidden="true" tabindex="-1"></a>                                  chr7_markers <span class="sc">&lt;=</span> (SENSI_saline_chr7_peak_pos <span class="sc">+</span> <span class="dv">5</span>))]</span>
<span id="cb31-51"><a href="#cb31-51" aria-hidden="true" tabindex="-1"></a>chr7_pheno1 <span class="ot">=</span> <span class="fu">as.data.frame</span>(qtl_SENSI_saline_exprs)[<span class="fu">names</span>(chr7_markers),<span class="dv">1</span>]</span>
<span id="cb31-52"><a href="#cb31-52" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(chr7_pheno1) <span class="ot">=</span> <span class="fu">names</span>(chr7_markers)</span>
<span id="cb31-53"><a href="#cb31-53" aria-hidden="true" tabindex="-1"></a>max_marker_SENSI_saline <span class="ot">=</span> <span class="fu">names</span>(chr7_pheno1[<span class="fu">which</span>(chr7_pheno1 <span class="sc">==</span> <span class="fu">max</span>(chr7_pheno1))])</span>
<span id="cb31-54"><a href="#cb31-54" aria-hidden="true" tabindex="-1"></a>max_marker_lod_SENSI_saline <span class="ot">=</span> chr7_pheno1[max_marker_SENSI_saline]</span>
<span id="cb31-55"><a href="#cb31-55" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-56"><a href="#cb31-56" aria-hidden="true" tabindex="-1"></a><span class="co"># Doing mediation analysis: looking at LOD score drops</span></span>
<span id="cb31-57"><a href="#cb31-57" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(chr7_genes)) {</span>
<span id="cb31-58"><a href="#cb31-58" aria-hidden="true" tabindex="-1"></a>  sig_gene_i <span class="ot">=</span> chr7_genes[i,<span class="st">&quot;ensembl_gene_id&quot;</span>]</span>
<span id="cb31-59"><a href="#cb31-59" aria-hidden="true" tabindex="-1"></a>  sig_label_i <span class="ot">=</span> chr7_genes[i,<span class="st">&quot;mgi_symbol&quot;</span>]</span>
<span id="cb31-60"><a href="#cb31-60" aria-hidden="true" tabindex="-1"></a>  pheno_col_i <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">colnames</span>(chr7_pheno) <span class="sc">==</span> sig_gene_i)</span>
<span id="cb31-61"><a href="#cb31-61" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb31-62"><a href="#cb31-62" aria-hidden="true" tabindex="-1"></a>  <span class="co"># RTG_a3</span></span>
<span id="cb31-63"><a href="#cb31-63" aria-hidden="true" tabindex="-1"></a>  RTG_a3_scan_i <span class="ot">=</span> <span class="fu">scan1</span>(<span class="at">genoprobs =</span> DO_str_2016_aprobs[,<span class="st">&quot;7&quot;</span>],</span>
<span id="cb31-64"><a href="#cb31-64" aria-hidden="true" tabindex="-1"></a>                        <span class="at">pheno =</span> RTG_a3_phenotype_mat[,<span class="st">&quot;RTG_a3&quot;</span>],</span>
<span id="cb31-65"><a href="#cb31-65" aria-hidden="true" tabindex="-1"></a>                        <span class="at">kinship =</span> DO_str_2016_kinship<span class="sc">$</span><span class="st">`</span><span class="at">7</span><span class="st">`</span>,</span>
<span id="cb31-66"><a href="#cb31-66" aria-hidden="true" tabindex="-1"></a>                        <span class="at">addcovar =</span> <span class="fu">cbind</span>(addcovar_i,</span>
<span id="cb31-67"><a href="#cb31-67" aria-hidden="true" tabindex="-1"></a>                                         chr7_pheno[,sig_gene_i]),</span>
<span id="cb31-68"><a href="#cb31-68" aria-hidden="true" tabindex="-1"></a>                        <span class="at">cores =</span> <span class="dv">9</span>)</span>
<span id="cb31-69"><a href="#cb31-69" aria-hidden="true" tabindex="-1"></a>  max_marker_RTG_a3_lod_i <span class="ot">=</span> <span class="fu">as.data.frame</span>(RTG_a3_scan_i)[max_marker_RTG_a3,<span class="dv">1</span>]</span>
<span id="cb31-70"><a href="#cb31-70" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="fu">exists</span>(<span class="st">&quot;max_markers_RTG_a3&quot;</span>)) {</span>
<span id="cb31-71"><a href="#cb31-71" aria-hidden="true" tabindex="-1"></a>    max_markers_RTG_a3 <span class="ot">=</span> <span class="fu">c</span>(max_markers_RTG_a3, max_marker_RTG_a3_lod_i)</span>
<span id="cb31-72"><a href="#cb31-72" aria-hidden="true" tabindex="-1"></a>    <span class="fu">names</span>(max_markers_RTG_a3)[i] <span class="ot">=</span> sig_label_i</span>
<span id="cb31-73"><a href="#cb31-73" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb31-74"><a href="#cb31-74" aria-hidden="true" tabindex="-1"></a>    max_markers_RTG_a3 <span class="ot">=</span> max_marker_RTG_a3_lod_i</span>
<span id="cb31-75"><a href="#cb31-75" aria-hidden="true" tabindex="-1"></a>    <span class="fu">names</span>(max_markers_RTG_a3) <span class="ot">=</span> sig_label_i</span>
<span id="cb31-76"><a href="#cb31-76" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb31-77"><a href="#cb31-77" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb31-78"><a href="#cb31-78" aria-hidden="true" tabindex="-1"></a>  <span class="co"># SENSI_cocaine</span></span>
<span id="cb31-79"><a href="#cb31-79" aria-hidden="true" tabindex="-1"></a>  SENSI_cocaine_scan_i <span class="ot">=</span> <span class="fu">scan1</span>(<span class="at">genoprobs =</span> DO_str_2016_aprobs[,<span class="st">&quot;7&quot;</span>],</span>
<span id="cb31-80"><a href="#cb31-80" aria-hidden="true" tabindex="-1"></a>                        <span class="at">pheno =</span> SENSI_cocaine_phenotype_mat[,<span class="st">&quot;SENSI_cocaine&quot;</span>],</span>
<span id="cb31-81"><a href="#cb31-81" aria-hidden="true" tabindex="-1"></a>                        <span class="at">kinship =</span> DO_str_2016_kinship<span class="sc">$</span><span class="st">`</span><span class="at">7</span><span class="st">`</span>,</span>
<span id="cb31-82"><a href="#cb31-82" aria-hidden="true" tabindex="-1"></a>                        <span class="at">addcovar =</span> <span class="fu">cbind</span>(addcovar_i,</span>
<span id="cb31-83"><a href="#cb31-83" aria-hidden="true" tabindex="-1"></a>                                         chr7_pheno[,sig_gene_i]),</span>
<span id="cb31-84"><a href="#cb31-84" aria-hidden="true" tabindex="-1"></a>                        <span class="at">cores =</span> <span class="dv">9</span>)</span>
<span id="cb31-85"><a href="#cb31-85" aria-hidden="true" tabindex="-1"></a>  max_marker_SENSI_cocaine_lod_i <span class="ot">=</span> <span class="fu">as.data.frame</span>(SENSI_cocaine_scan_i)[max_marker_SENSI_cocaine,<span class="dv">1</span>]</span>
<span id="cb31-86"><a href="#cb31-86" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="fu">exists</span>(<span class="st">&quot;max_markers_SENSI_cocaine&quot;</span>)) {</span>
<span id="cb31-87"><a href="#cb31-87" aria-hidden="true" tabindex="-1"></a>    max_markers_SENSI_cocaine <span class="ot">=</span> <span class="fu">c</span>(max_markers_SENSI_cocaine, max_marker_SENSI_cocaine_lod_i)</span>
<span id="cb31-88"><a href="#cb31-88" aria-hidden="true" tabindex="-1"></a>    <span class="fu">names</span>(max_markers_SENSI_cocaine)[i] <span class="ot">=</span> sig_label_i</span>
<span id="cb31-89"><a href="#cb31-89" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb31-90"><a href="#cb31-90" aria-hidden="true" tabindex="-1"></a>    max_markers_SENSI_cocaine <span class="ot">=</span> max_marker_SENSI_cocaine_lod_i</span>
<span id="cb31-91"><a href="#cb31-91" aria-hidden="true" tabindex="-1"></a>    <span class="fu">names</span>(max_markers_SENSI_cocaine) <span class="ot">=</span> sig_label_i</span>
<span id="cb31-92"><a href="#cb31-92" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb31-93"><a href="#cb31-93" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb31-94"><a href="#cb31-94" aria-hidden="true" tabindex="-1"></a>  <span class="co"># SENSI_saline</span></span>
<span id="cb31-95"><a href="#cb31-95" aria-hidden="true" tabindex="-1"></a>    SENSI_saline_scan_i <span class="ot">=</span> <span class="fu">scan1</span>(<span class="at">genoprobs =</span> DO_str_2016_aprobs[,<span class="st">&quot;7&quot;</span>],</span>
<span id="cb31-96"><a href="#cb31-96" aria-hidden="true" tabindex="-1"></a>                        <span class="at">pheno =</span> SENSI_saline_phenotype_mat[,<span class="st">&quot;SENSI_saline&quot;</span>],</span>
<span id="cb31-97"><a href="#cb31-97" aria-hidden="true" tabindex="-1"></a>                        <span class="at">kinship =</span> DO_str_2016_kinship<span class="sc">$</span><span class="st">`</span><span class="at">7</span><span class="st">`</span>,</span>
<span id="cb31-98"><a href="#cb31-98" aria-hidden="true" tabindex="-1"></a>                        <span class="at">addcovar =</span> <span class="fu">cbind</span>(addcovar_i,</span>
<span id="cb31-99"><a href="#cb31-99" aria-hidden="true" tabindex="-1"></a>                                         chr7_pheno[,sig_gene_i]),</span>
<span id="cb31-100"><a href="#cb31-100" aria-hidden="true" tabindex="-1"></a>                        <span class="at">cores =</span> <span class="dv">9</span>)</span>
<span id="cb31-101"><a href="#cb31-101" aria-hidden="true" tabindex="-1"></a>  max_marker_SENSI_saline_lod_i <span class="ot">=</span> <span class="fu">as.data.frame</span>(SENSI_saline_scan_i)[max_marker_SENSI_saline,<span class="dv">1</span>]</span>
<span id="cb31-102"><a href="#cb31-102" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="fu">exists</span>(<span class="st">&quot;max_markers_SENSI_saline&quot;</span>)) {</span>
<span id="cb31-103"><a href="#cb31-103" aria-hidden="true" tabindex="-1"></a>    max_markers_SENSI_saline <span class="ot">=</span> <span class="fu">c</span>(max_markers_SENSI_saline, max_marker_SENSI_saline_lod_i)</span>
<span id="cb31-104"><a href="#cb31-104" aria-hidden="true" tabindex="-1"></a>    <span class="fu">names</span>(max_markers_SENSI_saline)[i] <span class="ot">=</span> sig_label_i</span>
<span id="cb31-105"><a href="#cb31-105" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb31-106"><a href="#cb31-106" aria-hidden="true" tabindex="-1"></a>    max_markers_SENSI_saline <span class="ot">=</span> max_marker_SENSI_saline_lod_i</span>
<span id="cb31-107"><a href="#cb31-107" aria-hidden="true" tabindex="-1"></a>    <span class="fu">names</span>(max_markers_SENSI_saline) <span class="ot">=</span> sig_label_i</span>
<span id="cb31-108"><a href="#cb31-108" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb31-109"><a href="#cb31-109" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb31-110"><a href="#cb31-110" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>med_RTG_a3 <span class="ot">=</span> max_markers_RTG_a3[chr7_genes<span class="sc">$</span>mgi_symbol]</span>
<span id="cb31-111"><a href="#cb31-111" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>med_SENSI_cocaine <span class="ot">=</span> max_markers_SENSI_cocaine[chr7_genes<span class="sc">$</span>mgi_symbol]</span>
<span id="cb31-112"><a href="#cb31-112" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>med_SENSI_saline <span class="ot">=</span> max_markers_SENSI_saline[chr7_genes<span class="sc">$</span>mgi_symbol]</span>
<span id="cb31-113"><a href="#cb31-113" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-114"><a href="#cb31-114" aria-hidden="true" tabindex="-1"></a><span class="fu">rm</span>(<span class="at">list =</span> <span class="fu">ls</span>(<span class="at">pattern =</span> <span class="st">&quot;^max_markers_&quot;</span>))</span>
<span id="cb31-115"><a href="#cb31-115" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-116"><a href="#cb31-116" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculating a few variables that are useful for plotting</span></span>
<span id="cb31-117"><a href="#cb31-117" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>RTG_a3_lod_drop <span class="ot">=</span> max_marker_lod_RTG_a3 <span class="sc">-</span> chr7_genes<span class="sc">$</span>med_RTG_a3</span>
<span id="cb31-118"><a href="#cb31-118" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>SENSI_cocaine_lod_drop <span class="ot">=</span> max_marker_lod_SENSI_cocaine <span class="sc">-</span> chr7_genes<span class="sc">$</span>med_SENSI_cocaine</span>
<span id="cb31-119"><a href="#cb31-119" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>SENSI_saline_lod_drop <span class="ot">=</span> max_marker_lod_SENSI_saline <span class="sc">-</span> chr7_genes<span class="sc">$</span>med_SENSI_saline</span>
<span id="cb31-120"><a href="#cb31-120" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-121"><a href="#cb31-121" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting labels and colors and such.</span></span>
<span id="cb31-122"><a href="#cb31-122" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>label_RTG_a3 <span class="ot">=</span> <span class="fu">ifelse</span>((chr7_genes<span class="sc">$</span>RTG_a3_lod_drop <span class="sc">&gt;</span> <span class="fl">0.1</span>) <span class="sc">|</span> (chr7_genes<span class="sc">$</span>cor_RTG_a3 <span class="sc">&gt;</span> <span class="fl">0.3</span>),</span>
<span id="cb31-123"><a href="#cb31-123" aria-hidden="true" tabindex="-1"></a>                                 chr7_genes<span class="sc">$</span>mgi_symbol,<span class="st">&quot;&quot;</span>)</span>
<span id="cb31-124"><a href="#cb31-124" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>color_RTG_a3 <span class="ot">=</span> <span class="fu">ifelse</span>(chr7_genes<span class="sc">$</span>label_RTG_a3 <span class="sc">==</span> <span class="st">&quot;&quot;</span>, <span class="st">&quot;#666666&quot;</span>, <span class="st">&quot;#CC3333&quot;</span>)</span>
<span id="cb31-125"><a href="#cb31-125" aria-hidden="true" tabindex="-1"></a>chr7_genes[<span class="fu">which</span>(chr7_genes<span class="sc">$</span>mgi_symbol <span class="sc">==</span> <span class="st">&quot;Eed&quot;</span>),<span class="st">&quot;label_RTG_a3&quot;</span>] <span class="ot">=</span> <span class="st">&quot;Eed&quot;</span></span>
<span id="cb31-126"><a href="#cb31-126" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-127"><a href="#cb31-127" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>label_SENSI_cocaine <span class="ot">=</span> <span class="fu">ifelse</span>((chr7_genes<span class="sc">$</span>SENSI_cocaine_lod_drop <span class="sc">&gt;</span> <span class="fl">0.1</span>) <span class="sc">|</span> (chr7_genes<span class="sc">$</span>cor_SENSI_cocaine <span class="sc">&gt;</span> <span class="fl">0.3</span>),</span>
<span id="cb31-128"><a href="#cb31-128" aria-hidden="true" tabindex="-1"></a>                                 chr7_genes<span class="sc">$</span>mgi_symbol,<span class="st">&quot;&quot;</span>)</span>
<span id="cb31-129"><a href="#cb31-129" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>color_SENSI_cocaine <span class="ot">=</span> <span class="fu">ifelse</span>(chr7_genes<span class="sc">$</span>label_SENSI_cocaine <span class="sc">==</span> <span class="st">&quot;&quot;</span>, <span class="st">&quot;#666666&quot;</span>, <span class="st">&quot;#CC3333&quot;</span>)</span>
<span id="cb31-130"><a href="#cb31-130" aria-hidden="true" tabindex="-1"></a>chr7_genes[<span class="fu">which</span>(chr7_genes<span class="sc">$</span>mgi_symbol <span class="sc">==</span> <span class="st">&quot;Eed&quot;</span>),<span class="st">&quot;label_SENSI_cocaine&quot;</span>] <span class="ot">=</span> <span class="st">&quot;Eed&quot;</span></span>
<span id="cb31-131"><a href="#cb31-131" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-132"><a href="#cb31-132" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-133"><a href="#cb31-133" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>label_SENSI_saline <span class="ot">=</span> <span class="fu">ifelse</span>((chr7_genes<span class="sc">$</span>SENSI_saline_lod_drop <span class="sc">&gt;</span> <span class="fl">0.1</span>) <span class="sc">|</span> (chr7_genes<span class="sc">$</span>cor_SENSI_saline <span class="sc">&gt;</span> <span class="fl">0.3</span>),</span>
<span id="cb31-134"><a href="#cb31-134" aria-hidden="true" tabindex="-1"></a>                                 chr7_genes<span class="sc">$</span>mgi_symbol,<span class="st">&quot;&quot;</span>)</span>
<span id="cb31-135"><a href="#cb31-135" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>color_SENSI_saline <span class="ot">=</span> <span class="fu">ifelse</span>(chr7_genes<span class="sc">$</span>label_SENSI_saline <span class="sc">==</span> <span class="st">&quot;&quot;</span>, <span class="st">&quot;#666666&quot;</span>, <span class="st">&quot;#CC3333&quot;</span>)</span>
<span id="cb31-136"><a href="#cb31-136" aria-hidden="true" tabindex="-1"></a>chr7_genes[<span class="fu">which</span>(chr7_genes<span class="sc">$</span>mgi_symbol <span class="sc">==</span> <span class="st">&quot;Eed&quot;</span>),<span class="st">&quot;label_SENSI_saline&quot;</span>] <span class="ot">=</span> <span class="st">&quot;Eed&quot;</span></span>
<span id="cb31-137"><a href="#cb31-137" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-138"><a href="#cb31-138" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>in_exprs_set <span class="ot">=</span> chr7_genes<span class="sc">$</span>ensembl_gene_id <span class="sc">%in%</span> <span class="fu">colnames</span>(DO_str_2016_cross<span class="sc">$</span>pheno)</span>
<span id="cb31-139"><a href="#cb31-139" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-140"><a href="#cb31-140" aria-hidden="true" tabindex="-1"></a>chr7_genes<span class="sc">$</span>label_all <span class="ot">=</span> <span class="fu">ifelse</span>(chr7_genes<span class="sc">$</span>mgi_symbol <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">&quot;Eed&quot;</span>,<span class="st">&quot;Me3&quot;</span>),</span>
<span id="cb31-141"><a href="#cb31-141" aria-hidden="true" tabindex="-1"></a>                              chr7_genes<span class="sc">$</span>mgi_symbol, <span class="cn">NA</span>)</span></code></pre></div>
<p>Displaying correlations and mediation analysis (LOD score drops) with
the <code>RTG_a3</code> SNP mapping.</p>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a><span class="co"># 20</span></span>
<span id="cb32-2"><a href="#cb32-2" aria-hidden="true" tabindex="-1"></a>chr7_genes[<span class="fu">order</span>(chr7_genes<span class="sc">$</span>RTG_a3_lod_drop, <span class="at">decreasing =</span> <span class="cn">TRUE</span>),]</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["ensembl_gene_id"],"name":[1],"type":["chr"],"align":["left"]},{"label":["mgi_symbol"],"name":[2],"type":["chr"],"align":["left"]},{"label":["mgi_description"],"name":[3],"type":["chr"],"align":["left"]},{"label":["chromosome_name"],"name":[4],"type":["chr"],"align":["left"]},{"label":["start_position"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["end_position"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["strand"],"name":[7],"type":["int"],"align":["right"]},{"label":["cor_RTG_a3"],"name":[8],"type":["dbl"],"align":["right"]},{"label":["cor_SENSI_cocaine"],"name":[9],"type":["dbl"],"align":["right"]},{"label":["cor_SENSI_saline"],"name":[10],"type":["dbl"],"align":["right"]},{"label":["med_RTG_a3"],"name":[11],"type":["dbl"],"align":["right"]},{"label":["med_SENSI_cocaine"],"name":[12],"type":["dbl"],"align":["right"]},{"label":["med_SENSI_saline"],"name":[13],"type":["dbl"],"align":["right"]},{"label":["RTG_a3_lod_drop"],"name":[14],"type":["dbl"],"align":["right"]},{"label":["SENSI_cocaine_lod_drop"],"name":[15],"type":["dbl"],"align":["right"]},{"label":["SENSI_saline_lod_drop"],"name":[16],"type":["dbl"],"align":["right"]},{"label":["label_RTG_a3"],"name":[17],"type":["chr"],"align":["left"]},{"label":["color_RTG_a3"],"name":[18],"type":["chr"],"align":["left"]},{"label":["label_SENSI_cocaine"],"name":[19],"type":["chr"],"align":["left"]},{"label":["color_SENSI_cocaine"],"name":[20],"type":["chr"],"align":["left"]},{"label":["label_SENSI_saline"],"name":[21],"type":["chr"],"align":["left"]},{"label":["color_SENSI_saline"],"name":[22],"type":["chr"],"align":["left"]},{"label":["in_exprs_set"],"name":[23],"type":["lgl"],"align":["right"]},{"label":["label_all"],"name":[24],"type":["chr"],"align":["left"]}],"data":[{"1":"ENSMUSG00000030621","2":"Me3","3":"malic enzyme 3, NADP(+)-dependent, mitochondrial","4":"7","5":"89.63239","6":"89.85436","7":"1","8":"0.600417826","9":"0.355623671","10":"0.366597033","11":"2.573314","12":"2.620144","13":"2.807922","14":"6.944936e-01","15":"0.2523780809","16":"1.346227e-01","17":"Me3","18":"#CC3333","19":"Me3","20":"#CC3333","21":"Me3","22":"#CC3333","23":"TRUE","24":"Me3","_rn_":"7929"},{"1":"ENSMUSG00000109191","2":"Gm45131","3":"predicted gene 45131","4":"7","5":"92.06459","6":"92.06717","7":"1","8":"0.416834697","9":"0.361774223","10":"0.342266351","11":"2.621079","12":"2.224320","13":"2.610343","14":"6.467289e-01","15":"0.6482018125","16":"3.322019e-01","17":"Gm45131","18":"#CC3333","19":"Gm45131","20":"#CC3333","21":"Gm45131","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"46888"},{"1":"ENSMUSG00000109038","2":"Gm45120","3":"predicted gene 45120","4":"7","5":"89.96499","6":"89.96539","7":"-1","8":"0.815932705","9":"0.923859676","10":"0.930381337","11":"2.652996","12":"2.641292","13":"2.979527","14":"6.148123e-01","15":"0.2312299080","16":"-3.698204e-02","17":"Gm45120","18":"#CC3333","19":"Gm45120","20":"#CC3333","21":"Gm45120","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46741"},{"1":"ENSMUSG00000108528","2":"Gm8319","3":"predicted gene 8319","4":"7","5":"96.20290","6":"96.20354","7":"-1","8":"0.558106289","9":"0.573413114","10":"0.546852403","11":"2.750453","12":"2.562376","13":"2.603673","14":"5.173547e-01","15":"0.3101455550","16":"3.388711e-01","17":"Gm8319","18":"#CC3333","19":"Gm8319","20":"#CC3333","21":"Gm8319","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46263"},{"1":"ENSMUSG00000030638","2":"Sh3gl3","3":"SH3-domain GRB2-like 3","4":"7","5":"82.17384","6":"82.30742","7":"1","8":"0.285515287","9":"0.431474111","10":"0.456174073","11":"2.782769","12":"2.505688","13":"2.586839","14":"4.850388e-01","15":"0.3668336296","16":"3.557059e-01","17":"Sh3gl3","18":"#CC3333","19":"Sh3gl3","20":"#CC3333","21":"Sh3gl3","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"7934"},{"1":"ENSMUSG00000048897","2":"Zfp710","3":"zinc finger protein 710","4":"7","5":"80.02481","6":"80.09417","7":"1","8":"0.694733112","9":"0.717207144","10":"0.703536505","11":"2.794400","12":"2.650154","13":"2.436247","14":"4.734078e-01","15":"0.2223676260","16":"5.062972e-01","17":"Zfp710","18":"#CC3333","19":"Zfp710","20":"#CC3333","21":"Zfp710","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"14382"},{"1":"ENSMUSG00000025104","2":"Hdgfl3","3":"HDGF like 3","4":"7","5":"81.88125","6":"81.93447","7":"-1","8":"0.362281636","9":"0.143356327","10":"0.117772650","11":"2.798950","12":"2.473997","13":"2.720322","14":"4.688580e-01","15":"0.3985249580","16":"2.222221e-01","17":"Hdgfl3","18":"#CC3333","19":"Hdgfl3","20":"#CC3333","21":"Hdgfl3","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"4866"},{"1":"ENSMUSG00000004651","2":"Tyr","3":"tyrosinase","4":"7","5":"87.42477","6":"87.49351","7":"-1","8":"-0.162192833","9":"-0.203795884","10":"-0.212460606","11":"2.821314","12":"2.522022","13":"2.796982","14":"4.464935e-01","15":"0.3504996637","16":"1.455627e-01","17":"Tyr","18":"#CC3333","19":"Tyr","20":"#CC3333","21":"Tyr","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"737"},{"1":"ENSMUSG00000109150","2":"Gm45793","3":"predicted gene 45793","4":"7","5":"89.82722","6":"89.82856","7":"1","8":"-0.046247948","9":"-0.182941173","10":"-0.216905973","11":"2.828112","12":"2.887093","13":"2.933861","14":"4.396962e-01","15":"-0.0145711386","16":"8.683689e-03","17":"Gm45793","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46848"},{"1":"ENSMUSG00000108405","2":"Ywhaq-ps1","3":"tyrosine 3-monooxygenase/tryptophan 5-monooxygenase activation protein theta, pseudogene 1","4":"7","5":"97.59169","6":"97.59240","7":"-1","8":"0.223080363","9":"0.215028475","10":"0.266831068","11":"2.904536","12":"2.563861","13":"2.530827","14":"3.632719e-01","15":"0.3086602808","16":"4.117177e-01","17":"Ywhaq-ps1","18":"#CC3333","19":"Ywhaq-ps1","20":"#CC3333","21":"Ywhaq-ps1","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"46148"},{"1":"ENSMUSG00000108855","2":"Gm44917","3":"predicted gene 44917","4":"7","5":"82.67755","6":"82.67807","7":"1","8":"0.048212036","9":"0.067206926","10":"0.154335553","11":"2.937425","12":"2.737690","13":"2.620575","14":"3.303828e-01","15":"0.1348317106","16":"3.219693e-01","17":"Gm44917","18":"#CC3333","19":"Gm44917","20":"#CC3333","21":"Gm44917","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46565"},{"1":"ENSMUSG00000060688","2":"Olfr292","3":"olfactory receptor 292","4":"7","5":"86.68833","6":"86.69751","7":"1","8":"0.831769142","9":"0.913190863","10":"0.924627824","11":"2.955794","12":"2.530038","13":"2.591769","14":"3.120137e-01","15":"0.3424832542","16":"3.507755e-01","17":"Olfr292","18":"#CC3333","19":"Olfr292","20":"#CC3333","21":"Olfr292","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"17095"},{"1":"ENSMUSG00000025813","2":"Homer2","3":"homer scaffolding protein 2","4":"7","5":"81.60048","6":"81.70753","7":"-1","8":"0.267816592","9":"0.257400595","10":"0.345188643","11":"2.968777","12":"2.905959","13":"2.601974","14":"2.990311e-01","15":"-0.0334378182","16":"3.405703e-01","17":"Homer2","18":"#CC3333","19":"","20":"#666666","21":"Homer2","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"5207"},{"1":"ENSMUSG00000030606","2":"Hapln3","3":"hyaluronan and proteoglycan link protein 3","4":"7","5":"79.11510","6":"79.13102","7":"-1","8":"-0.209102914","9":"-0.266098499","10":"-0.276523167","11":"2.970993","12":"2.596176","13":"2.816156","14":"2.968149e-01","15":"0.2763458370","16":"1.263884e-01","17":"Hapln3","18":"#CC3333","19":"Hapln3","20":"#CC3333","21":"Hapln3","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"7917"},{"1":"ENSMUSG00000015709","2":"Arnt2","3":"aryl hydrocarbon receptor nuclear translocator 2","4":"7","5":"84.24628","6":"84.41018","7":"-1","8":"0.369624421","9":"0.477587945","10":"0.545739950","11":"2.975138","12":"2.847404","13":"2.890195","14":"2.926701e-01","15":"0.0251180525","16":"5.234929e-02","17":"Arnt2","18":"#CC3333","19":"Arnt2","20":"#CC3333","21":"Arnt2","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"1638"},{"1":"ENSMUSG00000025723","2":"Nmb","3":"neuromedin B","4":"7","5":"80.90223","6":"80.90508","7":"-1","8":"0.491310788","9":"0.579204988","10":"0.648912893","11":"2.989644","12":"2.918531","13":"2.712459","14":"2.781640e-01","15":"-0.0460094378","16":"2.300858e-01","17":"Nmb","18":"#CC3333","19":"Nmb","20":"#CC3333","21":"Nmb","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"5157"},{"1":"ENSMUSG00000108373","2":"Gm45168","3":"predicted gene 45168","4":"7","5":"79.57775","6":"79.57908","7":"-1","8":"0.037516313","9":"0.035108005","10":"-0.030061185","11":"3.008986","12":"2.863084","13":"2.789950","14":"2.588224e-01","15":"0.0094374777","16":"1.525949e-01","17":"Gm45168","18":"#CC3333","19":"","20":"#666666","21":"Gm45168","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46118"},{"1":"ENSMUSG00000097423","2":"Gm26522","3":"predicted gene, 26522","4":"7","5":"88.27879","6":"88.28353","7":"-1","8":"0.540513319","9":"0.488835387","10":"0.523444279","11":"3.015145","12":"3.095187","13":"2.820050","14":"2.526629e-01","15":"-0.2226648545","16":"1.224944e-01","17":"Gm26522","18":"#CC3333","19":"Gm26522","20":"#CC3333","21":"Gm26522","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"35939"},{"1":"ENSMUSG00000055407","2":"Map6","3":"microtubule-associated protein 6","4":"7","5":"99.26745","6":"99.33714","7":"1","8":"-0.196542458","9":"-0.200457986","10":"-0.182003258","11":"3.023077","12":"2.672952","13":"2.699586","14":"2.447311e-01","15":"0.1995698246","16":"2.429588e-01","17":"Map6","18":"#CC3333","19":"Map6","20":"#CC3333","21":"Map6","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"15933"},{"1":"ENSMUSG00000035314","2":"Gdpd5","3":"glycerophosphodiester phosphodiesterase domain containing 5","4":"7","5":"99.38141","6":"99.46188","7":"1","8":"-0.099474226","9":"-0.054687329","10":"0.012577263","11":"3.037611","12":"2.817985","13":"2.712108","14":"2.301970e-01","15":"0.0545370507","16":"2.304363e-01","17":"Gdpd5","18":"#CC3333","19":"","20":"#666666","21":"Gdpd5","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"9998"},{"1":"ENSMUSG00000030533","2":"Unc45a","3":"unc-45 myosin chaperone A","4":"7","5":"80.32529","6":"80.34100","7":"-1","8":"0.034076977","9":"0.099965044","10":"0.075148390","11":"3.037750","12":"2.926755","13":"2.860220","14":"2.300577e-01","15":"-0.0542337270","16":"8.232478e-02","17":"Unc45a","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7880"},{"1":"ENSMUSG00000030613","2":"Ccdc90b","3":"coiled-coil domain containing 90B","4":"7","5":"92.56115","6":"92.58229","7":"1","8":"-0.167732441","9":"-0.195787826","10":"-0.131143330","11":"3.039328","12":"3.255676","13":"2.759173","14":"2.284798e-01","15":"-0.3831545268","16":"1.833710e-01","17":"Ccdc90b","18":"#CC3333","19":"","20":"#666666","21":"Ccdc90b","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"7923"},{"1":"ENSMUSG00000057706","2":"Mex3b","3":"mex3 RNA binding family member B","4":"7","5":"82.86733","6":"82.87152","7":"1","8":"-0.244407910","9":"-0.254037630","10":"-0.212959735","11":"3.050188","12":"2.952798","13":"2.743827","14":"2.176201e-01","15":"-0.0802761132","16":"1.987170e-01","17":"Mex3b","18":"#CC3333","19":"","20":"#666666","21":"Mex3b","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"16432"},{"1":"ENSMUSG00000030616","2":"Sytl2","3":"synaptotagmin-like 2","4":"7","5":"90.30225","6":"90.41072","7":"1","8":"0.359826251","9":"0.270156214","10":"0.360009098","11":"3.053786","12":"2.852405","13":"2.786691","14":"2.140223e-01","15":"0.0201167105","16":"1.558531e-01","17":"Sytl2","18":"#CC3333","19":"","20":"#666666","21":"Sytl2","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"7926"},{"1":"ENSMUSG00000109452","2":"Gm44530","3":"predicted gene 44530","4":"7","5":"83.87436","6":"83.87792","7":"1","8":"0.089498095","9":"0.034528149","10":"0.047209158","11":"3.056168","12":"2.968975","13":"3.088469","14":"2.116402e-01","15":"-0.0964536656","16":"-1.459245e-01","17":"Gm44530","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47142"},{"1":"ENSMUSG00000109194","2":"Gm44570","3":"predicted gene 44570","4":"7","5":"81.61011","6":"81.61417","7":"1","8":"-0.055072532","9":"-0.083484383","10":"-0.005504820","11":"3.059740","12":"2.862892","13":"2.692116","14":"2.080684e-01","15":"0.0096293313","16":"2.504282e-01","17":"Gm44570","18":"#CC3333","19":"","20":"#666666","21":"Gm44570","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"46891"},{"1":"ENSMUSG00000108643","2":"Gm44931","3":"predicted gene 44931","4":"7","5":"80.60324","6":"80.60378","7":"-1","8":"0.486731012","9":"0.244847074","10":"0.239085688","11":"3.071554","12":"2.868384","13":"2.924841","14":"1.962542e-01","15":"0.0041378382","16":"1.770354e-02","17":"Gm44931","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46371"},{"1":"ENSMUSG00000109163","2":"3110009M11Rik","3":"RIKEN cDNA 3110009M11 gene","4":"7","5":"83.86459","6":"83.86548","7":"-1","8":"-0.130721618","9":"-0.152786068","10":"-0.150680013","11":"3.075906","12":"2.935237","13":"2.921882","14":"1.919023e-01","15":"-0.0627151134","16":"2.066248e-02","17":"3110009M11Rik","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46861"},{"1":"ENSMUSG00000050382","2":"Kif7","3":"kinesin family member 7","4":"7","5":"79.69810","6":"79.71572","7":"-1","8":"0.262245546","9":"0.145521756","10":"0.223620216","11":"3.076880","12":"2.811727","13":"2.940039","14":"1.909279e-01","15":"0.0607948675","16":"2.505469e-03","17":"Kif7","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"14780"},{"1":"ENSMUSG00000030753","2":"Thap12","3":"THAP domain containing 12","4":"7","5":"98.70310","6":"98.71806","7":"1","8":"-0.222608527","9":"-0.246522763","10":"-0.231218942","11":"3.098321","12":"2.815639","13":"2.770218","14":"1.694871e-01","15":"0.0568826803","16":"1.723260e-01","17":"Thap12","18":"#CC3333","19":"","20":"#666666","21":"Thap12","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"8006"},{"1":"ENSMUSG00000035704","2":"Alg8","3":"asparagine-linked glycosylation 8 (alpha-1,3-glucosyltransferase)","4":"7","5":"97.37161","6":"97.39218","7":"1","8":"-0.255404546","9":"-0.250190846","10":"-0.224498630","11":"3.104414","12":"2.850839","13":"2.753399","14":"1.633942e-01","15":"0.0216822394","16":"1.891453e-01","17":"Alg8","18":"#CC3333","19":"","20":"#666666","21":"Alg8","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"10123"},{"1":"ENSMUSG00000109479","2":"Gm44919","3":"predicted gene 44919","4":"7","5":"82.34227","6":"82.34334","7":"1","8":"-0.054335497","9":"0.022813600","10":"-0.002254276","11":"3.117871","12":"2.778423","13":"2.813180","14":"1.499367e-01","15":"0.0940986262","16":"1.293645e-01","17":"Gm44919","18":"#CC3333","19":"","20":"#666666","21":"Gm44919","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"47167"},{"1":"ENSMUSG00000038563","2":"Efl1","3":"elongation factor like GPTase 1","4":"7","5":"82.64861","6":"82.77785","7":"1","8":"0.197430378","9":"0.433479692","10":"0.402951260","11":"3.123553","12":"2.750133","13":"2.736122","14":"1.442548e-01","15":"0.1223884876","16":"2.064227e-01","17":"Efl1","18":"#CC3333","19":"Efl1","20":"#CC3333","21":"Efl1","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"11184"},{"1":"ENSMUSG00000030538","2":"Cib1","3":"calcium and integrin binding 1 (calmyrin)","4":"7","5":"80.22715","6":"80.23281","7":"-1","8":"0.043370013","9":"0.007825734","10":"-0.008074877","11":"3.133955","12":"2.681236","13":"2.774077","14":"1.338530e-01","15":"0.1912861656","16":"1.684678e-01","17":"Cib1","18":"#CC3333","19":"Cib1","20":"#CC3333","21":"Cib1","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"7883"},{"1":"ENSMUSG00000051451","2":"Crebzf","3":"CREB/ATF bZIP transcription factor","4":"7","5":"90.44273","6":"90.44799","7":"1","8":"-0.144270873","9":"-0.109144287","10":"-0.043185901","11":"3.141368","12":"2.788538","13":"2.931850","14":"1.264395e-01","15":"0.0839838034","16":"1.069462e-02","17":"Crebzf","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"15069"},{"1":"ENSMUSG00000059146","2":"Ntrk3","3":"neurotrophic tyrosine kinase, receptor, type 3","4":"7","5":"78.17596","6":"78.73801","7":"-1","8":"0.357657836","9":"0.576853178","10":"0.585217882","11":"3.142688","12":"2.684926","13":"2.750896","14":"1.251195e-01","15":"0.1875957915","16":"1.916486e-01","17":"Ntrk3","18":"#CC3333","19":"Ntrk3","20":"#CC3333","21":"Ntrk3","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"16740"},{"1":"ENSMUSG00000030532","2":"Hddc3","3":"HD domain containing 3","4":"7","5":"80.34310","6":"80.35055","7":"1","8":"-0.345176298","9":"-0.436263025","10":"-0.429438691","11":"3.152459","12":"2.846460","13":"2.904186","14":"1.153488e-01","15":"0.0260611714","16":"3.835868e-02","17":"Hddc3","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7879"},{"1":"ENSMUSG00000109382","2":"4930567K12Rik","3":"RIKEN cDNA 4930567K12 gene","4":"7","5":"92.09221","6":"92.11951","7":"-1","8":"-0.141964197","9":"-0.301567808","10":"-0.289605301","11":"3.152539","12":"2.693083","13":"2.902274","14":"1.152689e-01","15":"0.1794384793","16":"4.027095e-02","17":"4930567K12Rik","18":"#CC3333","19":"4930567K12Rik","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47073"},{"1":"ENSMUSG00000030623","2":"Prss23os","3":"protease, serine 23, opposite strand","4":"7","5":"89.51001","6":"89.51187","7":"1","8":"-0.207179363","9":"-0.205460752","10":"-0.191382185","11":"3.154199","12":"2.807646","13":"2.840019","14":"1.136089e-01","15":"0.0648760065","16":"1.025256e-01","17":"Prss23os","18":"#CC3333","19":"","20":"#666666","21":"Prss23os","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"7930"},{"1":"ENSMUSG00000108433","2":"Gm19761","3":"predicted gene, 19761","4":"7","5":"94.60202","6":"94.60241","7":"-1","8":"-0.016363213","9":"-0.162866031","10":"-0.169592359","11":"3.154874","12":"2.833917","13":"2.824777","14":"1.129343e-01","15":"0.0386049099","16":"1.177677e-01","17":"Gm19761","18":"#CC3333","19":"","20":"#666666","21":"Gm19761","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46175"},{"1":"ENSMUSG00000061787","2":"Rps17","3":"ribosomal protein S17","4":"7","5":"81.34273","6":"81.34525","7":"-1","8":"-0.268451032","9":"-0.263312272","10":"-0.266179980","11":"3.157424","12":"2.695292","13":"2.853943","14":"1.103840e-01","15":"0.1772294144","16":"8.860160e-02","17":"Rps17","18":"#CC3333","19":"Rps17","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"17313"},{"1":"ENSMUSG00000108878","2":"Gm49493","3":"predicted gene, 49493","4":"7","5":"83.89226","6":"83.89640","7":"1","8":"0.346740788","9":"0.253588350","10":"0.197111503","11":"3.157578","12":"2.811329","13":"2.974176","14":"1.102303e-01","15":"0.0611928094","16":"-3.163183e-02","17":"Gm49493","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46587"},{"1":"ENSMUSG00000109140","2":"Gm45089","3":"predicted gene 45089","4":"7","5":"82.17266","6":"82.17470","7":"-1","8":"-0.243336109","9":"-0.258916778","10":"-0.197553047","11":"3.157699","12":"2.960874","13":"2.745366","14":"1.101094e-01","15":"-0.0883527902","16":"1.971787e-01","17":"Gm45089","18":"#CC3333","19":"","20":"#666666","21":"Gm45089","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46840"},{"1":"ENSMUSG00000109279","2":"Gm45220","3":"predicted gene 45220","4":"7","5":"90.20128","6":"90.20379","7":"1","8":"-0.449163624","9":"-0.456878568","10":"-0.418314846","11":"3.162649","12":"2.484506","13":"2.930520","14":"1.051594e-01","15":"0.3880154520","16":"1.202460e-02","17":"Gm45220","18":"#CC3333","19":"Gm45220","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46974"},{"1":"ENSMUSG00000108776","2":"Gm45169","3":"predicted gene 45169","4":"7","5":"79.58665","6":"79.59261","7":"-1","8":"-0.133034183","9":"-0.186812792","10":"-0.194972020","11":"3.164368","12":"2.870963","13":"2.773323","14":"1.034400e-01","15":"0.0015583413","16":"1.692216e-01","17":"Gm45169","18":"#CC3333","19":"","20":"#666666","21":"Gm45169","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46489"},{"1":"ENSMUSG00000109450","2":"Gm39043","3":"predicted gene, 39043","4":"7","5":"83.81471","6":"83.83835","7":"1","8":"-0.257457902","9":"-0.262156747","10":"-0.188766920","11":"3.164488","12":"2.941858","13":"2.721704","14":"1.033200e-01","15":"-0.0693359478","16":"2.208404e-01","17":"Gm39043","18":"#CC3333","19":"","20":"#666666","21":"Gm39043","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"47140"},{"1":"ENSMUSG00000109243","2":"Gm45867","3":"predicted gene 45867","4":"7","5":"81.87420","6":"81.87652","7":"1","8":"-0.229888791","9":"-0.326901449","10":"-0.305945926","11":"3.165969","12":"3.020611","13":"2.940026","14":"1.018392e-01","15":"-0.1480891921","16":"2.518541e-03","17":"Gm45867","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46939"},{"1":"ENSMUSG00000109239","2":"Gm45176","3":"predicted gene 45176","4":"7","5":"91.13184","6":"91.13594","7":"1","8":"0.062191724","9":"-0.125140881","10":"-0.143132188","11":"3.168248","12":"3.093653","13":"3.141397","14":"9.956031e-02","15":"-0.2211311859","16":"-1.988524e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46935"},{"1":"ENSMUSG00000041343","2":"Ankrd42","3":"ankyrin repeat domain 42","4":"7","5":"92.58172","6":"92.63714","7":"-1","8":"-0.167929088","9":"-0.189264783","10":"-0.118972181","11":"3.170096","12":"2.923348","13":"2.872304","14":"9.771162e-02","15":"-0.0508265131","16":"7.024040e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"12212"},{"1":"ENSMUSG00000030609","2":"Aen","3":"apoptosis enhancing nuclease","4":"7","5":"78.89585","6":"78.91121","7":"1","8":"-0.291586944","9":"-0.296728616","10":"-0.249862230","11":"3.170170","12":"2.939946","13":"2.861311","14":"9.763745e-02","15":"-0.0674244924","16":"8.123364e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7919"},{"1":"ENSMUSG00000038459","2":"Abhd17c","3":"abhydrolase domain containing 17C","4":"7","5":"84.10936","6":"84.15189","7":"-1","8":"0.508907866","9":"0.243147514","10":"0.265512143","11":"3.170792","12":"2.848454","13":"2.924234","14":"9.701613e-02","15":"0.0240674696","16":"1.831088e-02","17":"Abhd17c","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11132"},{"1":"ENSMUSG00000030534","2":"Vps33b","3":"vacuolar protein sorting 33B","4":"7","5":"80.26965","6":"80.29175","7":"1","8":"0.130416486","9":"-0.066514425","10":"-0.002131725","11":"3.175337","12":"2.914986","13":"3.008304","14":"9.247123e-02","15":"-0.0424641092","16":"-6.575942e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7881"},{"1":"ENSMUSG00000049791","2":"Fzd4","3":"frizzled class receptor 4","4":"7","5":"89.40435","6":"89.41313","7":"1","8":"-0.362436715","9":"-0.290185052","10":"-0.267481037","11":"3.176176","12":"2.973155","13":"2.865283","14":"9.163244e-02","15":"-0.1006334230","16":"7.726137e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"14620"},{"1":"ENSMUSG00000061119","2":"Prcp","3":"prolylcarboxypeptidase (angiotensinase C)","4":"7","5":"92.87447","6":"92.93458","7":"1","8":"-0.246938549","9":"-0.274263636","10":"-0.217102049","11":"3.177473","12":"3.157450","13":"2.707276","14":"9.033465e-02","15":"-0.2849286022","16":"2.352685e-01","17":"","18":"#666666","19":"","20":"#666666","21":"Prcp","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"17182"},{"1":"ENSMUSG00000108460","2":"Gm39041","3":"predicted gene, 39041","4":"7","5":"79.29903","6":"79.30293","7":"1","8":"0.091817422","9":"0.290642302","10":"0.322310392","11":"3.181985","12":"2.877076","13":"2.958245","14":"8.582335e-02","15":"-0.0045543858","16":"-1.570082e-02","17":"","18":"#666666","19":"","20":"#666666","21":"Gm39041","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"46199"},{"1":"ENSMUSG00000108628","2":"Gm5037","3":"predicted gene 5037","4":"7","5":"95.91277","6":"95.91339","7":"-1","8":"0.271195021","9":"0.175478060","10":"0.206930876","11":"3.182477","12":"2.886618","13":"2.901523","14":"8.533072e-02","15":"-0.0140958969","16":"4.102109e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46359"},{"1":"ENSMUSG00000001773","2":"Folh1","3":"folate hydrolase 1","4":"7","5":"86.71898","6":"86.77594","7":"-1","8":"-0.218792080","9":"-0.194305487","10":"-0.143642765","11":"3.183633","12":"2.805714","13":"2.816360","14":"8.417510e-02","15":"0.0668072266","16":"1.261844e-01","17":"","18":"#666666","19":"","20":"#666666","21":"Folh1","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"313"},{"1":"ENSMUSG00000089453","2":"Gm24224","3":"predicted gene, 24224","4":"7","5":"79.35249","6":"79.35274","7":"-1","8":"0.456551437","9":"0.279968976","10":"0.281342608","11":"3.184322","12":"2.792319","13":"2.930878","14":"8.348600e-02","15":"0.0802022995","16":"1.166680e-02","17":"Gm24224","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"30433"},{"1":"ENSMUSG00000096033","2":"Gm5899","3":"predicted pseudogene 5899","4":"7","5":"94.19721","6":"94.19748","7":"1","8":"0.361973547","9":"0.243099464","10":"0.202831227","11":"3.185483","12":"2.863441","13":"2.911241","14":"8.232452e-02","15":"0.0090802637","16":"3.130327e-02","17":"Gm5899","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"34760"},{"1":"ENSMUSG00000030545","2":"Pex11a","3":"peroxisomal biogenesis factor 11 alpha","4":"7","5":"79.73596","6":"79.74313","7":"-1","8":"0.214999321","9":"0.075728913","10":"0.074419883","11":"3.185599","12":"2.738654","13":"2.890643","14":"8.220870e-02","15":"0.1338680412","16":"5.190119e-02","17":"","18":"#666666","19":"Pex11a","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7888"},{"1":"ENSMUSG00000015957","2":"Wnt11","3":"wingless-type MMTV integration site family, member 11","4":"7","5":"98.83511","6":"98.85519","7":"1","8":"0.364994407","9":"0.419471533","10":"0.379393618","11":"3.187307","12":"2.874126","13":"2.877523","14":"8.050120e-02","15":"-0.0016042274","16":"6.502157e-02","17":"Wnt11","18":"#CC3333","19":"Wnt11","20":"#CC3333","21":"Wnt11","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"1679"},{"1":"ENSMUSG00000098019","2":"Gm2546","3":"predicted gene 2546","4":"7","5":"89.60771","6":"89.60870","7":"1","8":"-0.255048459","9":"-0.317955463","10":"-0.286381525","11":"3.191407","12":"2.690117","13":"2.816861","14":"7.640145e-02","15":"0.1824047064","16":"1.256831e-01","17":"","18":"#666666","19":"Gm2546","20":"#CC3333","21":"Gm2546","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"36463"},{"1":"ENSMUSG00000030612","2":"Mrpl46","3":"mitochondrial ribosomal protein L46","4":"7","5":"78.77524","6":"78.78353","7":"-1","8":"0.154485436","9":"0.330673664","10":"0.338910024","11":"3.191431","12":"2.720196","13":"2.864607","14":"7.637725e-02","15":"0.1523254648","16":"7.793729e-02","17":"","18":"#666666","19":"Mrpl46","20":"#CC3333","21":"Mrpl46","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"7922"},{"1":"ENSMUSG00000064966","2":"Snord15b","3":"small nucleolar RNA, C/D box 14B","4":"7","5":"99.47956","6":"99.47971","7":"-1","8":"-0.179991635","9":"-0.108544226","10":"-0.075252758","11":"3.191854","12":"2.765195","13":"2.779556","14":"7.595355e-02","15":"0.1073267983","16":"1.629887e-01","17":"","18":"#666666","19":"Snord15b","20":"#CC3333","21":"Snord15b","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"18291"},{"1":"ENSMUSG00000108425","2":"Gm44706","3":"predicted gene 44706","4":"7","5":"79.74068","6":"79.74202","7":"1","8":"0.480111242","9":"0.275027563","10":"0.265856206","11":"3.192909","12":"2.941301","13":"2.958918","14":"7.489918e-02","15":"-0.0687789653","16":"-1.637377e-02","17":"Gm44706","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46167"},{"1":"ENSMUSG00000039391","2":"Ccdc81","3":"coiled-coil domain containing 81","4":"7","5":"89.86615","6":"89.90363","7":"-1","8":"0.027033002","9":"-0.079655419","10":"-0.004483190","11":"3.193049","12":"2.866154","13":"2.899665","14":"7.475862e-02","15":"0.0063679942","16":"4.287989e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11522"},{"1":"ENSMUSG00000081664","2":"Gm15544","3":"predicted gene 15544","4":"7","5":"80.42921","6":"80.43074","7":"1","8":"-0.330930817","9":"-0.348474900","10":"-0.298321800","11":"3.193425","12":"2.902495","13":"2.872643","14":"7.438285e-02","15":"-0.0299729828","16":"6.990168e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"24059"},{"1":"ENSMUSG00000109217","2":"Gm44676","3":"predicted gene 44676","4":"7","5":"91.60896","6":"91.61201","7":"1","8":"-0.298654152","9":"-0.266339826","10":"-0.225436969","11":"3.194100","12":"3.113496","13":"2.815215","14":"7.370805e-02","15":"-0.2409744406","16":"1.273295e-01","17":"","18":"#666666","19":"","20":"#666666","21":"Gm44676","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"46913"},{"1":"ENSMUSG00000108786","2":"Gm8309","3":"predicted gene 8309","4":"7","5":"95.46199","6":"95.46344","7":"1","8":"-0.354769310","9":"-0.446877780","10":"-0.435006139","11":"3.196928","12":"2.868012","13":"2.969025","14":"7.088014e-02","15":"0.0045098451","16":"-2.648099e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46499"},{"1":"ENSMUSG00000084821","2":"Gm15880","3":"predicted gene 15880","4":"7","5":"80.63602","6":"80.64438","7":"1","8":"0.084321635","9":"0.055426621","10":"0.007093799","11":"3.199146","12":"2.881886","13":"2.916775","14":"6.866178e-02","15":"-0.0093646295","16":"2.576921e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"26767"},{"1":"ENSMUSG00000109552","2":"Gm44531","3":"predicted gene 44531","4":"7","5":"83.86678","6":"83.86878","7":"1","8":"-0.156776244","9":"-0.186468255","10":"-0.186570706","11":"3.199531","12":"2.928953","13":"2.915598","14":"6.827717e-02","15":"-0.0564316568","16":"2.694656e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47237"},{"1":"ENSMUSG00000109431","2":"Gm45161","3":"predicted gene 45161","4":"7","5":"90.88544","6":"90.88771","7":"-1","8":"-0.396381049","9":"-0.453720239","10":"-0.414455551","11":"3.199983","12":"2.770553","13":"2.943339","14":"6.782489e-02","15":"0.1019689015","16":"-7.947927e-04","17":"","18":"#666666","19":"Gm45161","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47121"},{"1":"ENSMUSG00000109485","2":"Gm45016","3":"predicted gene 45016","4":"7","5":"81.78948","6":"81.79469","7":"1","8":"0.455228274","9":"0.344076553","10":"0.419866985","11":"3.207172","12":"2.893977","13":"2.877185","14":"6.063601e-02","15":"-0.0214558009","16":"6.535902e-02","17":"Gm45016","18":"#CC3333","19":"Gm45016","20":"#CC3333","21":"Gm45016","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"47172"},{"1":"ENSMUSG00000054061","2":"Gm9934","3":"predicted gene 9934","4":"7","5":"93.05208","6":"93.08103","7":"-1","8":"0.511870727","9":"0.302743814","10":"0.308433643","11":"3.208019","12":"2.793126","13":"2.878638","14":"5.978887e-02","15":"0.0793955335","16":"6.390683e-02","17":"Gm9934","18":"#CC3333","19":"Gm9934","20":"#CC3333","21":"Gm9934","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"15642"},{"1":"ENSMUSG00000030527","2":"Crtc3","3":"CREB regulated transcription coactivator 3","4":"7","5":"80.58663","6":"80.68888","7":"-1","8":"0.254540121","9":"0.078911010","10":"0.051323020","11":"3.208405","12":"3.145668","13":"3.011014","14":"5.940343e-02","15":"-0.2731467843","16":"-6.846921e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7876"},{"1":"ENSMUSG00000108956","2":"Gm45179","3":"predicted gene 45179","4":"7","5":"91.04900","6":"91.05334","7":"1","8":"-0.407131879","9":"-0.451640863","10":"-0.432980717","11":"3.211262","12":"2.860623","13":"2.951159","14":"5.654572e-02","15":"0.0118989439","16":"-8.614124e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46661"},{"1":"ENSMUSG00000030744","2":"Rps3","3":"ribosomal protein S3","4":"7","5":"99.47790","6":"99.48374","7":"-1","8":"-0.295601057","9":"-0.214407372","10":"-0.210895650","11":"3.215316","12":"2.918593","13":"2.886492","14":"5.249203e-02","15":"-0.0460711332","16":"5.605230e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7999"},{"1":"ENSMUSG00000108791","2":"1700011D18Rik","3":"RIKEN cDNA 1700011D18 gene","4":"7","5":"78.84725","6":"78.84753","7":"1","8":"-0.238300349","9":"-0.160084235","10":"-0.166381981","11":"3.215802","12":"2.874459","13":"2.956871","14":"5.200615e-02","15":"-0.0019371569","16":"-1.432632e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46503"},{"1":"ENSMUSG00000025102","2":"3110040N11Rik","3":"RIKEN cDNA 3110040N11 gene","4":"7","5":"81.78218","6":"81.78948","7":"-1","8":"-0.185746814","9":"-0.174523306","10":"-0.147746131","11":"3.217617","12":"2.943292","13":"2.870956","14":"5.019110e-02","15":"-0.0707699500","16":"7.158876e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"4864"},{"1":"ENSMUSG00000109125","2":"Gm45159","3":"predicted gene 45159","4":"7","5":"90.88707","6":"90.94005","7":"1","8":"-0.404417366","9":"-0.446948107","10":"-0.434407037","11":"3.217773","12":"2.971664","13":"2.963550","14":"5.003520e-02","15":"-0.0991426736","16":"-2.100507e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46825"},{"1":"ENSMUSG00000109244","2":"Gm44751","3":"predicted gene 44751","4":"7","5":"88.31148","6":"88.31586","7":"1","8":"-0.302141333","9":"-0.348460765","10":"-0.360736486","11":"3.217827","12":"2.885070","13":"2.950917","14":"4.998071e-02","15":"-0.0125480703","16":"-8.372518e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46940"},{"1":"ENSMUSG00000039428","2":"Tmem135","3":"transmembrane protein 135","4":"7","5":"89.13971","6":"89.40422","7":"-1","8":"-0.075391357","9":"-0.066532235","10":"-0.084717319","11":"3.219376","12":"2.978387","13":"2.935068","14":"4.843221e-02","15":"-0.1058651841","16":"7.476263e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11530"},{"1":"ENSMUSG00000048078","2":"Tenm4","3":"teneurin transmembrane protein 4","4":"7","5":"96.17125","6":"96.91109","7":"1","8":"0.083900035","9":"0.143069291","10":"0.196088476","11":"3.223017","12":"2.722220","13":"2.883589","14":"4.479079e-02","15":"0.1503016040","16":"5.895554e-02","17":"","18":"#666666","19":"Tenm4","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"14170"},{"1":"ENSMUSG00000046027","2":"Stard5","3":"StAR-related lipid transfer (START) domain containing 5","4":"7","5":"83.63196","6":"83.65313","7":"1","8":"-0.331568415","9":"-0.365320007","10":"-0.322192122","11":"3.224063","12":"2.855082","13":"2.930538","14":"4.374507e-02","15":"0.0174398745","16":"1.200648e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"13590"},{"1":"ENSMUSG00000097789","2":"Gm2115","3":"predicted gene 2115","4":"7","5":"84.52895","6":"84.58353","7":"1","8":"-0.188139487","9":"-0.203397446","10":"-0.147923636","11":"3.224593","12":"3.079589","13":"2.931494","14":"4.321528e-02","15":"-0.2070668667","16":"1.105070e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"36256"},{"1":"ENSMUSG00000109481","2":"Gm45130","3":"predicted gene 45130","4":"7","5":"91.81617","6":"91.81955","7":"1","8":"-0.315953993","9":"-0.343513236","10":"-0.360318947","11":"3.224598","12":"2.911323","13":"3.058179","14":"4.320977e-02","15":"-0.0388014654","16":"-1.156342e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47169"},{"1":"ENSMUSG00000109302","2":"Gm44967","3":"predicted gene 44967","4":"7","5":"80.99468","6":"80.99531","7":"1","8":"-0.259874325","9":"-0.270454186","10":"-0.201456048","11":"3.225777","12":"2.887762","13":"2.959617","14":"4.203137e-02","15":"-0.0152403916","16":"-1.707273e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46995"},{"1":"ENSMUSG00000035401","2":"Emsy","3":"EMSY, BRCA2-interacting transcriptional repressor","4":"7","5":"98.58714","6":"98.65678","7":"-1","8":"-0.438819854","9":"-0.449990376","10":"-0.400189331","11":"3.226110","12":"2.870629","13":"2.941865","14":"4.169841e-02","15":"0.0018924658","16":"6.791728e-04","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"10028"},{"1":"ENSMUSG00000108315","2":"Gm44601","3":"predicted gene 44601","4":"7","5":"94.36898","6":"94.37257","7":"-1","8":"-0.263511279","9":"-0.321154116","10":"-0.279305903","11":"3.226164","12":"2.854364","13":"2.870267","14":"4.164355e-02","15":"0.0181580689","16":"7.227789e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46061"},{"1":"ENSMUSG00000109387","2":"Gm45182","3":"predicted gene 45182","4":"7","5":"91.15860","6":"91.16100","7":"1","8":"-0.212098803","9":"-0.310430205","10":"-0.329922171","11":"3.226359","12":"3.121421","13":"3.029701","14":"4.144851e-02","15":"-0.2488997977","16":"-8.715672e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47077"},{"1":"ENSMUSG00000046591","2":"Ticrr","3":"TOPBP1-interacting checkpoint and replication regulator","4":"7","5":"79.66020","6":"79.69815","7":"1","8":"0.323304958","9":"0.478872100","10":"0.407486775","11":"3.228320","12":"2.785168","13":"2.870014","14":"3.948780e-02","15":"0.0873539370","16":"7.253007e-02","17":"Ticrr","18":"#CC3333","19":"Ticrr","20":"#CC3333","21":"Ticrr","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"13739"},{"1":"ENSMUSG00000052353","2":"Cemip","3":"cell migration inducing protein, hyaluronan binding","4":"7","5":"83.93286","6":"84.08650","7":"-1","8":"-0.138157122","9":"-0.159573773","10":"-0.087082345","11":"3.230950","12":"2.898760","13":"2.870854","14":"3.685772e-02","15":"-0.0262380625","16":"7.169048e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"15265"},{"1":"ENSMUSG00000030530","2":"Furin","3":"furin (paired basic amino acid cleaving enzyme)","4":"7","5":"80.38859","6":"80.40544","7":"-1","8":"-0.380926568","9":"-0.359165336","10":"-0.365891819","11":"3.232058","12":"2.791408","13":"2.932086","14":"3.575010e-02","15":"0.0811136870","16":"1.045853e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7878"},{"1":"ENSMUSG00000108555","2":"Gm18310","3":"predicted gene, 18310","4":"7","5":"80.34408","6":"80.34799","7":"-1","8":"-0.068524389","9":"-0.228812146","10":"-0.232682382","11":"3.233418","12":"2.944372","13":"3.054928","14":"3.438968e-02","15":"-0.0718506913","16":"-1.123838e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46289"},{"1":"ENSMUSG00000025726","2":"Slc28a1","3":"solute carrier family 28 (sodium-coupled nucleoside transporter), member 1","4":"7","5":"81.11480","6":"81.17042","7":"1","8":"-0.038125520","9":"-0.063346294","10":"-0.024448514","11":"3.234350","12":"2.858527","13":"2.872228","14":"3.345789e-02","15":"0.0139944316","16":"7.031632e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"5159"},{"1":"ENSMUSG00000097699","2":"5430400D12Rik","3":"RIKEN cDNA 5430400D12 gene","4":"7","5":"79.92072","6":"79.92252","7":"1","8":"-0.109843557","9":"-0.074870760","10":"-0.027028056","11":"3.235477","12":"2.942348","13":"3.030280","14":"3.233045e-02","15":"-0.0698262214","16":"-8.773508e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"36176"},{"1":"ENSMUSG00000087412","2":"Gm15501","3":"predicted pseudogene 15501","4":"7","5":"93.17898","6":"93.18418","7":"-1","8":"-0.102463493","9":"0.012082259","10":"-0.009890107","11":"3.237057","12":"2.807635","13":"2.901889","14":"3.075066e-02","15":"0.0648869218","16":"4.065539e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"28865"},{"1":"ENSMUSG00000005621","2":"Zfp592","3":"zinc finger protein 592","4":"7","5":"80.99368","6":"81.04516","7":"1","8":"0.170732306","9":"0.325461186","10":"0.357998255","11":"3.240084","12":"2.869852","13":"2.903080","14":"2.772385e-02","15":"0.0026693521","16":"3.946403e-02","17":"","18":"#666666","19":"Zfp592","20":"#CC3333","21":"Zfp592","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"872"},{"1":"ENSMUSG00000109395","2":"Gm45199","3":"predicted gene 45199","4":"7","5":"92.70677","6":"92.70828","7":"1","8":"-0.186475311","9":"-0.106557655","10":"-0.124357447","11":"3.240283","12":"2.796265","13":"2.805233","14":"2.752473e-02","15":"0.0762568615","16":"1.373118e-01","17":"","18":"#666666","19":"","20":"#666666","21":"Gm45199","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"47085"},{"1":"ENSMUSG00000109268","2":"9530078K11Rik","3":"RIKEN cDNA 9530078K11 gene","4":"7","5":"92.90218","6":"92.90287","7":"1","8":"-0.338119007","9":"-0.360000882","10":"-0.341177386","11":"3.240402","12":"2.861307","13":"3.006726","14":"2.740592e-02","15":"0.0112150029","16":"-6.418142e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46963"},{"1":"ENSMUSG00000109576","2":"Gm44704","3":"predicted gene 44704","4":"7","5":"89.21511","6":"89.21710","7":"-1","8":"-0.186092959","9":"-0.180110149","10":"-0.128709113","11":"3.241743","12":"2.971257","13":"2.919363","14":"2.606500e-02","15":"-0.0987358267","16":"2.318191e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47260"},{"1":"ENSMUSG00000025439","2":"Clns1a","3":"chloride channel, nucleotide-sensitive, 1A","4":"7","5":"97.69663","6":"97.72080","7":"1","8":"-0.145850744","9":"-0.240568044","10":"-0.186105631","11":"3.242289","12":"2.874316","13":"2.943571","14":"2.551904e-02","15":"-0.0017940446","16":"-1.026450e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"5049"},{"1":"ENSMUSG00000098041","2":"Gm26981","3":"predicted gene, 26981","4":"7","5":"92.72984","6":"92.73375","7":"1","8":"-0.311244759","9":"-0.295219537","10":"-0.255987144","11":"3.242303","12":"2.953969","13":"2.878147","14":"2.550471e-02","15":"-0.0814475631","16":"6.439709e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"36482"},{"1":"ENSMUSG00000097277","2":"2900076A07Rik","3":"RIKEN cDNA 2900076A07 gene","4":"7","5":"81.52355","6":"81.53150","7":"1","8":"-0.276519146","9":"-0.293619390","10":"-0.276459879","11":"3.242797","12":"2.868395","13":"2.985995","14":"2.501076e-02","15":"0.0041265235","16":"-4.345037e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"35807"},{"1":"ENSMUSG00000102555","2":"6430511E19Rik","3":"RIKEN cDNA 6430511E19 gene","4":"7","5":"92.63995","6":"92.64378","7":"1","8":"-0.119980435","9":"-0.130149908","10":"-0.134103460","11":"3.243400","12":"2.858641","13":"3.098624","14":"2.440755e-02","15":"0.0138811299","16":"-1.560797e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"40690"},{"1":"ENSMUSG00000108819","2":"Gm45177","3":"predicted gene 45177","4":"7","5":"91.13996","6":"91.14227","7":"1","8":"-0.426995126","9":"-0.432064627","10":"-0.390046307","11":"3.244198","12":"3.118122","13":"2.947361","14":"2.361004e-02","15":"-0.2455998829","16":"-4.816269e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46530"},{"1":"ENSMUSG00000086477","2":"Gm15506","3":"predicted gene 15506","4":"7","5":"98.68234","6":"98.70293","7":"-1","8":"-0.399134126","9":"-0.408482877","10":"-0.363916901","11":"3.245632","12":"2.947382","13":"2.903683","14":"2.217566e-02","15":"-0.0748608121","16":"3.886142e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"28100"},{"1":"ENSMUSG00000001741","2":"Il16","3":"interleukin 16","4":"7","5":"83.64283","6":"83.74573","7":"-1","8":"-0.138623025","9":"-0.276063795","10":"-0.260529235","11":"3.247025","12":"2.866402","13":"2.948018","14":"2.078273e-02","15":"0.0061198044","16":"-5.473100e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"305"},{"1":"ENSMUSG00000030611","2":"Mrps11","3":"mitochondrial ribosomal protein S11","4":"7","5":"78.78312","6":"78.79299","7":"1","8":"-0.149909539","9":"-0.271507641","10":"-0.262050900","11":"3.248226","12":"2.759820","13":"2.892223","14":"1.958184e-02","15":"0.1127017557","16":"5.032110e-02","17":"","18":"#666666","19":"Mrps11","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7921"},{"1":"ENSMUSG00000053528","2":"A530021J07Rik","3":"Riken cDNA A530021J07 gene","4":"7","5":"83.06178","6":"83.17943","7":"1","8":"-0.161662176","9":"-0.185877396","10":"-0.235864133","11":"3.248464","12":"2.804609","13":"2.898468","14":"1.934369e-02","15":"0.0679128466","16":"4.407694e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"15524"},{"1":"ENSMUSG00000074006","2":"Omp","3":"olfactory marker protein","4":"7","5":"98.14336","6":"98.14550","7":"-1","8":"-0.337683980","9":"-0.309247197","10":"-0.322841099","11":"3.248576","12":"2.941379","13":"2.933856","14":"1.923181e-02","15":"-0.0688575738","16":"8.688499e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"20667"},{"1":"ENSMUSG00000030617","2":"Ccdc83","3":"coiled-coil domain containing 83","4":"7","5":"90.22387","6":"90.26578","7":"-1","8":"-0.194958993","9":"-0.152922464","10":"-0.162528073","11":"3.249041","12":"2.883297","13":"2.938103","14":"1.876737e-02","15":"-0.0107754411","16":"4.441716e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"7927"},{"1":"ENSMUSG00000069972","2":"Rps13-ps2","3":"ribosomal protein S13, pseudogene 2","4":"7","5":"88.53045","6":"88.53090","7":"-1","8":"0.059146778","9":"0.064798038","10":"0.032213936","11":"3.249108","12":"2.783804","13":"2.893731","14":"1.869996e-02","15":"0.0887176485","16":"4.881358e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"19645"},{"1":"ENSMUSG00000030774","2":"Pak1","3":"p21 (RAC1) activated kinase 1","4":"7","5":"97.78854","6":"97.91238","7":"1","8":"-0.266286148","9":"-0.329750741","10":"-0.310540909","11":"3.249221","12":"2.914972","13":"2.951174","14":"1.858724e-02","15":"-0.0424507407","16":"-8.629320e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"8019"},{"1":"ENSMUSG00000064403","2":"Gm23928","3":"predicted gene, 23928","4":"7","5":"91.66715","6":"91.66729","7":"-1","8":"-0.048336316","9":"-0.149136384","10":"-0.122723759","11":"3.250530","12":"2.877446","13":"2.935215","14":"1.727826e-02","15":"-0.0049242924","16":"7.329435e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"17967"},{"1":"ENSMUSG00000108616","2":"Gm35040","3":"predicted gene, 35040","4":"7","5":"79.53604","6":"79.56003","7":"1","8":"-0.266185692","9":"-0.240611605","10":"-0.238843381","11":"3.251092","12":"2.768417","13":"2.916565","14":"1.671622e-02","15":"0.1041049850","16":"2.597989e-02","17":"","18":"#666666","19":"Gm35040","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46347"},{"1":"ENSMUSG00000109248","2":"Gm44993","3":"predicted gene 44993","4":"7","5":"83.80042","6":"83.83460","7":"-1","8":"-0.377041933","9":"-0.383409087","10":"-0.362601833","11":"3.253185","12":"2.847781","13":"2.866618","14":"1.462313e-02","15":"0.0247403882","16":"7.592691e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46944"},{"1":"ENSMUSG00000108984","2":"Gm44916","3":"predicted gene 44916","4":"7","5":"82.71559","6":"82.71779","7":"1","8":"-0.354588550","9":"-0.356364730","10":"-0.305232709","11":"3.253996","12":"2.879270","13":"2.990065","14":"1.381180e-02","15":"-0.0067484241","16":"-4.752071e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46689"},{"1":"ENSMUSG00000038623","2":"Tm6sf1","3":"transmembrane 6 superfamily member 1","4":"7","5":"81.85900","6":"81.88443","7":"1","8":"-0.198034000","9":"-0.209023839","10":"-0.214105110","11":"3.254456","12":"2.985960","13":"3.033772","14":"1.335244e-02","15":"-0.1134386523","16":"-9.122731e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11213"},{"1":"ENSMUSG00000097644","2":"Gm26862","3":"predicted gene, 26862","4":"7","5":"93.00698","6":"93.10479","7":"1","8":"-0.019873421","9":"-0.128044761","10":"-0.139262644","11":"3.254670","12":"2.791138","13":"2.970941","14":"1.313769e-02","15":"0.0813833302","16":"-2.839694e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"36127"},{"1":"ENSMUSG00000108603","2":"5330411O13Rik","3":"RIKEN cDNA 5330411O13 gene","4":"7","5":"78.81829","6":"78.82066","7":"1","8":"0.566462194","9":"0.586317224","10":"0.590253828","11":"3.256736","12":"2.842719","13":"2.960687","14":"1.107200e-02","15":"0.0298026856","16":"-1.814300e-02","17":"5330411O13Rik","18":"#CC3333","19":"5330411O13Rik","20":"#CC3333","21":"5330411O13Rik","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46334"},{"1":"ENSMUSG00000108993","2":"Gm44846","3":"predicted gene 44846","4":"7","5":"88.11916","6":"88.12240","7":"1","8":"-0.389058205","9":"-0.413895468","10":"-0.360033930","11":"3.256822","12":"2.820132","13":"2.945853","14":"1.098632e-02","15":"0.0523893289","16":"-3.308537e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46698"},{"1":"ENSMUSG00000109073","2":"Gm44975","3":"predicted gene 44975","4":"7","5":"99.12207","6":"99.12346","7":"-1","8":"-0.345026981","9":"-0.374048212","10":"-0.328644447","11":"3.257454","12":"2.871807","13":"2.941545","14":"1.035358e-02","15":"0.0007151291","16":"9.993208e-04","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46775"},{"1":"ENSMUSG00000109101","2":"Gm45132","3":"predicted gene 45132","4":"7","5":"98.67811","6":"98.68065","7":"-1","8":"0.209166695","9":"0.208371985","10":"0.222899735","11":"3.257542","12":"2.874567","13":"2.952404","14":"1.026562e-02","15":"-0.0020455343","16":"-9.859129e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46802"},{"1":"ENSMUSG00000089938","2":"Gm15744","3":"predicted gene 15744","4":"7","5":"89.76146","6":"89.76167","7":"-1","8":"-0.219831672","9":"-0.313912102","10":"-0.346683181","11":"3.258066","12":"2.883976","13":"2.944669","14":"9.742283e-03","15":"-0.0114542413","16":"-2.124447e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"30783"},{"1":"ENSMUSG00000108547","2":"Gm45186","3":"predicted gene 45186","4":"7","5":"97.65428","6":"97.65475","7":"-1","8":"-0.303557273","9":"-0.307053302","10":"-0.285806991","11":"3.258877","12":"2.843142","13":"2.775366","14":"8.931327e-03","15":"0.0293798737","16":"1.671787e-01","17":"","18":"#666666","19":"","20":"#666666","21":"Gm45186","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"46282"},{"1":"ENSMUSG00000063801","2":"Ap3s2","3":"adaptor-related protein complex 3, sigma 2 subunit","4":"7","5":"79.87533","6":"79.92065","7":"-1","8":"-0.250651857","9":"-0.287972937","10":"-0.229387949","11":"3.259136","12":"2.870760","13":"3.089748","14":"8.671573e-03","15":"0.0017617875","16":"-1.472035e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"17779"},{"1":"ENSMUSG00000109461","2":"Gm44848","3":"predicted gene 44848","4":"7","5":"88.06306","6":"88.06629","7":"1","8":"-0.008794755","9":"0.147611225","10":"0.126581584","11":"3.260197","12":"2.871528","13":"2.939063","14":"7.611097e-03","15":"0.0009938752","16":"3.481200e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47151"},{"1":"ENSMUSG00000038663","2":"Fsd2","3":"fibronectin type III and SPRY domain containing 2","4":"7","5":"81.53331","6":"81.56694","7":"-1","8":"-0.076911649","9":"-0.125284276","10":"-0.122565878","11":"3.260336","12":"2.911637","13":"2.905810","14":"7.472095e-03","15":"-0.0391154342","16":"3.673399e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11228"},{"1":"ENSMUSG00000083774","2":"Gm7180","3":"predicted pseudogene 7180","4":"7","5":"81.22236","6":"81.22287","7":"-1","8":"0.792861051","9":"0.822885593","10":"0.872721473","11":"3.260406","12":"2.877851","13":"2.995600","14":"7.401846e-03","15":"-0.0053293656","16":"-5.305596e-02","17":"Gm7180","18":"#CC3333","19":"Gm7180","20":"#CC3333","21":"Gm7180","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"25892"},{"1":"ENSMUSG00000039062","2":"Anpep","3":"alanyl (membrane) aminopeptidase","4":"7","5":"79.82180","6":"79.86106","7":"-1","8":"-0.336731165","9":"-0.384054607","10":"-0.340339320","11":"3.260513","12":"2.990758","13":"2.893945","14":"7.295087e-03","15":"-0.1182360863","16":"4.859901e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11382"},{"1":"ENSMUSG00000030549","2":"Rhcg","3":"Rhesus blood group-associated C glycoprotein","4":"7","5":"79.59336","6":"79.61766","7":"-1","8":"-0.201621690","9":"-0.177086397","10":"-0.220740025","11":"3.260889","12":"2.869610","13":"2.965837","14":"6.919197e-03","15":"0.0029119880","16":"-2.329292e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"7890"},{"1":"ENSMUSG00000109005","2":"Gm45221","3":"predicted gene 45221","4":"7","5":"90.18593","6":"90.18820","7":"1","8":"-0.408255511","9":"-0.412019305","10":"-0.376950901","11":"3.261336","12":"2.660800","13":"2.951246","14":"6.471925e-03","15":"0.2117214438","16":"-8.701348e-03","17":"","18":"#666666","19":"Gm45221","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46708"},{"1":"ENSMUSG00000109383","2":"Gm36584","3":"predicted gene, 36584","4":"7","5":"82.85250","6":"82.86707","7":"-1","8":"-0.122464243","9":"-0.112219724","10":"-0.127259545","11":"3.262069","12":"2.882533","13":"2.912826","14":"5.738771e-03","15":"-0.0100117278","16":"2.971807e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47074"},{"1":"ENSMUSG00000109290","2":"Phgdh-ps1","3":"3-phosphoglycerate dehydrogenase, pseudogene 1","4":"7","5":"83.71097","6":"83.71172","7":"-1","8":"-0.296475623","9":"-0.333429144","10":"-0.330531884","11":"3.262102","12":"2.926004","13":"3.008534","14":"5.705592e-03","15":"-0.0534819792","16":"-6.598904e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46984"},{"1":"ENSMUSG00000038886","2":"Man2a2","3":"mannosidase 2, alpha 2","4":"7","5":"80.34910","6":"80.37138","7":"-1","8":"-0.075457927","9":"-0.014507671","10":"0.059572330","11":"3.262271","12":"2.835230","13":"2.970278","14":"5.536548e-03","15":"0.0372920911","16":"-2.773389e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11321"},{"1":"ENSMUSG00000097685","2":"Gm26646","3":"predicted gene, 26646","4":"7","5":"79.91694","6":"79.92064","7":"1","8":"-0.213469115","9":"-0.205105255","10":"-0.175916096","11":"3.262388","12":"2.956500","13":"2.923975","14":"5.419932e-03","15":"-0.0839784786","16":"1.856961e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"36165"},{"1":"ENSMUSG00000108890","2":"Gm44847","3":"predicted gene 44847","4":"7","5":"88.08281","6":"88.08480","7":"1","8":"-0.381631633","9":"-0.385855568","10":"-0.334113079","11":"3.263505","12":"2.867068","13":"2.874903","14":"4.302481e-03","15":"0.0054539566","16":"6.764169e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46599"},{"1":"ENSMUSG00000041328","2":"Pcf11","3":"PCF11 cleavage and polyadenylation factor subunit","4":"7","5":"92.64354","6":"92.66993","7":"-1","8":"-0.281949651","9":"-0.301182385","10":"-0.283496979","11":"3.264623","12":"2.886183","13":"2.935852","14":"3.185108e-03","15":"-0.0136609751","16":"6.692056e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"12208"},{"1":"ENSMUSG00000097912","2":"6330403N20Rik","3":"RIKEN cDNA 6330403N20 gene","4":"7","5":"80.29773","6":"80.29962","7":"-1","8":"-0.309700323","9":"-0.231801778","10":"-0.252534216","11":"3.266521","12":"3.092669","13":"2.999955","14":"1.286775e-03","15":"-0.2201476783","16":"-5.741061e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"36362"},{"1":"ENSMUSG00000109428","2":"Gm45698","3":"predicted gene 45698","4":"7","5":"81.59089","6":"81.59203","7":"1","8":"-0.345492908","9":"-0.326685899","10":"-0.296145926","11":"3.266538","12":"2.845157","13":"2.940146","14":"1.270193e-03","15":"0.0273647848","16":"2.398410e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47118"},{"1":"ENSMUSG00000011154","2":"Cfap161","3":"cilia and flagella associated protein 161","4":"7","5":"83.77410","6":"83.79488","7":"-1","8":"-0.253620670","9":"-0.272877137","10":"-0.243135521","11":"3.267310","12":"2.868323","13":"2.937706","14":"4.978972e-04","15":"0.0041983095","16":"4.838244e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"1355"},{"1":"ENSMUSG00000030761","2":"Myo7a","3":"myosin VIIA","4":"7","5":"98.05106","6":"98.11952","7":"-1","8":"0.162078863","9":"-0.049478219","10":"0.003887768","11":"3.267581","12":"2.894036","13":"2.929362","14":"2.273263e-04","15":"-0.0215146645","16":"1.318269e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"8011"},{"1":"ENSMUSG00000025133","2":"Ints4","3":"integrator complex subunit 4","4":"7","5":"97.48096","6":"97.54139","7":"1","8":"-0.374635412","9":"-0.362560702","10":"-0.360820232","11":"3.267586","12":"2.794617","13":"2.944378","14":"2.216703e-04","15":"0.0779051163","16":"-1.833964e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"4873"},{"1":"ENSMUSG00000108630","2":"Gm44607","3":"predicted gene 44607","4":"7","5":"78.21161","6":"78.21270","7":"-1","8":"-0.060117288","9":"0.028866712","10":"0.036798412","11":"3.267691","12":"2.896374","13":"2.936025","14":"1.169557e-04","15":"-0.0238526601","16":"6.519570e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46361"},{"1":"ENSMUSG00000081747","2":"Rpl7a-ps9","3":"ribosomal protein L7A, pseudogene 9","4":"7","5":"81.44592","6":"81.44670","7":"1","8":"-0.247348396","9":"-0.214283043","10":"-0.168509265","11":"3.267712","12":"2.846569","13":"2.939530","14":"9.577984e-05","15":"0.0259527411","16":"3.014013e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"24135"},{"1":"ENSMUSG00000038763","2":"Alpk3","3":"alpha-kinase 3","4":"7","5":"81.05760","6":"81.10561","7":"1","8":"-0.279211813","9":"-0.282056501","10":"-0.308457829","11":"3.267730","12":"2.836272","13":"2.907908","14":"7.778691e-05","15":"0.0362492945","16":"3.463599e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11272"},{"1":"ENSMUSG00000045467","2":"Ttll13","3":"tubulin tyrosine ligase-like family, member 13","4":"7","5":"80.24638","6":"80.26082","7":"1","8":"-0.328778945","9":"-0.350320156","10":"-0.305390939","11":"3.267833","12":"2.952737","13":"2.969433","14":"-2.522906e-05","15":"-0.0802152738","16":"-2.688851e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"13441"},{"1":"ENSMUSG00000109074","2":"Gm45162","3":"predicted gene 45162","4":"7","5":"90.81154","6":"90.81216","7":"-1","8":"-0.153141220","9":"-0.001652146","10":"-0.026556558","11":"3.268256","12":"2.941209","13":"2.949900","14":"-4.478225e-04","15":"-0.0686874094","16":"-7.355439e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46776"},{"1":"ENSMUSG00000038503","2":"Mesd","3":"mesoderm development LRP chaperone","4":"7","5":"83.88447","6":"83.90153","7":"1","8":"-0.344137643","9":"-0.346163294","10":"-0.311971866","11":"3.268275","12":"2.866470","13":"2.946816","14":"-4.673381e-04","15":"0.0060517257","16":"-4.271239e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11149"},{"1":"ENSMUSG00000108670","2":"Gm44815","3":"predicted gene 44815","4":"7","5":"78.92225","6":"78.92389","7":"-1","8":"-0.401518339","9":"-0.394942082","10":"-0.369642967","11":"3.268417","12":"2.870642","13":"2.926044","14":"-6.094970e-04","15":"0.0018801159","16":"1.650004e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46394"},{"1":"ENSMUSG00000025103","2":"Btbd1","3":"BTB (POZ) domain containing 1","4":"7","5":"81.79207","6":"81.82943","7":"-1","8":"-0.244869760","9":"-0.272500998","10":"-0.212415383","11":"3.268863","12":"2.854685","13":"2.874408","14":"-1.055152e-03","15":"0.0178369287","16":"6.813698e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"4865"},{"1":"ENSMUSG00000082780","2":"Rps11-ps5","3":"ribosomal protein S11, pseudogene 5","4":"7","5":"96.43555","6":"96.43602","7":"-1","8":"-0.361147483","9":"-0.355869932","10":"-0.310570257","11":"3.269553","12":"2.878275","13":"2.960842","14":"-1.745153e-03","15":"-0.0057536530","16":"-1.829760e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"25036"},{"1":"ENSMUSG00000030760","2":"Acer3","3":"alkaline ceramidase 3","4":"7","5":"98.20639","6":"98.32121","7":"-1","8":"-0.393336935","9":"-0.407057105","10":"-0.358467011","11":"3.269861","12":"2.872019","13":"2.970382","14":"-2.053396e-03","15":"0.0005022996","16":"-2.783746e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"8010"},{"1":"ENSMUSG00000108970","2":"Gm45906","3":"predicted gene 45906","4":"7","5":"81.49243","6":"81.49386","7":"-1","8":"-0.224264149","9":"-0.208586440","10":"-0.197417865","11":"3.270216","12":"2.819513","13":"2.972781","14":"-2.408139e-03","15":"0.0530086707","16":"-3.023698e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46675"},{"1":"ENSMUSG00000104453","2":"Gm37829","3":"predicted gene, 37829","4":"7","5":"81.53031","6":"81.53252","7":"-1","8":"-0.383451621","9":"-0.412120765","10":"-0.378500945","11":"3.270572","12":"2.806030","13":"2.943807","14":"-2.763668e-03","15":"0.0664920147","16":"-1.262871e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"42437"},{"1":"ENSMUSG00000108365","2":"Gm44951","3":"predicted gene 44951","4":"7","5":"80.03670","6":"80.03873","7":"1","8":"-0.356841250","9":"-0.359038574","10":"-0.359059887","11":"3.270874","12":"3.038162","13":"2.934006","14":"-3.065564e-03","15":"-0.1656402423","16":"8.538228e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46111"},{"1":"ENSMUSG00000025722","2":"Wdr73","3":"WD repeat domain 73","4":"7","5":"80.89072","6":"80.90127","7":"-1","8":"-0.355829711","9":"-0.351787853","10":"-0.343540283","11":"3.270978","12":"2.869200","13":"2.913088","14":"-3.170530e-03","15":"0.0033212109","16":"2.945643e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"5156"},{"1":"ENSMUSG00000044362","2":"Ccdc89","3":"coiled-coil domain containing 89","4":"7","5":"90.42658","6":"90.42867","7":"1","8":"-0.134444833","9":"-0.131295447","10":"-0.067821285","11":"3.271029","12":"2.870446","13":"2.949752","14":"-3.220670e-03","15":"0.0020760868","16":"-7.207951e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"13124"},{"1":"ENSMUSG00000109505","2":"Gm44677","3":"predicted gene 44677","4":"7","5":"91.62727","6":"91.63101","7":"1","8":"-0.028381569","9":"-0.184222484","10":"-0.208598669","11":"3.271385","12":"2.995826","13":"3.105332","14":"-3.577431e-03","15":"-0.1233046074","16":"-1.627875e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47191"},{"1":"ENSMUSG00000085236","2":"2610206C17Rik","3":"RIKEN cDNA 2610206C17 gene","4":"7","5":"84.68964","6":"84.77905","7":"1","8":"-0.297001605","9":"-0.200595814","10":"-0.185876217","11":"3.271517","12":"2.879274","13":"2.925859","14":"-3.708770e-03","15":"-0.0067520321","16":"1.668533e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"27100"},{"1":"ENSMUSG00000109013","2":"Gm44723","3":"predicted gene 44723","4":"7","5":"83.37914","6":"83.38314","7":"-1","8":"-0.395422007","9":"-0.368122225","10":"-0.366499322","11":"3.272026","12":"2.843767","13":"2.961402","14":"-4.217675e-03","15":"0.0287543216","16":"-1.885761e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46716"},{"1":"ENSMUSG00000108532","2":"Gm32647","3":"predicted gene, 32647","4":"7","5":"94.04269","6":"94.70917","7":"1","8":"-0.439337146","9":"-0.435724580","10":"-0.413193491","11":"3.272030","12":"2.982273","13":"2.988365","14":"-4.221803e-03","15":"-0.1097511534","16":"-4.582017e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46267"},{"1":"ENSMUSG00000035642","2":"Aamdc","3":"adipogenesis associated Mth938 domain containing","4":"7","5":"97.55033","6":"97.57950","7":"-1","8":"-0.383901343","9":"-0.373521275","10":"-0.366415616","11":"3.272031","12":"2.862746","13":"2.987018","14":"-4.223481e-03","15":"0.0097756215","16":"-4.447365e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"10106"},{"1":"ENSMUSG00000097162","2":"2310010J17Rik","3":"RIKEN cDNA 2310010J17 gene","4":"7","5":"90.12482","6":"90.12998","7":"-1","8":"-0.328753013","9":"-0.351932912","10":"-0.336061038","11":"3.272188","12":"2.835996","13":"2.966342","14":"-4.379597e-03","15":"0.0365260096","16":"-2.379741e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"35706"},{"1":"ENSMUSG00000038943","2":"Prc1","3":"protein regulator of cytokinesis 1","4":"7","5":"80.29445","6":"80.31626","7":"1","8":"-0.281340207","9":"-0.289761040","10":"-0.288675941","11":"3.274072","12":"3.083832","13":"2.912882","14":"-6.264213e-03","15":"-0.2113103956","16":"2.966228e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11339"},{"1":"ENSMUSG00000045795","2":"Whamm","3":"WAS protein homolog associated with actin, golgi membranes and microtubules","4":"7","5":"81.57127","6":"81.59684","7":"1","8":"-0.371500458","9":"-0.342315270","10":"-0.316991068","11":"3.274452","12":"2.768025","13":"2.888896","14":"-6.644181e-03","15":"0.1044964495","16":"5.364832e-02","17":"","18":"#666666","19":"Whamm","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"13533"},{"1":"ENSMUSG00000039202","2":"Abhd2","3":"abhydrolase domain containing 2","4":"7","5":"79.27320","6":"79.36551","7":"1","8":"0.724047996","9":"0.821914165","10":"0.833482604","11":"3.274525","12":"3.169825","13":"3.209019","14":"-6.717273e-03","15":"-0.2973032284","16":"-2.664743e-01","17":"Abhd2","18":"#CC3333","19":"Abhd2","20":"#CC3333","21":"Abhd2","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"11451"},{"1":"ENSMUSG00000109037","2":"C030038I04Rik","3":"RIKEN cDNA C030038I04 gene","4":"7","5":"91.68947","6":"91.69021","7":"1","8":"-0.394212987","9":"-0.341882002","10":"-0.355136610","11":"3.274540","12":"3.017693","13":"2.958280","14":"-6.731980e-03","15":"-0.1451713659","16":"-1.573572e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46740"},{"1":"ENSMUSG00000108893","2":"Gm44927","3":"predicted gene 44927","4":"7","5":"87.89075","6":"87.89146","7":"1","8":"-0.236369817","9":"-0.257205081","10":"-0.185476237","11":"3.274888","12":"2.867399","13":"2.793756","14":"-7.080303e-03","15":"0.0051228830","16":"1.487883e-01","17":"","18":"#666666","19":"","20":"#666666","21":"Gm44927","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"46602"},{"1":"ENSMUSG00000091989","2":"Ndufab1-ps","3":"NADH:ubiquinone oxidoreductase subunit AB1b","4":"7","5":"80.50779","6":"80.50825","7":"1","8":"-0.258598362","9":"-0.219907103","10":"-0.248091351","11":"3.275415","12":"2.847657","13":"2.936547","14":"-7.607098e-03","15":"0.0248646946","16":"5.997275e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31778"},{"1":"ENSMUSG00000039194","2":"Rlbp1","3":"retinaldehyde binding protein 1","4":"7","5":"79.37487","6":"79.38705","7":"-1","8":"-0.007437470","9":"0.018782872","10":"0.013082816","11":"3.276204","12":"2.855143","13":"2.978865","14":"-8.395655e-03","15":"0.0173785891","16":"-3.632064e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11443"},{"1":"ENSMUSG00000092071","2":"A230065N10Rik","3":"RIKEN cDNA A230065N10 gene","4":"7","5":"89.63210","6":"89.63312","7":"-1","8":"0.120638555","9":"-0.063799851","10":"-0.079542825","11":"3.276694","12":"2.842332","13":"2.946826","14":"-8.885640e-03","15":"0.0301896033","16":"-4.281641e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"31813"},{"1":"ENSMUSG00000035547","2":"Capn5","3":"calpain 5","4":"7","5":"98.12156","6":"98.17827","7":"-1","8":"-0.378097175","9":"-0.410983814","10":"-0.376076014","11":"3.276759","12":"2.862745","13":"2.914777","14":"-8.951028e-03","15":"0.0097767034","16":"2.776781e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"10073"},{"1":"ENSMUSG00000108826","2":"4931412I15Rik","3":"RIKEN cDNA 4931412I15 gene","4":"7","5":"92.08109","6":"92.08254","7":"-1","8":"-0.418071404","9":"-0.405672946","10":"-0.405276243","11":"3.278422","12":"2.751115","13":"2.978837","14":"-1.061396e-02","15":"0.1214068820","16":"-3.629274e-02","17":"","18":"#666666","19":"4931412I15Rik","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46536"},{"1":"ENSMUSG00000025586","2":"Cpeb1","3":"cytoplasmic polyadenylation element binding protein 1","4":"7","5":"81.34703","6":"81.45547","7":"-1","8":"-0.359699256","9":"-0.348584324","10":"-0.327951963","11":"3.279156","12":"2.878863","13":"2.941194","14":"-1.134790e-02","15":"-0.0063416217","16":"1.350651e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"5126"},{"1":"ENSMUSG00000053158","2":"Fes","3":"feline sarcoma oncogene","4":"7","5":"80.37776","6":"80.38795","7":"-1","8":"0.218712435","9":"0.208807323","10":"0.156945031","11":"3.279502","12":"2.874137","13":"2.932682","14":"-1.169418e-02","15":"-0.0016156134","16":"9.862426e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"15443"},{"1":"ENSMUSG00000109390","2":"Gm45718","3":"predicted gene 45718","4":"7","5":"80.91037","6":"80.91417","7":"-1","8":"-0.397586041","9":"-0.433764066","10":"-0.399855656","11":"3.280078","12":"2.960381","13":"3.046079","14":"-1.226979e-02","15":"-0.0878597729","16":"-1.035343e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47080"},{"1":"ENSMUSG00000109506","2":"Gm44724","3":"predicted gene 44724","4":"7","5":"83.32774","6":"83.36943","7":"-1","8":"-0.417496192","9":"-0.432112818","10":"-0.406385831","11":"3.281418","12":"2.950697","13":"2.923846","14":"-1.361002e-02","15":"-0.0781750150","16":"1.869814e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47192"},{"1":"ENSMUSG00000030643","2":"Rab30","3":"RAB30, member RAS oncogene family","4":"7","5":"92.74160","6":"92.84453","7":"1","8":"-0.308711468","9":"-0.303816028","10":"-0.308263735","11":"3.281470","12":"2.736969","13":"2.945893","14":"-1.366212e-02","15":"0.1355523094","16":"-3.348127e-03","17":"","18":"#666666","19":"Rab30","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7936"},{"1":"ENSMUSG00000042797","2":"Aqp11","3":"aquaporin 11","4":"7","5":"97.72401","6":"97.73829","7":"-1","8":"-0.306511839","9":"-0.409151261","10":"-0.381387024","11":"3.281936","12":"2.841131","13":"2.940678","14":"-1.412833e-02","15":"0.0313901882","16":"1.866299e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"12707"},{"1":"ENSMUSG00000109256","2":"Gm44681","3":"predicted gene 44681","4":"7","5":"91.64771","6":"91.64855","7":"1","8":"-0.102770410","9":"-0.242078075","10":"-0.274792276","11":"3.282673","12":"3.048446","13":"3.072952","14":"-1.486512e-02","15":"-0.1759246375","16":"-1.304074e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46952"},{"1":"ENSMUSG00000108987","2":"Gm45056","3":"predicted gene 45056","4":"7","5":"82.11577","6":"82.12043","7":"-1","8":"-0.354297284","9":"-0.352677794","10":"-0.348292728","11":"3.282685","12":"3.069139","13":"2.889898","14":"-1.487752e-02","15":"-0.1966178277","16":"5.264604e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46692"},{"1":"ENSMUSG00000109327","2":"Gm44995","3":"predicted gene 44995","4":"7","5":"89.33837","6":"89.34629","7":"1","8":"-0.192624870","9":"-0.214341136","10":"-0.225810150","11":"3.282793","12":"2.921723","13":"2.950598","14":"-1.498521e-02","15":"-0.0492015803","16":"-8.053468e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47019"},{"1":"ENSMUSG00000030747","2":"Dgat2","3":"diacylglycerol O-acyltransferase 2","4":"7","5":"99.15366","6":"99.18272","7":"-1","8":"0.103403319","9":"0.166980701","10":"0.235512086","11":"3.283668","12":"3.032301","13":"3.013712","14":"-1.585976e-02","15":"-0.1597789309","16":"-7.116702e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"8001"},{"1":"ENSMUSG00000108476","2":"Gm44974","3":"predicted gene 44974","4":"7","5":"79.83059","6":"79.83187","7":"-1","8":"-0.105076142","9":"-0.107471693","10":"-0.038093528","11":"3.284741","12":"2.865497","13":"2.939322","14":"-1.693346e-02","15":"0.0070251380","16":"3.222262e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46213"},{"1":"ENSMUSG00000030629","2":"Zfand6","3":"zinc finger, AN1-type domain 6","4":"7","5":"84.61377","6":"84.68996","7":"-1","8":"-0.277564358","9":"-0.265625777","10":"-0.286114992","11":"3.285128","12":"2.952230","13":"2.996016","14":"-1.732017e-02","15":"-0.0797087807","16":"-5.347139e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7931"},{"1":"ENSMUSG00000030619","2":"Eed","3":"embryonic ectoderm development","4":"7","5":"89.95465","6":"89.98098","7":"-1","8":"0.843267603","9":"0.883609059","10":"0.898699477","11":"3.285379","12":"2.879296","13":"3.201045","14":"-1.757122e-02","15":"-0.0067745786","16":"-2.585005e-01","17":"Eed","18":"#CC3333","19":"Eed","20":"#CC3333","21":"Eed","22":"#CC3333","23":"TRUE","24":"Eed","_rn_":"7928"},{"1":"ENSMUSG00000108350","2":"Gm44950","3":"predicted gene 44950","4":"7","5":"79.92257","6":"79.92487","7":"-1","8":"-0.168730202","9":"-0.186528870","10":"-0.129102325","11":"3.285978","12":"2.885644","13":"3.055684","14":"-1.816976e-02","15":"-0.0131221949","16":"-1.131400e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46096"},{"1":"ENSMUSG00000039236","2":"Isg20","3":"interferon-stimulated protein","4":"7","5":"78.91342","6":"78.92040","7":"1","8":"-0.191000144","9":"-0.113836811","10":"-0.133945978","11":"3.286072","12":"2.856038","13":"2.969671","14":"-1.826360e-02","15":"0.0164839059","16":"-2.712671e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"11469"},{"1":"ENSMUSG00000109562","2":"Gm45178","3":"predicted gene 45178","4":"7","5":"90.99531","6":"90.99942","7":"1","8":"-0.421349178","9":"-0.434323829","10":"-0.407497110","11":"3.286993","12":"2.877305","13":"2.914254","14":"-1.918488e-02","15":"-0.0047830953","16":"2.829084e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47246"},{"1":"ENSMUSG00000030528","2":"Blm","3":"Bloom syndrome, RecQ like helicase","4":"7","5":"80.45473","6":"80.53512","7":"-1","8":"-0.310996059","9":"-0.303283591","10":"-0.293742020","11":"3.287264","12":"2.950173","13":"2.936340","14":"-1.945583e-02","15":"-0.0776516813","16":"6.204045e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7877"},{"1":"ENSMUSG00000063902","2":"Gm7964","3":"predicted gene 7964","4":"7","5":"83.75590","6":"83.75728","7":"1","8":"-0.303534404","9":"-0.366472614","10":"-0.377496510","11":"3.287500","12":"2.917653","13":"2.935356","14":"-1.969199e-02","15":"-0.0451313527","16":"7.188793e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"17811"},{"1":"ENSMUSG00000030607","2":"Acan","3":"aggrecan","4":"7","5":"79.05348","6":"79.11510","7":"1","8":"-0.374037456","9":"-0.379450887","10":"-0.393724678","11":"3.289267","12":"2.787714","13":"2.971995","14":"-2.145951e-02","15":"0.0848077660","16":"-2.945005e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7918"},{"1":"ENSMUSG00000109472","2":"Gm18943","3":"predicted gene, 18943","4":"7","5":"99.24760","6":"99.24882","7":"-1","8":"-0.001503626","9":"-0.064262722","10":"-0.078959190","11":"3.289369","12":"2.915923","13":"2.947774","14":"-2.156101e-02","15":"-0.0434012476","16":"-5.230010e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47161"},{"1":"ENSMUSG00000030546","2":"Plin1","3":"perilipin 1","4":"7","5":"79.72022","6":"79.73290","7":"-1","8":"0.176359739","9":"0.172200615","10":"0.170005558","11":"3.292088","12":"2.866301","13":"2.933097","14":"-2.428012e-02","15":"0.0062210575","16":"9.447777e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"7889"},{"1":"ENSMUSG00000108429","2":"Gm44603","3":"predicted gene 44603","4":"7","5":"95.47495","6":"95.47531","7":"1","8":"0.107561804","9":"0.270642117","10":"0.213371726","11":"3.292527","12":"2.864396","13":"2.857251","14":"-2.471932e-02","15":"0.0081259501","16":"8.529327e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46171"},{"1":"ENSMUSG00000109195","2":"Gm45038","3":"predicted gene 45038","4":"7","5":"93.52306","6":"93.52690","7":"1","8":"0.126172633","9":"0.320293942","10":"0.293083509","11":"3.293880","12":"2.874771","13":"2.929788","14":"-2.607217e-02","15":"-0.0022489214","16":"1.275658e-02","17":"","18":"#666666","19":"Gm45038","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46892"},{"1":"ENSMUSG00000035298","2":"Klhl35","3":"kelch-like 35","4":"7","5":"99.46600","6":"99.47402","7":"1","8":"0.420506997","9":"0.419201784","10":"0.484321087","11":"3.294519","12":"2.798645","13":"2.871173","14":"-2.671112e-02","15":"0.0738766166","16":"7.137193e-02","17":"Klhl35","18":"#CC3333","19":"Klhl35","20":"#CC3333","21":"Klhl35","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"9993"},{"1":"ENSMUSG00000108536","2":"Gm45059","3":"predicted gene 45059","4":"7","5":"78.77729","6":"78.77837","7":"-1","8":"-0.349510004","9":"-0.377265635","10":"-0.344644581","11":"3.294969","12":"2.890051","13":"2.967663","14":"-2.716058e-02","15":"-0.0175296223","16":"-2.511873e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46271"},{"1":"ENSMUSG00000090362","2":"Vmn2r79","3":"vomeronasal 2, receptor 79","4":"7","5":"86.99612","6":"87.04494","7":"1","8":"-0.030312054","9":"-0.046589500","10":"-0.082404861","11":"3.297141","12":"2.850619","13":"2.932771","14":"-2.933347e-02","15":"0.0219026623","16":"9.773870e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31061"},{"1":"ENSMUSG00000109429","2":"Gm45223","3":"predicted gene 45223","4":"7","5":"90.13249","6":"90.13763","7":"1","8":"-0.341382857","9":"-0.357788882","10":"-0.292070252","11":"3.298258","12":"2.850921","13":"2.967343","14":"-3.045030e-02","15":"0.0216004411","16":"-2.479821e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47119"},{"1":"ENSMUSG00000085792","2":"Gm15414","3":"predicted gene 15414","4":"7","5":"96.59665","6":"96.62907","7":"-1","8":"0.605496534","9":"0.609917913","10":"0.624006770","11":"3.299196","12":"2.864572","13":"3.028448","14":"-3.138823e-02","15":"0.0079491919","16":"-8.590354e-02","17":"Gm15414","18":"#CC3333","19":"Gm15414","20":"#CC3333","21":"Gm15414","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"27555"},{"1":"ENSMUSG00000039187","2":"Fanci","3":"Fanconi anemia, complementation group I","4":"7","5":"79.39193","6":"79.45026","7":"1","8":"-0.410306213","9":"-0.409009066","10":"-0.414060609","11":"3.299486","12":"2.854466","13":"2.952295","14":"-3.167763e-02","15":"0.0180552033","16":"-9.750442e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11440"},{"1":"ENSMUSG00000109460","2":"Gm45591","3":"predicted gene 45591","4":"7","5":"99.32120","6":"99.32246","7":"1","8":"0.022716234","9":"0.191680845","10":"0.147147811","11":"3.299848","12":"2.849083","13":"3.005053","14":"-3.204036e-02","15":"0.0234383987","16":"-6.250898e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47150"},{"1":"ENSMUSG00000109353","2":"Gm45183","3":"predicted gene 45183","4":"7","5":"91.30469","6":"91.30619","7":"1","8":"-0.245716284","9":"-0.307944324","10":"-0.310865342","11":"3.300856","12":"3.138517","13":"3.141224","14":"-3.304826e-02","15":"-0.2659950498","16":"-1.986796e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47044"},{"1":"ENSMUSG00000109498","2":"Gm45222","3":"predicted gene 45222","4":"7","5":"90.14595","6":"90.15025","7":"1","8":"-0.368817523","9":"-0.360405039","10":"-0.360740594","11":"3.301126","12":"2.828378","13":"2.986218","14":"-3.331763e-02","15":"0.0441436395","16":"-4.367355e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47184"},{"1":"ENSMUSG00000087088","2":"Gm16638","3":"predicted gene, 16638","4":"7","5":"83.59705","6":"83.63188","7":"-1","8":"-0.371596593","9":"-0.381663991","10":"-0.321787364","11":"3.301639","12":"2.705020","13":"3.181774","14":"-3.383096e-02","15":"0.1675011904","16":"-2.392291e-01","17":"","18":"#666666","19":"Gm16638","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"28594"},{"1":"ENSMUSG00000030610","2":"Det1","3":"de-etiolated homolog 1 (Arabidopsis)","4":"7","5":"78.82155","6":"78.84726","7":"-1","8":"-0.316795067","9":"-0.275109752","10":"-0.272550476","11":"3.303444","12":"2.915357","13":"2.948897","14":"-3.563601e-02","15":"-0.0428350098","16":"-6.352607e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7920"},{"1":"ENSMUSG00000074003","2":"Gucy2d","3":"guanylate cyclase 2d","4":"7","5":"98.44042","6":"98.47748","7":"1","8":"0.210787254","9":"0.185089067","10":"0.168561293","11":"3.304268","12":"2.863472","13":"2.926865","14":"-3.645962e-02","15":"0.0090499367","16":"1.567913e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"20665"},{"1":"ENSMUSG00000038570","2":"Saxo2","3":"stabilizer of axonemal microtubules 2","4":"7","5":"82.63296","6":"82.64853","7":"-1","8":"-0.071554272","9":"0.060708355","10":"0.066100343","11":"3.304388","12":"2.866442","13":"3.003823","14":"-3.657978e-02","15":"0.0060796522","16":"-6.127874e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11188"},{"1":"ENSMUSG00000065822","2":"Snord15a","3":"small nucleolar RNA, C/D box 15A","4":"7","5":"99.48279","6":"99.48293","7":"-1","8":"-0.218788121","9":"-0.166146702","10":"-0.204248962","11":"3.304711","12":"3.011920","13":"3.122168","14":"-3.690331e-02","15":"-0.1393978514","16":"-1.796239e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"18838"},{"1":"ENSMUSG00000108920","2":"Gm44678","3":"predicted gene 44678","4":"7","5":"91.34002","6":"91.34279","7":"1","8":"0.604196105","9":"0.506391985","10":"0.489791446","11":"3.306102","12":"3.085021","13":"3.272826","14":"-3.829424e-02","15":"-0.2124997611","16":"-3.302817e-01","17":"Gm44678","18":"#CC3333","19":"Gm44678","20":"#CC3333","21":"Gm44678","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"46627"},{"1":"ENSMUSG00000109245","2":"Gm44860","3":"predicted gene 44860","4":"7","5":"90.03057","6":"90.03292","7":"1","8":"-0.229556189","9":"-0.245022080","10":"-0.189460147","11":"3.306269","12":"2.868283","13":"2.780448","14":"-3.846127e-02","15":"0.0042386130","16":"1.620969e-01","17":"","18":"#666666","19":"","20":"#666666","21":"Gm44860","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"46941"},{"1":"ENSMUSG00000091962","2":"Vmn2r78","3":"vomeronasal 2, receptor 78","4":"7","5":"86.91509","6":"86.95531","7":"1","8":"-0.321686353","9":"-0.379705937","10":"-0.362216473","11":"3.306328","12":"2.883709","13":"2.993807","14":"-3.851988e-02","15":"-0.0111871303","16":"-5.126252e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31769"},{"1":"ENSMUSG00000108991","2":"Gm45807","3":"predicted gene 45807","4":"7","5":"82.69540","6":"82.69827","7":"1","8":"-0.109349406","9":"-0.241451331","10":"-0.249195605","11":"3.306968","12":"2.924870","13":"2.982401","14":"-3.915991e-02","15":"-0.0523484891","16":"-3.985666e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46696"},{"1":"ENSMUSG00000108953","2":"Gm45129","3":"predicted gene 45129","4":"7","5":"91.90806","6":"91.91029","7":"1","8":"-0.414651975","9":"-0.424829116","10":"-0.372630844","11":"3.308428","12":"2.961915","13":"2.959454","14":"-4.061984e-02","15":"-0.0893937179","16":"-1.690911e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46658"},{"1":"ENSMUSG00000109519","2":"Gm18806","3":"predicted gene, 18806","4":"7","5":"81.74235","6":"81.74432","7":"1","8":"-0.406248557","9":"-0.397775258","10":"-0.400338162","11":"3.308750","12":"2.877473","13":"2.941883","14":"-4.094166e-02","15":"-0.0049511518","16":"6.613160e-04","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47205"},{"1":"ENSMUSG00000030543","2":"Mesp2","3":"mesoderm posterior 2","4":"7","5":"79.81073","6":"79.81344","7":"1","8":"-0.343425188","9":"-0.343601998","10":"-0.347072291","11":"3.309243","12":"2.743844","13":"2.999711","14":"-4.143488e-02","15":"0.1286775982","16":"-5.716644e-02","17":"","18":"#666666","19":"Mesp2","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"7886"},{"1":"ENSMUSG00000109216","2":"Gm44907","3":"predicted gene 44907","4":"7","5":"90.45725","6":"90.45888","7":"1","8":"-0.329239859","9":"-0.314664695","10":"-0.280186695","11":"3.309319","12":"2.863221","13":"3.245421","14":"-4.151087e-02","15":"0.0093006802","16":"-3.028769e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46912"},{"1":"ENSMUSG00000049580","2":"Tsku","3":"tsukushi, small leucine rich proteoglycan","4":"7","5":"98.35067","6":"98.36133","7":"-1","8":"-0.440302112","9":"-0.418878962","10":"-0.426775397","11":"3.310220","12":"2.816934","13":"2.966323","14":"-4.241167e-02","15":"0.0555881174","16":"-2.377830e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"14563"},{"1":"ENSMUSG00000109149","2":"Gm19656","3":"predicted gene, 19656","4":"7","5":"98.72006","6":"98.72876","7":"-1","8":"-0.378698670","9":"-0.357442768","10":"-0.334352682","11":"3.310721","12":"3.060336","13":"2.980979","14":"-4.291323e-02","15":"-0.1878139000","16":"-3.843433e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46847"},{"1":"ENSMUSG00000030605","2":"Mfge8","3":"milk fat globule-EGF factor 8 protein","4":"7","5":"79.13377","6":"79.14906","7":"-1","8":"0.054224727","9":"-0.009103003","10":"0.004066888","11":"3.311468","12":"3.164361","13":"3.102645","14":"-4.365999e-02","15":"-0.2918388387","16":"-1.601002e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7916"},{"1":"ENSMUSG00000070460","2":"Olfr291","3":"olfactory receptor 291","4":"7","5":"84.85355","6":"84.85961","7":"1","8":"-0.260550286","9":"-0.222464807","10":"-0.231825174","11":"3.312860","12":"2.900231","13":"2.909317","14":"-4.505246e-02","15":"-0.0277091324","16":"3.322711e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"19783"},{"1":"ENSMUSG00000089501","2":"Gm23479","3":"predicted gene, 23479","4":"7","5":"98.93589","6":"98.93601","7":"1","8":"-0.184006692","9":"-0.152449990","10":"-0.185445824","11":"3.313608","12":"2.971360","13":"2.999384","14":"-4.579973e-02","15":"-0.0988382069","16":"-5.683980e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"30469"},{"1":"ENSMUSG00000109492","2":"Gm33882","3":"predicted gene, 33882","4":"7","5":"99.09023","6":"99.09088","7":"1","8":"0.171105053","9":"-0.022766370","10":"-0.022337731","11":"3.313851","12":"2.861967","13":"2.965301","14":"-4.604340e-02","15":"0.0105545188","16":"-2.275690e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47178"},{"1":"ENSMUSG00000030560","2":"Ctsc","3":"cathepsin C","4":"7","5":"88.27809","6":"88.31089","7":"1","8":"0.343818318","9":"0.270743926","10":"0.262738230","11":"3.313995","12":"3.217893","13":"3.050568","14":"-4.618670e-02","15":"-0.3453709584","16":"-1.080239e-01","17":"Ctsc","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7898"},{"1":"ENSMUSG00000039176","2":"Polg","3":"polymerase (DNA directed), gamma","4":"7","5":"79.44623","6":"79.46636","7":"-1","8":"0.441768943","9":"0.593174548","10":"0.592312746","11":"3.315052","12":"2.949568","13":"3.176676","14":"-4.724363e-02","15":"-0.0770465531","16":"-2.341311e-01","17":"Polg","18":"#CC3333","19":"Polg","20":"#CC3333","21":"Polg","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"11435"},{"1":"ENSMUSG00000039361","2":"Picalm","3":"phosphatidylinositol binding clathrin assembly protein","4":"7","5":"90.13021","6":"90.21346","7":"1","8":"-0.361948506","9":"-0.369425735","10":"-0.334833308","11":"3.315457","12":"2.620942","13":"2.990498","14":"-4.764922e-02","15":"0.2515799666","16":"-4.795368e-02","17":"","18":"#666666","19":"Picalm","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11512"},{"1":"ENSMUSG00000085443","2":"Gm15416","3":"predicted gene 15416","4":"7","5":"96.81243","6":"96.81749","7":"-1","8":"-0.193850121","9":"-0.207240791","10":"-0.232168336","11":"3.316249","12":"3.022890","13":"3.011554","14":"-4.844104e-02","15":"-0.1503687633","16":"-6.900902e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"27267"},{"1":"ENSMUSG00000039133","2":"9330171B17Rik","3":"RIKEN cDNA 9330171B17 gene","4":"7","5":"79.71028","6":"79.71236","7":"1","8":"0.326591636","9":"0.273861603","10":"0.351066605","11":"3.318315","12":"2.894966","13":"2.922625","14":"-5.050753e-02","15":"-0.0224445388","16":"1.991983e-02","17":"9330171B17Rik","18":"#CC3333","19":"","20":"#666666","21":"9330171B17Rik","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"11416"},{"1":"ENSMUSG00000085751","2":"Gm15412","3":"predicted gene 15412","4":"7","5":"96.33948","6":"96.34196","7":"-1","8":"0.110495168","9":"-0.096584235","10":"-0.098871574","11":"3.319611","12":"2.871900","13":"2.952702","14":"-5.180354e-02","15":"0.0006220596","16":"-1.015747e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"27522"},{"1":"ENSMUSG00000030630","2":"Fah","3":"fumarylacetoacetate hydrolase","4":"7","5":"84.58516","6":"84.60672","7":"-1","8":"0.256626683","9":"0.101274940","10":"0.131595612","11":"3.321129","12":"2.901175","13":"2.949281","14":"-5.332131e-02","15":"-0.0286532140","16":"-6.736192e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7932"},{"1":"ENSMUSG00000098066","2":"Gm26944","3":"predicted gene, 26944","4":"7","5":"92.63974","6":"92.64744","7":"1","8":"-0.344197963","9":"-0.363941827","10":"-0.370114671","11":"3.323412","12":"2.935898","13":"2.927776","14":"-5.560411e-02","15":"-0.0633763827","16":"1.476842e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"36505"},{"1":"ENSMUSG00000083820","2":"Ndufs6b","3":"NADH:ubiquinone oxidoreductase subunit S6B","4":"7","5":"81.16140","6":"81.16174","7":"-1","8":"0.303608760","9":"0.092751911","10":"0.092014571","11":"3.323958","12":"2.783683","13":"2.933296","14":"-5.615047e-02","15":"0.0888389185","16":"9.248861e-03","17":"Ndufs6b","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"25932"},{"1":"ENSMUSG00000109401","2":"Cpeb1os1","3":"cytoplasmic polyadenylation element binding protein 1, opposite strand 1","4":"7","5":"81.45589","6":"81.45801","7":"1","8":"-0.225376641","9":"-0.212370012","10":"-0.204280193","11":"3.327144","12":"2.885696","13":"2.886557","14":"-5.933651e-02","15":"-0.0131747552","16":"5.598778e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47091"},{"1":"ENSMUSG00000004508","2":"Gab2","3":"growth factor receptor bound protein 2-associated protein 2","4":"7","5":"97.08159","6":"97.30895","7":"1","8":"-0.346810521","9":"-0.387638796","10":"-0.363290754","11":"3.328057","12":"2.873066","13":"3.096768","14":"-6.024903e-02","15":"-0.0005445532","16":"-1.542231e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"715"},{"1":"ENSMUSG00000097441","2":"Gm26633","3":"predicted gene, 26633","4":"7","5":"79.01805","6":"79.05332","7":"-1","8":"-0.300669834","9":"-0.328117582","10":"-0.332731892","11":"3.329229","12":"3.223271","13":"3.021022","14":"-6.142139e-02","15":"-0.3507494554","16":"-7.847759e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"35954"},{"1":"ENSMUSG00000109196","2":"Gm44715","3":"predicted gene 44715","4":"7","5":"93.16883","6":"93.17267","7":"1","8":"0.026194370","9":"0.177927343","10":"0.164622567","11":"3.329357","12":"2.775575","13":"2.819159","14":"-6.154881e-02","15":"0.0969467016","16":"1.233857e-01","17":"","18":"#666666","19":"","20":"#666666","21":"Gm44715","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46893"},{"1":"ENSMUSG00000108321","2":"Gm44649","3":"predicted gene 44649","4":"7","5":"80.75182","6":"80.75861","7":"1","8":"-0.300152275","9":"-0.229351410","10":"-0.211156847","11":"3.329429","12":"2.984034","13":"2.962800","14":"-6.162108e-02","15":"-0.1115118729","16":"-2.025542e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46067"},{"1":"ENSMUSG00000025105","2":"Bnc1","3":"basonuclin 1","4":"7","5":"81.96667","6":"81.99229","7":"-1","8":"0.216433144","9":"0.123162573","10":"0.168864253","11":"3.330166","12":"2.895498","13":"2.918221","14":"-6.235800e-02","15":"-0.0229766274","16":"2.432347e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"4867"},{"1":"ENSMUSG00000109574","2":"B230206I08Rik","3":"RIKEN cDNA B230206I08 gene","4":"7","5":"92.40818","6":"92.41137","7":"1","8":"-0.385797585","9":"-0.393677326","10":"-0.340904905","11":"3.331759","12":"2.840301","13":"3.084233","14":"-6.395104e-02","15":"0.0322208529","16":"-1.416881e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47258"},{"1":"ENSMUSG00000108655","2":"Gm44949","3":"predicted gene 44949","4":"7","5":"80.08898","6":"80.09043","7":"-1","8":"-0.313990566","9":"-0.299964857","10":"-0.293647117","11":"3.332627","12":"2.873237","13":"3.121538","14":"-6.481953e-02","15":"-0.0007156700","16":"-1.789932e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46382"},{"1":"ENSMUSG00000053049","2":"Gm15413","3":"predicted gene 15413","4":"7","5":"96.79143","6":"96.80155","7":"-1","8":"-0.272271137","9":"-0.391047645","10":"-0.366444050","11":"3.334314","12":"2.884437","13":"2.951521","14":"-6.650607e-02","15":"-0.0119154115","16":"-8.977009e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"15419"},{"1":"ENSMUSG00000109031","2":"Gm45201","3":"predicted gene 45201","4":"7","5":"92.43117","6":"92.43385","7":"1","8":"-0.408623984","9":"-0.417559599","10":"-0.363581319","11":"3.336618","12":"2.823931","13":"2.996473","14":"-6.881012e-02","15":"0.0485911172","16":"-5.392842e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46734"},{"1":"ENSMUSG00000109090","2":"Gm45088","3":"predicted gene 45088","4":"7","5":"81.91894","6":"81.92035","7":"-1","8":"-0.376497148","9":"-0.399105548","10":"-0.337900932","11":"3.337835","12":"3.089085","13":"2.965411","14":"-7.002716e-02","15":"-0.2165634838","16":"-2.286614e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46791"},{"1":"ENSMUSG00000094950","2":"Vmn2r66","3":"vomeronasal 2, receptor 66","4":"7","5":"84.98537","6":"85.01227","7":"-1","8":"-0.230877731","9":"-0.172221213","10":"-0.211276136","11":"3.339229","12":"2.866948","13":"2.957971","14":"-7.142148e-02","15":"0.0055741133","16":"-1.542642e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"33893"},{"1":"ENSMUSG00000109311","2":"AI314278","3":"expressed sequence AI314278","4":"7","5":"89.42631","6":"89.44193","7":"1","8":"0.066197705","9":"-0.117268554","10":"-0.126482195","11":"3.339538","12":"2.871896","13":"2.959473","14":"-7.173046e-02","15":"0.0006253054","16":"-1.692883e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47003"},{"1":"ENSMUSG00000086993","2":"Rsf1os2","3":"remodeling and spacing factor 1, opposite strand 2","4":"7","5":"97.65001","6":"97.69683","7":"-1","8":"-0.246485567","9":"-0.242909216","10":"-0.260672974","11":"3.340140","12":"2.890115","13":"2.962195","14":"-7.233232e-02","15":"-0.0175935013","16":"-1.965008e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"28513"},{"1":"ENSMUSG00000047084","2":"Ngrn","3":"neugrin, neurite outgrowth associated","4":"7","5":"80.26122","6":"80.26537","7":"1","8":"-0.026366515","9":"-0.106331911","10":"-0.094000735","11":"3.340486","12":"2.954113","13":"3.240815","14":"-7.267791e-02","15":"-0.0815914068","16":"-2.982701e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"13888"},{"1":"ENSMUSG00000085080","2":"2310044K18Rik","3":"RIKEN cDNA 2310044K18 gene","4":"7","5":"83.65958","6":"83.66096","7":"1","8":"-0.389613187","9":"-0.384857092","10":"-0.367207532","11":"3.340822","12":"2.960952","13":"2.926944","14":"-7.301365e-02","15":"-0.0884301802","16":"1.560018e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"26976"},{"1":"ENSMUSG00000097015","2":"Gm26705","3":"predicted gene, 26705","4":"7","5":"99.26048","6":"99.26722","7":"-1","8":"0.335136797","9":"0.167233835","10":"0.156747839","11":"3.342037","12":"3.008616","13":"3.143275","14":"-7.422922e-02","15":"-0.1360943775","16":"-2.007302e-01","17":"Gm26705","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"35574"},{"1":"ENSMUSG00000109242","2":"A930002H02Rik","3":"RIKEN cDNA A930002H02 gene","4":"7","5":"90.87160","6":"90.87278","7":"1","8":"0.581503072","9":"0.359332422","10":"0.410995400","11":"3.342692","12":"2.876672","13":"3.089192","14":"-7.488421e-02","15":"-0.0041503456","16":"-1.466479e-01","17":"A930002H02Rik","18":"#CC3333","19":"A930002H02Rik","20":"#CC3333","21":"A930002H02Rik","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46938"},{"1":"ENSMUSG00000030615","2":"Tmem126a","3":"transmembrane protein 126A","4":"7","5":"90.45070","6":"90.45723","7":"-1","8":"-0.405826269","9":"-0.404406945","10":"-0.394935637","11":"3.343384","12":"2.890027","13":"3.004590","14":"-7.557608e-02","15":"-0.0175048943","16":"-6.204565e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7925"},{"1":"ENSMUSG00000109008","2":"Gm44899","3":"predicted gene 44899","4":"7","5":"80.98085","6":"80.99341","7":"-1","8":"-0.377333317","9":"-0.337397262","10":"-0.357252417","11":"3.343883","12":"3.175310","13":"3.175166","14":"-7.607480e-02","15":"-0.3027886952","16":"-2.326211e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46711"},{"1":"ENSMUSG00000030536","2":"Iqgap1","3":"IQ motif containing GTPase activating protein 1","4":"7","5":"80.71158","6":"80.82597","7":"-1","8":"-0.362549846","9":"-0.364165294","10":"-0.332289068","11":"3.345311","12":"3.039920","13":"2.986554","14":"-7.750354e-02","15":"-0.1673984915","16":"-4.400953e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7882"},{"1":"ENSMUSG00000108436","2":"Gm44851","3":"predicted gene 44851","4":"7","5":"80.40427","6":"80.40658","7":"1","8":"-0.364563492","9":"-0.323487418","10":"-0.314840674","11":"3.348356","12":"3.028968","13":"3.095236","14":"-8.054797e-02","15":"-0.1564458875","16":"-1.526915e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46178"},{"1":"ENSMUSG00000108857","2":"Gm44578","3":"predicted gene 44578","4":"7","5":"85.50130","6":"85.50513","7":"1","8":"0.112142398","9":"0.266321672","10":"0.254188262","11":"3.349658","12":"2.901167","13":"3.038456","14":"-8.185037e-02","15":"-0.0286456990","16":"-9.591131e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46567"},{"1":"ENSMUSG00000108788","2":"4921513I08Rik","3":"RIKEN cDNA 4921513I08 gene","4":"7","5":"78.25430","6":"78.25542","7":"1","8":"-0.252074044","9":"-0.327611916","10":"-0.344806660","11":"3.350100","12":"2.893353","13":"2.940696","14":"-8.229192e-02","15":"-0.0208309659","16":"1.848954e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46501"},{"1":"ENSMUSG00000108599","2":"Gm10616","3":"predicted gene 10616","4":"7","5":"79.46641","6":"79.46854","7":"1","8":"0.016210969","9":"0.056636067","10":"0.123554223","11":"3.350236","12":"2.880162","13":"3.079308","14":"-8.242785e-02","15":"-0.0076408290","16":"-1.367638e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46330"},{"1":"ENSMUSG00000108831","2":"Gm7957","3":"predicted gene 7957","4":"7","5":"83.41198","6":"83.41458","7":"1","8":"-0.068060674","9":"0.026837157","10":"0.021114558","11":"3.350502","12":"2.862738","13":"2.934696","14":"-8.269359e-02","15":"0.0097833262","16":"7.848010e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46541"},{"1":"ENSMUSG00000052396","2":"Mogat2","3":"monoacylglycerol O-acyltransferase 2","4":"7","5":"99.21908","6":"99.23862","7":"-1","8":"-0.023858004","9":"-0.063825982","10":"-0.053167772","11":"3.350676","12":"2.861376","13":"2.948305","14":"-8.286845e-02","15":"0.0111455871","16":"-5.760297e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"15280"},{"1":"ENSMUSG00000086690","2":"4933406J10Rik","3":"RIKEN cDNA 4933406J10 gene","4":"7","5":"82.74337","6":"82.75564","7":"-1","8":"-0.238911298","9":"-0.317150448","10":"-0.312707918","11":"3.351455","12":"2.862688","13":"2.942594","14":"-8.364735e-02","15":"0.0098339009","16":"-4.988944e-05","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"28265"},{"1":"ENSMUSG00000074024","2":"4632427E13Rik","3":"RIKEN cDNA 4632427E13 gene","4":"7","5":"92.73417","6":"92.74147","7":"-1","8":"-0.180910307","9":"-0.169873039","10":"-0.102192446","11":"3.352208","12":"2.846953","13":"3.024501","14":"-8.440013e-02","15":"0.0255690263","16":"-8.195609e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"20669"},{"1":"ENSMUSG00000109468","2":"Gm45185","3":"predicted gene 45185","4":"7","5":"99.14159","6":"99.14200","7":"1","8":"-0.158724566","9":"-0.308054783","10":"-0.304580974","11":"3.353667","12":"2.912772","13":"2.958190","14":"-8.585874e-02","15":"-0.0402504755","16":"-1.564561e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47158"},{"1":"ENSMUSG00000108825","2":"Gm45838","3":"predicted gene 45838","4":"7","5":"83.63337","6":"83.63582","7":"1","8":"-0.395246451","9":"-0.423189573","10":"-0.361667844","11":"3.353721","12":"2.630357","13":"3.221676","14":"-8.591343e-02","15":"0.2421644834","16":"-2.791310e-01","17":"","18":"#666666","19":"Gm45838","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46535"},{"1":"ENSMUSG00000035582","2":"Gdpd4","3":"glycerophosphodiester phosphodiesterase domain containing 4","4":"7","5":"97.91996","6":"98.04966","7":"1","8":"-0.094119498","9":"0.010285565","10":"0.014023464","11":"3.354674","12":"2.859010","13":"2.955572","14":"-8.686606e-02","15":"0.0135117655","16":"-1.302761e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"10086"},{"1":"ENSMUSG00000109427","2":"Gm45845","3":"predicted gene 45845","4":"7","5":"81.78974","6":"81.79032","7":"-1","8":"-0.164577252","9":"-0.197214985","10":"-0.168332441","11":"3.355839","12":"2.877661","13":"2.922858","14":"-8.803059e-02","15":"-0.0051389203","16":"1.968668e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47117"},{"1":"ENSMUSG00000039099","2":"Wdr93","3":"WD repeat domain 93","4":"7","5":"79.74316","6":"79.78595","7":"1","8":"0.305803011","9":"0.050527674","10":"0.038256690","11":"3.356536","12":"2.969541","13":"3.024038","14":"-8.872851e-02","15":"-0.0970197404","16":"-8.149395e-02","17":"Wdr93","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"11400"},{"1":"ENSMUSG00000108521","2":"Gm44639","3":"predicted gene 44639","4":"7","5":"79.27777","6":"79.28078","7":"1","8":"-0.008116648","9":"0.136164296","10":"0.185487747","11":"3.358346","12":"2.788899","13":"3.073562","14":"-9.053795e-02","15":"0.0836224032","16":"-1.310176e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46256"},{"1":"ENSMUSG00000030559","2":"Rab38","3":"RAB38, member RAS oncogene family","4":"7","5":"88.43027","6":"88.49157","7":"1","8":"-0.293386700","9":"-0.310403688","10":"-0.253710723","11":"3.358609","12":"2.809616","13":"3.363311","14":"-9.080098e-02","15":"0.0629059481","16":"-4.207667e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"7897"},{"1":"ENSMUSG00000108494","2":"Gm45203","3":"predicted gene 45203","4":"7","5":"80.11080","6":"80.11263","7":"-1","8":"-0.370196950","9":"-0.369343199","10":"-0.363923129","11":"3.365048","12":"2.802985","13":"3.108754","14":"-9.723983e-02","15":"0.0695371048","16":"-1.662094e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46231"},{"1":"ENSMUSG00000025584","2":"Pde8a","3":"phosphodiesterase 8A","4":"7","5":"81.21360","6":"81.33453","7":"1","8":"0.075958124","9":"-0.028856433","10":"0.032179545","11":"3.365823","12":"2.853469","13":"3.020855","14":"-9.801487e-02","15":"0.0190524987","16":"-7.831051e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"5125"},{"1":"ENSMUSG00000039405","2":"Prss23","3":"protease, serine 23","4":"7","5":"89.50778","6":"89.52719","7":"-1","8":"-0.365913837","9":"-0.385877671","10":"-0.327327492","11":"3.368756","12":"2.859351","13":"3.012408","14":"-1.009484e-01","15":"0.0131706576","16":"-6.986317e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11525"},{"1":"ENSMUSG00000108318","2":"Gm45170","3":"predicted gene 45170","4":"7","5":"79.60201","6":"79.60368","7":"-1","8":"-0.276636768","9":"-0.304856673","10":"-0.238950159","11":"3.368786","12":"2.840764","13":"3.016602","14":"-1.009780e-01","15":"0.0317572938","16":"-7.405748e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46064"},{"1":"ENSMUSG00000108390","2":"Gm39038","3":"predicted gene, 39038","4":"7","5":"78.56064","6":"78.57523","7":"-1","8":"0.036580527","9":"-0.143368399","10":"-0.176266048","11":"3.369184","12":"2.910918","13":"3.115783","14":"-1.013757e-01","15":"-0.0383959992","16":"-1.732381e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46134"},{"1":"ENSMUSG00000103887","2":"Gm37008","3":"predicted gene, 37008","4":"7","5":"92.63721","6":"92.63753","7":"1","8":"0.265102297","9":"0.350165395","10":"0.354634954","11":"3.369269","12":"2.898585","13":"2.960694","14":"-1.014611e-01","15":"-0.0260638016","16":"-1.814935e-02","17":"","18":"#666666","19":"Gm37008","20":"#CC3333","21":"Gm37008","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"41908"},{"1":"ENSMUSG00000074004","2":"B3gnt6","3":"UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 6 (core 3 synthase)","4":"7","5":"98.19242","6":"98.19948","7":"-1","8":"-0.233724462","9":"-0.255969030","10":"-0.188336244","11":"3.369679","12":"2.895918","13":"3.101594","14":"-1.018708e-01","15":"-0.0233959155","16":"-1.590498e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"20666"},{"1":"ENSMUSG00000030562","2":"Nox4","3":"NADPH oxidase 4","4":"7","5":"87.24610","6":"87.39871","7":"1","8":"-0.179942520","9":"-0.181308791","10":"-0.205773756","11":"3.370131","12":"2.873103","13":"3.045848","14":"-1.023226e-01","15":"-0.0005811809","16":"-1.033039e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"7899"},{"1":"ENSMUSG00000085560","2":"C230038L03Rik","3":"RIKEN cDNA C230038L03 gene","4":"7","5":"96.90683","6":"96.95189","7":"-1","8":"-0.325040727","9":"-0.339586474","10":"-0.281086570","11":"3.372309","12":"2.827248","13":"2.955972","14":"-1.045012e-01","15":"0.0452739954","16":"-1.342719e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"27360"},{"1":"ENSMUSG00000108845","2":"Gm45090","3":"predicted gene 45090","4":"7","5":"82.15588","6":"82.15702","7":"-1","8":"-0.239121064","9":"-0.270959068","10":"-0.277796975","11":"3.372370","12":"2.893527","13":"2.923348","14":"-1.045624e-01","15":"-0.0210051450","16":"1.919604e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46555"},{"1":"ENSMUSG00000097016","2":"Gm26708","3":"predicted gene, 26708","4":"7","5":"84.15214","6":"84.19512","7":"1","8":"-0.344107196","9":"-0.353314239","10":"-0.297445563","11":"3.373650","12":"2.866467","13":"2.993736","14":"-1.058421e-01","15":"0.0060548060","16":"-5.119129e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"35575"},{"1":"ENSMUSG00000103831","2":"Gm37608","3":"predicted gene, 37608","4":"7","5":"79.51688","6":"79.51876","7":"1","8":"-0.326350905","9":"-0.322756475","10":"-0.283517642","11":"3.373710","12":"2.815144","13":"2.961195","14":"-1.059018e-01","15":"0.0573775594","16":"-1.865056e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"41860"},{"1":"ENSMUSG00000108635","2":"Gm44633","3":"predicted gene 44633","4":"7","5":"96.81387","6":"96.81831","7":"-1","8":"0.063142115","9":"-0.016354016","10":"-0.048331670","11":"3.377162","12":"3.055318","13":"3.137421","14":"-1.093539e-01","15":"-0.1827967787","16":"-1.948768e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46364"},{"1":"ENSMUSG00000083974","2":"Gm16053","3":"predicted gene 16053","4":"7","5":"97.40614","6":"97.40633","7":"1","8":"-0.086768162","9":"-0.067063143","10":"-0.083524987","11":"3.377711","12":"2.939532","13":"2.977547","14":"-1.099030e-01","15":"-0.0670106448","16":"-3.500287e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"26074"},{"1":"ENSMUSG00000098805","2":"Gm27470","3":"predicted gene, 27470","4":"7","5":"91.12300","6":"91.12321","7":"-1","8":"0.135244513","9":"-0.034252378","10":"0.031172211","11":"3.378833","12":"3.137575","13":"3.202730","14":"-1.110253e-01","15":"-0.2650528814","16":"-2.601851e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"37183"},{"1":"ENSMUSG00000070469","2":"Adamtsl3","3":"ADAMTS-like 3","4":"7","5":"82.33569","6":"82.61445","7":"1","8":"-0.107485342","9":"-0.248098601","10":"-0.249440222","11":"3.379656","12":"2.969379","13":"2.947468","14":"-1.118481e-01","15":"-0.0968569240","16":"-4.923924e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"19788"},{"1":"ENSMUSG00000030641","2":"Ddias","3":"DNA damage-induced apoptosis suppressor","4":"7","5":"92.85752","6":"92.87425","7":"-1","8":"0.393297300","9":"0.504942860","10":"0.498753806","11":"3.385776","12":"2.882763","13":"3.030851","14":"-1.179681e-01","15":"-0.0102414808","16":"-8.830602e-02","17":"Ddias","18":"#CC3333","19":"Ddias","20":"#CC3333","21":"Ddias","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"7935"},{"1":"ENSMUSG00000109044","2":"Gm44680","3":"predicted gene 44680","4":"7","5":"91.63524","6":"91.63815","7":"1","8":"-0.433327681","9":"-0.463407276","10":"-0.426392071","11":"3.385946","12":"3.138881","13":"3.041154","14":"-1.181381e-01","15":"-0.2663590541","16":"-9.860999e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46747"},{"1":"ENSMUSG00000084636","2":"Gm23884","3":"predicted gene, 23884","4":"7","5":"95.17186","6":"95.17216","7":"1","8":"0.189882947","9":"0.399401018","10":"0.376642137","11":"3.387922","12":"2.867350","13":"2.995165","14":"-1.201137e-01","15":"0.0051716071","16":"-5.262071e-02","17":"","18":"#666666","19":"Gm23884","20":"#CC3333","21":"Gm23884","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"26630"},{"1":"ENSMUSG00000099001","2":"Gm27660","3":"predicted gene, 27660","4":"7","5":"91.12635","6":"91.12655","7":"-1","8":"-0.440537297","9":"-0.446248772","10":"-0.406615550","11":"3.388836","12":"2.716005","13":"3.110157","14":"-1.210281e-01","15":"0.1565171413","16":"-1.676128e-01","17":"","18":"#666666","19":"Gm27660","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"37361"},{"1":"ENSMUSG00000055446","2":"A630091E08Rik","3":"RIKEN cDNA A630091E08 gene","4":"7","5":"98.54332","6":"98.56299","7":"-1","8":"0.287318825","9":"0.166838810","10":"0.214111009","11":"3.389141","12":"2.991003","13":"2.926734","14":"-1.213334e-01","15":"-0.1184815987","16":"1.581039e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"15944"},{"1":"ENSMUSG00000062444","2":"Ap3b2","3":"adaptor-related protein complex 3, beta 2 subunit","4":"7","5":"81.46040","6":"81.49393","7":"-1","8":"-0.210087583","9":"-0.139551957","10":"-0.135631364","11":"3.389276","12":"2.730511","13":"2.989818","14":"-1.214684e-01","15":"0.1420108779","16":"-4.727324e-02","17":"","18":"#666666","19":"Ap3b2","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"17474"},{"1":"ENSMUSG00000062797","2":"Hikeshi","3":"heat shock protein nuclear import factor","4":"7","5":"89.91753","6":"89.94120","7":"-1","8":"-0.090389446","9":"0.057258046","10":"0.039595542","11":"3.389895","12":"2.705496","13":"2.971038","14":"-1.220868e-01","15":"0.1670257186","16":"-2.849364e-02","17":"","18":"#666666","19":"Hikeshi","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"17560"},{"1":"ENSMUSG00000044952","2":"Kctd21","3":"potassium channel tetramerisation domain containing 21","4":"7","5":"97.33233","6":"97.35021","7":"1","8":"-0.120031038","9":"-0.201738082","10":"-0.154860288","11":"3.391335","12":"2.986075","13":"3.073802","14":"-1.235271e-01","15":"-0.1135529710","16":"-1.312579e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"13289"},{"1":"ENSMUSG00000051515","2":"Fam181b","3":"family with sequence similarity 181, member B","4":"7","5":"93.07986","6":"93.08187","7":"1","8":"-0.160549731","9":"-0.300761165","10":"-0.308700781","11":"3.392737","12":"2.991246","13":"2.966768","14":"-1.249286e-01","15":"-0.1187246274","16":"-2.422346e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"15090"},{"1":"ENSMUSG00000109320","2":"Gm44679","3":"predicted gene 44679","4":"7","5":"91.51911","6":"91.52202","7":"1","8":"-0.279491032","9":"-0.358447294","10":"-0.376962914","11":"3.397266","12":"3.132898","13":"3.104762","14":"-1.294582e-01","15":"-0.2603759375","16":"-1.622170e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47012"},{"1":"ENSMUSG00000030541","2":"Idh2","3":"isocitrate dehydrogenase 2 (NADP+), mitochondrial","4":"7","5":"80.09485","6":"80.11539","7":"-1","8":"-0.091280903","9":"-0.169949622","10":"-0.180547498","11":"3.398256","12":"3.116987","13":"3.086868","14":"-1.304480e-01","15":"-0.2444658107","16":"-1.443233e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7885"},{"1":"ENSMUSG00000109091","2":"Gm44827","3":"predicted gene 44827","4":"7","5":"84.22683","6":"84.23292","7":"-1","8":"-0.327808079","9":"-0.295048216","10":"-0.306408286","11":"3.399668","12":"2.861951","13":"2.862938","14":"-1.318603e-01","15":"0.0105709107","16":"7.960647e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46792"},{"1":"ENSMUSG00000030614","2":"Tmem126b","3":"transmembrane protein 126B","4":"7","5":"90.46744","6":"90.47600","7":"-1","8":"-0.266037578","9":"-0.272686491","10":"-0.201241609","11":"3.400674","12":"2.947552","13":"3.054596","14":"-1.328658e-01","15":"-0.0750301477","16":"-1.120520e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7924"},{"1":"ENSMUSG00000086433","2":"Rsf1os1","3":"remodeling and spacing factor 1, opposite strand 1","4":"7","5":"97.59835","6":"97.64511","7":"-1","8":"0.120116339","9":"-0.052303899","10":"-0.050520196","11":"3.401454","12":"2.932580","13":"3.109406","14":"-1.336463e-01","15":"-0.0600580534","16":"-1.668619e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"28067"},{"1":"ENSMUSG00000030539","2":"Sema4b","3":"sema domain, immunoglobulin domain (Ig), transmembrane domain (TM) and short cytoplasmic domain, (semaphorin) 4B","4":"7","5":"80.18684","6":"80.22653","7":"1","8":"0.191787802","9":"0.060826886","10":"0.129849126","11":"3.403422","12":"2.893426","13":"3.202570","14":"-1.356139e-01","15":"-0.0209044495","16":"-2.600254e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7884"},{"1":"ENSMUSG00000030636","2":"1700010L04Rik","3":"RIKEN cDNA 1700010L04 gene","4":"7","5":"82.85160","6":"82.85985","7":"1","8":"-0.290928020","9":"-0.249555482","10":"-0.190645358","11":"3.403441","12":"3.075170","13":"2.974392","14":"-1.356331e-01","15":"-0.2026482581","16":"-3.184795e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"7933"},{"1":"ENSMUSG00000050973","2":"Gdpgp1","3":"GDP-D-glucose phosphorylase 1","4":"7","5":"80.23287","6":"80.24206","7":"1","8":"0.275467527","9":"0.079183823","10":"0.036898393","11":"3.406064","12":"2.915466","13":"2.907757","14":"-1.382559e-01","15":"-0.0429445809","16":"3.478777e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"14934"},{"1":"ENSMUSG00000061877","2":"BC048679","3":"cDNA sequence BC048679","4":"7","5":"81.49428","6":"81.49828","7":"-1","8":"0.373707860","9":"0.560610716","10":"0.536612314","11":"3.415074","12":"2.926190","13":"2.985823","14":"-1.472657e-01","15":"-0.0536678917","16":"-4.327868e-02","17":"BC048679","18":"#CC3333","19":"BC048679","20":"#CC3333","21":"BC048679","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"17332"},{"1":"ENSMUSG00000109130","2":"Gm45187","3":"predicted gene 45187","4":"7","5":"98.90427","6":"98.90806","7":"1","8":"0.612895434","9":"0.421900047","10":"0.424680626","11":"3.420032","12":"2.985627","13":"3.140917","14":"-1.522241e-01","15":"-0.1131053257","16":"-1.983726e-01","17":"Gm45187","18":"#CC3333","19":"Gm45187","20":"#CC3333","21":"Gm45187","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"46830"},{"1":"ENSMUSG00000035354","2":"Uvrag","3":"UV radiation resistance associated gene","4":"7","5":"98.88502","6":"99.14114","7":"-1","8":"0.375293537","9":"0.088125399","10":"0.121405653","11":"3.425596","12":"3.000958","13":"3.187184","14":"-1.577885e-01","15":"-0.1284362511","16":"-2.446398e-01","17":"Uvrag","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"10005"},{"1":"ENSMUSG00000062235","2":"Gm5341","3":"predicted pseudogene 5341","4":"7","5":"90.10748","6":"90.10797","7":"-1","8":"-0.336842442","9":"-0.323178616","10":"-0.315571756","11":"3.438675","12":"2.913301","13":"3.015322","14":"-1.708672e-01","15":"-0.0407789181","16":"-7.277790e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"17430"},{"1":"ENSMUSG00000049583","2":"Grm5","3":"glutamate receptor, metabotropic 5","4":"7","5":"87.58417","6":"88.13491","7":"1","8":"0.014628480","9":"0.048540297","10":"0.102935063","11":"3.446428","12":"2.978809","13":"2.904250","14":"-1.786201e-01","15":"-0.1062873974","16":"3.829427e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"14564"},{"1":"ENSMUSG00000108861","2":"5930435M05Rik","3":"RIKEN cDNA 5930435M05 gene","4":"7","5":"83.17857","6":"83.20159","7":"-1","8":"0.711528742","9":"0.827020879","10":"0.795481060","11":"3.450296","12":"2.895518","13":"3.001460","14":"-1.824878e-01","15":"-0.0229959135","16":"-5.891596e-02","17":"5930435M05Rik","18":"#CC3333","19":"5930435M05Rik","20":"#CC3333","21":"5930435M05Rik","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46571"},{"1":"ENSMUSG00000018995","2":"Nars2","3":"asparaginyl-tRNA synthetase 2 (mitochondrial)(putative)","4":"7","5":"96.95150","6":"97.06476","7":"1","8":"0.309888868","9":"0.134445566","10":"0.195243363","11":"3.456624","12":"2.913297","13":"2.993475","14":"-1.888162e-01","15":"-0.0407757790","16":"-5.093031e-02","17":"Nars2","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"2040"},{"1":"ENSMUSG00000068629","2":"Gm6155","3":"predicted pseudogene 6155","4":"7","5":"84.83454","6":"84.83488","7":"-1","8":"0.090231985","9":"0.074919552","10":"0.083929112","11":"3.457202","12":"2.963423","13":"2.997869","14":"-1.893944e-01","15":"-0.0909017555","16":"-5.532432e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"19409"},{"1":"ENSMUSG00000082954","2":"Gm15504","3":"predicted gene 15504","4":"7","5":"80.23768","6":"80.23794","7":"-1","8":"0.499278476","9":"0.722411533","10":"0.685961969","11":"3.469391","12":"2.937072","13":"2.931240","14":"-2.015829e-01","15":"-0.0645508024","16":"1.130495e-02","17":"Gm15504","18":"#CC3333","19":"Gm15504","20":"#CC3333","21":"Gm15504","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"25185"},{"1":"ENSMUSG00000108868","2":"Gm32850","3":"predicted gene, 32850","4":"7","5":"82.12005","6":"82.16172","7":"1","8":"0.395505147","9":"0.343649759","10":"0.324162574","11":"3.471670","12":"2.901307","13":"2.977069","14":"-2.038620e-01","15":"-0.0287855337","16":"-3.452451e-02","17":"Gm32850","18":"#CC3333","19":"Gm32850","20":"#CC3333","21":"Gm32850","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46578"},{"1":"ENSMUSG00000052629","2":"Gm9885","3":"predicted gene 9885","4":"7","5":"78.57883","6":"78.58104","7":"1","8":"0.280140493","9":"0.243927454","10":"0.223619459","11":"3.476032","12":"2.929778","13":"3.047069","14":"-2.082240e-01","15":"-0.0572558732","16":"-1.045246e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"15335"},{"1":"ENSMUSG00000108983","2":"Gm44926","3":"predicted gene 44926","4":"7","5":"87.77257","6":"87.77439","7":"1","8":"-0.250328440","9":"-0.256116579","10":"-0.188213955","11":"3.482442","12":"2.827666","13":"3.014179","14":"-2.146342e-01","15":"0.0448556825","16":"-7.163407e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46688"},{"1":"ENSMUSG00000030544","2":"Mesp1","3":"mesoderm posterior 1","4":"7","5":"79.79224","6":"79.79379","7":"-1","8":"0.158540619","9":"0.305685285","10":"0.278287465","11":"3.483008","12":"3.017930","13":"3.109609","14":"-2.152003e-01","15":"-0.1454081584","16":"-1.670644e-01","17":"","18":"#666666","19":"Mesp1","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"7887"},{"1":"ENSMUSG00000035713","2":"Usp35","3":"ubiquitin specific peptidase 35","4":"7","5":"97.30938","6":"97.33202","7":"-1","8":"-0.033098084","9":"-0.065123557","10":"0.015078484","11":"3.483378","12":"3.085649","13":"3.297350","14":"-2.155700e-01","15":"-0.2131276949","16":"-3.548051e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"10125"},{"1":"ENSMUSG00000070462","2":"Tlnrd1","3":"talin rod domain containing 1","4":"7","5":"83.87987","6":"83.88430","7":"-1","8":"-0.235446738","9":"-0.329626386","10":"-0.283831236","11":"3.484381","12":"2.742930","13":"3.047924","14":"-2.165727e-01","15":"0.1295919921","16":"-1.053793e-01","17":"","18":"#666666","19":"Tlnrd1","20":"#CC3333","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"19785"},{"1":"ENSMUSG00000108387","2":"Gm31510","3":"predicted gene, 31510","4":"7","5":"79.27169","6":"79.27303","7":"-1","8":"0.465968562","9":"0.416089118","10":"0.353115832","11":"3.486719","12":"3.020569","13":"2.995077","14":"-2.189114e-01","15":"-0.1480468877","16":"-5.253234e-02","17":"Gm31510","18":"#CC3333","19":"Gm31510","20":"#CC3333","21":"Gm31510","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46131"},{"1":"ENSMUSG00000052572","2":"Dlg2","3":"discs large MAGUK scaffold protein 2","4":"7","5":"90.47667","6":"92.44925","7":"1","8":"0.038728471","9":"-0.087511400","10":"-0.133969889","11":"3.487494","12":"2.876913","13":"3.151341","14":"-2.196862e-01","15":"-0.0043914038","16":"-2.087961e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"15325"},{"1":"ENSMUSG00000080748","2":"Gm15661","3":"predicted gene 15661","4":"7","5":"88.44867","6":"88.44923","7":"-1","8":"-0.313507982","9":"-0.364198291","10":"-0.323926749","11":"3.488524","12":"2.975951","13":"2.982308","14":"-2.207157e-01","15":"-0.1034296380","16":"-3.976374e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"23262"},{"1":"ENSMUSG00000070459","2":"Olfr290","3":"olfactory receptor 290","4":"7","5":"84.91308","6":"84.92086","7":"1","8":"0.256947440","9":"0.354610559","10":"0.332628965","11":"3.506064","12":"2.956358","13":"2.997278","14":"-2.382564e-01","15":"-0.0838358315","16":"-5.473303e-02","17":"","18":"#666666","19":"Olfr290","20":"#CC3333","21":"Olfr290","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"19782"},{"1":"ENSMUSG00000116179","2":"Olfr290","3":"olfactory receptor 290","4":"7","5":"84.91578","6":"84.91673","7":"1","8":"0.256947440","9":"0.354610559","10":"0.332628965","11":"3.506064","12":"2.956358","13":"2.997278","14":"-2.382564e-01","15":"-0.0838358315","16":"-5.473303e-02","17":"","18":"#666666","19":"Olfr290","20":"#CC3333","21":"Olfr290","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"53421"},{"1":"ENSMUSG00000108506","2":"Gm44705","3":"predicted gene 44705","4":"7","5":"79.79029","6":"79.79063","7":"-1","8":"0.375432324","9":"0.196901382","10":"0.233610267","11":"3.506335","12":"2.838249","13":"2.979612","14":"-2.385269e-01","15":"0.0342722900","16":"-3.706719e-02","17":"Gm44705","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46241"},{"1":"ENSMUSG00000038540","2":"Tmc3","3":"transmembrane channel-like gene family 3","4":"7","5":"83.58493","6":"83.62561","7":"1","8":"0.445585336","9":"0.425494329","10":"0.412354163","11":"3.506539","12":"2.918519","13":"3.168203","14":"-2.387310e-01","15":"-0.0459974405","16":"-2.256584e-01","17":"Tmc3","18":"#CC3333","19":"Tmc3","20":"#CC3333","21":"Tmc3","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"11173"},{"1":"ENSMUSG00000098174","2":"Gm8183","3":"predicted gene 8183","4":"7","5":"88.57187","6":"88.57285","7":"-1","8":"-0.353663726","9":"-0.371107906","10":"-0.331183032","11":"3.508451","12":"2.926446","13":"2.975723","14":"-2.406429e-01","15":"-0.0539245324","16":"-3.317880e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"36608"},{"1":"ENSMUSG00000035623","2":"Rsf1","3":"remodeling and spacing factor 1","4":"7","5":"97.57989","6":"97.69278","7":"1","8":"0.258269173","9":"0.099186545","10":"0.174300915","11":"3.508510","12":"2.900579","13":"2.930881","14":"-2.407021e-01","15":"-0.0280577472","16":"1.166329e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"10099"},{"1":"ENSMUSG00000066372","2":"Vmn2r65","3":"vomeronasal 2, receptor 65","4":"7","5":"84.93556","6":"84.96412","7":"-1","8":"0.228443642","9":"0.193867920","10":"0.187100429","11":"3.511167","12":"2.899147","13":"2.946648","14":"-2.433586e-01","15":"-0.0266249841","16":"-4.103727e-03","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"18987"},{"1":"ENSMUSG00000103663","2":"2900037B21Rik","3":"RIKEN cDNA 2900037B21 gene","4":"7","5":"79.51292","6":"79.51326","7":"1","8":"0.257266340","9":"0.443912636","10":"0.426911028","11":"3.514649","12":"2.924053","13":"3.061563","14":"-2.468414e-01","15":"-0.0515316913","16":"-1.190181e-01","17":"","18":"#666666","19":"2900037B21Rik","20":"#CC3333","21":"2900037B21Rik","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"41704"},{"1":"ENSMUSG00000109278","2":"Gm45144","3":"predicted gene 45144","4":"7","5":"88.85304","6":"88.86489","7":"-1","8":"0.068593182","9":"0.098341021","10":"0.068011786","11":"3.515994","12":"2.923719","13":"2.961513","14":"-2.481865e-01","15":"-0.0511970352","16":"-1.896836e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46973"},{"1":"ENSMUSG00000092195","2":"Vmn2r-ps87","3":"vomeronasal 2, receptor, pseudogene 87","4":"7","5":"86.88487","6":"86.88574","7":"1","8":"0.178056475","9":"0.318400627","10":"0.298032013","11":"3.527499","12":"3.003294","13":"3.062496","14":"-2.596910e-01","15":"-0.1307725498","16":"-1.199518e-01","17":"","18":"#666666","19":"Vmn2r-ps87","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31873"},{"1":"ENSMUSG00000109381","2":"Gm44826","3":"predicted gene 44826","4":"7","5":"84.20818","6":"84.21051","7":"-1","8":"-0.237817784","9":"-0.340569392","10":"-0.321792595","11":"3.528246","12":"3.007115","13":"2.996174","14":"-2.604384e-01","15":"-0.1345931834","16":"-5.362987e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47072"},{"1":"ENSMUSG00000109106","2":"4933430H16Rik","3":"RIKEN cDNA 4933430H16 gene","4":"7","5":"82.93794","6":"82.94069","7":"-1","8":"0.404138397","9":"0.204224402","10":"0.194440692","11":"3.528270","12":"2.917034","13":"2.923444","14":"-2.604621e-01","15":"-0.0445123692","16":"1.910037e-02","17":"4933430H16Rik","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46807"},{"1":"ENSMUSG00000108449","2":"Gm44507","3":"predicted gene 44507","4":"7","5":"98.38809","6":"98.41562","7":"-1","8":"0.402759511","9":"0.327676785","10":"0.294379927","11":"3.528713","12":"2.946173","13":"2.979019","14":"-2.609050e-01","15":"-0.0736509591","16":"-3.647425e-02","17":"Gm44507","18":"#CC3333","19":"Gm44507","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46190"},{"1":"ENSMUSG00000109226","2":"Gm45039","3":"predicted gene 45039","4":"7","5":"93.60484","6":"93.60832","7":"-1","8":"0.107020625","9":"-0.002891029","10":"-0.024472931","11":"3.531037","12":"2.917272","13":"2.983687","14":"-2.632288e-01","15":"-0.0447501222","16":"-4.114257e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46922"},{"1":"ENSMUSG00000039043","2":"Arpin","3":"actin-related protein 2/3 complex inhibitor","4":"7","5":"79.92536","6":"79.93536","7":"-1","8":"-0.224015645","9":"-0.247103717","10":"-0.179236160","11":"3.533702","12":"2.928811","13":"3.459850","14":"-2.658937e-01","15":"-0.0562892891","16":"-5.173058e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11373"},{"1":"ENSMUSG00000108389","2":"Gm45206","3":"predicted gene 45206","4":"7","5":"80.20939","6":"80.21168","7":"1","8":"-0.267035730","9":"-0.284665436","10":"-0.241668795","11":"3.539902","12":"2.889201","13":"3.100421","14":"-2.720939e-01","15":"-0.0166791910","16":"-1.578768e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46133"},{"1":"ENSMUSG00000090958","2":"Lrrc32","3":"leucine rich repeat containing 32","4":"7","5":"98.48928","6":"98.50218","7":"1","8":"-0.222607562","9":"-0.327064237","10":"-0.303083268","11":"3.540028","12":"3.016115","13":"3.178443","14":"-2.722203e-01","15":"-0.1435933874","16":"-2.358981e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"31321"},{"1":"ENSMUSG00000025724","2":"Sec11a","3":"SEC11 homolog A, signal peptidase complex subunit","4":"7","5":"80.90489","6":"80.94778","7":"-1","8":"0.515798574","9":"0.497091650","10":"0.487894932","11":"3.546532","12":"3.125740","13":"3.333277","14":"-2.787237e-01","15":"-0.2532186384","16":"-3.907330e-01","17":"Sec11a","18":"#CC3333","19":"Sec11a","20":"#CC3333","21":"Sec11a","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"5158"},{"1":"ENSMUSG00000109077","2":"Gm8285","3":"predicted gene 8285","4":"7","5":"93.56494","6":"93.56570","7":"-1","8":"0.666063345","9":"0.625438559","10":"0.584377139","11":"3.549526","12":"2.984933","13":"3.004427","14":"-2.817182e-01","15":"-0.1124112772","16":"-6.188265e-02","17":"Gm8285","18":"#CC3333","19":"Gm8285","20":"#CC3333","21":"Gm8285","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46778"},{"1":"ENSMUSG00000091239","2":"Vmn2r76","3":"vomeronasal 2, receptor 76","4":"7","5":"86.20202","6":"86.24631","7":"-1","8":"0.033721729","9":"0.052688564","10":"0.029097447","11":"3.562616","12":"2.998725","13":"3.049776","14":"-2.948085e-01","15":"-0.1262032346","16":"-1.072317e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31444"},{"1":"ENSMUSG00000109110","2":"Gm45014","3":"predicted gene 45014","4":"7","5":"81.86029","6":"81.86253","7":"-1","8":"-0.079684314","9":"-0.175720927","10":"-0.207280779","11":"3.566359","12":"3.073427","13":"3.074815","14":"-2.985506e-01","15":"-0.2009054250","16":"-1.322709e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46811"},{"1":"ENSMUSG00000109145","2":"Gm20744","3":"predicted gene, 20744","4":"7","5":"82.07039","6":"82.08176","7":"1","8":"0.543184736","9":"0.554873576","10":"0.525479751","11":"3.567767","12":"2.977892","13":"3.129194","14":"-2.999590e-01","15":"-0.1053707133","16":"-1.866491e-01","17":"Gm20744","18":"#CC3333","19":"Gm20744","20":"#CC3333","21":"Gm20744","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46844"},{"1":"ENSMUSG00000091496","2":"Gm8398","3":"predicted gene 8398","4":"7","5":"98.85473","6":"98.85510","7":"-1","8":"-0.178254508","9":"-0.154929031","10":"-0.179033575","11":"3.570008","12":"2.942828","13":"2.987280","14":"-3.021996e-01","15":"-0.0703067227","16":"-4.473584e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31560"},{"1":"ENSMUSG00000097585","2":"E230029C05Rik","3":"RIKEN cDNA E230029C05 gene","4":"7","5":"89.98072","6":"90.04907","7":"1","8":"-0.206760691","9":"-0.220124039","10":"-0.164832626","11":"3.571297","12":"3.035829","13":"3.217030","14":"-3.034890e-01","15":"-0.1633070824","16":"-2.744857e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"36075"},{"1":"ENSMUSG00000038646","2":"Rammet","3":"RNMT actvating mRNA cap methylating subunit","4":"7","5":"81.76292","6":"81.76949","7":"1","8":"0.479393887","9":"0.221162816","10":"0.232740772","11":"3.575929","12":"3.231014","13":"3.107097","14":"-3.081209e-01","15":"-0.3584919914","16":"-1.645529e-01","17":"Rammet","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11222"},{"1":"ENSMUSG00000108385","2":"Gm44730","3":"predicted gene 44730","4":"7","5":"78.85549","6":"78.85844","7":"-1","8":"0.453191062","9":"0.394728060","10":"0.353761893","11":"3.576409","12":"2.999455","13":"3.025255","14":"-3.086006e-01","15":"-0.1269329377","16":"-8.271020e-02","17":"Gm44730","18":"#CC3333","19":"Gm44730","20":"#CC3333","21":"Gm44730","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46130"},{"1":"ENSMUSG00000063394","2":"Olfr293","3":"olfactory receptor 293","4":"7","5":"86.64906","6":"86.66604","7":"1","8":"0.359170522","9":"0.400026417","10":"0.363273955","11":"3.576692","12":"2.948689","13":"3.045807","14":"-3.088839e-01","15":"-0.0761672610","16":"-1.032630e-01","17":"Olfr293","18":"#CC3333","19":"Olfr293","20":"#CC3333","21":"Olfr293","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"17688"},{"1":"ENSMUSG00000109341","2":"Gm30873","3":"predicted gene, 30873","4":"7","5":"84.33809","6":"84.34787","7":"1","8":"-0.268723160","9":"-0.267238459","10":"-0.247987251","11":"3.583378","12":"3.124151","13":"3.261728","14":"-3.155702e-01","15":"-0.2516296603","16":"-3.191831e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47033"},{"1":"ENSMUSG00000094841","2":"Gm10610","3":"predicted gene 10610","4":"7","5":"83.53619","6":"83.55026","7":"-1","8":"0.397714882","9":"0.558200975","10":"0.536618032","11":"3.589847","12":"2.987279","13":"3.102019","14":"-3.220394e-01","15":"-0.1147577731","16":"-1.594747e-01","17":"Gm10610","18":"#CC3333","19":"Gm10610","20":"#CC3333","21":"Gm10610","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"33805"},{"1":"ENSMUSG00000085194","2":"Platr32","3":"pluripotency associated transcript 32","4":"7","5":"81.05911","6":"81.06631","7":"1","8":"0.409466340","9":"0.301518196","10":"0.331806318","11":"3.590453","12":"2.901805","13":"3.108896","14":"-3.226451e-01","15":"-0.0292833347","16":"-1.663517e-01","17":"Platr32","18":"#CC3333","19":"Platr32","20":"#CC3333","21":"Platr32","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"27069"},{"1":"ENSMUSG00000092214","2":"Gm18560","3":"predicted gene, 18560","4":"7","5":"84.91938","6":"84.92050","7":"-1","8":"0.465512525","9":"0.440915854","10":"0.405477756","11":"3.592963","12":"3.002461","13":"3.013389","14":"-3.251547e-01","15":"-0.1299395503","16":"-7.084453e-02","17":"Gm18560","18":"#CC3333","19":"Gm18560","20":"#CC3333","21":"Gm18560","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"31887"},{"1":"ENSMUSG00000108764","2":"Gm2926","3":"predicted gene 2926","4":"7","5":"98.45526","6":"98.45550","7":"1","8":"0.328328776","9":"0.292524792","10":"0.273896872","11":"3.593964","12":"3.009690","13":"3.036670","14":"-3.261564e-01","15":"-0.1371685814","16":"-9.412565e-02","17":"Gm2926","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46478"},{"1":"ENSMUSG00000090773","2":"Vmn2r-ps69","3":"vomeronasal 2, receptor, pseudogene 69","4":"7","5":"85.30395","6":"85.34289","7":"1","8":"0.248937389","9":"0.208575021","10":"0.188930780","11":"3.595143","12":"3.040388","13":"3.066302","14":"-3.273348e-01","15":"-0.1678662544","16":"-1.237573e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31251"},{"1":"ENSMUSG00000109367","2":"Gm18782","3":"predicted gene, 18782","4":"7","5":"80.97989","6":"80.98129","7":"1","8":"0.828489004","9":"0.884436505","10":"0.913925331","11":"3.597985","12":"2.845600","13":"2.859495","14":"-3.301770e-01","15":"0.0269215554","16":"8.304993e-02","17":"Gm18782","18":"#CC3333","19":"Gm18782","20":"#CC3333","21":"Gm18782","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"47058"},{"1":"ENSMUSG00000092429","2":"Gm18958","3":"predicted gene, 18958","4":"7","5":"84.80103","6":"84.80316","7":"1","8":"0.710874373","9":"0.726876482","10":"0.695643540","11":"3.598405","12":"3.008391","13":"3.134406","14":"-3.305966e-01","15":"-0.1358693355","16":"-1.918614e-01","17":"Gm18958","18":"#CC3333","19":"Gm18958","20":"#CC3333","21":"Gm18958","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"32062"},{"1":"ENSMUSG00000054498","2":"Olfr308","3":"olfactory receptor 308","4":"7","5":"86.31822","6":"86.32563","7":"-1","8":"0.198145023","9":"0.200249744","10":"0.163976148","11":"3.600511","12":"3.010350","13":"3.008987","14":"-3.327030e-01","15":"-0.1378279569","16":"-6.644268e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"15743"},{"1":"ENSMUSG00000051727","2":"Kctd14","3":"potassium channel tetramerisation domain containing 14","4":"7","5":"97.45132","6":"97.45956","7":"1","8":"0.095986935","9":"0.098611808","10":"0.080496028","11":"3.601028","12":"2.932422","13":"3.306713","14":"-3.332196e-01","15":"-0.0599005857","16":"-3.641687e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"15137"},{"1":"ENSMUSG00000092401","2":"Olfr300-ps1","3":"olfactory receptor 300, pseudogene 1","4":"7","5":"86.43591","6":"86.44526","7":"1","8":"0.473743957","9":"0.496372831","10":"0.457745494","11":"3.601801","12":"3.028953","13":"3.078880","14":"-3.339935e-01","15":"-0.1564309672","16":"-1.363357e-01","17":"Olfr300-ps1","18":"#CC3333","19":"Olfr300-ps1","20":"#CC3333","21":"Olfr300-ps1","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"32040"},{"1":"ENSMUSG00000109135","2":"Gm6240","3":"predicted gene 6240","4":"7","5":"88.78193","6":"88.78240","7":"-1","8":"0.328870687","9":"0.311817055","10":"0.278558256","11":"3.605071","12":"3.032240","13":"3.102088","14":"-3.372633e-01","15":"-0.1597185970","16":"-1.595432e-01","17":"Gm6240","18":"#CC3333","19":"Gm6240","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46835"},{"1":"ENSMUSG00000108424","2":"Gm44590","3":"predicted gene 44590","4":"7","5":"97.91824","6":"97.91863","7":"1","8":"0.511082624","9":"0.503935957","10":"0.457276939","11":"3.612727","12":"3.010666","13":"3.034156","14":"-3.449188e-01","15":"-0.1381444727","16":"-9.161191e-02","17":"Gm44590","18":"#CC3333","19":"Gm44590","20":"#CC3333","21":"Gm44590","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46166"},{"1":"ENSMUSG00000109323","2":"Gm44968","3":"predicted gene 44968","4":"7","5":"81.20501","6":"81.21088","7":"1","8":"0.140400014","9":"0.209147672","10":"0.183380575","11":"3.612800","12":"2.928316","13":"3.023727","14":"-3.449923e-01","15":"-0.0557944706","16":"-8.118245e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47015"},{"1":"ENSMUSG00000092467","2":"Gm18360","3":"predicted gene, 18360","4":"7","5":"85.51672","6":"85.51942","7":"1","8":"0.352399097","9":"0.343574815","10":"0.326739956","11":"3.613141","12":"2.982940","13":"3.041290","14":"-3.453326e-01","15":"-0.1104185650","16":"-9.874519e-02","17":"Gm18360","18":"#CC3333","19":"Gm18360","20":"#CC3333","21":"Gm18360","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"32089"},{"1":"ENSMUSG00000109370","2":"Gm44675","3":"predicted gene 44675","4":"7","5":"91.56437","6":"91.56729","7":"1","8":"-0.374076534","9":"-0.414696202","10":"-0.365381886","11":"3.614630","12":"3.455698","13":"3.407096","14":"-3.468220e-01","15":"-0.5831763229","16":"-4.645516e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"47061"},{"1":"ENSMUSG00000109513","2":"Gm44928","3":"predicted gene 44928","4":"7","5":"84.57572","6":"84.57584","7":"-1","8":"0.057891345","9":"0.005026216","10":"-0.017440807","11":"3.616091","12":"3.049915","13":"3.086360","14":"-3.482828e-01","15":"-0.1773936380","16":"-1.438150e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47199"},{"1":"ENSMUSG00000059319","2":"Olfr295","3":"olfactory receptor 295","4":"7","5":"86.58386","6":"86.59113","7":"1","8":"0.243921792","9":"0.290726519","10":"0.255672652","11":"3.617063","12":"3.022616","13":"3.084920","14":"-3.492549e-01","15":"-0.1500942786","16":"-1.423758e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"16780"},{"1":"ENSMUSG00000096861","2":"Vmn2r68","3":"vomeronasal 2, receptor 68","4":"7","5":"85.22114","6":"85.23778","7":"-1","8":"0.062481671","9":"-0.007117861","10":"-0.034264709","11":"3.620855","12":"3.015898","13":"3.058599","14":"-3.530466e-01","15":"-0.1433759024","16":"-1.160549e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"35436"},{"1":"ENSMUSG00000109052","2":"Gm45012","3":"predicted gene 45012","4":"7","5":"99.20237","6":"99.22708","7":"1","8":"0.337697092","9":"0.237131897","10":"0.200198962","11":"3.621666","12":"3.157314","13":"3.142561","14":"-3.538579e-01","15":"-0.2847923990","16":"-2.000164e-01","17":"Gm45012","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46754"},{"1":"ENSMUSG00000108994","2":"Gm31663","3":"predicted gene, 31663","4":"7","5":"92.97988","6":"93.00651","7":"1","8":"0.904614748","9":"0.823345637","10":"0.834504935","11":"3.624184","12":"2.650078","13":"2.523341","14":"-3.563761e-01","15":"0.2224437495","16":"4.192033e-01","17":"Gm31663","18":"#CC3333","19":"Gm31663","20":"#CC3333","21":"Gm31663","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46699"},{"1":"ENSMUSG00000055571","2":"Olfr305","3":"olfactory receptor 305","4":"7","5":"86.36085","6":"86.36693","7":"-1","8":"0.352463984","9":"0.297716918","10":"0.266320902","11":"3.626574","12":"3.030762","13":"3.077651","14":"-3.587662e-01","15":"-0.1582405543","16":"-1.351062e-01","17":"Olfr305","18":"#CC3333","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"15969"},{"1":"ENSMUSG00000108816","2":"Gm6230","3":"predicted gene 6230","4":"7","5":"87.24211","6":"87.24289","7":"-1","8":"0.237256432","9":"0.171824519","10":"0.151843041","11":"3.629321","12":"3.041318","13":"3.088596","14":"-3.615129e-01","15":"-0.1687966838","16":"-1.460513e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46528"},{"1":"ENSMUSG00000054054","2":"Olfr309","3":"olfactory receptor 309","4":"7","5":"86.30447","6":"86.31371","7":"-1","8":"0.450280662","9":"0.387618307","10":"0.352939714","11":"3.630156","12":"3.014261","13":"3.073918","14":"-3.623485e-01","15":"-0.1417390205","16":"-1.313733e-01","17":"Olfr309","18":"#CC3333","19":"Olfr309","20":"#CC3333","21":"Olfr309","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"15640"},{"1":"ENSMUSG00000055610","2":"Olfr307","3":"olfactory receptor 307","4":"7","5":"86.33218","6":"86.33759","7":"-1","8":"0.146866144","9":"0.098324839","10":"0.081724408","11":"3.630708","12":"3.016225","13":"3.160006","14":"-3.628996e-01","15":"-0.1437030355","16":"-2.174612e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"15973"},{"1":"ENSMUSG00000092376","2":"Gm19203","3":"predicted gene, 19203","4":"7","5":"86.90700","6":"86.90793","7":"-1","8":"0.252426163","9":"0.181123608","10":"0.166681113","11":"3.631614","12":"3.053631","13":"3.106807","14":"-3.638063e-01","15":"-0.1811097033","16":"-1.642628e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32020"},{"1":"ENSMUSG00000092530","2":"Vmn2r-ps64","3":"vomeronasal 2, receptor, pseudogene 64","4":"7","5":"85.03403","6":"85.03455","7":"1","8":"0.506116305","9":"0.481533010","10":"0.449497845","11":"3.633442","12":"3.028686","13":"3.100217","14":"-3.656336e-01","15":"-0.1561643782","16":"-1.576722e-01","17":"Vmn2r-ps64","18":"#CC3333","19":"Vmn2r-ps64","20":"#CC3333","21":"Vmn2r-ps64","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"32140"},{"1":"ENSMUSG00000090436","2":"Vmn2r75","3":"vomeronasal 2, receptor 75","4":"7","5":"86.14529","6":"86.17178","7":"-1","8":"0.321842613","9":"0.331497832","10":"0.297103036","11":"3.634360","12":"3.020694","13":"3.073832","14":"-3.665518e-01","15":"-0.1481724455","16":"-1.312879e-01","17":"Vmn2r75","18":"#CC3333","19":"Vmn2r75","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31087"},{"1":"ENSMUSG00000092532","2":"Vmn2r-ps84","3":"vomeronasal 2, receptor, pseudogene 84","4":"7","5":"86.12633","6":"86.12723","7":"1","8":"0.530219743","9":"0.483855066","10":"0.442548423","11":"3.634566","12":"3.047645","13":"3.083244","14":"-3.667581e-01","15":"-0.1751230800","16":"-1.406993e-01","17":"Vmn2r-ps84","18":"#CC3333","19":"Vmn2r-ps84","20":"#CC3333","21":"Vmn2r-ps84","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"32141"},{"1":"ENSMUSG00000097749","2":"Gm16938","3":"predicted gene, 16938","4":"7","5":"98.17719","6":"98.17940","7":"1","8":"-0.381889382","9":"-0.397706604","10":"-0.403742704","11":"3.639011","12":"3.135963","13":"3.114425","14":"-3.712032e-01","15":"-0.2634413679","16":"-1.718801e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"36221"},{"1":"ENSMUSG00000092425","2":"Gm8018","3":"predicted gene 8018","4":"7","5":"84.87955","6":"84.88253","7":"1","8":"0.264380474","9":"0.179185689","10":"0.144619487","11":"3.639921","12":"3.045473","13":"3.085660","14":"-3.721127e-01","15":"-0.1729512412","16":"-1.431153e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32058"},{"1":"ENSMUSG00000108714","2":"Gm21057","3":"predicted gene, 21057","4":"7","5":"80.06271","6":"80.07527","7":"-1","8":"0.524890400","9":"0.426697889","10":"0.444795826","11":"3.640982","12":"3.137229","13":"3.322843","14":"-3.731740e-01","15":"-0.2647078161","16":"-3.802981e-01","17":"Gm21057","18":"#CC3333","19":"Gm21057","20":"#CC3333","21":"Gm21057","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46433"},{"1":"ENSMUSG00000108875","2":"Gm45064","3":"predicted gene 45064","4":"7","5":"89.55237","6":"89.56808","7":"1","8":"0.378764358","9":"0.358174756","10":"0.323148674","11":"3.641912","12":"3.037435","13":"3.066154","14":"-3.741037e-01","15":"-0.1649132957","16":"-1.236099e-01","17":"Gm45064","18":"#CC3333","19":"Gm45064","20":"#CC3333","21":"Gm45064","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46584"},{"1":"ENSMUSG00000020168","2":"Olfr299","3":"olfactory receptor 299","4":"7","5":"86.45992","6":"86.46956","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Olfr299","18":"#CC3333","19":"Olfr299","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"2356"},{"1":"ENSMUSG00000039608","2":"Olfr303","3":"olfactory receptor 303","4":"7","5":"86.39181","6":"86.39779","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Olfr303","18":"#CC3333","19":"Olfr303","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"11588"},{"1":"ENSMUSG00000051877","2":"Vmn2r72","3":"vomeronasal 2, receptor 72","4":"7","5":"85.73594","6":"85.75510","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r72","18":"#CC3333","19":"Vmn2r72","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"15168"},{"1":"ENSMUSG00000052779","2":"4833418N17Rik","3":"RIKEN cDNA 4833418N17 gene","4":"7","5":"81.99200","6":"81.99262","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"4833418N17Rik","18":"#CC3333","19":"4833418N17Rik","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"15367"},{"1":"ENSMUSG00000057067","2":"Olfr297","3":"olfactory receptor 297","4":"7","5":"86.51997","6":"86.52850","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Olfr297","18":"#CC3333","19":"Olfr297","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"16292"},{"1":"ENSMUSG00000057540","2":"Olfr310","3":"olfactory receptor 310","4":"7","5":"86.26849","6":"86.27717","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Olfr310","18":"#CC3333","19":"Olfr310","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"16400"},{"1":"ENSMUSG00000061549","2":"Olfr301","3":"olfactory receptor 301","4":"7","5":"86.40385","6":"86.41402","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Olfr301","18":"#CC3333","19":"Olfr301","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"17267"},{"1":"ENSMUSG00000062042","2":"Olfr294","3":"olfactory receptor 294","4":"7","5":"86.61564","6":"86.61664","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Olfr294","18":"#CC3333","19":"Olfr294","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"17373"},{"1":"ENSMUSG00000062426","2":"Olfr304","3":"olfactory receptor 304","4":"7","5":"86.38566","6":"86.38666","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Olfr304","18":"#CC3333","19":"Olfr304","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"17469"},{"1":"ENSMUSG00000062878","2":"Olfr298","3":"olfactory receptor 298","4":"7","5":"86.48632","6":"86.49248","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Olfr298","18":"#CC3333","19":"Olfr298","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"17576"},{"1":"ENSMUSG00000064940","2":"Gm23239","3":"predicted gene, 23239","4":"7","5":"77.91014","6":"77.91032","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm23239","18":"#CC3333","19":"Gm23239","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"18277"},{"1":"ENSMUSG00000065227","2":"Gm25790","3":"predicted gene, 25790","4":"7","5":"84.32331","6":"84.32341","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm25790","18":"#CC3333","19":"Gm25790","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"18417"},{"1":"ENSMUSG00000065311","2":"n-R5s156","3":"nuclear encoded rRNA 5S 156","4":"7","5":"99.49772","6":"99.49782","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"n-R5s156","18":"#CC3333","19":"n-R5s156","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"18471"},{"1":"ENSMUSG00000065360","2":"Gm24412","3":"predicted gene, 24412","4":"7","5":"97.52181","6":"97.52192","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm24412","18":"#CC3333","19":"Gm24412","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"18496"},{"1":"ENSMUSG00000065609","2":"Mir7-2","3":"microRNA 7-2","4":"7","5":"78.88828","6":"78.88837","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Mir7-2","18":"#CC3333","19":"Mir7-2","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"18706"},{"1":"ENSMUSG00000065803","2":"Gm23040","3":"predicted gene, 23040","4":"7","5":"98.50695","6":"98.50712","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm23040","18":"#CC3333","19":"Gm23040","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"18825"},{"1":"ENSMUSG00000065875","2":"Gm25684","3":"predicted gene, 25684","4":"7","5":"95.44235","6":"95.44246","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm25684","18":"#CC3333","19":"Gm25684","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"18863"},{"1":"ENSMUSG00000066378","2":"Gm10160","3":"predicted gene 10160","4":"7","5":"81.85553","6":"81.85579","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm10160","18":"#CC3333","19":"Gm10160","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"18988"},{"1":"ENSMUSG00000070458","2":"Vmn2r73","3":"vomeronasal 2, receptor 73","4":"7","5":"85.85067","6":"85.87627","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r73","18":"#CC3333","19":"Vmn2r73","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"19781"},{"1":"ENSMUSG00000076143","2":"Mir708","3":"microRNA 708","4":"7","5":"96.24942","6":"96.24953","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Mir708","18":"#CC3333","19":"Mir708","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"21385"},{"1":"ENSMUSG00000076389","2":"Gm23192","3":"predicted gene, 23192","4":"7","5":"86.29866","6":"86.29876","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm23192","18":"#CC3333","19":"Gm23192","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"21468"},{"1":"ENSMUSG00000077059","2":"Gm24541","3":"predicted gene, 24541","4":"7","5":"79.94919","6":"79.94928","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm24541","18":"#CC3333","19":"Gm24541","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"21768"},{"1":"ENSMUSG00000077149","2":"Gm22934","3":"predicted gene, 22934","4":"7","5":"82.99483","6":"82.99497","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm22934","18":"#CC3333","19":"Gm22934","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"21793"},{"1":"ENSMUSG00000077693","2":"Gm25345","3":"predicted gene, 25345","4":"7","5":"88.34397","6":"88.34410","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm25345","18":"#CC3333","19":"Gm25345","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"22174"},{"1":"ENSMUSG00000084462","2":"n-R5s154","3":"nuclear encoded rRNA 5S 154","4":"7","5":"82.75487","6":"82.75498","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"n-R5s154","18":"#CC3333","19":"n-R5s154","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"26499"},{"1":"ENSMUSG00000084480","2":"n-R5s155","3":"nuclear encoded rRNA 5S 155","4":"7","5":"90.29373","6":"90.29385","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"n-R5s155","18":"#CC3333","19":"n-R5s155","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"26514"},{"1":"ENSMUSG00000087947","2":"Gm24249","3":"predicted gene, 24249","4":"7","5":"88.08719","6":"88.08725","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm24249","18":"#CC3333","19":"Gm24249","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"29272"},{"1":"ENSMUSG00000087980","2":"Gm22226","3":"predicted gene, 22226","4":"7","5":"96.91999","6":"96.92010","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm22226","18":"#CC3333","19":"Gm22226","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"29292"},{"1":"ENSMUSG00000088057","2":"Gm26149","3":"predicted gene, 26149","4":"7","5":"81.66973","6":"81.66984","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm26149","18":"#CC3333","19":"Gm26149","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"29348"},{"1":"ENSMUSG00000088216","2":"Gm22080","3":"predicted gene, 22080","4":"7","5":"91.39258","6":"91.39269","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm22080","18":"#CC3333","19":"Gm22080","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"29470"},{"1":"ENSMUSG00000088877","2":"Gm25860","3":"predicted gene, 25860","4":"7","5":"93.14785","6":"93.14797","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm25860","18":"#CC3333","19":"Gm25860","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"29988"},{"1":"ENSMUSG00000089111","2":"Gm25907","3":"predicted gene, 25907","4":"7","5":"81.96571","6":"81.96583","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm25907","18":"#CC3333","19":"Gm25907","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"30173"},{"1":"ENSMUSG00000089259","2":"Gm25712","3":"predicted gene, 25712","4":"7","5":"96.86329","6":"96.86338","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm25712","18":"#CC3333","19":"Gm25712","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"30281"},{"1":"ENSMUSG00000089271","2":"Gm24552","3":"predicted gene, 24552","4":"7","5":"91.24299","6":"91.24310","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm24552","18":"#CC3333","19":"Gm24552","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"30293"},{"1":"ENSMUSG00000090774","2":"Vmn2r74","3":"vomeronasal 2, receptor 74","4":"7","5":"85.94461","6":"85.96163","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r74","18":"#CC3333","19":"Vmn2r74","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31252"},{"1":"ENSMUSG00000090806","2":"Vmn2r70","3":"vomeronasal 2, receptor 70","4":"7","5":"85.54841","6":"85.56919","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r70","18":"#CC3333","19":"Vmn2r70","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31269"},{"1":"ENSMUSG00000091006","2":"Vmn2r69","3":"vomeronasal 2, receptor 69","4":"7","5":"85.39383","6":"85.41592","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r69","18":"#CC3333","19":"Vmn2r69","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31340"},{"1":"ENSMUSG00000091205","2":"Vmn2r71","3":"vomeronasal 2, receptor 71","4":"7","5":"85.56309","6":"85.62468","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r71","18":"#CC3333","19":"Vmn2r71","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31431"},{"1":"ENSMUSG00000092179","2":"Vmn2r-ps75","3":"vomeronasal 2, receptor, pseudogene 75","4":"7","5":"85.79832","6":"85.79894","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps75","18":"#CC3333","19":"Vmn2r-ps75","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31860"},{"1":"ENSMUSG00000092197","2":"Gm20480","3":"predicted gene 20480","4":"7","5":"86.29864","6":"86.29889","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm20480","18":"#CC3333","19":"Gm20480","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31875"},{"1":"ENSMUSG00000092199","2":"Gm18358","3":"predicted gene, 18358","4":"7","5":"85.08877","6":"85.09116","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm18358","18":"#CC3333","19":"Gm18358","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31876"},{"1":"ENSMUSG00000092211","2":"Vmn2r-ps67","3":"vomeronasal 2, receptor, pseudogene 67","4":"7","5":"85.08398","6":"85.08436","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps67","18":"#CC3333","19":"Vmn2r-ps67","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31884"},{"1":"ENSMUSG00000092256","2":"Gm18359","3":"predicted gene, 18359","4":"7","5":"85.47696","6":"85.48202","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm18359","18":"#CC3333","19":"Gm18359","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31922"},{"1":"ENSMUSG00000092259","2":"Vmn2r-ps68","3":"vomeronasal 2, receptor, pseudogene 68","4":"7","5":"85.19220","6":"85.19306","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps68","18":"#CC3333","19":"Vmn2r-ps68","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31925"},{"1":"ENSMUSG00000092263","2":"Vmn2r-ps79","3":"vomeronasal 2, receptor, pseudogene 79","4":"7","5":"85.96702","6":"85.96739","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps79","18":"#CC3333","19":"Vmn2r-ps79","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31928"},{"1":"ENSMUSG00000092287","2":"Vmn2r-ps77","3":"vomeronasal 2, receptor, pseudogene 77","4":"7","5":"85.82194","6":"85.82226","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps77","18":"#CC3333","19":"Vmn2r-ps77","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31948"},{"1":"ENSMUSG00000092293","2":"Gm17919","3":"predicted gene, 17919","4":"7","5":"86.54197","6":"86.54266","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm17919","18":"#CC3333","19":"Gm17919","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31952"},{"1":"ENSMUSG00000092307","2":"Vmn2r-ps76","3":"vomeronasal 2, receptor, pseudogene 76","4":"7","5":"85.81545","6":"85.81619","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps76","18":"#CC3333","19":"Vmn2r-ps76","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31964"},{"1":"ENSMUSG00000092313","2":"Gm20415","3":"predicted gene 20415","4":"7","5":"85.28816","6":"85.28961","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm20415","18":"#CC3333","19":"Gm20415","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31970"},{"1":"ENSMUSG00000092330","2":"Vmn2r-ps82","3":"vomeronasal 2, receptor, pseudogene 82","4":"7","5":"86.05329","6":"86.06346","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps82","18":"#CC3333","19":"Vmn2r-ps82","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31981"},{"1":"ENSMUSG00000092352","2":"Vmn2r-ps74","3":"vomeronasal 2, receptor, pseudogene 74","4":"7","5":"85.71229","6":"85.71321","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps74","18":"#CC3333","19":"Vmn2r-ps74","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"31998"},{"1":"ENSMUSG00000092357","2":"Vmn2r-ps83","3":"vomeronasal 2, receptor, pseudogene 83","4":"7","5":"86.07315","6":"86.08378","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps83","18":"#CC3333","19":"Vmn2r-ps83","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32002"},{"1":"ENSMUSG00000092372","2":"Olfr296-ps1","3":"olfactory receptor 296, pseudogene 1","4":"7","5":"86.55948","6":"86.56268","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Olfr296-ps1","18":"#CC3333","19":"Olfr296-ps1","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32016"},{"1":"ENSMUSG00000092388","2":"Olfr306-ps1","3":"olfactory receptor 306, pseudogene 1","4":"7","5":"86.35298","6":"86.35372","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Olfr306-ps1","18":"#CC3333","19":"Olfr306-ps1","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32029"},{"1":"ENSMUSG00000092410","2":"Gm18354","3":"predicted gene, 18354","4":"7","5":"85.78820","6":"85.78965","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm18354","18":"#CC3333","19":"Gm18354","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32047"},{"1":"ENSMUSG00000092419","2":"Vmn2r-ps86","3":"vomeronasal 2, receptor, pseudogene 86","4":"7","5":"86.18573","6":"86.18660","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps86","18":"#CC3333","19":"Vmn2r-ps86","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32056"},{"1":"ENSMUSG00000092422","2":"Vmn2r-ps65","3":"vomeronasal 2, receptor, pseudogene 65","4":"7","5":"85.04178","6":"85.04200","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps65","18":"#CC3333","19":"Vmn2r-ps65","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32057"},{"1":"ENSMUSG00000092426","2":"Gm18779","3":"predicted gene, 18779","4":"7","5":"85.78502","6":"85.78560","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm18779","18":"#CC3333","19":"Gm18779","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32059"},{"1":"ENSMUSG00000092431","2":"Vmn2r-ps70","3":"vomeronasal 2, receptor, pseudogene 70","4":"7","5":"85.51185","6":"85.51215","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps70","18":"#CC3333","19":"Vmn2r-ps70","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32063"},{"1":"ENSMUSG00000092440","2":"Vmn2r-ps78","3":"vomeronasal 2, receptor, pseudogene 78","4":"7","5":"85.88446","6":"85.88476","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps78","18":"#CC3333","19":"Vmn2r-ps78","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32069"},{"1":"ENSMUSG00000092529","2":"Vmn2r-ps80","3":"vomeronasal 2, receptor, pseudogene 80","4":"7","5":"85.99798","6":"85.99959","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps80","18":"#CC3333","19":"Vmn2r-ps80","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32139"},{"1":"ENSMUSG00000092533","2":"Vmn2r-ps72","3":"vomeronasal 2, receptor, pseudogene 72","4":"7","5":"85.58965","6":"85.59056","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps72","18":"#CC3333","19":"Vmn2r-ps72","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32142"},{"1":"ENSMUSG00000092567","2":"Vmn2r-ps73","3":"vomeronasal 2, receptor, pseudogene 73","4":"7","5":"85.69636","6":"85.69858","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps73","18":"#CC3333","19":"Vmn2r-ps73","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32174"},{"1":"ENSMUSG00000092587","2":"Vmn2r-ps85","3":"vomeronasal 2, receptor, pseudogene 85","4":"7","5":"86.18120","6":"86.18214","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps85","18":"#CC3333","19":"Vmn2r-ps85","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32192"},{"1":"ENSMUSG00000092608","2":"Vmn2r-ps81","3":"vomeronasal 2, receptor, pseudogene 81","4":"7","5":"86.02672","6":"86.02755","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps81","18":"#CC3333","19":"Vmn2r-ps81","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32208"},{"1":"ENSMUSG00000092621","2":"Vmn2r-ps66","3":"vomeronasal 2, receptor, pseudogene 66","4":"7","5":"85.05308","6":"85.07094","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r-ps66","18":"#CC3333","19":"Vmn2r-ps66","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32220"},{"1":"ENSMUSG00000092743","2":"Gm22177","3":"predicted gene, 22177","4":"7","5":"84.17056","6":"84.17072","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm22177","18":"#CC3333","19":"Gm22177","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32287"},{"1":"ENSMUSG00000093238","2":"Mir9-3","3":"microRNA 9-3","4":"7","5":"79.50526","6":"79.50535","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Mir9-3","18":"#CC3333","19":"Mir9-3","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"32580"},{"1":"ENSMUSG00000094780","2":"Gm24659","3":"predicted gene, 24659","4":"7","5":"89.29119","6":"89.29130","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm24659","18":"#CC3333","19":"Gm24659","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"33756"},{"1":"ENSMUSG00000095664","2":"Vmn2r67","3":"vomeronasal 2, receptor 67","4":"7","5":"85.12697","6":"85.15600","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Vmn2r67","18":"#CC3333","19":"Vmn2r67","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"34469"},{"1":"ENSMUSG00000096631","2":"Gm22171","3":"predicted gene, 22171","4":"7","5":"89.29575","6":"89.29586","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm22171","18":"#CC3333","19":"Gm22171","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"35241"},{"1":"ENSMUSG00000099030","2":"Mir6394","3":"microRNA 6394","4":"7","5":"96.30567","6":"96.30576","7":"1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Mir6394","18":"#CC3333","19":"Mir6394","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"37385"},{"1":"ENSMUSG00000104817","2":"Gm44358","3":"predicted gene, 44358","4":"7","5":"78.02654","6":"78.02664","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm44358","18":"#CC3333","19":"Gm44358","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"42793"},{"1":"ENSMUSG00000108346","2":"Gm44980","3":"predicted gene 44980","4":"7","5":"98.43721","6":"98.43752","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm44980","18":"#CC3333","19":"Gm44980","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46092"},{"1":"ENSMUSG00000108508","2":"Gm44978","3":"predicted gene 44978","4":"7","5":"98.46513","6":"98.46545","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm44978","18":"#CC3333","19":"Gm44978","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46243"},{"1":"ENSMUSG00000108988","2":"Gm44934","3":"predicted gene 44934","4":"7","5":"98.73475","6":"98.73508","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm44934","18":"#CC3333","19":"Gm44934","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46693"},{"1":"ENSMUSG00000109440","2":"Bc1-ps1","3":"brain cytoplasmic RNA 1, pseudogene 1","4":"7","5":"93.66024","6":"93.66056","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Bc1-ps1","18":"#CC3333","19":"Bc1-ps1","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47130"},{"1":"ENSMUSG00000109541","2":"Gm6112","3":"predicted gene 6112","4":"7","5":"82.96659","6":"82.96722","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm6112","18":"#CC3333","19":"Gm6112","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47226"},{"1":"ENSMUSG00000110039","2":"Gm45541","3":"predicted gene 45541","4":"7","5":"86.40277","6":"86.40282","7":"-1","8":"0.353765916","9":"0.326532047","10":"0.293652423","11":"3.643420","12":"3.026417","13":"3.083946","14":"-3.756119e-01","15":"-0.1538958051","16":"-1.414012e-01","17":"Gm45541","18":"#CC3333","19":"Gm45541","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47716"},{"1":"ENSMUSG00000109199","2":"Gm44996","3":"predicted gene 44996","4":"7","5":"89.38834","6":"89.39132","7":"-1","8":"-0.277872498","9":"-0.282020358","10":"-0.217440133","11":"3.644643","12":"2.858940","13":"3.149975","14":"-3.768346e-01","15":"0.0135820965","16":"-2.074302e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46896"},{"1":"ENSMUSG00000093107","2":"Mir1839","3":"microRNA 1839","4":"7","5":"81.52985","6":"81.52998","7":"1","8":"0.613349813","9":"0.543167260","10":"0.539227691","11":"3.648296","12":"2.977537","13":"3.139259","14":"-3.804884e-01","15":"-0.1050157407","16":"-1.967150e-01","17":"Mir1839","18":"#CC3333","19":"Mir1839","20":"#CC3333","21":"Mir1839","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"32500"},{"1":"ENSMUSG00000104618","2":"Mir1839","3":"microRNA 1839","4":"7","5":"81.52992","6":"81.52999","7":"1","8":"0.613349813","9":"0.543167260","10":"0.539227691","11":"3.648296","12":"2.977537","13":"3.139259","14":"-3.804884e-01","15":"-0.1050157407","16":"-1.967150e-01","17":"Mir1839","18":"#CC3333","19":"Mir1839","20":"#CC3333","21":"Mir1839","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"42594"},{"1":"ENSMUSG00000092562","2":"Gm20414","3":"predicted gene 20414","4":"7","5":"85.26069","6":"85.26133","7":"-1","8":"0.508678555","9":"0.480175334","10":"0.443923307","11":"3.650954","12":"3.032499","13":"3.115042","14":"-3.831464e-01","15":"-0.1599777910","16":"-1.724970e-01","17":"Gm20414","18":"#CC3333","19":"Gm20414","20":"#CC3333","21":"Gm20414","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"32169"},{"1":"ENSMUSG00000082400","2":"Gm16017","3":"predicted gene 16017","4":"7","5":"79.41231","6":"79.41252","7":"-1","8":"0.308218829","9":"0.304449290","10":"0.289647495","11":"3.653570","12":"3.025596","13":"3.103099","14":"-3.857625e-01","15":"-0.1530743751","16":"-1.605543e-01","17":"Gm16017","18":"#CC3333","19":"Gm16017","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"24706"},{"1":"ENSMUSG00000109347","2":"Gm45175","3":"predicted gene 45175","4":"7","5":"84.44283","6":"84.44455","7":"-1","8":"0.529546602","9":"0.651822209","10":"0.639765493","11":"3.654785","12":"3.110256","13":"3.098069","14":"-3.869774e-01","15":"-0.2377345872","16":"-1.555247e-01","17":"Gm45175","18":"#CC3333","19":"Gm45175","20":"#CC3333","21":"Gm45175","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"47039"},{"1":"ENSMUSG00000108516","2":"Gm44602","3":"predicted gene 44602","4":"7","5":"94.47558","6":"94.47571","7":"1","8":"0.579300055","9":"0.546294466","10":"0.507148628","11":"3.663902","12":"3.016892","13":"3.080000","14":"-3.960936e-01","15":"-0.1443702855","16":"-1.374551e-01","17":"Gm44602","18":"#CC3333","19":"Gm44602","20":"#CC3333","21":"Gm44602","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46251"},{"1":"ENSMUSG00000109422","2":"Gm45174","3":"predicted gene 45174","4":"7","5":"84.55419","6":"84.55725","7":"1","8":"0.361333335","9":"0.352724186","10":"0.421322524","11":"3.667049","12":"2.890247","13":"3.049827","14":"-3.992407e-01","15":"-0.0177254163","16":"-1.072822e-01","17":"Gm45174","18":"#CC3333","19":"Gm45174","20":"#CC3333","21":"Gm45174","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"47112"},{"1":"ENSMUSG00000081372","2":"Gm16171","3":"predicted gene 16171","4":"7","5":"80.90783","6":"80.90815","7":"1","8":"0.508225758","9":"0.436308511","10":"0.402980722","11":"3.675226","12":"2.945190","13":"3.179318","14":"-4.074178e-01","15":"-0.0726680278","16":"-2.367733e-01","17":"Gm16171","18":"#CC3333","19":"Gm16171","20":"#CC3333","21":"Gm16171","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"23806"},{"1":"ENSMUSG00000090949","2":"Vmn2r77","3":"vomeronasal 2, receptor 77","4":"7","5":"86.79503","6":"86.81514","7":"1","8":"0.464076525","9":"0.392423523","10":"0.393079191","11":"3.678052","12":"3.155931","13":"3.167249","14":"-4.102437e-01","15":"-0.2834095406","16":"-2.247047e-01","17":"Vmn2r77","18":"#CC3333","19":"Vmn2r77","20":"#CC3333","21":"Vmn2r77","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"31316"},{"1":"ENSMUSG00000109042","2":"Gm44991","3":"predicted gene 44991","4":"7","5":"83.47171","6":"83.48547","7":"1","8":"0.203042063","9":"0.159130369","10":"0.132314184","11":"3.678488","12":"3.018663","13":"3.037291","14":"-4.106797e-01","15":"-0.1461417969","16":"-9.474684e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46745"},{"1":"ENSMUSG00000108897","2":"Gm44861","3":"predicted gene 44861","4":"7","5":"90.04270","6":"90.04392","7":"1","8":"-0.210223345","9":"-0.218427148","10":"-0.164028837","11":"3.689127","12":"2.960906","13":"3.100859","14":"-4.213194e-01","15":"-0.0883843197","16":"-1.583148e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46606"},{"1":"ENSMUSG00000108757","2":"Gm45202","3":"predicted gene 45202","4":"7","5":"80.15517","6":"80.15677","7":"1","8":"0.505463579","9":"0.426319759","10":"0.388204710","11":"3.695903","12":"3.017480","13":"3.095826","14":"-4.280947e-01","15":"-0.1449586528","16":"-1.532818e-01","17":"Gm45202","18":"#CC3333","19":"Gm45202","20":"#CC3333","21":"Gm45202","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46472"},{"1":"ENSMUSG00000070436","2":"Serpinh1","3":"serine (or cysteine) peptidase inhibitor, clade H, member 1","4":"7","5":"99.34538","6":"99.35324","7":"-1","8":"0.717110847","9":"0.570450417","10":"0.620047658","11":"3.696662","12":"3.265384","13":"3.300923","14":"-4.288539e-01","15":"-0.3928622954","16":"-3.583785e-01","17":"Serpinh1","18":"#CC3333","19":"Serpinh1","20":"#CC3333","21":"Serpinh1","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"19777"},{"1":"ENSMUSG00000109334","2":"Gm44925","3":"predicted gene 44925","4":"7","5":"87.11887","6":"87.12098","7":"1","8":"0.346858345","9":"0.312440003","10":"0.288650877","11":"3.701222","12":"3.072189","13":"3.145179","14":"-4.334142e-01","15":"-0.1996670540","16":"-2.026347e-01","17":"Gm44925","18":"#CC3333","19":"Gm44925","20":"#CC3333","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47026"},{"1":"ENSMUSG00000109335","2":"Gm44845","3":"predicted gene 44845","4":"7","5":"83.23454","6":"83.23488","7":"1","8":"0.208907391","9":"-0.010080191","10":"-0.017641527","11":"3.711433","12":"3.236246","13":"3.125814","14":"-4.436247e-01","15":"-0.3637240223","16":"-1.832691e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"47027"},{"1":"ENSMUSG00000035686","2":"Thrsp","3":"thyroid hormone responsive","4":"7","5":"97.41294","6":"97.41773","7":"-1","8":"0.215594318","9":"0.109666193","10":"0.173005235","11":"3.713651","12":"2.878351","13":"3.283396","14":"-4.458429e-01","15":"-0.0058290548","16":"-3.408517e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"10117"},{"1":"ENSMUSG00000102995","2":"A330074H02Rik","3":"RIKEN cDNA A330074H02 gene","4":"7","5":"79.51036","6":"79.51140","7":"1","8":"0.287419491","9":"0.491061607","10":"0.511465123","11":"3.722313","12":"2.841842","13":"3.410496","14":"-4.545047e-01","15":"0.0306801244","16":"-4.679515e-01","17":"","18":"#666666","19":"A330074H02Rik","20":"#CC3333","21":"A330074H02Rik","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"41096"},{"1":"ENSMUSG00000092217","2":"Vmn2r-ps62","3":"vomeronasal 2, receptor, pseudogene 62","4":"7","5":"84.81043","6":"84.81144","7":"-1","8":"0.280775501","9":"0.473961551","10":"0.448648437","11":"3.741415","12":"3.116050","13":"3.329156","14":"-4.736070e-01","15":"-0.2435282548","16":"-3.866120e-01","17":"","18":"#666666","19":"Vmn2r-ps62","20":"#CC3333","21":"Vmn2r-ps62","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"31889"},{"1":"ENSMUSG00000038930","2":"Rccd1","3":"RCC1 domain containing 1","4":"7","5":"80.29323","6":"80.32445","7":"-1","8":"0.015271491","9":"-0.022270179","10":"0.049144683","11":"3.774346","12":"3.396290","13":"3.206957","14":"-5.065378e-01","15":"-0.5237682720","16":"-2.644123e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11336"},{"1":"ENSMUSG00000097023","2":"Mir9-3hg","3":"Mir9-3 host gene","4":"7","5":"79.50003","6":"79.53440","7":"1","8":"0.449732950","9":"0.624094674","10":"0.630299304","11":"3.778079","12":"3.143868","13":"3.702013","14":"-5.102710e-01","15":"-0.2713459487","16":"-7.594687e-01","17":"Mir9-3hg","18":"#CC3333","19":"Mir9-3hg","20":"#CC3333","21":"Mir9-3hg","22":"#CC3333","23":"TRUE","24":"NA","_rn_":"35582"},{"1":"ENSMUSG00000108442","2":"Rpl15-ps5","3":"ribosomal protein L15, pseudogene 5","4":"7","5":"97.59484","6":"97.59544","7":"-1","8":"-0.022428289","9":"-0.031249591","10":"-0.004579962","11":"3.779337","12":"2.954387","13":"3.413061","14":"-5.115288e-01","15":"-0.0818650408","16":"-4.705170e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46183"},{"1":"ENSMUSG00000038797","2":"Zscan2","3":"zinc finger and SCAN domain containing 2","4":"7","5":"80.86092","6":"80.87654","7":"1","8":"0.211602592","9":"0.181269236","10":"0.257146192","11":"3.784288","12":"3.054266","13":"3.355973","14":"-5.164797e-01","15":"-0.1817443847","16":"-4.134288e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"11289"},{"1":"ENSMUSG00000109315","2":"Gm45037","3":"predicted gene 45037","4":"7","5":"93.36168","6":"93.38086","7":"1","8":"0.362472826","9":"0.363932317","10":"0.337255363","11":"3.801149","12":"3.046045","13":"3.150346","14":"-5.333415e-01","15":"-0.1735233406","16":"-2.078014e-01","17":"Gm45037","18":"#CC3333","19":"Gm45037","20":"#CC3333","21":"Gm45037","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"47007"},{"1":"ENSMUSG00000109257","2":"Gm45200","3":"predicted gene 45200","4":"7","5":"92.77850","6":"92.77942","7":"1","8":"0.027939742","9":"0.205564676","10":"0.229358057","11":"3.806879","12":"2.839053","13":"3.138256","14":"-5.390709e-01","15":"0.0334682987","16":"-1.957116e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"46953"},{"1":"ENSMUSG00000109175","2":"Gm44947","3":"predicted gene 44947","4":"7","5":"93.29070","6":"93.32395","7":"-1","8":"0.095588478","9":"0.202869307","10":"0.188758446","11":"3.882945","12":"3.114346","13":"3.294365","14":"-6.151375e-01","15":"-0.2418247187","16":"-3.518206e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46873"},{"1":"ENSMUSG00000109282","2":"Gm45188","3":"predicted gene 45188","4":"7","5":"98.89593","6":"98.89955","7":"1","8":"0.241812916","9":"0.093440190","10":"0.136274700","11":"3.906145","12":"3.069369","13":"3.257521","14":"-6.383375e-01","15":"-0.1968476844","16":"-3.149764e-01","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"FALSE","24":"NA","_rn_":"46977"},{"1":"ENSMUSG00000108864","2":"Gm44933","3":"predicted gene 44933","4":"7","5":"84.43599","6":"84.43923","7":"-1","8":"0.684163200","9":"0.807356231","10":"0.791630638","11":"3.984804","12":"3.000625","13":"3.164068","14":"-7.169960e-01","15":"-0.1281033035","16":"-2.215237e-01","17":"Gm44933","18":"#CC3333","19":"Gm44933","20":"#CC3333","21":"Gm44933","22":"#CC3333","23":"FALSE","24":"NA","_rn_":"46574"},{"1":"ENSMUSG00000030647","2":"Ndufc2","3":"NADH:ubiquinone oxidoreductase subunit C2","4":"7","5":"97.40000","6":"97.40780","7":"1","8":"0.053563079","9":"0.033343291","10":"0.088484349","11":"4.030980","12":"3.058726","13":"2.885898","14":"-7.631717e-01","15":"-0.1862042713","16":"5.664667e-02","17":"","18":"#666666","19":"","20":"#666666","21":"","22":"#666666","23":"TRUE","24":"NA","_rn_":"7937"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Identifying genes extreme on both mediation analysis and
correlational analysis for <code>RTG_a3</code> (IVSA all traits).</p>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Plotting mediation and SNP correlation variables with ggplot2 and ggrepel</span></span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>chr7_eQTL_cor_med_plot <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> chr7_genes[<span class="fu">which</span>(chr7_genes<span class="sc">$</span>in_exprs_set),], <span class="fu">aes</span>(<span class="at">x =</span> cor_RTG_a3,</span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">y =</span> RTG_a3_lod_drop,</span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">color =</span> color_RTG_a3,</span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">fill =</span> color_RTG_a3,</span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">label =</span> label_all)) <span class="sc">+</span></span>
<span id="cb33-7"><a href="#cb33-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span> </span>
<span id="cb33-8"><a href="#cb33-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>) <span class="sc">+</span></span>
<span id="cb33-9"><a href="#cb33-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="fl">0.1</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>) <span class="sc">+</span></span>
<span id="cb33-10"><a href="#cb33-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="fl">0.3</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>) <span class="sc">+</span></span>
<span id="cb33-11"><a href="#cb33-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">shape =</span> <span class="dv">21</span>, <span class="at">stroke =</span> <span class="fl">0.75</span>) <span class="sc">+</span></span>
<span id="cb33-12"><a href="#cb33-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#999999&quot;</span>,<span class="st">&quot;#000000&quot;</span>)) <span class="sc">+</span></span>
<span id="cb33-13"><a href="#cb33-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#FFFFFF&quot;</span>,<span class="st">&quot;#000000&quot;</span>)) <span class="sc">+</span></span>
<span id="cb33-14"><a href="#cb33-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_size_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>)) <span class="sc">+</span></span>
<span id="cb33-15"><a href="#cb33-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_label_repel</span>(<span class="fu">aes</span>(<span class="at">fill =</span> <span class="cn">NULL</span>)) <span class="sc">+</span></span>
<span id="cb33-16"><a href="#cb33-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;eQTL SNP Correlation with</span><span class="sc">\n</span><span class="st">Cocaine IVSA&quot;</span>) <span class="sc">+</span></span>
<span id="cb33-17"><a href="#cb33-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Cocaine IVSA</span><span class="sc">\n</span><span class="st">LOD Drop with Expression&quot;</span>)</span>
<span id="cb33-18"><a href="#cb33-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-19"><a href="#cb33-19" aria-hidden="true" tabindex="-1"></a><span class="co"># Displaying plot</span></span>
<span id="cb33-20"><a href="#cb33-20" aria-hidden="true" tabindex="-1"></a>chr7_eQTL_cor_med_plot</span></code></pre></div>
<pre><code>Warning: Removed 223 rows containing missing values (`geom_label_repel()`).</code></pre>
<p><img src="figure/17_mapping_interpretation.Rmd/unnamed-chunk-24-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-24-1">
Past versions of unnamed-chunk-24-1.png
</button>
</p>
<div id="fig-unnamed-chunk-24-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2485d7d
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-15
</td>
</tr>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
9b43495
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-19
</td>
</tr>
<tr>
<td>
da1922f
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Identifying genes extreme on both mediation analysis and
correlational analysis for <code>SENSI_cocaine</code>.</p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Plotting mediation and SNP correlation variables with ggplot2 and ggrepel</span></span>
<span id="cb35-2"><a href="#cb35-2" aria-hidden="true" tabindex="-1"></a>chr7_eQTL_cor_med_plot_SENSI_cocaine <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> chr7_genes[<span class="fu">which</span>(chr7_genes<span class="sc">$</span>in_exprs_set),], <span class="fu">aes</span>(<span class="at">x =</span> cor_SENSI_cocaine,</span>
<span id="cb35-3"><a href="#cb35-3" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">y =</span> SENSI_cocaine_lod_drop,</span>
<span id="cb35-4"><a href="#cb35-4" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">color =</span> color_SENSI_cocaine,</span>
<span id="cb35-5"><a href="#cb35-5" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">fill =</span> color_SENSI_cocaine,</span>
<span id="cb35-6"><a href="#cb35-6" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">label =</span> label_all)) <span class="sc">+</span></span>
<span id="cb35-7"><a href="#cb35-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span> </span>
<span id="cb35-8"><a href="#cb35-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>) <span class="sc">+</span></span>
<span id="cb35-9"><a href="#cb35-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="fl">0.1</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>) <span class="sc">+</span></span>
<span id="cb35-10"><a href="#cb35-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="fl">0.3</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>) <span class="sc">+</span></span>
<span id="cb35-11"><a href="#cb35-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">shape =</span> <span class="dv">21</span>, <span class="at">stroke =</span> <span class="fl">0.75</span>) <span class="sc">+</span></span>
<span id="cb35-12"><a href="#cb35-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#999999&quot;</span>,<span class="st">&quot;#000000&quot;</span>)) <span class="sc">+</span></span>
<span id="cb35-13"><a href="#cb35-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#FFFFFF&quot;</span>,<span class="st">&quot;#000000&quot;</span>)) <span class="sc">+</span></span>
<span id="cb35-14"><a href="#cb35-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_size_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>)) <span class="sc">+</span></span>
<span id="cb35-15"><a href="#cb35-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_label_repel</span>(<span class="fu">aes</span>(<span class="at">fill =</span> <span class="cn">NULL</span>)) <span class="sc">+</span></span>
<span id="cb35-16"><a href="#cb35-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;eQTL SNP Correlation with</span><span class="sc">\n</span><span class="st">Cocaine Sensitization&quot;</span>) <span class="sc">+</span></span>
<span id="cb35-17"><a href="#cb35-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Cocaine Sensitization</span><span class="sc">\n</span><span class="st">LOD Drop with Expression&quot;</span>)</span>
<span id="cb35-18"><a href="#cb35-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-19"><a href="#cb35-19" aria-hidden="true" tabindex="-1"></a><span class="co"># Displaying plot</span></span>
<span id="cb35-20"><a href="#cb35-20" aria-hidden="true" tabindex="-1"></a>chr7_eQTL_cor_med_plot_SENSI_cocaine</span></code></pre></div>
<pre><code>Warning: Removed 223 rows containing missing values (`geom_label_repel()`).</code></pre>
<p><img src="figure/17_mapping_interpretation.Rmd/unnamed-chunk-25-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-25-1">
Past versions of unnamed-chunk-25-1.png
</button>
</p>
<div id="fig-unnamed-chunk-25-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
9b43495
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Identifying genes extreme on both mediation analysis and
correlational analysis for <code>SENSI_saline</code>.</p>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Plotting mediation and SNP correlation variables with ggplot2 and ggrepel</span></span>
<span id="cb37-2"><a href="#cb37-2" aria-hidden="true" tabindex="-1"></a>chr7_eQTL_cor_med_plot_SENSI_saline <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> chr7_genes[<span class="fu">which</span>(chr7_genes<span class="sc">$</span>in_exprs_set),], <span class="fu">aes</span>(<span class="at">x =</span> cor_SENSI_saline,</span>
<span id="cb37-3"><a href="#cb37-3" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">y =</span> SENSI_saline_lod_drop,</span>
<span id="cb37-4"><a href="#cb37-4" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">color =</span> color_SENSI_saline,</span>
<span id="cb37-5"><a href="#cb37-5" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">fill =</span> color_SENSI_saline,</span>
<span id="cb37-6"><a href="#cb37-6" aria-hidden="true" tabindex="-1"></a>                                                       <span class="at">label =</span> label_all)) <span class="sc">+</span></span>
<span id="cb37-7"><a href="#cb37-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span> </span>
<span id="cb37-8"><a href="#cb37-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>) <span class="sc">+</span></span>
<span id="cb37-9"><a href="#cb37-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="fl">0.1</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>) <span class="sc">+</span></span>
<span id="cb37-10"><a href="#cb37-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="fl">0.3</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>) <span class="sc">+</span></span>
<span id="cb37-11"><a href="#cb37-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">shape =</span> <span class="dv">21</span>, <span class="at">stroke =</span> <span class="fl">0.75</span>) <span class="sc">+</span></span>
<span id="cb37-12"><a href="#cb37-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#999999&quot;</span>,<span class="st">&quot;#000000&quot;</span>)) <span class="sc">+</span></span>
<span id="cb37-13"><a href="#cb37-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#FFFFFF&quot;</span>,<span class="st">&quot;#000000&quot;</span>)) <span class="sc">+</span></span>
<span id="cb37-14"><a href="#cb37-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_size_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>)) <span class="sc">+</span></span>
<span id="cb37-15"><a href="#cb37-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_label_repel</span>(<span class="fu">aes</span>(<span class="at">fill =</span> <span class="cn">NULL</span>)) <span class="sc">+</span></span>
<span id="cb37-16"><a href="#cb37-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;eQTL SNP Correlation with</span><span class="sc">\n</span><span class="st">Sham/Control Locomotor Activity&quot;</span>) <span class="sc">+</span></span>
<span id="cb37-17"><a href="#cb37-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Sham/Control Locomotor Activity</span><span class="sc">\n</span><span class="st">LOD Drop with Expression&quot;</span>)</span>
<span id="cb37-18"><a href="#cb37-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-19"><a href="#cb37-19" aria-hidden="true" tabindex="-1"></a><span class="co"># Displaying plot</span></span>
<span id="cb37-20"><a href="#cb37-20" aria-hidden="true" tabindex="-1"></a>chr7_eQTL_cor_med_plot_SENSI_saline</span></code></pre></div>
<pre><code>Warning: Removed 223 rows containing missing values (`geom_label_repel()`).</code></pre>
<p><img src="figure/17_mapping_interpretation.Rmd/unnamed-chunk-26-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-26-1">
Past versions of unnamed-chunk-26-1.png
</button>
</p>
<div id="fig-unnamed-chunk-26-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
9b43495
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="st">&quot;./output/mediation_all_rta_figure.pdf&quot;</span>, <span class="at">plot =</span> <span class="fu">plot_grid</span>(chr7_eQTL_cor_med_plot <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">&quot;Cocaine IVSA&quot;</span>), chr7_eQTL_cor_med_plot_SENSI_cocaine <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">&quot;Cocaine Sensitization&quot;</span>), chr7_eQTL_cor_med_plot_SENSI_saline <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">&quot;Sham/Control Locomotor Activity&quot;</span>), <span class="at">ncol =</span> <span class="dv">3</span>), <span class="at">width =</span> <span class="dv">12</span>, <span class="at">height =</span> <span class="dv">4</span>, <span class="at">units =</span> <span class="st">&quot;in&quot;</span>)</span></code></pre></div>
<pre><code>Warning: Removed 223 rows containing missing values (`geom_label_repel()`).
Removed 223 rows containing missing values (`geom_label_repel()`).
Removed 223 rows containing missing values (`geom_label_repel()`).</code></pre>
<p>These data show that the strongest correlation and largest LOD drops
at the peak on 4 are with the genes <em>Me3</em>, <em>Sytl2</em>, and
<em>Arnt2</em> as well-annotated genes that meet criteria for potential
causal genes for this QTL. Plotting their respective eQTL effects on
chr7.</p>
<p>Starting with <em>Me3</em>.</p>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb41-1"><a href="#cb41-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot_BLUP_str_2016</span>(<span class="st">&quot;ENSMUSG00000030621&quot;</span>,<span class="st">&quot;7&quot;</span>,<span class="at">label=</span><span class="st">&quot;Me3&quot;</span>,<span class="at">legendpos =</span> <span class="st">&quot;bottomleft&quot;</span>)</span></code></pre></div>
<p><img src="figure/17_mapping_interpretation.Rmd/unnamed-chunk-28-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-28-1">
Past versions of unnamed-chunk-28-1.png
</button>
</p>
<div id="fig-unnamed-chunk-28-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2485d7d
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-15
</td>
</tr>
<tr>
<td>
f9c5856
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-16
</td>
</tr>
<tr>
<td>
bf1ce2e
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Now plotting <em>Arnt2</em>.</p>
<div class="sourceCode" id="cb42"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb42-1"><a href="#cb42-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot_BLUP_str_2016</span>(<span class="st">&quot;ENSMUSG00000015709&quot;</span>,<span class="st">&quot;7&quot;</span>,<span class="at">label=</span><span class="st">&quot;Arnt2&quot;</span>,<span class="at">legendpos =</span> <span class="st">&quot;bottomleft&quot;</span>)</span></code></pre></div>
<p><img src="figure/17_mapping_interpretation.Rmd/unnamed-chunk-29-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-29-1">
Past versions of unnamed-chunk-29-1.png
</button>
</p>
<div id="fig-unnamed-chunk-29-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2485d7d
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-15
</td>
</tr>
<tr>
<td>
da1922f
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-17
</td>
</tr>
<tr>
<td>
f9c5856
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-16
</td>
</tr>
<tr>
<td>
bf1ce2e
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Now plotting <em>Sytl2</em>.</p>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot_BLUP_str_2016</span>(<span class="st">&quot;ENSMUSG00000030616&quot;</span>,<span class="st">&quot;7&quot;</span>,<span class="at">label=</span><span class="st">&quot;Sytl2&quot;</span>,<span class="at">legendpos =</span> <span class="st">&quot;bottomleft&quot;</span>)</span></code></pre></div>
<p><img src="figure/17_mapping_interpretation.Rmd/unnamed-chunk-30-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-30-1">
Past versions of unnamed-chunk-30-1.png
</button>
</p>
<div id="fig-unnamed-chunk-30-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
aa52960
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-12
</td>
</tr>
<tr>
<td>
2485d7d
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-15
</td>
</tr>
<tr>
<td>
9b43495
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-19
</td>
</tr>
<tr>
<td>
da1922f
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-17
</td>
</tr>
<tr>
<td>
f9c5856
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-16
</td>
</tr>
<tr>
<td>
bf1ce2e
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Running 100 randomizations on positions other than the chr7 allele to
see how rare the mediation result is.</p>
<div class="sourceCode" id="cb44"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb44-1"><a href="#cb44-1" aria-hidden="true" tabindex="-1"></a>calculated_positions <span class="ot">=</span> Mm_maRt <span class="sc">%&gt;%</span> </span>
<span id="cb44-2"><a href="#cb44-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(chromosome_name) <span class="sc">%&gt;%</span> </span>
<span id="cb44-3"><a href="#cb44-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarize</span>(<span class="at">min_start =</span> <span class="fu">min</span>(start_position), </span>
<span id="cb44-4"><a href="#cb44-4" aria-hidden="true" tabindex="-1"></a>            <span class="at">min_end =</span> <span class="fu">min</span>(end_position), </span>
<span id="cb44-5"><a href="#cb44-5" aria-hidden="true" tabindex="-1"></a>            <span class="at">max_start =</span> <span class="fu">max</span>(start_position), </span>
<span id="cb44-6"><a href="#cb44-6" aria-hidden="true" tabindex="-1"></a>            <span class="at">max_end =</span> <span class="fu">max</span>(end_position)) <span class="sc">%&gt;%</span> </span>
<span id="cb44-7"><a href="#cb44-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(chromosome_name) <span class="sc">%&gt;%</span> </span>
<span id="cb44-8"><a href="#cb44-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">min_pos =</span> <span class="fu">min</span>(<span class="fu">c</span>(min_start, min_end)), </span>
<span id="cb44-9"><a href="#cb44-9" aria-hidden="true" tabindex="-1"></a>         <span class="at">max_pos =</span> <span class="fu">max</span>(<span class="fu">c</span>(max_start, max_end)),</span>
<span id="cb44-10"><a href="#cb44-10" aria-hidden="true" tabindex="-1"></a>         <span class="at">min_plus_5 =</span> <span class="fu">min</span>(<span class="fu">c</span>(min_start, min_end)) <span class="sc">+</span> <span class="dv">5</span>,</span>
<span id="cb44-11"><a href="#cb44-11" aria-hidden="true" tabindex="-1"></a>         <span class="at">max_minus_5 =</span> <span class="fu">max</span>(<span class="fu">c</span>(max_start, max_end)) <span class="sc">-</span> <span class="dv">5</span>) <span class="sc">%&gt;%</span></span>
<span id="cb44-12"><a href="#cb44-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">length =</span> max_minus_5 <span class="sc">-</span> min_plus_5)</span>
<span id="cb44-13"><a href="#cb44-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb44-14"><a href="#cb44-14" aria-hidden="true" tabindex="-1"></a>calculated_positions <span class="ot">=</span> <span class="fu">as.data.frame</span>(calculated_positions)</span>
<span id="cb44-15"><a href="#cb44-15" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(calculated_positions) <span class="ot">=</span> calculated_positions<span class="sc">$</span>chromosome_name</span>
<span id="cb44-16"><a href="#cb44-16" aria-hidden="true" tabindex="-1"></a>calculated_positions <span class="ot">=</span> calculated_positions[<span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">19</span>,<span class="st">&quot;X&quot;</span>),]</span>
<span id="cb44-17"><a href="#cb44-17" aria-hidden="true" tabindex="-1"></a>calculated_positions<span class="sc">$</span>cumulative_length <span class="ot">=</span> <span class="fu">cumsum</span>(calculated_positions<span class="sc">$</span>length)</span>
<span id="cb44-18"><a href="#cb44-18" aria-hidden="true" tabindex="-1"></a>calculated_positions<span class="sc">$</span>min_length <span class="ot">=</span> calculated_positions<span class="sc">$</span>cumulative_length <span class="sc">-</span> calculated_positions<span class="sc">$</span>length</span>
<span id="cb44-19"><a href="#cb44-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb44-20"><a href="#cb44-20" aria-hidden="true" tabindex="-1"></a>Me3_min <span class="ot">=</span> (chr7_genes[<span class="fu">which</span>(chr7_genes<span class="sc">$</span>mgi_symbol <span class="sc">==</span> <span class="st">&quot;Me3&quot;</span>),<span class="st">&quot;start_position&quot;</span>] <span class="sc">-</span> <span class="dv">5</span>) <span class="sc">+</span> calculated_positions[<span class="fu">which</span>(calculated_positions<span class="sc">$</span>chromosome_name <span class="sc">==</span> <span class="st">&quot;7&quot;</span>),<span class="st">&quot;min_length&quot;</span>]</span>
<span id="cb44-21"><a href="#cb44-21" aria-hidden="true" tabindex="-1"></a>Me3_max <span class="ot">=</span> (chr7_genes[<span class="fu">which</span>(chr7_genes<span class="sc">$</span>mgi_symbol <span class="sc">==</span> <span class="st">&quot;Me3&quot;</span>),<span class="st">&quot;end_position&quot;</span>] <span class="sc">+</span> <span class="dv">5</span>) <span class="sc">+</span> calculated_positions[<span class="fu">which</span>(calculated_positions<span class="sc">$</span>chromosome_name <span class="sc">==</span> <span class="st">&quot;7&quot;</span>),<span class="st">&quot;min_length&quot;</span>]</span>
<span id="cb44-22"><a href="#cb44-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb44-23"><a href="#cb44-23" aria-hidden="true" tabindex="-1"></a>max_pos <span class="ot">=</span> <span class="fu">max</span>(calculated_positions<span class="sc">$</span>cumulative_length)</span>
<span id="cb44-24"><a href="#cb44-24" aria-hidden="true" tabindex="-1"></a>random_peaks <span class="ot">=</span> <span class="fu">c</span>()</span>
<span id="cb44-25"><a href="#cb44-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb44-26"><a href="#cb44-26" aria-hidden="true" tabindex="-1"></a><span class="co"># Making random peaks excluding the Me3 region</span></span>
<span id="cb44-27"><a href="#cb44-27" aria-hidden="true" tabindex="-1"></a><span class="cf">while</span> (<span class="fu">length</span>(random_peaks) <span class="sc">&lt;</span> <span class="dv">100</span>) {</span>
<span id="cb44-28"><a href="#cb44-28" aria-hidden="true" tabindex="-1"></a>  candidate_peak <span class="ot">=</span> <span class="fu">runif</span>(<span class="dv">1</span>, <span class="at">min =</span> <span class="dv">0</span>, <span class="at">max =</span> max_pos)</span>
<span id="cb44-29"><a href="#cb44-29" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (candidate_peak <span class="sc">&gt;=</span> Me3_min <span class="sc">&amp;</span> candidate_peak <span class="sc">&lt;=</span> Me3_max) {</span>
<span id="cb44-30"><a href="#cb44-30" aria-hidden="true" tabindex="-1"></a>    <span class="cf">next</span></span>
<span id="cb44-31"><a href="#cb44-31" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb44-32"><a href="#cb44-32" aria-hidden="true" tabindex="-1"></a>    random_peaks <span class="ot">=</span> <span class="fu">c</span>(random_peaks, candidate_peak)</span>
<span id="cb44-33"><a href="#cb44-33" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb44-34"><a href="#cb44-34" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb44-35"><a href="#cb44-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb44-36"><a href="#cb44-36" aria-hidden="true" tabindex="-1"></a>random_peaks <span class="ot">=</span> random_peaks[<span class="fu">order</span>(random_peaks)]</span>
<span id="cb44-37"><a href="#cb44-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb44-38"><a href="#cb44-38" aria-hidden="true" tabindex="-1"></a>min_lod_drop <span class="ot">=</span> <span class="fu">c</span>()</span>
<span id="cb44-39"><a href="#cb44-39" aria-hidden="true" tabindex="-1"></a>max_lod_drop <span class="ot">=</span> <span class="fu">c</span>()</span>
<span id="cb44-40"><a href="#cb44-40" aria-hidden="true" tabindex="-1"></a>lods_drop <span class="ot">=</span> <span class="fu">list</span>()</span>
<span id="cb44-41"><a href="#cb44-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb44-42"><a href="#cb44-42" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> random_peaks) {</span>
<span id="cb44-43"><a href="#cb44-43" aria-hidden="true" tabindex="-1"></a>  chr_i <span class="ot">=</span> calculated_positions[<span class="fu">which</span>(calculated_positions<span class="sc">$</span>min_length <span class="sc">&lt;</span> i <span class="sc">&amp;</span> calculated_positions<span class="sc">$</span>cumulative_length <span class="sc">&gt;=</span> i),<span class="st">&quot;chromosome_name&quot;</span>]</span>
<span id="cb44-44"><a href="#cb44-44" aria-hidden="true" tabindex="-1"></a>  scale_i <span class="ot">=</span> calculated_positions[<span class="fu">which</span>(calculated_positions<span class="sc">$</span>min_length <span class="sc">&lt;</span> i <span class="sc">&amp;</span> calculated_positions<span class="sc">$</span>cumulative_length <span class="sc">&gt;=</span> i),<span class="st">&quot;min_length&quot;</span>]</span>
<span id="cb44-45"><a href="#cb44-45" aria-hidden="true" tabindex="-1"></a>  min_chr_i <span class="ot">=</span> calculated_positions[<span class="fu">which</span>(calculated_positions<span class="sc">$</span>min_length <span class="sc">&lt;</span> i <span class="sc">&amp;</span> calculated_positions<span class="sc">$</span>cumulative_length <span class="sc">&gt;=</span> i),<span class="st">&quot;min_plus_5&quot;</span>]</span>
<span id="cb44-46"><a href="#cb44-46" aria-hidden="true" tabindex="-1"></a>  peak_i <span class="ot">=</span> (i <span class="sc">-</span> scale_i) <span class="sc">+</span> min_chr_i</span>
<span id="cb44-47"><a href="#cb44-47" aria-hidden="true" tabindex="-1"></a>  genes_i <span class="ot">=</span> Mm_maRt[<span class="fu">which</span>(Mm_maRt<span class="sc">$</span>chromosome_name <span class="sc">==</span> chr_i <span class="sc">&amp;</span></span>
<span id="cb44-48"><a href="#cb44-48" aria-hidden="true" tabindex="-1"></a>                            (Mm_maRt<span class="sc">$</span>start_position <span class="sc">&gt;</span> (peak_i <span class="sc">-</span> <span class="dv">5</span>) <span class="sc">|</span> Mm_maRt<span class="sc">$</span>end_position <span class="sc">&gt;</span> (peak_i <span class="sc">-</span> <span class="dv">5</span>)) <span class="sc">&amp;</span></span>
<span id="cb44-49"><a href="#cb44-49" aria-hidden="true" tabindex="-1"></a>                           (Mm_maRt<span class="sc">$</span>start_position <span class="sc">&lt;</span> (peak_i <span class="sc">+</span> <span class="dv">5</span>) <span class="sc">|</span> Mm_maRt<span class="sc">$</span>end_position <span class="sc">&lt;</span> (peak_i <span class="sc">+</span> <span class="dv">5</span>))),]</span>
<span id="cb44-50"><a href="#cb44-50" aria-hidden="true" tabindex="-1"></a>  lods_i <span class="ot">=</span> <span class="fu">c</span>()</span>
<span id="cb44-51"><a href="#cb44-51" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (j <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(genes_i)) {</span>
<span id="cb44-52"><a href="#cb44-52" aria-hidden="true" tabindex="-1"></a>    sig_gene_j <span class="ot">=</span> genes_i[j,<span class="st">&quot;ensembl_gene_id&quot;</span>]</span>
<span id="cb44-53"><a href="#cb44-53" aria-hidden="true" tabindex="-1"></a>    sig_label_j <span class="ot">=</span> genes_i[j,<span class="st">&quot;mgi_symbol&quot;</span>]</span>
<span id="cb44-54"><a href="#cb44-54" aria-hidden="true" tabindex="-1"></a>    pheno_col_j <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">colnames</span>(DO_pheno) <span class="sc">==</span> sig_gene_i)</span>
<span id="cb44-55"><a href="#cb44-55" aria-hidden="true" tabindex="-1"></a>    RTG_a3_scan_j <span class="ot">=</span> <span class="fu">scan1</span>(<span class="at">genoprobs =</span> DO_str_2016_aprobs[,<span class="st">&quot;7&quot;</span>],</span>
<span id="cb44-56"><a href="#cb44-56" aria-hidden="true" tabindex="-1"></a>                          <span class="at">pheno =</span> RTG_a3_phenotype_mat[,<span class="st">&quot;RTG_a3&quot;</span>],</span>
<span id="cb44-57"><a href="#cb44-57" aria-hidden="true" tabindex="-1"></a>                          <span class="at">kinship =</span> DO_str_2016_kinship<span class="sc">$</span><span class="st">`</span><span class="at">7</span><span class="st">`</span>,</span>
<span id="cb44-58"><a href="#cb44-58" aria-hidden="true" tabindex="-1"></a>                          <span class="at">addcovar =</span> <span class="fu">cbind</span>(addcovar_i,</span>
<span id="cb44-59"><a href="#cb44-59" aria-hidden="true" tabindex="-1"></a>                                           DO_pheno[,sig_gene_j]),</span>
<span id="cb44-60"><a href="#cb44-60" aria-hidden="true" tabindex="-1"></a>                          <span class="at">cores =</span> <span class="dv">9</span>)</span>
<span id="cb44-61"><a href="#cb44-61" aria-hidden="true" tabindex="-1"></a>    max_marker_lod_j <span class="ot">=</span> <span class="fu">as.data.frame</span>(RTG_a3_scan_j)[max_marker_RTG_a3,<span class="dv">1</span>]</span>
<span id="cb44-62"><a href="#cb44-62" aria-hidden="true" tabindex="-1"></a>    lods_i <span class="ot">=</span> <span class="fu">c</span>(lods_i, max_marker_lod_j)</span>
<span id="cb44-63"><a href="#cb44-63" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb44-64"><a href="#cb44-64" aria-hidden="true" tabindex="-1"></a>  lods_drop_i <span class="ot">=</span> max_marker_lod_RTG_a3 <span class="sc">-</span> lods_i</span>
<span id="cb44-65"><a href="#cb44-65" aria-hidden="true" tabindex="-1"></a>  lods_drop[[<span class="fu">length</span>(lods_drop) <span class="sc">+</span> <span class="dv">1</span>]] <span class="ot">=</span> lods_drop_i</span>
<span id="cb44-66"><a href="#cb44-66" aria-hidden="true" tabindex="-1"></a>  min_lod_drop <span class="ot">=</span> <span class="fu">c</span>(min_lod_drop, <span class="fu">min</span>(lods_drop_i))</span>
<span id="cb44-67"><a href="#cb44-67" aria-hidden="true" tabindex="-1"></a>  max_lod_drop <span class="ot">=</span> <span class="fu">c</span>(max_lod_drop, <span class="fu">max</span>(lods_drop_i))</span>
<span id="cb44-68"><a href="#cb44-68" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
</div>
</div>
<div id="output" class="section level3">
<h3>Output</h3>
<p>The output data are saved in <code>./data/</code>.</p>
<div class="sourceCode" id="cb45"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(chr7_genes,<span class="st">&quot;./data/chr7_a3_QTL_results.RDS&quot;</span>)</span></code></pre></div>
</div>
</div>
<div id="discussion" class="section level1">
<h1>Discussion</h1>
<div id="qtl-figures" class="section level2">
<h2>QTL figures</h2>
<p>Plotting publication-quality figures for the QTL analysis. Starting
with the <code>SENS</code> QTL plot.</p>
<div class="sourceCode" id="cb46"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting QTL thresholds</span></span>
<span id="cb46-2"><a href="#cb46-2" aria-hidden="true" tabindex="-1"></a>SENS_COC_suggestive <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;37%&quot;</span>]</span>
<span id="cb46-3"><a href="#cb46-3" aria-hidden="true" tabindex="-1"></a>SENS_COC_ten_percent <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;90%&quot;</span>]</span>
<span id="cb46-4"><a href="#cb46-4" aria-hidden="true" tabindex="-1"></a>SENS_COC_five_percent <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;95%&quot;</span>]</span>
<span id="cb46-5"><a href="#cb46-5" aria-hidden="true" tabindex="-1"></a>SENS_COC_one_percent <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;99%&quot;</span>]</span>
<span id="cb46-6"><a href="#cb46-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-7"><a href="#cb46-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting information to plot</span></span>
<span id="cb46-8"><a href="#cb46-8" aria-hidden="true" tabindex="-1"></a>gap_Mb <span class="ot">=</span> <span class="dv">20</span></span>
<span id="cb46-9"><a href="#cb46-9" aria-hidden="true" tabindex="-1"></a>min_chr_i <span class="ot">=</span> <span class="dv">0</span></span>
<span id="cb46-10"><a href="#cb46-10" aria-hidden="true" tabindex="-1"></a>chrs <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">19</span>,<span class="st">&quot;X&quot;</span>))</span>
<span id="cb46-11"><a href="#cb46-11" aria-hidden="true" tabindex="-1"></a>SENS_COC_df <span class="ot">=</span> <span class="fu">as.data.frame</span>(SENS_COC_qtl)</span>
<span id="cb46-12"><a href="#cb46-12" aria-hidden="true" tabindex="-1"></a>SENS_COC_df<span class="sc">$</span>marker <span class="ot">=</span> <span class="fu">row.names</span>(SENS_COC_df)</span>
<span id="cb46-13"><a href="#cb46-13" aria-hidden="true" tabindex="-1"></a>SENS_COC_df<span class="sc">$</span>chr <span class="ot">=</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(SENS_COC_df))</span>
<span id="cb46-14"><a href="#cb46-14" aria-hidden="true" tabindex="-1"></a>SENS_COC_df<span class="sc">$</span>stagger <span class="ot">=</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(SENS_COC_df))</span>
<span id="cb46-15"><a href="#cb46-15" aria-hidden="true" tabindex="-1"></a>SENS_COC_df<span class="sc">$</span>pos <span class="ot">=</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">nrow</span>(SENS_COC_df))</span>
<span id="cb46-16"><a href="#cb46-16" aria-hidden="true" tabindex="-1"></a>SENS_COC_df<span class="sc">$</span>calc_pos <span class="ot">=</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">nrow</span>(SENS_COC_df))</span>
<span id="cb46-17"><a href="#cb46-17" aria-hidden="true" tabindex="-1"></a>SENS_COC_df<span class="sc">$</span>label <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(SENS_COC_df))</span>
<span id="cb46-18"><a href="#cb46-18" aria-hidden="true" tabindex="-1"></a>SENS_COC_min_max_mid <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">row.names =</span> chrs,</span>
<span id="cb46-19"><a href="#cb46-19" aria-hidden="true" tabindex="-1"></a>                                <span class="at">chr =</span> chrs,</span>
<span id="cb46-20"><a href="#cb46-20" aria-hidden="true" tabindex="-1"></a>                                <span class="at">max =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)),</span>
<span id="cb46-21"><a href="#cb46-21" aria-hidden="true" tabindex="-1"></a>                                <span class="at">calc_min =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)),</span>
<span id="cb46-22"><a href="#cb46-22" aria-hidden="true" tabindex="-1"></a>                                <span class="at">calc_mid =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)),</span>
<span id="cb46-23"><a href="#cb46-23" aria-hidden="true" tabindex="-1"></a>                                <span class="at">calc_max =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)))</span>
<span id="cb46-24"><a href="#cb46-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-25"><a href="#cb46-25" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting data frame for plotting RTG a1 in ggplot2</span></span>
<span id="cb46-26"><a href="#cb46-26" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(chrs)) {</span>
<span id="cb46-27"><a href="#cb46-27" aria-hidden="true" tabindex="-1"></a>  chr_i <span class="ot">=</span> chrs[i]</span>
<span id="cb46-28"><a href="#cb46-28" aria-hidden="true" tabindex="-1"></a>  pmap_i <span class="ot">=</span> pmap[[chr_i]]</span>
<span id="cb46-29"><a href="#cb46-29" aria-hidden="true" tabindex="-1"></a>  SENS_COC_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;chr&quot;</span>] <span class="ot">=</span> chr_i</span>
<span id="cb46-30"><a href="#cb46-30" aria-hidden="true" tabindex="-1"></a>  SENS_COC_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;stagger&quot;</span>] <span class="ot">=</span> <span class="fu">ifelse</span>(i <span class="sc">%%</span> <span class="dv">2</span> <span class="sc">==</span> <span class="dv">0</span>, <span class="st">&quot;darkblue&quot;</span>, <span class="st">&quot;lightblue&quot;</span>)</span>
<span id="cb46-31"><a href="#cb46-31" aria-hidden="true" tabindex="-1"></a>  SENS_COC_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;pos&quot;</span>] <span class="ot">=</span> pmap_i</span>
<span id="cb46-32"><a href="#cb46-32" aria-hidden="true" tabindex="-1"></a>  SENS_COC_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;calc_pos&quot;</span>] <span class="ot">=</span> min_chr_i <span class="sc">+</span> pmap_i</span>
<span id="cb46-33"><a href="#cb46-33" aria-hidden="true" tabindex="-1"></a>  max_lod_chr_i <span class="ot">=</span> <span class="fu">max</span>(SENS_COC_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;pheno1&quot;</span>])</span>
<span id="cb46-34"><a href="#cb46-34" aria-hidden="true" tabindex="-1"></a>  max_marker_chr_i <span class="ot">=</span> SENS_COC_df[<span class="fu">which</span>(SENS_COC_df<span class="sc">$</span>chr <span class="sc">==</span> chr_i <span class="sc">&amp;</span> SENS_COC_df<span class="sc">$</span>pheno1 <span class="sc">==</span> max_lod_chr_i),<span class="st">&quot;marker&quot;</span>]</span>
<span id="cb46-35"><a href="#cb46-35" aria-hidden="true" tabindex="-1"></a>  SENS_COC_df[max_marker_chr_i,<span class="st">&quot;label&quot;</span>] <span class="ot">=</span> <span class="fu">ifelse</span>(max_lod_chr_i <span class="sc">&gt;</span> SENS_COC_five_percent,</span>
<span id="cb46-36"><a href="#cb46-36" aria-hidden="true" tabindex="-1"></a>                                               <span class="fu">paste</span>(max_marker_chr_i, <span class="st">&quot; at chr&quot;</span>, chr_i,</span>
<span id="cb46-37"><a href="#cb46-37" aria-hidden="true" tabindex="-1"></a>                                                     <span class="st">&quot;:&quot;</span>,<span class="fu">round</span>(SENS_COC_df[max_marker_chr_i,<span class="st">&quot;pos&quot;</span>], <span class="dv">2</span>),</span>
<span id="cb46-38"><a href="#cb46-38" aria-hidden="true" tabindex="-1"></a>                                                     <span class="st">&quot; Mb (LOD = &quot;</span>,<span class="fu">round</span>(max_lod_chr_i, <span class="dv">2</span>),<span class="st">&quot;)&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>),</span>
<span id="cb46-39"><a href="#cb46-39" aria-hidden="true" tabindex="-1"></a>                                               <span class="st">&quot;&quot;</span>)</span>
<span id="cb46-40"><a href="#cb46-40" aria-hidden="true" tabindex="-1"></a>  max_pos_i <span class="ot">=</span> <span class="fu">max</span>(pmap_i)</span>
<span id="cb46-41"><a href="#cb46-41" aria-hidden="true" tabindex="-1"></a>  SENS_COC_min_max_mid[chr_i,<span class="st">&quot;calc_min&quot;</span>] <span class="ot">=</span> min_chr_i</span>
<span id="cb46-42"><a href="#cb46-42" aria-hidden="true" tabindex="-1"></a>  SENS_COC_min_max_mid[chr_i,<span class="st">&quot;calc_max&quot;</span>] <span class="ot">=</span> min_chr_i <span class="sc">+</span> max_pos_i</span>
<span id="cb46-43"><a href="#cb46-43" aria-hidden="true" tabindex="-1"></a>  SENS_COC_min_max_mid[chr_i,<span class="st">&quot;max&quot;</span>] <span class="ot">=</span> max_pos_i</span>
<span id="cb46-44"><a href="#cb46-44" aria-hidden="true" tabindex="-1"></a>  min_chr_i <span class="ot">=</span> min_chr_i <span class="sc">+</span> max_pos_i <span class="sc">+</span> gap_Mb</span>
<span id="cb46-45"><a href="#cb46-45" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb46-46"><a href="#cb46-46" aria-hidden="true" tabindex="-1"></a>SENS_COC_min_max_mid<span class="sc">$</span>calc_mid <span class="ot">=</span> SENS_COC_min_max_mid<span class="sc">$</span>calc_min <span class="sc">+</span> (<span class="fl">0.5</span> <span class="sc">*</span> (SENS_COC_min_max_mid<span class="sc">$</span>calc_max <span class="sc">-</span> SENS_COC_min_max_mid<span class="sc">$</span>calc_min))</span>
<span id="cb46-47"><a href="#cb46-47" aria-hidden="true" tabindex="-1"></a>SENS_COC_df<span class="sc">$</span>chr <span class="ot">=</span> <span class="fu">factor</span>(SENS_COC_df<span class="sc">$</span>chr, <span class="at">levels =</span> chrs, <span class="at">ordered =</span> <span class="cn">TRUE</span>)</span>
<span id="cb46-48"><a href="#cb46-48" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-49"><a href="#cb46-49" aria-hidden="true" tabindex="-1"></a><span class="co"># Plotting RTG a1</span></span>
<span id="cb46-50"><a href="#cb46-50" aria-hidden="true" tabindex="-1"></a>SENS_COC_mapping_plot <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> SENS_COC_df, <span class="fu">aes</span>(<span class="at">x =</span> calc_pos,</span>
<span id="cb46-51"><a href="#cb46-51" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">y =</span> pheno1,</span>
<span id="cb46-52"><a href="#cb46-52" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">color =</span> stagger,</span>
<span id="cb46-53"><a href="#cb46-53" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">group =</span> chr,</span>
<span id="cb46-54"><a href="#cb46-54" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">label =</span> label)) <span class="sc">+</span></span>
<span id="cb46-55"><a href="#cb46-55" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb46-56"><a href="#cb46-56" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>, <span class="at">panel.grid =</span> <span class="fu">element_line</span>(<span class="at">color =</span> <span class="st">&quot;#FFFFFF&quot;</span>)) <span class="sc">+</span></span>
<span id="cb46-57"><a href="#cb46-57" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">size =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb46-58"><a href="#cb46-58" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">labels =</span> SENS_COC_min_max_mid<span class="sc">$</span>chr,</span>
<span id="cb46-59"><a href="#cb46-59" aria-hidden="true" tabindex="-1"></a>                     <span class="at">breaks =</span> SENS_COC_min_max_mid<span class="sc">$</span>calc_mid,</span>
<span id="cb46-60"><a href="#cb46-60" aria-hidden="true" tabindex="-1"></a>                     <span class="at">minor_breaks =</span> <span class="fu">c</span>(SENS_COC_min_max_mid<span class="sc">$</span>calc_min, SENS_COC_min_max_mid<span class="sc">$</span>calc_max)) <span class="sc">+</span></span>
<span id="cb46-61"><a href="#cb46-61" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#002D72&quot;</span>,<span class="st">&quot;#0085CA&quot;</span>)) <span class="sc">+</span></span>
<span id="cb46-62"><a href="#cb46-62" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="cn">NULL</span>) <span class="sc">+</span></span>
<span id="cb46-63"><a href="#cb46-63" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;LOD Score&quot;</span>) <span class="sc">+</span></span>
<span id="cb46-64"><a href="#cb46-64" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> SENS_COC_five_percent, <span class="at">color =</span> <span class="st">&quot;#AB2328&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb46-65"><a href="#cb46-65" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> SENS_COC_ten_percent, <span class="at">color =</span> <span class="st">&quot;#ED8B00&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb46-66"><a href="#cb46-66" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> SENS_COC_suggestive, <span class="at">color =</span> <span class="st">&quot;#F1C400&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb46-67"><a href="#cb46-67" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">label =</span> <span class="fu">expression</span>(<span class="fu">paste</span>(alpha, <span class="st">&quot; = 0.05&quot;</span>)), <span class="at">x =</span> <span class="dv">80</span>, <span class="at">y =</span> SENS_COC_five_percent <span class="sc">+</span> <span class="fl">0.25</span>, <span class="at">color =</span> <span class="st">&quot;#AB2328&quot;</span>) <span class="sc">+</span></span>
<span id="cb46-68"><a href="#cb46-68" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">label =</span> <span class="fu">expression</span>(<span class="fu">paste</span>(alpha, <span class="st">&quot; = 0.10&quot;</span>)), <span class="at">x =</span> <span class="dv">80</span>, <span class="at">y =</span> SENS_COC_ten_percent <span class="sc">+</span> <span class="fl">0.25</span>, <span class="at">color =</span> <span class="st">&quot;#ED8B00&quot;</span>) <span class="sc">+</span></span>
<span id="cb46-69"><a href="#cb46-69" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">label =</span> <span class="st">&quot;Suggestive&quot;</span>, <span class="at">x =</span> <span class="dv">80</span>, <span class="at">y =</span> SENS_COC_suggestive <span class="sc">+</span> <span class="fl">0.25</span>, <span class="at">color =</span> <span class="st">&quot;#F1C400&quot;</span>) <span class="sc">+</span></span>
<span id="cb46-70"><a href="#cb46-70" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_label_repel</span>() <span class="sc">+</span></span>
<span id="cb46-71"><a href="#cb46-71" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">&quot;RTG Analysis 1: Mapping Results&quot;</span>)</span></code></pre></div>
<pre><code>Warning: Using `size` aesthetic for lines was deprecated in ggplot2 3.4.0.
ℹ Please use `linewidth` instead.
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
generated.</code></pre>
<div class="sourceCode" id="cb48"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb48-1"><a href="#cb48-1" aria-hidden="true" tabindex="-1"></a>SENS_COC_mapping_plot</span></code></pre></div>
<pre><code>Warning in is.na(x): is.na() applied to non-(list or vector) of type
&#39;expression&#39;</code></pre>
<pre><code>Warning in is.na(x): is.na() applied to non-(list or vector) of type
&#39;expression&#39;</code></pre>
<pre><code>Warning: ggrepel: 2 unlabeled data points (too many overlaps). Consider
increasing max.overlaps</code></pre>
<p><img src="figure/17_mapping_interpretation.Rmd/unnamed-chunk-33-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-33-1">
Past versions of unnamed-chunk-33-1.png
</button>
</p>
<div id="fig-unnamed-chunk-33-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
9b43495
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-19
</td>
</tr>
<tr>
<td>
da1922f
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Plotting <code>RTG_a2</code> QTL plot</p>
<div class="sourceCode" id="cb52"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb52-1"><a href="#cb52-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting QTL thresholds</span></span>
<span id="cb52-2"><a href="#cb52-2" aria-hidden="true" tabindex="-1"></a>RTG_a2_suggestive <span class="ot">=</span> RTG_a2_permu_thresholds[<span class="st">&quot;37%&quot;</span>]</span>
<span id="cb52-3"><a href="#cb52-3" aria-hidden="true" tabindex="-1"></a>RTG_a2_ten_percent <span class="ot">=</span> RTG_a2_permu_thresholds[<span class="st">&quot;90%&quot;</span>]</span>
<span id="cb52-4"><a href="#cb52-4" aria-hidden="true" tabindex="-1"></a>RTG_a2_five_percent <span class="ot">=</span> RTG_a2_permu_thresholds[<span class="st">&quot;95%&quot;</span>]</span>
<span id="cb52-5"><a href="#cb52-5" aria-hidden="true" tabindex="-1"></a>RTG_a2_one_percent <span class="ot">=</span> RTG_a2_permu_thresholds[<span class="st">&quot;99%&quot;</span>]</span>
<span id="cb52-6"><a href="#cb52-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb52-7"><a href="#cb52-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting information to plot</span></span>
<span id="cb52-8"><a href="#cb52-8" aria-hidden="true" tabindex="-1"></a>gap_Mb <span class="ot">=</span> <span class="dv">20</span></span>
<span id="cb52-9"><a href="#cb52-9" aria-hidden="true" tabindex="-1"></a>min_chr_i <span class="ot">=</span> <span class="dv">0</span></span>
<span id="cb52-10"><a href="#cb52-10" aria-hidden="true" tabindex="-1"></a>chrs <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">19</span>,<span class="st">&quot;X&quot;</span>))</span>
<span id="cb52-11"><a href="#cb52-11" aria-hidden="true" tabindex="-1"></a>RTG_a2_df <span class="ot">=</span> <span class="fu">as.data.frame</span>(RTG_a2_qtl)</span>
<span id="cb52-12"><a href="#cb52-12" aria-hidden="true" tabindex="-1"></a>RTG_a2_df<span class="sc">$</span>marker <span class="ot">=</span> <span class="fu">row.names</span>(RTG_a2_df)</span>
<span id="cb52-13"><a href="#cb52-13" aria-hidden="true" tabindex="-1"></a>RTG_a2_df<span class="sc">$</span>chr <span class="ot">=</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a2_df))</span>
<span id="cb52-14"><a href="#cb52-14" aria-hidden="true" tabindex="-1"></a>RTG_a2_df<span class="sc">$</span>stagger <span class="ot">=</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a2_df))</span>
<span id="cb52-15"><a href="#cb52-15" aria-hidden="true" tabindex="-1"></a>RTG_a2_df<span class="sc">$</span>pos <span class="ot">=</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a2_df))</span>
<span id="cb52-16"><a href="#cb52-16" aria-hidden="true" tabindex="-1"></a>RTG_a2_df<span class="sc">$</span>calc_pos <span class="ot">=</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a2_df))</span>
<span id="cb52-17"><a href="#cb52-17" aria-hidden="true" tabindex="-1"></a>RTG_a2_df<span class="sc">$</span>label <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a2_df))</span>
<span id="cb52-18"><a href="#cb52-18" aria-hidden="true" tabindex="-1"></a>RTG_a2_min_max_mid <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">row.names =</span> chrs,</span>
<span id="cb52-19"><a href="#cb52-19" aria-hidden="true" tabindex="-1"></a>                                <span class="at">chr =</span> chrs,</span>
<span id="cb52-20"><a href="#cb52-20" aria-hidden="true" tabindex="-1"></a>                                <span class="at">max =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)),</span>
<span id="cb52-21"><a href="#cb52-21" aria-hidden="true" tabindex="-1"></a>                                <span class="at">calc_min =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)),</span>
<span id="cb52-22"><a href="#cb52-22" aria-hidden="true" tabindex="-1"></a>                                <span class="at">calc_mid =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)),</span>
<span id="cb52-23"><a href="#cb52-23" aria-hidden="true" tabindex="-1"></a>                                <span class="at">calc_max =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)))</span>
<span id="cb52-24"><a href="#cb52-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb52-25"><a href="#cb52-25" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting data frame for plotting RTG a2 in ggplot2</span></span>
<span id="cb52-26"><a href="#cb52-26" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(chrs)) {</span>
<span id="cb52-27"><a href="#cb52-27" aria-hidden="true" tabindex="-1"></a>  chr_i <span class="ot">=</span> chrs[i]</span>
<span id="cb52-28"><a href="#cb52-28" aria-hidden="true" tabindex="-1"></a>  pmap_i <span class="ot">=</span> pmap[[chr_i]]</span>
<span id="cb52-29"><a href="#cb52-29" aria-hidden="true" tabindex="-1"></a>  RTG_a2_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;chr&quot;</span>] <span class="ot">=</span> chr_i</span>
<span id="cb52-30"><a href="#cb52-30" aria-hidden="true" tabindex="-1"></a>  RTG_a2_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;stagger&quot;</span>] <span class="ot">=</span> <span class="fu">ifelse</span>(i <span class="sc">%%</span> <span class="dv">2</span> <span class="sc">==</span> <span class="dv">0</span>, <span class="st">&quot;darkblue&quot;</span>, <span class="st">&quot;lightblue&quot;</span>)</span>
<span id="cb52-31"><a href="#cb52-31" aria-hidden="true" tabindex="-1"></a>  RTG_a2_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;pos&quot;</span>] <span class="ot">=</span> pmap_i</span>
<span id="cb52-32"><a href="#cb52-32" aria-hidden="true" tabindex="-1"></a>  RTG_a2_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;calc_pos&quot;</span>] <span class="ot">=</span> min_chr_i <span class="sc">+</span> pmap_i</span>
<span id="cb52-33"><a href="#cb52-33" aria-hidden="true" tabindex="-1"></a>  max_lod_chr_i <span class="ot">=</span> <span class="fu">max</span>(RTG_a2_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;pheno1&quot;</span>])</span>
<span id="cb52-34"><a href="#cb52-34" aria-hidden="true" tabindex="-1"></a>  max_marker_chr_i <span class="ot">=</span> RTG_a2_df[<span class="fu">which</span>(RTG_a2_df<span class="sc">$</span>chr <span class="sc">==</span> chr_i <span class="sc">&amp;</span> RTG_a2_df<span class="sc">$</span>pheno1 <span class="sc">==</span> max_lod_chr_i),<span class="st">&quot;marker&quot;</span>]</span>
<span id="cb52-35"><a href="#cb52-35" aria-hidden="true" tabindex="-1"></a>  RTG_a2_df[max_marker_chr_i,<span class="st">&quot;label&quot;</span>] <span class="ot">=</span> <span class="fu">ifelse</span>(max_lod_chr_i <span class="sc">&gt;</span> RTG_a2_five_percent,</span>
<span id="cb52-36"><a href="#cb52-36" aria-hidden="true" tabindex="-1"></a>                                               <span class="fu">paste</span>(max_marker_chr_i, <span class="st">&quot; at chr&quot;</span>, chr_i,</span>
<span id="cb52-37"><a href="#cb52-37" aria-hidden="true" tabindex="-1"></a>                                                     <span class="st">&quot;:&quot;</span>,<span class="fu">round</span>(RTG_a2_df[max_marker_chr_i,<span class="st">&quot;pos&quot;</span>], <span class="dv">2</span>),</span>
<span id="cb52-38"><a href="#cb52-38" aria-hidden="true" tabindex="-1"></a>                                                     <span class="st">&quot; Mb (LOD = &quot;</span>,<span class="fu">round</span>(max_lod_chr_i, <span class="dv">2</span>),<span class="st">&quot;)&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>),</span>
<span id="cb52-39"><a href="#cb52-39" aria-hidden="true" tabindex="-1"></a>                                               <span class="st">&quot;&quot;</span>)</span>
<span id="cb52-40"><a href="#cb52-40" aria-hidden="true" tabindex="-1"></a>  max_pos_i <span class="ot">=</span> <span class="fu">max</span>(pmap_i)</span>
<span id="cb52-41"><a href="#cb52-41" aria-hidden="true" tabindex="-1"></a>  RTG_a2_min_max_mid[chr_i,<span class="st">&quot;calc_min&quot;</span>] <span class="ot">=</span> min_chr_i</span>
<span id="cb52-42"><a href="#cb52-42" aria-hidden="true" tabindex="-1"></a>  RTG_a2_min_max_mid[chr_i,<span class="st">&quot;calc_max&quot;</span>] <span class="ot">=</span> min_chr_i <span class="sc">+</span> max_pos_i</span>
<span id="cb52-43"><a href="#cb52-43" aria-hidden="true" tabindex="-1"></a>  RTG_a2_min_max_mid[chr_i,<span class="st">&quot;max&quot;</span>] <span class="ot">=</span> max_pos_i</span>
<span id="cb52-44"><a href="#cb52-44" aria-hidden="true" tabindex="-1"></a>  min_chr_i <span class="ot">=</span> min_chr_i <span class="sc">+</span> max_pos_i <span class="sc">+</span> gap_Mb</span>
<span id="cb52-45"><a href="#cb52-45" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb52-46"><a href="#cb52-46" aria-hidden="true" tabindex="-1"></a>RTG_a2_min_max_mid<span class="sc">$</span>calc_mid <span class="ot">=</span> RTG_a2_min_max_mid<span class="sc">$</span>calc_min <span class="sc">+</span> (<span class="fl">0.5</span> <span class="sc">*</span> (RTG_a2_min_max_mid<span class="sc">$</span>calc_max <span class="sc">-</span> RTG_a2_min_max_mid<span class="sc">$</span>calc_min))</span>
<span id="cb52-47"><a href="#cb52-47" aria-hidden="true" tabindex="-1"></a>RTG_a2_df<span class="sc">$</span>chr <span class="ot">=</span> <span class="fu">factor</span>(RTG_a2_df<span class="sc">$</span>chr, <span class="at">levels =</span> chrs, <span class="at">ordered =</span> <span class="cn">TRUE</span>)</span>
<span id="cb52-48"><a href="#cb52-48" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb52-49"><a href="#cb52-49" aria-hidden="true" tabindex="-1"></a><span class="co"># Plotting RTG a2</span></span>
<span id="cb52-50"><a href="#cb52-50" aria-hidden="true" tabindex="-1"></a>RTG_a2_mapping_plot <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> RTG_a2_df, <span class="fu">aes</span>(<span class="at">x =</span> calc_pos,</span>
<span id="cb52-51"><a href="#cb52-51" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">y =</span> pheno1,</span>
<span id="cb52-52"><a href="#cb52-52" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">color =</span> stagger,</span>
<span id="cb52-53"><a href="#cb52-53" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">group =</span> chr,</span>
<span id="cb52-54"><a href="#cb52-54" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">label =</span> label)) <span class="sc">+</span></span>
<span id="cb52-55"><a href="#cb52-55" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb52-56"><a href="#cb52-56" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>, <span class="at">panel.grid =</span> <span class="fu">element_line</span>(<span class="at">color =</span> <span class="st">&quot;#FFFFFF&quot;</span>)) <span class="sc">+</span></span>
<span id="cb52-57"><a href="#cb52-57" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">size =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb52-58"><a href="#cb52-58" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">labels =</span> RTG_a2_min_max_mid<span class="sc">$</span>chr,</span>
<span id="cb52-59"><a href="#cb52-59" aria-hidden="true" tabindex="-1"></a>                     <span class="at">breaks =</span> RTG_a2_min_max_mid<span class="sc">$</span>calc_mid,</span>
<span id="cb52-60"><a href="#cb52-60" aria-hidden="true" tabindex="-1"></a>                     <span class="at">minor_breaks =</span> <span class="fu">c</span>(RTG_a2_min_max_mid<span class="sc">$</span>calc_min, RTG_a2_min_max_mid<span class="sc">$</span>calc_max)) <span class="sc">+</span></span>
<span id="cb52-61"><a href="#cb52-61" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#002D72&quot;</span>,<span class="st">&quot;#0085CA&quot;</span>)) <span class="sc">+</span></span>
<span id="cb52-62"><a href="#cb52-62" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="cn">NULL</span>) <span class="sc">+</span></span>
<span id="cb52-63"><a href="#cb52-63" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;LOD Score&quot;</span>) <span class="sc">+</span></span>
<span id="cb52-64"><a href="#cb52-64" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> RTG_a2_five_percent, <span class="at">color =</span> <span class="st">&quot;#AB2328&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb52-65"><a href="#cb52-65" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> RTG_a2_ten_percent, <span class="at">color =</span> <span class="st">&quot;#ED8B00&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb52-66"><a href="#cb52-66" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> RTG_a2_suggestive, <span class="at">color =</span> <span class="st">&quot;#F1C400&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb52-67"><a href="#cb52-67" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">label =</span> <span class="fu">expression</span>(<span class="fu">paste</span>(alpha, <span class="st">&quot; = 0.05&quot;</span>)), <span class="at">x =</span> <span class="dv">80</span>, <span class="at">y =</span> RTG_a2_five_percent <span class="sc">+</span> <span class="fl">0.25</span>, <span class="at">color =</span> <span class="st">&quot;#AB2328&quot;</span>) <span class="sc">+</span></span>
<span id="cb52-68"><a href="#cb52-68" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">label =</span> <span class="fu">expression</span>(<span class="fu">paste</span>(alpha, <span class="st">&quot; = 0.10&quot;</span>)), <span class="at">x =</span> <span class="dv">80</span>, <span class="at">y =</span> RTG_a2_ten_percent <span class="sc">+</span> <span class="fl">0.25</span>, <span class="at">color =</span> <span class="st">&quot;#ED8B00&quot;</span>) <span class="sc">+</span></span>
<span id="cb52-69"><a href="#cb52-69" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">label =</span> <span class="st">&quot;Suggestive&quot;</span>, <span class="at">x =</span> <span class="dv">80</span>, <span class="at">y =</span> RTG_a2_suggestive <span class="sc">+</span> <span class="fl">0.25</span>, <span class="at">color =</span> <span class="st">&quot;#F1C400&quot;</span>) <span class="sc">+</span></span>
<span id="cb52-70"><a href="#cb52-70" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_label_repel</span>() <span class="sc">+</span></span>
<span id="cb52-71"><a href="#cb52-71" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">&quot;RTG Analysis 2: Mapping Results&quot;</span>)</span>
<span id="cb52-72"><a href="#cb52-72" aria-hidden="true" tabindex="-1"></a>RTG_a2_mapping_plot</span></code></pre></div>
<pre><code>Warning in is.na(x): is.na() applied to non-(list or vector) of type
&#39;expression&#39;

Warning in is.na(x): is.na() applied to non-(list or vector) of type
&#39;expression&#39;</code></pre>
<p><img src="figure/17_mapping_interpretation.Rmd/unnamed-chunk-34-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-34-1">
Past versions of unnamed-chunk-34-1.png
</button>
</p>
<div id="fig-unnamed-chunk-34-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
2e38c65
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-30
</td>
</tr>
<tr>
<td>
c86ee90
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-20
</td>
</tr>
<tr>
<td>
9b43495
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-19
</td>
</tr>
<tr>
<td>
da1922f
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-17
</td>
</tr>
<tr>
<td>
f9c5856
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Plotting <code>RTG_a3</code> plot.</p>
<div class="sourceCode" id="cb54"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb54-1"><a href="#cb54-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting QTL thresholds</span></span>
<span id="cb54-2"><a href="#cb54-2" aria-hidden="true" tabindex="-1"></a>RTG_a3_permu_thresholds <span class="ot">=</span> <span class="fu">quantile</span>(RTG_a3_perms, <span class="at">probs =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">1</span><span class="fl">-0.63</span>, <span class="dv">1</span><span class="fl">-0.1</span>, <span class="dv">1</span><span class="fl">-0.05</span> ,<span class="dv">1</span><span class="fl">-0.01</span>, <span class="dv">1</span>))</span>
<span id="cb54-3"><a href="#cb54-3" aria-hidden="true" tabindex="-1"></a>RTG_a3_suggestive <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;37%&quot;</span>]</span>
<span id="cb54-4"><a href="#cb54-4" aria-hidden="true" tabindex="-1"></a>RTG_a3_ten_percent <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;90%&quot;</span>]</span>
<span id="cb54-5"><a href="#cb54-5" aria-hidden="true" tabindex="-1"></a>RTG_a3_five_percent <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;95%&quot;</span>]</span>
<span id="cb54-6"><a href="#cb54-6" aria-hidden="true" tabindex="-1"></a>RTG_a3_one_percent <span class="ot">=</span> RTG_a3_permu_thresholds[<span class="st">&quot;99%&quot;</span>]</span>
<span id="cb54-7"><a href="#cb54-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb54-8"><a href="#cb54-8" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting information to plot</span></span>
<span id="cb54-9"><a href="#cb54-9" aria-hidden="true" tabindex="-1"></a>gap_Mb <span class="ot">=</span> <span class="dv">20</span></span>
<span id="cb54-10"><a href="#cb54-10" aria-hidden="true" tabindex="-1"></a>min_chr_i <span class="ot">=</span> <span class="dv">0</span></span>
<span id="cb54-11"><a href="#cb54-11" aria-hidden="true" tabindex="-1"></a>chrs <span class="ot">=</span> <span class="fu">as.character</span>(<span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">19</span>,<span class="st">&quot;X&quot;</span>))</span>
<span id="cb54-12"><a href="#cb54-12" aria-hidden="true" tabindex="-1"></a>RTG_a3_df <span class="ot">=</span> <span class="fu">as.data.frame</span>(RTG_a3_qtl)</span>
<span id="cb54-13"><a href="#cb54-13" aria-hidden="true" tabindex="-1"></a>RTG_a3_df<span class="sc">$</span>marker <span class="ot">=</span> <span class="fu">row.names</span>(RTG_a3_df)</span>
<span id="cb54-14"><a href="#cb54-14" aria-hidden="true" tabindex="-1"></a>RTG_a3_df<span class="sc">$</span>chr <span class="ot">=</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a3_df))</span>
<span id="cb54-15"><a href="#cb54-15" aria-hidden="true" tabindex="-1"></a>RTG_a3_df<span class="sc">$</span>stagger <span class="ot">=</span> <span class="fu">rep</span>(<span class="cn">NA</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a3_df))</span>
<span id="cb54-16"><a href="#cb54-16" aria-hidden="true" tabindex="-1"></a>RTG_a3_df<span class="sc">$</span>pos <span class="ot">=</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a3_df))</span>
<span id="cb54-17"><a href="#cb54-17" aria-hidden="true" tabindex="-1"></a>RTG_a3_df<span class="sc">$</span>calc_pos <span class="ot">=</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a3_df))</span>
<span id="cb54-18"><a href="#cb54-18" aria-hidden="true" tabindex="-1"></a>RTG_a3_df<span class="sc">$</span>label <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a3_df))</span>
<span id="cb54-19"><a href="#cb54-19" aria-hidden="true" tabindex="-1"></a>RTG_a3_min_max_mid <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">row.names =</span> chrs,</span>
<span id="cb54-20"><a href="#cb54-20" aria-hidden="true" tabindex="-1"></a>                                <span class="at">chr =</span> chrs,</span>
<span id="cb54-21"><a href="#cb54-21" aria-hidden="true" tabindex="-1"></a>                                <span class="at">max =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)),</span>
<span id="cb54-22"><a href="#cb54-22" aria-hidden="true" tabindex="-1"></a>                                <span class="at">calc_min =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)),</span>
<span id="cb54-23"><a href="#cb54-23" aria-hidden="true" tabindex="-1"></a>                                <span class="at">calc_mid =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)),</span>
<span id="cb54-24"><a href="#cb54-24" aria-hidden="true" tabindex="-1"></a>                                <span class="at">calc_max =</span> <span class="fu">rep</span>(<span class="sc">-</span><span class="dv">10</span>, <span class="at">times =</span> <span class="fu">length</span>(chrs)))</span>
<span id="cb54-25"><a href="#cb54-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb54-26"><a href="#cb54-26" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting data frame for plotting RTG a3 in ggplot2</span></span>
<span id="cb54-27"><a href="#cb54-27" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(chrs)) {</span>
<span id="cb54-28"><a href="#cb54-28" aria-hidden="true" tabindex="-1"></a>  chr_i <span class="ot">=</span> chrs[i]</span>
<span id="cb54-29"><a href="#cb54-29" aria-hidden="true" tabindex="-1"></a>  pmap_i <span class="ot">=</span> pmap[[chr_i]]</span>
<span id="cb54-30"><a href="#cb54-30" aria-hidden="true" tabindex="-1"></a>  RTG_a3_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;chr&quot;</span>] <span class="ot">=</span> chr_i</span>
<span id="cb54-31"><a href="#cb54-31" aria-hidden="true" tabindex="-1"></a>  RTG_a3_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;stagger&quot;</span>] <span class="ot">=</span> <span class="fu">ifelse</span>(i <span class="sc">%%</span> <span class="dv">2</span> <span class="sc">==</span> <span class="dv">0</span>, <span class="st">&quot;darkblue&quot;</span>, <span class="st">&quot;lightblue&quot;</span>)</span>
<span id="cb54-32"><a href="#cb54-32" aria-hidden="true" tabindex="-1"></a>  RTG_a3_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;pos&quot;</span>] <span class="ot">=</span> pmap_i</span>
<span id="cb54-33"><a href="#cb54-33" aria-hidden="true" tabindex="-1"></a>  RTG_a3_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;calc_pos&quot;</span>] <span class="ot">=</span> min_chr_i <span class="sc">+</span> pmap_i</span>
<span id="cb54-34"><a href="#cb54-34" aria-hidden="true" tabindex="-1"></a>  max_lod_chr_i <span class="ot">=</span> <span class="fu">max</span>(RTG_a3_df[<span class="fu">names</span>(pmap_i),<span class="st">&quot;pheno1&quot;</span>], <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb54-35"><a href="#cb54-35" aria-hidden="true" tabindex="-1"></a>  max_marker_chr_i <span class="ot">=</span> RTG_a3_df[<span class="fu">which</span>(RTG_a3_df<span class="sc">$</span>chr <span class="sc">==</span> chr_i <span class="sc">&amp;</span> RTG_a3_df<span class="sc">$</span>pheno1 <span class="sc">==</span> max_lod_chr_i),<span class="st">&quot;marker&quot;</span>]</span>
<span id="cb54-36"><a href="#cb54-36" aria-hidden="true" tabindex="-1"></a>  RTG_a3_df[max_marker_chr_i,<span class="st">&quot;label&quot;</span>] <span class="ot">=</span> <span class="fu">ifelse</span>(max_lod_chr_i <span class="sc">&gt;</span> RTG_a3_five_percent,</span>
<span id="cb54-37"><a href="#cb54-37" aria-hidden="true" tabindex="-1"></a>                                               <span class="fu">paste</span>(max_marker_chr_i, <span class="st">&quot; at chr&quot;</span>, chr_i,</span>
<span id="cb54-38"><a href="#cb54-38" aria-hidden="true" tabindex="-1"></a>                                                     <span class="st">&quot;:&quot;</span>,<span class="fu">round</span>(RTG_a3_df[max_marker_chr_i,<span class="st">&quot;pos&quot;</span>], <span class="dv">2</span>),</span>
<span id="cb54-39"><a href="#cb54-39" aria-hidden="true" tabindex="-1"></a>                                                     <span class="st">&quot; Mb (LOD = &quot;</span>,<span class="fu">round</span>(max_lod_chr_i, <span class="dv">2</span>),<span class="st">&quot;)&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>),</span>
<span id="cb54-40"><a href="#cb54-40" aria-hidden="true" tabindex="-1"></a>                                               <span class="st">&quot;&quot;</span>)</span>
<span id="cb54-41"><a href="#cb54-41" aria-hidden="true" tabindex="-1"></a>  max_pos_i <span class="ot">=</span> <span class="fu">max</span>(pmap_i)</span>
<span id="cb54-42"><a href="#cb54-42" aria-hidden="true" tabindex="-1"></a>  RTG_a3_min_max_mid[chr_i,<span class="st">&quot;calc_min&quot;</span>] <span class="ot">=</span> min_chr_i</span>
<span id="cb54-43"><a href="#cb54-43" aria-hidden="true" tabindex="-1"></a>  RTG_a3_min_max_mid[chr_i,<span class="st">&quot;calc_max&quot;</span>] <span class="ot">=</span> min_chr_i <span class="sc">+</span> max_pos_i</span>
<span id="cb54-44"><a href="#cb54-44" aria-hidden="true" tabindex="-1"></a>  RTG_a3_min_max_mid[chr_i,<span class="st">&quot;max&quot;</span>] <span class="ot">=</span> max_pos_i</span>
<span id="cb54-45"><a href="#cb54-45" aria-hidden="true" tabindex="-1"></a>  min_chr_i <span class="ot">=</span> min_chr_i <span class="sc">+</span> max_pos_i <span class="sc">+</span> gap_Mb</span>
<span id="cb54-46"><a href="#cb54-46" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb54-47"><a href="#cb54-47" aria-hidden="true" tabindex="-1"></a>RTG_a3_min_max_mid<span class="sc">$</span>calc_mid <span class="ot">=</span> RTG_a3_min_max_mid<span class="sc">$</span>calc_min <span class="sc">+</span> (<span class="fl">0.5</span> <span class="sc">*</span> (RTG_a3_min_max_mid<span class="sc">$</span>calc_max <span class="sc">-</span> RTG_a3_min_max_mid<span class="sc">$</span>calc_min))</span>
<span id="cb54-48"><a href="#cb54-48" aria-hidden="true" tabindex="-1"></a>RTG_a3_df<span class="sc">$</span>chr <span class="ot">=</span> <span class="fu">factor</span>(RTG_a3_df<span class="sc">$</span>chr, <span class="at">levels =</span> chrs, <span class="at">ordered =</span> <span class="cn">TRUE</span>)</span>
<span id="cb54-49"><a href="#cb54-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb54-50"><a href="#cb54-50" aria-hidden="true" tabindex="-1"></a><span class="co"># Plotting RTG a3</span></span>
<span id="cb54-51"><a href="#cb54-51" aria-hidden="true" tabindex="-1"></a>RTG_a3_mapping_plot <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> RTG_a3_df, <span class="fu">aes</span>(<span class="at">x =</span> calc_pos,</span>
<span id="cb54-52"><a href="#cb54-52" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">y =</span> pheno1,</span>
<span id="cb54-53"><a href="#cb54-53" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">color =</span> stagger,</span>
<span id="cb54-54"><a href="#cb54-54" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">group =</span> chr, </span>
<span id="cb54-55"><a href="#cb54-55" aria-hidden="true" tabindex="-1"></a>                                                   <span class="at">label =</span> label)) <span class="sc">+</span></span>
<span id="cb54-56"><a href="#cb54-56" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb54-57"><a href="#cb54-57" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>, <span class="at">panel.grid =</span> <span class="fu">element_line</span>(<span class="at">color =</span> <span class="st">&quot;#FFFFFF&quot;</span>)) <span class="sc">+</span></span>
<span id="cb54-58"><a href="#cb54-58" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">size =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb54-59"><a href="#cb54-59" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">labels =</span> RTG_a3_min_max_mid<span class="sc">$</span>chr, </span>
<span id="cb54-60"><a href="#cb54-60" aria-hidden="true" tabindex="-1"></a>                     <span class="at">breaks =</span> RTG_a3_min_max_mid<span class="sc">$</span>calc_mid, </span>
<span id="cb54-61"><a href="#cb54-61" aria-hidden="true" tabindex="-1"></a>                     <span class="at">minor_breaks =</span> <span class="fu">c</span>(RTG_a3_min_max_mid<span class="sc">$</span>calc_min, RTG_a3_min_max_mid<span class="sc">$</span>calc_max)) <span class="sc">+</span></span>
<span id="cb54-62"><a href="#cb54-62" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#002D72&quot;</span>,<span class="st">&quot;#0085CA&quot;</span>)) <span class="sc">+</span></span>
<span id="cb54-63"><a href="#cb54-63" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="cn">NULL</span>) <span class="sc">+</span></span>
<span id="cb54-64"><a href="#cb54-64" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;LOD Score&quot;</span>) <span class="sc">+</span></span>
<span id="cb54-65"><a href="#cb54-65" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> RTG_a3_five_percent, <span class="at">color =</span> <span class="st">&quot;#AB2328&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb54-66"><a href="#cb54-66" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> RTG_a3_ten_percent, <span class="at">color =</span> <span class="st">&quot;#ED8B00&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb54-67"><a href="#cb54-67" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> RTG_a3_suggestive, <span class="at">color =</span> <span class="st">&quot;#F1C400&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb54-68"><a href="#cb54-68" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">label =</span> <span class="fu">expression</span>(<span class="fu">paste</span>(alpha, <span class="st">&quot; = 0.05&quot;</span>)), <span class="at">x =</span> <span class="dv">80</span>, <span class="at">y =</span> RTG_a3_five_percent <span class="sc">+</span> <span class="fl">0.25</span>, <span class="at">color =</span> <span class="st">&quot;#AB2328&quot;</span>) <span class="sc">+</span></span>
<span id="cb54-69"><a href="#cb54-69" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">label =</span> <span class="fu">expression</span>(<span class="fu">paste</span>(alpha, <span class="st">&quot; = 0.10&quot;</span>)), <span class="at">x =</span> <span class="dv">80</span>, <span class="at">y =</span> RTG_a3_ten_percent <span class="sc">+</span> <span class="fl">0.25</span>, <span class="at">color =</span> <span class="st">&quot;#ED8B00&quot;</span>) <span class="sc">+</span></span>
<span id="cb54-70"><a href="#cb54-70" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">&quot;text&quot;</span>, <span class="at">label =</span> <span class="st">&quot;Suggestive&quot;</span>, <span class="at">x =</span> <span class="dv">80</span>, <span class="at">y =</span> RTG_a3_suggestive <span class="sc">+</span> <span class="fl">0.25</span>, <span class="at">color =</span> <span class="st">&quot;#F1C400&quot;</span>) <span class="sc">+</span></span>
<span id="cb54-71"><a href="#cb54-71" aria-hidden="true" tabindex="-1"></a><span class="co">#   geom_label_repel() +</span></span>
<span id="cb54-72"><a href="#cb54-72" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">&quot;RTG Analysis 3: Mapping Results&quot;</span>)</span>
<span id="cb54-73"><a href="#cb54-73" aria-hidden="true" tabindex="-1"></a>RTG_a3_mapping_plot</span></code></pre></div>
<pre><code>Warning in is.na(x): is.na() applied to non-(list or vector) of type
&#39;expression&#39;

Warning in is.na(x): is.na() applied to non-(list or vector) of type
&#39;expression&#39;</code></pre>
<p><img src="figure/17_mapping_interpretation.Rmd/unnamed-chunk-35-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-35-1">
Past versions of unnamed-chunk-35-1.png
</button>
</p>
<div id="fig-unnamed-chunk-35-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
c86ee90
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-20
</td>
</tr>
<tr>
<td>
9b43495
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Compiling all RTG traits table</p>
<div class="sourceCode" id="cb56"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb56-1"><a href="#cb56-1" aria-hidden="true" tabindex="-1"></a><span class="co"># 30</span></span>
<span id="cb56-2"><a href="#cb56-2" aria-hidden="true" tabindex="-1"></a>SENS_COC_df<span class="sc">$</span>analysis <span class="ot">=</span> <span class="st">&quot;Sensitization&quot;</span></span>
<span id="cb56-3"><a href="#cb56-3" aria-hidden="true" tabindex="-1"></a>RTG_a2_df<span class="sc">$</span>analysis <span class="ot">=</span> <span class="st">&quot;IVSA Acquisition&quot;</span></span>
<span id="cb56-4"><a href="#cb56-4" aria-hidden="true" tabindex="-1"></a>RTG_a3_df<span class="sc">$</span>analysis <span class="ot">=</span> <span class="st">&quot;IVSA All Traits&quot;</span></span>
<span id="cb56-5"><a href="#cb56-5" aria-hidden="true" tabindex="-1"></a>RTG_a3_df<span class="sc">$</span>label <span class="ot">=</span> <span class="fu">ifelse</span>(RTG_a3_df<span class="sc">$</span>label <span class="sc">==</span> <span class="st">&quot;&quot;</span>, <span class="cn">NA</span>, RTG_a3_df<span class="sc">$</span>label)</span>
<span id="cb56-6"><a href="#cb56-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb56-7"><a href="#cb56-7" aria-hidden="true" tabindex="-1"></a>qtl_plot_df <span class="ot">=</span> <span class="fu">rbind</span>(SENS_COC_df, RTG_a2_df, RTG_a3_df)</span>
<span id="cb56-8"><a href="#cb56-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb56-9"><a href="#cb56-9" aria-hidden="true" tabindex="-1"></a>qtl_plot_df<span class="sc">$</span>analysis <span class="ot">=</span> <span class="fu">factor</span>(qtl_plot_df<span class="sc">$</span>analysis,</span>
<span id="cb56-10"><a href="#cb56-10" aria-hidden="true" tabindex="-1"></a>                              <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">&quot;Sensitization&quot;</span>,</span>
<span id="cb56-11"><a href="#cb56-11" aria-hidden="true" tabindex="-1"></a>                                         <span class="st">&quot;IVSA Acquisition&quot;</span>,</span>
<span id="cb56-12"><a href="#cb56-12" aria-hidden="true" tabindex="-1"></a>                                         <span class="st">&quot;IVSA All Traits&quot;</span>),</span>
<span id="cb56-13"><a href="#cb56-13" aria-hidden="true" tabindex="-1"></a>                              <span class="at">ordered =</span> <span class="cn">TRUE</span>)</span>
<span id="cb56-14"><a href="#cb56-14" aria-hidden="true" tabindex="-1"></a>qtl_plot_df<span class="sc">$</span>chr_analysis <span class="ot">=</span> <span class="fu">paste</span>(qtl_plot_df<span class="sc">$</span>analysis, </span>
<span id="cb56-15"><a href="#cb56-15" aria-hidden="true" tabindex="-1"></a>                                 <span class="fu">ifelse</span>(qtl_plot_df<span class="sc">$</span>stagger <span class="sc">==</span> <span class="st">&quot;lightblue&quot;</span>,</span>
<span id="cb56-16"><a href="#cb56-16" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">&quot;odd&quot;</span>, <span class="st">&quot;even&quot;</span>))</span>
<span id="cb56-17"><a href="#cb56-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb56-18"><a href="#cb56-18" aria-hidden="true" tabindex="-1"></a>qtl_plot_faceted <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> qtl_plot_df, <span class="fu">aes</span>(<span class="at">x =</span> calc_pos,</span>
<span id="cb56-19"><a href="#cb56-19" aria-hidden="true" tabindex="-1"></a>                                                  <span class="at">y =</span> pheno1,</span>
<span id="cb56-20"><a href="#cb56-20" aria-hidden="true" tabindex="-1"></a>                                                  <span class="at">color =</span> chr_analysis,</span>
<span id="cb56-21"><a href="#cb56-21" aria-hidden="true" tabindex="-1"></a>                                                  <span class="at">group =</span> chr,</span>
<span id="cb56-22"><a href="#cb56-22" aria-hidden="true" tabindex="-1"></a>                                                  <span class="at">label =</span> label)) <span class="sc">+</span></span>
<span id="cb56-23"><a href="#cb56-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb56-24"><a href="#cb56-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>, <span class="at">panel.grid =</span> <span class="fu">element_line</span>(<span class="at">color =</span> <span class="st">&quot;#FFFFFF&quot;</span>)) <span class="sc">+</span></span>
<span id="cb56-25"><a href="#cb56-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">size =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb56-26"><a href="#cb56-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">labels =</span> RTG_a3_min_max_mid<span class="sc">$</span>chr, </span>
<span id="cb56-27"><a href="#cb56-27" aria-hidden="true" tabindex="-1"></a>                     <span class="at">breaks =</span> RTG_a3_min_max_mid<span class="sc">$</span>calc_mid, </span>
<span id="cb56-28"><a href="#cb56-28" aria-hidden="true" tabindex="-1"></a>                     <span class="at">minor_breaks =</span> <span class="fu">c</span>(RTG_a3_min_max_mid<span class="sc">$</span>calc_min, RTG_a3_min_max_mid<span class="sc">$</span>calc_max)) <span class="sc">+</span></span>
<span id="cb56-29"><a href="#cb56-29" aria-hidden="true" tabindex="-1"></a>  <span class="co"># scale_color_manual(values = c(&quot;#66CC66&quot;,&quot;#448844&quot;,</span></span>
<span id="cb56-30"><a href="#cb56-30" aria-hidden="true" tabindex="-1"></a>  <span class="co">#                               &quot;#CC3333&quot;,&quot;#882222&quot;,</span></span>
<span id="cb56-31"><a href="#cb56-31" aria-hidden="true" tabindex="-1"></a>  <span class="co">#                               &quot;#3399CC&quot;,&quot;#226688&quot;)) +</span></span>
<span id="cb56-32"><a href="#cb56-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#333333&quot;</span>,<span class="st">&quot;#999999&quot;</span>,</span>
<span id="cb56-33"><a href="#cb56-33" aria-hidden="true" tabindex="-1"></a>                                <span class="st">&quot;#333333&quot;</span>,<span class="st">&quot;#999999&quot;</span>,</span>
<span id="cb56-34"><a href="#cb56-34" aria-hidden="true" tabindex="-1"></a>                                <span class="st">&quot;#333333&quot;</span>,<span class="st">&quot;#999999&quot;</span>)) <span class="sc">+</span></span>
<span id="cb56-35"><a href="#cb56-35" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="cn">NULL</span>) <span class="sc">+</span></span>
<span id="cb56-36"><a href="#cb56-36" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;LOD Score&quot;</span>) <span class="sc">+</span></span>
<span id="cb56-37"><a href="#cb56-37" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> RTG_a3_five_percent, <span class="at">color =</span> <span class="st">&quot;#444444&quot;</span>, <span class="at">linetype =</span> <span class="st">&quot;dashed&quot;</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb56-38"><a href="#cb56-38" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_grid</span>(analysis <span class="sc">~</span> ., <span class="at">scales =</span> <span class="st">&quot;free_y&quot;</span>) <span class="sc">+</span></span>
<span id="cb56-39"><a href="#cb56-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">panel.spacing =</span> <span class="fu">unit</span>(<span class="dv">0</span>, <span class="st">&quot;mm&quot;</span>))</span>
<span id="cb56-40"><a href="#cb56-40" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb56-41"><a href="#cb56-41" aria-hidden="true" tabindex="-1"></a>qtl_plot_faceted</span></code></pre></div>
<p><img src="figure/17_mapping_interpretation.Rmd/unnamed-chunk-36-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-36-1">
Past versions of unnamed-chunk-36-1.png
</button>
</p>
<div id="fig-unnamed-chunk-36-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
9b43495
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-19
</td>
</tr>
<tr>
<td>
da1922f
</td>
<td>
Michael C. Saul
</td>
<td>
2020-07-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="compiling-qtl-table" class="section level2">
<h2>Compiling QTL table</h2>
<p>Compiling table of QTL results</p>
<div class="sourceCode" id="cb57"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb57-1"><a href="#cb57-1" aria-hidden="true" tabindex="-1"></a><span class="co"># RTG_a1 peaks</span></span>
<span id="cb57-2"><a href="#cb57-2" aria-hidden="true" tabindex="-1"></a>RTG_a1_peaks <span class="ot">=</span> <span class="fu">find_peaks</span>(RTG_a1_qtl, <span class="at">map =</span> pmap, <span class="at">threshold =</span> RTG_a1_suggestive, <span class="at">prob =</span> <span class="fl">0.95</span>)</span>
<span id="cb57-3"><a href="#cb57-3" aria-hidden="true" tabindex="-1"></a>RTG_a1_peaks<span class="sc">$</span>lodcolumn <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;RTG_a1&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a1_peaks))</span>
<span id="cb57-4"><a href="#cb57-4" aria-hidden="true" tabindex="-1"></a>RTG_a1_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;suggestive&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a1_peaks))</span>
<span id="cb57-5"><a href="#cb57-5" aria-hidden="true" tabindex="-1"></a>RTG_a1_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(RTG_a1_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a1_ten_percent, <span class="st">&quot;ten_percent&quot;</span>, RTG_a1_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-6"><a href="#cb57-6" aria-hidden="true" tabindex="-1"></a>RTG_a1_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(RTG_a1_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a1_five_percent, <span class="st">&quot;five_percent&quot;</span>, RTG_a1_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-7"><a href="#cb57-7" aria-hidden="true" tabindex="-1"></a>RTG_a1_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(RTG_a1_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a1_one_percent, <span class="st">&quot;one_percent&quot;</span>, RTG_a1_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-8"><a href="#cb57-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb57-9"><a href="#cb57-9" aria-hidden="true" tabindex="-1"></a><span class="co"># RTG_a2 peaks</span></span>
<span id="cb57-10"><a href="#cb57-10" aria-hidden="true" tabindex="-1"></a>RTG_a2_peaks <span class="ot">=</span> <span class="fu">find_peaks</span>(RTG_a2_qtl, <span class="at">map =</span> pmap, <span class="at">threshold =</span> RTG_a2_suggestive, <span class="at">prob =</span> <span class="fl">0.95</span>)</span>
<span id="cb57-11"><a href="#cb57-11" aria-hidden="true" tabindex="-1"></a>RTG_a2_peaks<span class="sc">$</span>lodcolumn <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;RTG_a2&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a2_peaks))</span>
<span id="cb57-12"><a href="#cb57-12" aria-hidden="true" tabindex="-1"></a>RTG_a2_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;suggestive&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a2_peaks))</span>
<span id="cb57-13"><a href="#cb57-13" aria-hidden="true" tabindex="-1"></a>RTG_a2_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(RTG_a2_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a2_ten_percent, <span class="st">&quot;ten_percent&quot;</span>, RTG_a2_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-14"><a href="#cb57-14" aria-hidden="true" tabindex="-1"></a>RTG_a2_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(RTG_a2_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a2_five_percent, <span class="st">&quot;five_percent&quot;</span>, RTG_a2_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-15"><a href="#cb57-15" aria-hidden="true" tabindex="-1"></a>RTG_a2_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(RTG_a2_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a2_one_percent, <span class="st">&quot;one_percent&quot;</span>, RTG_a2_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-16"><a href="#cb57-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb57-17"><a href="#cb57-17" aria-hidden="true" tabindex="-1"></a><span class="co"># RTG_a3 peaks</span></span>
<span id="cb57-18"><a href="#cb57-18" aria-hidden="true" tabindex="-1"></a>RTG_a3_peaks <span class="ot">=</span> <span class="fu">find_peaks</span>(RTG_a3_qtl, <span class="at">map =</span> pmap, <span class="at">threshold =</span> RTG_a3_suggestive, <span class="at">prob =</span> <span class="fl">0.95</span>)</span>
<span id="cb57-19"><a href="#cb57-19" aria-hidden="true" tabindex="-1"></a>RTG_a3_peaks<span class="sc">$</span>lodcolumn <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;RTG_a3&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a3_peaks))</span>
<span id="cb57-20"><a href="#cb57-20" aria-hidden="true" tabindex="-1"></a>RTG_a3_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;suggestive&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(RTG_a3_peaks))</span>
<span id="cb57-21"><a href="#cb57-21" aria-hidden="true" tabindex="-1"></a>RTG_a3_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(RTG_a3_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a3_ten_percent, <span class="st">&quot;ten_percent&quot;</span>, RTG_a3_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-22"><a href="#cb57-22" aria-hidden="true" tabindex="-1"></a>RTG_a3_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(RTG_a3_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a3_five_percent, <span class="st">&quot;five_percent&quot;</span>, RTG_a3_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-23"><a href="#cb57-23" aria-hidden="true" tabindex="-1"></a>RTG_a3_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(RTG_a3_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a3_one_percent, <span class="st">&quot;one_percent&quot;</span>, RTG_a3_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-24"><a href="#cb57-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb57-25"><a href="#cb57-25" aria-hidden="true" tabindex="-1"></a><span class="co"># SENS Cocaine peaks</span></span>
<span id="cb57-26"><a href="#cb57-26" aria-hidden="true" tabindex="-1"></a>SENS_COC_peaks <span class="ot">=</span> <span class="fu">find_peaks</span>(SENS_COC_qtl, <span class="at">map =</span> pmap, <span class="at">threshold =</span> RTG_a3_suggestive, <span class="at">prob =</span> <span class="fl">0.95</span>)</span>
<span id="cb57-27"><a href="#cb57-27" aria-hidden="true" tabindex="-1"></a>SENS_COC_peaks<span class="sc">$</span>lodcolumn <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;SENS&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(SENS_COC_peaks))</span>
<span id="cb57-28"><a href="#cb57-28" aria-hidden="true" tabindex="-1"></a>SENS_COC_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">rep</span>(<span class="st">&quot;suggestive&quot;</span>, <span class="at">times =</span> <span class="fu">nrow</span>(SENS_COC_peaks))</span>
<span id="cb57-29"><a href="#cb57-29" aria-hidden="true" tabindex="-1"></a>SENS_COC_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(SENS_COC_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a3_ten_percent, <span class="st">&quot;ten_percent&quot;</span>, SENS_COC_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-30"><a href="#cb57-30" aria-hidden="true" tabindex="-1"></a>SENS_COC_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(SENS_COC_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a3_five_percent, <span class="st">&quot;five_percent&quot;</span>, SENS_COC_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-31"><a href="#cb57-31" aria-hidden="true" tabindex="-1"></a>SENS_COC_peaks<span class="sc">$</span>threshold <span class="ot">=</span> <span class="fu">ifelse</span>(SENS_COC_peaks<span class="sc">$</span>lod <span class="sc">&gt;</span> RTG_a3_one_percent, <span class="st">&quot;one_percent&quot;</span>, SENS_COC_peaks<span class="sc">$</span>threshold)</span>
<span id="cb57-32"><a href="#cb57-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb57-33"><a href="#cb57-33" aria-hidden="true" tabindex="-1"></a><span class="co"># Making and displaying an RTG peaks table</span></span>
<span id="cb57-34"><a href="#cb57-34" aria-hidden="true" tabindex="-1"></a>RTG_peaks <span class="ot">=</span> <span class="fu">rbind</span>(RTG_a1_peaks, RTG_a2_peaks, RTG_a3_peaks, SENS_COC_peaks)</span>
<span id="cb57-35"><a href="#cb57-35" aria-hidden="true" tabindex="-1"></a>RTG_peaks <span class="ot">=</span> RTG_peaks[<span class="fu">order</span>(RTG_peaks<span class="sc">$</span>lod, <span class="at">decreasing =</span> <span class="cn">TRUE</span>),]</span>
<span id="cb57-36"><a href="#cb57-36" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(RTG_peaks[<span class="fu">order</span>(RTG_peaks<span class="sc">$</span>lod, <span class="at">decreasing =</span> <span class="cn">TRUE</span>),], <span class="at">file =</span> <span class="st">&quot;./data/RTG_QTL_peaks.RDS&quot;</span>)</span>
<span id="cb57-37"><a href="#cb57-37" aria-hidden="true" tabindex="-1"></a>RTG_peaks</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":[""],"name":["_rn_"],"type":[""],"align":["left"]},{"label":["lodindex"],"name":[1],"type":["int"],"align":["right"]},{"label":["lodcolumn"],"name":[2],"type":["chr"],"align":["left"]},{"label":["chr"],"name":[3],"type":["fct"],"align":["left"]},{"label":["pos"],"name":[4],"type":["dbl"],"align":["right"]},{"label":["lod"],"name":[5],"type":["dbl"],"align":["right"]},{"label":["ci_lo"],"name":[6],"type":["dbl"],"align":["right"]},{"label":["ci_hi"],"name":[7],"type":["dbl"],"align":["right"]},{"label":["threshold"],"name":[8],"type":["chr"],"align":["left"]}],"data":[{"1":"1","2":"RTG_a3","3":"7","4":"87.650445","5":"13.247377","6":"87.127600","7":"89.522474","8":"one_percent","_rn_":"18"},{"1":"1","2":"RTG_a3","3":"10","4":"126.992303","5":"9.573921","6":"126.910230","7":"128.357832","8":"one_percent","_rn_":"19"},{"1":"1","2":"SENS","3":"7","4":"89.522474","5":"9.410413","6":"87.244659","7":"91.284113","8":"one_percent","_rn_":"26"},{"1":"1","2":"SENS","3":"6","4":"5.211712","5":"9.122582","6":"4.997746","7":"5.753635","8":"one_percent","_rn_":"25"},{"1":"1","2":"RTG_a3","3":"X","4":"145.412921","5":"7.697957","6":"143.607480","7":"153.184507","8":"five_percent","_rn_":"23"},{"1":"1","2":"RTG_a3","3":"2","4":"156.947274","5":"7.516119","6":"21.537364","7":"159.974939","8":"ten_percent","_rn_":"14"},{"1":"1","2":"SENS","3":"3","4":"68.721822","5":"7.457026","6":"66.826817","7":"139.243295","8":"ten_percent","_rn_":"24"},{"1":"1","2":"SENS","3":"18","4":"81.704609","5":"7.331789","6":"77.645166","7":"83.757465","8":"ten_percent","_rn_":"29"},{"1":"1","2":"RTG_a3","3":"18","4":"81.677972","5":"7.063818","6":"77.645166","7":"82.530836","8":"suggestive","_rn_":"21"},{"1":"1","2":"RTG_a2","3":"4","4":"68.409256","5":"6.886803","6":"65.622424","7":"104.858027","8":"suggestive","_rn_":"10"},{"1":"1","2":"RTG_a1","3":"X","4":"126.063036","5":"6.871369","6":"126.063036","7":"126.668641","8":"suggestive","_rn_":"7"},{"1":"1","2":"RTG_a3","3":"4","4":"97.758260","5":"6.789763","6":"66.847420","7":"137.202836","8":"suggestive","_rn_":"16"},{"1":"1","2":"RTG_a1","3":"17","4":"30.794429","5":"6.767366","6":"24.319710","7":"31.191195","8":"suggestive","_rn_":"6"},{"1":"1","2":"RTG_a1","3":"4","4":"66.941798","5":"6.672830","6":"65.314612","7":"104.192049","8":"suggestive","_rn_":"2"},{"1":"1","2":"RTG_a2","3":"17","4":"26.580200","5":"6.646707","6":"23.927610","7":"30.924967","8":"suggestive","_rn_":"12"},{"1":"1","2":"RTG_a1","3":"8","4":"102.900587","5":"6.628390","6":"95.515283","7":"104.049311","8":"suggestive","_rn_":"4"},{"1":"1","2":"RTG_a2","3":"18","4":"79.073327","5":"6.502239","6":"35.133799","7":"88.557104","8":"suggestive","_rn_":"13"},{"1":"1","2":"RTG_a1","3":"10","4":"126.196944","5":"6.449488","6":"73.683324","7":"128.583553","8":"suggestive","_rn_":"5"},{"1":"1","2":"RTG_a3","3":"14","4":"109.432102","5":"6.314892","6":"31.239525","7":"113.914779","8":"suggestive","_rn_":"20"},{"1":"1","2":"RTG_a2","3":"2","4":"152.691424","5":"6.298269","6":"5.700593","7":"178.759883","8":"suggestive","_rn_":"8"},{"1":"1","2":"RTG_a3","3":"19","4":"28.103942","5":"6.289914","6":"27.612116","7":"32.738270","8":"suggestive","_rn_":"22"},{"1":"1","2":"SENS","3":"16","4":"30.819494","5":"6.287702","6":"24.498310","7":"78.155446","8":"suggestive","_rn_":"28"},{"1":"1","2":"RTG_a1","3":"3","4":"64.794282","5":"6.266369","6":"27.906078","7":"87.268036","8":"suggestive","_rn_":"1"},{"1":"1","2":"RTG_a2","3":"10","4":"79.060561","5":"6.192272","6":"28.485286","7":"127.644256","8":"suggestive","_rn_":"11"},{"1":"1","2":"RTG_a2","3":"3","4":"87.082227","5":"6.147783","6":"27.906078","7":"147.102650","8":"suggestive","_rn_":"9"},{"1":"1","2":"RTG_a3","3":"3","4":"41.717680","5":"6.103629","6":"8.849682","7":"141.224270","8":"suggestive","_rn_":"15"},{"1":"1","2":"RTG_a3","3":"6","4":"117.446498","5":"6.100421","6":"4.997746","7":"142.477920","8":"suggestive","_rn_":"17"},{"1":"1","2":"SENS","3":"10","4":"105.630599","5":"6.078273","6":"17.234900","7":"128.382937","8":"suggestive","_rn_":"27"},{"1":"1","2":"RTG_a1","3":"7","4":"90.552310","5":"5.964469","6":"19.443555","7":"139.357249","8":"suggestive","_rn_":"3"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<div id="document-control" class="section level3">
<h3>Document Control</h3>
<p>This document was prepared using RMarkdown in RStudio.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb58"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb58-1"><a href="#cb58-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.6 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/liblapack.so.3

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] edgeR_3.40.2      limma_3.54.2      data.table_1.14.8 ggplotify_0.1.0  
 [5] cowplot_1.1.1     qtl2_0.32         biomaRt_2.54.1    ggrepel_0.9.3    
 [9] forcats_0.5.2     stringr_1.5.0     dplyr_1.1.2       purrr_1.0.1      
[13] readr_2.1.3       tidyr_1.3.0       tibble_3.2.1      ggplot2_3.4.2    
[17] tidyverse_1.3.2  

loaded via a namespace (and not attached):
  [1] googledrive_2.0.0      colorspace_2.0-3       ellipsis_0.3.2        
  [4] rprojroot_2.0.3        XVector_0.38.0         fs_1.5.2              
  [7] rstudioapi_0.14        farver_2.1.1           bit64_4.0.5           
 [10] AnnotationDbi_1.60.2   fansi_1.0.3            lubridate_1.8.0       
 [13] xml2_1.3.3             cachem_1.0.6           knitr_1.42            
 [16] jsonlite_1.8.3         workflowr_1.7.0        broom_1.0.1           
 [19] dbplyr_2.2.1           png_0.1-7              compiler_4.2.1        
 [22] httr_1.4.4             backports_1.4.1        assertthat_0.2.1      
 [25] fastmap_1.1.0          gargle_1.2.1           cli_3.4.1             
 [28] later_1.3.0            htmltools_0.5.3        prettyunits_1.1.1     
 [31] tools_4.2.1            gtable_0.3.1           glue_1.6.2            
 [34] GenomeInfoDbData_1.2.9 rappdirs_0.3.3         Rcpp_1.0.9            
 [37] Biobase_2.58.0         cellranger_1.1.0       jquerylib_0.1.4       
 [40] vctrs_0.6.2            Biostrings_2.66.0      xfun_0.39             
 [43] rvest_1.0.3            lifecycle_1.0.3        XML_3.99-0.11         
 [46] googlesheets4_1.0.1    zlibbioc_1.44.0        scales_1.2.1          
 [49] ragg_1.2.4             hms_1.1.2              promises_1.2.0.1      
 [52] parallel_4.2.1         yaml_2.3.6             curl_4.3.3            
 [55] memoise_2.0.1          yulab.utils_0.0.6      sass_0.4.2            
 [58] stringi_1.7.8          RSQLite_2.2.18         highr_0.9             
 [61] S4Vectors_0.36.2       BiocGenerics_0.44.0    filelock_1.0.2        
 [64] GenomeInfoDb_1.34.9    systemfonts_1.0.4      rlang_1.1.1           
 [67] pkgconfig_2.0.3        bitops_1.0-7           evaluate_0.17         
 [70] lattice_0.20-45        labeling_0.4.2         bit_4.0.4             
 [73] tidyselect_1.2.0       magrittr_2.0.3         R6_2.5.1              
 [76] IRanges_2.32.0         generics_0.1.3         DBI_1.1.3             
 [79] pillar_1.9.0           haven_2.5.1            whisker_0.4           
 [82] withr_2.5.0            KEGGREST_1.38.0        RCurl_1.98-1.9        
 [85] modelr_0.1.9           crayon_1.5.2           utf8_1.2.2            
 [88] BiocFileCache_2.6.1    tzdb_0.3.0             rmarkdown_2.17        
 [91] progress_1.2.2         locfit_1.5-9.6         readxl_1.4.2          
 [94] blob_1.2.3             git2r_0.32.0           reprex_2.0.2          
 [97] digest_0.6.30          httpuv_1.6.6           textshaping_0.3.6     
[100] gridGraphics_0.5-1     stats4_4.2.1           munsell_0.5.0         
[103] bslib_0.4.0           </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
