<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Michael C. Saul (michael.saul [at] jax.org)" />


<title>DO Elastic Net to Expression Analysis</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>


<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Diversity Outbred Reference Traits Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">DO Elastic Net to Expression Analysis</h1>
<h4 class="author">Michael C. Saul (michael.saul [at] jax.org)</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-12-29
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rta/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20200501code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20200501)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20200501code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20200501)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong14d61ce"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 14d61ce </a>
</p>
</div>
<div id="strongRepositoryversionstrong14d61ce" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 14d61ce. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    code/.Rapp.history
    Ignored:    data/.Rhistory
    Ignored:    data/RDS/
    Ignored:    run/

Untracked files:
    Untracked:  LICENSE
    Untracked:  Untitled.R
    Untracked:  analysis/17_mapping_interpretation_old.Rmd
    Untracked:  analysis/temp.txt
    Untracked:  code/DO_eQTL_perms.R
    Untracked:  code/DO_eQTL_score_calc.R
    Untracked:  code/STAR_RSEM_416.sh
    Untracked:  code/eQTL_plotting_DO_striatum_416.R
    Untracked:  code/ggplot_pub_themes.R
    Untracked:  data/20_elastic_mv_results.csv
    Untracked:  data/20_elasticnet_predictions.RDS
    Untracked:  data/DO_416_DGEList.RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 3).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 4).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_DGEList_unfiltered.RDS
    Untracked:  data/DO_416_ERCC_DGEList.RDS
    Untracked:  data/DO_416_ERCC_cpm.RDS
    Untracked:  data/DO_416_normrt.RDS
    Untracked:  data/DO_416_pheno.RDS
    Untracked:  data/DO_416_pheno_allgenes.RDS
    Untracked:  data/DO_416_voom (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom.RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom_allgenes.RDS
    Untracked:  data/DO_416_voom_cpresiduals.RDS
    Untracked:  data/DO_AQ_active_CoV_toptable_annotated.RDS
    Untracked:  data/DO_AQ_lever_ratio_toptable_annotated.RDS
    Untracked:  data/DO_AQ_mean_inf_toptable_annotated.RDS
    Untracked:  data/DO_AQ_rate_toptable_annotated.RDS
    Untracked:  data/DO_IVSA_acquisition_cancor_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_glm.RDS
    Untracked:  data/DO_IVSA_all_cancor_df.RDS
    Untracked:  data/DO_IVSA_data.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_ALL.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_AQ.RDS
    Untracked:  data/DO_IVSA_novelty_data.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_old.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_a1_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_a2_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a4_male_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a5_female_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_raw_data.RDS
    Untracked:  data/DO_IVSA_transformed_data.RDS
    Untracked:  data/DO_LIMS_data.RDS
    Untracked:  data/DO_RTG_a1_new.qtl.RData
    Untracked:  data/DO_RTG_a1_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a1_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a2_new.qtl.RData
    Untracked:  data/DO_RTG_a2_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a2_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a3_new.qtl.RData
    Untracked:  data/DO_RTG_a3_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_01.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_02.RData
    Untracked:  data/DO_RTG_a4_new.qtl.RData
    Untracked:  data/DO_RTG_a5_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_project_df_for_mapping.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_2020-11-06.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_a2.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_sexspecific.RDS
    Untracked:  data/DO_demographic_data_df.RDS
    Untracked:  data/DO_eigengene_significance.RDS
    Untracked:  data/DO_eigengenes.RDS
    Untracked:  data/DO_highcor_cpresiduals_datatable.RDS
    Untracked:  data/DO_holeboard_data_df.RDS
    Untracked:  data/DO_holeboard_summarized.RDS
    Untracked:  data/DO_lightdark_data_df.RDS
    Untracked:  data/DO_mQTL_peaks.RDS
    Untracked:  data/DO_mQTL_scan1s.RDS
    Untracked:  data/DO_module_membership.RDS
    Untracked:  data/DO_novelplace_data_df.RDS
    Untracked:  data/DO_novelty_data.RDS
    Untracked:  data/DO_novelty_response_data.RDS
    Untracked:  data/DO_novelty_response_data_for_RTG.RDS
    Untracked:  data/DO_novelty_response_data_with_transform.csv
    Untracked:  data/DO_openfield_data_df.RDS
    Untracked:  data/DO_paraclique_data.el
    Untracked:  data/DO_paraclique_eigengenes.RDS
    Untracked:  data/DO_paracliques.RDS
    Untracked:  data/DO_paracliques_significance.RDS
    Untracked:  data/DO_ploidy_calls.RDS
    Untracked:  data/DO_pmap.RDS
    Untracked:  data/DO_pvalue_histograms.RData
    Untracked:  data/DO_rtg_a1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a3_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m3_toptable_annotated.RDS
    Untracked:  data/DO_str_2016_eQTL.RData
    Untracked:  data/DO_str_2016_gm4qtl2.zip
    Untracked:  data/DO_str_2016_gm4qtl2_intensities.fst
    Untracked:  data/DO_voom_cpresiduals_NArm.RDS
    Untracked:  data/DO_wgcna.RDS
    Untracked:  data/IVSA_RTG_a3_blup.pdf
    Untracked:  data/IVSA_h2_2019-11-18.tsv
    Untracked:  data/IVSA_metadata_sheet.RDS
    Untracked:  data/ME16_edges.tsv
    Untracked:  data/Mus_musculus.GRCm38.94.parsed.RDS
    Untracked:  data/Paraclique_19_MF_AMIGO.txt
    Untracked:  data/RTG_QTL_peaks.RDS
    Untracked:  data/RTG_figure.RDS
    Untracked:  data/SENS_RTG/
    Untracked:  data/Tyr2_genotypes.RDS
    Untracked:  data/Tyr_genotypes.RDS
    Untracked:  data/blup_chr7_DO_RTG_a3.RDS
    Untracked:  data/cc_variants/
    Untracked:  data/chr18_QTL_results.RDS
    Untracked:  data/chr4_QTL_results.RDS
    Untracked:  data/chr7_a3_QTL_results.RDS
    Untracked:  data/cis/
    Untracked:  data/eQTL_annotated_peaks.RDS
    Untracked:  data/eQTL_plots.RData
    Untracked:  data/figures/
    Untracked:  data/founders_key/
    Untracked:  data/gigamuga/
    Untracked:  data/gwas/
    Untracked:  data/info/
    Untracked:  data/m2G.a2.RTG.chr4snp.RData
    Untracked:  data/mapping_from_hao/
    Untracked:  data/perms/
    Untracked:  data/qtl2/
    Untracked:  data/raw/
    Untracked:  data/resources/
    Untracked:  data/tads_loops/
    Untracked:  data/transcripts/
    Untracked:  data/variants/
    Untracked:  logs/
    Untracked:  new_RTG_a3.jpeg
    Untracked:  output/RTG_out/
    Untracked:  output/docs_backup_2020-07-30.tar.gz
    Untracked:  output/elastic_net_out/
    Untracked:  rstudio_rta.sif
    Untracked:  singularity/
    Untracked:  tmp/
    Untracked:  var/

Unstaged changes:
    Deleted:    .Rprofile
    Modified:   .gitignore
    Modified:   README.md
    Modified:   analysis/_site.yml
    Modified:   analysis/about.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/21_DO_elasticnet_correlation.Rmd</code>) and HTML (<code>docs/21_DO_elasticnet_correlation.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
14d61ce
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-29
</td>
<td>
Committing for Sumner
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="background" class="section level1">
<h1>Background</h1>
<div id="motivation" class="section level2">
<h2>Motivation</h2>
<div id="purpose-of-analysis" class="section level3">
<h3>Purpose of analysis</h3>
<p>This script is used to correlate elastic net predictions from Robyn Ball to expression.</p>
</div>
<div id="r-libraries" class="section level3">
<h3>R libraries</h3>
<p>Calling R libraries necessary for this analysis.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;limma&quot;</span>)</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;edgeR&quot;</span>)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggplot2&quot;</span>)</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;biomaRt&quot;</span>)</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;statmod&quot;</span>)</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;qvalue&quot;</span>)</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;msaul&quot;</span>)</span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;WGCNA&quot;</span>)</span></code></pre></div>
<pre><code>Loading required package: dynamicTreeCut</code></pre>
<pre><code>Loading required package: fastcluster</code></pre>
<pre><code>
Attaching package: &#39;fastcluster&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    hclust</code></pre>
<pre><code></code></pre>
<pre><code>
Attaching package: &#39;WGCNA&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:qvalue&#39;:

    qvalue</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    cor</code></pre>
</div>
<div id="norm_rank_transform-function" class="section level3">
<h3><code>norm_rank_transform()</code> function</h3>
<p>Getting a normal rank transformation function that can handle <code>NA</code> values.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a>norm_rank_transform <span class="ot">=</span> <span class="cf">function</span>(x, <span class="at">c =</span> (<span class="dv">0</span>)) {</span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stopifnot</span>(<span class="fu">is.numeric</span>(x) <span class="sc">&amp;</span> <span class="fu">is.vector</span>(x))</span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a>  x_noNA <span class="ot">=</span> <span class="fu">which</span>(<span class="sc">!</span><span class="fu">is.na</span>(x))</span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a>  N <span class="ot">=</span> <span class="fu">length</span>(x_noNA)</span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>  x[x_noNA] <span class="ot">=</span> <span class="fu">qnorm</span>((<span class="fu">rank</span>(x[x_noNA], <span class="at">ties.method =</span> <span class="st">&quot;average&quot;</span>) <span class="sc">-</span> c) <span class="sc">/</span> (N <span class="sc">-</span> (<span class="dv">2</span> <span class="sc">*</span> c) <span class="sc">+</span> <span class="dv">1</span>))</span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(x)</span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
</div>
<div id="importing-data" class="section level3">
<h3>Importing data</h3>
<p>Importing data.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a>DO_pheno       <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_416_pheno.RDS&quot;</span>)</span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a>DO_dge         <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_416_DGEList.RDS&quot;</span>)</span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>DO_elasticnet  <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/20_elasticnet_predictions.RDS&quot;</span>)</span></code></pre></div>
</div>
<div id="reconciling-predictions-and-expression-data" class="section level3">
<h3>Reconciling predictions and expression data</h3>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>DO_pheno<span class="sc">$</span>pred_AQ_lever_ratio <span class="ot">=</span> DO_elasticnet[<span class="fu">row.names</span>(DO_pheno),<span class="st">&quot;AQ_lever_ratio&quot;</span>]</span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a>DO_pheno<span class="sc">$</span>pred_AQ_mean_inf    <span class="ot">=</span> DO_elasticnet[<span class="fu">row.names</span>(DO_pheno),<span class="st">&quot;AQ_mean_inf&quot;</span>]</span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>DO_pheno<span class="sc">$</span>pred_AQ_active_CoV  <span class="ot">=</span> DO_elasticnet[<span class="fu">row.names</span>(DO_pheno),<span class="st">&quot;AQ_active_CoV&quot;</span>]</span>
<span id="cb12-4"><a href="#cb12-4" aria-hidden="true" tabindex="-1"></a>DO_pheno<span class="sc">$</span>pred_AQ_rate        <span class="ot">=</span> DO_elasticnet[<span class="fu">row.names</span>(DO_pheno),<span class="st">&quot;AQ_rate&quot;</span>]</span>
<span id="cb12-5"><a href="#cb12-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-6"><a href="#cb12-6" aria-hidden="true" tabindex="-1"></a>has_AQ_lever_ratio_express <span class="ot">=</span> <span class="fu">row.names</span>(DO_pheno[<span class="fu">which</span>((<span class="sc">!</span><span class="fu">is.na</span>(DO_pheno[,<span class="st">&quot;pred_AQ_lever_ratio&quot;</span>])) <span class="sc">&amp;</span> (DO_pheno<span class="sc">$</span>Sex <span class="sc">==</span> <span class="st">&quot;M&quot;</span> <span class="sc">|</span> DO_pheno<span class="sc">$</span>Sex <span class="sc">==</span> <span class="st">&quot;F&quot;</span>)),])</span>
<span id="cb12-7"><a href="#cb12-7" aria-hidden="true" tabindex="-1"></a>has_AQ_mean_inf_express <span class="ot">=</span> <span class="fu">row.names</span>(DO_pheno[<span class="fu">which</span>((<span class="sc">!</span><span class="fu">is.na</span>(DO_pheno[,<span class="st">&quot;pred_AQ_mean_inf&quot;</span>])) <span class="sc">&amp;</span> (DO_pheno<span class="sc">$</span>Sex <span class="sc">==</span> <span class="st">&quot;M&quot;</span> <span class="sc">|</span> DO_pheno<span class="sc">$</span>Sex <span class="sc">==</span> <span class="st">&quot;F&quot;</span>)),])</span>
<span id="cb12-8"><a href="#cb12-8" aria-hidden="true" tabindex="-1"></a>has_AQ_active_CoV_express <span class="ot">=</span> <span class="fu">row.names</span>(DO_pheno[<span class="fu">which</span>((<span class="sc">!</span><span class="fu">is.na</span>(DO_pheno[,<span class="st">&quot;pred_AQ_active_CoV&quot;</span>])) <span class="sc">&amp;</span> (DO_pheno<span class="sc">$</span>Sex <span class="sc">==</span> <span class="st">&quot;M&quot;</span> <span class="sc">|</span> DO_pheno<span class="sc">$</span>Sex <span class="sc">==</span> <span class="st">&quot;F&quot;</span>)),])</span>
<span id="cb12-9"><a href="#cb12-9" aria-hidden="true" tabindex="-1"></a>has_AQ_rate_express <span class="ot">=</span> <span class="fu">row.names</span>(DO_pheno[<span class="fu">which</span>((<span class="sc">!</span><span class="fu">is.na</span>(DO_pheno[,<span class="st">&quot;pred_AQ_rate&quot;</span>])) <span class="sc">&amp;</span> (DO_pheno<span class="sc">$</span>Sex <span class="sc">==</span> <span class="st">&quot;M&quot;</span> <span class="sc">|</span> DO_pheno<span class="sc">$</span>Sex <span class="sc">==</span> <span class="st">&quot;F&quot;</span>)),])</span>
<span id="cb12-10"><a href="#cb12-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-11"><a href="#cb12-11" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="st">&quot;There are &quot;</span>,<span class="fu">length</span>(has_AQ_lever_ratio_express),<span class="st">&quot; samples with both predicted lever ratio and expression data.&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;There are 383 samples with both predicted lever ratio and expression data.&quot;</code></pre>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="st">&quot;There are &quot;</span>,<span class="fu">length</span>(has_AQ_mean_inf_express),<span class="st">&quot; samples with both predicted mean infusion and expression data.&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;There are 383 samples with both predicted mean infusion and expression data.&quot;</code></pre>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="st">&quot;There are &quot;</span>,<span class="fu">length</span>(has_AQ_active_CoV_express),<span class="st">&quot; samples with both preditcted CoV of IEI and expression data.&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;There are 383 samples with both preditcted CoV of IEI and expression data.&quot;</code></pre>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="st">&quot;There are &quot;</span>,<span class="fu">length</span>(has_AQ_rate_express),<span class="st">&quot; samples with both predicted rate of acquisition and expression data.&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;There are 383 samples with both predicted rate of acquisition and expression data.&quot;</code></pre>
</div>
</div>
<div id="gene-expression-analysis" class="section level2">
<h2>Gene Expression Analysis</h2>
<div id="pred_aq_lever_ratio-analysis-setup" class="section level3">
<h3><code>pred_AQ_lever_ratio</code> analysis: setup</h3>
<p>Getting factors for <code>pred_AQ_lever_ratio</code>.</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Model factors for overall eigentraits</span></span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_Generation <span class="ot">=</span> DO_pheno[has_AQ_lever_ratio_express,<span class="st">&quot;Generation&quot;</span>]</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_ChoroidPlexus <span class="ot">=</span> DO_pheno[has_AQ_lever_ratio_express,<span class="st">&quot;choroid_plexus_covariate&quot;</span>]</span>
<span id="cb20-4"><a href="#cb20-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_Sex <span class="ot">=</span> DO_pheno[has_AQ_lever_ratio_express,<span class="st">&quot;Sex&quot;</span>]</span>
<span id="cb20-5"><a href="#cb20-5" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_Predict <span class="ot">=</span> DO_pheno[has_AQ_lever_ratio_express,<span class="st">&quot;pred_AQ_lever_ratio&quot;</span>]</span>
<span id="cb20-6"><a href="#cb20-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-7"><a href="#cb20-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Gene expression list for rtg trait</span></span>
<span id="cb20-8"><a href="#cb20-8" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_dge <span class="ot">=</span> DO_dge[,has_AQ_lever_ratio_express]</span></code></pre></div>
<p>Writing model matrix for this correlation.</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_modmat <span class="ot">=</span> <span class="fu">model.matrix</span>(<span class="sc">~</span> DO_AQ_lever_ratio_ChoroidPlexus <span class="sc">+</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_lever_ratio_Sex <span class="sc">+</span> </span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_lever_ratio_Generation <span class="sc">+</span></span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_lever_ratio_Predict)</span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_AQ_lever_ratio_modmat) <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;DO_AQ_lever_ratio_&quot;</span>,<span class="st">&quot;&quot;</span>,<span class="fu">colnames</span>(DO_AQ_lever_ratio_modmat))</span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_AQ_lever_ratio_modmat)</span></code></pre></div>
<pre><code>[1] &quot;(Intercept)&quot;   &quot;ChoroidPlexus&quot; &quot;SexM&quot;          &quot;GenerationG22&quot;
[5] &quot;GenerationG23&quot; &quot;Predict&quot;      </code></pre>
<p>Fitting the model for the <code>DO_AQ_lever_ratio_modmat</code>.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_dge <span class="ot">=</span> <span class="fu">estimateGLMRobustDisp</span>(DO_AQ_lever_ratio_dge, <span class="at">design =</span> DO_AQ_lever_ratio_modmat)</span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_glmFit <span class="ot">=</span> <span class="fu">glmFit</span>(DO_AQ_lever_ratio_dge, <span class="at">design =</span> DO_AQ_lever_ratio_modmat)</span></code></pre></div>
<p>Summarizing the genes correlated with predicted IVSA data using <code>DO_AQ_lever_ratio_modmat</code>.</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_toptable <span class="ot">=</span> <span class="fu">as.data.frame</span>(<span class="fu">topTags</span>(<span class="fu">glmLRT</span>(DO_AQ_lever_ratio_glmFit,</span>
<span id="cb24-2"><a href="#cb24-2" aria-hidden="true" tabindex="-1"></a>                                                          <span class="at">coef =</span> <span class="fu">grep</span>(<span class="st">&quot;^Predict$&quot;</span>, <span class="fu">colnames</span>(DO_AQ_lever_ratio_modmat))),</span>
<span id="cb24-3"><a href="#cb24-3" aria-hidden="true" tabindex="-1"></a>                                           <span class="at">n =</span> <span class="fu">nrow</span>(DO_AQ_lever_ratio_dge)))</span>
<span id="cb24-4"><a href="#cb24-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_toptable<span class="sc">$</span>qvalue <span class="ot">=</span> <span class="fu">qvalue</span>(DO_AQ_lever_ratio_toptable<span class="sc">$</span>PValue)<span class="sc">$</span>qvalues</span>
<span id="cb24-5"><a href="#cb24-5" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_lever_ratio_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.01</span>)), <span class="st">&quot; genes correlated with lever ratio prediction at q &lt; 0.01.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;59 genes correlated with lever ratio prediction at q &lt; 0.01.&quot;</code></pre>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_lever_ratio_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.05</span>)), <span class="st">&quot; genes correlated with lever ratio prediction at q &lt; 0.05.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;314 genes correlated with lever ratio prediction at q &lt; 0.05.&quot;</code></pre>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_lever_ratio_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.10</span>)), <span class="st">&quot; genes correlated with lever ratio prediction at q &lt; 0.10.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;777 genes correlated with lever ratio prediction at q &lt; 0.10.&quot;</code></pre>
<div class="sourceCode" id="cb30"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_lever_ratio_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.20</span>)), <span class="st">&quot; genes correlated with lever ratio prediction at q &lt; 0.20.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;2069 genes correlated with lever ratio prediction at q &lt; 0.20.&quot;</code></pre>
<p>Plotting lever ratio p-value histogram.</p>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_hist <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> DO_AQ_lever_ratio_toptable, <span class="fu">aes</span>(<span class="at">x =</span> PValue)) <span class="sc">+</span></span>
<span id="cb32-2"><a href="#cb32-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_histogram</span>(<span class="at">fill =</span> <span class="st">&quot;#66CC66&quot;</span>, <span class="at">bins =</span> <span class="dv">50</span>) <span class="sc">+</span></span>
<span id="cb32-3"><a href="#cb32-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb32-4"><a href="#cb32-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_hist</span></code></pre></div>
<p><img src="figure/21_DO_elasticnet_correlation.Rmd/unnamed-chunk-9-1.png" style="display: block; margin: auto;" /></p>
</div>
<div id="pred_aq_mean_inf-analysis-setup" class="section level3">
<h3><code>pred_AQ_mean_inf</code> analysis: setup</h3>
<p>Getting factors for <code>pred_AQ_mean_inf</code>.</p>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Model factors</span></span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_Generation <span class="ot">=</span> DO_pheno[has_AQ_mean_inf_express,<span class="st">&quot;Generation&quot;</span>]</span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_ChoroidPlexus <span class="ot">=</span> DO_pheno[has_AQ_mean_inf_express,<span class="st">&quot;choroid_plexus_covariate&quot;</span>]</span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_Sex <span class="ot">=</span> DO_pheno[has_AQ_mean_inf_express,<span class="st">&quot;Sex&quot;</span>]</span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_Predict <span class="ot">=</span> DO_pheno[has_AQ_mean_inf_express,<span class="st">&quot;pred_AQ_mean_inf&quot;</span>]</span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-7"><a href="#cb33-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Gene expression list for trait</span></span>
<span id="cb33-8"><a href="#cb33-8" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_dge <span class="ot">=</span> DO_dge[,has_AQ_mean_inf_express]</span></code></pre></div>
<p>Writing model matrix for this correlation.</p>
<div class="sourceCode" id="cb34"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb34-1"><a href="#cb34-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_modmat <span class="ot">=</span> <span class="fu">model.matrix</span>(<span class="sc">~</span> DO_AQ_mean_inf_ChoroidPlexus <span class="sc">+</span></span>
<span id="cb34-2"><a href="#cb34-2" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_mean_inf_Sex <span class="sc">+</span> </span>
<span id="cb34-3"><a href="#cb34-3" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_mean_inf_Generation <span class="sc">+</span></span>
<span id="cb34-4"><a href="#cb34-4" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_mean_inf_Predict)</span>
<span id="cb34-5"><a href="#cb34-5" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_AQ_mean_inf_modmat) <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;DO_AQ_mean_inf_&quot;</span>,<span class="st">&quot;&quot;</span>,<span class="fu">colnames</span>(DO_AQ_mean_inf_modmat))</span>
<span id="cb34-6"><a href="#cb34-6" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_AQ_mean_inf_modmat)</span></code></pre></div>
<pre><code>[1] &quot;(Intercept)&quot;   &quot;ChoroidPlexus&quot; &quot;SexM&quot;          &quot;GenerationG22&quot;
[5] &quot;GenerationG23&quot; &quot;Predict&quot;      </code></pre>
<p>Fitting the model for the <code>DO_AQ_mean_inf_modmat</code>.</p>
<div class="sourceCode" id="cb36"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_dge <span class="ot">=</span> <span class="fu">estimateGLMRobustDisp</span>(DO_AQ_mean_inf_dge, <span class="at">design =</span> DO_AQ_mean_inf_modmat)</span>
<span id="cb36-2"><a href="#cb36-2" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_glmFit <span class="ot">=</span> <span class="fu">glmFit</span>(DO_AQ_mean_inf_dge, <span class="at">design =</span> DO_AQ_mean_inf_modmat)</span></code></pre></div>
<p>Summarizing the genes correlated with predicted IVSA data using <code>DO_AQ_mean_inf_modmat</code>.</p>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_toptable <span class="ot">=</span> <span class="fu">as.data.frame</span>(<span class="fu">topTags</span>(<span class="fu">glmLRT</span>(DO_AQ_mean_inf_glmFit,</span>
<span id="cb37-2"><a href="#cb37-2" aria-hidden="true" tabindex="-1"></a>                                                          <span class="at">coef =</span> <span class="fu">grep</span>(<span class="st">&quot;^Predict$&quot;</span>, <span class="fu">colnames</span>(DO_AQ_mean_inf_modmat))),</span>
<span id="cb37-3"><a href="#cb37-3" aria-hidden="true" tabindex="-1"></a>                                           <span class="at">n =</span> <span class="fu">nrow</span>(DO_AQ_mean_inf_dge)))</span>
<span id="cb37-4"><a href="#cb37-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_toptable<span class="sc">$</span>qvalue <span class="ot">=</span> <span class="fu">qvalue</span>(DO_AQ_mean_inf_toptable<span class="sc">$</span>PValue)<span class="sc">$</span>qvalues</span>
<span id="cb37-5"><a href="#cb37-5" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_mean_inf_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.01</span>)), <span class="st">&quot; genes correlated with mean infusion prediction at q &lt; 0.01.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;55 genes correlated with mean infusion prediction at q &lt; 0.01.&quot;</code></pre>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_mean_inf_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.05</span>)), <span class="st">&quot; genes correlated with mean infusion prediction at q &lt; 0.05.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;328 genes correlated with mean infusion prediction at q &lt; 0.05.&quot;</code></pre>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb41-1"><a href="#cb41-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_mean_inf_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.10</span>)), <span class="st">&quot; genes correlated with mean infusion prediction at q &lt; 0.10.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;743 genes correlated with mean infusion prediction at q &lt; 0.10.&quot;</code></pre>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_mean_inf_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.20</span>)), <span class="st">&quot; genes correlated with mean infusion prediction at q &lt; 0.20.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;1910 genes correlated with mean infusion prediction at q &lt; 0.20.&quot;</code></pre>
<p>Plotting mean infusion p-value histogram.</p>
<div class="sourceCode" id="cb45"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_hist <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> DO_AQ_mean_inf_toptable, <span class="fu">aes</span>(<span class="at">x =</span> PValue)) <span class="sc">+</span></span>
<span id="cb45-2"><a href="#cb45-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_histogram</span>(<span class="at">fill =</span> <span class="st">&quot;#66CC66&quot;</span>, <span class="at">bins =</span> <span class="dv">50</span>) <span class="sc">+</span></span>
<span id="cb45-3"><a href="#cb45-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb45-4"><a href="#cb45-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_hist</span></code></pre></div>
<p><img src="figure/21_DO_elasticnet_correlation.Rmd/unnamed-chunk-14-1.png" style="display: block; margin: auto;" /></p>
</div>
<div id="pred_aq_active_cov-analysis-setup" class="section level3">
<h3><code>pred_AQ_active_CoV</code> analysis: setup</h3>
<p>Getting factors for <code>pred_AQ_active_CoV</code>.</p>
<div class="sourceCode" id="cb46"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Model factors for overall eigentraits</span></span>
<span id="cb46-2"><a href="#cb46-2" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_Generation <span class="ot">=</span> DO_pheno[has_AQ_active_CoV_express,<span class="st">&quot;Generation&quot;</span>]</span>
<span id="cb46-3"><a href="#cb46-3" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_ChoroidPlexus <span class="ot">=</span> DO_pheno[has_AQ_active_CoV_express,<span class="st">&quot;choroid_plexus_covariate&quot;</span>]</span>
<span id="cb46-4"><a href="#cb46-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_Sex <span class="ot">=</span> DO_pheno[has_AQ_active_CoV_express,<span class="st">&quot;Sex&quot;</span>]</span>
<span id="cb46-5"><a href="#cb46-5" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_Predict <span class="ot">=</span> DO_pheno[has_AQ_active_CoV_express,<span class="st">&quot;pred_AQ_active_CoV&quot;</span>]</span>
<span id="cb46-6"><a href="#cb46-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-7"><a href="#cb46-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Gene expression list for rtg trait</span></span>
<span id="cb46-8"><a href="#cb46-8" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_dge <span class="ot">=</span> DO_dge[,has_AQ_active_CoV_express]</span></code></pre></div>
<p>Writing model matrix for this correlation.</p>
<div class="sourceCode" id="cb47"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb47-1"><a href="#cb47-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_modmat <span class="ot">=</span> <span class="fu">model.matrix</span>(<span class="sc">~</span> DO_AQ_active_CoV_ChoroidPlexus <span class="sc">+</span></span>
<span id="cb47-2"><a href="#cb47-2" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_active_CoV_Sex <span class="sc">+</span> </span>
<span id="cb47-3"><a href="#cb47-3" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_active_CoV_Generation <span class="sc">+</span></span>
<span id="cb47-4"><a href="#cb47-4" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_active_CoV_Predict)</span>
<span id="cb47-5"><a href="#cb47-5" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_AQ_active_CoV_modmat) <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;DO_AQ_active_CoV_&quot;</span>,<span class="st">&quot;&quot;</span>,<span class="fu">colnames</span>(DO_AQ_active_CoV_modmat))</span>
<span id="cb47-6"><a href="#cb47-6" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_AQ_active_CoV_modmat)</span></code></pre></div>
<pre><code>[1] &quot;(Intercept)&quot;   &quot;ChoroidPlexus&quot; &quot;SexM&quot;          &quot;GenerationG22&quot;
[5] &quot;GenerationG23&quot; &quot;Predict&quot;      </code></pre>
<p>Fitting the model for the <code>DO_AQ_active_CoV_modmat</code>.</p>
<div class="sourceCode" id="cb49"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb49-1"><a href="#cb49-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_dge <span class="ot">=</span> <span class="fu">estimateGLMRobustDisp</span>(DO_AQ_active_CoV_dge, <span class="at">design =</span> DO_AQ_active_CoV_modmat)</span>
<span id="cb49-2"><a href="#cb49-2" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_glmFit <span class="ot">=</span> <span class="fu">glmFit</span>(DO_AQ_active_CoV_dge, <span class="at">design =</span> DO_AQ_active_CoV_modmat)</span></code></pre></div>
<p>Summarizing the genes correlated with predicted IVSA data using <code>DO_AQ_active_CoV_modmat</code>.</p>
<div class="sourceCode" id="cb50"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb50-1"><a href="#cb50-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_toptable <span class="ot">=</span> <span class="fu">as.data.frame</span>(<span class="fu">topTags</span>(<span class="fu">glmLRT</span>(DO_AQ_active_CoV_glmFit,</span>
<span id="cb50-2"><a href="#cb50-2" aria-hidden="true" tabindex="-1"></a>                                                          <span class="at">coef =</span> <span class="fu">grep</span>(<span class="st">&quot;^Predict$&quot;</span>, <span class="fu">colnames</span>(DO_AQ_active_CoV_modmat))),</span>
<span id="cb50-3"><a href="#cb50-3" aria-hidden="true" tabindex="-1"></a>                                           <span class="at">n =</span> <span class="fu">nrow</span>(DO_AQ_active_CoV_dge)))</span>
<span id="cb50-4"><a href="#cb50-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_toptable<span class="sc">$</span>qvalue <span class="ot">=</span> <span class="fu">qvalue</span>(DO_AQ_active_CoV_toptable<span class="sc">$</span>PValue)<span class="sc">$</span>qvalues</span>
<span id="cb50-5"><a href="#cb50-5" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_active_CoV_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.01</span>)), <span class="st">&quot; genes correlated with IEI CoV prediction at q &lt; 0.01.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;56 genes correlated with IEI CoV prediction at q &lt; 0.01.&quot;</code></pre>
<div class="sourceCode" id="cb52"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb52-1"><a href="#cb52-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_active_CoV_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.05</span>)), <span class="st">&quot; genes correlated with IEI CoV prediction at q &lt; 0.05.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;304 genes correlated with IEI CoV prediction at q &lt; 0.05.&quot;</code></pre>
<div class="sourceCode" id="cb54"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb54-1"><a href="#cb54-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_active_CoV_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.10</span>)), <span class="st">&quot; genes correlated with IEI CoV prediction at q &lt; 0.10.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;723 genes correlated with IEI CoV prediction at q &lt; 0.10.&quot;</code></pre>
<div class="sourceCode" id="cb56"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb56-1"><a href="#cb56-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_active_CoV_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.20</span>)), <span class="st">&quot; genes correlated with IEI CoV prediction at q &lt; 0.20.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;2002 genes correlated with IEI CoV prediction at q &lt; 0.20.&quot;</code></pre>
<p>Plotting lever ratio p-value histogram.</p>
<div class="sourceCode" id="cb58"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb58-1"><a href="#cb58-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_hist <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> DO_AQ_active_CoV_toptable, <span class="fu">aes</span>(<span class="at">x =</span> PValue)) <span class="sc">+</span></span>
<span id="cb58-2"><a href="#cb58-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_histogram</span>(<span class="at">fill =</span> <span class="st">&quot;#66CC66&quot;</span>, <span class="at">bins =</span> <span class="dv">50</span>) <span class="sc">+</span></span>
<span id="cb58-3"><a href="#cb58-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb58-4"><a href="#cb58-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_hist</span></code></pre></div>
<p><img src="figure/21_DO_elasticnet_correlation.Rmd/unnamed-chunk-19-1.png" style="display: block; margin: auto;" /></p>
</div>
<div id="pred_aq_rate-analysis-setup" class="section level3">
<h3><code>pred_AQ_rate</code> analysis: setup</h3>
<p>Getting factors for <code>pred_AQ_rate</code>.</p>
<div class="sourceCode" id="cb59"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb59-1"><a href="#cb59-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Model factors for overall eigentraits</span></span>
<span id="cb59-2"><a href="#cb59-2" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_Generation <span class="ot">=</span> DO_pheno[has_AQ_rate_express,<span class="st">&quot;Generation&quot;</span>]</span>
<span id="cb59-3"><a href="#cb59-3" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_ChoroidPlexus <span class="ot">=</span> DO_pheno[has_AQ_rate_express,<span class="st">&quot;choroid_plexus_covariate&quot;</span>]</span>
<span id="cb59-4"><a href="#cb59-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_Sex <span class="ot">=</span> DO_pheno[has_AQ_rate_express,<span class="st">&quot;Sex&quot;</span>]</span>
<span id="cb59-5"><a href="#cb59-5" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_Predict <span class="ot">=</span> DO_pheno[has_AQ_rate_express,<span class="st">&quot;pred_AQ_rate&quot;</span>]</span>
<span id="cb59-6"><a href="#cb59-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb59-7"><a href="#cb59-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Gene expression list for rtg trait</span></span>
<span id="cb59-8"><a href="#cb59-8" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_dge <span class="ot">=</span> DO_dge[,has_AQ_rate_express]</span></code></pre></div>
<p>Writing model matrix for this correlation.</p>
<div class="sourceCode" id="cb60"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb60-1"><a href="#cb60-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_modmat <span class="ot">=</span> <span class="fu">model.matrix</span>(<span class="sc">~</span> DO_AQ_rate_ChoroidPlexus <span class="sc">+</span></span>
<span id="cb60-2"><a href="#cb60-2" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_rate_Sex <span class="sc">+</span> </span>
<span id="cb60-3"><a href="#cb60-3" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_rate_Generation <span class="sc">+</span></span>
<span id="cb60-4"><a href="#cb60-4" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_rate_Predict)</span>
<span id="cb60-5"><a href="#cb60-5" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_AQ_rate_modmat) <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;DO_AQ_rate_&quot;</span>,<span class="st">&quot;&quot;</span>,<span class="fu">colnames</span>(DO_AQ_rate_modmat))</span>
<span id="cb60-6"><a href="#cb60-6" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_AQ_rate_modmat)</span></code></pre></div>
<pre><code>[1] &quot;(Intercept)&quot;   &quot;ChoroidPlexus&quot; &quot;SexM&quot;          &quot;GenerationG22&quot;
[5] &quot;GenerationG23&quot; &quot;Predict&quot;      </code></pre>
<p>Fitting the model for the <code>DO_AQ_rate_modmat</code>.</p>
<div class="sourceCode" id="cb62"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb62-1"><a href="#cb62-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_dge <span class="ot">=</span> <span class="fu">estimateGLMRobustDisp</span>(DO_AQ_rate_dge, <span class="at">design =</span> DO_AQ_rate_modmat)</span>
<span id="cb62-2"><a href="#cb62-2" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_glmFit <span class="ot">=</span> <span class="fu">glmFit</span>(DO_AQ_rate_dge, <span class="at">design =</span> DO_AQ_rate_modmat)</span></code></pre></div>
<p>Summarizing the genes correlated with predicted IVSA data using <code>DO_AQ_rate_modmat</code>.</p>
<div class="sourceCode" id="cb63"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb63-1"><a href="#cb63-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_toptable <span class="ot">=</span> <span class="fu">as.data.frame</span>(<span class="fu">topTags</span>(<span class="fu">glmLRT</span>(DO_AQ_rate_glmFit,</span>
<span id="cb63-2"><a href="#cb63-2" aria-hidden="true" tabindex="-1"></a>                                                          <span class="at">coef =</span> <span class="fu">grep</span>(<span class="st">&quot;^Predict$&quot;</span>, <span class="fu">colnames</span>(DO_AQ_rate_modmat))),</span>
<span id="cb63-3"><a href="#cb63-3" aria-hidden="true" tabindex="-1"></a>                                           <span class="at">n =</span> <span class="fu">nrow</span>(DO_AQ_rate_dge)))</span>
<span id="cb63-4"><a href="#cb63-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_toptable<span class="sc">$</span>qvalue <span class="ot">=</span> <span class="fu">qvalue</span>(DO_AQ_rate_toptable<span class="sc">$</span>PValue)<span class="sc">$</span>qvalues</span>
<span id="cb63-5"><a href="#cb63-5" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_rate_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.01</span>)), <span class="st">&quot; genes correlated with acquisition rate prediction at q &lt; 0.01.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;59 genes correlated with acquisition rate prediction at q &lt; 0.01.&quot;</code></pre>
<div class="sourceCode" id="cb65"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb65-1"><a href="#cb65-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_rate_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.05</span>)), <span class="st">&quot; genes correlated with acquisition rate prediction at q &lt; 0.05.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;299 genes correlated with acquisition rate prediction at q &lt; 0.05.&quot;</code></pre>
<div class="sourceCode" id="cb67"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb67-1"><a href="#cb67-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_rate_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.10</span>)), <span class="st">&quot; genes correlated with acquisition rate prediction at q &lt; 0.10.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;736 genes correlated with acquisition rate prediction at q &lt; 0.10.&quot;</code></pre>
<div class="sourceCode" id="cb69"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb69-1"><a href="#cb69-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="fu">length</span>(<span class="fu">which</span>(DO_AQ_rate_toptable<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.20</span>)), <span class="st">&quot; genes correlated with acquisition rate prediction at q &lt; 0.20.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;1908 genes correlated with acquisition rate prediction at q &lt; 0.20.&quot;</code></pre>
<p>Plotting acquisition rate p-value histogram.</p>
<div class="sourceCode" id="cb71"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb71-1"><a href="#cb71-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_hist <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> DO_AQ_rate_toptable, <span class="fu">aes</span>(<span class="at">x =</span> PValue)) <span class="sc">+</span></span>
<span id="cb71-2"><a href="#cb71-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_histogram</span>(<span class="at">fill =</span> <span class="st">&quot;#66CC66&quot;</span>, <span class="at">bins =</span> <span class="dv">50</span>) <span class="sc">+</span></span>
<span id="cb71-3"><a href="#cb71-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb71-4"><a href="#cb71-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_hist</span></code></pre></div>
<p><img src="figure/21_DO_elasticnet_correlation.Rmd/unnamed-chunk-24-1.png" style="display: block; margin: auto;" /></p>
</div>
<div id="annotating-toptables" class="section level3">
<h3>Annotating toptables</h3>
<p>Using <code>biomaRt</code> to annotate the toptables.</p>
<div class="sourceCode" id="cb72"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb72-1"><a href="#cb72-1" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_genes <span class="ot">=</span> <span class="fu">row.names</span>(DO_AQ_lever_ratio_toptable)</span>
<span id="cb72-2"><a href="#cb72-2" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_genes <span class="ot">=</span> <span class="fu">row.names</span>(DO_AQ_mean_inf_toptable)</span>
<span id="cb72-3"><a href="#cb72-3" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_genes <span class="ot">=</span> <span class="fu">row.names</span>(DO_AQ_active_CoV_toptable)</span>
<span id="cb72-4"><a href="#cb72-4" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_genes <span class="ot">=</span> <span class="fu">row.names</span>(DO_AQ_rate_toptable)</span>
<span id="cb72-5"><a href="#cb72-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-6"><a href="#cb72-6" aria-hidden="true" tabindex="-1"></a>DO_elasticnet_genes <span class="ot">=</span> <span class="fu">unique</span>(<span class="fu">c</span>(DO_AQ_lever_ratio_genes,</span>
<span id="cb72-7"><a href="#cb72-7" aria-hidden="true" tabindex="-1"></a>                               DO_AQ_mean_inf_genes,</span>
<span id="cb72-8"><a href="#cb72-8" aria-hidden="true" tabindex="-1"></a>                               DO_AQ_active_CoV_genes,</span>
<span id="cb72-9"><a href="#cb72-9" aria-hidden="true" tabindex="-1"></a>                               DO_AQ_rate_genes))</span>
<span id="cb72-10"><a href="#cb72-10" aria-hidden="true" tabindex="-1"></a>                      </span>
<span id="cb72-11"><a href="#cb72-11" aria-hidden="true" tabindex="-1"></a>maRt <span class="ot">=</span> <span class="fu">useMart</span>(<span class="at">biomart =</span> <span class="st">&quot;ENSEMBL_MART_ENSEMBL&quot;</span>,</span>
<span id="cb72-12"><a href="#cb72-12" aria-hidden="true" tabindex="-1"></a>               <span class="at">host =</span> <span class="st">&quot;oct2018.archive.ensembl.org&quot;</span>,</span>
<span id="cb72-13"><a href="#cb72-13" aria-hidden="true" tabindex="-1"></a>               <span class="at">dataset =</span> <span class="st">&quot;mmusculus_gene_ensembl&quot;</span>)</span></code></pre></div>
<pre><code>Warning: Ensembl will soon enforce the use of https.
Ensure the &#39;host&#39; argument includes &quot;https://&quot;</code></pre>
<div class="sourceCode" id="cb74"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb74-1"><a href="#cb74-1" aria-hidden="true" tabindex="-1"></a>maRt_filter <span class="ot">=</span> <span class="st">&quot;ensembl_gene_id&quot;</span></span>
<span id="cb74-2"><a href="#cb74-2" aria-hidden="true" tabindex="-1"></a>maRt_attributes <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;mgi_symbol&quot;</span>,<span class="st">&quot;mgi_description&quot;</span>,<span class="st">&quot;mgi_id&quot;</span>,<span class="st">&quot;chromosome_name&quot;</span>,<span class="st">&quot;start_position&quot;</span>,</span>
<span id="cb74-3"><a href="#cb74-3" aria-hidden="true" tabindex="-1"></a>                    <span class="st">&quot;end_position&quot;</span>,<span class="st">&quot;strand&quot;</span>,<span class="st">&quot;band&quot;</span>,<span class="st">&quot;ensembl_gene_id&quot;</span>)</span>
<span id="cb74-4"><a href="#cb74-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb74-5"><a href="#cb74-5" aria-hidden="true" tabindex="-1"></a>maRt_annotation_raw <span class="ot">=</span> <span class="fu">getBM</span>(maRt_attributes, maRt_filter, DO_elasticnet_genes, maRt)</span>
<span id="cb74-6"><a href="#cb74-6" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(maRt_annotation_raw) <span class="ot">=</span> <span class="fu">as.character</span>(maRt_annotation_raw<span class="sc">$</span>ensembl_gene_id)</span>
<span id="cb74-7"><a href="#cb74-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb74-8"><a href="#cb74-8" aria-hidden="true" tabindex="-1"></a><span class="co"># AQ_lever_ratio</span></span>
<span id="cb74-9"><a href="#cb74-9" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_toptable_annotated <span class="ot">=</span> <span class="fu">cbind</span>(maRt_annotation_raw[DO_AQ_lever_ratio_genes,],</span>
<span id="cb74-10"><a href="#cb74-10" aria-hidden="true" tabindex="-1"></a>                                             DO_AQ_lever_ratio_toptable)</span>
<span id="cb74-11"><a href="#cb74-11" aria-hidden="true" tabindex="-1"></a>DO_AQ_lever_ratio_toptable_annotated<span class="sc">$</span>ensembl_gene_id <span class="ot">=</span> DO_AQ_lever_ratio_genes; <span class="fu">row.names</span>(DO_AQ_lever_ratio_toptable_annotated) <span class="ot">=</span> DO_AQ_lever_ratio_genes</span>
<span id="cb74-12"><a href="#cb74-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb74-13"><a href="#cb74-13" aria-hidden="true" tabindex="-1"></a><span class="co"># AQ_mean_inf</span></span>
<span id="cb74-14"><a href="#cb74-14" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_toptable_annotated <span class="ot">=</span> <span class="fu">cbind</span>(maRt_annotation_raw[DO_AQ_mean_inf_genes,],</span>
<span id="cb74-15"><a href="#cb74-15" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_mean_inf_toptable)</span>
<span id="cb74-16"><a href="#cb74-16" aria-hidden="true" tabindex="-1"></a>DO_AQ_mean_inf_toptable_annotated<span class="sc">$</span>ensembl_gene_id <span class="ot">=</span> DO_AQ_mean_inf_genes; <span class="fu">row.names</span>(DO_AQ_mean_inf_toptable_annotated) <span class="ot">=</span> DO_AQ_mean_inf_genes</span>
<span id="cb74-17"><a href="#cb74-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb74-18"><a href="#cb74-18" aria-hidden="true" tabindex="-1"></a><span class="co"># AQ_active_CoV</span></span>
<span id="cb74-19"><a href="#cb74-19" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_toptable_annotated <span class="ot">=</span> <span class="fu">cbind</span>(maRt_annotation_raw[DO_AQ_active_CoV_genes,],</span>
<span id="cb74-20"><a href="#cb74-20" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_active_CoV_toptable)</span>
<span id="cb74-21"><a href="#cb74-21" aria-hidden="true" tabindex="-1"></a>DO_AQ_active_CoV_toptable_annotated<span class="sc">$</span>ensembl_gene_id <span class="ot">=</span> DO_AQ_active_CoV_genes; <span class="fu">row.names</span>(DO_AQ_active_CoV_toptable_annotated) <span class="ot">=</span> DO_AQ_active_CoV_genes</span>
<span id="cb74-22"><a href="#cb74-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb74-23"><a href="#cb74-23" aria-hidden="true" tabindex="-1"></a><span class="co"># AQ_rate</span></span>
<span id="cb74-24"><a href="#cb74-24" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_toptable_annotated <span class="ot">=</span> <span class="fu">cbind</span>(maRt_annotation_raw[DO_AQ_rate_genes,],</span>
<span id="cb74-25"><a href="#cb74-25" aria-hidden="true" tabindex="-1"></a>                                          DO_AQ_rate_toptable)</span>
<span id="cb74-26"><a href="#cb74-26" aria-hidden="true" tabindex="-1"></a>DO_AQ_rate_toptable_annotated<span class="sc">$</span>ensembl_gene_id <span class="ot">=</span> DO_AQ_rate_genes; <span class="fu">row.names</span>(DO_AQ_rate_toptable_annotated) <span class="ot">=</span> DO_AQ_rate_genes</span></code></pre></div>
</div>
<div id="paraclique-analysis" class="section level3">
<h3>Paraclique Analysis</h3>
<p>Evaluating relationship between elastic net regression predictions and paraclique analyses.</p>
<div class="sourceCode" id="cb75"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb75-1"><a href="#cb75-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Loading paracliques</span></span>
<span id="cb75-2"><a href="#cb75-2" aria-hidden="true" tabindex="-1"></a>DO_paraclique_eigengenes <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_paraclique_eigengenes.RDS&quot;</span>)</span>
<span id="cb75-3"><a href="#cb75-3" aria-hidden="true" tabindex="-1"></a>DO_paracliques <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_paracliques.RDS&quot;</span>)</span>
<span id="cb75-4"><a href="#cb75-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb75-5"><a href="#cb75-5" aria-hidden="true" tabindex="-1"></a>DO_phenotypes <span class="ot">=</span> <span class="fu">colnames</span>(DO_pheno)[<span class="fu">grep</span>(<span class="st">&quot;AQ&quot;</span>, <span class="fu">colnames</span>(DO_pheno))]</span>
<span id="cb75-6"><a href="#cb75-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb75-7"><a href="#cb75-7" aria-hidden="true" tabindex="-1"></a>DO_paraclique_significance <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="fu">matrix</span>(<span class="at">nrow =</span> (<span class="fu">ncol</span>(DO_paraclique_eigengenes) <span class="sc">-</span> <span class="dv">1</span>), <span class="at">ncol =</span> <span class="dv">1</span> <span class="sc">+</span> (<span class="dv">2</span> <span class="sc">*</span> <span class="dv">4</span>)))</span>
<span id="cb75-8"><a href="#cb75-8" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(DO_paraclique_significance) <span class="ot">=</span> <span class="fu">colnames</span>(DO_paraclique_eigengenes)[<span class="sc">-</span><span class="dv">1</span>]</span>
<span id="cb75-9"><a href="#cb75-9" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(DO_paraclique_significance) <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;paraclique&quot;</span>,</span>
<span id="cb75-10"><a href="#cb75-10" aria-hidden="true" tabindex="-1"></a>                                         <span class="fu">paste</span>(DO_phenotypes,<span class="st">&quot;_bicor_p&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>),</span>
<span id="cb75-11"><a href="#cb75-11" aria-hidden="true" tabindex="-1"></a>                                         <span class="fu">paste</span>(DO_phenotypes,<span class="st">&quot;_overlap_p&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>))</span>
<span id="cb75-12"><a href="#cb75-12" aria-hidden="true" tabindex="-1"></a>DO_paraclique_significance<span class="sc">$</span>paraclique <span class="ot">=</span> <span class="fu">colnames</span>(DO_paraclique_eigengenes)[<span class="sc">-</span><span class="dv">1</span>]</span>
<span id="cb75-13"><a href="#cb75-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb75-14"><a href="#cb75-14" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(DO_paraclique_significance)) {</span>
<span id="cb75-15"><a href="#cb75-15" aria-hidden="true" tabindex="-1"></a>  paraclique_i <span class="ot">=</span> DO_paraclique_significance[i,<span class="st">&quot;paraclique&quot;</span>]</span>
<span id="cb75-16"><a href="#cb75-16" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (j <span class="cf">in</span> DO_phenotypes) {</span>
<span id="cb75-17"><a href="#cb75-17" aria-hidden="true" tabindex="-1"></a>    col_j <span class="ot">=</span> <span class="fu">paste</span>(j,<span class="st">&quot;_bicor_p&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)</span>
<span id="cb75-18"><a href="#cb75-18" aria-hidden="true" tabindex="-1"></a>    DO_paraclique_significance[i,col_j] <span class="ot">=</span> <span class="fu">bicorAndPvalue</span>(DO_paraclique_eigengenes[,paraclique_i],</span>
<span id="cb75-19"><a href="#cb75-19" aria-hidden="true" tabindex="-1"></a>                                                         DO_pheno[,j],</span>
<span id="cb75-20"><a href="#cb75-20" aria-hidden="true" tabindex="-1"></a>                                                         <span class="at">use =</span> <span class="st">&quot;pairwise.complete.obs&quot;</span>)<span class="sc">$</span>p[<span class="dv">1</span>,<span class="dv">1</span>]</span>
<span id="cb75-21"><a href="#cb75-21" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb75-22"><a href="#cb75-22" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb75-23"><a href="#cb75-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb75-24"><a href="#cb75-24" aria-hidden="true" tabindex="-1"></a>MSET_B <span class="ot">=</span> <span class="fl">1e6</span></span>
<span id="cb75-25"><a href="#cb75-25" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> DO_phenotypes) {</span>
<span id="cb75-26"><a href="#cb75-26" aria-hidden="true" tabindex="-1"></a>  col_i <span class="ot">=</span> <span class="fu">paste</span>(i,<span class="st">&quot;_overlap_p&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)</span>
<span id="cb75-27"><a href="#cb75-27" aria-hidden="true" tabindex="-1"></a>  toptable_i <span class="ot">=</span> <span class="fu">eval</span>(<span class="fu">parse</span>(<span class="at">text =</span> <span class="fu">paste</span>(<span class="st">&quot;DO_&quot;</span>,<span class="fu">gsub</span>(<span class="st">&quot;^pred_&quot;</span>, <span class="st">&quot;&quot;</span>, i),<span class="st">&quot;_toptable&quot;</span>,<span class="at">sep=</span><span class="st">&quot;&quot;</span>)))</span>
<span id="cb75-28"><a href="#cb75-28" aria-hidden="true" tabindex="-1"></a>  sig_genes_i <span class="ot">=</span> <span class="fu">row.names</span>(toptable_i[<span class="fu">which</span>(toptable_i<span class="sc">$</span>qvalue <span class="sc">&lt;</span> <span class="fl">0.10</span>),])</span>
<span id="cb75-29"><a href="#cb75-29" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (j <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(DO_paracliques)) {</span>
<span id="cb75-30"><a href="#cb75-30" aria-hidden="true" tabindex="-1"></a>    paraclique_genes_j <span class="ot">=</span>  DO_paracliques[[j]]</span>
<span id="cb75-31"><a href="#cb75-31" aria-hidden="true" tabindex="-1"></a>    DO_paraclique_significance[j,col_i] <span class="ot">=</span> <span class="fu">MSET.overlap.test</span>(sig_genes_i,</span>
<span id="cb75-32"><a href="#cb75-32" aria-hidden="true" tabindex="-1"></a>                                                            paraclique_genes_j,</span>
<span id="cb75-33"><a href="#cb75-33" aria-hidden="true" tabindex="-1"></a>                                                            <span class="fu">row.names</span>(toptable_i),</span>
<span id="cb75-34"><a href="#cb75-34" aria-hidden="true" tabindex="-1"></a>                                                            <span class="fu">row.names</span>(DO_dge), </span>
<span id="cb75-35"><a href="#cb75-35" aria-hidden="true" tabindex="-1"></a>                                                            <span class="at">B =</span> MSET_B)<span class="sc">$</span>p.value <span class="sc">+</span> (<span class="dv">1</span><span class="sc">/</span>MSET_B)</span>
<span id="cb75-36"><a href="#cb75-36" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb75-37"><a href="#cb75-37" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
</div>
<div id="output" class="section level3">
<h3>Output</h3>
<p>Saving the annotated toptables.</p>
<div class="sourceCode" id="cb76"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb76-1"><a href="#cb76-1" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_AQ_lever_ratio_toptable_annotated,<span class="st">&quot;./data/DO_AQ_lever_ratio_toptable_annotated.RDS&quot;</span>)</span>
<span id="cb76-2"><a href="#cb76-2" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_AQ_mean_inf_toptable_annotated,<span class="st">&quot;./data/DO_AQ_mean_inf_toptable_annotated.RDS&quot;</span>)</span>
<span id="cb76-3"><a href="#cb76-3" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_AQ_active_CoV_toptable_annotated,<span class="st">&quot;./data/DO_AQ_active_CoV_toptable_annotated.RDS&quot;</span>)</span>
<span id="cb76-4"><a href="#cb76-4" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_AQ_rate_toptable_annotated,<span class="st">&quot;./data/DO_AQ_rate_toptable_annotated.RDS&quot;</span>)</span>
<span id="cb76-5"><a href="#cb76-5" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(DO_paraclique_significance,<span class="st">&quot;./data/DO_paraclique_significance_elasticnet.RDS&quot;</span>)</span></code></pre></div>
<p>Saving the histograms.</p>
<div class="sourceCode" id="cb77"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb77-1"><a href="#cb77-1" aria-hidden="true" tabindex="-1"></a><span class="fu">save</span>(<span class="at">list =</span> <span class="fu">ls</span>(<span class="at">pattern =</span> <span class="st">&quot;hist$&quot;</span>), <span class="at">file =</span> <span class="st">&quot;./data/DO_elasticnet_GEX_pvalue_histograms.RData&quot;</span>)</span></code></pre></div>
<p>The output data are saved in <code>./data/</code>.</p>
</div>
<div id="document-control" class="section level3">
<h3>Document Control</h3>
<p>This document was prepared using RMarkdown in RStudio.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb78"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb78-1"><a href="#cb78-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.3 LTS

Matrix products: default
BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] WGCNA_1.70-3          fastcluster_1.2.3     dynamicTreeCut_1.63-1
 [4] msaul_1.16.6          qvalue_2.26.0         statmod_1.4.36       
 [7] biomaRt_2.50.1        ggplot2_3.3.5         edgeR_3.36.0         
[10] limma_3.50.0         

loaded via a namespace (and not attached):
  [1] colorspace_2.0-2       ellipsis_0.3.2         rprojroot_2.0.2       
  [4] htmlTable_2.3.0        XVector_0.34.0         base64enc_0.1-3       
  [7] fs_1.5.2               rstudioapi_0.13        farver_2.1.0          
 [10] bit64_4.0.5            AnnotationDbi_1.56.2   fansi_0.5.0           
 [13] xml2_1.3.3             codetools_0.2-18       splines_4.1.2         
 [16] doParallel_1.0.16      cachem_1.0.6           impute_1.68.0         
 [19] knitr_1.36             Formula_1.2-4          workflowr_1.6.2       
 [22] cluster_2.1.2          GO.db_3.14.0           dbplyr_2.1.1          
 [25] png_0.1-7              compiler_4.1.2         httr_1.4.2            
 [28] backports_1.4.0        assertthat_0.2.1       Matrix_1.3-4          
 [31] fastmap_1.1.0          later_1.3.0            htmltools_0.5.2       
 [34] prettyunits_1.1.1      tools_4.1.2            gtable_0.3.0          
 [37] glue_1.5.1             GenomeInfoDbData_1.2.7 reshape2_1.4.4        
 [40] dplyr_1.0.7            rappdirs_0.3.3         Rcpp_1.0.7            
 [43] Biobase_2.54.0         jquerylib_0.1.4        vctrs_0.3.8           
 [46] Biostrings_2.62.0      preprocessCore_1.56.0  iterators_1.0.13      
 [49] xfun_0.28              stringr_1.4.0          lifecycle_1.0.1       
 [52] XML_3.99-0.8           zlibbioc_1.40.0        scales_1.1.1          
 [55] hms_1.1.1              promises_1.2.0.1       parallel_4.1.2        
 [58] RColorBrewer_1.1-2     yaml_2.2.1             curl_4.3.2            
 [61] memoise_2.0.1          gridExtra_2.3          rpart_4.1-15          
 [64] latticeExtra_0.6-29    stringi_1.7.6          RSQLite_2.2.9         
 [67] highr_0.9              S4Vectors_0.32.3       foreach_1.5.1         
 [70] checkmate_2.0.0        BiocGenerics_0.40.0    filelock_1.0.2        
 [73] GenomeInfoDb_1.30.0    rlang_0.4.12           pkgconfig_2.0.3       
 [76] bitops_1.0-7           matrixStats_0.61.0     evaluate_0.14         
 [79] lattice_0.20-45        purrr_0.3.4            htmlwidgets_1.5.4     
 [82] labeling_0.4.2         bit_4.0.4              tidyselect_1.1.1      
 [85] plyr_1.8.6             magrittr_2.0.1         R6_2.5.1              
 [88] IRanges_2.28.0         generics_0.1.1         Hmisc_4.6-0           
 [91] DBI_1.1.1              pillar_1.6.4           whisker_0.4           
 [94] foreign_0.8-81         withr_2.4.3            survival_3.2-13       
 [97] KEGGREST_1.34.0        RCurl_1.98-1.5         nnet_7.3-16           
[100] tibble_3.1.6           crayon_1.4.2           utf8_1.2.2            
[103] BiocFileCache_2.2.0    rmarkdown_2.11         jpeg_0.1-9            
[106] progress_1.2.2         locfit_1.5-9.4         grid_4.1.2            
[109] data.table_1.14.2      blob_1.2.2             git2r_0.29.0          
[112] digest_0.6.29          httpuv_1.6.3           stats4_4.1.2          
[115] munsell_0.5.0         </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
