<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Michael C. Saul (michael.saul [at] jax.org)" />


<title>DO Cocaine and Saline Sensitization Canonical Correlations</title>

<script src="site_libs/header-attrs-2.17/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Diversity Outbred Reference Traits Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">DO Cocaine and Saline Sensitization
Canonical Correlations</h1>
<h4 class="author">Michael C. Saul (michael.saul [at] jax.org)</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-04-02
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rta/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20200501code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20200501)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20200501code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20200501)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong6065daa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong> 6065daa
</a>
</p>
</div>
<div id="strongRepositoryversionstrong6065daa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version 6065daa.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/.Rapp.history
    Ignored:    data/.Rhistory
    Ignored:    data/RDS/
    Ignored:    run/

Untracked files:
    Untracked:  LICENSE
    Untracked:  Rplot.pdf
    Untracked:  SENSI_chr7_snps.R
    Untracked:  Untitled.R
    Untracked:  analysis/17_mapping_interpretation_old.Rmd
    Untracked:  analysis/temp.txt
    Untracked:  code/DO_IVSA_SALINE_pleiotropy.R
    Untracked:  code/DO_IVSA_SENS_pleiotropy.R
    Untracked:  code/DO_SENS_SALINE_pleiotropy.R
    Untracked:  code/DO_eQTL_perms.R
    Untracked:  code/DO_eQTL_score_calc.R
    Untracked:  code/SENSI_chr7_snps.R
    Untracked:  code/STAR_RSEM_416.sh
    Untracked:  code/eQTL_plotting_DO_striatum_416.R
    Untracked:  code/ggplot_pub_themes.R
    Untracked:  code/pleitropy_analysis.sbatch
    Untracked:  code/pleitropy_saline_analysis.sbatch
    Untracked:  code/pleitropy_sensi_saline_analysis.sbatch
    Untracked:  code/rta_mapping_figure_leftover.R
    Untracked:  data/20_elastic_mv_results.csv
    Untracked:  data/20_elasticnet_predictions.RDS
    Untracked:  data/DO_416_DGEList.RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 3).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 4).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_DGEList_unfiltered.RDS
    Untracked:  data/DO_416_ERCC_DGEList.RDS
    Untracked:  data/DO_416_ERCC_cpm.RDS
    Untracked:  data/DO_416_normrt.RDS
    Untracked:  data/DO_416_pheno.RDS
    Untracked:  data/DO_416_pheno_allgenes.RDS
    Untracked:  data/DO_416_voom (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom.RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom_allgenes.RDS
    Untracked:  data/DO_416_voom_cpresiduals.RDS
    Untracked:  data/DO_AQ_active_CoV_toptable_annotated.RDS
    Untracked:  data/DO_AQ_lever_ratio_toptable_annotated.RDS
    Untracked:  data/DO_AQ_mean_inf_toptable_annotated.RDS
    Untracked:  data/DO_AQ_rate_toptable_annotated.RDS
    Untracked:  data/DO_IVSA_SALINE_pleio_lods.RDS
    Untracked:  data/DO_IVSA_SENS_pleio_lods.RDS
    Untracked:  data/DO_IVSA_SENS_pleio_pvl.RDS
    Untracked:  data/DO_IVSA_acquisition_cancor_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_glm.RDS
    Untracked:  data/DO_IVSA_all_cancor_df.RDS
    Untracked:  data/DO_IVSA_all_cancor_df_2024-03-27.RDS
    Untracked:  data/DO_IVSA_all_cancor_df_2024-03-29.RDS
    Untracked:  data/DO_IVSA_data.RDS
    Untracked:  data/DO_IVSA_data_2024-03-29.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_ALL.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_ALL_2024-03-29.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_AQ.RDS
    Untracked:  data/DO_IVSA_novelty_data.RDS
    Untracked:  data/DO_IVSA_novelty_data_2024-03-29.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_old.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_a1_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_a2_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor_
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor_2024-03-25.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor_2024-03-27.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor_2024-03-29.RDS
    Untracked:  data/DO_IVSA_prediction_a4_male_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a5_female_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_raw_data.RDS
    Untracked:  data/DO_IVSA_transformed_data.RDS
    Untracked:  data/DO_LIMS_data.RDS
    Untracked:  data/DO_RTG_a1_new.qtl.RData
    Untracked:  data/DO_RTG_a1_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a1_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a2_new.qtl.RData
    Untracked:  data/DO_RTG_a2_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a2_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a3_new.qtl.RData
    Untracked:  data/DO_RTG_a3_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_01.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_02.RData
    Untracked:  data/DO_RTG_a4_new.qtl.RData
    Untracked:  data/DO_RTG_a5_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_project_df_for_mapping.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_2020-11-06.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_a2.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_sexspecific.RDS
    Untracked:  data/DO_SENS_SALINE_pleio_lods.RDS
    Untracked:  data/DO_cocaine_measures_data.RDS
    Untracked:  data/DO_demographic_data_df.RDS
    Untracked:  data/DO_eigengene_significance.RDS
    Untracked:  data/DO_eigengenes.RDS
    Untracked:  data/DO_elasticnet_GEX_pvalue_histograms.RData
    Untracked:  data/DO_highcor_cpresiduals_datatable.RDS
    Untracked:  data/DO_holeboard_data_df.RDS
    Untracked:  data/DO_holeboard_summarized.RDS
    Untracked:  data/DO_lightdark_data_df.RDS
    Untracked:  data/DO_mQTL_peaks.RDS
    Untracked:  data/DO_mQTL_scan1s.RDS
    Untracked:  data/DO_module_membership.RDS
    Untracked:  data/DO_novelplace_data_df.RDS
    Untracked:  data/DO_novelty_COC_cancor.RDS
    Untracked:  data/DO_novelty_COC_cancor_2024-04-02.RDS
    Untracked:  data/DO_novelty_SAL_cancor.RDS
    Untracked:  data/DO_novelty_SAL_cancor_2024-04-02.RDS
    Untracked:  data/DO_novelty_data.RDS
    Untracked:  data/DO_novelty_data_2024-03-29.RDS
    Untracked:  data/DO_novelty_response_data.RDS
    Untracked:  data/DO_novelty_response_data_for_RTG.RDS
    Untracked:  data/DO_novelty_response_data_with_transform.csv
    Untracked:  data/DO_novelty_saline_cocaine_cancor.RDS
    Untracked:  data/DO_novelty_saline_cocaine_cancor_2024-04-02.RDS
    Untracked:  data/DO_openfield_data_df.RDS
    Untracked:  data/DO_paraclique_data.el
    Untracked:  data/DO_paraclique_eigengenes.RDS
    Untracked:  data/DO_paraclique_significance_elasticnet.RDS
    Untracked:  data/DO_paracliques.RDS
    Untracked:  data/DO_paracliques_significance.RDS
    Untracked:  data/DO_ploidy_calls.RDS
    Untracked:  data/DO_pmap.RDS
    Untracked:  data/DO_pvalue_histograms.RData
    Untracked:  data/DO_rtg_a1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a3_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m3_toptable_annotated.RDS
    Untracked:  data/DO_saline.qtl.RData
    Untracked:  data/DO_saline_measures_data.RDS
    Untracked:  data/DO_sensi_cocaine.qtl.RData
    Untracked:  data/DO_str_2016_eQTL.RData
    Untracked:  data/DO_str_2016_gm4qtl2.zip
    Untracked:  data/DO_str_2016_gm4qtl2_intensities.fst
    Untracked:  data/DO_voom_cpresiduals_NArm.RDS
    Untracked:  data/DO_wgcna.RDS
    Untracked:  data/IVSA_RTG_a3_blup.pdf
    Untracked:  data/IVSA_h2_2019-11-18.tsv
    Untracked:  data/IVSA_metadata_sheet.RDS
    Untracked:  data/ME16_edges.tsv
    Untracked:  data/Mus_musculus.GRCm38.94.parsed.RDS
    Untracked:  data/Paraclique_19_MF_AMIGO.txt
    Untracked:  data/RTG_QTL_peaks.RDS
    Untracked:  data/RTG_figure.RDS
    Untracked:  data/SENSI_chr7_snps.RData
    Untracked:  data/SENS_RTG/
    Untracked:  data/Tyr2_genotypes.RDS
    Untracked:  data/Tyr_genotypes.RDS
    Untracked:  data/blup_chr7_DO_RTG_a3.RDS
    Untracked:  data/blup_spearman_cors.RData
    Untracked:  data/cc_variants/
    Untracked:  data/chr18_QTL_results.RDS
    Untracked:  data/chr4_QTL_results.RDS
    Untracked:  data/chr7_a3_QTL_results.RDS
    Untracked:  data/chr7_blups_with_rtg.RDS
    Untracked:  data/chr7_cocaine_BLUP.RDS
    Untracked:  data/chr7_gene_blup.RDS
    Untracked:  data/chr7_qtl_cors_with_spearman_blups.RDS
    Untracked:  data/chr7_sensi_cocaine_BLUP.RDS
    Untracked:  data/cis/
    Untracked:  data/eQTL_annotated_peaks.RDS
    Untracked:  data/eQTL_plots.RData
    Untracked:  data/figures/
    Untracked:  data/founders_key/
    Untracked:  data/gigamuga/
    Untracked:  data/gwas/
    Untracked:  data/info/
    Untracked:  data/m2G.a2.RTG.chr4snp.RData
    Untracked:  data/mapping/
    Untracked:  data/mapping_from_hao/
    Untracked:  data/mpd_measures_saline.txt
    Untracked:  data/mpd_measures_saline_cocaine.txt
    Untracked:  data/perms/
    Untracked:  data/pleio_df.RDS
    Untracked:  data/qtl2/
    Untracked:  data/raw/
    Untracked:  data/resources/
    Untracked:  data/tads_loops/
    Untracked:  data/transcripts/
    Untracked:  data/variants/
    Untracked:  logs/
    Untracked:  new_RTG_a3.jpeg
    Untracked:  output/DO_module_membership.tsv
    Untracked:  output/DO_paraclique_df.tsv
    Untracked:  output/RDS/
    Untracked:  output/RTG_out/
    Untracked:  output/docs_backup_2020-07-30.tar.gz
    Untracked:  output/elastic_net_out/
    Untracked:  output/mapping_figure.pdf
    Untracked:  output/mapping_plot.pdf
    Untracked:  output/mediation_all_rta_figure.pdf
    Untracked:  output/mediation_all_rta_figure_with_eed.pdf
    Untracked:  output/pleio_plot2.pdf
    Untracked:  rstudio_rta.sif
    Untracked:  singularity/
    Untracked:  tmp/
    Untracked:  var/

Unstaged changes:
    Deleted:    .Rprofile
    Modified:   .gitignore
    Modified:   README.md
    Modified:   analysis/17_mapping_interpretation.Rmd
    Modified:   analysis/_site.yml

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/22_DO_RTA_saline_cocaine.Rmd</code>) and HTML
(<code>docs/22_DO_RTA_saline_cocaine.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
6065daa
</td>
<td>
Michael C. Saul
</td>
<td>
2024-04-02
</td>
<td>
Edited the MPD call so that the public version is the only version that
is conserved.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
96203d5
</td>
<td>
Michael C. Saul
</td>
<td>
2024-04-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
fc82bd1
</td>
<td>
Michael C. Saul
</td>
<td>
2024-04-02
</td>
<td>
Changing MPD to public version (note: old version preserved for now.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
9a95311
</td>
<td>
Michael C. Saul
</td>
<td>
2022-04-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
7d756c4
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
7567181
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-11
</td>
<td>
Saving the saline and cocaine measures from MPD
</td>
</tr>
<tr>
<td>
html
</td>
<td>
6c5ce0e
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
8fa6f9a
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-11
</td>
<td>
Committing to fix another absolute file path
</td>
</tr>
<tr>
<td>
html
</td>
<td>
ad47706
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
1396c1e
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-11
</td>
<td>
Committing to fix an absolute file path
</td>
</tr>
<tr>
<td>
html
</td>
<td>
7deeb6d
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
4c0fdc3
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-11
</td>
<td>
Initial commit of the saline and cocaine RTA
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="background" class="section level1">
<h1>Background</h1>
<div id="motivation" class="section level2">
<h2>Motivation</h2>
<div id="purpose-of-analysis" class="section level3">
<h3>Purpose of analysis</h3>
<p>This analysis is performed to process the novelty data, sensitization
saline data, and sensitization cocaine data. 2) Strain-by-sex means and
standard errors for plotting</p>
<p><strong>These data were collected by the CSNA behavioral phenotyping
core.</strong></p>
</div>
</div>
</div>
<div id="analysis" class="section level1">
<h1>Analysis</h1>
<div id="setup" class="section level2">
<h2>Setup</h2>
<div id="r-libraries" class="section level3">
<h3>R libraries</h3>
<p>Calling R libraries necessary for this analysis.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;dplyr&quot;</span>)</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;tidyr&quot;</span>)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;tidyverse&quot;</span>)</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;lubridate&quot;</span>)</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggplot2&quot;</span>)</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;cowplot&quot;</span>)</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;MASS&quot;</span>)</span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(<span class="st">&quot;ggbeeswarm&quot;</span>)</span></code></pre></div>
</div>
<div id="current-date" class="section level3">
<h3>Current Date</h3>
<p>Getting current date</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a>current_date <span class="ot">=</span> <span class="fu">format</span>(<span class="fu">Sys.time</span>(),<span class="st">&quot;%Y-%m-%d&quot;</span>)</span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>current_date</span></code></pre></div>
<pre><code>[1] &quot;2024-04-02&quot;</code></pre>
</div>
</div>
<div id="importing-data" class="section level2">
<h2>Importing data</h2>
<div id="getting-mpd-site-information" class="section level3">
<h3>Getting MPD Site Information</h3>
<p>Currently using internal access to the JAX MPD curation site.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>MPD_prefix <span class="ot">=</span> <span class="st">&quot;http://mpdcuration.jax.org:82/api/pheno/animalvals/&quot;</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>MPD_suffix <span class="ot">=</span> <span class="st">&quot;?csv=yes&quot;</span></span></code></pre></div>
</div>
<div id="importing-raw-saline-and-cocaine-response-data-from-mpd"
class="section level3">
<h3>Importing raw saline and cocaine response data from MPD</h3>
<p>First, getting table of saline and cocaine measures in MPD</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>saline_cocaine_measures <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">&quot;./data/mpd_measures_saline_cocaine.txt&quot;</span>,</span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">&quot;</span><span class="sc">\t</span><span class="st">&quot;</span>)</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a><span class="co"># filtering to only post-injection data</span></span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a>saline_measures_post <span class="ot">=</span> saline_cocaine_measures[<span class="fu">which</span>(saline_cocaine_measures<span class="sc">$</span>injection_time <span class="sc">==</span> <span class="st">&quot;post&quot;</span> <span class="sc">&amp;</span></span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a>                                                     saline_cocaine_measures<span class="sc">$</span>saline_cocaine <span class="sc">==</span> <span class="st">&quot;saline&quot;</span>),]</span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a>cocaine_measures_post <span class="ot">=</span> saline_cocaine_measures[<span class="fu">which</span>(saline_cocaine_measures<span class="sc">$</span>injection_time <span class="sc">==</span> <span class="st">&quot;post&quot;</span> <span class="sc">&amp;</span></span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a>                                                      saline_cocaine_measures<span class="sc">$</span>saline_cocaine <span class="sc">==</span> <span class="st">&quot;cocaine&quot;</span>),]</span></code></pre></div>
<p>Next, getting cocaine and saline datasets from MPD.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting a base for the MPD measures</span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>mpd_series_base <span class="ot">=</span> <span class="st">&quot;https://phenome.jax.org/api/pheno/animalvals/series/MEASURE?csv=1&quot;</span></span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting cocaine sensitization MPD measures IDs</span></span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a>cocaine_sensi_measures <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;90309&quot;</span>,<span class="st">&quot;90315&quot;</span>,<span class="st">&quot;90321&quot;</span>,<span class="st">&quot;90327&quot;</span>,</span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a>                           <span class="st">&quot;90442&quot;</span>,<span class="st">&quot;90333&quot;</span>,<span class="st">&quot;90387&quot;</span>)</span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a>cocaine_sensi_colnames <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;animal_id&quot;</span>, cocaine_measures_post<span class="sc">$</span>measure)</span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting saline sensitization MPD measures IDs</span></span>
<span id="cb6-10"><a href="#cb6-10" aria-hidden="true" tabindex="-1"></a>saline_sensi_measures <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;90306&quot;</span>,<span class="st">&quot;90312&quot;</span>,<span class="st">&quot;90318&quot;</span>,<span class="st">&quot;90324&quot;</span>,</span>
<span id="cb6-11"><a href="#cb6-11" aria-hidden="true" tabindex="-1"></a>                          <span class="st">&quot;90439&quot;</span>,<span class="st">&quot;90330&quot;</span>,<span class="st">&quot;90384&quot;</span>)</span>
<span id="cb6-12"><a href="#cb6-12" aria-hidden="true" tabindex="-1"></a>saline_sensi_colnames <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;animal_id&quot;</span>, saline_measures_post<span class="sc">$</span>measure)</span>
<span id="cb6-13"><a href="#cb6-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-14"><a href="#cb6-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Pulling data from MPD for cocaine locomotor sensitization</span></span>
<span id="cb6-15"><a href="#cb6-15" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="fu">seq_len</span>(<span class="fu">length</span>(cocaine_sensi_measures))) {</span>
<span id="cb6-16"><a href="#cb6-16" aria-hidden="true" tabindex="-1"></a>  mpd_csv_i <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;MEASURE&quot;</span>, cocaine_sensi_measures[i], mpd_series_base)</span>
<span id="cb6-17"><a href="#cb6-17" aria-hidden="true" tabindex="-1"></a>  mpd_table_i <span class="ot">=</span> <span class="fu">read_csv</span>(mpd_csv_i)</span>
<span id="cb6-18"><a href="#cb6-18" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (i <span class="sc">==</span> <span class="dv">1</span>) {</span>
<span id="cb6-19"><a href="#cb6-19" aria-hidden="true" tabindex="-1"></a>    mpd_cocaine_table_long <span class="ot">=</span> mpd_table_i</span>
<span id="cb6-20"><a href="#cb6-20" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb6-21"><a href="#cb6-21" aria-hidden="true" tabindex="-1"></a>    mpd_cocaine_table_long <span class="ot">=</span> <span class="fu">rbind</span>(mpd_cocaine_table_long, mpd_table_i)</span>
<span id="cb6-22"><a href="#cb6-22" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb6-23"><a href="#cb6-23" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<pre><code>Rows: 490 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (4): measnum, value, animal_projid, strainid
lgl (1): label

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 490 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (4): measnum, value, animal_projid, strainid
lgl (1): label

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 483 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (4): measnum, value, animal_projid, strainid
lgl (1): label

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 406 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (4): measnum, value, animal_projid, strainid
lgl (1): label

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 377 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (4): measnum, value, animal_projid, strainid
lgl (1): label

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 4327 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (5): measnum, label, value, animal_projid, strainid

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 4323 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (7): projsym, varname, label, strain, sex, animal_id, stocknum
dbl (4): measnum, value, animal_projid, strainid

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting cocaine measure names</span></span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>cocaine_measure_names <span class="ot">=</span> <span class="fu">unique</span>(<span class="fu">pull</span>(mpd_cocaine_table_long, varname))</span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-4"><a href="#cb8-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Making a wide format table for cocaine measures</span></span>
<span id="cb8-5"><a href="#cb8-5" aria-hidden="true" tabindex="-1"></a>mpd_cocaine_sensi_table <span class="ot">=</span> mpd_cocaine_table_long <span class="sc">|&gt;</span></span>
<span id="cb8-6"><a href="#cb8-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(<span class="at">id_cols =</span> <span class="fu">c</span>(animal_id, animal_projid, strain,</span>
<span id="cb8-7"><a href="#cb8-7" aria-hidden="true" tabindex="-1"></a>                          sex, stocknum, strainid),</span>
<span id="cb8-8"><a href="#cb8-8" aria-hidden="true" tabindex="-1"></a>              <span class="at">names_from =</span> varname,</span>
<span id="cb8-9"><a href="#cb8-9" aria-hidden="true" tabindex="-1"></a>              <span class="at">values_from =</span> value) <span class="sc">|&gt;</span></span>
<span id="cb8-10"><a href="#cb8-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(strain <span class="sc">==</span> <span class="st">&quot;J:DO&quot;</span>) <span class="sc">|&gt;</span></span>
<span id="cb8-11"><a href="#cb8-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">as.data.frame</span>()</span>
<span id="cb8-12"><a href="#cb8-12" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(mpd_cocaine_sensi_table) <span class="ot">=</span> mpd_cocaine_sensi_table<span class="sc">$</span>animal_id</span>
<span id="cb8-13"><a href="#cb8-13" aria-hidden="true" tabindex="-1"></a>cocaine_measures_data <span class="ot">=</span> mpd_cocaine_sensi_table[,cocaine_sensi_colnames]</span>
<span id="cb8-14"><a href="#cb8-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-15"><a href="#cb8-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Pulling data from MPD for saline locomotor sensitization</span></span>
<span id="cb8-16"><a href="#cb8-16" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="fu">seq_len</span>(<span class="fu">length</span>(saline_sensi_measures))) {</span>
<span id="cb8-17"><a href="#cb8-17" aria-hidden="true" tabindex="-1"></a>  mpd_csv_i <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;MEASURE&quot;</span>, saline_sensi_measures[i], mpd_series_base)</span>
<span id="cb8-18"><a href="#cb8-18" aria-hidden="true" tabindex="-1"></a>  mpd_table_i <span class="ot">=</span> <span class="fu">read_csv</span>(mpd_csv_i)</span>
<span id="cb8-19"><a href="#cb8-19" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (i <span class="sc">==</span> <span class="dv">1</span>) {</span>
<span id="cb8-20"><a href="#cb8-20" aria-hidden="true" tabindex="-1"></a>    mpd_saline_table_long <span class="ot">=</span> mpd_table_i</span>
<span id="cb8-21"><a href="#cb8-21" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb8-22"><a href="#cb8-22" aria-hidden="true" tabindex="-1"></a>    mpd_saline_table_long <span class="ot">=</span> <span class="fu">rbind</span>(mpd_saline_table_long, mpd_table_i)</span>
<span id="cb8-23"><a href="#cb8-23" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb8-24"><a href="#cb8-24" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<pre><code>Rows: 474 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (4): measnum, value, animal_projid, strainid
lgl (1): label

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 473 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (4): measnum, value, animal_projid, strainid
lgl (1): label

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 471 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (4): measnum, value, animal_projid, strainid
lgl (1): label

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 399 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (4): measnum, value, animal_projid, strainid
lgl (1): label

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 371 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (4): measnum, value, animal_projid, strainid
lgl (1): label

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 4212 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (5): measnum, label, value, animal_projid, strainid

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 4208 Columns: 11
── Column specification ────────────────────────────────────────────────────────
Delimiter: &quot;,&quot;
chr (6): projsym, varname, strain, sex, animal_id, stocknum
dbl (5): measnum, label, value, animal_projid, strainid

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting saline measure names</span></span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a>saline_measure_names <span class="ot">=</span> <span class="fu">unique</span>(<span class="fu">pull</span>(mpd_saline_table_long, varname))</span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Making a wide format table for saline measures</span></span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>mpd_saline_sensi_table <span class="ot">=</span> mpd_saline_table_long <span class="sc">|&gt;</span></span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(<span class="at">id_cols =</span> <span class="fu">c</span>(animal_id, animal_projid, strain,</span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a>                          sex, stocknum, strainid),</span>
<span id="cb10-8"><a href="#cb10-8" aria-hidden="true" tabindex="-1"></a>              <span class="at">names_from =</span> varname,</span>
<span id="cb10-9"><a href="#cb10-9" aria-hidden="true" tabindex="-1"></a>              <span class="at">values_from =</span> value) <span class="sc">|&gt;</span></span>
<span id="cb10-10"><a href="#cb10-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(strain <span class="sc">==</span> <span class="st">&quot;J:DO&quot;</span>) <span class="sc">|&gt;</span></span>
<span id="cb10-11"><a href="#cb10-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">as.data.frame</span>()</span>
<span id="cb10-12"><a href="#cb10-12" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(mpd_saline_sensi_table) <span class="ot">=</span> mpd_saline_sensi_table<span class="sc">$</span>animal_id</span>
<span id="cb10-13"><a href="#cb10-13" aria-hidden="true" tabindex="-1"></a>saline_measures_data <span class="ot">=</span> mpd_saline_sensi_table[,saline_sensi_colnames]</span></code></pre></div>
</div>
<div id="importing-novelty-response-data" class="section level3">
<h3>Importing novelty response data</h3>
<p>Loading novelty response data and preparing for RTA.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a>DO_novelty_response <span class="ot">=</span> <span class="fu">readRDS</span>(<span class="st">&quot;./data/DO_novelty_response_data.RDS&quot;</span>)</span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a>DO_novelty_response<span class="sc">$</span>Sex <span class="ot">=</span> <span class="fu">gsub</span>(<span class="st">&quot;^([MF])</span><span class="sc">\\</span><span class="st">w+$&quot;</span>,<span class="st">&quot;</span><span class="sc">\\</span><span class="st">1&quot;</span>,DO_novelty_response<span class="sc">$</span>Sex)</span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>DO_saline_novelty <span class="ot">=</span> DO_novelty_response[<span class="fu">row.names</span>(saline_measures_data),]</span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a>DO_saline_novelty <span class="ot">=</span> <span class="fu">cbind</span>(DO_saline_novelty, saline_measures_data[,<span class="sc">-</span><span class="dv">1</span>])</span>
<span id="cb11-5"><a href="#cb11-5" aria-hidden="true" tabindex="-1"></a>DO_cocaine_novelty <span class="ot">=</span> DO_novelty_response[<span class="fu">row.names</span>(cocaine_measures_data),]</span>
<span id="cb11-6"><a href="#cb11-6" aria-hidden="true" tabindex="-1"></a>DO_cocaine_novelty <span class="ot">=</span> <span class="fu">cbind</span>(DO_cocaine_novelty, cocaine_measures_data[,<span class="sc">-</span><span class="dv">1</span>])</span>
<span id="cb11-7"><a href="#cb11-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-8"><a href="#cb11-8" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting columns of each type of measure</span></span>
<span id="cb11-9"><a href="#cb11-9" aria-hidden="true" tabindex="-1"></a>saline_columns <span class="ot">=</span> <span class="fu">colnames</span>(saline_measures_data[,<span class="sc">-</span><span class="dv">1</span>])</span>
<span id="cb11-10"><a href="#cb11-10" aria-hidden="true" tabindex="-1"></a>cocaine_columns <span class="ot">=</span> <span class="fu">colnames</span>(cocaine_measures_data[,<span class="sc">-</span><span class="dv">1</span>])</span>
<span id="cb11-11"><a href="#cb11-11" aria-hidden="true" tabindex="-1"></a>novelty_columns <span class="ot">=</span> <span class="fu">colnames</span>(DO_novelty_response)</span>
<span id="cb11-12"><a href="#cb11-12" aria-hidden="true" tabindex="-1"></a>novelty_columns <span class="ot">=</span> novelty_columns[<span class="fu">grep</span>(<span class="st">&quot;batch_ranknorm$&quot;</span>,novelty_columns)]</span></code></pre></div>
</div>
<div id="saline-rta" class="section level3">
<h3>Saline RTA</h3>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting highest correlation variables to start out</span></span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a>highest_cor <span class="ot">=</span> <span class="fu">cor</span>(DO_saline_novelty[,<span class="st">&quot;dist_d3_d11_auc_cont_do&quot;</span>], </span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>                  DO_saline_novelty[,novelty_columns],</span>
<span id="cb12-4"><a href="#cb12-4" aria-hidden="true" tabindex="-1"></a>                  <span class="at">use =</span> <span class="st">&quot;pairwise.complete.obs&quot;</span>)</span>
<span id="cb12-5"><a href="#cb12-5" aria-hidden="true" tabindex="-1"></a>highest_cor_r <span class="ot">=</span> <span class="fu">max</span>(<span class="fu">abs</span>(highest_cor))</span>
<span id="cb12-6"><a href="#cb12-6" aria-hidden="true" tabindex="-1"></a>highest_cor_ind <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">abs</span>(highest_cor) <span class="sc">==</span> highest_cor_r, <span class="at">arr.ind =</span> <span class="cn">TRUE</span>)</span>
<span id="cb12-7"><a href="#cb12-7" aria-hidden="true" tabindex="-1"></a>saline_highcor_id <span class="ot">=</span> <span class="st">&quot;dist_d3_d11_auc_cont_do&quot;</span></span>
<span id="cb12-8"><a href="#cb12-8" aria-hidden="true" tabindex="-1"></a>novelty_highcor_id <span class="ot">=</span> <span class="fu">colnames</span>(highest_cor)[highest_cor_ind[<span class="dv">1</span>,<span class="st">&quot;col&quot;</span>]]</span>
<span id="cb12-9"><a href="#cb12-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-10"><a href="#cb12-10" aria-hidden="true" tabindex="-1"></a><span class="fu">paste0</span>(<span class="st">&quot;The highest correlation (&quot;</span>, <span class="fu">round</span>(highest_cor_r, <span class="dv">2</span>),<span class="st">&quot;) is between &quot;</span>, saline_highcor_id, <span class="st">&quot; and &quot;</span>,</span>
<span id="cb12-11"><a href="#cb12-11" aria-hidden="true" tabindex="-1"></a>       novelty_highcor_id, <span class="st">&quot;.&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;The highest correlation (0.53) is between dist_d3_d11_auc_cont_do and OFA_total_distance_traveled_batch_ranknorm.&quot;</code></pre>
<p>Running canonical correlation on saline-treated animals</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Starting with these as best novelty and IVSA variables</span></span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>best_novelty_vars <span class="ot">=</span> novelty_highcor_id</span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a>best_saline_vars <span class="ot">=</span> saline_highcor_id</span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a>noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_saline_novelty[,<span class="fu">c</span>(best_novelty_vars, best_saline_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb14-5"><a href="#cb14-5" aria-hidden="true" tabindex="-1"></a>starting_cancor <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_saline_novelty[noNA_rows,best_novelty_vars], </span>
<span id="cb14-6"><a href="#cb14-6" aria-hidden="true" tabindex="-1"></a>                                 DO_saline_novelty[noNA_rows,best_saline_vars],</span>
<span id="cb14-7"><a href="#cb14-7" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb14-8"><a href="#cb14-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-9"><a href="#cb14-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Selecting appropriate saline variables</span></span>
<span id="cb14-10"><a href="#cb14-10" aria-hidden="true" tabindex="-1"></a>saline_vars_unselected <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;dist_d3_d2_cont_do&quot;</span>,<span class="st">&quot;dist_d5_d3_cont_do&quot;</span>,<span class="st">&quot;dist_d12_d2_cont_do&quot;</span>,<span class="st">&quot;dist_d19_d11_cont_do&quot;</span>)</span>
<span id="cb14-11"><a href="#cb14-11" aria-hidden="true" tabindex="-1"></a>novelty_vars_unselected <span class="ot">=</span> novelty_columns[(<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(novelty_columns <span class="sc">==</span> best_novelty_vars))]</span>
<span id="cb14-12"><a href="#cb14-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-13"><a href="#cb14-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-14"><a href="#cb14-14" aria-hidden="true" tabindex="-1"></a>not_optimized <span class="ot">=</span> <span class="cn">TRUE</span></span>
<span id="cb14-15"><a href="#cb14-15" aria-hidden="true" tabindex="-1"></a>min_obs <span class="ot">=</span> <span class="dv">225</span></span>
<span id="cb14-16"><a href="#cb14-16" aria-hidden="true" tabindex="-1"></a>max_var <span class="ot">=</span> <span class="dv">10</span></span>
<span id="cb14-17"><a href="#cb14-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-18"><a href="#cb14-18" aria-hidden="true" tabindex="-1"></a><span class="cf">while</span> (not_optimized) {</span>
<span id="cb14-19"><a href="#cb14-19" aria-hidden="true" tabindex="-1"></a>  best_cancor <span class="ot">=</span> starting_cancor<span class="sc">$</span>cor[<span class="dv">1</span>]</span>
<span id="cb14-20"><a href="#cb14-20" aria-hidden="true" tabindex="-1"></a>  best_var <span class="ot">=</span> <span class="st">&quot;&quot;</span></span>
<span id="cb14-21"><a href="#cb14-21" aria-hidden="true" tabindex="-1"></a>  best_paradigm <span class="ot">=</span> <span class="st">&quot;&quot;</span></span>
<span id="cb14-22"><a href="#cb14-22" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb14-23"><a href="#cb14-23" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (i <span class="cf">in</span> novelty_vars_unselected) {</span>
<span id="cb14-24"><a href="#cb14-24" aria-hidden="true" tabindex="-1"></a>    noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_saline_novelty[,<span class="fu">c</span>(best_novelty_vars, i, best_saline_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb14-25"><a href="#cb14-25" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="fu">length</span>(noNA_rows) <span class="sc">&gt;=</span> min_obs) {</span>
<span id="cb14-26"><a href="#cb14-26" aria-hidden="true" tabindex="-1"></a>      cancor_i <span class="ot">=</span> stats<span class="sc">::</span><span class="fu">cancor</span>(DO_saline_novelty[noNA_rows, <span class="fu">c</span>(best_novelty_vars, i)], DO_saline_novelty[noNA_rows, best_saline_vars],</span>
<span id="cb14-27"><a href="#cb14-27" aria-hidden="true" tabindex="-1"></a>                        <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb14-28"><a href="#cb14-28" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span> (cancor_i<span class="sc">$</span>cor[<span class="dv">1</span>] <span class="sc">&gt;</span> best_cancor) {</span>
<span id="cb14-29"><a href="#cb14-29" aria-hidden="true" tabindex="-1"></a>        best_cancor <span class="ot">=</span> cancor_i<span class="sc">$</span>cor[<span class="dv">1</span>]</span>
<span id="cb14-30"><a href="#cb14-30" aria-hidden="true" tabindex="-1"></a>        best_var <span class="ot">=</span> i</span>
<span id="cb14-31"><a href="#cb14-31" aria-hidden="true" tabindex="-1"></a>        best_paradigm <span class="ot">=</span> <span class="st">&quot;novelty&quot;</span></span>
<span id="cb14-32"><a href="#cb14-32" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb14-33"><a href="#cb14-33" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb14-34"><a href="#cb14-34" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb14-35"><a href="#cb14-35" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb14-36"><a href="#cb14-36" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (j <span class="cf">in</span> saline_vars_unselected) {</span>
<span id="cb14-37"><a href="#cb14-37" aria-hidden="true" tabindex="-1"></a>    noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_saline_novelty[,<span class="fu">c</span>(best_novelty_vars, j, best_saline_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb14-38"><a href="#cb14-38" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="fu">length</span>(noNA_rows) <span class="sc">&gt;=</span> min_obs) {</span>
<span id="cb14-39"><a href="#cb14-39" aria-hidden="true" tabindex="-1"></a>      cancor_j <span class="ot">=</span> stats<span class="sc">::</span><span class="fu">cancor</span>(DO_saline_novelty[noNA_rows, best_novelty_vars], DO_saline_novelty[noNA_rows,<span class="fu">c</span>(best_saline_vars, j)],</span>
<span id="cb14-40"><a href="#cb14-40" aria-hidden="true" tabindex="-1"></a>                        <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb14-41"><a href="#cb14-41" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span> (cancor_j<span class="sc">$</span>cor[<span class="dv">1</span>] <span class="sc">&gt;</span> best_cancor) {</span>
<span id="cb14-42"><a href="#cb14-42" aria-hidden="true" tabindex="-1"></a>        best_cancor <span class="ot">=</span> cancor_j<span class="sc">$</span>cor[<span class="dv">1</span>]</span>
<span id="cb14-43"><a href="#cb14-43" aria-hidden="true" tabindex="-1"></a>        best_var <span class="ot">=</span> j</span>
<span id="cb14-44"><a href="#cb14-44" aria-hidden="true" tabindex="-1"></a>        best_paradigm <span class="ot">=</span> <span class="st">&quot;saline&quot;</span></span>
<span id="cb14-45"><a href="#cb14-45" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb14-46"><a href="#cb14-46" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb14-47"><a href="#cb14-47" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb14-48"><a href="#cb14-48" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb14-49"><a href="#cb14-49" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;novelty&quot;</span>) {</span>
<span id="cb14-50"><a href="#cb14-50" aria-hidden="true" tabindex="-1"></a>    novelty_vars_test <span class="ot">=</span> <span class="fu">c</span>(best_novelty_vars, best_var)</span>
<span id="cb14-51"><a href="#cb14-51" aria-hidden="true" tabindex="-1"></a>    saline_vars_test <span class="ot">=</span> best_saline_vars</span>
<span id="cb14-52"><a href="#cb14-52" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;saline&quot;</span>) {</span>
<span id="cb14-53"><a href="#cb14-53" aria-hidden="true" tabindex="-1"></a>    novelty_vars_test <span class="ot">=</span> best_novelty_vars</span>
<span id="cb14-54"><a href="#cb14-54" aria-hidden="true" tabindex="-1"></a>    saline_vars_test <span class="ot">=</span> <span class="fu">c</span>(best_saline_vars, best_var)</span>
<span id="cb14-55"><a href="#cb14-55" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb14-56"><a href="#cb14-56" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb14-57"><a href="#cb14-57" aria-hidden="true" tabindex="-1"></a>  noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_saline_novelty[,<span class="fu">c</span>(novelty_vars_test, saline_vars_test)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb14-58"><a href="#cb14-58" aria-hidden="true" tabindex="-1"></a>  k_gtr <span class="ot">=</span> <span class="fu">c</span>()</span>
<span id="cb14-59"><a href="#cb14-59" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb14-60"><a href="#cb14-60" aria-hidden="true" tabindex="-1"></a>  B <span class="ot">=</span> <span class="fl">1e4</span></span>
<span id="cb14-61"><a href="#cb14-61" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (k <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span>B) {</span>
<span id="cb14-62"><a href="#cb14-62" aria-hidden="true" tabindex="-1"></a>    cancor_k <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_saline_novelty[<span class="fu">sample</span>(noNA_rows), novelty_vars_test], </span>
<span id="cb14-63"><a href="#cb14-63" aria-hidden="true" tabindex="-1"></a>                      DO_saline_novelty[noNA_rows, saline_vars_test],</span>
<span id="cb14-64"><a href="#cb14-64" aria-hidden="true" tabindex="-1"></a>                      <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb14-65"><a href="#cb14-65" aria-hidden="true" tabindex="-1"></a>    k_gtr <span class="ot">=</span> <span class="fu">c</span>(k_gtr, cancor_k<span class="sc">$</span>cor[<span class="dv">1</span>])</span>
<span id="cb14-66"><a href="#cb14-66" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb14-67"><a href="#cb14-67" aria-hidden="true" tabindex="-1"></a>  p_value_improvement <span class="ot">=</span> (<span class="fu">length</span>(<span class="fu">which</span>(k_gtr <span class="sc">&gt;</span> best_cancor)) <span class="sc">+</span> <span class="dv">1</span>) <span class="sc">/</span> (B <span class="sc">+</span> <span class="dv">1</span>)</span>
<span id="cb14-68"><a href="#cb14-68" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb14-69"><a href="#cb14-69" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> ((<span class="fu">length</span>(best_novelty_vars) <span class="sc">+</span> <span class="fu">length</span>(best_saline_vars)) <span class="sc">&gt;</span> max_var) {</span>
<span id="cb14-70"><a href="#cb14-70" aria-hidden="true" tabindex="-1"></a>    not_optimized <span class="ot">=</span> <span class="cn">FALSE</span></span>
<span id="cb14-71"><a href="#cb14-71" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> <span class="cf">if</span> (p_value_improvement <span class="sc">&lt;</span> <span class="fl">0.001</span>) {</span>
<span id="cb14-72"><a href="#cb14-72" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;novelty&quot;</span>) {</span>
<span id="cb14-73"><a href="#cb14-73" aria-hidden="true" tabindex="-1"></a>      best_novelty_vars <span class="ot">=</span> <span class="fu">c</span>(best_novelty_vars, best_var)</span>
<span id="cb14-74"><a href="#cb14-74" aria-hidden="true" tabindex="-1"></a>      novelty_vars_unselected <span class="ot">=</span> novelty_vars_unselected[<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(novelty_vars_unselected <span class="sc">==</span> best_var)]</span>
<span id="cb14-75"><a href="#cb14-75" aria-hidden="true" tabindex="-1"></a>    } <span class="cf">else</span> <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;saline&quot;</span>) {</span>
<span id="cb14-76"><a href="#cb14-76" aria-hidden="true" tabindex="-1"></a>      best_saline_vars <span class="ot">=</span> <span class="fu">c</span>(best_saline_vars, best_var)</span>
<span id="cb14-77"><a href="#cb14-77" aria-hidden="true" tabindex="-1"></a>      saline_vars_unselected <span class="ot">=</span> saline_vars_unselected[<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(saline_vars_unselected <span class="sc">==</span> best_var)]</span>
<span id="cb14-78"><a href="#cb14-78" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb14-79"><a href="#cb14-79" aria-hidden="true" tabindex="-1"></a>    starting_cancor <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_saline_novelty[noNA_rows,best_novelty_vars], DO_saline_novelty[noNA_rows,best_saline_vars],</span>
<span id="cb14-80"><a href="#cb14-80" aria-hidden="true" tabindex="-1"></a>                             <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb14-81"><a href="#cb14-81" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb14-82"><a href="#cb14-82" aria-hidden="true" tabindex="-1"></a>    not_optimized <span class="ot">=</span> <span class="cn">FALSE</span></span>
<span id="cb14-83"><a href="#cb14-83" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb14-84"><a href="#cb14-84" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb14-85"><a href="#cb14-85" aria-hidden="true" tabindex="-1"></a>noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_saline_novelty[,<span class="fu">c</span>(best_novelty_vars, best_saline_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb14-86"><a href="#cb14-86" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-87"><a href="#cb14-87" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Best novelty variables:</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="fu">paste</span>(best_novelty_vars, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>, <span class="at">collapse =</span> <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>),<span class="st">&quot;</span><span class="sc">\n\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Best novelty variables:
OFA_total_distance_traveled_batch_ranknorm
NPP_NoveltyPreference_ZoneTime_WhiteVsBlack_Total_batch_ranknorm
OFA_total_resting_time_in_center_batch_ranknorm
OFA_distance_traveled_last_five_batch_ranknorm
OFA_total_distance_traveled_in_corner_batch_ranknorm
OFA_pct_time_corner_batch_ranknorm
OFA_pct_resting_time_center_batch_ranknorm</code></pre>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Best saline variables:</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="fu">paste</span>(best_saline_vars, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>, <span class="at">collapse =</span> <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>), <span class="st">&quot;</span><span class="sc">\n\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Best saline variables:
dist_d3_d11_auc_cont_do
dist_d3_d2_cont_do
dist_d5_d3_cont_do
dist_d19_d11_cont_do</code></pre>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;First canonical correlate magnitude: &quot;</span>, <span class="fu">round</span>(starting_cancor<span class="sc">$</span>cor[<span class="dv">1</span>], <span class="dv">4</span>), <span class="st">&quot;</span><span class="sc">\n\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>First canonical correlate magnitude: 0.6532</code></pre>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Canonical correlation trained on &quot;</span>, <span class="fu">length</span>(noNA_rows), <span class="st">&quot; complete observations.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Canonical correlation trained on 237 complete observations.</code></pre>
<p>Adjusting this canonical correlation for sex.</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>DO_saline_novelty<span class="sc">$</span>DE_Sex <span class="ot">=</span> DO_saline_novelty<span class="sc">$</span>Sex</span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>DO_saline_novelty<span class="sc">$</span>Sex <span class="ot">=</span> <span class="fu">ifelse</span>(DO_saline_novelty<span class="sc">$</span>DE_Sex <span class="sc">==</span> <span class="st">&quot;M&quot;</span>, <span class="dv">1</span>, <span class="dv">0</span>)</span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a>novelty_cancor_vars_SAL <span class="ot">=</span> <span class="fu">c</span>(best_novelty_vars,<span class="st">&quot;Sex&quot;</span>)</span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a>saline_cancor_vars_SAL <span class="ot">=</span> <span class="fu">c</span>(best_saline_vars,<span class="st">&quot;Sex&quot;</span>)</span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a>saline_cancor_rows_SAL <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_saline_novelty[,<span class="fu">c</span>(novelty_cancor_vars_SAL,saline_cancor_vars_SAL)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb22-7"><a href="#cb22-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-8"><a href="#cb22-8" aria-hidden="true" tabindex="-1"></a>saline_novelty_cancor_SAL <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_saline_novelty[saline_cancor_rows_SAL,saline_cancor_vars_SAL],</span>
<span id="cb22-9"><a href="#cb22-9" aria-hidden="true" tabindex="-1"></a>                                 DO_saline_novelty[saline_cancor_rows_SAL,novelty_cancor_vars_SAL],</span>
<span id="cb22-10"><a href="#cb22-10" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb22-11"><a href="#cb22-11" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="st">&quot;The first canonical correlate of these saline variables to novelty variables has a magnitude of &quot;</span>, </span>
<span id="cb22-12"><a href="#cb22-12" aria-hidden="true" tabindex="-1"></a>      <span class="fu">round</span>(saline_novelty_cancor_SAL<span class="sc">$</span>cor[<span class="dv">2</span>], <span class="at">digits =</span> <span class="dv">3</span>), </span>
<span id="cb22-13"><a href="#cb22-13" aria-hidden="true" tabindex="-1"></a>      <span class="st">&quot;.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;The first canonical correlate of these saline variables to novelty variables has a magnitude of 0.651.&quot;</code></pre>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="st">&quot;This canonical correlation is based upon &quot;</span>,</span>
<span id="cb24-2"><a href="#cb24-2" aria-hidden="true" tabindex="-1"></a>      <span class="fu">length</span>(saline_cancor_rows_SAL), <span class="st">&quot; complete observations.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;This canonical correlation is based upon 237 complete observations.&quot;</code></pre>
<p>This is an adequate fit for the canonical correlation. Plotting the
variables.</p>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a><span class="fu">source</span>(<span class="st">&quot;./code/ggplot_pub_themes.R&quot;</span>)</span></code></pre></div>
<pre><code>Warning: The `size` argument of `element_line()` is deprecated as of ggplot2 3.4.0.
ℹ Please use the `linewidth` argument instead.
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
generated.</code></pre>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a>saline_cancor <span class="ot">=</span> <span class="fu">as.matrix</span>(DO_saline_novelty[saline_cancor_rows_SAL,saline_cancor_vars_SAL]) <span class="sc">%*%</span> saline_novelty_cancor_SAL<span class="sc">$</span>xcoef[,<span class="dv">2</span>]</span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>novelty_cancor <span class="ot">=</span> <span class="fu">as.matrix</span>(DO_saline_novelty[saline_cancor_rows_SAL,novelty_cancor_vars_SAL]) <span class="sc">%*%</span> saline_novelty_cancor_SAL<span class="sc">$</span>ycoef[,<span class="dv">2</span>]</span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a>saline_novelty_cancor_SAL_df <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">saline_cancor =</span> saline_cancor, <span class="at">novelty_cancor =</span> novelty_cancor)</span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>saline_novelty_cancor_SAL_df<span class="sc">$</span>Sex <span class="ot">=</span> DO_saline_novelty[<span class="fu">row.names</span>(saline_novelty_cancor_SAL_df),<span class="st">&quot;DE_Sex&quot;</span>]</span>
<span id="cb28-6"><a href="#cb28-6" aria-hidden="true" tabindex="-1"></a>saline_novelty_cancor_SAL_df<span class="sc">$</span>Subject <span class="ot">=</span> <span class="fu">as.numeric</span>(<span class="fu">as.character</span>(<span class="fu">row.names</span>(saline_novelty_cancor_SAL_df)))</span>
<span id="cb28-7"><a href="#cb28-7" aria-hidden="true" tabindex="-1"></a>saline_novelty_cancor_SAL_df<span class="sc">$</span>Acquired <span class="ot">=</span> DO_saline_novelty[<span class="fu">row.names</span>(saline_novelty_cancor_SAL_df),<span class="st">&quot;DE_Exit_Reason&quot;</span>]</span>
<span id="cb28-8"><a href="#cb28-8" aria-hidden="true" tabindex="-1"></a>saline_novelty_cancor_lm <span class="ot">=</span> <span class="fu">lm</span>(saline_cancor <span class="sc">~</span> novelty_cancor, <span class="at">data =</span> saline_novelty_cancor_SAL_df)</span>
<span id="cb28-9"><a href="#cb28-9" aria-hidden="true" tabindex="-1"></a>novelty_saline_SAL_cancor_plot <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> saline_novelty_cancor_SAL_df, <span class="fu">aes</span>(<span class="at">x =</span> novelty_cancor, <span class="at">y =</span> saline_cancor, <span class="at">color =</span> Sex)) <span class="sc">+</span></span>
<span id="cb28-10"><a href="#cb28-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">size =</span> (pt_to_mm <span class="sc">*</span> <span class="fl">1.5</span>)) <span class="sc">+</span></span>
<span id="cb28-11"><a href="#cb28-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_abline</span>(<span class="at">slope =</span> saline_novelty_cancor_lm<span class="sc">$</span>coefficients[<span class="st">&quot;novelty_cancor&quot;</span>], </span>
<span id="cb28-12"><a href="#cb28-12" aria-hidden="true" tabindex="-1"></a>              <span class="at">intercept =</span> saline_novelty_cancor_lm<span class="sc">$</span>coefficients[<span class="st">&quot;(Intercept)&quot;</span>],</span>
<span id="cb28-13"><a href="#cb28-13" aria-hidden="true" tabindex="-1"></a>              <span class="at">size =</span> (<span class="dv">1</span> <span class="sc">*</span> pt_to_mm)) <span class="sc">+</span></span>
<span id="cb28-14"><a href="#cb28-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#3399CC&quot;</span>,<span class="st">&quot;#66CC66&quot;</span>)) <span class="sc">+</span></span>
<span id="cb28-15"><a href="#cb28-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Novelty Canonical Correlate&quot;</span>) <span class="sc">+</span></span>
<span id="cb28-16"><a href="#cb28-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Saline Canonical Correlate&quot;</span>)</span></code></pre></div>
<pre><code>Warning: Using `size` aesthetic for lines was deprecated in ggplot2 3.4.0.
ℹ Please use `linewidth` instead.
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
generated.</code></pre>
<div class="sourceCode" id="cb30"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1" aria-hidden="true" tabindex="-1"></a>novelty_saline_SAL_cancor_plot <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">panel.grid =</span> <span class="fu">element_line</span>(<span class="at">color =</span> <span class="st">&quot;#FFFFFF&quot;</span>))</span></code></pre></div>
<p><img src="figure/22_DO_RTA_saline_cocaine.Rmd/unnamed-chunk-8-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-8-1">
Past versions of unnamed-chunk-8-1.png
</button>
</p>
<div id="fig-unnamed-chunk-8-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
96203d5
</td>
<td>
Michael C. Saul
</td>
<td>
2024-04-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cocaine-rta" class="section level3">
<h3>Cocaine RTA</h3>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Getting highest correlation variables to start out</span></span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a>highest_cor <span class="ot">=</span> <span class="fu">cor</span>(DO_cocaine_novelty[,<span class="st">&quot;dist_d3_d11_auc_cocaine_do&quot;</span>], </span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a>                  DO_cocaine_novelty[,novelty_columns],</span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a>                  <span class="at">use =</span> <span class="st">&quot;pairwise.complete.obs&quot;</span>)</span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a>highest_cor_r <span class="ot">=</span> <span class="fu">max</span>(<span class="fu">abs</span>(highest_cor))</span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a>highest_cor_ind <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">abs</span>(highest_cor) <span class="sc">==</span> highest_cor_r, <span class="at">arr.ind =</span> <span class="cn">TRUE</span>)</span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a>cocaine_highcor_id <span class="ot">=</span> <span class="st">&quot;dist_d3_d11_auc_cocaine_do&quot;</span></span>
<span id="cb31-8"><a href="#cb31-8" aria-hidden="true" tabindex="-1"></a>novelty_highcor_id <span class="ot">=</span> <span class="fu">colnames</span>(highest_cor)[highest_cor_ind[<span class="dv">1</span>,<span class="st">&quot;col&quot;</span>]]</span>
<span id="cb31-9"><a href="#cb31-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-10"><a href="#cb31-10" aria-hidden="true" tabindex="-1"></a><span class="fu">paste0</span>(<span class="st">&quot;The highest correlation (&quot;</span>, <span class="fu">round</span>(highest_cor_r, <span class="dv">2</span>),<span class="st">&quot;) is between &quot;</span>, cocaine_highcor_id, <span class="st">&quot; and &quot;</span>,</span>
<span id="cb31-11"><a href="#cb31-11" aria-hidden="true" tabindex="-1"></a>       novelty_highcor_id, <span class="st">&quot;.&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;The highest correlation (0.53) is between dist_d3_d11_auc_cocaine_do and OFA_total_distance_traveled_batch_ranknorm.&quot;</code></pre>
<p>Running canonical correlation on cocaine-treated animals</p>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Starting with these as best novelty and IVSA variables</span></span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>best_novelty_vars <span class="ot">=</span> novelty_highcor_id</span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a>best_cocaine_vars <span class="ot">=</span> cocaine_highcor_id</span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a>noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_cocaine_novelty[,<span class="fu">c</span>(best_novelty_vars, best_cocaine_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a>starting_cancor <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_cocaine_novelty[noNA_rows,best_novelty_vars], </span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a>                                 DO_cocaine_novelty[noNA_rows,best_cocaine_vars],</span>
<span id="cb33-7"><a href="#cb33-7" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb33-8"><a href="#cb33-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-9"><a href="#cb33-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Selecting cocaine sensi variables</span></span>
<span id="cb33-10"><a href="#cb33-10" aria-hidden="true" tabindex="-1"></a>cocaine_vars_unselected <span class="ot">=</span> <span class="fu">c</span>(<span class="st">&quot;dist_d3_d2_cocaine_do&quot;</span>,<span class="st">&quot;dist_d5_d3_cocaine_do&quot;</span>,<span class="st">&quot;dist_d12_d2_cocaine_do&quot;</span>,<span class="st">&quot;dist_d19_d11_cocaine_do&quot;</span>)</span>
<span id="cb33-11"><a href="#cb33-11" aria-hidden="true" tabindex="-1"></a>novelty_vars_unselected <span class="ot">=</span> novelty_columns[(<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(novelty_columns <span class="sc">==</span> best_novelty_vars))]</span>
<span id="cb33-12"><a href="#cb33-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-13"><a href="#cb33-13" aria-hidden="true" tabindex="-1"></a>not_optimized <span class="ot">=</span> <span class="cn">TRUE</span></span>
<span id="cb33-14"><a href="#cb33-14" aria-hidden="true" tabindex="-1"></a>min_obs <span class="ot">=</span> <span class="dv">225</span></span>
<span id="cb33-15"><a href="#cb33-15" aria-hidden="true" tabindex="-1"></a>max_var <span class="ot">=</span> <span class="dv">10</span></span>
<span id="cb33-16"><a href="#cb33-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-17"><a href="#cb33-17" aria-hidden="true" tabindex="-1"></a><span class="cf">while</span> (not_optimized) {</span>
<span id="cb33-18"><a href="#cb33-18" aria-hidden="true" tabindex="-1"></a>  best_cancor <span class="ot">=</span> starting_cancor<span class="sc">$</span>cor[<span class="dv">1</span>]</span>
<span id="cb33-19"><a href="#cb33-19" aria-hidden="true" tabindex="-1"></a>  best_var <span class="ot">=</span> <span class="st">&quot;&quot;</span></span>
<span id="cb33-20"><a href="#cb33-20" aria-hidden="true" tabindex="-1"></a>  best_paradigm <span class="ot">=</span> <span class="st">&quot;&quot;</span></span>
<span id="cb33-21"><a href="#cb33-21" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb33-22"><a href="#cb33-22" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (i <span class="cf">in</span> novelty_vars_unselected) {</span>
<span id="cb33-23"><a href="#cb33-23" aria-hidden="true" tabindex="-1"></a>    noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_cocaine_novelty[,<span class="fu">c</span>(best_novelty_vars, i, best_cocaine_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb33-24"><a href="#cb33-24" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="fu">length</span>(noNA_rows) <span class="sc">&gt;=</span> min_obs) {</span>
<span id="cb33-25"><a href="#cb33-25" aria-hidden="true" tabindex="-1"></a>      cancor_i <span class="ot">=</span> stats<span class="sc">::</span><span class="fu">cancor</span>(DO_cocaine_novelty[noNA_rows, <span class="fu">c</span>(best_novelty_vars, i)], DO_cocaine_novelty[noNA_rows, best_cocaine_vars],</span>
<span id="cb33-26"><a href="#cb33-26" aria-hidden="true" tabindex="-1"></a>                        <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb33-27"><a href="#cb33-27" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span> (cancor_i<span class="sc">$</span>cor[<span class="dv">1</span>] <span class="sc">&gt;</span> best_cancor) {</span>
<span id="cb33-28"><a href="#cb33-28" aria-hidden="true" tabindex="-1"></a>        best_cancor <span class="ot">=</span> cancor_i<span class="sc">$</span>cor[<span class="dv">1</span>]</span>
<span id="cb33-29"><a href="#cb33-29" aria-hidden="true" tabindex="-1"></a>        best_var <span class="ot">=</span> i</span>
<span id="cb33-30"><a href="#cb33-30" aria-hidden="true" tabindex="-1"></a>        best_paradigm <span class="ot">=</span> <span class="st">&quot;novelty&quot;</span></span>
<span id="cb33-31"><a href="#cb33-31" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb33-32"><a href="#cb33-32" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb33-33"><a href="#cb33-33" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb33-34"><a href="#cb33-34" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb33-35"><a href="#cb33-35" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (j <span class="cf">in</span> cocaine_vars_unselected) {</span>
<span id="cb33-36"><a href="#cb33-36" aria-hidden="true" tabindex="-1"></a>    noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_cocaine_novelty[,<span class="fu">c</span>(best_novelty_vars, j, best_cocaine_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb33-37"><a href="#cb33-37" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="fu">length</span>(noNA_rows) <span class="sc">&gt;=</span> min_obs) {</span>
<span id="cb33-38"><a href="#cb33-38" aria-hidden="true" tabindex="-1"></a>      cancor_j <span class="ot">=</span> stats<span class="sc">::</span><span class="fu">cancor</span>(DO_cocaine_novelty[noNA_rows, best_novelty_vars], DO_cocaine_novelty[noNA_rows,<span class="fu">c</span>(best_cocaine_vars, j)],</span>
<span id="cb33-39"><a href="#cb33-39" aria-hidden="true" tabindex="-1"></a>                        <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb33-40"><a href="#cb33-40" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span> (cancor_j<span class="sc">$</span>cor[<span class="dv">1</span>] <span class="sc">&gt;</span> best_cancor) {</span>
<span id="cb33-41"><a href="#cb33-41" aria-hidden="true" tabindex="-1"></a>        best_cancor <span class="ot">=</span> cancor_j<span class="sc">$</span>cor[<span class="dv">1</span>]</span>
<span id="cb33-42"><a href="#cb33-42" aria-hidden="true" tabindex="-1"></a>        best_var <span class="ot">=</span> j</span>
<span id="cb33-43"><a href="#cb33-43" aria-hidden="true" tabindex="-1"></a>        best_paradigm <span class="ot">=</span> <span class="st">&quot;cocaine&quot;</span></span>
<span id="cb33-44"><a href="#cb33-44" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb33-45"><a href="#cb33-45" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb33-46"><a href="#cb33-46" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb33-47"><a href="#cb33-47" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb33-48"><a href="#cb33-48" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;novelty&quot;</span>) {</span>
<span id="cb33-49"><a href="#cb33-49" aria-hidden="true" tabindex="-1"></a>    novelty_vars_test <span class="ot">=</span> <span class="fu">c</span>(best_novelty_vars, best_var)</span>
<span id="cb33-50"><a href="#cb33-50" aria-hidden="true" tabindex="-1"></a>    cocaine_vars_test <span class="ot">=</span> best_cocaine_vars</span>
<span id="cb33-51"><a href="#cb33-51" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;cocaine&quot;</span>) {</span>
<span id="cb33-52"><a href="#cb33-52" aria-hidden="true" tabindex="-1"></a>    novelty_vars_test <span class="ot">=</span> best_novelty_vars</span>
<span id="cb33-53"><a href="#cb33-53" aria-hidden="true" tabindex="-1"></a>    cocaine_vars_test <span class="ot">=</span> <span class="fu">c</span>(best_cocaine_vars, best_var)</span>
<span id="cb33-54"><a href="#cb33-54" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb33-55"><a href="#cb33-55" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb33-56"><a href="#cb33-56" aria-hidden="true" tabindex="-1"></a>  noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_cocaine_novelty[,<span class="fu">c</span>(novelty_vars_test, cocaine_vars_test)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb33-57"><a href="#cb33-57" aria-hidden="true" tabindex="-1"></a>  k_gtr <span class="ot">=</span> <span class="fu">c</span>()</span>
<span id="cb33-58"><a href="#cb33-58" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb33-59"><a href="#cb33-59" aria-hidden="true" tabindex="-1"></a>  B <span class="ot">=</span> <span class="fl">1e4</span></span>
<span id="cb33-60"><a href="#cb33-60" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (k <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span>B) {</span>
<span id="cb33-61"><a href="#cb33-61" aria-hidden="true" tabindex="-1"></a>    cancor_k <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_cocaine_novelty[<span class="fu">sample</span>(noNA_rows), novelty_vars_test], </span>
<span id="cb33-62"><a href="#cb33-62" aria-hidden="true" tabindex="-1"></a>                      DO_cocaine_novelty[noNA_rows, cocaine_vars_test],</span>
<span id="cb33-63"><a href="#cb33-63" aria-hidden="true" tabindex="-1"></a>                      <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb33-64"><a href="#cb33-64" aria-hidden="true" tabindex="-1"></a>    k_gtr <span class="ot">=</span> <span class="fu">c</span>(k_gtr, cancor_k<span class="sc">$</span>cor[<span class="dv">1</span>])</span>
<span id="cb33-65"><a href="#cb33-65" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb33-66"><a href="#cb33-66" aria-hidden="true" tabindex="-1"></a>  p_value_improvement <span class="ot">=</span> (<span class="fu">length</span>(<span class="fu">which</span>(k_gtr <span class="sc">&gt;</span> best_cancor)) <span class="sc">+</span> <span class="dv">1</span>) <span class="sc">/</span> (B <span class="sc">+</span> <span class="dv">1</span>)</span>
<span id="cb33-67"><a href="#cb33-67" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb33-68"><a href="#cb33-68" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> ((<span class="fu">length</span>(best_novelty_vars) <span class="sc">+</span> <span class="fu">length</span>(best_cocaine_vars)) <span class="sc">&gt;</span> max_var) {</span>
<span id="cb33-69"><a href="#cb33-69" aria-hidden="true" tabindex="-1"></a>    not_optimized <span class="ot">=</span> <span class="cn">FALSE</span></span>
<span id="cb33-70"><a href="#cb33-70" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> <span class="cf">if</span> (p_value_improvement <span class="sc">&lt;</span> <span class="fl">0.001</span>) {</span>
<span id="cb33-71"><a href="#cb33-71" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;novelty&quot;</span>) {</span>
<span id="cb33-72"><a href="#cb33-72" aria-hidden="true" tabindex="-1"></a>      best_novelty_vars <span class="ot">=</span> <span class="fu">c</span>(best_novelty_vars, best_var)</span>
<span id="cb33-73"><a href="#cb33-73" aria-hidden="true" tabindex="-1"></a>      novelty_vars_unselected <span class="ot">=</span> novelty_vars_unselected[<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(novelty_vars_unselected <span class="sc">==</span> best_var)]</span>
<span id="cb33-74"><a href="#cb33-74" aria-hidden="true" tabindex="-1"></a>    } <span class="cf">else</span> <span class="cf">if</span> (best_paradigm <span class="sc">==</span> <span class="st">&quot;cocaine&quot;</span>) {</span>
<span id="cb33-75"><a href="#cb33-75" aria-hidden="true" tabindex="-1"></a>      best_cocaine_vars <span class="ot">=</span> <span class="fu">c</span>(best_cocaine_vars, best_var)</span>
<span id="cb33-76"><a href="#cb33-76" aria-hidden="true" tabindex="-1"></a>      cocaine_vars_unselected <span class="ot">=</span> cocaine_vars_unselected[<span class="sc">-</span><span class="dv">1</span> <span class="sc">*</span> <span class="fu">which</span>(cocaine_vars_unselected <span class="sc">==</span> best_var)]</span>
<span id="cb33-77"><a href="#cb33-77" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb33-78"><a href="#cb33-78" aria-hidden="true" tabindex="-1"></a>    starting_cancor <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_cocaine_novelty[noNA_rows,best_novelty_vars], DO_cocaine_novelty[noNA_rows,best_cocaine_vars],</span>
<span id="cb33-79"><a href="#cb33-79" aria-hidden="true" tabindex="-1"></a>                             <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb33-80"><a href="#cb33-80" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb33-81"><a href="#cb33-81" aria-hidden="true" tabindex="-1"></a>    not_optimized <span class="ot">=</span> <span class="cn">FALSE</span></span>
<span id="cb33-82"><a href="#cb33-82" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb33-83"><a href="#cb33-83" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb33-84"><a href="#cb33-84" aria-hidden="true" tabindex="-1"></a>noNA_rows <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_cocaine_novelty[,<span class="fu">c</span>(best_novelty_vars, best_cocaine_vars)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb33-85"><a href="#cb33-85" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-86"><a href="#cb33-86" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Best novelty variables:</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="fu">paste</span>(best_novelty_vars, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>, <span class="at">collapse =</span> <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>),<span class="st">&quot;</span><span class="sc">\n\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Best novelty variables:
OFA_total_distance_traveled_batch_ranknorm
OFA_distance_traveled_slope_batch_ranknorm
LD_pct_ambulatory_Counts_in_light_batch_ranknorm
OFA_total_resting_time_in_perimeter_batch_ranknorm
LD_total_transitions_batch_ranknorm
OFA_total_distance_traveled_in_center_batch_ranknorm
OFA_pct_time_center_batch_ranknorm</code></pre>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Best cocaine variables:</span><span class="sc">\n</span><span class="st">&quot;</span>, <span class="fu">paste</span>(best_cocaine_vars, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>, <span class="at">collapse =</span> <span class="st">&quot;</span><span class="sc">\n</span><span class="st">&quot;</span>), <span class="st">&quot;</span><span class="sc">\n\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Best cocaine variables:
dist_d3_d11_auc_cocaine_do
dist_d3_d2_cocaine_do
dist_d5_d3_cocaine_do
dist_d19_d11_cocaine_do</code></pre>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;First canonical correlate magnitude: &quot;</span>, <span class="fu">round</span>(starting_cancor<span class="sc">$</span>cor[<span class="dv">1</span>], <span class="dv">4</span>), <span class="st">&quot;</span><span class="sc">\n\n</span><span class="st">&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>First canonical correlate magnitude: 0.6679</code></pre>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;Canonical correlation trained on &quot;</span>, <span class="fu">length</span>(noNA_rows), <span class="st">&quot; complete observations.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>Canonical correlation trained on 261 complete observations.</code></pre>
<p>Adjusting this canonical correlation for sex.</p>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb41-1"><a href="#cb41-1" aria-hidden="true" tabindex="-1"></a>DO_cocaine_novelty<span class="sc">$</span>DE_Sex <span class="ot">=</span> DO_cocaine_novelty<span class="sc">$</span>Sex</span>
<span id="cb41-2"><a href="#cb41-2" aria-hidden="true" tabindex="-1"></a>DO_cocaine_novelty<span class="sc">$</span>Sex <span class="ot">=</span> <span class="fu">ifelse</span>(DO_cocaine_novelty<span class="sc">$</span>DE_Sex <span class="sc">==</span> <span class="st">&quot;M&quot;</span>, <span class="dv">1</span>, <span class="dv">0</span>)</span>
<span id="cb41-3"><a href="#cb41-3" aria-hidden="true" tabindex="-1"></a>novelty_cancor_vars_COC <span class="ot">=</span> <span class="fu">c</span>(best_novelty_vars,<span class="st">&quot;Sex&quot;</span>)</span>
<span id="cb41-4"><a href="#cb41-4" aria-hidden="true" tabindex="-1"></a>cocaine_cancor_vars_COC <span class="ot">=</span> <span class="fu">c</span>(best_cocaine_vars,<span class="st">&quot;Sex&quot;</span>)</span>
<span id="cb41-5"><a href="#cb41-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-6"><a href="#cb41-6" aria-hidden="true" tabindex="-1"></a>cocaine_cancor_rows_COC <span class="ot">=</span> <span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_cocaine_novelty[,<span class="fu">c</span>(novelty_cancor_vars_COC,cocaine_cancor_vars_COC)])) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb41-7"><a href="#cb41-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-8"><a href="#cb41-8" aria-hidden="true" tabindex="-1"></a>cocaine_novelty_cancor_COC <span class="ot">=</span>  stats<span class="sc">::</span><span class="fu">cancor</span>(DO_cocaine_novelty[cocaine_cancor_rows_COC,cocaine_cancor_vars_COC],</span>
<span id="cb41-9"><a href="#cb41-9" aria-hidden="true" tabindex="-1"></a>                                 DO_cocaine_novelty[cocaine_cancor_rows_COC,novelty_cancor_vars_COC],</span>
<span id="cb41-10"><a href="#cb41-10" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">xcenter =</span> <span class="cn">TRUE</span>, <span class="at">ycenter =</span> <span class="cn">TRUE</span>)</span>
<span id="cb41-11"><a href="#cb41-11" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="st">&quot;The first canonical correlate of these cocaine variables to novelty variables has a magnitude of &quot;</span>, </span>
<span id="cb41-12"><a href="#cb41-12" aria-hidden="true" tabindex="-1"></a>      <span class="fu">round</span>(cocaine_novelty_cancor_COC<span class="sc">$</span>cor[<span class="dv">2</span>], <span class="at">digits =</span> <span class="dv">3</span>), </span>
<span id="cb41-13"><a href="#cb41-13" aria-hidden="true" tabindex="-1"></a>      <span class="st">&quot;.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;The first canonical correlate of these cocaine variables to novelty variables has a magnitude of 0.674.&quot;</code></pre>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1" aria-hidden="true" tabindex="-1"></a><span class="fu">paste</span>(<span class="st">&quot;This canonical correlation is based upon &quot;</span>,</span>
<span id="cb43-2"><a href="#cb43-2" aria-hidden="true" tabindex="-1"></a>      <span class="fu">length</span>(cocaine_cancor_rows_COC), <span class="st">&quot; complete observations.&quot;</span>, <span class="at">sep =</span> <span class="st">&quot;&quot;</span>)</span></code></pre></div>
<pre><code>[1] &quot;This canonical correlation is based upon 261 complete observations.&quot;</code></pre>
<p>This is an adequate fit for the canonical correlation. Plotting the
variables.</p>
<div class="sourceCode" id="cb45"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1" aria-hidden="true" tabindex="-1"></a><span class="fu">source</span>(<span class="st">&quot;./code/ggplot_pub_themes.R&quot;</span>)</span>
<span id="cb45-2"><a href="#cb45-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb45-3"><a href="#cb45-3" aria-hidden="true" tabindex="-1"></a>cocaine_cancor <span class="ot">=</span> <span class="fu">as.matrix</span>(DO_cocaine_novelty[cocaine_cancor_rows_COC,cocaine_cancor_vars_COC]) <span class="sc">%*%</span> cocaine_novelty_cancor_COC<span class="sc">$</span>xcoef[,<span class="dv">2</span>]</span>
<span id="cb45-4"><a href="#cb45-4" aria-hidden="true" tabindex="-1"></a>novelty_cancor <span class="ot">=</span> <span class="fu">as.matrix</span>(DO_cocaine_novelty[cocaine_cancor_rows_COC,novelty_cancor_vars_COC]) <span class="sc">%*%</span> cocaine_novelty_cancor_COC<span class="sc">$</span>ycoef[,<span class="dv">2</span>]</span>
<span id="cb45-5"><a href="#cb45-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb45-6"><a href="#cb45-6" aria-hidden="true" tabindex="-1"></a>cocaine_novelty_cancor_COC_df <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">cocaine_cancor =</span> cocaine_cancor, <span class="at">novelty_cancor =</span> novelty_cancor)</span>
<span id="cb45-7"><a href="#cb45-7" aria-hidden="true" tabindex="-1"></a>cocaine_novelty_cancor_COC_df<span class="sc">$</span>Sex <span class="ot">=</span> DO_cocaine_novelty[<span class="fu">row.names</span>(cocaine_novelty_cancor_COC_df),<span class="st">&quot;DE_Sex&quot;</span>]</span>
<span id="cb45-8"><a href="#cb45-8" aria-hidden="true" tabindex="-1"></a>cocaine_novelty_cancor_COC_df<span class="sc">$</span>Subject <span class="ot">=</span> <span class="fu">as.numeric</span>(<span class="fu">as.character</span>(<span class="fu">row.names</span>(cocaine_novelty_cancor_COC_df)))</span>
<span id="cb45-9"><a href="#cb45-9" aria-hidden="true" tabindex="-1"></a>cocaine_novelty_cancor_COC_df<span class="sc">$</span>Acquired <span class="ot">=</span> DO_cocaine_novelty[<span class="fu">row.names</span>(cocaine_novelty_cancor_COC_df),<span class="st">&quot;DE_Exit_Reason&quot;</span>]</span>
<span id="cb45-10"><a href="#cb45-10" aria-hidden="true" tabindex="-1"></a>cocaine_novelty_cancor_lm <span class="ot">=</span> <span class="fu">lm</span>(cocaine_cancor <span class="sc">~</span> novelty_cancor, <span class="at">data =</span> cocaine_novelty_cancor_COC_df)</span>
<span id="cb45-11"><a href="#cb45-11" aria-hidden="true" tabindex="-1"></a>novelty_cocaine_COC_cancor_plot <span class="ot">=</span> <span class="fu">ggplot</span>(<span class="at">data =</span> cocaine_novelty_cancor_COC_df, <span class="fu">aes</span>(<span class="at">x =</span> novelty_cancor, <span class="at">y =</span> cocaine_cancor, <span class="at">color =</span> Sex)) <span class="sc">+</span></span>
<span id="cb45-12"><a href="#cb45-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">size =</span> (pt_to_mm <span class="sc">*</span> <span class="fl">1.5</span>)) <span class="sc">+</span></span>
<span id="cb45-13"><a href="#cb45-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_abline</span>(<span class="at">slope =</span> cocaine_novelty_cancor_lm<span class="sc">$</span>coefficients[<span class="st">&quot;novelty_cancor&quot;</span>], </span>
<span id="cb45-14"><a href="#cb45-14" aria-hidden="true" tabindex="-1"></a>              <span class="at">intercept =</span> cocaine_novelty_cancor_lm<span class="sc">$</span>coefficients[<span class="st">&quot;(Intercept)&quot;</span>],</span>
<span id="cb45-15"><a href="#cb45-15" aria-hidden="true" tabindex="-1"></a>              <span class="at">size =</span> (<span class="dv">1</span> <span class="sc">*</span> pt_to_mm)) <span class="sc">+</span></span>
<span id="cb45-16"><a href="#cb45-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">&quot;#3399CC&quot;</span>,<span class="st">&quot;#66CC66&quot;</span>)) <span class="sc">+</span></span>
<span id="cb45-17"><a href="#cb45-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Novelty Canonical Correlate&quot;</span>) <span class="sc">+</span></span>
<span id="cb45-18"><a href="#cb45-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Cocaine Sensitization Canonical Correlate&quot;</span>)</span>
<span id="cb45-19"><a href="#cb45-19" aria-hidden="true" tabindex="-1"></a>novelty_cocaine_COC_cancor_plot <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">panel.grid =</span> <span class="fu">element_line</span>(<span class="at">color =</span> <span class="st">&quot;#FFFFFF&quot;</span>))</span></code></pre></div>
<p><img src="figure/22_DO_RTA_saline_cocaine.Rmd/unnamed-chunk-12-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-12-1">
Past versions of unnamed-chunk-12-1.png
</button>
</p>
<div id="fig-unnamed-chunk-12-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
96203d5
</td>
<td>
Michael C. Saul
</td>
<td>
2024-04-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="making-predictions" class="section level3">
<h3>Making Predictions</h3>
<p>Generating predicted variables from these data.</p>
<div class="sourceCode" id="cb46"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1" aria-hidden="true" tabindex="-1"></a>DO_novelty <span class="ot">=</span> DO_novelty_response</span>
<span id="cb46-2"><a href="#cb46-2" aria-hidden="true" tabindex="-1"></a>DO_novelty<span class="sc">$</span>Sex <span class="ot">=</span> <span class="fu">ifelse</span>(DO_novelty<span class="sc">$</span>Sex <span class="sc">==</span> <span class="st">&quot;M&quot;</span>, <span class="dv">1</span>, <span class="dv">0</span>)</span>
<span id="cb46-3"><a href="#cb46-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-4"><a href="#cb46-4" aria-hidden="true" tabindex="-1"></a><span class="co"># saline</span></span>
<span id="cb46-5"><a href="#cb46-5" aria-hidden="true" tabindex="-1"></a>DO_predict_rows_SAL <span class="ot">=</span> <span class="fu">row.names</span>(DO_novelty)[<span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_novelty[,novelty_cancor_vars_SAL])) <span class="sc">==</span> <span class="dv">0</span>)]</span>
<span id="cb46-6"><a href="#cb46-6" aria-hidden="true" tabindex="-1"></a>saline_prediction_SAL <span class="ot">=</span> <span class="fu">as.numeric</span>(<span class="fu">as.matrix</span>(DO_novelty[DO_predict_rows_SAL,novelty_cancor_vars_SAL]) <span class="sc">%*%</span> saline_novelty_cancor_SAL<span class="sc">$</span>ycoef[,<span class="dv">2</span>])</span>
<span id="cb46-7"><a href="#cb46-7" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(saline_prediction_SAL) <span class="ot">=</span> DO_predict_rows_SAL</span>
<span id="cb46-8"><a href="#cb46-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-9"><a href="#cb46-9" aria-hidden="true" tabindex="-1"></a><span class="co"># cocaine</span></span>
<span id="cb46-10"><a href="#cb46-10" aria-hidden="true" tabindex="-1"></a>DO_predict_rows_COC <span class="ot">=</span> <span class="fu">row.names</span>(DO_novelty)[<span class="fu">which</span>(<span class="fu">rowSums</span>(<span class="fu">is.na</span>(DO_novelty[,novelty_cancor_vars_COC])) <span class="sc">==</span> <span class="dv">0</span>)]</span>
<span id="cb46-11"><a href="#cb46-11" aria-hidden="true" tabindex="-1"></a>cocaine_prediction_COC <span class="ot">=</span> <span class="fu">as.numeric</span>(<span class="fu">as.matrix</span>(DO_novelty[DO_predict_rows_COC,novelty_cancor_vars_COC]) <span class="sc">%*%</span> cocaine_novelty_cancor_COC<span class="sc">$</span>ycoef[,<span class="dv">2</span>])</span>
<span id="cb46-12"><a href="#cb46-12" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(cocaine_prediction_COC) <span class="ot">=</span> DO_predict_rows_COC</span>
<span id="cb46-13"><a href="#cb46-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-14"><a href="#cb46-14" aria-hidden="true" tabindex="-1"></a><span class="co"># union</span></span>
<span id="cb46-15"><a href="#cb46-15" aria-hidden="true" tabindex="-1"></a>DO_predict_rows_ALL <span class="ot">=</span> <span class="fu">unique</span>(<span class="fu">c</span>(DO_predict_rows_SAL, DO_predict_rows_COC))</span>
<span id="cb46-16"><a href="#cb46-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-17"><a href="#cb46-17" aria-hidden="true" tabindex="-1"></a>saline_cocaine_pred <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">row.names =</span> DO_predict_rows_ALL,</span>
<span id="cb46-18"><a href="#cb46-18" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">Mouse_ID =</span> DO_predict_rows_ALL,</span>
<span id="cb46-19"><a href="#cb46-19" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">Sex =</span> DO_novelty[DO_predict_rows_ALL,<span class="st">&quot;Sex&quot;</span>],</span>
<span id="cb46-20"><a href="#cb46-20" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">Generation =</span> DO_novelty[DO_predict_rows_ALL,<span class="st">&quot;Generation&quot;</span>],</span>
<span id="cb46-21"><a href="#cb46-21" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">saline_novelty_cancor =</span> saline_prediction_SAL[DO_predict_rows_ALL],</span>
<span id="cb46-22"><a href="#cb46-22" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">cocaine_novelty_cancor =</span> cocaine_prediction_COC[DO_predict_rows_ALL],</span>
<span id="cb46-23"><a href="#cb46-23" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>)</span>
<span id="cb46-24"><a href="#cb46-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-25"><a href="#cb46-25" aria-hidden="true" tabindex="-1"></a>today <span class="ot">=</span> <span class="fu">as.character</span>(lubridate<span class="sc">::</span><span class="fu">today</span>())</span>
<span id="cb46-26"><a href="#cb46-26" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(saline_cocaine_pred, <span class="fu">paste0</span>(<span class="st">&quot;./data/DO_novelty_saline_cocaine_cancor_&quot;</span>,today,<span class="st">&quot;.RDS&quot;</span>))</span>
<span id="cb46-27"><a href="#cb46-27" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(cocaine_novelty_cancor_COC, <span class="fu">paste0</span>(<span class="st">&quot;./data/DO_novelty_COC_cancor_&quot;</span>,today,<span class="st">&quot;.RDS&quot;</span>))</span>
<span id="cb46-28"><a href="#cb46-28" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(saline_novelty_cancor_SAL, <span class="fu">paste0</span>(<span class="st">&quot;./data/DO_novelty_SAL_cancor_&quot;</span>,today,<span class="st">&quot;.RDS&quot;</span>))</span></code></pre></div>
</div>
<div id="r-session-and-os-information" class="section level3">
<h3>R session and OS information</h3>
<div class="sourceCode" id="cb47"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb47-1"><a href="#cb47-1" aria-hidden="true" tabindex="-1"></a>novelty_saline_cancor_sessionInfo <span class="ot">=</span> <span class="fu">sessionInfo</span>()</span>
<span id="cb47-2"><a href="#cb47-2" aria-hidden="true" tabindex="-1"></a>novelty_saline_cancor_sessionInfo</span></code></pre></div>
<pre><code>R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.6 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/liblapack.so.3

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] ggbeeswarm_0.7.2 MASS_7.3-57      cowplot_1.1.1    lubridate_1.8.0 
 [5] forcats_0.5.2    stringr_1.5.0    purrr_1.0.1      readr_2.1.3     
 [9] tibble_3.2.1     ggplot2_3.4.2    tidyverse_1.3.2  tidyr_1.3.0     
[13] dplyr_1.1.2     

loaded via a namespace (and not attached):
 [1] httr_1.4.4          sass_0.4.2          bit64_4.0.5        
 [4] vroom_1.6.0         jsonlite_1.8.3      modelr_0.1.9       
 [7] bslib_0.4.0         assertthat_0.2.1    highr_0.9          
[10] vipor_0.4.5         googlesheets4_1.0.1 cellranger_1.1.0   
[13] yaml_2.3.6          pillar_1.9.0        backports_1.4.1    
[16] glue_1.6.2          digest_0.6.30       promises_1.2.0.1   
[19] rvest_1.0.3         colorspace_2.0-3    htmltools_0.5.3    
[22] httpuv_1.6.6        pkgconfig_2.0.3     broom_1.0.1        
[25] haven_2.5.1         scales_1.2.1        whisker_0.4        
[28] later_1.3.0         tzdb_0.3.0          git2r_0.32.0       
[31] googledrive_2.0.0   farver_2.1.1        generics_0.1.3     
[34] ellipsis_0.3.2      cachem_1.0.6        withr_2.5.0        
[37] cli_3.4.1           magrittr_2.0.3      crayon_1.5.2       
[40] readxl_1.4.2        evaluate_0.17       fs_1.5.2           
[43] fansi_1.0.3         xml2_1.3.3          beeswarm_0.4.0     
[46] tools_4.2.1         hms_1.1.2           gargle_1.2.1       
[49] lifecycle_1.0.3     munsell_0.5.0       reprex_2.0.2       
[52] compiler_4.2.1      jquerylib_0.1.4     rlang_1.1.1        
[55] grid_4.2.1          rstudioapi_0.14     labeling_0.4.2     
[58] rmarkdown_2.17      gtable_0.3.1        curl_4.3.3         
[61] DBI_1.1.3           R6_2.5.1            knitr_1.42         
[64] fastmap_1.1.0       bit_4.0.4           utf8_1.2.2         
[67] workflowr_1.7.0     rprojroot_2.0.3     stringi_1.7.8      
[70] parallel_4.2.1      Rcpp_1.0.9          vctrs_0.6.2        
[73] dbplyr_2.2.1        tidyselect_1.2.0    xfun_0.39          </code></pre>
</div>
</div>
<div id="document-control" class="section level2">
<h2>Document Control</h2>
<p>This document was prepared using RMarkdown in RStudio.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb49"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb49-1"><a href="#cb49-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.6 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/liblapack.so.3

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] ggbeeswarm_0.7.2 MASS_7.3-57      cowplot_1.1.1    lubridate_1.8.0 
 [5] forcats_0.5.2    stringr_1.5.0    purrr_1.0.1      readr_2.1.3     
 [9] tibble_3.2.1     ggplot2_3.4.2    tidyverse_1.3.2  tidyr_1.3.0     
[13] dplyr_1.1.2     

loaded via a namespace (and not attached):
 [1] httr_1.4.4          sass_0.4.2          bit64_4.0.5        
 [4] vroom_1.6.0         jsonlite_1.8.3      modelr_0.1.9       
 [7] bslib_0.4.0         assertthat_0.2.1    highr_0.9          
[10] vipor_0.4.5         googlesheets4_1.0.1 cellranger_1.1.0   
[13] yaml_2.3.6          pillar_1.9.0        backports_1.4.1    
[16] glue_1.6.2          digest_0.6.30       promises_1.2.0.1   
[19] rvest_1.0.3         colorspace_2.0-3    htmltools_0.5.3    
[22] httpuv_1.6.6        pkgconfig_2.0.3     broom_1.0.1        
[25] haven_2.5.1         scales_1.2.1        whisker_0.4        
[28] later_1.3.0         tzdb_0.3.0          git2r_0.32.0       
[31] googledrive_2.0.0   farver_2.1.1        generics_0.1.3     
[34] ellipsis_0.3.2      cachem_1.0.6        withr_2.5.0        
[37] cli_3.4.1           magrittr_2.0.3      crayon_1.5.2       
[40] readxl_1.4.2        evaluate_0.17       fs_1.5.2           
[43] fansi_1.0.3         xml2_1.3.3          beeswarm_0.4.0     
[46] tools_4.2.1         hms_1.1.2           gargle_1.2.1       
[49] lifecycle_1.0.3     munsell_0.5.0       reprex_2.0.2       
[52] compiler_4.2.1      jquerylib_0.1.4     rlang_1.1.1        
[55] grid_4.2.1          rstudioapi_0.14     labeling_0.4.2     
[58] rmarkdown_2.17      gtable_0.3.1        curl_4.3.3         
[61] DBI_1.1.3           R6_2.5.1            knitr_1.42         
[64] fastmap_1.1.0       bit_4.0.4           utf8_1.2.2         
[67] workflowr_1.7.0     rprojroot_2.0.3     stringi_1.7.8      
[70] parallel_4.2.1      Rcpp_1.0.9          vctrs_0.6.2        
[73] dbplyr_2.2.1        tidyselect_1.2.0    xfun_0.39          </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
