<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>About</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>


<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Diversity Outbred Reference Traits Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">About</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-03-11
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rta/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version 1.7.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20200501code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20200501)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20200501code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20200501)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong8b820dc"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 8b820dc </a>
</p>
</div>
<div id="strongRepositoryversionstrong8b820dc" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 8b820dc. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    code/.Rapp.history
    Ignored:    data/.Rhistory
    Ignored:    data/RDS/
    Ignored:    run/

Untracked files:
    Untracked:  LICENSE
    Untracked:  Untitled.R
    Untracked:  analysis/17_mapping_interpretation_old.Rmd
    Untracked:  analysis/temp.txt
    Untracked:  code/DO_eQTL_perms.R
    Untracked:  code/DO_eQTL_score_calc.R
    Untracked:  code/STAR_RSEM_416.sh
    Untracked:  code/eQTL_plotting_DO_striatum_416.R
    Untracked:  code/ggplot_pub_themes.R
    Untracked:  data/20_elastic_mv_results.csv
    Untracked:  data/20_elasticnet_predictions.RDS
    Untracked:  data/DO_416_DGEList.RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 3).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org 4).RDS
    Untracked:  data/DO_416_DGEList_unfiltered (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_DGEList_unfiltered.RDS
    Untracked:  data/DO_416_ERCC_DGEList.RDS
    Untracked:  data/DO_416_ERCC_cpm.RDS
    Untracked:  data/DO_416_normrt.RDS
    Untracked:  data/DO_416_pheno.RDS
    Untracked:  data/DO_416_pheno_allgenes.RDS
    Untracked:  data/DO_416_voom (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom.RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org 2).RDS
    Untracked:  data/DO_416_voom_allgenes (michael.saul@jax.org).RDS
    Untracked:  data/DO_416_voom_allgenes.RDS
    Untracked:  data/DO_416_voom_cpresiduals.RDS
    Untracked:  data/DO_AQ_active_CoV_toptable_annotated.RDS
    Untracked:  data/DO_AQ_lever_ratio_toptable_annotated.RDS
    Untracked:  data/DO_AQ_mean_inf_toptable_annotated.RDS
    Untracked:  data/DO_AQ_rate_toptable_annotated.RDS
    Untracked:  data/DO_IVSA_acquisition_cancor_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_df.RDS
    Untracked:  data/DO_IVSA_acquisition_logit_glm.RDS
    Untracked:  data/DO_IVSA_all_cancor_df.RDS
    Untracked:  data/DO_IVSA_data.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_ALL.RDS
    Untracked:  data/DO_IVSA_novelty_cancor_AQ.RDS
    Untracked:  data/DO_IVSA_novelty_data.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_ACQUISITION_logit_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_ALL_cancor_old.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_AQ_cancor_2021-01-04.RDS
    Untracked:  data/DO_IVSA_prediction_a1_ACQUISITION_logit.RDS
    Untracked:  data/DO_IVSA_prediction_a2_AQ_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a3_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a4_male_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_prediction_a5_female_ALL_cancor.RDS
    Untracked:  data/DO_IVSA_raw_data.RDS
    Untracked:  data/DO_IVSA_transformed_data.RDS
    Untracked:  data/DO_LIMS_data.RDS
    Untracked:  data/DO_RTG_a1_new.qtl.RData
    Untracked:  data/DO_RTG_a1_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a1_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a2_new.qtl.RData
    Untracked:  data/DO_RTG_a2_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a2_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_a3_new.qtl.RData
    Untracked:  data/DO_RTG_a3_new_1000x.qtlpermu.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_01.RData
    Untracked:  data/DO_RTG_a3_new_mapping_1000x_02.RData
    Untracked:  data/DO_RTG_a4_new.qtl.RData
    Untracked:  data/DO_RTG_a5_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new.qtl.RData
    Untracked:  data/DO_RTG_elasticnet_new_mapping_1000x.RData
    Untracked:  data/DO_RTG_project_df_for_mapping.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_2020-11-06.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_a2.RDS
    Untracked:  data/DO_RTG_project_df_for_mapping_sexspecific.RDS
    Untracked:  data/DO_cocaine_measures_data.RDS
    Untracked:  data/DO_demographic_data_df.RDS
    Untracked:  data/DO_eigengene_significance.RDS
    Untracked:  data/DO_eigengenes.RDS
    Untracked:  data/DO_elasticnet_GEX_pvalue_histograms.RData
    Untracked:  data/DO_highcor_cpresiduals_datatable.RDS
    Untracked:  data/DO_holeboard_data_df.RDS
    Untracked:  data/DO_holeboard_summarized.RDS
    Untracked:  data/DO_lightdark_data_df.RDS
    Untracked:  data/DO_mQTL_peaks.RDS
    Untracked:  data/DO_mQTL_scan1s.RDS
    Untracked:  data/DO_module_membership.RDS
    Untracked:  data/DO_novelplace_data_df.RDS
    Untracked:  data/DO_novelty_COC_cancor.RDS
    Untracked:  data/DO_novelty_SAL_cancor.RDS
    Untracked:  data/DO_novelty_data.RDS
    Untracked:  data/DO_novelty_response_data.RDS
    Untracked:  data/DO_novelty_response_data_for_RTG.RDS
    Untracked:  data/DO_novelty_response_data_with_transform.csv
    Untracked:  data/DO_novelty_saline_cocaine_cancor.RDS
    Untracked:  data/DO_openfield_data_df.RDS
    Untracked:  data/DO_paraclique_data.el
    Untracked:  data/DO_paraclique_eigengenes.RDS
    Untracked:  data/DO_paraclique_significance_elasticnet.RDS
    Untracked:  data/DO_paracliques.RDS
    Untracked:  data/DO_paracliques_significance.RDS
    Untracked:  data/DO_ploidy_calls.RDS
    Untracked:  data/DO_pmap.RDS
    Untracked:  data/DO_pvalue_histograms.RData
    Untracked:  data/DO_rtg_a1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_a3_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m1_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m2_toptable_annotated.RDS
    Untracked:  data/DO_rtg_m3_toptable_annotated.RDS
    Untracked:  data/DO_saline_measures_data.RDS
    Untracked:  data/DO_str_2016_eQTL.RData
    Untracked:  data/DO_str_2016_gm4qtl2.zip
    Untracked:  data/DO_str_2016_gm4qtl2_intensities.fst
    Untracked:  data/DO_voom_cpresiduals_NArm.RDS
    Untracked:  data/DO_wgcna.RDS
    Untracked:  data/IVSA_RTG_a3_blup.pdf
    Untracked:  data/IVSA_h2_2019-11-18.tsv
    Untracked:  data/IVSA_metadata_sheet.RDS
    Untracked:  data/ME16_edges.tsv
    Untracked:  data/Mus_musculus.GRCm38.94.parsed.RDS
    Untracked:  data/Paraclique_19_MF_AMIGO.txt
    Untracked:  data/RTG_QTL_peaks.RDS
    Untracked:  data/RTG_figure.RDS
    Untracked:  data/SENS_RTG/
    Untracked:  data/Tyr2_genotypes.RDS
    Untracked:  data/Tyr_genotypes.RDS
    Untracked:  data/blup_chr7_DO_RTG_a3.RDS
    Untracked:  data/cc_variants/
    Untracked:  data/chr18_QTL_results.RDS
    Untracked:  data/chr4_QTL_results.RDS
    Untracked:  data/chr7_a3_QTL_results.RDS
    Untracked:  data/cis/
    Untracked:  data/eQTL_annotated_peaks.RDS
    Untracked:  data/eQTL_plots.RData
    Untracked:  data/figures/
    Untracked:  data/founders_key/
    Untracked:  data/gigamuga/
    Untracked:  data/gwas/
    Untracked:  data/info/
    Untracked:  data/m2G.a2.RTG.chr4snp.RData
    Untracked:  data/mapping_from_hao/
    Untracked:  data/mpd_measures_saline.txt
    Untracked:  data/mpd_measures_saline_cocaine.txt
    Untracked:  data/perms/
    Untracked:  data/qtl2/
    Untracked:  data/raw/
    Untracked:  data/resources/
    Untracked:  data/tads_loops/
    Untracked:  data/transcripts/
    Untracked:  data/variants/
    Untracked:  logs/
    Untracked:  new_RTG_a3.jpeg
    Untracked:  output/RTG_out/
    Untracked:  output/docs_backup_2020-07-30.tar.gz
    Untracked:  output/elastic_net_out/
    Untracked:  rstudio_rta.sif
    Untracked:  singularity/
    Untracked:  tmp/
    Untracked:  var/

Unstaged changes:
    Deleted:    .Rprofile
    Modified:   .gitignore
    Modified:   README.md
    Modified:   analysis/_site.yml

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/about.Rmd</code>) and HTML (<code>docs/about.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
8b820dc
</td>
<td>
Michael C. Saul
</td>
<td>
2022-03-11
</td>
<td>
Updating the Singularity definition file
</td>
</tr>
<tr>
<td>
html
</td>
<td>
30eb880
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
509338c
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-08
</td>
<td>
Adding list of packages
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
5309b17
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-08
</td>
<td>
Adding list of installed packages to the RMarkdown document
</td>
</tr>
<tr>
<td>
html
</td>
<td>
4360a14
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
32417e1
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
<td>
Committing change to Singularity def that fixed the argyle package
</td>
</tr>
<tr>
<td>
html
</td>
<td>
401dc6a
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
9325bf3
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
<td>
Recommitting to mention the Singularity registry at JAX
</td>
</tr>
<tr>
<td>
html
</td>
<td>
330c0ef
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
3adb32c
</td>
<td>
Michael C. Saul
</td>
<td>
2021-12-06
</td>
<td>
Initial commit on Sumner
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
c43e250
</td>
<td>
Michael C. Saul
</td>
<td>
2020-05-01
</td>
<td>
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>This project utilizes a container from the <a href="https://www.rocker-project.org/">Rocker project</a>. Upon publication, this Singularity container will be published on the <a href="https://jaxreg.jax.org/collections">JAX Singularity Registry</a>.</p>
<p>The container used here was compiled from the following definition file:</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="ex">bootstrap:</span> docker</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="ex">from:</span> rocker/rstudio:4.1.2</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="ex">%environment</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="ex">%post</span></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="ex">apt-get</span> <span class="at">-y</span> update</span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="ex">apt-get</span> <span class="at">-y</span> install curl</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="ex">apt-get</span> <span class="at">-y</span> install openssl</span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="fu">ln</span> <span class="at">-snf</span> /usr/share/zoneinfo/<span class="va">$(</span><span class="ex">curl</span> https://ipapi.co/timezone<span class="va">)</span> /etc/localtime</span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="ex">apt-get</span> <span class="at">-y</span> install git</span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a><span class="ex">apt-get</span> <span class="at">-y</span> install libgit2-dev</span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="ex">apt-get</span> <span class="at">-y</span> install build-essential</span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a><span class="ex">apt-get</span> <span class="at">-y</span> install libxml2-dev</span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a><span class="ex">apt-get</span> <span class="at">-y</span> install libpng-dev <span class="co"># Necessary for WGCNA</span></span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a><span class="co"># Installing R packages hosted from CRAN</span></span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;tidyverse&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;ggplot2&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;ggrepel&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;ggpubr&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;grid&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;gridExtra&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;gplots&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;cowplot&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;ggplotify&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;data.table&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;readxl&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;GGally&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;ggbeeswarm&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;RColorBrewer&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;corrplot&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;statmod&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;dplyr&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;glmnet&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;git2r&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;workflowr&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;qtl2&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;qtl2convert&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-41"><a href="#cb1-41" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;qtl2pleio&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-42"><a href="#cb1-42" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;broman&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-43"><a href="#cb1-43" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;fst&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-44"><a href="#cb1-44" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;glmnet&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-45"><a href="#cb1-45" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;nlme&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-46"><a href="#cb1-46" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;DiagrammeR&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-47"><a href="#cb1-47" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;knitr&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-48"><a href="#cb1-48" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;https://cran.rstudio.com/src/contrib/Rcpp_1.0.7.tar.gz&quot;, repos=NULL, type = &quot;source&quot;)&#39;</span></span>
<span id="cb1-49"><a href="#cb1-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-50"><a href="#cb1-50" aria-hidden="true" tabindex="-1"></a><span class="co"># Installing Bioconductor packages</span></span>
<span id="cb1-51"><a href="#cb1-51" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;BiocManager&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-52"><a href="#cb1-52" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;BiocManager::install(&quot;biomaRt&quot;)&#39;</span></span>
<span id="cb1-53"><a href="#cb1-53" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;BiocManager::install(&quot;GenomicRanges&quot;)&#39;</span></span>
<span id="cb1-54"><a href="#cb1-54" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;BiocManager::install(&quot;Gviz&quot;)&#39;</span></span>
<span id="cb1-55"><a href="#cb1-55" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;BiocManager::install(&quot;tximport&quot;)&#39;</span></span>
<span id="cb1-56"><a href="#cb1-56" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;BiocManager::install(&quot;limma&quot;)&#39;</span></span>
<span id="cb1-57"><a href="#cb1-57" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;BiocManager::install(&quot;edgeR&quot;)&#39;</span></span>
<span id="cb1-58"><a href="#cb1-58" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;BiocManager::install(&quot;qvalue&quot;)&#39;</span></span>
<span id="cb1-59"><a href="#cb1-59" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;BiocManager::install(&quot;preprocessCore&quot;)&#39;</span></span>
<span id="cb1-60"><a href="#cb1-60" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;BiocManager::install(&quot;impute&quot;)&#39;</span></span>
<span id="cb1-61"><a href="#cb1-61" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;BiocManager::install(&quot;GO.db&quot;)&#39;</span></span>
<span id="cb1-62"><a href="#cb1-62" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;BiocManager::install(&quot;AnnotationDbi&quot;)&#39;</span></span>
<span id="cb1-63"><a href="#cb1-63" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-64"><a href="#cb1-64" aria-hidden="true" tabindex="-1"></a><span class="co"># Installing WGCNA, which has multiple dependencies from Bioconductor</span></span>
<span id="cb1-65"><a href="#cb1-65" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;WGCNA&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-66"><a href="#cb1-66" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-67"><a href="#cb1-67" aria-hidden="true" tabindex="-1"></a><span class="co"># Installing devtools packages</span></span>
<span id="cb1-68"><a href="#cb1-68" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;roxygen2&quot;, repos=&quot;http://cran.rstudio.com&quot;, type=&quot;source&quot;)&#39;</span></span>
<span id="cb1-69"><a href="#cb1-69" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;rversions&quot;, repos=&quot;http://cran.rstudio.com&quot;, type=&quot;source&quot;)&#39;</span></span>
<span id="cb1-70"><a href="#cb1-70" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;install.packages(&quot;devtools&quot;, repos=&quot;http://cran.rstudio.com&quot;)&#39;</span></span>
<span id="cb1-71"><a href="#cb1-71" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;devtools::install_github(&quot;jtliso/cliqueR&quot;)&#39;</span></span>
<span id="cb1-72"><a href="#cb1-72" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;devtools::install_github(&quot;msaul/msaul&quot;)&#39;</span></span>
<span id="cb1-73"><a href="#cb1-73" aria-hidden="true" tabindex="-1"></a><span class="ex">R</span> <span class="at">--slave</span> <span class="at">-e</span> <span class="st">&#39;devtools::install_github(&quot;andrewparkermorgan/argyle&quot;)&#39;</span></span>
<span id="cb1-74"><a href="#cb1-74" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-75"><a href="#cb1-75" aria-hidden="true" tabindex="-1"></a><span class="ex">%labels</span></span>
<span id="cb1-76"><a href="#cb1-76" aria-hidden="true" tabindex="-1"></a>    <span class="ex">Author</span> Michael C. Saul, michael.saul [at] jax.org</span>
<span id="cb1-77"><a href="#cb1-77" aria-hidden="true" tabindex="-1"></a>    <span class="ex">Version</span> v2022-01-31</span>
<span id="cb1-78"><a href="#cb1-78" aria-hidden="true" tabindex="-1"></a>    <span class="ex">MyLabel</span> Reference Trait Analysis Singularity Container</span>
<span id="cb1-79"><a href="#cb1-79" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-80"><a href="#cb1-80" aria-hidden="true" tabindex="-1"></a><span class="ex">%help</span></span>
<span id="cb1-81"><a href="#cb1-81" aria-hidden="true" tabindex="-1"></a>    <span class="ex">RStudio</span> Singularity Container for Reference Trait Analysis</span>
<span id="cb1-82"><a href="#cb1-82" aria-hidden="true" tabindex="-1"></a>    <span class="ex">2021-12-09</span></span>
<span id="cb1-83"><a href="#cb1-83" aria-hidden="true" tabindex="-1"></a>    <span class="ex">Singularity</span> Container Complied By: Michael C. Saul</span>
<span id="cb1-84"><a href="#cb1-84" aria-hidden="true" tabindex="-1"></a>    <span class="ex">Docker</span> Repository: rocker/rstudio:4.1.2</span>
<span id="cb1-85"><a href="#cb1-85" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-86"><a href="#cb1-86" aria-hidden="true" tabindex="-1"></a>    <span class="ex">This</span> container derives from the RStudio Rocker container for R v4.1.2 as it existed on 2022-01-31.</span>
<span id="cb1-87"><a href="#cb1-87" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-88"><a href="#cb1-88" aria-hidden="true" tabindex="-1"></a>    <span class="ex">This</span> container was compiled on 2022-01-31 by Michael C. Saul.</span></code></pre></div>
<p>A list of the installed packages is below.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># From user hc_haha</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="co"># https://stackoverflow.com/questions/38481980/get-the-list-of-installed-packages-by-user-in-r</span></span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a>pkg <span class="ot">=</span> tibble<span class="sc">::</span><span class="fu">tibble</span>(</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">Package =</span> <span class="fu">names</span>(<span class="fu">installed.packages</span>()[,<span class="dv">3</span>]),</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">Version =</span> <span class="fu">unname</span>(<span class="fu">installed.packages</span>()[,<span class="dv">3</span>])</span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>pkg</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["Package"],"name":[1],"type":["chr"],"align":["left"]},{"label":["Version"],"name":[2],"type":["chr"],"align":["left"]}],"data":[{"1":"markdown","2":"1.1"},{"1":"abind","2":"1.4-5"},{"1":"AnnotationDbi","2":"1.56.2"},{"1":"AnnotationFilter","2":"1.18.0"},{"1":"argyle","2":"0.2.2"},{"1":"askpass","2":"1.1"},{"1":"assertthat","2":"0.2.1"},{"1":"backports","2":"1.4.1"},{"1":"base64enc","2":"0.1-3"},{"1":"beeswarm","2":"0.4.0"},{"1":"BH","2":"1.78.0-0"},{"1":"Biobase","2":"2.54.0"},{"1":"BiocFileCache","2":"2.2.1"},{"1":"BiocGenerics","2":"0.40.0"},{"1":"BiocIO","2":"1.4.0"},{"1":"BiocManager","2":"1.30.16"},{"1":"BiocParallel","2":"1.28.3"},{"1":"BiocVersion","2":"3.14.0"},{"1":"biomaRt","2":"2.50.2"},{"1":"Biostrings","2":"2.62.0"},{"1":"bit","2":"4.0.4"},{"1":"bit64","2":"4.0.5"},{"1":"bitops","2":"1.0-7"},{"1":"blob","2":"1.2.2"},{"1":"brew","2":"1.0-6"},{"1":"brio","2":"1.1.3"},{"1":"broman","2":"0.76"},{"1":"broom","2":"0.7.12"},{"1":"cachem","2":"1.0.6"},{"1":"callr","2":"3.7.0"},{"1":"carData","2":"3.0-5"},{"1":"caTools","2":"1.18.2"},{"1":"cellranger","2":"1.1.0"},{"1":"checkmate","2":"2.0.0"},{"1":"cli","2":"3.1.1"},{"1":"clipr","2":"0.7.1"},{"1":"cliqueR","2":"1.0"},{"1":"colorspace","2":"2.0-2"},{"1":"commonmark","2":"1.7"},{"1":"corrplot","2":"0.92"},{"1":"cowplot","2":"1.1.1"},{"1":"cpp11","2":"0.4.2"},{"1":"crayon","2":"1.4.2"},{"1":"credentials","2":"1.3.2"},{"1":"curl","2":"4.3.2"},{"1":"data.table","2":"1.14.2"},{"1":"DBI","2":"1.1.2"},{"1":"dbplyr","2":"2.1.1"},{"1":"DelayedArray","2":"0.20.0"},{"1":"desc","2":"1.4.0"},{"1":"devtools","2":"2.4.3"},{"1":"DiagrammeR","2":"1.0.8"},{"1":"dichromat","2":"2.0-0"},{"1":"diffobj","2":"0.3.5"},{"1":"digest","2":"0.6.29"},{"1":"docopt","2":"0.7.1"},{"1":"doParallel","2":"1.0.16"},{"1":"downloader","2":"0.4"},{"1":"dplyr","2":"1.0.7"},{"1":"dtplyr","2":"1.2.1"},{"1":"dynamicTreeCut","2":"1.63-1"},{"1":"edgeR","2":"3.36.0"},{"1":"ellipsis","2":"0.3.2"},{"1":"evaluate","2":"0.14"},{"1":"fansi","2":"1.0.2"},{"1":"farver","2":"2.1.0"},{"1":"fastcluster","2":"1.2.3"},{"1":"fastmap","2":"1.1.0"},{"1":"filelock","2":"1.0.2"},{"1":"forcats","2":"0.5.1"},{"1":"foreach","2":"1.5.1"},{"1":"formatR","2":"1.11"},{"1":"Formula","2":"1.2-4"},{"1":"fs","2":"1.5.2"},{"1":"fst","2":"0.9.4"},{"1":"futile.logger","2":"1.4.3"},{"1":"futile.options","2":"1.0.1"},{"1":"gargle","2":"1.2.0"},{"1":"gemma2","2":"0.1.3"},{"1":"generics","2":"0.1.1"},{"1":"GenomeInfoDb","2":"1.30.1"},{"1":"GenomeInfoDbData","2":"1.2.7"},{"1":"GenomicRanges","2":"1.46.1"},{"1":"gert","2":"1.5.0"},{"1":"getPass","2":"0.2-2"},{"1":"GGally","2":"2.1.2"},{"1":"ggbeeswarm","2":"0.6.0"},{"1":"ggplot2","2":"3.3.5"},{"1":"ggplotify","2":"0.1.0"},{"1":"ggrepel","2":"0.9.1"},{"1":"ggsci","2":"2.9"},{"1":"ggsignif","2":"0.6.3"},{"1":"gh","2":"1.3.0"},{"1":"git2r","2":"0.29.0"},{"1":"gitcreds","2":"0.1.1"},{"1":"glmnet","2":"4.1-3"},{"1":"glue","2":"1.6.1"},{"1":"GO.db","2":"3.14.0"},{"1":"googledrive","2":"2.0.0"},{"1":"googlesheets4","2":"1.0.0"},{"1":"gplots","2":"3.1.1"},{"1":"gridExtra","2":"2.3"},{"1":"gridGraphics","2":"0.5-1"},{"1":"gtable","2":"0.3.0"},{"1":"gtools","2":"3.9.2"},{"1":"haven","2":"2.4.3"},{"1":"highr","2":"0.9"},{"1":"Hmisc","2":"4.6-0"},{"1":"hms","2":"1.1.1"},{"1":"htmlTable","2":"2.4.0"},{"1":"htmltools","2":"0.5.2"},{"1":"htmlwidgets","2":"1.5.4"},{"1":"httpuv","2":"1.6.5"},{"1":"httr","2":"1.4.2"},{"1":"ids","2":"1.0.1"},{"1":"igraph","2":"1.2.11"},{"1":"impute","2":"1.68.0"},{"1":"influenceR","2":"0.1.0.1"},{"1":"ini","2":"0.3.1"},{"1":"IRanges","2":"2.28.0"},{"1":"isoband","2":"0.2.5"},{"1":"iterators","2":"1.0.13"},{"1":"jpeg","2":"0.1-9"},{"1":"jquerylib","2":"0.1.4"},{"1":"jsonlite","2":"1.7.3"},{"1":"KEGGREST","2":"1.34.0"},{"1":"knitr","2":"1.37"},{"1":"labeling","2":"0.4.2"},{"1":"lambda.r","2":"1.2.4"},{"1":"later","2":"1.3.0"},{"1":"latticeExtra","2":"0.6-29"},{"1":"lazyeval","2":"0.2.2"},{"1":"lifecycle","2":"1.0.1"},{"1":"limma","2":"3.50.0"},{"1":"littler","2":"0.3.15"},{"1":"locfit","2":"1.5-9.4"},{"1":"lubridate","2":"1.8.0"},{"1":"magrittr","2":"2.0.2"},{"1":"maptools","2":"1.1-2"},{"1":"MatrixGenerics","2":"1.6.0"},{"1":"MatrixModels","2":"0.5-0"},{"1":"matrixStats","2":"0.61.0"},{"1":"memoise","2":"2.0.1"},{"1":"mime","2":"0.12"},{"1":"minqa","2":"1.2.4"},{"1":"modelr","2":"0.1.8"},{"1":"msaul","2":"1.16.6"},{"1":"munsell","2":"0.5.0"},{"1":"mvtnorm","2":"1.1-3"},{"1":"nlme","2":"3.1-155"},{"1":"numDeriv","2":"2016.8-1.1"},{"1":"openssl","2":"1.4.6"},{"1":"pillar","2":"1.6.5"},{"1":"pkgbuild","2":"1.3.1"},{"1":"pkgconfig","2":"2.0.3"},{"1":"pkgload","2":"1.2.4"},{"1":"plogr","2":"0.2.0"},{"1":"plyr","2":"1.8.6"},{"1":"png","2":"0.1-7"},{"1":"polynom","2":"1.4-0"},{"1":"praise","2":"1.0.0"},{"1":"preprocessCore","2":"1.56.0"},{"1":"prettyunits","2":"1.1.1"},{"1":"processx","2":"3.5.2"},{"1":"progress","2":"1.2.2"},{"1":"promises","2":"1.2.0.1"},{"1":"ProtGenerics","2":"1.26.0"},{"1":"ps","2":"1.6.0"},{"1":"purrr","2":"0.3.4"},{"1":"qtl","2":"1.50"},{"1":"qtl2","2":"0.28"},{"1":"qtl2convert","2":"0.26"},{"1":"qtl2pleio","2":"1.4.3"},{"1":"quantreg","2":"5.87"},{"1":"qvalue","2":"2.26.0"},{"1":"R6","2":"2.5.1"},{"1":"rappdirs","2":"0.3.3"},{"1":"rcmdcheck","2":"1.4.0"},{"1":"RColorBrewer","2":"1.1-2"},{"1":"Rcpp","2":"1.0.8"},{"1":"RcppEigen","2":"0.3.3.9.1"},{"1":"RCurl","2":"1.98-1.5"},{"1":"readr","2":"2.1.2"},{"1":"readxl","2":"1.3.1"},{"1":"rematch","2":"1.0.1"},{"1":"rematch2","2":"2.1.2"},{"1":"remotes","2":"2.4.2"},{"1":"reprex","2":"2.0.1"},{"1":"reshape","2":"0.8.8"},{"1":"reshape2","2":"1.4.4"},{"1":"restfulr","2":"0.0.13"},{"1":"rjson","2":"0.2.21"},{"1":"rlang","2":"1.0.0"},{"1":"rmarkdown","2":"2.11"},{"1":"roxygen2","2":"7.1.2"},{"1":"rprojroot","2":"2.0.2"},{"1":"RSQLite","2":"2.2.9"},{"1":"rstudioapi","2":"0.13"},{"1":"rversions","2":"2.1.1"},{"1":"rvest","2":"1.0.2"},{"1":"S4Vectors","2":"0.32.3"},{"1":"scales","2":"1.1.1"},{"1":"selectr","2":"0.4-2"},{"1":"sessioninfo","2":"1.2.2"},{"1":"shape","2":"1.4.6"},{"1":"snow","2":"0.4-4"},{"1":"sp","2":"1.4-6"},{"1":"SparseM","2":"1.81"},{"1":"statmod","2":"1.4.36"},{"1":"stringi","2":"1.7.6"},{"1":"stringr","2":"1.4.0"},{"1":"SummarizedExperiment","2":"1.24.0"},{"1":"sys","2":"3.4"},{"1":"testthat","2":"3.1.2"},{"1":"tibble","2":"3.1.6"},{"1":"tidyr","2":"1.1.4"},{"1":"tidyselect","2":"1.1.1"},{"1":"tidyverse","2":"1.3.1"},{"1":"tinytex","2":"0.36"},{"1":"tximport","2":"1.22.0"},{"1":"tzdb","2":"0.2.0"},{"1":"usethis","2":"2.1.5"},{"1":"utf8","2":"1.2.2"},{"1":"uuid","2":"1.0-3"},{"1":"vctrs","2":"0.3.8"},{"1":"vipor","2":"0.4.5"},{"1":"viridis","2":"0.6.2"},{"1":"viridisLite","2":"0.4.0"},{"1":"visNetwork","2":"2.1.0"},{"1":"vroom","2":"1.5.7"},{"1":"waldo","2":"0.3.1"},{"1":"WGCNA","2":"1.70-3"},{"1":"whisker","2":"0.4"},{"1":"withr","2":"2.4.3"},{"1":"workflowr","2":"1.7.0"},{"1":"xfun","2":"0.29"},{"1":"XML","2":"3.99-0.8"},{"1":"xml2","2":"1.3.3"},{"1":"xopen","2":"1.0.0"},{"1":"XVector","2":"0.34.0"},{"1":"yaml","2":"2.2.2"},{"1":"yulab.utils","2":"0.0.4"},{"1":"zip","2":"2.2.0"},{"1":"zlibbioc","2":"1.40.0"},{"1":"base","2":"4.1.2"},{"1":"boot","2":"1.3-28"},{"1":"class","2":"7.3-19"},{"1":"cluster","2":"2.1.2"},{"1":"codetools","2":"0.2-18"},{"1":"compiler","2":"4.1.2"},{"1":"datasets","2":"4.1.2"},{"1":"foreign","2":"0.8-81"},{"1":"graphics","2":"4.1.2"},{"1":"grDevices","2":"4.1.2"},{"1":"grid","2":"4.1.2"},{"1":"KernSmooth","2":"2.23-20"},{"1":"lattice","2":"0.20-45"},{"1":"MASS","2":"7.3-54"},{"1":"Matrix","2":"1.3-4"},{"1":"methods","2":"4.1.2"},{"1":"mgcv","2":"1.8-38"},{"1":"nlme","2":"3.1-153"},{"1":"nnet","2":"7.3-16"},{"1":"parallel","2":"4.1.2"},{"1":"rpart","2":"4.1-15"},{"1":"spatial","2":"7.3-14"},{"1":"splines","2":"4.1.2"},{"1":"stats","2":"4.1.2"},{"1":"stats4","2":"4.1.2"},{"1":"survival","2":"3.2-13"},{"1":"tcltk","2":"4.1.2"},{"1":"tools","2":"4.1.2"},{"1":"utils","2":"4.1.2"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.3 LTS

Matrix products: default
BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.8       rstudioapi_0.13  whisker_0.4      knitr_1.37      
 [5] magrittr_2.0.2   workflowr_1.7.0  R6_2.5.1         rlang_1.0.0     
 [9] fastmap_1.1.0    fansi_1.0.2      stringr_1.4.0    tools_4.1.2     
[13] xfun_0.29        utf8_1.2.2       cli_3.1.1        git2r_0.29.0    
[17] jquerylib_0.1.4  htmltools_0.5.2  ellipsis_0.3.2   rprojroot_2.0.2 
[21] yaml_2.2.2       digest_0.6.29    tibble_3.1.6     lifecycle_1.0.1 
[25] crayon_1.4.2     later_1.3.0      vctrs_0.3.8      fs_1.5.2        
[29] promises_1.2.0.1 glue_1.6.1       evaluate_0.14    rmarkdown_2.11  
[33] stringi_1.7.6    compiler_4.1.2   pillar_1.6.5     jsonlite_1.7.3  
[37] httpuv_1.6.5     pkgconfig_2.0.3 </code></pre>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
